<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!-- Generated by Apache Maven Doxia at Jan 30, 2015 -->
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
    <title>FindBugs Bug Detector Report</title>
    <style type="text/css" media="all">
      @import url("./css/maven-base.css");
      @import url("./css/maven-theme.css");
      @import url("./css/site.css");
    </style>
    <link rel="stylesheet" href="./css/print.css" type="text/css" media="print" />
    <meta name="Date-Revision-yyyymmdd" content="20150130" />
    <meta http-equiv="Content-Language" content="en" />
        
        </head>
  <body class="composite">
    <div id="banner">
                    <div id="bannerLeft">
                biojava-genome
                </div>
                    <div class="clear">
        <hr/>
      </div>
    </div>
    <div id="breadcrumbs">
            
        
                <div class="xleft">
        <span id="publishDate">Last Published: 2015-01-30</span>
                  &nbsp;| <span id="projectVersion">Version: 4.0.0</span>
                      </div>
            <div class="xright">                    <a href="./" title="biojava-genome">biojava-genome</a>
              
        
      </div>
      <div class="clear">
        <hr/>
      </div>
    </div>
    <div id="leftColumn">
      <div id="navcolumn">
             
        
                                <h5>Parent Project</h5>
                  <ul>
                  <li class="none">
                          <a href="../index.html" title="biojava">biojava</a>
            </li>
          </ul>
                       <h5>Project Documentation</h5>
                  <ul>
                                                                                                                                                                                <li class="collapsed">
                          <a href="project-info.html" title="Project Information">Project Information</a>
                  </li>
                                                                                                                                    <li class="expanded">
                          <a href="project-reports.html" title="Project Reports">Project Reports</a>
                    <ul>
                      <li class="none">
            <strong>FindBugs Report</strong>
          </li>
                      <li class="none">
                          <a href="checkstyle.html" title="Checkstyle">Checkstyle</a>
            </li>
                      <li class="none">
                          <a href="xref-test/index.html" title="Test Source Xref">Test Source Xref</a>
            </li>
                      <li class="none">
                          <a href="xref/index.html" title="Source Xref">Source Xref</a>
            </li>
              </ul>
        </li>
          </ul>
                             <a href="http://maven.apache.org/" title="Built by Maven" class="poweredBy">
        <img class="poweredBy" alt="Built by Maven" src="./images/logos/maven-feather.png" />
      </a>
                   
        
            </div>
    </div>
    <div id="bodyColumn">
      <div id="contentBox">
        <div class="section"><h2>FindBugs Bug Detector Report<a name="FindBugs_Bug_Detector_Report"></a></h2><p>The following document contains the results of <a class="externalLink" href="http://findbugs.sourceforge.net">FindBugs Report</a></p><p>FindBugs Version is <i>2.0.2</i></p><p>Threshold is <i>medium</i></p><p>Effort is <i>min</i></p></div><div class="section"><h2>Summary<a name="Summary"></a></h2><table border="0" class="bodyTable"><tr class="a"><th>Classes</th><th>Bugs</th><th>Errors</th><th>Missing Classes</th></tr><tr class="b"><td>31</td><td>51</td><td>0</td><td>0</td></tr></table></div><div class="section"><h2>Files<a name="Files"></a></h2><table border="0" class="bodyTable"><tr class="a"><th>Class</th><th>Bugs</th></tr><tr class="b"><td><a href="#org.biojava.nbio.genome.GeneFeatureHelper">org.biojava.nbio.genome.GeneFeatureHelper</a></td><td>3</td></tr><tr class="a"><td><a href="#org.biojava.nbio.genome.homology.BlastHomologyHits">org.biojava.nbio.genome.homology.BlastHomologyHits</a></td><td>1</td></tr><tr class="b"><td><a href="#org.biojava.nbio.genome.homology.GFF3FromUniprotBlastHits">org.biojava.nbio.genome.homology.GFF3FromUniprotBlastHits</a></td><td>8</td></tr><tr class="a"><td><a href="#org.biojava.nbio.genome.parsers.cytoband.Cytoband">org.biojava.nbio.genome.parsers.cytoband.Cytoband</a></td><td>1</td></tr><tr class="b"><td><a href="#org.biojava.nbio.genome.parsers.cytoband.CytobandParser">org.biojava.nbio.genome.parsers.cytoband.CytobandParser</a></td><td>1</td></tr><tr class="a"><td><a href="#org.biojava.nbio.genome.parsers.geneid.GeneIDXMLReader">org.biojava.nbio.genome.parsers.geneid.GeneIDXMLReader</a></td><td>2</td></tr><tr class="b"><td><a href="#org.biojava.nbio.genome.parsers.genename.GeneChromosomePosition">org.biojava.nbio.genome.parsers.genename.GeneChromosomePosition</a></td><td>1</td></tr><tr class="a"><td><a href="#org.biojava.nbio.genome.parsers.genename.GeneChromosomePositionParser">org.biojava.nbio.genome.parsers.genename.GeneChromosomePositionParser</a></td><td>1</td></tr><tr class="b"><td><a href="#org.biojava.nbio.genome.parsers.genename.GeneName">org.biojava.nbio.genome.parsers.genename.GeneName</a></td><td>3</td></tr><tr class="a"><td><a href="#org.biojava.nbio.genome.parsers.genename.GeneNamesParser">org.biojava.nbio.genome.parsers.genename.GeneNamesParser</a></td><td>1</td></tr><tr class="b"><td><a href="#org.biojava.nbio.genome.parsers.gff.FeatureList">org.biojava.nbio.genome.parsers.gff.FeatureList</a></td><td>1</td></tr><tr class="a"><td><a href="#org.biojava.nbio.genome.parsers.gff.FeatureList$FeatureComparator">org.biojava.nbio.genome.parsers.gff.FeatureList$FeatureComparator</a></td><td>1</td></tr><tr class="b"><td><a href="#org.biojava.nbio.genome.parsers.gff.GFF3Reader">org.biojava.nbio.genome.parsers.gff.GFF3Reader</a></td><td>2</td></tr><tr class="a"><td><a href="#org.biojava.nbio.genome.parsers.gff.GFF3Writer">org.biojava.nbio.genome.parsers.gff.GFF3Writer</a></td><td>5</td></tr><tr class="b"><td><a href="#org.biojava.nbio.genome.parsers.gff.GeneIDGFF2Reader">org.biojava.nbio.genome.parsers.gff.GeneIDGFF2Reader</a></td><td>3</td></tr><tr class="a"><td><a href="#org.biojava.nbio.genome.parsers.gff.GeneMarkGTFReader">org.biojava.nbio.genome.parsers.gff.GeneMarkGTFReader</a></td><td>2</td></tr><tr class="b"><td><a href="#org.biojava.nbio.genome.parsers.gff.Location">org.biojava.nbio.genome.parsers.gff.Location</a></td><td>5</td></tr><tr class="a"><td><a href="#org.biojava.nbio.genome.parsers.twobit.TwoBitParser">org.biojava.nbio.genome.parsers.twobit.TwoBitParser</a></td><td>4</td></tr><tr class="b"><td><a href="#org.biojava.nbio.genome.uniprot.UniprotToFasta">org.biojava.nbio.genome.uniprot.UniprotToFasta</a></td><td>3</td></tr><tr class="a"><td><a href="#org.biojava.nbio.genome.util.SplitFasta">org.biojava.nbio.genome.util.SplitFasta</a></td><td>3</td></tr></table></div><a name="org.biojava.nbio.genome.GeneFeatureHelper"></a><div class="section"><h3>org.biojava.nbio.genome.GeneFeatureHelper<a name="org.biojava.nbio.genome.GeneFeatureHelper"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Found reliance on default encoding in org.biojava.nbio.genome.GeneFeatureHelper.outputFastaSequenceLengthGFF3(File, File): new java.io.FileWriter(File)</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/genome/GeneFeatureHelper.html#171">171</a></td><td>High</td></tr><tr class="b"><td>org.biojava.nbio.genome.GeneFeatureHelper.getChromosomeSequenceFromDNASequence(LinkedHashMap) makes inefficient use of keySet iterator instead of entrySet iterator</td><td>PERFORMANCE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#WMI_WRONG_MAP_ITERATOR">WMI_WRONG_MAP_ITERATOR</a></td><td><a href="./xref/org/biojava/nbio/genome/GeneFeatureHelper.html#323">323</a></td><td>Medium</td></tr><tr class="a"><td>org.biojava.nbio.genome.GeneFeatureHelper.loadFastaAddGeneFeaturesFromUpperCaseExonFastaFile(File, File, boolean) makes inefficient use of keySet iterator instead of entrySet iterator</td><td>PERFORMANCE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#WMI_WRONG_MAP_ITERATOR">WMI_WRONG_MAP_ITERATOR</a></td><td><a href="./xref/org/biojava/nbio/genome/GeneFeatureHelper.html#52">52</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.genome.homology.BlastHomologyHits"></a><div class="section"><h3>org.biojava.nbio.genome.homology.BlastHomologyHits<a name="org.biojava.nbio.genome.homology.BlastHomologyHits"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>org.biojava.nbio.genome.homology.BlastHomologyHits.getMatches(File, double) makes inefficient use of keySet iterator instead of entrySet iterator</td><td>PERFORMANCE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#WMI_WRONG_MAP_ITERATOR">WMI_WRONG_MAP_ITERATOR</a></td><td><a href="./xref/org/biojava/nbio/genome/homology/BlastHomologyHits.html#46">46</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.genome.homology.GFF3FromUniprotBlastHits"></a><div class="section"><h3>org.biojava.nbio.genome.homology.GFF3FromUniprotBlastHits<a name="org.biojava.nbio.genome.homology.GFF3FromUniprotBlastHits"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Hard coded reference to an absolute pathname in org.biojava.nbio.genome.homology.GFF3FromUniprotBlastHits.main(String[])</td><td>STYLE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DMI_HARDCODED_ABSOLUTE_FILENAME">DMI_HARDCODED_ABSOLUTE_FILENAME</a></td><td><a href="./xref/org/biojava/nbio/genome/homology/GFF3FromUniprotBlastHits.html#297">297</a></td><td>Medium</td></tr><tr class="b"><td>Hard coded reference to an absolute pathname in org.biojava.nbio.genome.homology.GFF3FromUniprotBlastHits.main(String[])</td><td>STYLE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DMI_HARDCODED_ABSOLUTE_FILENAME">DMI_HARDCODED_ABSOLUTE_FILENAME</a></td><td><a href="./xref/org/biojava/nbio/genome/homology/GFF3FromUniprotBlastHits.html#297">297</a></td><td>Medium</td></tr><tr class="a"><td>Hard coded reference to an absolute pathname in org.biojava.nbio.genome.homology.GFF3FromUniprotBlastHits.main(String[])</td><td>STYLE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DMI_HARDCODED_ABSOLUTE_FILENAME">DMI_HARDCODED_ABSOLUTE_FILENAME</a></td><td><a href="./xref/org/biojava/nbio/genome/homology/GFF3FromUniprotBlastHits.html#300">300</a></td><td>Medium</td></tr><tr class="b"><td>Found reliance on default encoding in org.biojava.nbio.genome.homology.GFF3FromUniprotBlastHits.process(LinkedHashMap, LinkedHashMap, OutputStream): String.getBytes()</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/genome/homology/GFF3FromUniprotBlastHits.html#261">261</a></td><td>High</td></tr><tr class="a"><td>Suspicious comparison of Integer references in org.biojava.nbio.genome.homology.GFF3FromUniprotBlastHits.process(LinkedHashMap, LinkedHashMap, OutputStream)</td><td>CORRECTNESS</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#RC_REF_COMPARISON">RC_REF_COMPARISON</a></td><td><a href="./xref/org/biojava/nbio/genome/homology/GFF3FromUniprotBlastHits.html#145">145</a></td><td>High</td></tr><tr class="b"><td>Exception is caught when Exception is not thrown in org.biojava.nbio.genome.homology.GFF3FromUniprotBlastHits.process(LinkedHashMap, LinkedHashMap, OutputStream)</td><td>STYLE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#REC_CATCH_EXCEPTION">REC_CATCH_EXCEPTION</a></td><td><a href="./xref/org/biojava/nbio/genome/homology/GFF3FromUniprotBlastHits.html#265">265</a></td><td>Medium</td></tr><tr class="a"><td>org.biojava.nbio.genome.homology.GFF3FromUniprotBlastHits.process(LinkedHashMap, LinkedHashMap, OutputStream) concatenates strings using + in a loop</td><td>PERFORMANCE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#SBSC_USE_STRINGBUFFER_CONCATENATION">SBSC_USE_STRINGBUFFER_CONCATENATION</a></td><td><a href="./xref/org/biojava/nbio/genome/homology/GFF3FromUniprotBlastHits.html#97">97</a></td><td>Medium</td></tr><tr class="b"><td>org.biojava.nbio.genome.homology.GFF3FromUniprotBlastHits.process(LinkedHashMap, LinkedHashMap, OutputStream) makes inefficient use of keySet iterator instead of entrySet iterator</td><td>PERFORMANCE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#WMI_WRONG_MAP_ITERATOR">WMI_WRONG_MAP_ITERATOR</a></td><td><a href="./xref/org/biojava/nbio/genome/homology/GFF3FromUniprotBlastHits.html#82">82</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.genome.parsers.cytoband.Cytoband"></a><div class="section"><h3>org.biojava.nbio.genome.parsers.cytoband.Cytoband<a name="org.biojava.nbio.genome.parsers.cytoband.Cytoband"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>org.biojava.nbio.genome.parsers.cytoband.Cytoband defines compareTo(Cytoband) and uses Object.equals()</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#EQ_COMPARETO_USE_OBJECT_EQUALS">EQ_COMPARETO_USE_OBJECT_EQUALS</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/cytoband/Cytoband.html#94">94-112</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.genome.parsers.cytoband.CytobandParser"></a><div class="section"><h3>org.biojava.nbio.genome.parsers.cytoband.CytobandParser<a name="org.biojava.nbio.genome.parsers.cytoband.CytobandParser"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>Found reliance on default encoding in org.biojava.nbio.genome.parsers.cytoband.CytobandParser.getAllCytobands(InputStream): new java.io.InputStreamReader(InputStream)</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/cytoband/CytobandParser.html#77">77</a></td><td>High</td></tr></table></div><a name="org.biojava.nbio.genome.parsers.geneid.GeneIDXMLReader"></a><div class="section"><h3>org.biojava.nbio.genome.parsers.geneid.GeneIDXMLReader<a name="org.biojava.nbio.genome.parsers.geneid.GeneIDXMLReader"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>Hard coded reference to an absolute pathname in org.biojava.nbio.genome.parsers.geneid.GeneIDXMLReader.main(String[])</td><td>STYLE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DMI_HARDCODED_ABSOLUTE_FILENAME">DMI_HARDCODED_ABSOLUTE_FILENAME</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/geneid/GeneIDXMLReader.html#95">95</a></td><td>Medium</td></tr><tr class="a"><td>Hard coded reference to an absolute pathname in org.biojava.nbio.genome.parsers.geneid.GeneIDXMLReader.main(String[])</td><td>STYLE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DMI_HARDCODED_ABSOLUTE_FILENAME">DMI_HARDCODED_ABSOLUTE_FILENAME</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/geneid/GeneIDXMLReader.html#98">98</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.genome.parsers.genename.GeneChromosomePosition"></a><div class="section"><h3>org.biojava.nbio.genome.parsers.genename.GeneChromosomePosition<a name="org.biojava.nbio.genome.parsers.genename.GeneChromosomePosition"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>org.biojava.nbio.genome.parsers.genename.GeneChromosomePosition defines compareTo(GeneChromosomePosition) and uses Object.equals()</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#EQ_COMPARETO_USE_OBJECT_EQUALS">EQ_COMPARETO_USE_OBJECT_EQUALS</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#158">158</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.genome.parsers.genename.GeneChromosomePositionParser"></a><div class="section"><h3>org.biojava.nbio.genome.parsers.genename.GeneChromosomePositionParser<a name="org.biojava.nbio.genome.parsers.genename.GeneChromosomePositionParser"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Found reliance on default encoding in org.biojava.nbio.genome.parsers.genename.GeneChromosomePositionParser.getChromosomeMappings(InputStream): new java.io.InputStreamReader(InputStream)</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/genename/GeneChromosomePositionParser.html#80">80</a></td><td>High</td></tr></table></div><a name="org.biojava.nbio.genome.parsers.genename.GeneName"></a><div class="section"><h3>org.biojava.nbio.genome.parsers.genename.GeneName<a name="org.biojava.nbio.genome.parsers.genename.GeneName"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>org.biojava.nbio.genome.parsers.genename.GeneName defines compareTo(GeneName) and uses Object.equals()</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#EQ_COMPARETO_USE_OBJECT_EQUALS">EQ_COMPARETO_USE_OBJECT_EQUALS</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/genename/GeneName.html#136">136</a></td><td>Medium</td></tr><tr class="b"><td>org.biojava.nbio.genome.parsers.genename.GeneName defines equals(GeneName) method and uses Object.equals(Object)</td><td>CORRECTNESS</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#EQ_SELF_USE_OBJECT">EQ_SELF_USE_OBJECT</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/genename/GeneName.html#140">140</a></td><td>Medium</td></tr><tr class="a"><td>org.biojava.nbio.genome.parsers.genename.GeneName defines equals and uses Object.hashCode()</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#HE_EQUALS_USE_HASHCODE">HE_EQUALS_USE_HASHCODE</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/genename/GeneName.html#140">140</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.genome.parsers.genename.GeneNamesParser"></a><div class="section"><h3>org.biojava.nbio.genome.parsers.genename.GeneNamesParser<a name="org.biojava.nbio.genome.parsers.genename.GeneNamesParser"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Found reliance on default encoding in org.biojava.nbio.genome.parsers.genename.GeneNamesParser.getGeneNames(InputStream): new java.io.InputStreamReader(InputStream)</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/genename/GeneNamesParser.html#95">95</a></td><td>High</td></tr></table></div><a name="org.biojava.nbio.genome.parsers.gff.FeatureList"></a><div class="section"><h3>org.biojava.nbio.genome.parsers.gff.FeatureList<a name="org.biojava.nbio.genome.parsers.gff.FeatureList"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Class org.biojava.nbio.genome.parsers.gff.FeatureList defines non-transient non-serializable instance field mLocation</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#SE_BAD_FIELD">SE_BAD_FIELD</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/gff/FeatureList.html#">Not available</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.genome.parsers.gff.FeatureListFeatureComparator"></a><div class="section"><h3>org.biojava.nbio.genome.parsers.gff.FeatureList$FeatureComparator<a name="org.biojava.nbio.genome.parsers.gff.FeatureListFeatureComparator"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Should org.biojava.nbio.genome.parsers.gff.FeatureList$FeatureComparator be a _static_ inner class?</td><td>PERFORMANCE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#SIC_INNER_SHOULD_BE_STATIC">SIC_INNER_SHOULD_BE_STATIC</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/gff/FeatureList.html#470">470-477</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.genome.parsers.gff.GFF3Reader"></a><div class="section"><h3>org.biojava.nbio.genome.parsers.gff.GFF3Reader<a name="org.biojava.nbio.genome.parsers.gff.GFF3Reader"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Dead store to listGenes in org.biojava.nbio.genome.parsers.gff.GFF3Reader.main(String[])</td><td>STYLE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DLS_DEAD_LOCAL_STORE">DLS_DEAD_LOCAL_STORE</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/gff/GFF3Reader.html#176">176</a></td><td>Medium</td></tr><tr class="b"><td>Found reliance on default encoding in org.biojava.nbio.genome.parsers.gff.GFF3Reader.read(String, List): new java.io.FileReader(String)</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/gff/GFF3Reader.html#74">74</a></td><td>High</td></tr></table></div><a name="org.biojava.nbio.genome.parsers.gff.GFF3Writer"></a><div class="section"><h3>org.biojava.nbio.genome.parsers.gff.GFF3Writer<a name="org.biojava.nbio.genome.parsers.gff.GFF3Writer"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>Hard coded reference to an absolute pathname in org.biojava.nbio.genome.parsers.gff.GFF3Writer.main(String[])</td><td>STYLE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DMI_HARDCODED_ABSOLUTE_FILENAME">DMI_HARDCODED_ABSOLUTE_FILENAME</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/gff/GFF3Writer.html#149">149</a></td><td>Medium</td></tr><tr class="a"><td>Hard coded reference to an absolute pathname in org.biojava.nbio.genome.parsers.gff.GFF3Writer.main(String[])</td><td>STYLE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DMI_HARDCODED_ABSOLUTE_FILENAME">DMI_HARDCODED_ABSOLUTE_FILENAME</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/gff/GFF3Writer.html#149">149</a></td><td>Medium</td></tr><tr class="b"><td>Found reliance on default encoding in org.biojava.nbio.genome.parsers.gff.GFF3Writer.write(OutputStream, LinkedHashMap): String.getBytes()</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/gff/GFF3Writer.html#51">51</a></td><td>High</td></tr><tr class="a"><td>org.biojava.nbio.genome.parsers.gff.GFF3Writer.getGFF3Note(ArrayList) concatenates strings using + in a loop</td><td>PERFORMANCE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#SBSC_USE_STRINGBUFFER_CONCATENATION">SBSC_USE_STRINGBUFFER_CONCATENATION</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/gff/GFF3Writer.html#137">137</a></td><td>Medium</td></tr><tr class="b"><td>org.biojava.nbio.genome.parsers.gff.GFF3Writer.write(OutputStream, LinkedHashMap) makes inefficient use of keySet iterator instead of entrySet iterator</td><td>PERFORMANCE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#WMI_WRONG_MAP_ITERATOR">WMI_WRONG_MAP_ITERATOR</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/gff/GFF3Writer.html#53">53</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.genome.parsers.gff.GeneIDGFF2Reader"></a><div class="section"><h3>org.biojava.nbio.genome.parsers.gff.GeneIDGFF2Reader<a name="org.biojava.nbio.genome.parsers.gff.GeneIDGFF2Reader"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>Found reliance on default encoding in org.biojava.nbio.genome.parsers.gff.GeneIDGFF2Reader.read(String): new java.io.FileReader(String)</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/gff/GeneIDGFF2Reader.html#65">65</a></td><td>High</td></tr><tr class="a"><td>Found reliance on default encoding in org.biojava.nbio.genome.parsers.gff.GeneIDGFF2Reader.write(FeatureList, String): new java.io.FileWriter(String)</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/gff/GeneIDGFF2Reader.html#171">171</a></td><td>High</td></tr><tr class="b"><td>org.biojava.nbio.genome.parsers.gff.GeneIDGFF2Reader.parseLine(String) invokes inefficient new String(String) constructor</td><td>PERFORMANCE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_STRING_CTOR">DM_STRING_CTOR</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/gff/GeneIDGFF2Reader.html#150">150</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.genome.parsers.gff.GeneMarkGTFReader"></a><div class="section"><h3>org.biojava.nbio.genome.parsers.gff.GeneMarkGTFReader<a name="org.biojava.nbio.genome.parsers.gff.GeneMarkGTFReader"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>Found reliance on default encoding in org.biojava.nbio.genome.parsers.gff.GeneMarkGTFReader.read(String): new java.io.FileReader(String)</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/gff/GeneMarkGTFReader.html#66">66</a></td><td>High</td></tr><tr class="a"><td>org.biojava.nbio.genome.parsers.gff.GeneMarkGTFReader.parseLine(String) invokes inefficient new String(String) constructor</td><td>PERFORMANCE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_STRING_CTOR">DM_STRING_CTOR</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/gff/GeneMarkGTFReader.html#150">150</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.genome.parsers.gff.Location"></a><div class="section"><h3>org.biojava.nbio.genome.parsers.gff.Location<a name="org.biojava.nbio.genome.parsers.gff.Location"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Dead store to p14_14 in org.biojava.nbio.genome.parsers.gff.Location.main(String[])</td><td>STYLE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DLS_DEAD_LOCAL_STORE">DLS_DEAD_LOCAL_STORE</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/gff/Location.html#900">900</a></td><td>High</td></tr><tr class="b"><td>Dead store to p3_7 in org.biojava.nbio.genome.parsers.gff.Location.main(String[])</td><td>STYLE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DLS_DEAD_LOCAL_STORE">DLS_DEAD_LOCAL_STORE</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/gff/Location.html#893">893</a></td><td>High</td></tr><tr class="a"><td>org.biojava.nbio.genome.parsers.gff.Location.toString() invokes inefficient new String(String) constructor</td><td>PERFORMANCE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_STRING_CTOR">DM_STRING_CTOR</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/gff/Location.html#849">849</a></td><td>Medium</td></tr><tr class="b"><td>org.biojava.nbio.genome.parsers.gff.Location defines equals(Location) method and uses Object.equals(Object)</td><td>CORRECTNESS</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#EQ_SELF_USE_OBJECT">EQ_SELF_USE_OBJECT</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/gff/Location.html#860">860</a></td><td>Medium</td></tr><tr class="a"><td>org.biojava.nbio.genome.parsers.gff.Location defines equals and uses Object.hashCode()</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#HE_EQUALS_USE_HASHCODE">HE_EQUALS_USE_HASHCODE</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/gff/Location.html#860">860</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.genome.parsers.twobit.TwoBitParser"></a><div class="section"><h3>org.biojava.nbio.genome.parsers.twobit.TwoBitParser<a name="org.biojava.nbio.genome.parsers.twobit.TwoBitParser"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Result of integer multiplication cast to long in org.biojava.nbio.genome.parsers.twobit.TwoBitParser.readFourBytes()</td><td>STYLE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#ICAST_INTEGER_MULTIPLY_CAST_TO_LONG">ICAST_INTEGER_MULTIPLY_CAST_TO_LONG</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html#106">106</a></td><td>High</td></tr><tr class="b"><td>Inconsistent synchronization of org.biojava.nbio.genome.parsers.twobit.TwoBitParser.start_file_pos; locked 66% of time</td><td>MT_CORRECTNESS</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#IS2_INCONSISTENT_SYNC">IS2_INCONSISTENT_SYNC</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html#166">166</a></td><td>Medium</td></tr><tr class="a"><td>org.biojava.nbio.genome.parsers.twobit.TwoBitParser.main(String[]) ignores result of org.biojava.nbio.genome.parsers.twobit.TwoBitParser.skip(long)</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#SR_NOT_CHECKED">SR_NOT_CHECKED</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html#388">388</a></td><td>Medium</td></tr><tr class="b"><td>org.biojava.nbio.genome.parsers.twobit.TwoBitParser.setCurrentSequencePosition(long) ignores result of org.biojava.nbio.genome.parsers.twobit.TwoBitParser.skip(long)</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#SR_NOT_CHECKED">SR_NOT_CHECKED</a></td><td><a href="./xref/org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html#196">196</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.genome.uniprot.UniprotToFasta"></a><div class="section"><h3>org.biojava.nbio.genome.uniprot.UniprotToFasta<a name="org.biojava.nbio.genome.uniprot.UniprotToFasta"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>Found reliance on default encoding in org.biojava.nbio.genome.uniprot.UniprotToFasta.process(String, String): new java.io.FileReader(String)</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/genome/uniprot/UniprotToFasta.html#70">70</a></td><td>High</td></tr><tr class="a"><td>Dereference of the result of readLine() without nullcheck in org.biojava.nbio.genome.uniprot.UniprotToFasta.process(String, String)</td><td>STYLE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#NP_DEREFERENCE_OF_READLINE_VALUE">NP_DEREFERENCE_OF_READLINE_VALUE</a></td><td><a href="./xref/org/biojava/nbio/genome/uniprot/UniprotToFasta.html#85">85</a></td><td>Medium</td></tr><tr class="b"><td>Dereference of the result of readLine() without nullcheck in org.biojava.nbio.genome.uniprot.UniprotToFasta.process(String, String)</td><td>STYLE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#NP_DEREFERENCE_OF_READLINE_VALUE">NP_DEREFERENCE_OF_READLINE_VALUE</a></td><td><a href="./xref/org/biojava/nbio/genome/uniprot/UniprotToFasta.html#85">85</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.genome.util.SplitFasta"></a><div class="section"><h3>org.biojava.nbio.genome.util.SplitFasta<a name="org.biojava.nbio.genome.util.SplitFasta"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>Hard coded reference to an absolute pathname in org.biojava.nbio.genome.util.SplitFasta.main(String[])</td><td>STYLE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DMI_HARDCODED_ABSOLUTE_FILENAME">DMI_HARDCODED_ABSOLUTE_FILENAME</a></td><td><a href="./xref/org/biojava/nbio/genome/util/SplitFasta.html#69">69</a></td><td>Medium</td></tr><tr class="a"><td>Hard coded reference to an absolute pathname in org.biojava.nbio.genome.util.SplitFasta.main(String[])</td><td>STYLE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DMI_HARDCODED_ABSOLUTE_FILENAME">DMI_HARDCODED_ABSOLUTE_FILENAME</a></td><td><a href="./xref/org/biojava/nbio/genome/util/SplitFasta.html#69">69</a></td><td>Medium</td></tr><tr class="b"><td>exceptional return value of java.io.File.mkdirs() ignored in org.biojava.nbio.genome.util.SplitFasta.processNucleotides(File, String, File)</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#RV_RETURN_VALUE_IGNORED_BAD_PRACTICE">RV_RETURN_VALUE_IGNORED_BAD_PRACTICE</a></td><td><a href="./xref/org/biojava/nbio/genome/util/SplitFasta.html#49">49</a></td><td>Medium</td></tr></table></div>
      </div>
    </div>
    <div class="clear">
      <hr/>
    </div>
    <div id="footer">
      <div class="xright">
              Copyright &#169;                    2000-2015
                        <a href="http://www.biojava.org">BioJava</a>.
            All Rights Reserved.      
        
                  </div>
      <div class="clear">
        <hr/>
      </div>
    </div>
  </body>
</html>
