<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head><meta http-equiv="content-type" content="text/html; charset=UTF-8" />
<title>GeneSequence xref</title>
<link type="text/css" rel="stylesheet" href="../../../../../stylesheet.css" />
</head>
<body>
<div id="overview"><a href="../../../../../../apidocs/org/biojava/nbio/core/sequence/GeneSequence.html">View Javadoc</a></div><pre>
<a class="jxr_linenumber" name="L1" href="#L1">1</a>   <em class="jxr_comment">/*</em>
<a class="jxr_linenumber" name="L2" href="#L2">2</a>   <em class="jxr_comment"> *                    BioJava development code</em>
<a class="jxr_linenumber" name="L3" href="#L3">3</a>   <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L4" href="#L4">4</a>   <em class="jxr_comment"> * This code may be freely distributed and modified under the</em>
<a class="jxr_linenumber" name="L5" href="#L5">5</a>   <em class="jxr_comment"> * terms of the GNU Lesser General Public Licence.  This should</em>
<a class="jxr_linenumber" name="L6" href="#L6">6</a>   <em class="jxr_comment"> * be distributed with the code.  If you do not have a copy,</em>
<a class="jxr_linenumber" name="L7" href="#L7">7</a>   <em class="jxr_comment"> * see:</em>
<a class="jxr_linenumber" name="L8" href="#L8">8</a>   <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L9" href="#L9">9</a>   <em class="jxr_comment"> *      <a href="http://www.gnu.org/copyleft/lesser.htm" target="alexandria_uri">http://www.gnu.org/copyleft/lesser.htm</a>l</em>
<a class="jxr_linenumber" name="L10" href="#L10">10</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L11" href="#L11">11</a>  <em class="jxr_comment"> * Copyright for this code is held jointly by the individual</em>
<a class="jxr_linenumber" name="L12" href="#L12">12</a>  <em class="jxr_comment"> * authors.  These should be listed in @author doc comments.</em>
<a class="jxr_linenumber" name="L13" href="#L13">13</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L14" href="#L14">14</a>  <em class="jxr_comment"> * For more information on the BioJava project and its aims,</em>
<a class="jxr_linenumber" name="L15" href="#L15">15</a>  <em class="jxr_comment"> * or to join the biojava-l mailing list, visit the home page</em>
<a class="jxr_linenumber" name="L16" href="#L16">16</a>  <em class="jxr_comment"> * at:</em>
<a class="jxr_linenumber" name="L17" href="#L17">17</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L18" href="#L18">18</a>  <em class="jxr_comment"> *      <a href="http://www.biojava.org" target="alexandria_uri">http://www.biojava.org</a>/</em>
<a class="jxr_linenumber" name="L19" href="#L19">19</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L20" href="#L20">20</a>  <em class="jxr_comment"> * Created on DATE</em>
<a class="jxr_linenumber" name="L21" href="#L21">21</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L22" href="#L22">22</a>  <em class="jxr_comment"> */</em>
<a class="jxr_linenumber" name="L23" href="#L23">23</a>  <strong class="jxr_keyword">package</strong> org.biojava.nbio.core.sequence;
<a class="jxr_linenumber" name="L24" href="#L24">24</a>  
<a class="jxr_linenumber" name="L25" href="#L25">25</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.exceptions.CompoundNotFoundException;
<a class="jxr_linenumber" name="L26" href="#L26">26</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.compound.DNACompoundSet;
<a class="jxr_linenumber" name="L27" href="#L27">27</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.compound.NucleotideCompound;
<a class="jxr_linenumber" name="L28" href="#L28">28</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.template.CompoundSet;
<a class="jxr_linenumber" name="L29" href="#L29">29</a>  <strong class="jxr_keyword">import</strong> org.slf4j.Logger;
<a class="jxr_linenumber" name="L30" href="#L30">30</a>  <strong class="jxr_keyword">import</strong> org.slf4j.LoggerFactory;
<a class="jxr_linenumber" name="L31" href="#L31">31</a>  
<a class="jxr_linenumber" name="L32" href="#L32">32</a>  <strong class="jxr_keyword">import</strong> java.util.ArrayList;
<a class="jxr_linenumber" name="L33" href="#L33">33</a>  <strong class="jxr_keyword">import</strong> java.util.Collections;
<a class="jxr_linenumber" name="L34" href="#L34">34</a>  <strong class="jxr_keyword">import</strong> java.util.LinkedHashMap;
<a class="jxr_linenumber" name="L35" href="#L35">35</a>  
<a class="jxr_linenumber" name="L36" href="#L36">36</a>  <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L37" href="#L37">37</a>  <em class="jxr_javadoccomment"> *</em>
<a class="jxr_linenumber" name="L38" href="#L38">38</a>  <em class="jxr_javadoccomment"> * @author Scooter Willis</em>
<a class="jxr_linenumber" name="L39" href="#L39">39</a>  <em class="jxr_javadoccomment"> */</em>
<a class="jxr_linenumber" name="L40" href="#L40">40</a>  <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">class</strong> <a href="../../../../../org/biojava/nbio/core/sequence/GeneSequence.html">GeneSequence</a> <strong class="jxr_keyword">extends</strong> <a href="../../../../../org/biojava/nbio/core/sequence/DNASequence.html">DNASequence</a> {
<a class="jxr_linenumber" name="L41" href="#L41">41</a>  	
<a class="jxr_linenumber" name="L42" href="#L42">42</a>  	<strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">final</strong> <strong class="jxr_keyword">static</strong> Logger logger = LoggerFactory.getLogger(GeneSequence.<strong class="jxr_keyword">class</strong>);
<a class="jxr_linenumber" name="L43" href="#L43">43</a>  
<a class="jxr_linenumber" name="L44" href="#L44">44</a>      <strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">final</strong> LinkedHashMap&lt;String, TranscriptSequence&gt; transcriptSequenceHashMap = <strong class="jxr_keyword">new</strong> LinkedHashMap&lt;String, TranscriptSequence&gt;();    
<a class="jxr_linenumber" name="L45" href="#L45">45</a>      <strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">final</strong> LinkedHashMap&lt;String, IntronSequence&gt; intronSequenceHashMap = <strong class="jxr_keyword">new</strong> LinkedHashMap&lt;String, IntronSequence&gt;();
<a class="jxr_linenumber" name="L46" href="#L46">46</a>      <strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">final</strong> LinkedHashMap&lt;String, ExonSequence&gt; exonSequenceHashMap = <strong class="jxr_keyword">new</strong> LinkedHashMap&lt;String, ExonSequence&gt;();
<a class="jxr_linenumber" name="L47" href="#L47">47</a>      <strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">final</strong> ArrayList&lt;IntronSequence&gt; intronSequenceList = <strong class="jxr_keyword">new</strong> ArrayList&lt;IntronSequence&gt;();
<a class="jxr_linenumber" name="L48" href="#L48">48</a>      <strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">final</strong> ArrayList&lt;ExonSequence&gt; exonSequenceList = <strong class="jxr_keyword">new</strong> ArrayList&lt;ExonSequence&gt;();
<a class="jxr_linenumber" name="L49" href="#L49">49</a>      <strong class="jxr_keyword">boolean</strong> intronAdded = false; <em class="jxr_comment">// need to deal with the problem that typically introns are not added when validating the list and adding in introns as the regions not included in exons</em>
<a class="jxr_linenumber" name="L50" href="#L50">50</a>      <strong class="jxr_keyword">private</strong> <a href="../../../../../org/biojava/nbio/core/sequence/Strand.html">Strand</a> strand = Strand.UNDEFINED;
<a class="jxr_linenumber" name="L51" href="#L51">51</a>      <strong class="jxr_keyword">private</strong> <a href="../../../../../org/biojava/nbio/core/sequence/ChromosomeSequence.html">ChromosomeSequence</a> chromosomeSequence;
<a class="jxr_linenumber" name="L52" href="#L52">52</a>  
<a class="jxr_linenumber" name="L53" href="#L53">53</a>      <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L54" href="#L54">54</a>  <em class="jxr_javadoccomment">     * A class that keeps track of the details of a GeneSequence which is difficult to properly model. Two important concepts that is difficult</em>
<a class="jxr_linenumber" name="L55" href="#L55">55</a>  <em class="jxr_javadoccomment">     * to make everything flexible but still work. You can have GFF features that only describe Exons or Exons/Introns or CDS regions and one</em>
<a class="jxr_linenumber" name="L56" href="#L56">56</a>  <em class="jxr_javadoccomment">     * or more Transcriptions. You can have exon sequences but that does not imply transcription to the actual protein.</em>
<a class="jxr_linenumber" name="L57" href="#L57">57</a>  <em class="jxr_javadoccomment">     *</em>
<a class="jxr_linenumber" name="L58" href="#L58">58</a>  <em class="jxr_javadoccomment">     * The GeneSequence will keep track of Exons and Introns but to get a Protein sequence you need to start with a</em>
<a class="jxr_linenumber" name="L59" href="#L59">59</a>  <em class="jxr_javadoccomment">     * TranscriptSequence and then add CDS sequences.</em>
<a class="jxr_linenumber" name="L60" href="#L60">60</a>  <em class="jxr_javadoccomment">     *</em>
<a class="jxr_linenumber" name="L61" href="#L61">61</a>  <em class="jxr_javadoccomment">     * This is also a key class in the biojava-3-genome module for reading and writing GFF3 files</em>
<a class="jxr_linenumber" name="L62" href="#L62">62</a>  <em class="jxr_javadoccomment">     *</em>
<a class="jxr_linenumber" name="L63" href="#L63">63</a>  <em class="jxr_javadoccomment">     * @param parentDNASequence</em>
<a class="jxr_linenumber" name="L64" href="#L64">64</a>  <em class="jxr_javadoccomment">     * @param begin</em>
<a class="jxr_linenumber" name="L65" href="#L65">65</a>  <em class="jxr_javadoccomment">     * @param end inclusive of end</em>
<a class="jxr_linenumber" name="L66" href="#L66">66</a>  <em class="jxr_javadoccomment">     * @param strand force a gene to have strand and transcription sequence will inherit</em>
<a class="jxr_linenumber" name="L67" href="#L67">67</a>  <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L68" href="#L68">68</a>      <strong class="jxr_keyword">public</strong> <a href="../../../../../org/biojava/nbio/core/sequence/GeneSequence.html">GeneSequence</a>(<a href="../../../../../org/biojava/nbio/core/sequence/ChromosomeSequence.html">ChromosomeSequence</a> parentSequence, <strong class="jxr_keyword">int</strong> begin, <strong class="jxr_keyword">int</strong> end, <a href="../../../../../org/biojava/nbio/core/sequence/Strand.html">Strand</a> strand) {
<a class="jxr_linenumber" name="L69" href="#L69">69</a>          chromosomeSequence = parentSequence;
<a class="jxr_linenumber" name="L70" href="#L70">70</a>          setParentSequence(parentSequence);
<a class="jxr_linenumber" name="L71" href="#L71">71</a>          setBioBegin(begin);
<a class="jxr_linenumber" name="L72" href="#L72">72</a>          setBioEnd(end);
<a class="jxr_linenumber" name="L73" href="#L73">73</a>          setStrand(strand);
<a class="jxr_linenumber" name="L74" href="#L74">74</a>          <strong class="jxr_keyword">this</strong>.setCompoundSet(DNACompoundSet.getDNACompoundSet());
<a class="jxr_linenumber" name="L75" href="#L75">75</a>      }
<a class="jxr_linenumber" name="L76" href="#L76">76</a>  
<a class="jxr_linenumber" name="L77" href="#L77">77</a>      <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L78" href="#L78">78</a>  <em class="jxr_javadoccomment">     * The parent ChromosomeSequence which contains the actual DNA sequence data</em>
<a class="jxr_linenumber" name="L79" href="#L79">79</a>  <em class="jxr_javadoccomment">     * @return Chromosome sequence</em>
<a class="jxr_linenumber" name="L80" href="#L80">80</a>  <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L81" href="#L81">81</a>      <strong class="jxr_keyword">public</strong> <a href="../../../../../org/biojava/nbio/core/sequence/ChromosomeSequence.html">ChromosomeSequence</a> getParentChromosomeSequence() {
<a class="jxr_linenumber" name="L82" href="#L82">82</a>          <strong class="jxr_keyword">return</strong> chromosomeSequence;
<a class="jxr_linenumber" name="L83" href="#L83">83</a>      }
<a class="jxr_linenumber" name="L84" href="#L84">84</a>  
<a class="jxr_linenumber" name="L85" href="#L85">85</a>      @Override
<a class="jxr_linenumber" name="L86" href="#L86">86</a>      <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">int</strong> getLength() {
<a class="jxr_linenumber" name="L87" href="#L87">87</a>          <strong class="jxr_keyword">return</strong> Math.abs(<strong class="jxr_keyword">this</strong>.getBioEnd() - <strong class="jxr_keyword">this</strong>.getBioBegin()) + 1;
<a class="jxr_linenumber" name="L88" href="#L88">88</a>      }
<a class="jxr_linenumber" name="L89" href="#L89">89</a>  
<a class="jxr_linenumber" name="L90" href="#L90">90</a>      <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L91" href="#L91">91</a>  <em class="jxr_javadoccomment">     * Once everything has been added to the gene sequence where you might have added exon sequences only then you</em>
<a class="jxr_linenumber" name="L92" href="#L92">92</a>  <em class="jxr_javadoccomment">     * can infer the intron sequences and add them. You may also have the case where you only added one or more</em>
<a class="jxr_linenumber" name="L93" href="#L93">93</a>  <em class="jxr_javadoccomment">     * TranscriptSequences and from that you can infer the exon sequences and intron sequences.</em>
<a class="jxr_linenumber" name="L94" href="#L94">94</a>  <em class="jxr_javadoccomment">     * Currently not implement</em>
<a class="jxr_linenumber" name="L95" href="#L95">95</a>  <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L96" href="#L96">96</a>      <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> addIntronsUsingExons() <strong class="jxr_keyword">throws</strong> Exception {
<a class="jxr_linenumber" name="L97" href="#L97">97</a>          <strong class="jxr_keyword">if</strong> (intronAdded) { <em class="jxr_comment">//going to assume introns are correct</em>
<a class="jxr_linenumber" name="L98" href="#L98">98</a>              <strong class="jxr_keyword">return</strong>;
<a class="jxr_linenumber" name="L99" href="#L99">99</a>          }
<a class="jxr_linenumber" name="L100" href="#L100">100</a>         <strong class="jxr_keyword">if</strong> (exonSequenceList.size() == 0) {
<a class="jxr_linenumber" name="L101" href="#L101">101</a>             <strong class="jxr_keyword">return</strong>;
<a class="jxr_linenumber" name="L102" href="#L102">102</a>         }
<a class="jxr_linenumber" name="L103" href="#L103">103</a>         <a href="../../../../../org/biojava/nbio/core/sequence/ExonComparator.html">ExonComparator</a> exonComparator = <strong class="jxr_keyword">new</strong> <a href="../../../../../org/biojava/nbio/core/sequence/ExonComparator.html">ExonComparator</a>();
<a class="jxr_linenumber" name="L104" href="#L104">104</a>         <em class="jxr_comment">//sort based on start position and sense;</em>
<a class="jxr_linenumber" name="L105" href="#L105">105</a>         Collections.sort(exonSequenceList, exonComparator);
<a class="jxr_linenumber" name="L106" href="#L106">106</a>         <strong class="jxr_keyword">int</strong> shift = -1;
<a class="jxr_linenumber" name="L107" href="#L107">107</a>         <strong class="jxr_keyword">if</strong> (getStrand() == Strand.NEGATIVE) {
<a class="jxr_linenumber" name="L108" href="#L108">108</a>             shift = 1;
<a class="jxr_linenumber" name="L109" href="#L109">109</a>         }
<a class="jxr_linenumber" name="L110" href="#L110">110</a>         <em class="jxr_comment">//ExonSequence firstExonSequence = exonSequenceList.get(0);</em>
<a class="jxr_linenumber" name="L111" href="#L111">111</a>         <strong class="jxr_keyword">int</strong> intronIndex = 1;
<a class="jxr_linenumber" name="L112" href="#L112">112</a>  <em class="jxr_comment">//       if (firstExonSequence.getBioBegin().intValue() != getBioBegin().intValue()) {</em>
<a class="jxr_linenumber" name="L113" href="#L113">113</a>  <em class="jxr_comment">//           this.addIntron(new AccessionID(this.getAccession().getID() + "-" + "intron" + intronIndex), getBioBegin(), firstExonSequence.getBioBegin() + shift);</em>
<a class="jxr_linenumber" name="L114" href="#L114">114</a>  <em class="jxr_comment">//           intronIndex++;</em>
<a class="jxr_linenumber" name="L115" href="#L115">115</a>  <em class="jxr_comment">//       }</em>
<a class="jxr_linenumber" name="L116" href="#L116">116</a>         <strong class="jxr_keyword">for</strong> (<strong class="jxr_keyword">int</strong> i = 0; i &lt; exonSequenceList.size() - 1; i++) {
<a class="jxr_linenumber" name="L117" href="#L117">117</a>             <a href="../../../../../org/biojava/nbio/core/sequence/ExonSequence.html">ExonSequence</a> exon1 = exonSequenceList.get(i);
<a class="jxr_linenumber" name="L118" href="#L118">118</a>             <a href="../../../../../org/biojava/nbio/core/sequence/ExonSequence.html">ExonSequence</a> exon2 = exonSequenceList.get(i + 1);
<a class="jxr_linenumber" name="L119" href="#L119">119</a>             <strong class="jxr_keyword">this</strong>.addIntron(<strong class="jxr_keyword">new</strong> <a href="../../../../../org/biojava/nbio/core/sequence/AccessionID.html">AccessionID</a>(<strong class="jxr_keyword">this</strong>.getAccession().getID() + <span class="jxr_string">"-"</span> + <span class="jxr_string">"intron"</span> + intronIndex), exon1.getBioEnd() - shift, exon2.getBioBegin() + shift);
<a class="jxr_linenumber" name="L120" href="#L120">120</a>             intronIndex++;
<a class="jxr_linenumber" name="L121" href="#L121">121</a>         }
<a class="jxr_linenumber" name="L122" href="#L122">122</a> 
<a class="jxr_linenumber" name="L123" href="#L123">123</a>  <em class="jxr_comment">//       ExonSequence lastExonSequence = exonSequenceList.get(exonSequenceList.size() - 1);</em>
<a class="jxr_linenumber" name="L124" href="#L124">124</a>  <em class="jxr_comment">//       if (lastExonSequence.getBioEnd().intValue() != getBioEnd().intValue()) {</em>
<a class="jxr_linenumber" name="L125" href="#L125">125</a>  <em class="jxr_comment">//           this.addIntron(new AccessionID(this.getAccession().getID() + "-" + "intron" + intronIndex), lastExonSequence.getBioEnd() - shift, getBioEnd());</em>
<a class="jxr_linenumber" name="L126" href="#L126">126</a>  <em class="jxr_comment">//           intronIndex++;</em>
<a class="jxr_linenumber" name="L127" href="#L127">127</a>  <em class="jxr_comment">//       }</em>
<a class="jxr_linenumber" name="L128" href="#L128">128</a> 
<a class="jxr_linenumber" name="L129" href="#L129">129</a>         <em class="jxr_comment">//    log.severe("Add in support for building introns based on added exons");</em>
<a class="jxr_linenumber" name="L130" href="#L130">130</a> 
<a class="jxr_linenumber" name="L131" href="#L131">131</a>     }
<a class="jxr_linenumber" name="L132" href="#L132">132</a> 
<a class="jxr_linenumber" name="L133" href="#L133">133</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L134" href="#L134">134</a> <em class="jxr_javadoccomment">     * A gene should have Strand</em>
<a class="jxr_linenumber" name="L135" href="#L135">135</a> <em class="jxr_javadoccomment">     * @return the strand</em>
<a class="jxr_linenumber" name="L136" href="#L136">136</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L137" href="#L137">137</a>     <strong class="jxr_keyword">public</strong> <a href="../../../../../org/biojava/nbio/core/sequence/Strand.html">Strand</a> getStrand() {
<a class="jxr_linenumber" name="L138" href="#L138">138</a>         <strong class="jxr_keyword">return</strong> strand;
<a class="jxr_linenumber" name="L139" href="#L139">139</a>     }
<a class="jxr_linenumber" name="L140" href="#L140">140</a> 
<a class="jxr_linenumber" name="L141" href="#L141">141</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L142" href="#L142">142</a> <em class="jxr_javadoccomment">     * @param strand the strand to set</em>
<a class="jxr_linenumber" name="L143" href="#L143">143</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L144" href="#L144">144</a>     <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> setStrand(<a href="../../../../../org/biojava/nbio/core/sequence/Strand.html">Strand</a> strand) {
<a class="jxr_linenumber" name="L145" href="#L145">145</a>         <strong class="jxr_keyword">this</strong>.strand = strand;
<a class="jxr_linenumber" name="L146" href="#L146">146</a>     }
<a class="jxr_linenumber" name="L147" href="#L147">147</a> 
<a class="jxr_linenumber" name="L148" href="#L148">148</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L149" href="#L149">149</a> <em class="jxr_javadoccomment">     * Get the transcript sequence by accession</em>
<a class="jxr_linenumber" name="L150" href="#L150">150</a> <em class="jxr_javadoccomment">     * @param accession</em>
<a class="jxr_linenumber" name="L151" href="#L151">151</a> <em class="jxr_javadoccomment">     * @return the transcript</em>
<a class="jxr_linenumber" name="L152" href="#L152">152</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L153" href="#L153">153</a>     <strong class="jxr_keyword">public</strong> <a href="../../../../../org/biojava/nbio/core/sequence/TranscriptSequence.html">TranscriptSequence</a> getTranscript(String accession) {
<a class="jxr_linenumber" name="L154" href="#L154">154</a>         <strong class="jxr_keyword">return</strong> transcriptSequenceHashMap.get(accession);
<a class="jxr_linenumber" name="L155" href="#L155">155</a>     }
<a class="jxr_linenumber" name="L156" href="#L156">156</a> 
<a class="jxr_linenumber" name="L157" href="#L157">157</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L158" href="#L158">158</a> <em class="jxr_javadoccomment">     * Get the collection of transcription sequences assigned to this gene</em>
<a class="jxr_linenumber" name="L159" href="#L159">159</a> <em class="jxr_javadoccomment">     * @return transcripts</em>
<a class="jxr_linenumber" name="L160" href="#L160">160</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L161" href="#L161">161</a>     <strong class="jxr_keyword">public</strong> LinkedHashMap&lt;String, TranscriptSequence&gt; getTranscripts() {
<a class="jxr_linenumber" name="L162" href="#L162">162</a>         <strong class="jxr_keyword">return</strong> transcriptSequenceHashMap;
<a class="jxr_linenumber" name="L163" href="#L163">163</a>     }
<a class="jxr_linenumber" name="L164" href="#L164">164</a> 
<a class="jxr_linenumber" name="L165" href="#L165">165</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L166" href="#L166">166</a> <em class="jxr_javadoccomment">     * Remove the transcript sequence from the gene</em>
<a class="jxr_linenumber" name="L167" href="#L167">167</a> <em class="jxr_javadoccomment">     * @param accession</em>
<a class="jxr_linenumber" name="L168" href="#L168">168</a> <em class="jxr_javadoccomment">     * @return transcriptsequence</em>
<a class="jxr_linenumber" name="L169" href="#L169">169</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L170" href="#L170">170</a>     <strong class="jxr_keyword">public</strong> <a href="../../../../../org/biojava/nbio/core/sequence/TranscriptSequence.html">TranscriptSequence</a> removeTranscript(String accession) {
<a class="jxr_linenumber" name="L171" href="#L171">171</a> 
<a class="jxr_linenumber" name="L172" href="#L172">172</a> 
<a class="jxr_linenumber" name="L173" href="#L173">173</a>         <strong class="jxr_keyword">return</strong> transcriptSequenceHashMap.remove(accession);
<a class="jxr_linenumber" name="L174" href="#L174">174</a>     }
<a class="jxr_linenumber" name="L175" href="#L175">175</a> 
<a class="jxr_linenumber" name="L176" href="#L176">176</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L177" href="#L177">177</a> <em class="jxr_javadoccomment">     * Add a transcription sequence to a gene which describes a ProteinSequence</em>
<a class="jxr_linenumber" name="L178" href="#L178">178</a> <em class="jxr_javadoccomment">     * @param accession</em>
<a class="jxr_linenumber" name="L179" href="#L179">179</a> <em class="jxr_javadoccomment">     * @param begin</em>
<a class="jxr_linenumber" name="L180" href="#L180">180</a> <em class="jxr_javadoccomment">     * @param end</em>
<a class="jxr_linenumber" name="L181" href="#L181">181</a> <em class="jxr_javadoccomment">     * @return transcript sequence</em>
<a class="jxr_linenumber" name="L182" href="#L182">182</a> <em class="jxr_javadoccomment">     * @throws Exception If the accession id is already used</em>
<a class="jxr_linenumber" name="L183" href="#L183">183</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L184" href="#L184">184</a>     <strong class="jxr_keyword">public</strong> <a href="../../../../../org/biojava/nbio/core/sequence/TranscriptSequence.html">TranscriptSequence</a> addTranscript(<a href="../../../../../org/biojava/nbio/core/sequence/AccessionID.html">AccessionID</a> accession, <strong class="jxr_keyword">int</strong> begin, <strong class="jxr_keyword">int</strong> end) <strong class="jxr_keyword">throws</strong> Exception {
<a class="jxr_linenumber" name="L185" href="#L185">185</a>         <strong class="jxr_keyword">if</strong> (transcriptSequenceHashMap.containsKey(accession.getID())) {
<a class="jxr_linenumber" name="L186" href="#L186">186</a>             <strong class="jxr_keyword">throw</strong> <strong class="jxr_keyword">new</strong> Exception(<span class="jxr_string">"Duplicate accesion id "</span> + accession.getID());
<a class="jxr_linenumber" name="L187" href="#L187">187</a>         }
<a class="jxr_linenumber" name="L188" href="#L188">188</a>         <a href="../../../../../org/biojava/nbio/core/sequence/TranscriptSequence.html">TranscriptSequence</a> transcriptSequence = <strong class="jxr_keyword">new</strong> <a href="../../../../../org/biojava/nbio/core/sequence/TranscriptSequence.html">TranscriptSequence</a>(<strong class="jxr_keyword">this</strong>, begin, end);
<a class="jxr_linenumber" name="L189" href="#L189">189</a>         transcriptSequence.setAccession(accession);
<a class="jxr_linenumber" name="L190" href="#L190">190</a>         transcriptSequenceHashMap.put(accession.getID(), transcriptSequence);
<a class="jxr_linenumber" name="L191" href="#L191">191</a>         <strong class="jxr_keyword">return</strong> transcriptSequence;
<a class="jxr_linenumber" name="L192" href="#L192">192</a>     }
<a class="jxr_linenumber" name="L193" href="#L193">193</a> 
<a class="jxr_linenumber" name="L194" href="#L194">194</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L195" href="#L195">195</a> <em class="jxr_javadoccomment">     * Remove the intron by accession</em>
<a class="jxr_linenumber" name="L196" href="#L196">196</a> <em class="jxr_javadoccomment">     * @param accession</em>
<a class="jxr_linenumber" name="L197" href="#L197">197</a> <em class="jxr_javadoccomment">     * @return intron sequence</em>
<a class="jxr_linenumber" name="L198" href="#L198">198</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L199" href="#L199">199</a>     <strong class="jxr_keyword">public</strong> <a href="../../../../../org/biojava/nbio/core/sequence/IntronSequence.html">IntronSequence</a> removeIntron(String accession) {
<a class="jxr_linenumber" name="L200" href="#L200">200</a>         <strong class="jxr_keyword">for</strong> (IntronSequence intronSequence : intronSequenceList) {
<a class="jxr_linenumber" name="L201" href="#L201">201</a>             <strong class="jxr_keyword">if</strong> (intronSequence.getAccession().getID().equals(accession)) {
<a class="jxr_linenumber" name="L202" href="#L202">202</a>                 intronSequenceList.remove(intronSequence);
<a class="jxr_linenumber" name="L203" href="#L203">203</a>                 intronSequenceHashMap.remove(accession);
<a class="jxr_linenumber" name="L204" href="#L204">204</a>                 <strong class="jxr_keyword">return</strong> intronSequence;
<a class="jxr_linenumber" name="L205" href="#L205">205</a>             }
<a class="jxr_linenumber" name="L206" href="#L206">206</a>         }
<a class="jxr_linenumber" name="L207" href="#L207">207</a>         <strong class="jxr_keyword">return</strong> <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L208" href="#L208">208</a>     }
<a class="jxr_linenumber" name="L209" href="#L209">209</a> 
<a class="jxr_linenumber" name="L210" href="#L210">210</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L211" href="#L211">211</a> <em class="jxr_javadoccomment">     * Add an Intron Currently used to mark an IntronSequence as a feature</em>
<a class="jxr_linenumber" name="L212" href="#L212">212</a> <em class="jxr_javadoccomment">     * @param accession</em>
<a class="jxr_linenumber" name="L213" href="#L213">213</a> <em class="jxr_javadoccomment">     * @param begin</em>
<a class="jxr_linenumber" name="L214" href="#L214">214</a> <em class="jxr_javadoccomment">     * @param end</em>
<a class="jxr_linenumber" name="L215" href="#L215">215</a> <em class="jxr_javadoccomment">     * @return intron sequence</em>
<a class="jxr_linenumber" name="L216" href="#L216">216</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L217" href="#L217">217</a>     <strong class="jxr_keyword">public</strong> <a href="../../../../../org/biojava/nbio/core/sequence/IntronSequence.html">IntronSequence</a> addIntron(<a href="../../../../../org/biojava/nbio/core/sequence/AccessionID.html">AccessionID</a> accession, <strong class="jxr_keyword">int</strong> begin, <strong class="jxr_keyword">int</strong> end) <strong class="jxr_keyword">throws</strong> Exception {
<a class="jxr_linenumber" name="L218" href="#L218">218</a>         <strong class="jxr_keyword">if</strong> (intronSequenceHashMap.containsKey(accession.getID())) {
<a class="jxr_linenumber" name="L219" href="#L219">219</a>             <strong class="jxr_keyword">throw</strong> <strong class="jxr_keyword">new</strong> Exception(<span class="jxr_string">"Duplicate accesion id "</span> + accession.getID());
<a class="jxr_linenumber" name="L220" href="#L220">220</a>         }
<a class="jxr_linenumber" name="L221" href="#L221">221</a>         intronAdded = <strong class="jxr_keyword">true</strong>;
<a class="jxr_linenumber" name="L222" href="#L222">222</a>         <a href="../../../../../org/biojava/nbio/core/sequence/IntronSequence.html">IntronSequence</a> intronSequence = <strong class="jxr_keyword">new</strong> <a href="../../../../../org/biojava/nbio/core/sequence/IntronSequence.html">IntronSequence</a>(<strong class="jxr_keyword">this</strong>, begin, end); <em class="jxr_comment">// working off the assumption that intron frame is always 0 or doesn't matter and same sense as parent</em>
<a class="jxr_linenumber" name="L223" href="#L223">223</a>         intronSequence.setAccession(accession);
<a class="jxr_linenumber" name="L224" href="#L224">224</a>         intronSequenceList.add(intronSequence);
<a class="jxr_linenumber" name="L225" href="#L225">225</a>         intronSequenceHashMap.put(accession.getID(), intronSequence);
<a class="jxr_linenumber" name="L226" href="#L226">226</a>         <strong class="jxr_keyword">return</strong> intronSequence;
<a class="jxr_linenumber" name="L227" href="#L227">227</a>     }
<a class="jxr_linenumber" name="L228" href="#L228">228</a> 
<a class="jxr_linenumber" name="L229" href="#L229">229</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L230" href="#L230">230</a> <em class="jxr_javadoccomment">     * Remove the exon sequence</em>
<a class="jxr_linenumber" name="L231" href="#L231">231</a> <em class="jxr_javadoccomment">     * @param accession</em>
<a class="jxr_linenumber" name="L232" href="#L232">232</a> <em class="jxr_javadoccomment">     * @return exon sequence</em>
<a class="jxr_linenumber" name="L233" href="#L233">233</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L234" href="#L234">234</a>     <strong class="jxr_keyword">public</strong> <a href="../../../../../org/biojava/nbio/core/sequence/ExonSequence.html">ExonSequence</a> removeExon(String accession) {
<a class="jxr_linenumber" name="L235" href="#L235">235</a>         <strong class="jxr_keyword">for</strong> (ExonSequence exonSequence : exonSequenceList) {
<a class="jxr_linenumber" name="L236" href="#L236">236</a>             <strong class="jxr_keyword">if</strong> (exonSequence.getAccession().getID().equals(accession)) {
<a class="jxr_linenumber" name="L237" href="#L237">237</a>                 exonSequenceList.remove(exonSequence);
<a class="jxr_linenumber" name="L238" href="#L238">238</a>                 exonSequenceHashMap.remove(accession);
<a class="jxr_linenumber" name="L239" href="#L239">239</a>                 <em class="jxr_comment">// we now have a new gap which creates an intron</em>
<a class="jxr_linenumber" name="L240" href="#L240">240</a>                 intronSequenceList.clear();
<a class="jxr_linenumber" name="L241" href="#L241">241</a>                 intronSequenceHashMap.clear();
<a class="jxr_linenumber" name="L242" href="#L242">242</a>                 intronAdded = false;
<a class="jxr_linenumber" name="L243" href="#L243">243</a>                 <strong class="jxr_keyword">try</strong>{
<a class="jxr_linenumber" name="L244" href="#L244">244</a>                     addIntronsUsingExons();
<a class="jxr_linenumber" name="L245" href="#L245">245</a>                 } <strong class="jxr_keyword">catch</strong>(Exception e){
<a class="jxr_linenumber" name="L246" href="#L246">246</a>                     logger.error(<span class="jxr_string">"Remove Exon validate() error "</span> + e.getMessage());
<a class="jxr_linenumber" name="L247" href="#L247">247</a>                 }
<a class="jxr_linenumber" name="L248" href="#L248">248</a>                 <strong class="jxr_keyword">return</strong> exonSequence;
<a class="jxr_linenumber" name="L249" href="#L249">249</a>             }
<a class="jxr_linenumber" name="L250" href="#L250">250</a>         }
<a class="jxr_linenumber" name="L251" href="#L251">251</a>         <strong class="jxr_keyword">return</strong> <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L252" href="#L252">252</a>     }
<a class="jxr_linenumber" name="L253" href="#L253">253</a> 
<a class="jxr_linenumber" name="L254" href="#L254">254</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L255" href="#L255">255</a> <em class="jxr_javadoccomment">     * Add an ExonSequence mainly used to mark as a feature</em>
<a class="jxr_linenumber" name="L256" href="#L256">256</a> <em class="jxr_javadoccomment">     * @param accession</em>
<a class="jxr_linenumber" name="L257" href="#L257">257</a> <em class="jxr_javadoccomment">     * @param begin</em>
<a class="jxr_linenumber" name="L258" href="#L258">258</a> <em class="jxr_javadoccomment">     * @param end</em>
<a class="jxr_linenumber" name="L259" href="#L259">259</a> <em class="jxr_javadoccomment">     * @return exon sequence</em>
<a class="jxr_linenumber" name="L260" href="#L260">260</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L261" href="#L261">261</a>     <strong class="jxr_keyword">public</strong> <a href="../../../../../org/biojava/nbio/core/sequence/ExonSequence.html">ExonSequence</a> addExon(<a href="../../../../../org/biojava/nbio/core/sequence/AccessionID.html">AccessionID</a> accession, <strong class="jxr_keyword">int</strong> begin, <strong class="jxr_keyword">int</strong> end) <strong class="jxr_keyword">throws</strong> Exception {
<a class="jxr_linenumber" name="L262" href="#L262">262</a>         <strong class="jxr_keyword">if</strong> (exonSequenceHashMap.containsKey(accession.getID())) {
<a class="jxr_linenumber" name="L263" href="#L263">263</a>             <strong class="jxr_keyword">throw</strong> <strong class="jxr_keyword">new</strong> Exception(<span class="jxr_string">"Duplicate accesion id "</span> + accession.getID());
<a class="jxr_linenumber" name="L264" href="#L264">264</a>         }
<a class="jxr_linenumber" name="L265" href="#L265">265</a> 
<a class="jxr_linenumber" name="L266" href="#L266">266</a>         <a href="../../../../../org/biojava/nbio/core/sequence/ExonSequence.html">ExonSequence</a> exonSequence = <strong class="jxr_keyword">new</strong> <a href="../../../../../org/biojava/nbio/core/sequence/ExonSequence.html">ExonSequence</a>(<strong class="jxr_keyword">this</strong>, begin, end); <em class="jxr_comment">//sense should be the same as parent</em>
<a class="jxr_linenumber" name="L267" href="#L267">267</a>         exonSequence.setAccession(accession);
<a class="jxr_linenumber" name="L268" href="#L268">268</a>         exonSequenceList.add(exonSequence);
<a class="jxr_linenumber" name="L269" href="#L269">269</a>         exonSequenceHashMap.put(accession.getID(), exonSequence);
<a class="jxr_linenumber" name="L270" href="#L270">270</a>         <strong class="jxr_keyword">return</strong> exonSequence;
<a class="jxr_linenumber" name="L271" href="#L271">271</a>     }
<a class="jxr_linenumber" name="L272" href="#L272">272</a> 
<a class="jxr_linenumber" name="L273" href="#L273">273</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L274" href="#L274">274</a> <em class="jxr_javadoccomment">     * Get the exons as an ArrayList</em>
<a class="jxr_linenumber" name="L275" href="#L275">275</a> <em class="jxr_javadoccomment">     * @return exons</em>
<a class="jxr_linenumber" name="L276" href="#L276">276</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L277" href="#L277">277</a>     <strong class="jxr_keyword">public</strong> ArrayList&lt;ExonSequence&gt; getExonSequences() {
<a class="jxr_linenumber" name="L278" href="#L278">278</a>         <strong class="jxr_keyword">return</strong> exonSequenceList;
<a class="jxr_linenumber" name="L279" href="#L279">279</a>     }
<a class="jxr_linenumber" name="L280" href="#L280">280</a> 
<a class="jxr_linenumber" name="L281" href="#L281">281</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L282" href="#L282">282</a> <em class="jxr_javadoccomment">     * Get the introns as an ArrayList</em>
<a class="jxr_linenumber" name="L283" href="#L283">283</a> <em class="jxr_javadoccomment">     * @return introns </em>
<a class="jxr_linenumber" name="L284" href="#L284">284</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L285" href="#L285">285</a>     <strong class="jxr_keyword">public</strong> ArrayList&lt;IntronSequence&gt; getIntronSequences() {
<a class="jxr_linenumber" name="L286" href="#L286">286</a>         <strong class="jxr_keyword">return</strong> intronSequenceList;
<a class="jxr_linenumber" name="L287" href="#L287">287</a>     }
<a class="jxr_linenumber" name="L288" href="#L288">288</a> 
<a class="jxr_linenumber" name="L289" href="#L289">289</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L290" href="#L290">290</a> <em class="jxr_javadoccomment">     * Try to give method clarity where you want a DNASequence coding in the 5' to 3' direction</em>
<a class="jxr_linenumber" name="L291" href="#L291">291</a> <em class="jxr_javadoccomment">     * Returns the DNASequence representative of the 5' and 3' reading based on strand</em>
<a class="jxr_linenumber" name="L292" href="#L292">292</a> <em class="jxr_javadoccomment">     * @return dna sequence</em>
<a class="jxr_linenumber" name="L293" href="#L293">293</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L294" href="#L294">294</a>     <strong class="jxr_keyword">public</strong> <a href="../../../../../org/biojava/nbio/core/sequence/DNASequence.html">DNASequence</a> getSequence5PrimeTo3Prime() {
<a class="jxr_linenumber" name="L295" href="#L295">295</a>         String sequence = getSequenceAsString(<strong class="jxr_keyword">this</strong>.getBioBegin(), <strong class="jxr_keyword">this</strong>.getBioEnd(), <strong class="jxr_keyword">this</strong>.getStrand());
<a class="jxr_linenumber" name="L296" href="#L296">296</a>         <strong class="jxr_keyword">if</strong> (getStrand() == Strand.NEGATIVE) {
<a class="jxr_linenumber" name="L297" href="#L297">297</a>             <em class="jxr_comment">//need to take complement of sequence because it is negative and we are returning the gene sequence from the opposite strand</em>
<a class="jxr_linenumber" name="L298" href="#L298">298</a>             StringBuilder b = <strong class="jxr_keyword">new</strong> StringBuilder(getLength());
<a class="jxr_linenumber" name="L299" href="#L299">299</a>             CompoundSet&lt;NucleotideCompound&gt; compoundSet = <strong class="jxr_keyword">this</strong>.getCompoundSet();
<a class="jxr_linenumber" name="L300" href="#L300">300</a>             <strong class="jxr_keyword">for</strong> (<strong class="jxr_keyword">int</strong> i = 0; i &lt; sequence.length(); i++) {
<a class="jxr_linenumber" name="L301" href="#L301">301</a>                 String nucleotide = sequence.charAt(i) + <span class="jxr_string">""</span>;
<a class="jxr_linenumber" name="L302" href="#L302">302</a>                 <a href="../../../../../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html">NucleotideCompound</a> nucleotideCompound = compoundSet.getCompoundForString(nucleotide);
<a class="jxr_linenumber" name="L303" href="#L303">303</a>                 b.append(nucleotideCompound.getComplement().getShortName());
<a class="jxr_linenumber" name="L304" href="#L304">304</a>             }
<a class="jxr_linenumber" name="L305" href="#L305">305</a>             sequence = b.toString();
<a class="jxr_linenumber" name="L306" href="#L306">306</a>         }
<a class="jxr_linenumber" name="L307" href="#L307">307</a>         <a href="../../../../../org/biojava/nbio/core/sequence/DNASequence.html">DNASequence</a> dnaSequence = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L308" href="#L308">308</a>         <strong class="jxr_keyword">try</strong> {
<a class="jxr_linenumber" name="L309" href="#L309">309</a>         	dnaSequence = <strong class="jxr_keyword">new</strong> <a href="../../../../../org/biojava/nbio/core/sequence/DNASequence.html">DNASequence</a>(sequence.toUpperCase());
<a class="jxr_linenumber" name="L310" href="#L310">310</a>         } <strong class="jxr_keyword">catch</strong> (CompoundNotFoundException e) {
<a class="jxr_linenumber" name="L311" href="#L311">311</a>         	<em class="jxr_comment">// this should not happen, the sequence is DNA originally, if it does, there's a bug somewhere</em>
<a class="jxr_linenumber" name="L312" href="#L312">312</a>         	logger.error(<span class="jxr_string">"Could not create new DNA sequence in getSequence5PrimeTo3Prime(). Error: {}"</span>,e.getMessage());
<a class="jxr_linenumber" name="L313" href="#L313">313</a>         }
<a class="jxr_linenumber" name="L314" href="#L314">314</a>         dnaSequence.setAccession(<strong class="jxr_keyword">new</strong> <a href="../../../../../org/biojava/nbio/core/sequence/AccessionID.html">AccessionID</a>(<strong class="jxr_keyword">this</strong>.getAccession().getID()));
<a class="jxr_linenumber" name="L315" href="#L315">315</a>         <strong class="jxr_keyword">return</strong> dnaSequence;
<a class="jxr_linenumber" name="L316" href="#L316">316</a>     }
<a class="jxr_linenumber" name="L317" href="#L317">317</a> }
</pre>
<hr/>
<div id="footer">Copyright &#169; 2000&#x2013;2015 <a href="http://www.biojava.org">BioJava</a>. All rights reserved.</div>
</body>
</html>
