<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head><meta http-equiv="content-type" content="text/html; charset=UTF-8" />
<title>ProteinSequence xref</title>
<link type="text/css" rel="stylesheet" href="../../../../../stylesheet.css" />
</head>
<body>
<div id="overview"><a href="../../../../../../apidocs/org/biojava/nbio/core/sequence/ProteinSequence.html">View Javadoc</a></div><pre>
<a class="jxr_linenumber" name="L1" href="#L1">1</a>   <em class="jxr_comment">/*</em>
<a class="jxr_linenumber" name="L2" href="#L2">2</a>   <em class="jxr_comment"> *                    BioJava development code</em>
<a class="jxr_linenumber" name="L3" href="#L3">3</a>   <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L4" href="#L4">4</a>   <em class="jxr_comment"> * This code may be freely distributed and modified under the</em>
<a class="jxr_linenumber" name="L5" href="#L5">5</a>   <em class="jxr_comment"> * terms of the GNU Lesser General Public Licence.  This should</em>
<a class="jxr_linenumber" name="L6" href="#L6">6</a>   <em class="jxr_comment"> * be distributed with the code.  If you do not have a copy,</em>
<a class="jxr_linenumber" name="L7" href="#L7">7</a>   <em class="jxr_comment"> * see:</em>
<a class="jxr_linenumber" name="L8" href="#L8">8</a>   <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L9" href="#L9">9</a>   <em class="jxr_comment"> *      <a href="http://www.gnu.org/copyleft/lesser.htm" target="alexandria_uri">http://www.gnu.org/copyleft/lesser.htm</a>l</em>
<a class="jxr_linenumber" name="L10" href="#L10">10</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L11" href="#L11">11</a>  <em class="jxr_comment"> * Copyright for this code is held jointly by the individual</em>
<a class="jxr_linenumber" name="L12" href="#L12">12</a>  <em class="jxr_comment"> * authors.  These should be listed in @author doc comments.</em>
<a class="jxr_linenumber" name="L13" href="#L13">13</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L14" href="#L14">14</a>  <em class="jxr_comment"> * For more information on the BioJava project and its aims,</em>
<a class="jxr_linenumber" name="L15" href="#L15">15</a>  <em class="jxr_comment"> * or to join the biojava-l mailing list, visit the home page</em>
<a class="jxr_linenumber" name="L16" href="#L16">16</a>  <em class="jxr_comment"> * at:</em>
<a class="jxr_linenumber" name="L17" href="#L17">17</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L18" href="#L18">18</a>  <em class="jxr_comment"> *      <a href="http://www.biojava.org" target="alexandria_uri">http://www.biojava.org</a>/</em>
<a class="jxr_linenumber" name="L19" href="#L19">19</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L20" href="#L20">20</a>  <em class="jxr_comment"> * Created on DATE</em>
<a class="jxr_linenumber" name="L21" href="#L21">21</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L22" href="#L22">22</a>  <em class="jxr_comment"> */</em>
<a class="jxr_linenumber" name="L23" href="#L23">23</a>  <strong class="jxr_keyword">package</strong> org.biojava.nbio.core.sequence;
<a class="jxr_linenumber" name="L24" href="#L24">24</a>  
<a class="jxr_linenumber" name="L25" href="#L25">25</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.exceptions.CompoundNotFoundException;
<a class="jxr_linenumber" name="L26" href="#L26">26</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.compound.*;
<a class="jxr_linenumber" name="L27" href="#L27">27</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.features.FeatureInterface;
<a class="jxr_linenumber" name="L28" href="#L28">28</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.io.DNASequenceCreator;
<a class="jxr_linenumber" name="L29" href="#L29">29</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.io.FastaReader;
<a class="jxr_linenumber" name="L30" href="#L30">30</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.io.PlainFastaHeaderParser;
<a class="jxr_linenumber" name="L31" href="#L31">31</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.loader.StringProxySequenceReader;
<a class="jxr_linenumber" name="L32" href="#L32">32</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.location.InsdcParser;
<a class="jxr_linenumber" name="L33" href="#L33">33</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.location.template.Location;
<a class="jxr_linenumber" name="L34" href="#L34">34</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.template.AbstractSequence;
<a class="jxr_linenumber" name="L35" href="#L35">35</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.template.CompoundSet;
<a class="jxr_linenumber" name="L36" href="#L36">36</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.template.ProxySequenceReader;
<a class="jxr_linenumber" name="L37" href="#L37">37</a>  <strong class="jxr_keyword">import</strong> org.slf4j.Logger;
<a class="jxr_linenumber" name="L38" href="#L38">38</a>  <strong class="jxr_keyword">import</strong> org.slf4j.LoggerFactory;
<a class="jxr_linenumber" name="L39" href="#L39">39</a>  
<a class="jxr_linenumber" name="L40" href="#L40">40</a>  <strong class="jxr_keyword">import</strong> java.io.IOException;
<a class="jxr_linenumber" name="L41" href="#L41">41</a>  <strong class="jxr_keyword">import</strong> java.io.InputStream;
<a class="jxr_linenumber" name="L42" href="#L42">42</a>  <strong class="jxr_keyword">import</strong> java.net.URL;
<a class="jxr_linenumber" name="L43" href="#L43">43</a>  <strong class="jxr_keyword">import</strong> java.util.LinkedHashMap;
<a class="jxr_linenumber" name="L44" href="#L44">44</a>  
<a class="jxr_linenumber" name="L45" href="#L45">45</a>  <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L46" href="#L46">46</a>  <em class="jxr_javadoccomment"> * The representation of a ProteinSequence</em>
<a class="jxr_linenumber" name="L47" href="#L47">47</a>  <em class="jxr_javadoccomment"> *</em>
<a class="jxr_linenumber" name="L48" href="#L48">48</a>  <em class="jxr_javadoccomment"> * @author Scooter Willis</em>
<a class="jxr_linenumber" name="L49" href="#L49">49</a>  <em class="jxr_javadoccomment"> * @author Paolo Pavan</em>
<a class="jxr_linenumber" name="L50" href="#L50">50</a>  <em class="jxr_javadoccomment"> */</em>
<a class="jxr_linenumber" name="L51" href="#L51">51</a>  <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">class</strong> <a href="../../../../../org/biojava/nbio/core/sequence/ProteinSequence.html">ProteinSequence</a> <strong class="jxr_keyword">extends</strong> AbstractSequence&lt;AminoAcidCompound&gt; {
<a class="jxr_linenumber" name="L52" href="#L52">52</a>  
<a class="jxr_linenumber" name="L53" href="#L53">53</a>  	<strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">final</strong> <strong class="jxr_keyword">static</strong> Logger logger = LoggerFactory.getLogger(ProteinSequence.<strong class="jxr_keyword">class</strong>);
<a class="jxr_linenumber" name="L54" href="#L54">54</a>          
<a class="jxr_linenumber" name="L55" href="#L55">55</a>          <em class="jxr_comment">/*</em>
<a class="jxr_linenumber" name="L56" href="#L56">56</a>  <em class="jxr_comment">    private ArrayList&lt;FeatureInterface&lt;AbstractSequence&lt;AminoAcidCompound&gt;, AminoAcidCompound&gt;&gt; features</em>
<a class="jxr_linenumber" name="L57" href="#L57">57</a>  <em class="jxr_comment">            = new ArrayList&lt;FeatureInterface&lt;AbstractSequence&lt;AminoAcidCompound&gt;, AminoAcidCompound&gt;&gt;();</em>
<a class="jxr_linenumber" name="L58" href="#L58">58</a>  <em class="jxr_comment">    private LinkedHashMap&lt;String, ArrayList&lt;FeatureInterface&lt;AbstractSequence&lt;AminoAcidCompound&gt;, AminoAcidCompound&gt;&gt;&gt; groupedFeatures</em>
<a class="jxr_linenumber" name="L59" href="#L59">59</a>  <em class="jxr_comment">            = new LinkedHashMap&lt;String, ArrayList&lt;FeatureInterface&lt;AbstractSequence&lt;AminoAcidCompound&gt;, AminoAcidCompound&gt;&gt;&gt;();</em>
<a class="jxr_linenumber" name="L60" href="#L60">60</a>  <em class="jxr_comment">        */</em>
<a class="jxr_linenumber" name="L61" href="#L61">61</a>  
<a class="jxr_linenumber" name="L62" href="#L62">62</a>      <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L63" href="#L63">63</a>  <em class="jxr_javadoccomment">     * Create a protein from a string</em>
<a class="jxr_linenumber" name="L64" href="#L64">64</a>  <em class="jxr_javadoccomment">     *</em>
<a class="jxr_linenumber" name="L65" href="#L65">65</a>  <em class="jxr_javadoccomment">     * @param seqString</em>
<a class="jxr_linenumber" name="L66" href="#L66">66</a>  <em class="jxr_javadoccomment">     * @throws CompoundNotFoundException </em>
<a class="jxr_linenumber" name="L67" href="#L67">67</a>  <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L68" href="#L68">68</a>      <strong class="jxr_keyword">public</strong> <a href="../../../../../org/biojava/nbio/core/sequence/ProteinSequence.html">ProteinSequence</a>(String seqString) <strong class="jxr_keyword">throws</strong> CompoundNotFoundException { 
<a class="jxr_linenumber" name="L69" href="#L69">69</a>          <strong class="jxr_keyword">this</strong>(seqString, AminoAcidCompoundSet.getAminoAcidCompoundSet());
<a class="jxr_linenumber" name="L70" href="#L70">70</a>      }
<a class="jxr_linenumber" name="L71" href="#L71">71</a>  
<a class="jxr_linenumber" name="L72" href="#L72">72</a>      <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L73" href="#L73">73</a>  <em class="jxr_javadoccomment">     * Create a protein from a string with a user defined set of amino acids</em>
<a class="jxr_linenumber" name="L74" href="#L74">74</a>  <em class="jxr_javadoccomment">     *</em>
<a class="jxr_linenumber" name="L75" href="#L75">75</a>  <em class="jxr_javadoccomment">     * @param seqString</em>
<a class="jxr_linenumber" name="L76" href="#L76">76</a>  <em class="jxr_javadoccomment">     * @param compoundSet</em>
<a class="jxr_linenumber" name="L77" href="#L77">77</a>  <em class="jxr_javadoccomment">     * @throws CompoundNotFoundException </em>
<a class="jxr_linenumber" name="L78" href="#L78">78</a>  <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L79" href="#L79">79</a>      <strong class="jxr_keyword">public</strong> <a href="../../../../../org/biojava/nbio/core/sequence/ProteinSequence.html">ProteinSequence</a>(String seqString, CompoundSet&lt;AminoAcidCompound&gt; compoundSet) <strong class="jxr_keyword">throws</strong> CompoundNotFoundException {
<a class="jxr_linenumber" name="L80" href="#L80">80</a>          <strong class="jxr_keyword">super</strong>(seqString, compoundSet);
<a class="jxr_linenumber" name="L81" href="#L81">81</a>      }
<a class="jxr_linenumber" name="L82" href="#L82">82</a>  
<a class="jxr_linenumber" name="L83" href="#L83">83</a>      <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L84" href="#L84">84</a>  <em class="jxr_javadoccomment">     * A protein sequence where the storage of the sequence is somewhere else.</em>
<a class="jxr_linenumber" name="L85" href="#L85">85</a>  <em class="jxr_javadoccomment">     * Could be loaded from a large Fasta file or via a Uniprot Proxy reader via</em>
<a class="jxr_linenumber" name="L86" href="#L86">86</a>  <em class="jxr_javadoccomment">     * Uniprot ID</em>
<a class="jxr_linenumber" name="L87" href="#L87">87</a>  <em class="jxr_javadoccomment">     *</em>
<a class="jxr_linenumber" name="L88" href="#L88">88</a>  <em class="jxr_javadoccomment">     * @param proxyLoader</em>
<a class="jxr_linenumber" name="L89" href="#L89">89</a>  <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L90" href="#L90">90</a>      <strong class="jxr_keyword">public</strong> <a href="../../../../../org/biojava/nbio/core/sequence/ProteinSequence.html">ProteinSequence</a>(ProxySequenceReader&lt;AminoAcidCompound&gt; proxyLoader) {
<a class="jxr_linenumber" name="L91" href="#L91">91</a>          <strong class="jxr_keyword">this</strong>(proxyLoader, AminoAcidCompoundSet.getAminoAcidCompoundSet());
<a class="jxr_linenumber" name="L92" href="#L92">92</a>      }
<a class="jxr_linenumber" name="L93" href="#L93">93</a>  
<a class="jxr_linenumber" name="L94" href="#L94">94</a>      <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L95" href="#L95">95</a>  <em class="jxr_javadoccomment">     * A protein sequence where the storage of the sequence is somewhere else</em>
<a class="jxr_linenumber" name="L96" href="#L96">96</a>  <em class="jxr_javadoccomment">     * with user defined set of amino acids. Could be loaded from a large Fasta</em>
<a class="jxr_linenumber" name="L97" href="#L97">97</a>  <em class="jxr_javadoccomment">     * file or via a Uniprot Proxy reader via Uniprot ID</em>
<a class="jxr_linenumber" name="L98" href="#L98">98</a>  <em class="jxr_javadoccomment">     *</em>
<a class="jxr_linenumber" name="L99" href="#L99">99</a>  <em class="jxr_javadoccomment">     * @param proxyLoader</em>
<a class="jxr_linenumber" name="L100" href="#L100">100</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L101" href="#L101">101</a>     <strong class="jxr_keyword">public</strong> <a href="../../../../../org/biojava/nbio/core/sequence/ProteinSequence.html">ProteinSequence</a>(ProxySequenceReader&lt;AminoAcidCompound&gt; proxyLoader, CompoundSet&lt;AminoAcidCompound&gt; compoundSet) {
<a class="jxr_linenumber" name="L102" href="#L102">102</a>         <strong class="jxr_keyword">super</strong>(proxyLoader, compoundSet);
<a class="jxr_linenumber" name="L103" href="#L103">103</a>     }
<a class="jxr_linenumber" name="L104" href="#L104">104</a> 
<a class="jxr_linenumber" name="L105" href="#L105">105</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L106" href="#L106">106</a> <em class="jxr_javadoccomment">     * A Protein sequence can be stand alone or loaded from a transcript</em>
<a class="jxr_linenumber" name="L107" href="#L107">107</a> <em class="jxr_javadoccomment">     * sequence. The design goal is to allow the creation of a Protein sequence</em>
<a class="jxr_linenumber" name="L108" href="#L108">108</a> <em class="jxr_javadoccomment">     * from a Uniprot ID or some other Protein ID that based on cross reference</em>
<a class="jxr_linenumber" name="L109" href="#L109">109</a> <em class="jxr_javadoccomment">     * you should be able to get the GeneSequence that codes for the protein if</em>
<a class="jxr_linenumber" name="L110" href="#L110">110</a> <em class="jxr_javadoccomment">     * the CDS/Gene region is known. From the GeneSequence you should then be</em>
<a class="jxr_linenumber" name="L111" href="#L111">111</a> <em class="jxr_javadoccomment">     * able to get the ChromosomeSequence which then allows you explore flaning</em>
<a class="jxr_linenumber" name="L112" href="#L112">112</a> <em class="jxr_javadoccomment">     * regions of the gene sequences. The framework is in place to do this but</em>
<a class="jxr_linenumber" name="L113" href="#L113">113</a> <em class="jxr_javadoccomment">     * currently hasn't been implement in the reverse direction starting from</em>
<a class="jxr_linenumber" name="L114" href="#L114">114</a> <em class="jxr_javadoccomment">     * the Protein sequence.</em>
<a class="jxr_linenumber" name="L115" href="#L115">115</a> <em class="jxr_javadoccomment">     *</em>
<a class="jxr_linenumber" name="L116" href="#L116">116</a> <em class="jxr_javadoccomment">     * @param parentDNASequence</em>
<a class="jxr_linenumber" name="L117" href="#L117">117</a> <em class="jxr_javadoccomment">     * @param begin</em>
<a class="jxr_linenumber" name="L118" href="#L118">118</a> <em class="jxr_javadoccomment">     * @param end</em>
<a class="jxr_linenumber" name="L119" href="#L119">119</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L120" href="#L120">120</a>     <em class="jxr_comment">//TODO - Someone needs to check if this is a bug.  Shouldn't a parentDNASequence be something other then AminoAcid?</em>
<a class="jxr_linenumber" name="L121" href="#L121">121</a>     <em class="jxr_comment">//However, due to the derivation of this class, this is the only possible type argument for this parameter...</em>
<a class="jxr_linenumber" name="L122" href="#L122">122</a>     <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> setParentDNASequence(AbstractSequence&lt;NucleotideCompound&gt; parentDNASequence, Integer begin, Integer end) {
<a class="jxr_linenumber" name="L123" href="#L123">123</a>         <strong class="jxr_keyword">this</strong>.setParentSequence(parentDNASequence);
<a class="jxr_linenumber" name="L124" href="#L124">124</a>         setBioBegin(begin);
<a class="jxr_linenumber" name="L125" href="#L125">125</a>         setBioEnd(end);
<a class="jxr_linenumber" name="L126" href="#L126">126</a>     }
<a class="jxr_linenumber" name="L127" href="#L127">127</a> 
<a class="jxr_linenumber" name="L128" href="#L128">128</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L129" href="#L129">129</a> <em class="jxr_javadoccomment">     * Add feature.</em>
<a class="jxr_linenumber" name="L130" href="#L130">130</a> <em class="jxr_javadoccomment">     * &lt;p&gt;</em>
<a class="jxr_linenumber" name="L131" href="#L131">131</a> <em class="jxr_javadoccomment">     * If feature is type 'coded_by' than resolves parent DNA sequence.</em>
<a class="jxr_linenumber" name="L132" href="#L132">132</a> <em class="jxr_javadoccomment">     * &lt;/p&gt;</em>
<a class="jxr_linenumber" name="L133" href="#L133">133</a> <em class="jxr_javadoccomment">     * @param feature </em>
<a class="jxr_linenumber" name="L134" href="#L134">134</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L135" href="#L135">135</a>     @Override
<a class="jxr_linenumber" name="L136" href="#L136">136</a>     <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> addFeature(FeatureInterface&lt;AbstractSequence&lt;AminoAcidCompound&gt;, AminoAcidCompound&gt; feature) {
<a class="jxr_linenumber" name="L137" href="#L137">137</a>         <strong class="jxr_keyword">super</strong>.addFeature(feature);
<a class="jxr_linenumber" name="L138" href="#L138">138</a> 
<a class="jxr_linenumber" name="L139" href="#L139">139</a>         <em class="jxr_comment">// if feature is called 'coded_by' than add parent DNA location</em>
<a class="jxr_linenumber" name="L140" href="#L140">140</a>         <strong class="jxr_keyword">if</strong> (feature.getType().equals(<span class="jxr_string">"coded_by"</span>)) {
<a class="jxr_linenumber" name="L141" href="#L141">141</a>             <a href="../../../../../org/biojava/nbio/core/sequence/location/InsdcParser.html">InsdcParser</a> parser = <strong class="jxr_keyword">new</strong> <a href="../../../../../org/biojava/nbio/core/sequence/location/InsdcParser.html">InsdcParser</a>(DataSource.GENBANK);
<a class="jxr_linenumber" name="L142" href="#L142">142</a> 
<a class="jxr_linenumber" name="L143" href="#L143">143</a>             <a href="../../../../../org/biojava/nbio/core/sequence/location/template/Location.html">Location</a> location = parser.parse(feature.getSource());
<a class="jxr_linenumber" name="L144" href="#L144">144</a>             <em class="jxr_comment">// convert location into DNASequence</em>
<a class="jxr_linenumber" name="L145" href="#L145">145</a>             <strong class="jxr_keyword">try</strong> {
<a class="jxr_linenumber" name="L146" href="#L146">146</a>             	DNASequence dnaSeq = <strong class="jxr_keyword">new</strong> <a href="../../../../../org/biojava/nbio/core/sequence/DNASequence.html">DNASequence</a>(getSequence(location), DNACompoundSet.getDNACompoundSet());
<a class="jxr_linenumber" name="L147" href="#L147">147</a>             	setParentDNASequence(dnaSeq, location.getStart().getPosition(), location.getEnd().getPosition());
<a class="jxr_linenumber" name="L148" href="#L148">148</a>             } <strong class="jxr_keyword">catch</strong> (CompoundNotFoundException e) {
<a class="jxr_linenumber" name="L149" href="#L149">149</a>             	<em class="jxr_comment">// TODO is there another solution to handle this exception?</em>
<a class="jxr_linenumber" name="L150" href="#L150">150</a>             	logger.error(<span class="jxr_string">"Could not add 'coded_by' parent DNA location feature, unrecognised compounds found in DNA sequence: {}"</span>,e.getMessage());
<a class="jxr_linenumber" name="L151" href="#L151">151</a>             }
<a class="jxr_linenumber" name="L152" href="#L152">152</a>         }
<a class="jxr_linenumber" name="L153" href="#L153">153</a>     }
<a class="jxr_linenumber" name="L154" href="#L154">154</a>     
<a class="jxr_linenumber" name="L155" href="#L155">155</a>     <strong class="jxr_keyword">private</strong> <a href="../../../../../org/biojava/nbio/core/sequence/DNASequence.html">DNASequence</a> getRawParentSequence(String accessId) <strong class="jxr_keyword">throws</strong> IOException {
<a class="jxr_linenumber" name="L156" href="#L156">156</a>         String seqUrlTemplate = <span class="jxr_string">"http://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore&amp;id=%s&amp;rettype=fasta&amp;retmode=text"</span>;
<a class="jxr_linenumber" name="L157" href="#L157">157</a>         URL url = <strong class="jxr_keyword">new</strong> URL(String.format(seqUrlTemplate, accessId));
<a class="jxr_linenumber" name="L158" href="#L158">158</a>         
<a class="jxr_linenumber" name="L159" href="#L159">159</a>         logger.info(<span class="jxr_string">"Getting parent DNA sequence from URL: {}"</span>, url.toString());
<a class="jxr_linenumber" name="L160" href="#L160">160</a>         
<a class="jxr_linenumber" name="L161" href="#L161">161</a>         InputStream is = url.openConnection().getInputStream();
<a class="jxr_linenumber" name="L162" href="#L162">162</a> 
<a class="jxr_linenumber" name="L163" href="#L163">163</a>         FastaReader&lt;DNASequence, NucleotideCompound&gt; parentReader
<a class="jxr_linenumber" name="L164" href="#L164">164</a>                 = <strong class="jxr_keyword">new</strong> FastaReader&lt;DNASequence, NucleotideCompound&gt;(is,
<a class="jxr_linenumber" name="L165" href="#L165">165</a>                         <strong class="jxr_keyword">new</strong> PlainFastaHeaderParser&lt;DNASequence, NucleotideCompound&gt;(),
<a class="jxr_linenumber" name="L166" href="#L166">166</a>                         <strong class="jxr_keyword">new</strong> <a href="../../../../../org/biojava/nbio/core/sequence/io/DNASequenceCreator.html">DNASequenceCreator</a>(AmbiguityDNACompoundSet.getDNACompoundSet()));
<a class="jxr_linenumber" name="L167" href="#L167">167</a>         LinkedHashMap&lt;String, DNASequence&gt; seq = parentReader.process();
<a class="jxr_linenumber" name="L168" href="#L168">168</a> 
<a class="jxr_linenumber" name="L169" href="#L169">169</a>         <a href="../../../../../org/biojava/nbio/core/sequence/DNASequence.html">DNASequence</a> parentSeq = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L170" href="#L170">170</a>         <strong class="jxr_keyword">if</strong> (seq.size() == 1) {
<a class="jxr_linenumber" name="L171" href="#L171">171</a>             parentSeq = seq.values().iterator().next();
<a class="jxr_linenumber" name="L172" href="#L172">172</a>         }
<a class="jxr_linenumber" name="L173" href="#L173">173</a>         is.close();
<a class="jxr_linenumber" name="L174" href="#L174">174</a> 
<a class="jxr_linenumber" name="L175" href="#L175">175</a>         <strong class="jxr_keyword">return</strong> parentSeq;
<a class="jxr_linenumber" name="L176" href="#L176">176</a>     }
<a class="jxr_linenumber" name="L177" href="#L177">177</a> 
<a class="jxr_linenumber" name="L178" href="#L178">178</a>     <strong class="jxr_keyword">private</strong> String getSequence(<a href="../../../../../org/biojava/nbio/core/sequence/location/template/Location.html">Location</a> cdna) {
<a class="jxr_linenumber" name="L179" href="#L179">179</a>         <a href="../../../../../org/biojava/nbio/core/sequence/DNASequence.html">DNASequence</a> rawParent = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L180" href="#L180">180</a>         <strong class="jxr_keyword">if</strong> (!cdna.isComplex()) {
<a class="jxr_linenumber" name="L181" href="#L181">181</a>             <strong class="jxr_keyword">try</strong> {
<a class="jxr_linenumber" name="L182" href="#L182">182</a>                 rawParent = getRawParentSequence(cdna.getAccession().getID());
<a class="jxr_linenumber" name="L183" href="#L183">183</a>                 <strong class="jxr_keyword">return</strong> cdna.getSubSequence(rawParent).getSequenceAsString();
<a class="jxr_linenumber" name="L184" href="#L184">184</a>             } <strong class="jxr_keyword">catch</strong> (IOException e) {
<a class="jxr_linenumber" name="L185" href="#L185">185</a>                 <em class="jxr_comment">// return null</em>
<a class="jxr_linenumber" name="L186" href="#L186">186</a>             	logger.error(<span class="jxr_string">"Caught IOException when getting DNA sequence for id {}. Error: {}"</span>, cdna.getAccession().getID(), e.getMessage());
<a class="jxr_linenumber" name="L187" href="#L187">187</a>                 <strong class="jxr_keyword">return</strong> <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L188" href="#L188">188</a>             }
<a class="jxr_linenumber" name="L189" href="#L189">189</a>         } <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L190" href="#L190">190</a>             <em class="jxr_comment">// in case of complex</em>
<a class="jxr_linenumber" name="L191" href="#L191">191</a>             StringBuilder sb = <strong class="jxr_keyword">new</strong> StringBuilder();
<a class="jxr_linenumber" name="L192" href="#L192">192</a> 
<a class="jxr_linenumber" name="L193" href="#L193">193</a>             <strong class="jxr_keyword">for</strong> (Location sub : cdna.getSubLocations()) {
<a class="jxr_linenumber" name="L194" href="#L194">194</a>                 String sebStr = getSequence(sub);
<a class="jxr_linenumber" name="L195" href="#L195">195</a>                 sb.append((sebStr == <strong class="jxr_keyword">null</strong> ? <span class="jxr_string">""</span> : sebStr));
<a class="jxr_linenumber" name="L196" href="#L196">196</a>             }
<a class="jxr_linenumber" name="L197" href="#L197">197</a> 
<a class="jxr_linenumber" name="L198" href="#L198">198</a>             <strong class="jxr_keyword">return</strong> sb.toString();
<a class="jxr_linenumber" name="L199" href="#L199">199</a>         }
<a class="jxr_linenumber" name="L200" href="#L200">200</a>     }
<a class="jxr_linenumber" name="L201" href="#L201">201</a> 
<a class="jxr_linenumber" name="L202" href="#L202">202</a>     <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">void</strong> main(String[] args) <strong class="jxr_keyword">throws</strong> Exception {
<a class="jxr_linenumber" name="L203" href="#L203">203</a>         <a href="../../../../../org/biojava/nbio/core/sequence/ProteinSequence.html">ProteinSequence</a> proteinSequence = <strong class="jxr_keyword">new</strong> <a href="../../../../../org/biojava/nbio/core/sequence/ProteinSequence.html">ProteinSequence</a>(<span class="jxr_string">"ARNDCEQGHILKMFPSTWYVBZJX"</span>);
<a class="jxr_linenumber" name="L204" href="#L204">204</a>         logger.info(<span class="jxr_string">"Protein Sequence: {}"</span>, proteinSequence.toString());
<a class="jxr_linenumber" name="L205" href="#L205">205</a> 
<a class="jxr_linenumber" name="L206" href="#L206">206</a>         StringProxySequenceReader&lt;AminoAcidCompound&gt; sequenceStringProxyLoader = <strong class="jxr_keyword">new</strong> StringProxySequenceReader&lt;AminoAcidCompound&gt;(<span class="jxr_string">"XRNDCEQGHILKMFPSTWYVBZJA"</span>, AminoAcidCompoundSet.getAminoAcidCompoundSet());
<a class="jxr_linenumber" name="L207" href="#L207">207</a>         <a href="../../../../../org/biojava/nbio/core/sequence/ProteinSequence.html">ProteinSequence</a> proteinSequenceFromProxy = <strong class="jxr_keyword">new</strong> <a href="../../../../../org/biojava/nbio/core/sequence/ProteinSequence.html">ProteinSequence</a>(sequenceStringProxyLoader);
<a class="jxr_linenumber" name="L208" href="#L208">208</a>         logger.info(<span class="jxr_string">"Protein Sequence from Proxy: {}"</span>, proteinSequenceFromProxy.toString());
<a class="jxr_linenumber" name="L209" href="#L209">209</a> 
<a class="jxr_linenumber" name="L210" href="#L210">210</a>     }
<a class="jxr_linenumber" name="L211" href="#L211">211</a> }
</pre>
<hr/>
<div id="footer">Copyright &#169; 2000&#x2013;2015 <a href="http://www.biojava.org">BioJava</a>. All rights reserved.</div>
</body>
</html>
