<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_71) on Fri Jan 30 08:16:01 PST 2015 -->
<meta http-equiv="Content-Type" content="text/html" charset="UTF-8">
<title>Index (biojava-protein-disorder 4.0.0 API)</title>
<meta name="date" content="2015-01-30">
<link rel="stylesheet" type="text/css" href="./stylesheet.css" title="Style">
</head>
<body>
<script type="text/javascript"><!--
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="Index (biojava-protein-disorder 4.0.0 API)";
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar_top">
<!--   -->
</a><a href="#skip-navbar_top" title="Skip navigation links"></a><a name="navbar_top_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="./overview-summary.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li>Use</li>
<li><a href="./overview-tree.html">Tree</a></li>
<li><a href="./deprecated-list.html">Deprecated</a></li>
<li class="navBarCell1Rev">Index</li>
<li><a href="./help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="./index.html?index-all.html" target="_top">Frames</a></li>
<li><a href="index-all.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="./allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<div class="contentContainer"><a href="#_A_">A</a>&nbsp;<a href="#_B_">B</a>&nbsp;<a href="#_C_">C</a>&nbsp;<a href="#_D_">D</a>&nbsp;<a href="#_E_">E</a>&nbsp;<a href="#_F_">F</a>&nbsp;<a href="#_G_">G</a>&nbsp;<a href="#_H_">H</a>&nbsp;<a href="#_I_">I</a>&nbsp;<a href="#_J_">J</a>&nbsp;<a href="#_M_">M</a>&nbsp;<a href="#_N_">N</a>&nbsp;<a href="#_O_">O</a>&nbsp;<a href="#_P_">P</a>&nbsp;<a href="#_R_">R</a>&nbsp;<a href="#_S_">S</a>&nbsp;<a href="#_T_">T</a>&nbsp;<a href="#_V_">V</a>&nbsp;<a href="#_W_">W</a>&nbsp;<a name="_A_">
<!--   -->
</a>
<h2 class="title">A</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/SequenceUtil.html#AA">AA</a></span> - Static variable in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/SequenceUtil.html" title="class in org.biojava.nbio.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Valid Amino acids</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/SequenceUtil.html#AMBIGUOUS_AA">AMBIGUOUS_AA</a></span> - Static variable in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/SequenceUtil.html" title="class in org.biojava.nbio.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Same as AA pattern but with two additional letters - XU</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/SequenceUtil.html#AMBIGUOUS_NUCLEOTIDE">AMBIGUOUS_NUCLEOTIDE</a></span> - Static variable in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/SequenceUtil.html" title="class in org.biojava.nbio.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Ambiguous nucleotide</div>
</dd>
</dl>
<a name="_B_">
<!--   -->
</a>
<h2 class="title">B</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/RonnConstraint.html#Blosum62">Blosum62</a></span> - Static variable in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/RonnConstraint.html" title="class in org.biojava.nbio.ronn">RonnConstraint</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_C_">
<!--   -->
</a>
<h2 class="title">C</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/ORonn.html#call()">call()</a></span> - Method in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/ORonn.html" title="class in org.biojava.nbio.ronn">ORonn</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/Timer.html#checkPoint()">checkPoint()</a></span> - Method in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/Timer.html" title="class in org.biojava.nbio.ronn">Timer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/SequenceUtil.html#cleanSequence(java.lang.String)">cleanSequence(String)</a></span> - Static method in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/SequenceUtil.html" title="class in org.biojava.nbio.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Removes all whitespace chars in the sequence string</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/FastaSequence.html#compareTo(org.biojava.nbio.data.sequence.FastaSequence)">compareTo(FastaSequence)</a></span> - Method in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/FastaSequence.html" title="class in org.biojava.nbio.data.sequence">FastaSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/Jronn.html#convertProteinSequencetoFasta(org.biojava.nbio.core.sequence.ProteinSequence)">convertProteinSequencetoFasta(ProteinSequence)</a></span> - Static method in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/Jronn.html" title="class in org.biojava.nbio.ronn">Jronn</a></dt>
<dd>
<div class="block">Utility method to convert a BioJava ProteinSequence object to the FastaSequence 
  object used internally in JRonn.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/FastaSequence.html#countMatchesInSequence(java.lang.String,%20java.lang.String)">countMatchesInSequence(String, String)</a></span> - Static method in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/FastaSequence.html" title="class in org.biojava.nbio.data.sequence">FastaSequence</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_D_">
<!--   -->
</a>
<h2 class="title">D</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/SequenceUtil.html#deepCleanSequence(java.lang.String)">deepCleanSequence(String)</a></span> - Static method in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/SequenceUtil.html" title="class in org.biojava.nbio.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Removes all special characters and digits as well as whitespace chars
 from the sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/RonnConstraint.html#DEFAULT_DISORDER">DEFAULT_DISORDER</a></span> - Static variable in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/RonnConstraint.html" title="class in org.biojava.nbio.ronn">RonnConstraint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/RonnConstraint.html#DEFAULT_ORDER">DEFAULT_ORDER</a></span> - Static variable in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/RonnConstraint.html" title="class in org.biojava.nbio.ronn">RonnConstraint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/RonnConstraint.html#DEFAULT_RANGE_PROBABILITY_THRESHOLD">DEFAULT_RANGE_PROBABILITY_THRESHOLD</a></span> - Static variable in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/RonnConstraint.html" title="class in org.biojava.nbio.ronn">RonnConstraint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/package-summary.html">demo</a> - package demo</dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/SequenceUtil.html#DIGIT">DIGIT</a></span> - Static variable in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/SequenceUtil.html" title="class in org.biojava.nbio.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">A digit</div>
</dd>
</dl>
<a name="_E_">
<!--   -->
</a>
<h2 class="title">E</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/FastaSequence.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/FastaSequence.html" title="class in org.biojava.nbio.data.sequence">FastaSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/Jronn.Range.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/Jronn.Range.html" title="class in org.biojava.nbio.ronn">Jronn.Range</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/ModelLoader.Model.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/ModelLoader.Model.html" title="class in org.biojava.nbio.ronn">ModelLoader.Model</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_F_">
<!--   -->
</a>
<h2 class="title">F</h2>
<dl>
<dt><a href="./org/biojava/nbio/data/sequence/FastaSequence.html" title="class in org.biojava.nbio.data.sequence"><span class="strong">FastaSequence</span></a> - Class in <a href="./org/biojava/nbio/data/sequence/package-summary.html">org.biojava.nbio.data.sequence</a></dt>
<dd>
<div class="block">A FASTA formatted sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/FastaSequence.html#FastaSequence(java.lang.String,%20java.lang.String)">FastaSequence(String, String)</a></span> - Constructor for class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/FastaSequence.html" title="class in org.biojava.nbio.data.sequence">FastaSequence</a></dt>
<dd>
<div class="block">Upon construction the any whitespace characters are removed from the
 sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/Jronn.Range.html#from">from</a></span> - Variable in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/Jronn.Range.html" title="class in org.biojava.nbio.ronn">Jronn.Range</a></dt>
<dd>
<div class="block">Range starting position counts from 1 (the first position on the sequence is 1)</div>
</dd>
</dl>
<a name="_G_">
<!--   -->
</a>
<h2 class="title">G</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/Jronn.html#getDisorder(org.biojava.nbio.data.sequence.FastaSequence)">getDisorder(FastaSequence)</a></span> - Static method in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/Jronn.html" title="class in org.biojava.nbio.ronn">Jronn</a></dt>
<dd>
<div class="block">Calculates the disordered regions of the sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/Jronn.html#getDisorder(java.util.List)">getDisorder(List&lt;FastaSequence&gt;)</a></span> - Static method in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/Jronn.html" title="class in org.biojava.nbio.ronn">Jronn</a></dt>
<dd>
<div class="block">Calculates the disordered regions of the sequence for many sequences in the input.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/Jronn.html#getDisorder(java.lang.String)">getDisorder(String)</a></span> - Static method in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/Jronn.html" title="class in org.biojava.nbio.ronn">Jronn</a></dt>
<dd>
<div class="block">Calculates the disordered regions of the protein sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/Jronn.html#getDisorderScores(org.biojava.nbio.data.sequence.FastaSequence)">getDisorderScores(FastaSequence)</a></span> - Static method in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/Jronn.html" title="class in org.biojava.nbio.ronn">Jronn</a></dt>
<dd>
<div class="block">Calculates the probability value for each residue in the protein sequence, 
 telling the probability that the residue belongs to disordered region.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/Jronn.html#getDisorderScores(org.biojava.nbio.core.sequence.ProteinSequence)">getDisorderScores(ProteinSequence)</a></span> - Static method in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/Jronn.html" title="class in org.biojava.nbio.ronn">Jronn</a></dt>
<dd>
<div class="block">Calculates the probability value for each residue in the protein sequence, 
 telling the probability that the residue belongs to disordered region.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/Jronn.html#getDisorderScores(java.util.List)">getDisorderScores(List&lt;FastaSequence&gt;)</a></span> - Static method in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/Jronn.html" title="class in org.biojava.nbio.ronn">Jronn</a></dt>
<dd>
<div class="block">Calculates the probability of disorder scores for each residue in the sequence for 
 many sequences in the input.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/FastaSequence.html#getFormatedSequence(int)">getFormatedSequence(int)</a></span> - Method in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/FastaSequence.html" title="class in org.biojava.nbio.data.sequence">FastaSequence</a></dt>
<dd>
<div class="block">Format sequence per width letter in one string.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/FastaSequence.html#getFormattedFasta()">getFormattedFasta()</a></span> - Method in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/FastaSequence.html" title="class in org.biojava.nbio.data.sequence">FastaSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/FastaSequence.html#getId()">getId()</a></span> - Method in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/FastaSequence.html" title="class in org.biojava.nbio.data.sequence">FastaSequence</a></dt>
<dd>
<div class="block">Gets the value of id</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/FastaSequence.html#getLength()">getLength()</a></span> - Method in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/FastaSequence.html" title="class in org.biojava.nbio.data.sequence">FastaSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/ModelLoader.html#getModel(int)">getModel(int)</a></span> - Method in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/ModelLoader.html" title="class in org.biojava.nbio.ronn">ModelLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/FastaSequence.html#getOnelineFasta()">getOnelineFasta()</a></span> - Method in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/FastaSequence.html" title="class in org.biojava.nbio.data.sequence">FastaSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/ORonnModel.html#getScores(java.io.File)">getScores(File)</a></span> - Method in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/ORonnModel.html" title="class in org.biojava.nbio.ronn">ORonnModel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/FastaSequence.html#getSequence()">getSequence()</a></span> - Method in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/FastaSequence.html" title="class in org.biojava.nbio.data.sequence">FastaSequence</a></dt>
<dd>
<div class="block">Gets the value of sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/RonnConstraint.Threshold.html#getTreshold(int)">getTreshold(int)</a></span> - Static method in enum org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/RonnConstraint.Threshold.html" title="enum in org.biojava.nbio.ronn">RonnConstraint.Threshold</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/RonnConstraint.Threshold.html#getValues()">getValues()</a></span> - Method in enum org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/RonnConstraint.Threshold.html" title="enum in org.biojava.nbio.ronn">RonnConstraint.Threshold</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_H_">
<!--   -->
</a>
<h2 class="title">H</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/FastaSequence.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/FastaSequence.html" title="class in org.biojava.nbio.data.sequence">FastaSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/Jronn.Range.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/Jronn.Range.html" title="class in org.biojava.nbio.ronn">Jronn.Range</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/ModelLoader.Model.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/ModelLoader.Model.html" title="class in org.biojava.nbio.ronn">ModelLoader.Model</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_I_">
<!--   -->
</a>
<h2 class="title">I</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/RonnConstraint.html#INDEX">INDEX</a></span> - Static variable in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/RonnConstraint.html" title="class in org.biojava.nbio.ronn">RonnConstraint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/SequenceUtil.html#isAmbiguosProtein(java.lang.String)">isAmbiguosProtein(String)</a></span> - Static method in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/SequenceUtil.html" title="class in org.biojava.nbio.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Check whether the sequence confirms to amboguous protein sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/SequenceUtil.html#isNonAmbNucleotideSequence(java.lang.String)">isNonAmbNucleotideSequence(String)</a></span> - Static method in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/SequenceUtil.html" title="class in org.biojava.nbio.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Ambiguous DNA chars : AGTCRYMKSWHBVDN // differs from protein in only one
 (!) - B char</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/SequenceUtil.html#isNucleotideSequence(org.biojava.nbio.data.sequence.FastaSequence)">isNucleotideSequence(FastaSequence)</a></span> - Static method in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/SequenceUtil.html" title="class in org.biojava.nbio.data.sequence">SequenceUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/SequenceUtil.html#isProteinSequence(java.lang.String)">isProteinSequence(String)</a></span> - Static method in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/SequenceUtil.html" title="class in org.biojava.nbio.data.sequence">SequenceUtil</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_J_">
<!--   -->
</a>
<h2 class="title">J</h2>
<dl>
<dt><a href="./org/biojava/nbio/ronn/Jronn.html" title="class in org.biojava.nbio.ronn"><span class="strong">Jronn</span></a> - Class in <a href="./org/biojava/nbio/ronn/package-summary.html">org.biojava.nbio.ronn</a></dt>
<dd>
<div class="block">This class gives public API to RONN functions.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/Jronn.html#Jronn()">Jronn()</a></span> - Constructor for class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/Jronn.html" title="class in org.biojava.nbio.ronn">Jronn</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/ronn/Jronn.Range.html" title="class in org.biojava.nbio.ronn"><span class="strong">Jronn.Range</span></a> - Class in <a href="./org/biojava/nbio/ronn/package-summary.html">org.biojava.nbio.ronn</a></dt>
<dd>
<div class="block">Holder for the ranges, contain pointers to starting and ending position 
 on the sequence which comprises a disordered region.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/Jronn.Range.html#Jronn.Range(int,%20int,%20float)">Jronn.Range(int, int, float)</a></span> - Constructor for class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/Jronn.Range.html" title="class in org.biojava.nbio.ronn">Jronn.Range</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_M_">
<!--   -->
</a>
<h2 class="title">M</h2>
<dl>
<dt><span class="strong"><a href="./demo/PredictDisorder.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/PredictDisorder.html" title="class in demo">PredictDisorder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/ModelLoader.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/ModelLoader.html" title="class in org.biojava.nbio.ronn">ModelLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/ORonn.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/ORonn.html" title="class in org.biojava.nbio.ronn">ORonn</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/ronn/ModelLoader.html" title="class in org.biojava.nbio.ronn"><span class="strong">ModelLoader</span></a> - Class in <a href="./org/biojava/nbio/ronn/package-summary.html">org.biojava.nbio.ronn</a></dt>
<dd>
<div class="block">Class that loads data from the model files into <a href="./org/biojava/nbio/ronn/ModelLoader.Model.html" title="class in org.biojava.nbio.ronn"><code>ModelLoader.Model</code></a> objects</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/ModelLoader.html#ModelLoader()">ModelLoader()</a></span> - Constructor for class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/ModelLoader.html" title="class in org.biojava.nbio.ronn">ModelLoader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/ronn/ModelLoader.Model.html" title="class in org.biojava.nbio.ronn"><span class="strong">ModelLoader.Model</span></a> - Class in <a href="./org/biojava/nbio/ronn/package-summary.html">org.biojava.nbio.ronn</a></dt>
<dd>
<div class="block">Represent a RONN model</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/ModelLoader.Model.html#ModelLoader.Model(int,%20int)">ModelLoader.Model(int, int)</a></span> - Constructor for class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/ModelLoader.Model.html" title="class in org.biojava.nbio.ronn">ModelLoader.Model</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/ronn/ModelLoader.Threshold.html" title="class in org.biojava.nbio.ronn"><span class="strong">ModelLoader.Threshold</span></a> - Class in <a href="./org/biojava/nbio/ronn/package-summary.html">org.biojava.nbio.ronn</a></dt>
<dd>
<div class="block">Represents a Threshold</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/ModelLoader.Threshold.html#ModelLoader.Threshold(int)">ModelLoader.Threshold(int)</a></span> - Constructor for class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/ModelLoader.Threshold.html" title="class in org.biojava.nbio.ronn">ModelLoader.Threshold</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_N_">
<!--   -->
</a>
<h2 class="title">N</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/SequenceUtil.html#NON_AA">NON_AA</a></span> - Static variable in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/SequenceUtil.html" title="class in org.biojava.nbio.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">inversion of AA pattern</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/SequenceUtil.html#NON_NUCLEOTIDE">NON_NUCLEOTIDE</a></span> - Static variable in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/SequenceUtil.html" title="class in org.biojava.nbio.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Non nucleotide</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/SequenceUtil.html#NONWORD">NONWORD</a></span> - Static variable in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/SequenceUtil.html" title="class in org.biojava.nbio.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Non word</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/SequenceUtil.html#NUCLEOTIDE">NUCLEOTIDE</a></span> - Static variable in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/SequenceUtil.html" title="class in org.biojava.nbio.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Nucleotides a, t, g, c, u</div>
</dd>
<dt><a href="./org/biojava/nbio/ronn/NullOutputStream.html" title="class in org.biojava.nbio.ronn"><span class="strong">NullOutputStream</span></a> - Class in <a href="./org/biojava/nbio/ronn/package-summary.html">org.biojava.nbio.ronn</a></dt>
<dd>
<div class="block">The stream that void its input</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/NullOutputStream.html#NullOutputStream()">NullOutputStream()</a></span> - Constructor for class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/NullOutputStream.html" title="class in org.biojava.nbio.ronn">NullOutputStream</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_O_">
<!--   -->
</a>
<h2 class="title">O</h2>
<dl>
<dt><a href="./org/biojava/nbio/data/sequence/package-summary.html">org.biojava.nbio.data.sequence</a> - package org.biojava.nbio.data.sequence</dt>
<dd>
<div class="block">Set of classes that responsible for data handling.</div>
</dd>
<dt><a href="./org/biojava/nbio/ronn/package-summary.html">org.biojava.nbio.ronn</a> - package org.biojava.nbio.ronn</dt>
<dd>
<div class="block">
 BioJava provide a module biojava-protein-disorder for prediction disordered regions
 from a protein sequence.</div>
</dd>
<dt><a href="./org/biojava/nbio/ronn/ORonn.html" title="class in org.biojava.nbio.ronn"><span class="strong">ORonn</span></a> - Class in <a href="./org/biojava/nbio/ronn/package-summary.html">org.biojava.nbio.ronn</a></dt>
<dd>
<div class="block">Fully re-factored and enhanced version of RONN.</div>
</dd>
<dt><a href="./org/biojava/nbio/ronn/ORonnModel.html" title="class in org.biojava.nbio.ronn"><span class="strong">ORonnModel</span></a> - Class in <a href="./org/biojava/nbio/ronn/package-summary.html">org.biojava.nbio.ronn</a></dt>
<dd>
<div class="block">Fully re-factored version of RONN model.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/ORonnModel.html#ORonnModel(java.lang.String,%20org.biojava.nbio.ronn.ModelLoader.Model,%20float)">ORonnModel(String, ModelLoader.Model, float)</a></span> - Constructor for class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/ORonnModel.html" title="class in org.biojava.nbio.ronn">ORonnModel</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_P_">
<!--   -->
</a>
<h2 class="title">P</h2>
<dl>
<dt><a href="./demo/PredictDisorder.html" title="class in demo"><span class="strong">PredictDisorder</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/PredictDisorder.html#PredictDisorder()">PredictDisorder()</a></span> - Constructor for class demo.<a href="./demo/PredictDisorder.html" title="class in demo">PredictDisorder</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_R_">
<!--   -->
</a>
<h2 class="title">R</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/SequenceUtil.html#readFasta(java.io.InputStream)">readFasta(InputStream)</a></span> - Static method in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/SequenceUtil.html" title="class in org.biojava.nbio.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Reads fasta sequences from inStream into the list of FastaSequence
 objects</div>
</dd>
<dt><a href="./org/biojava/nbio/ronn/RonnConstraint.html" title="class in org.biojava.nbio.ronn"><span class="strong">RonnConstraint</span></a> - Class in <a href="./org/biojava/nbio/ronn/package-summary.html">org.biojava.nbio.ronn</a></dt>
<dd>
<div class="block">A collection of various constrain values used by RONN</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/RonnConstraint.html#RonnConstraint()">RonnConstraint()</a></span> - Constructor for class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/RonnConstraint.html" title="class in org.biojava.nbio.ronn">RonnConstraint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/ronn/RonnConstraint.Threshold.html" title="enum in org.biojava.nbio.ronn"><span class="strong">RonnConstraint.Threshold</span></a> - Enum in <a href="./org/biojava/nbio/ronn/package-summary.html">org.biojava.nbio.ronn</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_S_">
<!--   -->
</a>
<h2 class="title">S</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/Jronn.Range.html#score">score</a></span> - Variable in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/Jronn.Range.html" title="class in org.biojava.nbio.ronn">Jronn.Range</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/Jronn.html#scoresToRanges(float[],%20float)">scoresToRanges(float[], float)</a></span> - Static method in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/Jronn.html" title="class in org.biojava.nbio.ronn">Jronn</a></dt>
<dd>
<div class="block">Convert raw scores to ranges.</div>
</dd>
<dt><a href="./org/biojava/nbio/data/sequence/SequenceUtil.html" title="class in org.biojava.nbio.data.sequence"><span class="strong">SequenceUtil</span></a> - Class in <a href="./org/biojava/nbio/data/sequence/package-summary.html">org.biojava.nbio.data.sequence</a></dt>
<dd>
<div class="block">Utility class for operations on sequences</div>
</dd>
</dl>
<a name="_T_">
<!--   -->
</a>
<h2 class="title">T</h2>
<dl>
<dt><a href="./org/biojava/nbio/ronn/Timer.html" title="class in org.biojava.nbio.ronn"><span class="strong">Timer</span></a> - Class in <a href="./org/biojava/nbio/ronn/package-summary.html">org.biojava.nbio.ronn</a></dt>
<dd>
<div class="block">A simple timer, calculates the time interval between two events.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/Timer.html#Timer()">Timer()</a></span> - Constructor for class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/Timer.html" title="class in org.biojava.nbio.ronn">Timer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/Timer.html#Timer(java.util.concurrent.TimeUnit)">Timer(TimeUnit)</a></span> - Constructor for class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/Timer.html" title="class in org.biojava.nbio.ronn">Timer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/Jronn.Range.html#to">to</a></span> - Variable in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/Jronn.Range.html" title="class in org.biojava.nbio.ronn">Jronn.Range</a></dt>
<dd>
<div class="block">The range ending position includes the last residue.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/FastaSequence.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/FastaSequence.html" title="class in org.biojava.nbio.data.sequence">FastaSequence</a></dt>
<dd>
<div class="block">Same as oneLineFasta</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/Jronn.Range.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/Jronn.Range.html" title="class in org.biojava.nbio.ronn">Jronn.Range</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/ModelLoader.Model.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/ModelLoader.Model.html" title="class in org.biojava.nbio.ronn">ModelLoader.Model</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_V_">
<!--   -->
</a>
<h2 class="title">V</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/RonnConstraint.Threshold.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/RonnConstraint.Threshold.html" title="enum in org.biojava.nbio.ronn">RonnConstraint.Threshold</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/RonnConstraint.Threshold.html#values()">values()</a></span> - Static method in enum org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/RonnConstraint.Threshold.html" title="enum in org.biojava.nbio.ronn">RonnConstraint.Threshold</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
</dl>
<a name="_W_">
<!--   -->
</a>
<h2 class="title">W</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/SequenceUtil.html#WHITE_SPACE">WHITE_SPACE</a></span> - Static variable in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/SequenceUtil.html" title="class in org.biojava.nbio.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">A whitespace character: [\t\n\x0B\f\r]</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/ronn/NullOutputStream.html#write(int)">write(int)</a></span> - Method in class org.biojava.nbio.ronn.<a href="./org/biojava/nbio/ronn/NullOutputStream.html" title="class in org.biojava.nbio.ronn">NullOutputStream</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/SequenceUtil.html#writeFasta(java.io.OutputStream,%20java.util.List,%20int)">writeFasta(OutputStream, List&lt;FastaSequence&gt;, int)</a></span> - Static method in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/SequenceUtil.html" title="class in org.biojava.nbio.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Writes list of FastaSequeces into the outstream formatting the sequence
 so that it contains width chars on each line</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/data/sequence/SequenceUtil.html#writeFasta(java.io.OutputStream,%20java.util.List)">writeFasta(OutputStream, List&lt;FastaSequence&gt;)</a></span> - Static method in class org.biojava.nbio.data.sequence.<a href="./org/biojava/nbio/data/sequence/SequenceUtil.html" title="class in org.biojava.nbio.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Writes FastaSequence in the file, each sequence will take one line only</div>
</dd>
</dl>
<a href="#_A_">A</a>&nbsp;<a href="#_B_">B</a>&nbsp;<a href="#_C_">C</a>&nbsp;<a href="#_D_">D</a>&nbsp;<a href="#_E_">E</a>&nbsp;<a href="#_F_">F</a>&nbsp;<a href="#_G_">G</a>&nbsp;<a href="#_H_">H</a>&nbsp;<a href="#_I_">I</a>&nbsp;<a href="#_J_">J</a>&nbsp;<a href="#_M_">M</a>&nbsp;<a href="#_N_">N</a>&nbsp;<a href="#_O_">O</a>&nbsp;<a href="#_P_">P</a>&nbsp;<a href="#_R_">R</a>&nbsp;<a href="#_S_">S</a>&nbsp;<a href="#_T_">T</a>&nbsp;<a href="#_V_">V</a>&nbsp;<a href="#_W_">W</a>&nbsp;</div>
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar_bottom">
<!--   -->
</a><a href="#skip-navbar_bottom" title="Skip navigation links"></a><a name="navbar_bottom_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="./overview-summary.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li>Use</li>
<li><a href="./overview-tree.html">Tree</a></li>
<li><a href="./deprecated-list.html">Deprecated</a></li>
<li class="navBarCell1Rev">Index</li>
<li><a href="./help-doc.html">Help</a></li>
</ul>
<div class="aboutLanguage"><em><script src="http://www.google-analytics.com/urchin.js" 							type="text/javascript"></script><script 							type="text/javascript">_uacct = 							"UA-1326640-1";urchinTracker();</script></em></div>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="./index.html?index-all.html" target="_top">Frames</a></li>
<li><a href="index-all.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="./allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
<p class="legalCopy"><small>Copyright &#169; 2000-2015 <a href="http://www.biojava.org">BioJava</a>. All Rights Reserved.</small></p>
</body>
</html>
