<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head><meta http-equiv="content-type" content="text/html; charset=UTF-8" />
<title>TestDNAAlignment xref</title>
<link type="text/css" rel="stylesheet" href="../../../../stylesheet.css" />
</head>
<body>
<div id="overview"><a href="../../../../../testapidocs/org/biojava/nbio/alignment/TestDNAAlignment.html">View Javadoc</a></div><pre>
<a class="jxr_linenumber" name="L1" href="#L1">1</a>   <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L2" href="#L2">2</a>   <em class="jxr_javadoccomment"> *                    BioJava development code</em>
<a class="jxr_linenumber" name="L3" href="#L3">3</a>   <em class="jxr_javadoccomment"> *</em>
<a class="jxr_linenumber" name="L4" href="#L4">4</a>   <em class="jxr_javadoccomment"> * This code may be freely distributed and modified under the terms of the GNU</em>
<a class="jxr_linenumber" name="L5" href="#L5">5</a>   <em class="jxr_javadoccomment"> * Lesser General Public Licence. This should be distributed with the code. If</em>
<a class="jxr_linenumber" name="L6" href="#L6">6</a>   <em class="jxr_javadoccomment"> * you do not have a copy, see:</em>
<a class="jxr_linenumber" name="L7" href="#L7">7</a>   <em class="jxr_javadoccomment"> *</em>
<a class="jxr_linenumber" name="L8" href="#L8">8</a>   <em class="jxr_javadoccomment"> *      <a href="http://www.gnu.org/copyleft/lesser.htm" target="alexandria_uri">http://www.gnu.org/copyleft/lesser.htm</a>l</em>
<a class="jxr_linenumber" name="L9" href="#L9">9</a>   <em class="jxr_javadoccomment"> *</em>
<a class="jxr_linenumber" name="L10" href="#L10">10</a>  <em class="jxr_javadoccomment"> * Copyright for this code is held jointly by the individual</em>
<a class="jxr_linenumber" name="L11" href="#L11">11</a>  <em class="jxr_javadoccomment"> * authors.  These should be listed in @author doc comments.</em>
<a class="jxr_linenumber" name="L12" href="#L12">12</a>  <em class="jxr_javadoccomment"> *</em>
<a class="jxr_linenumber" name="L13" href="#L13">13</a>  <em class="jxr_javadoccomment"> * For more information on the BioJava project and its aims,</em>
<a class="jxr_linenumber" name="L14" href="#L14">14</a>  <em class="jxr_javadoccomment"> * or to join the biojava-l mailing list, visit the home page</em>
<a class="jxr_linenumber" name="L15" href="#L15">15</a>  <em class="jxr_javadoccomment"> * at:</em>
<a class="jxr_linenumber" name="L16" href="#L16">16</a>  <em class="jxr_javadoccomment"> *</em>
<a class="jxr_linenumber" name="L17" href="#L17">17</a>  <em class="jxr_javadoccomment"> *      <a href="http://www.biojava.org" target="alexandria_uri">http://www.biojava.org</a>/</em>
<a class="jxr_linenumber" name="L18" href="#L18">18</a>  <em class="jxr_javadoccomment"> *</em>
<a class="jxr_linenumber" name="L19" href="#L19">19</a>  <em class="jxr_javadoccomment"> * Created on Oct 5, 2011 </em>
<a class="jxr_linenumber" name="L20" href="#L20">20</a>  <em class="jxr_javadoccomment"> * Created by Andreas Prlic</em>
<a class="jxr_linenumber" name="L21" href="#L21">21</a>  <em class="jxr_javadoccomment"> *</em>
<a class="jxr_linenumber" name="L22" href="#L22">22</a>  <em class="jxr_javadoccomment"> * @since 3.0.2</em>
<a class="jxr_linenumber" name="L23" href="#L23">23</a>  <em class="jxr_javadoccomment"> */</em>
<a class="jxr_linenumber" name="L24" href="#L24">24</a>  <strong class="jxr_keyword">package</strong> org.biojava.nbio.alignment;
<a class="jxr_linenumber" name="L25" href="#L25">25</a>  
<a class="jxr_linenumber" name="L26" href="#L26">26</a>  <strong class="jxr_keyword">import</strong> junit.framework.TestCase;
<a class="jxr_linenumber" name="L27" href="#L27">27</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.alignment.Alignments.PairwiseSequenceAlignerType;
<a class="jxr_linenumber" name="L28" href="#L28">28</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.alignment.template.PairwiseSequenceAligner;
<a class="jxr_linenumber" name="L29" href="#L29">29</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.alignment.template.Profile;
<a class="jxr_linenumber" name="L30" href="#L30">30</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.alignment.template.SequencePair;
<a class="jxr_linenumber" name="L31" href="#L31">31</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.alignment.template.SubstitutionMatrix;
<a class="jxr_linenumber" name="L32" href="#L32">32</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.exceptions.CompoundNotFoundException;
<a class="jxr_linenumber" name="L33" href="#L33">33</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.DNASequence;
<a class="jxr_linenumber" name="L34" href="#L34">34</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.compound.AmbiguityDNACompoundSet;
<a class="jxr_linenumber" name="L35" href="#L35">35</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.compound.DNACompoundSet;
<a class="jxr_linenumber" name="L36" href="#L36">36</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.compound.NucleotideCompound;
<a class="jxr_linenumber" name="L37" href="#L37">37</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.io.FastaReaderHelper;
<a class="jxr_linenumber" name="L38" href="#L38">38</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.util.ConcurrencyTools;
<a class="jxr_linenumber" name="L39" href="#L39">39</a>  
<a class="jxr_linenumber" name="L40" href="#L40">40</a>  <strong class="jxr_keyword">import</strong> java.io.InputStream;
<a class="jxr_linenumber" name="L41" href="#L41">41</a>  <strong class="jxr_keyword">import</strong> java.util.ArrayList;
<a class="jxr_linenumber" name="L42" href="#L42">42</a>  <strong class="jxr_keyword">import</strong> java.util.LinkedHashMap;
<a class="jxr_linenumber" name="L43" href="#L43">43</a>  <strong class="jxr_keyword">import</strong> java.util.List;
<a class="jxr_linenumber" name="L44" href="#L44">44</a>  
<a class="jxr_linenumber" name="L45" href="#L45">45</a>  <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">class</strong> <a href="../../../../org/biojava/nbio/alignment/TestDNAAlignment.html">TestDNAAlignment</a> <strong class="jxr_keyword">extends</strong> TestCase {
<a class="jxr_linenumber" name="L46" href="#L46">46</a>  
<a class="jxr_linenumber" name="L47" href="#L47">47</a>  	<strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">final</strong> <strong class="jxr_keyword">double</strong> PRECISION = 0.00000001;
<a class="jxr_linenumber" name="L48" href="#L48">48</a>  	
<a class="jxr_linenumber" name="L49" href="#L49">49</a>  
<a class="jxr_linenumber" name="L50" href="#L50">50</a>      <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> testDNAAlignment() {
<a class="jxr_linenumber" name="L51" href="#L51">51</a>  
<a class="jxr_linenumber" name="L52" href="#L52">52</a>          <strong class="jxr_keyword">try</strong> {
<a class="jxr_linenumber" name="L53" href="#L53">53</a>              List&lt;DNASequence&gt; lst = getDNAFASTAFile();
<a class="jxr_linenumber" name="L54" href="#L54">54</a>  
<a class="jxr_linenumber" name="L55" href="#L55">55</a>              Profile&lt;DNASequence, NucleotideCompound&gt; profile = Alignments.getMultipleSequenceAlignment(lst);
<a class="jxr_linenumber" name="L56" href="#L56">56</a>  
<a class="jxr_linenumber" name="L57" href="#L57">57</a>              assertTrue(profile.getSize() == 10);
<a class="jxr_linenumber" name="L58" href="#L58">58</a>  
<a class="jxr_linenumber" name="L59" href="#L59">59</a>              assertTrue(profile.getAlignedSequence(1).getSequenceAsString().length() &gt; 50);
<a class="jxr_linenumber" name="L60" href="#L60">60</a>  
<a class="jxr_linenumber" name="L61" href="#L61">61</a>  
<a class="jxr_linenumber" name="L62" href="#L62">62</a>              <em class="jxr_comment">// here how to print the MSA:</em>
<a class="jxr_linenumber" name="L63" href="#L63">63</a>  
<a class="jxr_linenumber" name="L64" href="#L64">64</a>              <em class="jxr_comment">//System.out.printf("MSA:%n%s%n", profile);</em>
<a class="jxr_linenumber" name="L65" href="#L65">65</a>          } <strong class="jxr_keyword">catch</strong> (Exception e) {
<a class="jxr_linenumber" name="L66" href="#L66">66</a>              e.printStackTrace();
<a class="jxr_linenumber" name="L67" href="#L67">67</a>              fail(e.getMessage());
<a class="jxr_linenumber" name="L68" href="#L68">68</a>          }
<a class="jxr_linenumber" name="L69" href="#L69">69</a>          ConcurrencyTools.shutdown();
<a class="jxr_linenumber" name="L70" href="#L70">70</a>      }
<a class="jxr_linenumber" name="L71" href="#L71">71</a>  
<a class="jxr_linenumber" name="L72" href="#L72">72</a>      <strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">static</strong> List&lt;DNASequence&gt; getDNAFASTAFile() <strong class="jxr_keyword">throws</strong> Exception {
<a class="jxr_linenumber" name="L73" href="#L73">73</a>  
<a class="jxr_linenumber" name="L74" href="#L74">74</a>          InputStream inStream = TestDNAAlignment.<strong class="jxr_keyword">class</strong>.getResourceAsStream(String.format(<span class="jxr_string">"/dna-fasta.txt"</span>));
<a class="jxr_linenumber" name="L75" href="#L75">75</a>          LinkedHashMap&lt;String, DNASequence&gt; fastas = FastaReaderHelper.readFastaDNASequence(inStream);
<a class="jxr_linenumber" name="L76" href="#L76">76</a>  
<a class="jxr_linenumber" name="L77" href="#L77">77</a>          List&lt;DNASequence&gt; sequences = <strong class="jxr_keyword">new</strong> ArrayList&lt;DNASequence&gt;();
<a class="jxr_linenumber" name="L78" href="#L78">78</a>  
<a class="jxr_linenumber" name="L79" href="#L79">79</a>          <strong class="jxr_keyword">for</strong> (String key : fastas.keySet()) {
<a class="jxr_linenumber" name="L80" href="#L80">80</a>              DNASequence seq = fastas.get(key);
<a class="jxr_linenumber" name="L81" href="#L81">81</a>              sequences.add(seq);
<a class="jxr_linenumber" name="L82" href="#L82">82</a>          }
<a class="jxr_linenumber" name="L83" href="#L83">83</a>  
<a class="jxr_linenumber" name="L84" href="#L84">84</a>          <strong class="jxr_keyword">return</strong> sequences;
<a class="jxr_linenumber" name="L85" href="#L85">85</a>      }
<a class="jxr_linenumber" name="L86" href="#L86">86</a>  
<a class="jxr_linenumber" name="L87" href="#L87">87</a>      <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L88" href="#L88">88</a>  <em class="jxr_javadoccomment">     * @author brandstaetter </em>
<a class="jxr_linenumber" name="L89" href="#L89">89</a>  <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L90" href="#L90">90</a>      <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> testDNAMultipleAlignmentWithMixedCompoundSets() <strong class="jxr_keyword">throws</strong> CompoundNotFoundException {
<a class="jxr_linenumber" name="L91" href="#L91">91</a>  
<a class="jxr_linenumber" name="L92" href="#L92">92</a>          DNASequence target = <strong class="jxr_keyword">new</strong> DNASequence(<span class="jxr_string">"ACTGACGTGTAGCTGACTGA"</span>, DNACompoundSet.getDNACompoundSet());
<a class="jxr_linenumber" name="L93" href="#L93">93</a>          DNASequence query = <strong class="jxr_keyword">new</strong> DNASequence(<span class="jxr_string">"ACTGACGTGTAGCTGACTGTA"</span>, AmbiguityDNACompoundSet.getDNACompoundSet());
<a class="jxr_linenumber" name="L94" href="#L94">94</a>  
<a class="jxr_linenumber" name="L95" href="#L95">95</a>          List&lt;DNASequence&gt; lst = <strong class="jxr_keyword">new</strong> ArrayList&lt;DNASequence&gt;();
<a class="jxr_linenumber" name="L96" href="#L96">96</a>          lst.add(target);
<a class="jxr_linenumber" name="L97" href="#L97">97</a>          lst.add(query);
<a class="jxr_linenumber" name="L98" href="#L98">98</a>  
<a class="jxr_linenumber" name="L99" href="#L99">99</a>          <strong class="jxr_keyword">try</strong> {
<a class="jxr_linenumber" name="L100" href="#L100">100</a>         	@SuppressWarnings(<span class="jxr_string">"unused"</span>)
<a class="jxr_linenumber" name="L101" href="#L101">101</a> 			Profile&lt;DNASequence, NucleotideCompound&gt; profile = Alignments.getMultipleSequenceAlignment(lst);
<a class="jxr_linenumber" name="L102" href="#L102">102</a>         	fail(<span class="jxr_string">"Alignments.getMultipleSequenceAlignment(lst) expected exception with differing compound sets"</span>);
<a class="jxr_linenumber" name="L103" href="#L103">103</a>         } <strong class="jxr_keyword">catch</strong> (IllegalArgumentException ex) {
<a class="jxr_linenumber" name="L104" href="#L104">104</a>         	<em class="jxr_comment">// expected exception</em>
<a class="jxr_linenumber" name="L105" href="#L105">105</a>         }
<a class="jxr_linenumber" name="L106" href="#L106">106</a>     }
<a class="jxr_linenumber" name="L107" href="#L107">107</a> 
<a class="jxr_linenumber" name="L108" href="#L108">108</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L109" href="#L109">109</a> <em class="jxr_javadoccomment">     * @author brandstaetter</em>
<a class="jxr_linenumber" name="L110" href="#L110">110</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L111" href="#L111">111</a>     <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> testDNAPairwiseAlignmentWithMixedCompoundSets() <strong class="jxr_keyword">throws</strong> CompoundNotFoundException {
<a class="jxr_linenumber" name="L112" href="#L112">112</a>         DNASequence target = <strong class="jxr_keyword">new</strong> DNASequence(<span class="jxr_string">"ACTGACGTGTAGCTGACTGA"</span>, DNACompoundSet.getDNACompoundSet());
<a class="jxr_linenumber" name="L113" href="#L113">113</a>         DNASequence query = <strong class="jxr_keyword">new</strong> DNASequence(<span class="jxr_string">"ACTGACGTGTAGCTGACTGT"</span>, AmbiguityDNACompoundSet.getDNACompoundSet());
<a class="jxr_linenumber" name="L114" href="#L114">114</a>         SubstitutionMatrix&lt;NucleotideCompound&gt; matrix = SubstitutionMatrixHelper.getNuc4_4();
<a class="jxr_linenumber" name="L115" href="#L115">115</a>         SimpleGapPenalty gapP = <strong class="jxr_keyword">new</strong> SimpleGapPenalty();
<a class="jxr_linenumber" name="L116" href="#L116">116</a>         gapP.setOpenPenalty((<strong class="jxr_keyword">short</strong>) 5);
<a class="jxr_linenumber" name="L117" href="#L117">117</a>         gapP.setExtensionPenalty((<strong class="jxr_keyword">short</strong>) 2);
<a class="jxr_linenumber" name="L118" href="#L118">118</a>         
<a class="jxr_linenumber" name="L119" href="#L119">119</a>         <strong class="jxr_keyword">try</strong> {
<a class="jxr_linenumber" name="L120" href="#L120">120</a>         	@SuppressWarnings(<span class="jxr_string">"unused"</span>)
<a class="jxr_linenumber" name="L121" href="#L121">121</a> 			SequencePair&lt;DNASequence, NucleotideCompound&gt; psa = Alignments.getPairwiseAlignment(query, target, PairwiseSequenceAlignerType.LOCAL, gapP, matrix);
<a class="jxr_linenumber" name="L122" href="#L122">122</a>         	fail(<span class="jxr_string">"Alignments.getPairwiseAlignment() expected exception with differing compound sets"</span>);
<a class="jxr_linenumber" name="L123" href="#L123">123</a>         } <strong class="jxr_keyword">catch</strong> (IllegalArgumentException ex) {
<a class="jxr_linenumber" name="L124" href="#L124">124</a>         	<em class="jxr_comment">// expected exception</em>
<a class="jxr_linenumber" name="L125" href="#L125">125</a>         }
<a class="jxr_linenumber" name="L126" href="#L126">126</a>     }
<a class="jxr_linenumber" name="L127" href="#L127">127</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L128" href="#L128">128</a> <em class="jxr_javadoccomment">     * @author Daniel Cameron</em>
<a class="jxr_linenumber" name="L129" href="#L129">129</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L130" href="#L130">130</a>     <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> testMixedCaseInputStringsMatchUnderlyingBases() <strong class="jxr_keyword">throws</strong> CompoundNotFoundException {
<a class="jxr_linenumber" name="L131" href="#L131">131</a>         DNASequence target = <strong class="jxr_keyword">new</strong> DNASequence(<span class="jxr_string">"AAAAAAAAGTC"</span>, DNACompoundSet.getDNACompoundSet());
<a class="jxr_linenumber" name="L132" href="#L132">132</a>         DNASequence query = <strong class="jxr_keyword">new</strong> DNASequence(<span class="jxr_string">"aaaaaaaagtc"</span>, DNACompoundSet.getDNACompoundSet());
<a class="jxr_linenumber" name="L133" href="#L133">133</a>         SubstitutionMatrix&lt;NucleotideCompound&gt; matrix = SubstitutionMatrixHelper.getNuc4_4();
<a class="jxr_linenumber" name="L134" href="#L134">134</a>         SimpleGapPenalty gapP = <strong class="jxr_keyword">new</strong> SimpleGapPenalty((<strong class="jxr_keyword">short</strong>)5, (<strong class="jxr_keyword">short</strong>)2);
<a class="jxr_linenumber" name="L135" href="#L135">135</a>         <em class="jxr_comment">// should be a full match with +5 per match</em>
<a class="jxr_linenumber" name="L136" href="#L136">136</a>         assertEquals(5.0 * query.getLength(), Alignments.getPairwiseAligner(query, target, PairwiseSequenceAlignerType.LOCAL, gapP, matrix).getScore(), PRECISION);
<a class="jxr_linenumber" name="L137" href="#L137">137</a>     }
<a class="jxr_linenumber" name="L138" href="#L138">138</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L139" href="#L139">139</a> <em class="jxr_javadoccomment">     * @author Daniel Cameron</em>
<a class="jxr_linenumber" name="L140" href="#L140">140</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L141" href="#L141">141</a>     <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> testNoAlignedBases() <strong class="jxr_keyword">throws</strong> CompoundNotFoundException {
<a class="jxr_linenumber" name="L142" href="#L142">142</a>         DNASequence target = <strong class="jxr_keyword">new</strong> DNASequence(<span class="jxr_string">"A"</span>, DNACompoundSet.getDNACompoundSet());
<a class="jxr_linenumber" name="L143" href="#L143">143</a>         DNASequence query = <strong class="jxr_keyword">new</strong> DNASequence(<span class="jxr_string">"T"</span>, DNACompoundSet.getDNACompoundSet());
<a class="jxr_linenumber" name="L144" href="#L144">144</a>         SubstitutionMatrix&lt;NucleotideCompound&gt; matrix = SubstitutionMatrixHelper.getNuc4_4();
<a class="jxr_linenumber" name="L145" href="#L145">145</a>         SimpleGapPenalty gapP = <strong class="jxr_keyword">new</strong> SimpleGapPenalty((<strong class="jxr_keyword">short</strong>)0, (<strong class="jxr_keyword">short</strong>)1);
<a class="jxr_linenumber" name="L146" href="#L146">146</a>         PairwiseSequenceAligner&lt;DNASequence, NucleotideCompound&gt; aligner = Alignments.getPairwiseAligner(query, target, PairwiseSequenceAlignerType.GLOBAL, gapP, matrix);
<a class="jxr_linenumber" name="L147" href="#L147">147</a>         assertEquals(2, aligner.getPair().getLength());
<a class="jxr_linenumber" name="L148" href="#L148">148</a>     }
<a class="jxr_linenumber" name="L149" href="#L149">149</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L150" href="#L150">150</a> <em class="jxr_javadoccomment">    * @author Daniel Cameron</em>
<a class="jxr_linenumber" name="L151" href="#L151">151</a> <em class="jxr_javadoccomment">    */</em>
<a class="jxr_linenumber" name="L152" href="#L152">152</a> 	<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> testLinearAlignment() <strong class="jxr_keyword">throws</strong> CompoundNotFoundException {
<a class="jxr_linenumber" name="L153" href="#L153">153</a> 		DNASequence query = <strong class="jxr_keyword">new</strong> DNASequence(<span class="jxr_string">"GTAAAAG"</span>, DNACompoundSet.getDNACompoundSet());
<a class="jxr_linenumber" name="L154" href="#L154">154</a> 		DNASequence target = <strong class="jxr_keyword">new</strong> DNASequence(<span class="jxr_string">"GAAAACGTTTTTTTTTT"</span>, DNACompoundSet.getDNACompoundSet());
<a class="jxr_linenumber" name="L155" href="#L155">155</a> 		SubstitutionMatrix&lt;NucleotideCompound&gt; matrix = SubstitutionMatrixHelper.getNuc4_4();
<a class="jxr_linenumber" name="L156" href="#L156">156</a> 		SimpleGapPenalty gapP = <strong class="jxr_keyword">new</strong> SimpleGapPenalty((<strong class="jxr_keyword">short</strong>)0, (<strong class="jxr_keyword">short</strong>)3);
<a class="jxr_linenumber" name="L157" href="#L157">157</a> 		PairwiseSequenceAligner&lt;DNASequence, NucleotideCompound&gt; aligner = Alignments.getPairwiseAligner(query, target, PairwiseSequenceAlignerType.GLOBAL, gapP, matrix);
<a class="jxr_linenumber" name="L158" href="#L158">158</a> 		assertEquals(String.format(<span class="jxr_string">"GTAAAA-G----------%nG-AAAACGTTTTTTTTTT%n"</span>), aligner.getPair().toString());;
<a class="jxr_linenumber" name="L159" href="#L159">159</a> 	}
<a class="jxr_linenumber" name="L160" href="#L160">160</a> }
</pre>
<hr/>
<div id="footer">Copyright &#169; 2000&#x2013;2015 <a href="http://www.biojava.org">BioJava</a>. All rights reserved.</div>
</body>
</html>
