<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head><meta http-equiv="content-type" content="text/html; charset=UTF-8" />
<title>TranslationTest xref</title>
<link type="text/css" rel="stylesheet" href="../../../../../stylesheet.css" />
</head>
<body>
<div id="overview"><a href="../../../../../../testapidocs/org/biojava/nbio/core/sequence/TranslationTest.html">View Javadoc</a></div><pre>
<a class="jxr_linenumber" name="L1" href="#L1">1</a>   <em class="jxr_comment">/*</em>
<a class="jxr_linenumber" name="L2" href="#L2">2</a>   <em class="jxr_comment"> *                    BioJava development code</em>
<a class="jxr_linenumber" name="L3" href="#L3">3</a>   <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L4" href="#L4">4</a>   <em class="jxr_comment"> * This code may be freely distributed and modified under the</em>
<a class="jxr_linenumber" name="L5" href="#L5">5</a>   <em class="jxr_comment"> * terms of the GNU Lesser General Public Licence.  This should</em>
<a class="jxr_linenumber" name="L6" href="#L6">6</a>   <em class="jxr_comment"> * be distributed with the code.  If you do not have a copy,</em>
<a class="jxr_linenumber" name="L7" href="#L7">7</a>   <em class="jxr_comment"> * see:</em>
<a class="jxr_linenumber" name="L8" href="#L8">8</a>   <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L9" href="#L9">9</a>   <em class="jxr_comment"> *      <a href="http://www.gnu.org/copyleft/lesser.htm" target="alexandria_uri">http://www.gnu.org/copyleft/lesser.htm</a>l</em>
<a class="jxr_linenumber" name="L10" href="#L10">10</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L11" href="#L11">11</a>  <em class="jxr_comment"> * Copyright for this code is held jointly by the individual</em>
<a class="jxr_linenumber" name="L12" href="#L12">12</a>  <em class="jxr_comment"> * authors.  These should be listed in @author doc comments.</em>
<a class="jxr_linenumber" name="L13" href="#L13">13</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L14" href="#L14">14</a>  <em class="jxr_comment"> * For more information on the BioJava project and its aims,</em>
<a class="jxr_linenumber" name="L15" href="#L15">15</a>  <em class="jxr_comment"> * or to join the biojava-l mailing list, visit the home page</em>
<a class="jxr_linenumber" name="L16" href="#L16">16</a>  <em class="jxr_comment"> * at:</em>
<a class="jxr_linenumber" name="L17" href="#L17">17</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L18" href="#L18">18</a>  <em class="jxr_comment"> *      <a href="http://www.biojava.org" target="alexandria_uri">http://www.biojava.org</a>/</em>
<a class="jxr_linenumber" name="L19" href="#L19">19</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L20" href="#L20">20</a>  <em class="jxr_comment"> */</em>
<a class="jxr_linenumber" name="L21" href="#L21">21</a>  <strong class="jxr_keyword">package</strong> org.biojava.nbio.core.sequence;
<a class="jxr_linenumber" name="L22" href="#L22">22</a>  
<a class="jxr_linenumber" name="L23" href="#L23">23</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.exceptions.CompoundNotFoundException;
<a class="jxr_linenumber" name="L24" href="#L24">24</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.compound.AminoAcidCompound;
<a class="jxr_linenumber" name="L25" href="#L25">25</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.compound.AminoAcidCompoundSet;
<a class="jxr_linenumber" name="L26" href="#L26">26</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.compound.DNACompoundSet;
<a class="jxr_linenumber" name="L27" href="#L27">27</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.compound.NucleotideCompound;
<a class="jxr_linenumber" name="L28" href="#L28">28</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.io.*;
<a class="jxr_linenumber" name="L29" href="#L29">29</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.io.util.ClasspathResource;
<a class="jxr_linenumber" name="L30" href="#L30">30</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.template.Sequence;
<a class="jxr_linenumber" name="L31" href="#L31">31</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.transcription.Frame;
<a class="jxr_linenumber" name="L32" href="#L32">32</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.transcription.TranscriptionEngine;
<a class="jxr_linenumber" name="L33" href="#L33">33</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.transcription.TranscriptionEngine.Builder;
<a class="jxr_linenumber" name="L34" href="#L34">34</a>  <strong class="jxr_keyword">import</strong> org.junit.Assert;
<a class="jxr_linenumber" name="L35" href="#L35">35</a>  <strong class="jxr_keyword">import</strong> org.junit.BeforeClass;
<a class="jxr_linenumber" name="L36" href="#L36">36</a>  <strong class="jxr_keyword">import</strong> org.junit.Test;
<a class="jxr_linenumber" name="L37" href="#L37">37</a>  <strong class="jxr_keyword">import</strong> org.slf4j.Logger;
<a class="jxr_linenumber" name="L38" href="#L38">38</a>  <strong class="jxr_keyword">import</strong> org.slf4j.LoggerFactory;
<a class="jxr_linenumber" name="L39" href="#L39">39</a>  
<a class="jxr_linenumber" name="L40" href="#L40">40</a>  <strong class="jxr_keyword">import</strong> java.io.IOException;
<a class="jxr_linenumber" name="L41" href="#L41">41</a>  <strong class="jxr_keyword">import</strong> java.io.InputStream;
<a class="jxr_linenumber" name="L42" href="#L42">42</a>  <strong class="jxr_keyword">import</strong> java.util.EnumMap;
<a class="jxr_linenumber" name="L43" href="#L43">43</a>  <strong class="jxr_keyword">import</strong> java.util.Map;
<a class="jxr_linenumber" name="L44" href="#L44">44</a>  <strong class="jxr_keyword">import</strong> java.util.Map.Entry;
<a class="jxr_linenumber" name="L45" href="#L45">45</a>  
<a class="jxr_linenumber" name="L46" href="#L46">46</a>  <strong class="jxr_keyword">import</strong> <strong class="jxr_keyword">static</strong> org.biojava.nbio.core.sequence.io.util.IOUtils.close;
<a class="jxr_linenumber" name="L47" href="#L47">47</a>  <strong class="jxr_keyword">import</strong> <strong class="jxr_keyword">static</strong> org.hamcrest.CoreMatchers.is;
<a class="jxr_linenumber" name="L48" href="#L48">48</a>  <strong class="jxr_keyword">import</strong> <strong class="jxr_keyword">static</strong> org.junit.Assert.*;
<a class="jxr_linenumber" name="L49" href="#L49">49</a>  
<a class="jxr_linenumber" name="L50" href="#L50">50</a>  <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">class</strong> <a href="../../../../../org/biojava/nbio/core/sequence/TranslationTest.html">TranslationTest</a> {
<a class="jxr_linenumber" name="L51" href="#L51">51</a>  
<a class="jxr_linenumber" name="L52" href="#L52">52</a>  	<strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">final</strong> <strong class="jxr_keyword">static</strong> Logger logger = LoggerFactory.getLogger(TranslationTest.<strong class="jxr_keyword">class</strong>);
<a class="jxr_linenumber" name="L53" href="#L53">53</a>  
<a class="jxr_linenumber" name="L54" href="#L54">54</a>  	<strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">static</strong> DNACompoundSet dnaCs = DNACompoundSet.getDNACompoundSet();
<a class="jxr_linenumber" name="L55" href="#L55">55</a>  	<strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">static</strong> AminoAcidCompoundSet aaCs = AminoAcidCompoundSet.getAminoAcidCompoundSet();
<a class="jxr_linenumber" name="L56" href="#L56">56</a>  	<strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">static</strong> DNASequence brca2Dna;
<a class="jxr_linenumber" name="L57" href="#L57">57</a>  	<strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">static</strong> Sequence&lt;AminoAcidCompound&gt; brca2Pep;
<a class="jxr_linenumber" name="L58" href="#L58">58</a>  	<strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">static</strong> Sequence&lt;NucleotideCompound&gt; volvoxDna;
<a class="jxr_linenumber" name="L59" href="#L59">59</a>  	<strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">static</strong> Sequence&lt;AminoAcidCompound&gt; volvoxPep;
<a class="jxr_linenumber" name="L60" href="#L60">60</a>  
<a class="jxr_linenumber" name="L61" href="#L61">61</a>  	@BeforeClass
<a class="jxr_linenumber" name="L62" href="#L62">62</a>  	<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">void</strong> parseSequences() {
<a class="jxr_linenumber" name="L63" href="#L63">63</a>  		InputStream cdsIs = <strong class="jxr_keyword">new</strong> ClasspathResource(
<a class="jxr_linenumber" name="L64" href="#L64">64</a>  				<span class="jxr_string">"org/biojava/nbio/core/sequence/BRCA2-cds.fasta"</span>).getInputStream();
<a class="jxr_linenumber" name="L65" href="#L65">65</a>  		InputStream pepIs = <strong class="jxr_keyword">new</strong> ClasspathResource(
<a class="jxr_linenumber" name="L66" href="#L66">66</a>  				<span class="jxr_string">"org/biojava/nbio/core/sequence/BRCA2-peptide.fasta"</span>).getInputStream();
<a class="jxr_linenumber" name="L67" href="#L67">67</a>  		InputStream volDnaIs = <strong class="jxr_keyword">new</strong> ClasspathResource(
<a class="jxr_linenumber" name="L68" href="#L68">68</a>  				<span class="jxr_string">"org/biojava/nbio/core/sequence/volvox-cds.fasta"</span>).getInputStream();
<a class="jxr_linenumber" name="L69" href="#L69">69</a>  		InputStream volPepIs = <strong class="jxr_keyword">new</strong> ClasspathResource(
<a class="jxr_linenumber" name="L70" href="#L70">70</a>  				<span class="jxr_string">"org/biojava/nbio/core/sequence/volvox-peptide.fasta"</span>).getInputStream();
<a class="jxr_linenumber" name="L71" href="#L71">71</a>  
<a class="jxr_linenumber" name="L72" href="#L72">72</a>  		<strong class="jxr_keyword">try</strong> {
<a class="jxr_linenumber" name="L73" href="#L73">73</a>  			FastaReader&lt;DNASequence, NucleotideCompound&gt; dnaReader = <strong class="jxr_keyword">new</strong> FastaReader&lt;DNASequence, NucleotideCompound&gt;(cdsIs,
<a class="jxr_linenumber" name="L74" href="#L74">74</a>  					<strong class="jxr_keyword">new</strong> GenericFastaHeaderParser&lt;DNASequence, NucleotideCompound&gt;(), <strong class="jxr_keyword">new</strong> DNASequenceCreator(dnaCs));
<a class="jxr_linenumber" name="L75" href="#L75">75</a>  			brca2Dna = dnaReader.process().values().iterator().next();
<a class="jxr_linenumber" name="L76" href="#L76">76</a>  			FastaReader&lt;ProteinSequence, AminoAcidCompound&gt; pReader = <strong class="jxr_keyword">new</strong> FastaReader&lt;ProteinSequence, AminoAcidCompound&gt;(
<a class="jxr_linenumber" name="L77" href="#L77">77</a>  					pepIs, <strong class="jxr_keyword">new</strong> GenericFastaHeaderParser&lt;ProteinSequence, AminoAcidCompound&gt;(), <strong class="jxr_keyword">new</strong> ProteinSequenceCreator(
<a class="jxr_linenumber" name="L78" href="#L78">78</a>  							aaCs));
<a class="jxr_linenumber" name="L79" href="#L79">79</a>  			brca2Pep = pReader.process().values().iterator().next();
<a class="jxr_linenumber" name="L80" href="#L80">80</a>  
<a class="jxr_linenumber" name="L81" href="#L81">81</a>  			FastaReader&lt;DNASequence, NucleotideCompound&gt; volvoxDnaReader = <strong class="jxr_keyword">new</strong> FastaReader&lt;DNASequence, NucleotideCompound&gt;(volDnaIs,
<a class="jxr_linenumber" name="L82" href="#L82">82</a>  					<strong class="jxr_keyword">new</strong> GenericFastaHeaderParser&lt;DNASequence, NucleotideCompound&gt;(), <strong class="jxr_keyword">new</strong> DNASequenceCreator(dnaCs));
<a class="jxr_linenumber" name="L83" href="#L83">83</a>  			volvoxDna = volvoxDnaReader.process().values().iterator().next();
<a class="jxr_linenumber" name="L84" href="#L84">84</a>  			FastaReader&lt;ProteinSequence, AminoAcidCompound&gt; volvoxPepReader = <strong class="jxr_keyword">new</strong> FastaReader&lt;ProteinSequence, AminoAcidCompound&gt;(
<a class="jxr_linenumber" name="L85" href="#L85">85</a>  					volPepIs, <strong class="jxr_keyword">new</strong> GenericFastaHeaderParser&lt;ProteinSequence, AminoAcidCompound&gt;(), <strong class="jxr_keyword">new</strong> ProteinSequenceCreator(
<a class="jxr_linenumber" name="L86" href="#L86">86</a>  							aaCs));
<a class="jxr_linenumber" name="L87" href="#L87">87</a>  			volvoxPep = volvoxPepReader.process().values().iterator().next();
<a class="jxr_linenumber" name="L88" href="#L88">88</a>  		}
<a class="jxr_linenumber" name="L89" href="#L89">89</a>  		<strong class="jxr_keyword">catch</strong> (IOException e) {
<a class="jxr_linenumber" name="L90" href="#L90">90</a>  			logger.error(<span class="jxr_string">"Exception: "</span>, e);
<a class="jxr_linenumber" name="L91" href="#L91">91</a>  			Assert.fail(<span class="jxr_string">"Encountered exception"</span>);
<a class="jxr_linenumber" name="L92" href="#L92">92</a>  		}
<a class="jxr_linenumber" name="L93" href="#L93">93</a>  		<strong class="jxr_keyword">finally</strong> {
<a class="jxr_linenumber" name="L94" href="#L94">94</a>  			close(cdsIs);
<a class="jxr_linenumber" name="L95" href="#L95">95</a>  			close(pepIs);
<a class="jxr_linenumber" name="L96" href="#L96">96</a>  			close(volDnaIs);
<a class="jxr_linenumber" name="L97" href="#L97">97</a>  			close(volPepIs);
<a class="jxr_linenumber" name="L98" href="#L98">98</a>  		}
<a class="jxr_linenumber" name="L99" href="#L99">99</a>  	}
<a class="jxr_linenumber" name="L100" href="#L100">100</a> 
<a class="jxr_linenumber" name="L101" href="#L101">101</a> 	@Test
<a class="jxr_linenumber" name="L102" href="#L102">102</a> 	<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> getUniversal() {
<a class="jxr_linenumber" name="L103" href="#L103">103</a> 		IUPACParser.getInstance().getTable(1);
<a class="jxr_linenumber" name="L104" href="#L104">104</a> 		IUPACParser.getInstance().getTable(<span class="jxr_string">"UNIVERSAL"</span>);
<a class="jxr_linenumber" name="L105" href="#L105">105</a> 	}
<a class="jxr_linenumber" name="L106" href="#L106">106</a> 
<a class="jxr_linenumber" name="L107" href="#L107">107</a> 	@Test
<a class="jxr_linenumber" name="L108" href="#L108">108</a> 	<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> basicTranslation() <strong class="jxr_keyword">throws</strong> CompoundNotFoundException {
<a class="jxr_linenumber" name="L109" href="#L109">109</a> 		TranscriptionEngine e = TranscriptionEngine.getDefault();
<a class="jxr_linenumber" name="L110" href="#L110">110</a> 		DNASequence dna = <strong class="jxr_keyword">new</strong> DNASequence(<span class="jxr_string">"ATG"</span>);
<a class="jxr_linenumber" name="L111" href="#L111">111</a> 		RNASequence rna = dna.getRNASequence(e);
<a class="jxr_linenumber" name="L112" href="#L112">112</a> 		ProteinSequence protein = rna.getProteinSequence(e);
<a class="jxr_linenumber" name="L113" href="#L113">113</a> 		AminoAcidCompound initMet = protein.getCompoundAt(1);
<a class="jxr_linenumber" name="L114" href="#L114">114</a> 		assertThat(<span class="jxr_string">"Initator methionine wrong"</span>, initMet.toString(), is(<span class="jxr_string">"M"</span>));
<a class="jxr_linenumber" name="L115" href="#L115">115</a> 	}
<a class="jxr_linenumber" name="L116" href="#L116">116</a> 
<a class="jxr_linenumber" name="L117" href="#L117">117</a> 	@Test
<a class="jxr_linenumber" name="L118" href="#L118">118</a> 	<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> translateN() <strong class="jxr_keyword">throws</strong> CompoundNotFoundException {
<a class="jxr_linenumber" name="L119" href="#L119">119</a> 		TranscriptionEngine.Builder b = <strong class="jxr_keyword">new</strong> TranscriptionEngine.Builder();
<a class="jxr_linenumber" name="L120" href="#L120">120</a> 		b.translateNCodons(<strong class="jxr_keyword">true</strong>).initMet(<strong class="jxr_keyword">true</strong>);
<a class="jxr_linenumber" name="L121" href="#L121">121</a> 		TranscriptionEngine e = b.build();
<a class="jxr_linenumber" name="L122" href="#L122">122</a> 		DNASequence dna = <strong class="jxr_keyword">new</strong> DNASequence(<span class="jxr_string">"ATN"</span>);
<a class="jxr_linenumber" name="L123" href="#L123">123</a> 		RNASequence rna = dna.getRNASequence(e);
<a class="jxr_linenumber" name="L124" href="#L124">124</a> 		ProteinSequence protein = rna.getProteinSequence(e);
<a class="jxr_linenumber" name="L125" href="#L125">125</a> 		assertThat(<span class="jxr_string">"Ambiguous translation problem"</span>, protein.toString(), is(<span class="jxr_string">"X"</span>));
<a class="jxr_linenumber" name="L126" href="#L126">126</a> 		DNASequence dna2 = <strong class="jxr_keyword">new</strong> DNASequence(<span class="jxr_string">"GTGGTNTAA"</span>);
<a class="jxr_linenumber" name="L127" href="#L127">127</a> 		RNASequence rna2 = dna2.getRNASequence(e);
<a class="jxr_linenumber" name="L128" href="#L128">128</a> 		ProteinSequence protein2 = rna2.getProteinSequence(e);
<a class="jxr_linenumber" name="L129" href="#L129">129</a> 		assertThat(<span class="jxr_string">"Ambiguous translation problem"</span>, protein2.toString(), is(<span class="jxr_string">"VX"</span>));
<a class="jxr_linenumber" name="L130" href="#L130">130</a> 	}
<a class="jxr_linenumber" name="L131" href="#L131">131</a> 
<a class="jxr_linenumber" name="L132" href="#L132">132</a> 	@SuppressWarnings(<span class="jxr_string">"serial"</span>)
<a class="jxr_linenumber" name="L133" href="#L133">133</a> 	@Test
<a class="jxr_linenumber" name="L134" href="#L134">134</a> 	<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> multiFrameTranslation() <strong class="jxr_keyword">throws</strong> CompoundNotFoundException {
<a class="jxr_linenumber" name="L135" href="#L135">135</a> 		TranscriptionEngine e = TranscriptionEngine.getDefault();
<a class="jxr_linenumber" name="L136" href="#L136">136</a> 		DNASequence dna = <strong class="jxr_keyword">new</strong> DNASequence(<span class="jxr_string">"ATGGCGTGA"</span>);
<a class="jxr_linenumber" name="L137" href="#L137">137</a> 
<a class="jxr_linenumber" name="L138" href="#L138">138</a> 		Map&lt;Frame, String&gt; expectedTranslations = <strong class="jxr_keyword">new</strong> EnumMap&lt;Frame, String&gt;(Frame.<strong class="jxr_keyword">class</strong>) {
<a class="jxr_linenumber" name="L139" href="#L139">139</a> 
<a class="jxr_linenumber" name="L140" href="#L140">140</a> 			{
<a class="jxr_linenumber" name="L141" href="#L141">141</a> 				put(Frame.ONE, <span class="jxr_string">"MA"</span>);
<a class="jxr_linenumber" name="L142" href="#L142">142</a> 				put(Frame.TWO, <span class="jxr_string">"WR"</span>);
<a class="jxr_linenumber" name="L143" href="#L143">143</a> 				put(Frame.THREE, <span class="jxr_string">"GV"</span>);
<a class="jxr_linenumber" name="L144" href="#L144">144</a> 				put(Frame.REVERSED_ONE, <span class="jxr_string">"SRH"</span>);
<a class="jxr_linenumber" name="L145" href="#L145">145</a> 				put(Frame.REVERSED_TWO, <span class="jxr_string">"HA"</span>);
<a class="jxr_linenumber" name="L146" href="#L146">146</a> 				put(Frame.REVERSED_THREE, <span class="jxr_string">"TP"</span>);
<a class="jxr_linenumber" name="L147" href="#L147">147</a> 			}
<a class="jxr_linenumber" name="L148" href="#L148">148</a> 		};
<a class="jxr_linenumber" name="L149" href="#L149">149</a> 
<a class="jxr_linenumber" name="L150" href="#L150">150</a> 		Map&lt;Frame, Sequence&lt;AminoAcidCompound&gt;&gt; translations =
<a class="jxr_linenumber" name="L151" href="#L151">151</a> 				e.multipleFrameTranslation(dna, Frame.getAllFrames());
<a class="jxr_linenumber" name="L152" href="#L152">152</a> 
<a class="jxr_linenumber" name="L153" href="#L153">153</a> 		<strong class="jxr_keyword">for</strong> (Entry&lt;Frame, Sequence&lt;AminoAcidCompound&gt;&gt; entry : translations.entrySet()) {
<a class="jxr_linenumber" name="L154" href="#L154">154</a> 			String expected = expectedTranslations.get(entry.getKey());
<a class="jxr_linenumber" name="L155" href="#L155">155</a> 			Sequence&lt;AminoAcidCompound&gt; protein = entry.getValue();
<a class="jxr_linenumber" name="L156" href="#L156">156</a> 			assertThat(<span class="jxr_string">"Checking 6 frame translation"</span>, protein.toString(), is(expected));
<a class="jxr_linenumber" name="L157" href="#L157">157</a> 		}
<a class="jxr_linenumber" name="L158" href="#L158">158</a> 	}
<a class="jxr_linenumber" name="L159" href="#L159">159</a> 
<a class="jxr_linenumber" name="L160" href="#L160">160</a> 	@Test
<a class="jxr_linenumber" name="L161" href="#L161">161</a> 	<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> lowerCases() <strong class="jxr_keyword">throws</strong> CompoundNotFoundException {
<a class="jxr_linenumber" name="L162" href="#L162">162</a> 		TranscriptionEngine e = TranscriptionEngine.getDefault();
<a class="jxr_linenumber" name="L163" href="#L163">163</a> 		DNASequence dna = <strong class="jxr_keyword">new</strong> DNASequence(<span class="jxr_string">"atgcCt"</span>);
<a class="jxr_linenumber" name="L164" href="#L164">164</a> 		RNASequence rna = dna.getRNASequence(e);
<a class="jxr_linenumber" name="L165" href="#L165">165</a> 		Sequence&lt;AminoAcidCompound&gt; peptide = rna.getProteinSequence(e);
<a class="jxr_linenumber" name="L166" href="#L166">166</a> 		assertThat(<span class="jxr_string">"Checking lower casing is respected"</span>, peptide.getSequenceAsString(),
<a class="jxr_linenumber" name="L167" href="#L167">167</a> 				is(<span class="jxr_string">"MP"</span>));
<a class="jxr_linenumber" name="L168" href="#L168">168</a> 	}
<a class="jxr_linenumber" name="L169" href="#L169">169</a> 
<a class="jxr_linenumber" name="L170" href="#L170">170</a> 	@Test
<a class="jxr_linenumber" name="L171" href="#L171">171</a> 	<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> translateBrca2ExonOne() <strong class="jxr_keyword">throws</strong> CompoundNotFoundException {
<a class="jxr_linenumber" name="L172" href="#L172">172</a> 		TranscriptionEngine e = TranscriptionEngine.getDefault();
<a class="jxr_linenumber" name="L173" href="#L173">173</a> 		DNASequence dna = <strong class="jxr_keyword">new</strong> DNASequence(
<a class="jxr_linenumber" name="L174" href="#L174">174</a> 				<span class="jxr_string">"ATGCCTATTGGATCCAAAGAGAGGCCAACATTTTTTGAAATTTTTAAGACACGCTGCAACAAAGCA"</span>);
<a class="jxr_linenumber" name="L175" href="#L175">175</a> 		RNASequence rna = dna.getRNASequence(e);
<a class="jxr_linenumber" name="L176" href="#L176">176</a> 		Sequence&lt;AminoAcidCompound&gt; peptide = rna.getProteinSequence(e);
<a class="jxr_linenumber" name="L177" href="#L177">177</a> 		assertThat(<span class="jxr_string">"Initator methionine wrong"</span>, peptide.getSequenceAsString(),
<a class="jxr_linenumber" name="L178" href="#L178">178</a> 				is(<span class="jxr_string">"MPIGSKERPTFFEIFKTRCNKA"</span>));
<a class="jxr_linenumber" name="L179" href="#L179">179</a> 	}
<a class="jxr_linenumber" name="L180" href="#L180">180</a> 
<a class="jxr_linenumber" name="L181" href="#L181">181</a> 	@Test(timeout=2000)
<a class="jxr_linenumber" name="L182" href="#L182">182</a> 	<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> translateBrca2() {
<a class="jxr_linenumber" name="L183" href="#L183">183</a> 		TranscriptionEngine e =
<a class="jxr_linenumber" name="L184" href="#L184">184</a> 				<strong class="jxr_keyword">new</strong> TranscriptionEngine.Builder().decorateRna(<strong class="jxr_keyword">true</strong>).build();
<a class="jxr_linenumber" name="L185" href="#L185">185</a> 		<strong class="jxr_keyword">for</strong> (<strong class="jxr_keyword">int</strong> i = 0; i &lt; 100; i++) {
<a class="jxr_linenumber" name="L186" href="#L186">186</a> 			RNASequence rna = brca2Dna.getRNASequence(e);
<a class="jxr_linenumber" name="L187" href="#L187">187</a> 			ProteinSequence protein = rna.getProteinSequence(e);
<a class="jxr_linenumber" name="L188" href="#L188">188</a> 			assertThat(<span class="jxr_string">"BRCA2 does not translate"</span>, protein.getSequenceAsString(),
<a class="jxr_linenumber" name="L189" href="#L189">189</a> 					is(brca2Pep.getSequenceAsString()));
<a class="jxr_linenumber" name="L190" href="#L190">190</a> 		}
<a class="jxr_linenumber" name="L191" href="#L191">191</a> 	}
<a class="jxr_linenumber" name="L192" href="#L192">192</a> 
<a class="jxr_linenumber" name="L193" href="#L193">193</a> 	@Test
<a class="jxr_linenumber" name="L194" href="#L194">194</a> 	<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> translateInternalStops() {
<a class="jxr_linenumber" name="L195" href="#L195">195</a> 		TranscriptionEngine e = TranscriptionEngine.getDefault();
<a class="jxr_linenumber" name="L196" href="#L196">196</a> 		Sequence&lt;AminoAcidCompound&gt; pep = e.translate(volvoxDna);
<a class="jxr_linenumber" name="L197" href="#L197">197</a> 		assertThat(<span class="jxr_string">"Ensure internal stops stay"</span>, pep.toString(), is(volvoxPep.toString()));
<a class="jxr_linenumber" name="L198" href="#L198">198</a> 	}
<a class="jxr_linenumber" name="L199" href="#L199">199</a> 	
<a class="jxr_linenumber" name="L200" href="#L200">200</a> 	@Test
<a class="jxr_linenumber" name="L201" href="#L201">201</a> 	<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> translateStopAtInternalStops(){
<a class="jxr_linenumber" name="L202" href="#L202">202</a> 		<em class="jxr_comment">//This should stop translation at the first stop codon encountered</em>
<a class="jxr_linenumber" name="L203" href="#L203">203</a> 		TranscriptionEngine e = <strong class="jxr_keyword">new</strong> TranscriptionEngine.Builder().stopAtStopCodons(<strong class="jxr_keyword">true</strong>).build();
<a class="jxr_linenumber" name="L204" href="#L204">204</a> 		RNASequence rna = ((DNASequence)volvoxDna).getRNASequence();
<a class="jxr_linenumber" name="L205" href="#L205">205</a> 		String pep = rna.getProteinSequence(e).getSequenceAsString();
<a class="jxr_linenumber" name="L206" href="#L206">206</a> 		String testpep = volvoxPep.getSequenceAsString().split(<span class="jxr_string">"&#92;&#92;*"</span>)[0];
<a class="jxr_linenumber" name="L207" href="#L207">207</a> 		assertThat(<span class="jxr_string">"Translation stops at Stop"</span>, pep, is(testpep));
<a class="jxr_linenumber" name="L208" href="#L208">208</a> 	}
<a class="jxr_linenumber" name="L209" href="#L209">209</a> 	
<a class="jxr_linenumber" name="L210" href="#L210">210</a> 	@Test
<a class="jxr_linenumber" name="L211" href="#L211">211</a> 	<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> waitForStartCodon() <strong class="jxr_keyword">throws</strong> CompoundNotFoundException{
<a class="jxr_linenumber" name="L212" href="#L212">212</a> 		<em class="jxr_comment">//Should not start translation until a start codon is encountered</em>
<a class="jxr_linenumber" name="L213" href="#L213">213</a> 		TranscriptionEngine e = <strong class="jxr_keyword">new</strong> TranscriptionEngine.Builder().waitForStartCodon(<strong class="jxr_keyword">true</strong>).build();
<a class="jxr_linenumber" name="L214" href="#L214">214</a> 		RNASequence rna = <strong class="jxr_keyword">new</strong> RNASequence(<span class="jxr_string">"UCCAUGAGC"</span>);
<a class="jxr_linenumber" name="L215" href="#L215">215</a> 		String pep = rna.getProteinSequence(e).getSequenceAsString();
<a class="jxr_linenumber" name="L216" href="#L216">216</a> 		assertThat(<span class="jxr_string">"Translation starts at Start Codon"</span>,pep, is(<span class="jxr_string">"MS"</span>));
<a class="jxr_linenumber" name="L217" href="#L217">217</a> 		
<a class="jxr_linenumber" name="L218" href="#L218">218</a> 		<em class="jxr_comment">//And should start at start of sequence (NB this is implied by success of all other tests)</em>
<a class="jxr_linenumber" name="L219" href="#L219">219</a> 		e = <strong class="jxr_keyword">new</strong> TranscriptionEngine.Builder().waitForStartCodon(false).build();
<a class="jxr_linenumber" name="L220" href="#L220">220</a> 		pep = rna.getProteinSequence(e).getSequenceAsString();
<a class="jxr_linenumber" name="L221" href="#L221">221</a> 		assertThat(<span class="jxr_string">"Translation starts at start of sequence"</span>, pep, is(<span class="jxr_string">"SMS"</span>));
<a class="jxr_linenumber" name="L222" href="#L222">222</a> 	}
<a class="jxr_linenumber" name="L223" href="#L223">223</a> 
<a class="jxr_linenumber" name="L224" href="#L224">224</a> 	@Test
<a class="jxr_linenumber" name="L225" href="#L225">225</a> 	<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> translateInitMet() <strong class="jxr_keyword">throws</strong> CompoundNotFoundException {
<a class="jxr_linenumber" name="L226" href="#L226">226</a> 		TranscriptionEngine e = TranscriptionEngine.getDefault();
<a class="jxr_linenumber" name="L227" href="#L227">227</a> 		assertThat(<span class="jxr_string">"Leucene (CTA) is not changed to init met"</span>, e.translate(<strong class="jxr_keyword">new</strong> DNASequence(<span class="jxr_string">"CTA"</span>)).toString(), is(<span class="jxr_string">"L"</span>));
<a class="jxr_linenumber" name="L228" href="#L228">228</a> 		assertThat(<span class="jxr_string">"Leucene (CTG) is changed to init met"</span>, e.translate(<strong class="jxr_keyword">new</strong> DNASequence(<span class="jxr_string">"CTG"</span>)).toString(), is(<span class="jxr_string">"M"</span>));
<a class="jxr_linenumber" name="L229" href="#L229">229</a> 
<a class="jxr_linenumber" name="L230" href="#L230">230</a> 		e = <strong class="jxr_keyword">new</strong> TranscriptionEngine.Builder().initMet(false).build();
<a class="jxr_linenumber" name="L231" href="#L231">231</a> 		assertThat(<span class="jxr_string">"Leucene (CTG) is not changed to init met"</span>, e.translate(<strong class="jxr_keyword">new</strong> DNASequence(<span class="jxr_string">"CTG"</span>)).toString(), is(<span class="jxr_string">"L"</span>));
<a class="jxr_linenumber" name="L232" href="#L232">232</a> 	}
<a class="jxr_linenumber" name="L233" href="#L233">233</a> 
<a class="jxr_linenumber" name="L234" href="#L234">234</a> 	<em class="jxr_javadoccomment">/** test for https:</em><em class="jxr_comment">//github.com/biojava/biojava/issues/53  */</em>
<a class="jxr_linenumber" name="L235" href="#L235">235</a> <em class="jxr_javadoccomment">	@Test</em>
<a class="jxr_linenumber" name="L236" href="#L236">236</a> <em class="jxr_javadoccomment">	public void testHashCollision() throws CompoundNotFoundException{</em>
<a class="jxr_linenumber" name="L237" href="#L237">237</a> <em class="jxr_javadoccomment">		Builder builder = new TranscriptionEngine.Builder();</em>
<a class="jxr_linenumber" name="L238" href="#L238">238</a> <em class="jxr_javadoccomment">		builder.initMet(false);</em>
<a class="jxr_linenumber" name="L239" href="#L239">239</a> <em class="jxr_javadoccomment">		builder.translateNCodons(true);</em>
<a class="jxr_linenumber" name="L240" href="#L240">240</a> <em class="jxr_javadoccomment">		builder.trimStop(false);</em>
<a class="jxr_linenumber" name="L241" href="#L241">241</a> <em class="jxr_javadoccomment">		TranscriptionEngine engine = builder.build();</em>
<a class="jxr_linenumber" name="L242" href="#L242">242</a> <em class="jxr_javadoccomment">		Sequence&lt;AminoAcidCompound&gt; seq=engine.translate(new</em>
<a class="jxr_linenumber" name="L243" href="#L243">243</a> <em class="jxr_javadoccomment">				DNASequence("GTNTGTTAGTGT"));</em>
<a class="jxr_linenumber" name="L244" href="#L244">244</a> <em class="jxr_javadoccomment">		assertThat("XC*C", is(seq.toString()));</em>
<a class="jxr_linenumber" name="L245" href="#L245">245</a> <em class="jxr_javadoccomment">		Sequence&lt;AminoAcidCompound&gt; seq2=engine.translate(new</em>
<a class="jxr_linenumber" name="L246" href="#L246">246</a> <em class="jxr_javadoccomment">				DNASequence("ANAANG"));</em>
<a class="jxr_linenumber" name="L247" href="#L247">247</a> <em class="jxr_javadoccomment">		assertEquals("XX",seq2.toString());</em>
<a class="jxr_linenumber" name="L248" href="#L248">248</a> <em class="jxr_javadoccomment">		assertNotSame("HR",seq2.toString());</em>
<a class="jxr_linenumber" name="L249" href="#L249">249</a> <em class="jxr_javadoccomment">	}</em>
<a class="jxr_linenumber" name="L250" href="#L250">250</a> <em class="jxr_javadoccomment">}</em>
</pre>
<hr/>
<div id="footer">Copyright &#169; 2000&#x2013;2015 <a href="http://www.biojava.org">BioJava</a>. All rights reserved.</div>
</body>
</html>
