<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head><meta http-equiv="content-type" content="text/html; charset=UTF-8" />
<title>FastaReader xref</title>
<link type="text/css" rel="stylesheet" href="../../../../../../stylesheet.css" />
</head>
<body>
<div id="overview"><a href="../../../../../../../apidocs/org/biojava/nbio/core/sequence/io/FastaReader.html">View Javadoc</a></div><pre>
<a class="jxr_linenumber" name="L1" href="#L1">1</a>   <em class="jxr_comment">/*</em>
<a class="jxr_linenumber" name="L2" href="#L2">2</a>   <em class="jxr_comment"> *                    BioJava development code</em>
<a class="jxr_linenumber" name="L3" href="#L3">3</a>   <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L4" href="#L4">4</a>   <em class="jxr_comment"> * This code may be freely distributed and modified under the</em>
<a class="jxr_linenumber" name="L5" href="#L5">5</a>   <em class="jxr_comment"> * terms of the GNU Lesser General Public Licence.  This should</em>
<a class="jxr_linenumber" name="L6" href="#L6">6</a>   <em class="jxr_comment"> * be distributed with the code.  If you do not have a copy,</em>
<a class="jxr_linenumber" name="L7" href="#L7">7</a>   <em class="jxr_comment"> * see:</em>
<a class="jxr_linenumber" name="L8" href="#L8">8</a>   <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L9" href="#L9">9</a>   <em class="jxr_comment"> *      <a href="http://www.gnu.org/copyleft/lesser.htm" target="alexandria_uri">http://www.gnu.org/copyleft/lesser.htm</a>l</em>
<a class="jxr_linenumber" name="L10" href="#L10">10</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L11" href="#L11">11</a>  <em class="jxr_comment"> * Copyright for this code is held jointly by the individual</em>
<a class="jxr_linenumber" name="L12" href="#L12">12</a>  <em class="jxr_comment"> * authors.  These should be listed in @author doc comments.</em>
<a class="jxr_linenumber" name="L13" href="#L13">13</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L14" href="#L14">14</a>  <em class="jxr_comment"> * For more information on the BioJava project and its aims,</em>
<a class="jxr_linenumber" name="L15" href="#L15">15</a>  <em class="jxr_comment"> * or to join the biojava-l mailing list, visit the home page</em>
<a class="jxr_linenumber" name="L16" href="#L16">16</a>  <em class="jxr_comment"> * at:</em>
<a class="jxr_linenumber" name="L17" href="#L17">17</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L18" href="#L18">18</a>  <em class="jxr_comment"> *      <a href="http://www.biojava.org" target="alexandria_uri">http://www.biojava.org</a>/</em>
<a class="jxr_linenumber" name="L19" href="#L19">19</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L20" href="#L20">20</a>  <em class="jxr_comment"> * Created on 01-21-2010</em>
<a class="jxr_linenumber" name="L21" href="#L21">21</a>  <em class="jxr_comment"> */</em>
<a class="jxr_linenumber" name="L22" href="#L22">22</a>  <strong class="jxr_keyword">package</strong> org.biojava.nbio.core.sequence.io;
<a class="jxr_linenumber" name="L23" href="#L23">23</a>  
<a class="jxr_linenumber" name="L24" href="#L24">24</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.exceptions.CompoundNotFoundException;
<a class="jxr_linenumber" name="L25" href="#L25">25</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.ProteinSequence;
<a class="jxr_linenumber" name="L26" href="#L26">26</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.compound.AminoAcidCompound;
<a class="jxr_linenumber" name="L27" href="#L27">27</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.compound.AminoAcidCompoundSet;
<a class="jxr_linenumber" name="L28" href="#L28">28</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.io.template.SequenceCreatorInterface;
<a class="jxr_linenumber" name="L29" href="#L29">29</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.io.template.SequenceHeaderParserInterface;
<a class="jxr_linenumber" name="L30" href="#L30">30</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.template.Compound;
<a class="jxr_linenumber" name="L31" href="#L31">31</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.template.Sequence;
<a class="jxr_linenumber" name="L32" href="#L32">32</a>  <strong class="jxr_keyword">import</strong> org.slf4j.Logger;
<a class="jxr_linenumber" name="L33" href="#L33">33</a>  <strong class="jxr_keyword">import</strong> org.slf4j.LoggerFactory;
<a class="jxr_linenumber" name="L34" href="#L34">34</a>  
<a class="jxr_linenumber" name="L35" href="#L35">35</a>  <strong class="jxr_keyword">import</strong> java.io.*;
<a class="jxr_linenumber" name="L36" href="#L36">36</a>  <strong class="jxr_keyword">import</strong> java.util.HashMap;
<a class="jxr_linenumber" name="L37" href="#L37">37</a>  <strong class="jxr_keyword">import</strong> java.util.LinkedHashMap;
<a class="jxr_linenumber" name="L38" href="#L38">38</a>  
<a class="jxr_linenumber" name="L39" href="#L39">39</a>  <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L40" href="#L40">40</a>  <em class="jxr_javadoccomment"> * Use FastaReaderHelper as an example of how to use this class where FastaReaderHelper should be the</em>
<a class="jxr_linenumber" name="L41" href="#L41">41</a>  <em class="jxr_javadoccomment"> * primary class used to read Fasta files</em>
<a class="jxr_linenumber" name="L42" href="#L42">42</a>  <em class="jxr_javadoccomment"> * @author Scooter Willis ;lt;willishf at gmail dot com&amp;gt;</em>
<a class="jxr_linenumber" name="L43" href="#L43">43</a>  <em class="jxr_javadoccomment"> */</em>
<a class="jxr_linenumber" name="L44" href="#L44">44</a>  <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">class</strong> FastaReader&lt;S <strong class="jxr_keyword">extends</strong> Sequence&lt;?&gt;, C <strong class="jxr_keyword">extends</strong> Compound&gt; {
<a class="jxr_linenumber" name="L45" href="#L45">45</a>  
<a class="jxr_linenumber" name="L46" href="#L46">46</a>  	<strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">final</strong> <strong class="jxr_keyword">static</strong> Logger logger = LoggerFactory.getLogger(FastaReader.<strong class="jxr_keyword">class</strong>);
<a class="jxr_linenumber" name="L47" href="#L47">47</a>  
<a class="jxr_linenumber" name="L48" href="#L48">48</a>      SequenceCreatorInterface&lt;C&gt; sequenceCreator;
<a class="jxr_linenumber" name="L49" href="#L49">49</a>      SequenceHeaderParserInterface&lt;S,C&gt; headerParser;
<a class="jxr_linenumber" name="L50" href="#L50">50</a>      <a href="../../../../../../org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html">BufferedReaderBytesRead</a> br;
<a class="jxr_linenumber" name="L51" href="#L51">51</a>      InputStreamReader isr;
<a class="jxr_linenumber" name="L52" href="#L52">52</a>      FileInputStream fi = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L53" href="#L53">53</a>      <strong class="jxr_keyword">long</strong> fileIndex = 0;
<a class="jxr_linenumber" name="L54" href="#L54">54</a>      <strong class="jxr_keyword">long</strong> sequenceIndex = 0;
<a class="jxr_linenumber" name="L55" href="#L55">55</a>      String line = <span class="jxr_string">""</span>;
<a class="jxr_linenumber" name="L56" href="#L56">56</a>      String header= <span class="jxr_string">""</span>;
<a class="jxr_linenumber" name="L57" href="#L57">57</a>      
<a class="jxr_linenumber" name="L58" href="#L58">58</a>      <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L59" href="#L59">59</a>  <em class="jxr_javadoccomment">     * If you are going to use FileProxyProteinSequenceCreator then do not use this constructor because we need details about</em>
<a class="jxr_linenumber" name="L60" href="#L60">60</a>  <em class="jxr_javadoccomment">     * local file offsets for quick reads. InputStreams does not give you the name of the stream to access quickly via file seek. A seek in</em>
<a class="jxr_linenumber" name="L61" href="#L61">61</a>  <em class="jxr_javadoccomment">     * an inputstream is forced to read all the data so you don't gain anything.</em>
<a class="jxr_linenumber" name="L62" href="#L62">62</a>  <em class="jxr_javadoccomment">     * @param br</em>
<a class="jxr_linenumber" name="L63" href="#L63">63</a>  <em class="jxr_javadoccomment">     * @param headerParser</em>
<a class="jxr_linenumber" name="L64" href="#L64">64</a>  <em class="jxr_javadoccomment">     * @param sequenceCreator</em>
<a class="jxr_linenumber" name="L65" href="#L65">65</a>  <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L66" href="#L66">66</a>      <strong class="jxr_keyword">public</strong> <a href="../../../../../../org/biojava/nbio/core/sequence/io/FastaReader.html">FastaReader</a>(InputStream is, SequenceHeaderParserInterface&lt;S,C&gt; headerParser,
<a class="jxr_linenumber" name="L67" href="#L67">67</a>      		SequenceCreatorInterface&lt;C&gt; sequenceCreator) {
<a class="jxr_linenumber" name="L68" href="#L68">68</a>          <strong class="jxr_keyword">this</strong>.headerParser = headerParser;
<a class="jxr_linenumber" name="L69" href="#L69">69</a>          isr = <strong class="jxr_keyword">new</strong> InputStreamReader(is);
<a class="jxr_linenumber" name="L70" href="#L70">70</a>          <strong class="jxr_keyword">this</strong>.br = <strong class="jxr_keyword">new</strong> <a href="../../../../../../org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html">BufferedReaderBytesRead</a>(isr);
<a class="jxr_linenumber" name="L71" href="#L71">71</a>          <strong class="jxr_keyword">this</strong>.sequenceCreator = sequenceCreator;
<a class="jxr_linenumber" name="L72" href="#L72">72</a>      }
<a class="jxr_linenumber" name="L73" href="#L73">73</a>  
<a class="jxr_linenumber" name="L74" href="#L74">74</a>      <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L75" href="#L75">75</a>  <em class="jxr_javadoccomment">     * If you are going to use the FileProxyProteinSequenceCreator then you</em>
<a class="jxr_linenumber" name="L76" href="#L76">76</a>  <em class="jxr_javadoccomment">     * need to use this constructor because we need details about</em>
<a class="jxr_linenumber" name="L77" href="#L77">77</a>  <em class="jxr_javadoccomment">     * the location of the file.</em>
<a class="jxr_linenumber" name="L78" href="#L78">78</a>  <em class="jxr_javadoccomment">     * @param file</em>
<a class="jxr_linenumber" name="L79" href="#L79">79</a>  <em class="jxr_javadoccomment">     * @param headerParser</em>
<a class="jxr_linenumber" name="L80" href="#L80">80</a>  <em class="jxr_javadoccomment">     * @param sequenceCreator</em>
<a class="jxr_linenumber" name="L81" href="#L81">81</a>  <em class="jxr_javadoccomment">     * @throws FileNotFoundException if the file does not exist, is a directory </em>
<a class="jxr_linenumber" name="L82" href="#L82">82</a>  <em class="jxr_javadoccomment">     * 	rather than a regular file, or for some other reason cannot be opened</em>
<a class="jxr_linenumber" name="L83" href="#L83">83</a>  <em class="jxr_javadoccomment">     * 	for reading.</em>
<a class="jxr_linenumber" name="L84" href="#L84">84</a>  <em class="jxr_javadoccomment">     * @throws SecurityException if a security manager exists and its checkRead</em>
<a class="jxr_linenumber" name="L85" href="#L85">85</a>  <em class="jxr_javadoccomment">     * 	method denies read access to the file.</em>
<a class="jxr_linenumber" name="L86" href="#L86">86</a>  <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L87" href="#L87">87</a>      <strong class="jxr_keyword">public</strong> <a href="../../../../../../org/biojava/nbio/core/sequence/io/FastaReader.html">FastaReader</a>(File file, SequenceHeaderParserInterface&lt;S,C&gt; headerParser,
<a class="jxr_linenumber" name="L88" href="#L88">88</a>      		SequenceCreatorInterface&lt;C&gt; sequenceCreator) <strong class="jxr_keyword">throws</strong> FileNotFoundException {
<a class="jxr_linenumber" name="L89" href="#L89">89</a>          <strong class="jxr_keyword">this</strong>.headerParser = headerParser;
<a class="jxr_linenumber" name="L90" href="#L90">90</a>          fi = <strong class="jxr_keyword">new</strong> FileInputStream(file);
<a class="jxr_linenumber" name="L91" href="#L91">91</a>          isr = <strong class="jxr_keyword">new</strong> InputStreamReader(fi);
<a class="jxr_linenumber" name="L92" href="#L92">92</a>          <strong class="jxr_keyword">this</strong>.br = <strong class="jxr_keyword">new</strong> <a href="../../../../../../org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html">BufferedReaderBytesRead</a>(isr);
<a class="jxr_linenumber" name="L93" href="#L93">93</a>          <strong class="jxr_keyword">this</strong>.sequenceCreator = sequenceCreator;
<a class="jxr_linenumber" name="L94" href="#L94">94</a>      }
<a class="jxr_linenumber" name="L95" href="#L95">95</a>  
<a class="jxr_linenumber" name="L96" href="#L96">96</a>      <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L97" href="#L97">97</a>  <em class="jxr_javadoccomment">     * The parsing is done in this method.&lt;br&gt;</em>
<a class="jxr_linenumber" name="L98" href="#L98">98</a>  <em class="jxr_javadoccomment">     * This method tries to process all the available fasta records </em>
<a class="jxr_linenumber" name="L99" href="#L99">99</a>  <em class="jxr_javadoccomment">     * in the File or InputStream, closes the underlying resource, </em>
<a class="jxr_linenumber" name="L100" href="#L100">100</a> <em class="jxr_javadoccomment">     * and return the results in {@link LinkedHashMap}.&lt;br&gt;</em>
<a class="jxr_linenumber" name="L101" href="#L101">101</a> <em class="jxr_javadoccomment">     * You don't need to call {@link #close()} after calling this method.</em>
<a class="jxr_linenumber" name="L102" href="#L102">102</a> <em class="jxr_javadoccomment">     * @see #process(int)</em>
<a class="jxr_linenumber" name="L103" href="#L103">103</a> <em class="jxr_javadoccomment">     * @return {@link HashMap} containing all the parsed fasta records </em>
<a class="jxr_linenumber" name="L104" href="#L104">104</a> <em class="jxr_javadoccomment">     * present, starting current fileIndex onwards.</em>
<a class="jxr_linenumber" name="L105" href="#L105">105</a> <em class="jxr_javadoccomment">     * @throws IOException if an error occurs reading the input file</em>
<a class="jxr_linenumber" name="L106" href="#L106">106</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L107" href="#L107">107</a>     <strong class="jxr_keyword">public</strong> LinkedHashMap&lt;String,S&gt; process() <strong class="jxr_keyword">throws</strong> IOException {
<a class="jxr_linenumber" name="L108" href="#L108">108</a>     	LinkedHashMap&lt;String,S&gt; sequences = process(-1);
<a class="jxr_linenumber" name="L109" href="#L109">109</a>     	close();
<a class="jxr_linenumber" name="L110" href="#L110">110</a>     	<strong class="jxr_keyword">return</strong> sequences;
<a class="jxr_linenumber" name="L111" href="#L111">111</a>     }
<a class="jxr_linenumber" name="L112" href="#L112">112</a> 
<a class="jxr_linenumber" name="L113" href="#L113">113</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L114" href="#L114">114</a> <em class="jxr_javadoccomment">     * This method tries to parse maximum &lt;code&gt;max&lt;/code&gt; records from</em>
<a class="jxr_linenumber" name="L115" href="#L115">115</a> <em class="jxr_javadoccomment">     * the open File or InputStream, and leaves the underlying resource open.&lt;br&gt;</em>
<a class="jxr_linenumber" name="L116" href="#L116">116</a> <em class="jxr_javadoccomment">     * Subsequent calls to the same method continue parsing the rest of the file.&lt;br&gt;</em>
<a class="jxr_linenumber" name="L117" href="#L117">117</a> <em class="jxr_javadoccomment">     * This is particularly useful when dealing with very big data files,</em>
<a class="jxr_linenumber" name="L118" href="#L118">118</a> <em class="jxr_javadoccomment">     * (e.g. NCBI nr database), which can't fit into memory and will take long</em>
<a class="jxr_linenumber" name="L119" href="#L119">119</a> <em class="jxr_javadoccomment">     * time before the first result is available.&lt;br&gt;</em>
<a class="jxr_linenumber" name="L120" href="#L120">120</a> <em class="jxr_javadoccomment">     * &lt;b&gt;N.B.&lt;/b&gt;</em>
<a class="jxr_linenumber" name="L121" href="#L121">121</a> <em class="jxr_javadoccomment">     * &lt;ul&gt;</em>
<a class="jxr_linenumber" name="L122" href="#L122">122</a> <em class="jxr_javadoccomment">     * &lt;li&gt;This method can't be called after calling its NO-ARGUMENT twin.&lt;/li&gt; </em>
<a class="jxr_linenumber" name="L123" href="#L123">123</a> <em class="jxr_javadoccomment">     * &lt;li&gt;remember to close the underlying resource when you are done.&lt;/li&gt; </em>
<a class="jxr_linenumber" name="L124" href="#L124">124</a> <em class="jxr_javadoccomment">     * &lt;/ul&gt;</em>
<a class="jxr_linenumber" name="L125" href="#L125">125</a> <em class="jxr_javadoccomment">     * @see #process()</em>
<a class="jxr_linenumber" name="L126" href="#L126">126</a> <em class="jxr_javadoccomment">     * @author Amr AL-Hossary</em>
<a class="jxr_linenumber" name="L127" href="#L127">127</a> <em class="jxr_javadoccomment">     * @since 3.0.6</em>
<a class="jxr_linenumber" name="L128" href="#L128">128</a> <em class="jxr_javadoccomment">     * @param max maximum number of records to return, &lt;code&gt;-1&lt;/code&gt; for infinity.</em>
<a class="jxr_linenumber" name="L129" href="#L129">129</a> <em class="jxr_javadoccomment">     * @return {@link HashMap} containing maximum &lt;code&gt;max&lt;/code&gt; parsed fasta records </em>
<a class="jxr_linenumber" name="L130" href="#L130">130</a> <em class="jxr_javadoccomment">     * present, starting current fileIndex onwards.</em>
<a class="jxr_linenumber" name="L131" href="#L131">131</a> <em class="jxr_javadoccomment">     * @throws IOException if an error occurs reading the input file</em>
<a class="jxr_linenumber" name="L132" href="#L132">132</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L133" href="#L133">133</a> 	<strong class="jxr_keyword">public</strong> LinkedHashMap&lt;String,S&gt; process(<strong class="jxr_keyword">int</strong> max) <strong class="jxr_keyword">throws</strong> IOException {
<a class="jxr_linenumber" name="L134" href="#L134">134</a>         LinkedHashMap&lt;String,S&gt; sequences = <strong class="jxr_keyword">new</strong> LinkedHashMap&lt;String,S&gt;();
<a class="jxr_linenumber" name="L135" href="#L135">135</a> 
<a class="jxr_linenumber" name="L136" href="#L136">136</a>         String line = <span class="jxr_string">""</span>;
<a class="jxr_linenumber" name="L137" href="#L137">137</a>         <strong class="jxr_keyword">if</strong>(<strong class="jxr_keyword">this</strong>.line != <strong class="jxr_keyword">null</strong> &amp;&amp; <strong class="jxr_keyword">this</strong>.line.length() &gt; 0){
<a class="jxr_linenumber" name="L138" href="#L138">138</a>         	line=<strong class="jxr_keyword">this</strong>.line;
<a class="jxr_linenumber" name="L139" href="#L139">139</a>         }
<a class="jxr_linenumber" name="L140" href="#L140">140</a>         String header = <span class="jxr_string">""</span>;
<a class="jxr_linenumber" name="L141" href="#L141">141</a>         <strong class="jxr_keyword">if</strong>(<strong class="jxr_keyword">this</strong>.header != <strong class="jxr_keyword">null</strong> &amp;&amp; <strong class="jxr_keyword">this</strong>.header.length() &gt; 0){
<a class="jxr_linenumber" name="L142" href="#L142">142</a>         	header=<strong class="jxr_keyword">this</strong>.header;
<a class="jxr_linenumber" name="L143" href="#L143">143</a>         }
<a class="jxr_linenumber" name="L144" href="#L144">144</a>         
<a class="jxr_linenumber" name="L145" href="#L145">145</a>         StringBuilder sb = <strong class="jxr_keyword">new</strong> StringBuilder();
<a class="jxr_linenumber" name="L146" href="#L146">146</a>         <strong class="jxr_keyword">int</strong> processedSequences=0;
<a class="jxr_linenumber" name="L147" href="#L147">147</a>         <strong class="jxr_keyword">boolean</strong> keepGoing = <strong class="jxr_keyword">true</strong>;
<a class="jxr_linenumber" name="L148" href="#L148">148</a> 
<a class="jxr_linenumber" name="L149" href="#L149">149</a>         <strong class="jxr_keyword">do</strong> {
<a class="jxr_linenumber" name="L150" href="#L150">150</a>             line = line.trim(); <em class="jxr_comment">// nice to have but probably not needed</em>
<a class="jxr_linenumber" name="L151" href="#L151">151</a>             <strong class="jxr_keyword">if</strong> (line.length() != 0) {
<a class="jxr_linenumber" name="L152" href="#L152">152</a>                 <strong class="jxr_keyword">if</strong> (line.startsWith(<span class="jxr_string">"&gt;"</span>)) {<em class="jxr_comment">//start of new fasta record</em>
<a class="jxr_linenumber" name="L153" href="#L153">153</a>                     <strong class="jxr_keyword">if</strong> (sb.length() &gt; 0) {<em class="jxr_comment">//i.e. if there is already a sequence before</em>
<a class="jxr_linenumber" name="L154" href="#L154">154</a>                     <em class="jxr_comment">//    logger.debug("Sequence index=" + sequenceIndex);</em>
<a class="jxr_linenumber" name="L155" href="#L155">155</a>                     	
<a class="jxr_linenumber" name="L156" href="#L156">156</a>                     	<strong class="jxr_keyword">try</strong> {    
<a class="jxr_linenumber" name="L157" href="#L157">157</a>                     	    @SuppressWarnings(<span class="jxr_string">"unchecked"</span>)
<a class="jxr_linenumber" name="L158" href="#L158">158</a>                     		S sequence = (S)sequenceCreator.getSequence(sb.toString(), sequenceIndex);
<a class="jxr_linenumber" name="L159" href="#L159">159</a>                             headerParser.parseHeader(header, sequence);
<a class="jxr_linenumber" name="L160" href="#L160">160</a>                             sequences.put(sequence.getAccession().getID(),sequence);
<a class="jxr_linenumber" name="L161" href="#L161">161</a>                             processedSequences++;
<a class="jxr_linenumber" name="L162" href="#L162">162</a> 
<a class="jxr_linenumber" name="L163" href="#L163">163</a>                     	} <strong class="jxr_keyword">catch</strong> (CompoundNotFoundException e) {
<a class="jxr_linenumber" name="L164" href="#L164">164</a>                     		logger.warn(<span class="jxr_string">"Sequence with header '{}' has unrecognised compounds ({}), it will be ignored"</span>, 
<a class="jxr_linenumber" name="L165" href="#L165">165</a>                     				header, e.getMessage());
<a class="jxr_linenumber" name="L166" href="#L166">166</a>                     	}
<a class="jxr_linenumber" name="L167" href="#L167">167</a> <em class="jxr_comment">//                        if (maxSequenceLength &lt; sb.length()) {</em>
<a class="jxr_linenumber" name="L168" href="#L168">168</a> <em class="jxr_comment">//                            maxSequenceLength = sb.length();</em>
<a class="jxr_linenumber" name="L169" href="#L169">169</a> <em class="jxr_comment">//                        }</em>
<a class="jxr_linenumber" name="L170" href="#L170">170</a> <em class="jxr_comment">//                        sb = new StringBuilder(maxSequenceLength);</em>
<a class="jxr_linenumber" name="L171" href="#L171">171</a>                         sb.setLength(0); <em class="jxr_comment">//this is faster, better memory utilization (same buffer)</em>
<a class="jxr_linenumber" name="L172" href="#L172">172</a>                     }
<a class="jxr_linenumber" name="L173" href="#L173">173</a>                     header = line.substring(1);
<a class="jxr_linenumber" name="L174" href="#L174">174</a>                 } <strong class="jxr_keyword">else</strong> <strong class="jxr_keyword">if</strong> (line.startsWith(<span class="jxr_string">";"</span>)) {
<a class="jxr_linenumber" name="L175" href="#L175">175</a>                 } <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L176" href="#L176">176</a>                     <em class="jxr_comment">//mark the start of the sequence with the fileIndex before the line was read</em>
<a class="jxr_linenumber" name="L177" href="#L177">177</a>                     <strong class="jxr_keyword">if</strong>(sb.length() == 0){
<a class="jxr_linenumber" name="L178" href="#L178">178</a>                         sequenceIndex = fileIndex;
<a class="jxr_linenumber" name="L179" href="#L179">179</a>                     }
<a class="jxr_linenumber" name="L180" href="#L180">180</a>                     sb.append(line);
<a class="jxr_linenumber" name="L181" href="#L181">181</a>                 }
<a class="jxr_linenumber" name="L182" href="#L182">182</a>             }
<a class="jxr_linenumber" name="L183" href="#L183">183</a>             fileIndex = br.getBytesRead();
<a class="jxr_linenumber" name="L184" href="#L184">184</a>             line = br.readLine();
<a class="jxr_linenumber" name="L185" href="#L185">185</a> 			<strong class="jxr_keyword">if</strong> (line == <strong class="jxr_keyword">null</strong>) {<em class="jxr_comment">//i.e. EOF</em>
<a class="jxr_linenumber" name="L186" href="#L186">186</a>                 String seq = sb.toString();
<a class="jxr_linenumber" name="L187" href="#L187">187</a>                 <strong class="jxr_keyword">if</strong> ( seq.length() == 0) {
<a class="jxr_linenumber" name="L188" href="#L188">188</a>                     logger.warn(<span class="jxr_string">"Can't parse sequence {}. Got sequence of length 0!"</span>, sequenceIndex);
<a class="jxr_linenumber" name="L189" href="#L189">189</a>                     logger.warn(<span class="jxr_string">"header: {}"</span>, header);
<a class="jxr_linenumber" name="L190" href="#L190">190</a>                 }
<a class="jxr_linenumber" name="L191" href="#L191">191</a>                 <em class="jxr_comment">//    logger.debug("Sequence index=" + sequenceIndex + " " + fileIndex );</em>
<a class="jxr_linenumber" name="L192" href="#L192">192</a>                 <strong class="jxr_keyword">try</strong> {
<a class="jxr_linenumber" name="L193" href="#L193">193</a>                 	@SuppressWarnings(<span class="jxr_string">"unchecked"</span>)
<a class="jxr_linenumber" name="L194" href="#L194">194</a>                 	S sequence = (S)sequenceCreator.getSequence(seq, sequenceIndex);
<a class="jxr_linenumber" name="L195" href="#L195">195</a>                 	headerParser.parseHeader(header, sequence);
<a class="jxr_linenumber" name="L196" href="#L196">196</a>                 	sequences.put(sequence.getAccession().getID(),sequence);
<a class="jxr_linenumber" name="L197" href="#L197">197</a>                 	processedSequences++;                	
<a class="jxr_linenumber" name="L198" href="#L198">198</a>                 } <strong class="jxr_keyword">catch</strong> (CompoundNotFoundException e) {
<a class="jxr_linenumber" name="L199" href="#L199">199</a>             		logger.warn(<span class="jxr_string">"Sequence with header '{}' has unrecognised compounds ({}), it will be ignored"</span>, 
<a class="jxr_linenumber" name="L200" href="#L200">200</a>             				header, e.getMessage());
<a class="jxr_linenumber" name="L201" href="#L201">201</a>             	}
<a class="jxr_linenumber" name="L202" href="#L202">202</a>                 keepGoing = false;
<a class="jxr_linenumber" name="L203" href="#L203">203</a>             }
<a class="jxr_linenumber" name="L204" href="#L204">204</a> 			<strong class="jxr_keyword">if</strong> (max &gt; -1 &amp;&amp; processedSequences&gt;=max) {
<a class="jxr_linenumber" name="L205" href="#L205">205</a> 				keepGoing=false;
<a class="jxr_linenumber" name="L206" href="#L206">206</a> 			}
<a class="jxr_linenumber" name="L207" href="#L207">207</a>         } <strong class="jxr_keyword">while</strong> (keepGoing);
<a class="jxr_linenumber" name="L208" href="#L208">208</a>         <strong class="jxr_keyword">this</strong>.line  = line;
<a class="jxr_linenumber" name="L209" href="#L209">209</a>         <strong class="jxr_keyword">this</strong>.header= header;
<a class="jxr_linenumber" name="L210" href="#L210">210</a>         <strong class="jxr_keyword">return</strong> sequences;
<a class="jxr_linenumber" name="L211" href="#L211">211</a>     }
<a class="jxr_linenumber" name="L212" href="#L212">212</a> 
<a class="jxr_linenumber" name="L213" href="#L213">213</a> 	<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> close() <strong class="jxr_keyword">throws</strong> IOException {
<a class="jxr_linenumber" name="L214" href="#L214">214</a> 		br.close();
<a class="jxr_linenumber" name="L215" href="#L215">215</a>         isr.close();
<a class="jxr_linenumber" name="L216" href="#L216">216</a>         <em class="jxr_comment">//If stream was created from File object then we need to close it</em>
<a class="jxr_linenumber" name="L217" href="#L217">217</a>         <strong class="jxr_keyword">if</strong> (fi != <strong class="jxr_keyword">null</strong>) {
<a class="jxr_linenumber" name="L218" href="#L218">218</a>             fi.close();
<a class="jxr_linenumber" name="L219" href="#L219">219</a>         }
<a class="jxr_linenumber" name="L220" href="#L220">220</a>         <strong class="jxr_keyword">this</strong>.line=<strong class="jxr_keyword">this</strong>.header = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L221" href="#L221">221</a> 	}
<a class="jxr_linenumber" name="L222" href="#L222">222</a> 
<a class="jxr_linenumber" name="L223" href="#L223">223</a>     <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">void</strong> main(String[] args) {
<a class="jxr_linenumber" name="L224" href="#L224">224</a>         <strong class="jxr_keyword">try</strong> {
<a class="jxr_linenumber" name="L225" href="#L225">225</a>             String inputFile = <span class="jxr_string">"src/test/resources/PF00104_small.fasta"</span>;
<a class="jxr_linenumber" name="L226" href="#L226">226</a>             FileInputStream is = <strong class="jxr_keyword">new</strong> FileInputStream(inputFile);
<a class="jxr_linenumber" name="L227" href="#L227">227</a> 
<a class="jxr_linenumber" name="L228" href="#L228">228</a>             FastaReader&lt;ProteinSequence, AminoAcidCompound&gt; fastaReader = <strong class="jxr_keyword">new</strong> FastaReader&lt;ProteinSequence, AminoAcidCompound&gt;(is, <strong class="jxr_keyword">new</strong> GenericFastaHeaderParser&lt;ProteinSequence,AminoAcidCompound&gt;(), <strong class="jxr_keyword">new</strong> ProteinSequenceCreator(AminoAcidCompoundSet.getAminoAcidCompoundSet()));
<a class="jxr_linenumber" name="L229" href="#L229">229</a>             LinkedHashMap&lt;String,ProteinSequence&gt; proteinSequences = fastaReader.process();
<a class="jxr_linenumber" name="L230" href="#L230">230</a>             is.close();
<a class="jxr_linenumber" name="L231" href="#L231">231</a> 
<a class="jxr_linenumber" name="L232" href="#L232">232</a> 
<a class="jxr_linenumber" name="L233" href="#L233">233</a>             logger.info(<span class="jxr_string">"Protein Sequences: {}"</span>, proteinSequences);
<a class="jxr_linenumber" name="L234" href="#L234">234</a> 
<a class="jxr_linenumber" name="L235" href="#L235">235</a>             File file = <strong class="jxr_keyword">new</strong> File(inputFile);
<a class="jxr_linenumber" name="L236" href="#L236">236</a>             FastaReader&lt;ProteinSequence,AminoAcidCompound&gt; fastaProxyReader = 
<a class="jxr_linenumber" name="L237" href="#L237">237</a>             		<strong class="jxr_keyword">new</strong> FastaReader&lt;ProteinSequence,AminoAcidCompound&gt;(
<a class="jxr_linenumber" name="L238" href="#L238">238</a>             				file, 
<a class="jxr_linenumber" name="L239" href="#L239">239</a>             				<strong class="jxr_keyword">new</strong> GenericFastaHeaderParser&lt;ProteinSequence,AminoAcidCompound&gt;(),
<a class="jxr_linenumber" name="L240" href="#L240">240</a>             				<strong class="jxr_keyword">new</strong> <a href="../../../../../../org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html">FileProxyProteinSequenceCreator</a>(
<a class="jxr_linenumber" name="L241" href="#L241">241</a>             						file, 
<a class="jxr_linenumber" name="L242" href="#L242">242</a>             						AminoAcidCompoundSet.getAminoAcidCompoundSet(), 
<a class="jxr_linenumber" name="L243" href="#L243">243</a>             						<strong class="jxr_keyword">new</strong> <a href="../../../../../../org/biojava/nbio/core/sequence/io/FastaSequenceParser.html">FastaSequenceParser</a>()
<a class="jxr_linenumber" name="L244" href="#L244">244</a>         						) 
<a class="jxr_linenumber" name="L245" href="#L245">245</a>         				);
<a class="jxr_linenumber" name="L246" href="#L246">246</a>             LinkedHashMap&lt;String,ProteinSequence&gt; proteinProxySequences = fastaProxyReader.process();
<a class="jxr_linenumber" name="L247" href="#L247">247</a> 
<a class="jxr_linenumber" name="L248" href="#L248">248</a>             <strong class="jxr_keyword">for</strong>(String key : proteinProxySequences.keySet()){
<a class="jxr_linenumber" name="L249" href="#L249">249</a>                 <a href="../../../../../../org/biojava/nbio/core/sequence/ProteinSequence.html">ProteinSequence</a> proteinSequence = proteinProxySequences.get(key);
<a class="jxr_linenumber" name="L250" href="#L250">250</a>                 logger.info(<span class="jxr_string">"Protein Proxy Sequence Key: {}"</span>, key);
<a class="jxr_linenumber" name="L251" href="#L251">251</a> <em class="jxr_comment">//                if(key.equals("Q98SJ1_CHICK/15-61")){</em>
<a class="jxr_linenumber" name="L252" href="#L252">252</a> <em class="jxr_comment">//                    int dummy = 1;</em>
<a class="jxr_linenumber" name="L253" href="#L253">253</a> <em class="jxr_comment">//                }</em>
<a class="jxr_linenumber" name="L254" href="#L254">254</a>                 logger.info(<span class="jxr_string">"Protein Sequence: {}"</span>, proteinSequence.toString());
<a class="jxr_linenumber" name="L255" href="#L255">255</a> 
<a class="jxr_linenumber" name="L256" href="#L256">256</a>             }
<a class="jxr_linenumber" name="L257" href="#L257">257</a> 
<a class="jxr_linenumber" name="L258" href="#L258">258</a>         } <strong class="jxr_keyword">catch</strong> (Exception e) {
<a class="jxr_linenumber" name="L259" href="#L259">259</a>             logger.warn(<span class="jxr_string">"Exception: "</span>, e);
<a class="jxr_linenumber" name="L260" href="#L260">260</a>         }
<a class="jxr_linenumber" name="L261" href="#L261">261</a>     }
<a class="jxr_linenumber" name="L262" href="#L262">262</a> }
</pre>
<hr/>
<div id="footer">Copyright &#169; 2000&#x2013;2015 <a href="http://www.biojava.org">BioJava</a>. All rights reserved.</div>
</body>
</html>
