<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head><meta http-equiv="content-type" content="text/html; charset=UTF-8" />
<title>IUPACParser xref</title>
<link type="text/css" rel="stylesheet" href="../../../../../../stylesheet.css" />
</head>
<body>
<div id="overview"><a href="../../../../../../../apidocs/org/biojava/nbio/core/sequence/io/IUPACParser.html">View Javadoc</a></div><pre>
<a class="jxr_linenumber" name="L1" href="#L1">1</a>   <em class="jxr_comment">/*</em>
<a class="jxr_linenumber" name="L2" href="#L2">2</a>   <em class="jxr_comment"> *                    BioJava development code</em>
<a class="jxr_linenumber" name="L3" href="#L3">3</a>   <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L4" href="#L4">4</a>   <em class="jxr_comment"> * This code may be freely distributed and modified under the</em>
<a class="jxr_linenumber" name="L5" href="#L5">5</a>   <em class="jxr_comment"> * terms of the GNU Lesser General Public Licence.  This should</em>
<a class="jxr_linenumber" name="L6" href="#L6">6</a>   <em class="jxr_comment"> * be distributed with the code.  If you do not have a copy,</em>
<a class="jxr_linenumber" name="L7" href="#L7">7</a>   <em class="jxr_comment"> * see:</em>
<a class="jxr_linenumber" name="L8" href="#L8">8</a>   <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L9" href="#L9">9</a>   <em class="jxr_comment"> *      <a href="http://www.gnu.org/copyleft/lesser.htm" target="alexandria_uri">http://www.gnu.org/copyleft/lesser.htm</a>l</em>
<a class="jxr_linenumber" name="L10" href="#L10">10</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L11" href="#L11">11</a>  <em class="jxr_comment"> * Copyright for this code is held jointly by the individual</em>
<a class="jxr_linenumber" name="L12" href="#L12">12</a>  <em class="jxr_comment"> * authors.  These should be listed in @author doc comments.</em>
<a class="jxr_linenumber" name="L13" href="#L13">13</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L14" href="#L14">14</a>  <em class="jxr_comment"> * For more information on the BioJava project and its aims,</em>
<a class="jxr_linenumber" name="L15" href="#L15">15</a>  <em class="jxr_comment"> * or to join the biojava-l mailing list, visit the home page</em>
<a class="jxr_linenumber" name="L16" href="#L16">16</a>  <em class="jxr_comment"> * at:</em>
<a class="jxr_linenumber" name="L17" href="#L17">17</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L18" href="#L18">18</a>  <em class="jxr_comment"> *      <a href="http://www.biojava.org" target="alexandria_uri">http://www.biojava.org</a>/</em>
<a class="jxr_linenumber" name="L19" href="#L19">19</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L20" href="#L20">20</a>  <em class="jxr_comment"> * Created on 01-21-2010</em>
<a class="jxr_linenumber" name="L21" href="#L21">21</a>  <em class="jxr_comment"> */</em>
<a class="jxr_linenumber" name="L22" href="#L22">22</a>  <strong class="jxr_keyword">package</strong> org.biojava.nbio.core.sequence.io;
<a class="jxr_linenumber" name="L23" href="#L23">23</a>  
<a class="jxr_linenumber" name="L24" href="#L24">24</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.exceptions.ParserException;
<a class="jxr_linenumber" name="L25" href="#L25">25</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.compound.AminoAcidCompound;
<a class="jxr_linenumber" name="L26" href="#L26">26</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.compound.NucleotideCompound;
<a class="jxr_linenumber" name="L27" href="#L27">27</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.io.util.ClasspathResource;
<a class="jxr_linenumber" name="L28" href="#L28">28</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.io.util.IOUtils;
<a class="jxr_linenumber" name="L29" href="#L29">29</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.template.AbstractCompoundSet;
<a class="jxr_linenumber" name="L30" href="#L30">30</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.template.CompoundSet;
<a class="jxr_linenumber" name="L31" href="#L31">31</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.transcription.Table;
<a class="jxr_linenumber" name="L32" href="#L32">32</a>  
<a class="jxr_linenumber" name="L33" href="#L33">33</a>  <strong class="jxr_keyword">import</strong> java.io.InputStream;
<a class="jxr_linenumber" name="L34" href="#L34">34</a>  <strong class="jxr_keyword">import</strong> java.util.*;
<a class="jxr_linenumber" name="L35" href="#L35">35</a>  
<a class="jxr_linenumber" name="L36" href="#L36">36</a>  
<a class="jxr_linenumber" name="L37" href="#L37">37</a>  <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L38" href="#L38">38</a>  <em class="jxr_javadoccomment"> * Available translations</em>
<a class="jxr_linenumber" name="L39" href="#L39">39</a>  <em class="jxr_javadoccomment"> *</em>
<a class="jxr_linenumber" name="L40" href="#L40">40</a>  <em class="jxr_javadoccomment"> * &lt;ul&gt;</em>
<a class="jxr_linenumber" name="L41" href="#L41">41</a>  <em class="jxr_javadoccomment"> * &lt;li&gt;1 - UNIVERSAL&lt;/li&gt;</em>
<a class="jxr_linenumber" name="L42" href="#L42">42</a>  <em class="jxr_javadoccomment"> * &lt;li&gt;2 - VERTEBRATE_MITOCHONDRIAL&lt;/li&gt;</em>
<a class="jxr_linenumber" name="L43" href="#L43">43</a>  <em class="jxr_javadoccomment"> * &lt;li&gt;3 - YEAST_MITOCHONDRIAL&lt;/li&gt;</em>
<a class="jxr_linenumber" name="L44" href="#L44">44</a>  <em class="jxr_javadoccomment"> * &lt;li&gt;4 - MOLD_MITOCHONDRIAL&lt;/li&gt;</em>
<a class="jxr_linenumber" name="L45" href="#L45">45</a>  <em class="jxr_javadoccomment"> * &lt;li&gt;5 - INVERTEBRATE_MITOCHONDRIAL&lt;/li&gt;</em>
<a class="jxr_linenumber" name="L46" href="#L46">46</a>  <em class="jxr_javadoccomment"> * &lt;li&gt;6 - CILIATE_NUCLEAR&lt;/li&gt;</em>
<a class="jxr_linenumber" name="L47" href="#L47">47</a>  <em class="jxr_javadoccomment"> * &lt;li&gt;9 - ECHINODERM_MITOCHONDRIAL&lt;/li&gt;</em>
<a class="jxr_linenumber" name="L48" href="#L48">48</a>  <em class="jxr_javadoccomment"> * &lt;li&gt;10 - EUPLOTID_NUCLEAR&lt;/li&gt;</em>
<a class="jxr_linenumber" name="L49" href="#L49">49</a>  <em class="jxr_javadoccomment"> * &lt;li&gt;11 - BACTERIAL&lt;/li&gt;</em>
<a class="jxr_linenumber" name="L50" href="#L50">50</a>  <em class="jxr_javadoccomment"> * &lt;li&gt;12 - ALTERNATIVE_YEAST_NUCLEAR&lt;/li&gt;</em>
<a class="jxr_linenumber" name="L51" href="#L51">51</a>  <em class="jxr_javadoccomment"> * &lt;li&gt;13 - ASCIDIAN_MITOCHONDRIAL&lt;/li&gt;</em>
<a class="jxr_linenumber" name="L52" href="#L52">52</a>  <em class="jxr_javadoccomment"> * &lt;li&gt;14 - FLATWORM_MITOCHONDRIAL&lt;/li&gt;</em>
<a class="jxr_linenumber" name="L53" href="#L53">53</a>  <em class="jxr_javadoccomment"> * &lt;li&gt;15 - BLEPHARISMA_MACRONUCLEAR&lt;/li&gt;</em>
<a class="jxr_linenumber" name="L54" href="#L54">54</a>  <em class="jxr_javadoccomment"> * &lt;li&gt;16 - 2CHLOROPHYCEAN_MITOCHONDRIAL&lt;/li&gt;</em>
<a class="jxr_linenumber" name="L55" href="#L55">55</a>  <em class="jxr_javadoccomment"> * &lt;li&gt;21 - TREMATODE_MITOCHONDRIAL&lt;/li&gt;</em>
<a class="jxr_linenumber" name="L56" href="#L56">56</a>  <em class="jxr_javadoccomment"> * &lt;li&gt;23 - SCENEDESMUS_MITOCHONDRIAL&lt;/li&gt;</em>
<a class="jxr_linenumber" name="L57" href="#L57">57</a>  <em class="jxr_javadoccomment"> * &lt;/ul&gt;</em>
<a class="jxr_linenumber" name="L58" href="#L58">58</a>  <em class="jxr_javadoccomment"> *</em>
<a class="jxr_linenumber" name="L59" href="#L59">59</a>  <em class="jxr_javadoccomment"> * Taken from &lt;a</em>
<a class="jxr_linenumber" name="L60" href="#L60">60</a>  <em class="jxr_javadoccomment"> * href="<a href="http://www.ncbi.nlm.nih.gov/Taxonomy/Utils/wprintgc.cgi?mode=c" target="alexandria_uri">http://www.ncbi.nlm.nih.gov/Taxonomy/Utils/wprintgc.cgi?mode=c</a>"</em>
<a class="jxr_linenumber" name="L61" href="#L61">61</a>  <em class="jxr_javadoccomment"> * &gt;NCBI&lt;/a&gt; with slight modification and put into the classpath resource.</em>
<a class="jxr_linenumber" name="L62" href="#L62">62</a>  <em class="jxr_javadoccomment"> *</em>
<a class="jxr_linenumber" name="L63" href="#L63">63</a>  <em class="jxr_javadoccomment"> * Takes in an ID, name, amino acid string and the locations of amino acids</em>
<a class="jxr_linenumber" name="L64" href="#L64">64</a>  <em class="jxr_javadoccomment"> * which acts as start codons in the translation table. You can give the 3 codon</em>
<a class="jxr_linenumber" name="L65" href="#L65">65</a>  <em class="jxr_javadoccomment"> * position strings that correspond to the amino acid string or if you are using</em>
<a class="jxr_linenumber" name="L66" href="#L66">66</a>  <em class="jxr_javadoccomment"> * the default IUPAC codes you can use the hardcoded ones which are consistent</em>
<a class="jxr_linenumber" name="L67" href="#L67">67</a>  <em class="jxr_javadoccomment"> * amongst all &lt;a</em>
<a class="jxr_linenumber" name="L68" href="#L68">68</a>  <em class="jxr_javadoccomment"> * href="<a href="http://www.ncbi.nlm.nih.gov/Taxonomy/Utils/wprintgc.cgi?mode=c" target="alexandria_uri">http://www.ncbi.nlm.nih.gov/Taxonomy/Utils/wprintgc.cgi?mode=c</a>"&gt; codon</em>
<a class="jxr_linenumber" name="L69" href="#L69">69</a>  <em class="jxr_javadoccomment"> * tables&lt;/a&gt;.</em>
<a class="jxr_linenumber" name="L70" href="#L70">70</a>  <em class="jxr_javadoccomment"> *</em>
<a class="jxr_linenumber" name="L71" href="#L71">71</a>  <em class="jxr_javadoccomment"> * The generated {@link IUPACTable} objects do not parse the data further until</em>
<a class="jxr_linenumber" name="L72" href="#L72">72</a>  <em class="jxr_javadoccomment"> * requested so if you do not use a translation table your only penalty is the</em>
<a class="jxr_linenumber" name="L73" href="#L73">73</a>  <em class="jxr_javadoccomment"> * loading of the IUPAC data from the classpath.</em>
<a class="jxr_linenumber" name="L74" href="#L74">74</a>  <em class="jxr_javadoccomment"> *</em>
<a class="jxr_linenumber" name="L75" href="#L75">75</a>  <em class="jxr_javadoccomment"> * @author Andy Yates</em>
<a class="jxr_linenumber" name="L76" href="#L76">76</a>  <em class="jxr_javadoccomment"> */</em>
<a class="jxr_linenumber" name="L77" href="#L77">77</a>  <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">class</strong> <a href="../../../../../../org/biojava/nbio/core/sequence/io/IUPACParser.html">IUPACParser</a> {
<a class="jxr_linenumber" name="L78" href="#L78">78</a>  
<a class="jxr_linenumber" name="L79" href="#L79">79</a>    <strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">class</strong> <a href="../../../../../../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html">IOD</a> {
<a class="jxr_linenumber" name="L80" href="#L80">80</a>      <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">final</strong> <a href="../../../../../../org/biojava/nbio/core/sequence/io/IUPACParser.html">IUPACParser</a> INSTANCE = <strong class="jxr_keyword">new</strong> <a href="../../../../../../org/biojava/nbio/core/sequence/io/IUPACParser.html">IUPACParser</a>();
<a class="jxr_linenumber" name="L81" href="#L81">81</a>    }
<a class="jxr_linenumber" name="L82" href="#L82">82</a>  
<a class="jxr_linenumber" name="L83" href="#L83">83</a>    <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">static</strong> <a href="../../../../../../org/biojava/nbio/core/sequence/io/IUPACParser.html">IUPACParser</a> getInstance() {
<a class="jxr_linenumber" name="L84" href="#L84">84</a>      <strong class="jxr_keyword">return</strong> IOD.INSTANCE;
<a class="jxr_linenumber" name="L85" href="#L85">85</a>    }
<a class="jxr_linenumber" name="L86" href="#L86">86</a>  
<a class="jxr_linenumber" name="L87" href="#L87">87</a>    <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">final</strong> String      IUPAC_LOCATION = <span class="jxr_string">"org/biojava/nbio/core/sequence/iupac.txt"</span>;
<a class="jxr_linenumber" name="L88" href="#L88">88</a>  
<a class="jxr_linenumber" name="L89" href="#L89">89</a>    <strong class="jxr_keyword">private</strong> InputStream              is;
<a class="jxr_linenumber" name="L90" href="#L90">90</a>    <strong class="jxr_keyword">private</strong> List&lt;IUPACTable&gt;         tables;
<a class="jxr_linenumber" name="L91" href="#L91">91</a>    <strong class="jxr_keyword">private</strong> Map&lt;String, IUPACTable&gt;  nameLookup;
<a class="jxr_linenumber" name="L92" href="#L92">92</a>    <strong class="jxr_keyword">private</strong> Map&lt;Integer, IUPACTable&gt; idLookup;
<a class="jxr_linenumber" name="L93" href="#L93">93</a>  
<a class="jxr_linenumber" name="L94" href="#L94">94</a>    <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L95" href="#L95">95</a>  <em class="jxr_javadoccomment">   * Default version and uses the classpath based IUPAC table</em>
<a class="jxr_linenumber" name="L96" href="#L96">96</a>  <em class="jxr_javadoccomment">   */</em>
<a class="jxr_linenumber" name="L97" href="#L97">97</a>    <strong class="jxr_keyword">public</strong> <a href="../../../../../../org/biojava/nbio/core/sequence/io/IUPACParser.html">IUPACParser</a>() {
<a class="jxr_linenumber" name="L98" href="#L98">98</a>      <em class="jxr_comment">//use the preCache version to make sure we don't keep a IO handle open</em>
<a class="jxr_linenumber" name="L99" href="#L99">99</a>      is = <strong class="jxr_keyword">new</strong> <a href="../../../../../../org/biojava/nbio/core/sequence/io/util/ClasspathResource.html">ClasspathResource</a>(IUPAC_LOCATION, <strong class="jxr_keyword">true</strong>).getInputStream();
<a class="jxr_linenumber" name="L100" href="#L100">100</a>   }
<a class="jxr_linenumber" name="L101" href="#L101">101</a> 
<a class="jxr_linenumber" name="L102" href="#L102">102</a>   <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L103" href="#L103">103</a> <em class="jxr_javadoccomment">   * Allows you to specify a different IUPAC table.</em>
<a class="jxr_linenumber" name="L104" href="#L104">104</a> <em class="jxr_javadoccomment">   */</em>
<a class="jxr_linenumber" name="L105" href="#L105">105</a>   <strong class="jxr_keyword">public</strong> <a href="../../../../../../org/biojava/nbio/core/sequence/io/IUPACParser.html">IUPACParser</a>(InputStream is) {
<a class="jxr_linenumber" name="L106" href="#L106">106</a>     <strong class="jxr_keyword">this</strong>.is = is;
<a class="jxr_linenumber" name="L107" href="#L107">107</a>   }
<a class="jxr_linenumber" name="L108" href="#L108">108</a> 
<a class="jxr_linenumber" name="L109" href="#L109">109</a>   <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L110" href="#L110">110</a> <em class="jxr_javadoccomment">   * Returns a list of all available IUPAC tables</em>
<a class="jxr_linenumber" name="L111" href="#L111">111</a> <em class="jxr_javadoccomment">   */</em>
<a class="jxr_linenumber" name="L112" href="#L112">112</a>   <strong class="jxr_keyword">public</strong> List&lt;IUPACTable&gt; getTables() {
<a class="jxr_linenumber" name="L113" href="#L113">113</a>     <strong class="jxr_keyword">if</strong> (tables == <strong class="jxr_keyword">null</strong>) {
<a class="jxr_linenumber" name="L114" href="#L114">114</a>       tables = parseTables();
<a class="jxr_linenumber" name="L115" href="#L115">115</a>     }
<a class="jxr_linenumber" name="L116" href="#L116">116</a>     <strong class="jxr_keyword">return</strong> tables;
<a class="jxr_linenumber" name="L117" href="#L117">117</a>   }
<a class="jxr_linenumber" name="L118" href="#L118">118</a> 
<a class="jxr_linenumber" name="L119" href="#L119">119</a>   <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L120" href="#L120">120</a> <em class="jxr_javadoccomment">   * Returns a table by its name</em>
<a class="jxr_linenumber" name="L121" href="#L121">121</a> <em class="jxr_javadoccomment">   */</em>
<a class="jxr_linenumber" name="L122" href="#L122">122</a>   <strong class="jxr_keyword">public</strong> <a href="../../../../../../org/biojava/nbio/core/sequence/io/IUPACParser.html">IUPACTable</a> getTable(String name) {
<a class="jxr_linenumber" name="L123" href="#L123">123</a>     populateLookups();
<a class="jxr_linenumber" name="L124" href="#L124">124</a>     <strong class="jxr_keyword">return</strong> nameLookup.get(name);
<a class="jxr_linenumber" name="L125" href="#L125">125</a>   }
<a class="jxr_linenumber" name="L126" href="#L126">126</a> 
<a class="jxr_linenumber" name="L127" href="#L127">127</a>   <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L128" href="#L128">128</a> <em class="jxr_javadoccomment">   * Returns a table by its identifier i.e. 1 means universal codon tables</em>
<a class="jxr_linenumber" name="L129" href="#L129">129</a> <em class="jxr_javadoccomment">   */</em>
<a class="jxr_linenumber" name="L130" href="#L130">130</a>   <strong class="jxr_keyword">public</strong> <a href="../../../../../../org/biojava/nbio/core/sequence/io/IUPACParser.html">IUPACTable</a> getTable(Integer id) {
<a class="jxr_linenumber" name="L131" href="#L131">131</a>     populateLookups();
<a class="jxr_linenumber" name="L132" href="#L132">132</a>     <strong class="jxr_keyword">return</strong> idLookup.get(id);
<a class="jxr_linenumber" name="L133" href="#L133">133</a>   }
<a class="jxr_linenumber" name="L134" href="#L134">134</a> 
<a class="jxr_linenumber" name="L135" href="#L135">135</a>   <strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">void</strong> populateLookups() {
<a class="jxr_linenumber" name="L136" href="#L136">136</a>     <strong class="jxr_keyword">if</strong>(nameLookup == <strong class="jxr_keyword">null</strong>) {
<a class="jxr_linenumber" name="L137" href="#L137">137</a>       nameLookup = <strong class="jxr_keyword">new</strong> HashMap&lt;String, IUPACTable&gt;();
<a class="jxr_linenumber" name="L138" href="#L138">138</a>       idLookup = <strong class="jxr_keyword">new</strong> HashMap&lt;Integer, IUPACTable&gt;();
<a class="jxr_linenumber" name="L139" href="#L139">139</a>       <strong class="jxr_keyword">for</strong>(<a href="../../../../../../org/biojava/nbio/core/sequence/io/IUPACParser.html">IUPACTable</a> t: getTables()) {
<a class="jxr_linenumber" name="L140" href="#L140">140</a>         nameLookup.put(t.getName(), t);
<a class="jxr_linenumber" name="L141" href="#L141">141</a>         idLookup.put(t.getId(), t);
<a class="jxr_linenumber" name="L142" href="#L142">142</a>       }
<a class="jxr_linenumber" name="L143" href="#L143">143</a>     }
<a class="jxr_linenumber" name="L144" href="#L144">144</a>   }
<a class="jxr_linenumber" name="L145" href="#L145">145</a> 
<a class="jxr_linenumber" name="L146" href="#L146">146</a>   <strong class="jxr_keyword">private</strong> List&lt;IUPACTable&gt; parseTables() {
<a class="jxr_linenumber" name="L147" href="#L147">147</a>     List&lt;IUPACTable&gt; localTables = <strong class="jxr_keyword">new</strong> ArrayList&lt;IUPACTable&gt;();
<a class="jxr_linenumber" name="L148" href="#L148">148</a>     List&lt;String&gt; lines = IOUtils.getList(is);
<a class="jxr_linenumber" name="L149" href="#L149">149</a>     Integer id = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L150" href="#L150">150</a>     String name, aa, starts, baseone, basetwo, basethree;
<a class="jxr_linenumber" name="L151" href="#L151">151</a>     name = aa = starts = baseone = basetwo = basethree = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L152" href="#L152">152</a>     <strong class="jxr_keyword">for</strong> (String line : lines) {
<a class="jxr_linenumber" name="L153" href="#L153">153</a>       <strong class="jxr_keyword">if</strong> (line.equalsIgnoreCase(<span class="jxr_string">"//"</span>)) {
<a class="jxr_linenumber" name="L154" href="#L154">154</a>         localTables.add(<strong class="jxr_keyword">new</strong> <a href="../../../../../../org/biojava/nbio/core/sequence/io/IUPACParser.html">IUPACTable</a>(name, id, aa, starts, baseone, basetwo,
<a class="jxr_linenumber" name="L155" href="#L155">155</a>             basethree));
<a class="jxr_linenumber" name="L156" href="#L156">156</a>         name = aa = starts = baseone = basetwo = basethree = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L157" href="#L157">157</a>         id = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L158" href="#L158">158</a>       }
<a class="jxr_linenumber" name="L159" href="#L159">159</a>       <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L160" href="#L160">160</a>         String[] keyValue = line.split(<span class="jxr_string">"&#92;&#92;s*=&#92;&#92;s*"</span>);
<a class="jxr_linenumber" name="L161" href="#L161">161</a>         <strong class="jxr_keyword">if</strong> (keyValue[0].equals(<span class="jxr_string">"AAs"</span>)) {
<a class="jxr_linenumber" name="L162" href="#L162">162</a>           aa = keyValue[1];
<a class="jxr_linenumber" name="L163" href="#L163">163</a>         }
<a class="jxr_linenumber" name="L164" href="#L164">164</a>         <strong class="jxr_keyword">else</strong> <strong class="jxr_keyword">if</strong> (keyValue[0].equals(<span class="jxr_string">"Starts"</span>)) {
<a class="jxr_linenumber" name="L165" href="#L165">165</a>           starts = keyValue[1];
<a class="jxr_linenumber" name="L166" href="#L166">166</a>         }
<a class="jxr_linenumber" name="L167" href="#L167">167</a>         <strong class="jxr_keyword">else</strong> <strong class="jxr_keyword">if</strong> (keyValue[0].equals(<span class="jxr_string">"Base1"</span>)) {
<a class="jxr_linenumber" name="L168" href="#L168">168</a>           baseone = keyValue[1];
<a class="jxr_linenumber" name="L169" href="#L169">169</a>         }
<a class="jxr_linenumber" name="L170" href="#L170">170</a>         <strong class="jxr_keyword">else</strong> <strong class="jxr_keyword">if</strong> (keyValue[0].equals(<span class="jxr_string">"Base2"</span>)) {
<a class="jxr_linenumber" name="L171" href="#L171">171</a>           basetwo = keyValue[1];
<a class="jxr_linenumber" name="L172" href="#L172">172</a>         }
<a class="jxr_linenumber" name="L173" href="#L173">173</a>         <strong class="jxr_keyword">else</strong> <strong class="jxr_keyword">if</strong> (keyValue[0].equals(<span class="jxr_string">"Base3"</span>)) {
<a class="jxr_linenumber" name="L174" href="#L174">174</a>           basethree = keyValue[1];
<a class="jxr_linenumber" name="L175" href="#L175">175</a>         }
<a class="jxr_linenumber" name="L176" href="#L176">176</a>         <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L177" href="#L177">177</a>           name = keyValue[0];
<a class="jxr_linenumber" name="L178" href="#L178">178</a>           id = Integer.parseInt(keyValue[1]);
<a class="jxr_linenumber" name="L179" href="#L179">179</a>         }
<a class="jxr_linenumber" name="L180" href="#L180">180</a>       }
<a class="jxr_linenumber" name="L181" href="#L181">181</a>     }
<a class="jxr_linenumber" name="L182" href="#L182">182</a> 
<a class="jxr_linenumber" name="L183" href="#L183">183</a>     <strong class="jxr_keyword">return</strong> localTables;
<a class="jxr_linenumber" name="L184" href="#L184">184</a>   }
<a class="jxr_linenumber" name="L185" href="#L185">185</a> 
<a class="jxr_linenumber" name="L186" href="#L186">186</a>   <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L187" href="#L187">187</a> <em class="jxr_javadoccomment">   * Holds the concept of a codon table from the IUPAC format</em>
<a class="jxr_linenumber" name="L188" href="#L188">188</a> <em class="jxr_javadoccomment">   *</em>
<a class="jxr_linenumber" name="L189" href="#L189">189</a> <em class="jxr_javadoccomment">   * @author Andy Yates</em>
<a class="jxr_linenumber" name="L190" href="#L190">190</a> <em class="jxr_javadoccomment">   */</em>
<a class="jxr_linenumber" name="L191" href="#L191">191</a>   <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">class</strong> <a href="../../../../../../org/biojava/nbio/core/sequence/io/IUPACParser.html">IUPACTable</a> <strong class="jxr_keyword">implements</strong> <a href="../../../../../../org/biojava/nbio/core/sequence/transcription/Table.html">Table</a> {
<a class="jxr_linenumber" name="L192" href="#L192">192</a> 
<a class="jxr_linenumber" name="L193" href="#L193">193</a>     <strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">final</strong> Integer      id;
<a class="jxr_linenumber" name="L194" href="#L194">194</a>     <strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">final</strong> String       name;
<a class="jxr_linenumber" name="L195" href="#L195">195</a>     <strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">final</strong> String       aminoAcidString;
<a class="jxr_linenumber" name="L196" href="#L196">196</a>     <strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">final</strong> String       startCodons;
<a class="jxr_linenumber" name="L197" href="#L197">197</a>     <strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">final</strong> String       baseOne;
<a class="jxr_linenumber" name="L198" href="#L198">198</a>     <strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">final</strong> String       baseTwo;
<a class="jxr_linenumber" name="L199" href="#L199">199</a>     <strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">final</strong> String       baseThree;
<a class="jxr_linenumber" name="L200" href="#L200">200</a> 
<a class="jxr_linenumber" name="L201" href="#L201">201</a>     <strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">final</strong> List&lt;Codon&gt;  codons    = <strong class="jxr_keyword">new</strong> ArrayList&lt;Codon&gt;();
<a class="jxr_linenumber" name="L202" href="#L202">202</a>     <strong class="jxr_keyword">private</strong> CompoundSet&lt;Codon&gt; compounds = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L203" href="#L203">203</a> 
<a class="jxr_linenumber" name="L204" href="#L204">204</a>     <strong class="jxr_keyword">public</strong> <a href="../../../../../../org/biojava/nbio/core/sequence/io/IUPACParser.html">IUPACTable</a>(String name, <strong class="jxr_keyword">int</strong> id, String aminoAcidString,
<a class="jxr_linenumber" name="L205" href="#L205">205</a>         String startCodons, String baseOne, String baseTwo, String baseThree) {
<a class="jxr_linenumber" name="L206" href="#L206">206</a>       <strong class="jxr_keyword">this</strong>.aminoAcidString = aminoAcidString;
<a class="jxr_linenumber" name="L207" href="#L207">207</a>       <strong class="jxr_keyword">this</strong>.startCodons = startCodons;
<a class="jxr_linenumber" name="L208" href="#L208">208</a>       <strong class="jxr_keyword">this</strong>.name = name;
<a class="jxr_linenumber" name="L209" href="#L209">209</a>       <strong class="jxr_keyword">this</strong>.id = id;
<a class="jxr_linenumber" name="L210" href="#L210">210</a>       <strong class="jxr_keyword">this</strong>.baseOne = baseOne;
<a class="jxr_linenumber" name="L211" href="#L211">211</a>       <strong class="jxr_keyword">this</strong>.baseTwo = baseTwo;
<a class="jxr_linenumber" name="L212" href="#L212">212</a>       <strong class="jxr_keyword">this</strong>.baseThree = baseThree;
<a class="jxr_linenumber" name="L213" href="#L213">213</a>     }
<a class="jxr_linenumber" name="L214" href="#L214">214</a> 
<a class="jxr_linenumber" name="L215" href="#L215">215</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L216" href="#L216">216</a> <em class="jxr_javadoccomment">     * Constructor which uses the basic IUPAC codon table format. Useful</em>
<a class="jxr_linenumber" name="L217" href="#L217">217</a> <em class="jxr_javadoccomment">     * if you need to specify your own IUPAC table with minimal</em>
<a class="jxr_linenumber" name="L218" href="#L218">218</a> <em class="jxr_javadoccomment">     * definitions from your side.</em>
<a class="jxr_linenumber" name="L219" href="#L219">219</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L220" href="#L220">220</a>     <strong class="jxr_keyword">public</strong> <a href="../../../../../../org/biojava/nbio/core/sequence/io/IUPACParser.html">IUPACTable</a>(String name, Integer id, String aminoAcidString,
<a class="jxr_linenumber" name="L221" href="#L221">221</a>         String startCodons) {
<a class="jxr_linenumber" name="L222" href="#L222">222</a>       <strong class="jxr_keyword">this</strong>(name, id, aminoAcidString, startCodons,
<a class="jxr_linenumber" name="L223" href="#L223">223</a>           <span class="jxr_string">"TTTTTTTTTTTTTTTTCCCCCCCCCCCCCCCCAAAAAAAAAAAAAAAAGGGGGGGGGGGGGGGG"</span>,
<a class="jxr_linenumber" name="L224" href="#L224">224</a>           <span class="jxr_string">"TTTTCCCCAAAAGGGGTTTTCCCCAAAAGGGGTTTTCCCCAAAAGGGGTTTTCCCCAAAAGGGG"</span>,
<a class="jxr_linenumber" name="L225" href="#L225">225</a>           <span class="jxr_string">"TCAGTCAGTCAGTCAGTCAGTCAGTCAGTCAGTCAGTCAGTCAGTCAGTCAGTCAGTCAGTCAG"</span>);
<a class="jxr_linenumber" name="L226" href="#L226">226</a>     }
<a class="jxr_linenumber" name="L227" href="#L227">227</a> 
<a class="jxr_linenumber" name="L228" href="#L228">228</a>     <strong class="jxr_keyword">public</strong> Integer getId() {
<a class="jxr_linenumber" name="L229" href="#L229">229</a>       <strong class="jxr_keyword">return</strong> id;
<a class="jxr_linenumber" name="L230" href="#L230">230</a>     }
<a class="jxr_linenumber" name="L231" href="#L231">231</a> 
<a class="jxr_linenumber" name="L232" href="#L232">232</a>     <strong class="jxr_keyword">public</strong> String getName() {
<a class="jxr_linenumber" name="L233" href="#L233">233</a>       <strong class="jxr_keyword">return</strong> name;
<a class="jxr_linenumber" name="L234" href="#L234">234</a>     }
<a class="jxr_linenumber" name="L235" href="#L235">235</a> 
<a class="jxr_linenumber" name="L236" href="#L236">236</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L237" href="#L237">237</a> <em class="jxr_javadoccomment">     * Returns true if the given compound was a start codon in this</em>
<a class="jxr_linenumber" name="L238" href="#L238">238</a> <em class="jxr_javadoccomment">     * codon table. This will report true if the compound could ever have</em>
<a class="jxr_linenumber" name="L239" href="#L239">239</a> <em class="jxr_javadoccomment">     * been a start codon.</em>
<a class="jxr_linenumber" name="L240" href="#L240">240</a> <em class="jxr_javadoccomment">     *</em>
<a class="jxr_linenumber" name="L241" href="#L241">241</a> <em class="jxr_javadoccomment">     * @throws IllegalStateException Thrown if</em>
<a class="jxr_linenumber" name="L242" href="#L242">242</a> <em class="jxr_javadoccomment">     * {@link #getCodons(CompoundSet, CompoundSet)} was not called first.</em>
<a class="jxr_linenumber" name="L243" href="#L243">243</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L244" href="#L244">244</a>         @Override
<a class="jxr_linenumber" name="L245" href="#L245">245</a>     <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">boolean</strong> isStart(<a href="../../../../../../org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html">AminoAcidCompound</a> compound) <strong class="jxr_keyword">throws</strong> IllegalStateException {
<a class="jxr_linenumber" name="L246" href="#L246">246</a>       <strong class="jxr_keyword">if</strong>(<strong class="jxr_keyword">this</strong>.codons.isEmpty()) {
<a class="jxr_linenumber" name="L247" href="#L247">247</a>         <strong class="jxr_keyword">throw</strong> <strong class="jxr_keyword">new</strong> IllegalStateException(<span class="jxr_string">"Codons are empty; please request getCodons() fist before asking this"</span>);
<a class="jxr_linenumber" name="L248" href="#L248">248</a>       }
<a class="jxr_linenumber" name="L249" href="#L249">249</a>       <strong class="jxr_keyword">for</strong>(<a href="../../../../../../org/biojava/nbio/core/sequence/transcription/Table.html">Codon</a> codon: codons) {
<a class="jxr_linenumber" name="L250" href="#L250">250</a>         <em class="jxr_comment">//Only check if the codon was a start codon and then ask if the compound was encoded by it</em>
<a class="jxr_linenumber" name="L251" href="#L251">251</a>         <strong class="jxr_keyword">if</strong>(codon.isStart()) {
<a class="jxr_linenumber" name="L252" href="#L252">252</a>           <strong class="jxr_keyword">if</strong>(codon.getAminoAcid().equalsIgnoreCase(compound)) {
<a class="jxr_linenumber" name="L253" href="#L253">253</a>             <strong class="jxr_keyword">return</strong> <strong class="jxr_keyword">true</strong>;
<a class="jxr_linenumber" name="L254" href="#L254">254</a>           }
<a class="jxr_linenumber" name="L255" href="#L255">255</a>         }
<a class="jxr_linenumber" name="L256" href="#L256">256</a>       }
<a class="jxr_linenumber" name="L257" href="#L257">257</a>       <strong class="jxr_keyword">return</strong> false;
<a class="jxr_linenumber" name="L258" href="#L258">258</a>     }
<a class="jxr_linenumber" name="L259" href="#L259">259</a> 
<a class="jxr_linenumber" name="L260" href="#L260">260</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L261" href="#L261">261</a> <em class="jxr_javadoccomment">     * Returns a list of codons where the source and target compounds</em>
<a class="jxr_linenumber" name="L262" href="#L262">262</a> <em class="jxr_javadoccomment">     * are the same as those given by the parameters.</em>
<a class="jxr_linenumber" name="L263" href="#L263">263</a> <em class="jxr_javadoccomment">     *</em>
<a class="jxr_linenumber" name="L264" href="#L264">264</a> <em class="jxr_javadoccomment">     * @param nucleotides The nucleotide set to use when building BioJava </em>
<a class="jxr_linenumber" name="L265" href="#L265">265</a> <em class="jxr_javadoccomment">     * representations of codons</em>
<a class="jxr_linenumber" name="L266" href="#L266">266</a> <em class="jxr_javadoccomment">     * @param aminoAcids The target amino acid compounds objects</em>
<a class="jxr_linenumber" name="L267" href="#L267">267</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L268" href="#L268">268</a>         @Override
<a class="jxr_linenumber" name="L269" href="#L269">269</a>     <strong class="jxr_keyword">public</strong> List&lt;Codon&gt; getCodons(CompoundSet&lt;NucleotideCompound&gt; nucelotides,
<a class="jxr_linenumber" name="L270" href="#L270">270</a>         CompoundSet&lt;AminoAcidCompound&gt; aminoAcids) {
<a class="jxr_linenumber" name="L271" href="#L271">271</a> 
<a class="jxr_linenumber" name="L272" href="#L272">272</a>       <strong class="jxr_keyword">if</strong> (<strong class="jxr_keyword">this</strong>.codons.isEmpty()) {
<a class="jxr_linenumber" name="L273" href="#L273">273</a>         List&lt;String&gt; aminoAcidStrings = aminoAcids();
<a class="jxr_linenumber" name="L274" href="#L274">274</a>         List&lt;String&gt; startCodonStrings = startCodons();
<a class="jxr_linenumber" name="L275" href="#L275">275</a>         List&lt;List&lt;String&gt;&gt; codonStrings = codonStrings();
<a class="jxr_linenumber" name="L276" href="#L276">276</a> 
<a class="jxr_linenumber" name="L277" href="#L277">277</a>         <strong class="jxr_keyword">for</strong> (<strong class="jxr_keyword">int</strong> i = 0; i &lt; aminoAcidStrings.size(); i++) {
<a class="jxr_linenumber" name="L278" href="#L278">278</a> 
<a class="jxr_linenumber" name="L279" href="#L279">279</a>           List&lt;String&gt; codonString    = codonStrings.get(i);
<a class="jxr_linenumber" name="L280" href="#L280">280</a>           <a href="../../../../../../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html">NucleotideCompound</a> one      = getCompound(codonString, 0, nucelotides);
<a class="jxr_linenumber" name="L281" href="#L281">281</a>           <a href="../../../../../../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html">NucleotideCompound</a> two      = getCompound(codonString, 1, nucelotides);
<a class="jxr_linenumber" name="L282" href="#L282">282</a>           <a href="../../../../../../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html">NucleotideCompound</a> three    = getCompound(codonString, 2, nucelotides);
<a class="jxr_linenumber" name="L283" href="#L283">283</a>           <strong class="jxr_keyword">boolean</strong> start               = (<span class="jxr_string">"M"</span>.equals(startCodonStrings.get(i)));
<a class="jxr_linenumber" name="L284" href="#L284">284</a>           <strong class="jxr_keyword">boolean</strong> stop                = (<span class="jxr_string">"*"</span>.equals(aminoAcidStrings.get(i)));
<a class="jxr_linenumber" name="L285" href="#L285">285</a>           <a href="../../../../../../org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html">AminoAcidCompound</a> aminoAcid = aminoAcids
<a class="jxr_linenumber" name="L286" href="#L286">286</a>               .getCompoundForString(aminoAcidStrings.get(i));
<a class="jxr_linenumber" name="L287" href="#L287">287</a>           codons.add(<strong class="jxr_keyword">new</strong> <a href="../../../../../../org/biojava/nbio/core/sequence/transcription/Table.html">Codon</a>(<strong class="jxr_keyword">new</strong> <a href="../../../../../../org/biojava/nbio/core/sequence/transcription/Table.html">CaseInsensitiveTriplet</a>(one, two, three), aminoAcid, start, stop));
<a class="jxr_linenumber" name="L288" href="#L288">288</a>         }
<a class="jxr_linenumber" name="L289" href="#L289">289</a>       }
<a class="jxr_linenumber" name="L290" href="#L290">290</a> 
<a class="jxr_linenumber" name="L291" href="#L291">291</a>       <strong class="jxr_keyword">return</strong> codons;
<a class="jxr_linenumber" name="L292" href="#L292">292</a>     }
<a class="jxr_linenumber" name="L293" href="#L293">293</a> 
<a class="jxr_linenumber" name="L294" href="#L294">294</a>     <strong class="jxr_keyword">private</strong> <a href="../../../../../../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html">NucleotideCompound</a> getCompound(List&lt;String&gt; compounds,
<a class="jxr_linenumber" name="L295" href="#L295">295</a>         <strong class="jxr_keyword">int</strong> position, CompoundSet&lt;NucleotideCompound&gt; nucelotides) {
<a class="jxr_linenumber" name="L296" href="#L296">296</a>       String compound = compounds.get(position);
<a class="jxr_linenumber" name="L297" href="#L297">297</a>       <a href="../../../../../../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html">NucleotideCompound</a> returnCompound = nucelotides
<a class="jxr_linenumber" name="L298" href="#L298">298</a>           .getCompoundForString(compound);
<a class="jxr_linenumber" name="L299" href="#L299">299</a>       <strong class="jxr_keyword">if</strong> (returnCompound == <strong class="jxr_keyword">null</strong>) {
<a class="jxr_linenumber" name="L300" href="#L300">300</a>         <strong class="jxr_keyword">if</strong> (<span class="jxr_string">"T"</span>.equalsIgnoreCase(compound)) {
<a class="jxr_linenumber" name="L301" href="#L301">301</a>             returnCompound = nucelotides.getCompoundForString(<span class="jxr_string">"U"</span>);
<a class="jxr_linenumber" name="L302" href="#L302">302</a>         }
<a class="jxr_linenumber" name="L303" href="#L303">303</a>         <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L304" href="#L304">304</a>           <strong class="jxr_keyword">throw</strong> <strong class="jxr_keyword">new</strong> <a href="../../../../../../org/biojava/nbio/core/exceptions/ParserException.html">ParserException</a>(<span class="jxr_string">"Cannot find a compound for string "</span>
<a class="jxr_linenumber" name="L305" href="#L305">305</a>               + compound);
<a class="jxr_linenumber" name="L306" href="#L306">306</a>         }
<a class="jxr_linenumber" name="L307" href="#L307">307</a>       }
<a class="jxr_linenumber" name="L308" href="#L308">308</a>       <strong class="jxr_keyword">return</strong> returnCompound;
<a class="jxr_linenumber" name="L309" href="#L309">309</a>     }
<a class="jxr_linenumber" name="L310" href="#L310">310</a> 
<a class="jxr_linenumber" name="L311" href="#L311">311</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L312" href="#L312">312</a> <em class="jxr_javadoccomment">     * Returns the compound set of codons</em>
<a class="jxr_linenumber" name="L313" href="#L313">313</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L314" href="#L314">314</a>     @Override
<a class="jxr_linenumber" name="L315" href="#L315">315</a> 	<strong class="jxr_keyword">public</strong> CompoundSet&lt;Codon&gt; getCodonCompoundSet(
<a class="jxr_linenumber" name="L316" href="#L316">316</a>         <strong class="jxr_keyword">final</strong> CompoundSet&lt;NucleotideCompound&gt; rnaCompounds,
<a class="jxr_linenumber" name="L317" href="#L317">317</a>         <strong class="jxr_keyword">final</strong> CompoundSet&lt;AminoAcidCompound&gt; aminoAcidCompounds) {
<a class="jxr_linenumber" name="L318" href="#L318">318</a>       <strong class="jxr_keyword">if</strong> (compounds == <strong class="jxr_keyword">null</strong>) {
<a class="jxr_linenumber" name="L319" href="#L319">319</a>         compounds = <strong class="jxr_keyword">new</strong> AbstractCompoundSet&lt;Codon&gt;() {
<a class="jxr_linenumber" name="L320" href="#L320">320</a>           {
<a class="jxr_linenumber" name="L321" href="#L321">321</a>             <strong class="jxr_keyword">for</strong> (Codon c : getCodons(rnaCompounds, aminoAcidCompounds)) {
<a class="jxr_linenumber" name="L322" href="#L322">322</a>               addCompound(c);
<a class="jxr_linenumber" name="L323" href="#L323">323</a>             }
<a class="jxr_linenumber" name="L324" href="#L324">324</a>           }
<a class="jxr_linenumber" name="L325" href="#L325">325</a>         };
<a class="jxr_linenumber" name="L326" href="#L326">326</a>       }
<a class="jxr_linenumber" name="L327" href="#L327">327</a>       <strong class="jxr_keyword">return</strong> compounds;
<a class="jxr_linenumber" name="L328" href="#L328">328</a>     }
<a class="jxr_linenumber" name="L329" href="#L329">329</a> 
<a class="jxr_linenumber" name="L330" href="#L330">330</a>     <strong class="jxr_keyword">private</strong> List&lt;List&lt;String&gt;&gt; codonStrings() {
<a class="jxr_linenumber" name="L331" href="#L331">331</a>       List&lt;List&lt;String&gt;&gt; codons = <strong class="jxr_keyword">new</strong> ArrayList&lt;List&lt;String&gt;&gt;();
<a class="jxr_linenumber" name="L332" href="#L332">332</a>       <strong class="jxr_keyword">for</strong> (<strong class="jxr_keyword">int</strong> i = 0; i &lt; baseOne.length(); i++) {
<a class="jxr_linenumber" name="L333" href="#L333">333</a>         List&lt;String&gt; codon = Arrays.asList(Character
<a class="jxr_linenumber" name="L334" href="#L334">334</a>             .toString(baseOne.charAt(i)),
<a class="jxr_linenumber" name="L335" href="#L335">335</a>             Character.toString(baseTwo.charAt(i)), Character.toString(baseThree
<a class="jxr_linenumber" name="L336" href="#L336">336</a>                 .charAt(i)));
<a class="jxr_linenumber" name="L337" href="#L337">337</a>         codons.add(codon);
<a class="jxr_linenumber" name="L338" href="#L338">338</a>       }
<a class="jxr_linenumber" name="L339" href="#L339">339</a>       <strong class="jxr_keyword">return</strong> codons;
<a class="jxr_linenumber" name="L340" href="#L340">340</a>     }
<a class="jxr_linenumber" name="L341" href="#L341">341</a> 
<a class="jxr_linenumber" name="L342" href="#L342">342</a>     <strong class="jxr_keyword">private</strong> List&lt;String&gt; aminoAcids() {
<a class="jxr_linenumber" name="L343" href="#L343">343</a>       <strong class="jxr_keyword">return</strong> split(aminoAcidString);
<a class="jxr_linenumber" name="L344" href="#L344">344</a>     }
<a class="jxr_linenumber" name="L345" href="#L345">345</a> 
<a class="jxr_linenumber" name="L346" href="#L346">346</a>     <strong class="jxr_keyword">private</strong> List&lt;String&gt; startCodons() {
<a class="jxr_linenumber" name="L347" href="#L347">347</a>       <strong class="jxr_keyword">return</strong> split(startCodons);
<a class="jxr_linenumber" name="L348" href="#L348">348</a>     }
<a class="jxr_linenumber" name="L349" href="#L349">349</a> 
<a class="jxr_linenumber" name="L350" href="#L350">350</a>     <strong class="jxr_keyword">private</strong> List&lt;String&gt; split(String string) {
<a class="jxr_linenumber" name="L351" href="#L351">351</a>       List&lt;String&gt; split = <strong class="jxr_keyword">new</strong> ArrayList&lt;String&gt;();
<a class="jxr_linenumber" name="L352" href="#L352">352</a>       <strong class="jxr_keyword">for</strong> (<strong class="jxr_keyword">int</strong> i = 0; i &lt; string.length(); i++) {
<a class="jxr_linenumber" name="L353" href="#L353">353</a>         split.add(Character.toString(string.charAt(i)));
<a class="jxr_linenumber" name="L354" href="#L354">354</a>       }
<a class="jxr_linenumber" name="L355" href="#L355">355</a>       <strong class="jxr_keyword">return</strong> split;
<a class="jxr_linenumber" name="L356" href="#L356">356</a>     }
<a class="jxr_linenumber" name="L357" href="#L357">357</a>   }
<a class="jxr_linenumber" name="L358" href="#L358">358</a> }
</pre>
<hr/>
<div id="footer">Copyright &#169; 2000&#x2013;2015 <a href="http://www.biojava.org">BioJava</a>. All rights reserved.</div>
</body>
</html>
