<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_71) on Fri Jan 30 08:16:46 PST 2015 -->
<meta http-equiv="Content-Type" content="text/html" charset="UTF-8">
<title>Index (biojava-core 4.0.0 API)</title>
<meta name="date" content="2015-01-30">
<link rel="stylesheet" type="text/css" href="./stylesheet.css" title="Style">
</head>
<body>
<script type="text/javascript"><!--
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="Index (biojava-core 4.0.0 API)";
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar_top">
<!--   -->
</a><a href="#skip-navbar_top" title="Skip navigation links"></a><a name="navbar_top_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="./overview-summary.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li>Use</li>
<li><a href="./overview-tree.html">Tree</a></li>
<li><a href="./deprecated-list.html">Deprecated</a></li>
<li class="navBarCell1Rev">Index</li>
<li><a href="./help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="./index.html?index-all.html" target="_top">Frames</a></li>
<li><a href="index-all.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="./allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<div class="contentContainer"><a href="#_A_">A</a>&nbsp;<a href="#_B_">B</a>&nbsp;<a href="#_C_">C</a>&nbsp;<a href="#_D_">D</a>&nbsp;<a href="#_E_">E</a>&nbsp;<a href="#_F_">F</a>&nbsp;<a href="#_G_">G</a>&nbsp;<a href="#_H_">H</a>&nbsp;<a href="#_I_">I</a>&nbsp;<a href="#_J_">J</a>&nbsp;<a href="#_K_">K</a>&nbsp;<a href="#_L_">L</a>&nbsp;<a href="#_M_">M</a>&nbsp;<a href="#_N_">N</a>&nbsp;<a href="#_O_">O</a>&nbsp;<a href="#_P_">P</a>&nbsp;<a href="#_Q_">Q</a>&nbsp;<a href="#_R_">R</a>&nbsp;<a href="#_S_">S</a>&nbsp;<a href="#_T_">T</a>&nbsp;<a href="#_U_">U</a>&nbsp;<a href="#_V_">V</a>&nbsp;<a href="#_W_">W</a>&nbsp;<a href="#_X_">X</a>&nbsp;<a href="#___">_</a>&nbsp;<a name="_A_">
<!--   -->
</a>
<h2 class="title">A</h2>
<dl>
<dt><a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template"><span class="strong">AbstractCompound</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>
<div class="block">The details of a Compound</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html#AbstractCompound(java.lang.String)">AbstractCompound(String)</a></span> - Constructor for class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template"><span class="strong">AbstractCompoundSet</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="type parameter in AbstractCompoundSet">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#AbstractCompoundSet()">AbstractCompoundSet()</a></span> - Constructor for class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template"><span class="strong">AbstractCompoundTranslator</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="type parameter in AbstractCompoundTranslator">F</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>,<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="type parameter in AbstractCompoundTranslator">T</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#AbstractCompoundTranslator(org.biojava.nbio.core.sequence.io.template.SequenceCreatorInterface,%20org.biojava.nbio.core.sequence.template.CompoundSet,%20org.biojava.nbio.core.sequence.template.CompoundSet)">AbstractCompoundTranslator(SequenceCreatorInterface&lt;T&gt;, CompoundSet&lt;F&gt;, CompoundSet&lt;T&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features"><span class="strong">AbstractFeature</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="type parameter in AbstractFeature">S</a> extends <a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="type parameter in AbstractFeature">C</a>&gt;,<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="type parameter in AbstractFeature">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/features/package-summary.html">org.biojava.nbio.core.sequence.features</a></dt>
<dd>
<div class="block">A feature is currently any descriptive item that can be associated with a sequence position(s)
 A feature has a type and a source which is currently a string to allow flexibility for the user
 Ideally well defined features should have a class to describe attributes of that feature</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html#AbstractFeature(java.lang.String,%20java.lang.String)">AbstractFeature(String, String)</a></span> - Constructor for class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">A feature has a type and a source</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template"><span class="strong">AbstractLocation</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/location/template/package-summary.html">org.biojava.nbio.core.sequence.location.template</a></dt>
<dd>
<div class="block">Base abstraction of a location which encodes for the majority of important
 features about a location such as the start, end and strand</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#AbstractLocation()">AbstractLocation()</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#AbstractLocation(org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.Strand,%20boolean,%20boolean,%20java.util.List)">AbstractLocation(Point, Point, Strand, boolean, boolean, List&lt;Location&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>
<div class="block">Default constructor</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#AbstractLocation(org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.Strand,%20boolean,%20boolean,%20org.biojava.nbio.core.sequence.AccessionID,%20java.util.List)">AbstractLocation(Point, Point, Strand, boolean, boolean, AccessionID, List&lt;Location&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>
<div class="block">Default constructor</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template"><span class="strong">AbstractNucleotideCompoundSet</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html" title="type parameter in AbstractNucleotideCompoundSet">C</a> extends <a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#AbstractNucleotideCompoundSet()">AbstractNucleotideCompoundSet()</a></span> - Constructor for class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractNucleotideCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template"><span class="strong">AbstractSequence</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="type parameter in AbstractSequence">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>
<div class="block">The base class for DNA, RNA and Protein sequences.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#AbstractSequence()">AbstractSequence()</a></span> - Constructor for class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#AbstractSequence(java.lang.String,%20org.biojava.nbio.core.sequence.template.CompoundSet)">AbstractSequence(String, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Create a Sequence from a simple string where the values should be found in compoundSet</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#AbstractSequence(org.biojava.nbio.core.sequence.template.SequenceReader,%20org.biojava.nbio.core.sequence.template.CompoundSet)">AbstractSequence(SequenceReader&lt;C&gt;, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">A ProxySequenceReader allows abstraction of both the storage of the sequence data and the location
 of the sequence data.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.AnnotationType.html" title="enum in org.biojava.nbio.core.sequence.template"><span class="strong">AbstractSequence.AnnotationType</span></a> - Enum in <a href="./org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/location/template/AccesionedLocation.html" title="interface in org.biojava.nbio.core.sequence.location.template"><span class="strong">AccesionedLocation</span></a> - Interface in <a href="./org/biojava/nbio/core/sequence/location/template/package-summary.html">org.biojava.nbio.core.sequence.location.template</a></dt>
<dd>
<div class="block">A location which is bound to an AccessionID.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#ACCESSION_TAG">ACCESSION_TAG</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/template/Accessioned.html" title="interface in org.biojava.nbio.core.sequence.template"><span class="strong">Accessioned</span></a> - Interface in <a href="./org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>
<div class="block">Indicates an entity is accessioned</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/AccessionID.html" title="class in org.biojava.nbio.core.sequence"><span class="strong">AccessionID</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">Used in Sequences as the unique indentifier.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/AccessionID.html#AccessionID()">AccessionID()</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/AccessionID.html#AccessionID(java.lang.String)">AccessionID(String)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/AccessionID.html#AccessionID(java.lang.String,%20org.biojava.nbio.core.sequence.DataSource)">AccessionID(String, DataSource)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#addAlignedSequence(S)">addAlignedSequence(S)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava.nbio.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">A sequence that has been aligned to other sequences will have inserts.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html#addCDS(org.biojava.nbio.core.sequence.AccessionID,%20int,%20int,%20int)">addCDS(AccessionID, int, int, int)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>
<div class="block">Add a Coding Sequence region with phase to the transcript sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/XMLHelper.html#addChildElement(org.w3c.dom.Element,%20java.lang.String)">addChildElement(Element, String)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/XMLHelper.html" title="class in org.biojava.nbio.core.util">XMLHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#addCompound(C,%20C,%20java.lang.Iterable)">addCompound(C, C, Iterable&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#addCompound(C,%20C,%20C...)">addCompound(C, C, C...)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#addCompound(C)">addCompound(C)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#addCompounds(F,%20T...)">addCompounds(F, T...)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#addCompoundsToList(java.util.List,%20java.util.List)">addCompoundsToList(List&lt;T&gt;, List&lt;List&lt;T&gt;&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#addCompoundToLists(java.util.List,%20T)">addCompoundToLists(List&lt;List&lt;T&gt;&gt;, T)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#addEquivalent(C,%20C)">addEquivalent(C, C)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/GeneSequence.html#addExon(org.biojava.nbio.core.sequence.AccessionID,%20int,%20int)">addExon(AccessionID, int, int)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Add an ExonSequence mainly used to mark as a feature</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/ProteinSequence.html#addFeature(org.biojava.nbio.core.sequence.features.FeatureInterface)">addFeature(FeatureInterface&lt;AbstractSequence&lt;AminoAcidCompound&gt;, AminoAcidCompound&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/ProteinSequence.html" title="class in org.biojava.nbio.core.sequence">ProteinSequence</a></dt>
<dd>
<div class="block">Add feature.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#addFeature(int,%20int,%20org.biojava.nbio.core.sequence.features.FeatureInterface)">addFeature(int, int, FeatureInterface&lt;AbstractSequence&lt;C&gt;, C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Method to help set the proper details for a feature as it relates to a sequence
 where the feature needs to have a location on the sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#addFeature(org.biojava.nbio.core.sequence.features.FeatureInterface)">addFeature(FeatureInterface&lt;AbstractSequence&lt;C&gt;, C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Add a feature to this sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/ChromosomeSequence.html#addGene(org.biojava.nbio.core.sequence.AccessionID,%20int,%20int,%20org.biojava.nbio.core.sequence.Strand)">addGene(AccessionID, int, int, Strand)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/ChromosomeSequence.html" title="class in org.biojava.nbio.core.sequence">ChromosomeSequence</a></dt>
<dd>
<div class="block">Add a gene to the chromosome sequence using bioIndexing starts at 1 instead of 0.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/GeneSequence.html#addIntron(org.biojava.nbio.core.sequence.AccessionID,%20int,%20int)">addIntron(AccessionID, int, int)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Add an Intron Currently used to mark an IntronSequence as a feature</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/GeneSequence.html#addIntronsUsingExons()">addIntronsUsingExons()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Once everything has been added to the gene sequence where you might have added exon sequences only then you
 can infer the intron sequences and add them.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#addNote(java.lang.String)">addNote(String)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Add notes about this sequence that will get exported for GFF3</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#addNucleotideCompound(java.lang.String,%20java.lang.String,%20java.lang.String...)">addNucleotideCompound(String, String, String...)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractNucleotideCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#addProperty(java.lang.String,%20java.lang.String)">addProperty(String, String)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">DBReferenceInfo</a></dt>
<dd>
<div class="block">Add a property and type to associate with this DBReferenceInfo</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html#addQualifier(java.lang.String,%20org.biojava.nbio.core.sequence.features.Qualifier)">addQualifier(String, Qualifier)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#addQualifier(java.lang.String,%20org.biojava.nbio.core.sequence.features.Qualifier)">addQualifier(String, Qualifier)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html#addQualifier(java.lang.String,%20org.biojava.nbio.core.sequence.features.Qualifier)">addQualifier(String, Qualifier)</a></span> - Method in interface org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Add a qualifier</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/QuantityFeature.html#addQuantity(java.lang.Number)">addQuantity(Number)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/QuantityFeature.html" title="class in org.biojava.nbio.core.sequence.features">QuantityFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html#addStartCodonSequence(org.biojava.nbio.core.sequence.AccessionID,%20int,%20int)">addStartCodonSequence(AccessionID, int, int)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html#addStopCodonSequence(org.biojava.nbio.core.sequence.AccessionID,%20int,%20int)">addStopCodonSequence(AccessionID, int, int)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#addStrings(java.lang.String,%20java.lang.String...)">addStrings(String, String...)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/FlatFileCache.html#addToCache(java.lang.String,%20java.io.File)">addToCache(String, File)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/FlatFileCache.html" title="class in org.biojava.nbio.core.util">FlatFileCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/GeneSequence.html#addTranscript(org.biojava.nbio.core.sequence.AccessionID,%20int,%20int)">addTranscript(AccessionID, int, int)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Add a transcription sequence to a gene which describes a ProteinSequence</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound"><span class="strong">AmbiguityDNACompoundSet</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/compound/package-summary.html">org.biojava.nbio.core.sequence.compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundSet.html#AmbiguityDNACompoundSet()">AmbiguityDNACompoundSet()</a></span> - Constructor for class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AmbiguityDNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/compound/AmbiguityRNACompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound"><span class="strong">AmbiguityRNACompoundSet</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/compound/package-summary.html">org.biojava.nbio.core.sequence.compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/AmbiguityRNACompoundSet.html#AmbiguityRNACompoundSet()">AmbiguityRNACompoundSet()</a></span> - Constructor for class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/AmbiguityRNACompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AmbiguityRNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava.nbio.core.sequence.compound"><span class="strong">AminoAcidCompound</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/compound/package-summary.html">org.biojava.nbio.core.sequence.compound</a></dt>
<dd>
<div class="block">Used to describe an Amino Acid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#AminoAcidCompound(org.biojava.nbio.core.sequence.compound.AminoAcidCompoundSet,%20java.lang.String,%20java.lang.String,%20java.lang.String,%20java.lang.Float)">AminoAcidCompound(AminoAcidCompoundSet, String, String, String, Float)</a></span> - Constructor for class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound"><span class="strong">AminoAcidCompoundSet</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/compound/package-summary.html">org.biojava.nbio.core.sequence.compound</a></dt>
<dd>
<div class="block">Set of proteinogenic amino acids.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#AminoAcidCompoundSet()">AminoAcidCompoundSet()</a></span> - Constructor for class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#aminoAcidsCompounds(org.biojava.nbio.core.sequence.template.CompoundSet)">aminoAcidsCompounds(CompoundSet&lt;AminoAcidCompound&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/loader/ArrayListProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader"><span class="strong">ArrayListProxySequenceReader</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/loader/ArrayListProxySequenceReader.html" title="type parameter in ArrayListProxySequenceReader">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/loader/package-summary.html">org.biojava.nbio.core.sequence.loader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/ArrayListProxySequenceReader.html#ArrayListProxySequenceReader()">ArrayListProxySequenceReader()</a></span> - Constructor for class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/ArrayListProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">ArrayListProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage"><span class="strong">ArrayListSequenceReader</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="type parameter in ArrayListSequenceReader">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/storage/package-summary.html">org.biojava.nbio.core.sequence.storage</a></dt>
<dd>
<div class="block">Stores a Sequence as a collection of compounds in an ArrayList</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#ArrayListSequenceReader()">ArrayListSequenceReader()</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#ArrayListSequenceReader(java.util.List,%20org.biojava.nbio.core.sequence.template.CompoundSet)">ArrayListSequenceReader(List&lt;C&gt;, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#ArrayListSequenceReader(java.lang.String,%20org.biojava.nbio.core.sequence.template.CompoundSet)">ArrayListSequenceReader(String, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.BondLocation.html#assertLocation()">assertLocation()</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.BondLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.BondLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#assertLocation()">assertLocation()</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html#attribute(java.lang.String,%20java.lang.String,%20java.lang.String)">attribute(String, String, String)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html#attribute(java.lang.String,%20java.lang.String)">attribute(String, String)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/XMLWriter.html#attribute(java.lang.String,%20java.lang.String,%20java.lang.String)">attribute(String, String, String)</a></span> - Method in interface org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/XMLWriter.html" title="interface in org.biojava.nbio.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Add an attribute to an element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/XMLWriter.html#attribute(java.lang.String,%20java.lang.String)">attribute(String, String)</a></span> - Method in interface org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/XMLWriter.html" title="interface in org.biojava.nbio.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Add an un-qualified attribute to an element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#AUTHORS_TAG">AUTHORS_TAG</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/UncompressInputStream.html#available()">available()</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/UncompressInputStream.html" title="class in org.biojava.nbio.core.util">UncompressInputStream</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_B_">
<!--   -->
</a>
<h2 class="title">B</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#BASE_COUNT_TAG">BASE_COUNT_TAG</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#BASE_COUNT_TAG_FULL">BASE_COUNT_TAG_FULL</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/BasicSequence.html" title="class in org.biojava.nbio.core.sequence"><span class="strong">BasicSequence</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/BasicSequence.html" title="type parameter in BasicSequence">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">Bare bones version of the Sequence object to be used sparingly.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/BasicSequence.html#BasicSequence(java.lang.String,%20org.biojava.nbio.core.sequence.template.CompoundSet)">BasicSequence(String, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/BasicSequence.html" title="class in org.biojava.nbio.core.sequence">BasicSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/BasicSequence.html#BasicSequence(org.biojava.nbio.core.sequence.template.ProxySequenceReader)">BasicSequence(ProxySequenceReader&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/BasicSequence.html" title="class in org.biojava.nbio.core.sequence">BasicSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#bitMask()">bitMask()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">This method should return the bit mask to be used to extract the
 bytes you are interested in working with.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html#bitMask()">bitMask()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader.FourBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html#bitMask()">bitMask()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader.TwoBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage"><span class="strong">BitSequenceReader</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="type parameter in BitSequenceReader">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/storage/package-summary.html">org.biojava.nbio.core.sequence.storage</a></dt>
<dd>
<div class="block">An implementation of the popular bit encodings.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#BitSequenceReader(org.biojava.nbio.core.sequence.storage.BitSequenceReader.BitArrayWorker,%20org.biojava.nbio.core.sequence.AccessionID)">BitSequenceReader(BitSequenceReader.BitArrayWorker&lt;C&gt;, AccessionID)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Instance which allows you to supply a different @{BitArrayWorker}
 object.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage"><span class="strong">BitSequenceReader.BitArrayWorker</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="type parameter in BitSequenceReader.BitArrayWorker">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/storage/package-summary.html">org.biojava.nbio.core.sequence.storage</a></dt>
<dd>
<div class="block">The logic of working with a bit has been separated out into this class
 to help developers create the bit data structures without having to
 put the code into an intermediate format and to also use the format
 without the need to copy this code.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#BitSequenceReader.BitArrayWorker(org.biojava.nbio.core.sequence.template.Sequence)">BitSequenceReader.BitArrayWorker(Sequence&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#BitSequenceReader.BitArrayWorker(java.lang.String,%20org.biojava.nbio.core.sequence.template.CompoundSet)">BitSequenceReader.BitArrayWorker(String, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#BitSequenceReader.BitArrayWorker(org.biojava.nbio.core.sequence.template.CompoundSet,%20int)">BitSequenceReader.BitArrayWorker(CompoundSet&lt;C&gt;, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#BitSequenceReader.BitArrayWorker(org.biojava.nbio.core.sequence.template.CompoundSet,%20int[])">BitSequenceReader.BitArrayWorker(CompoundSet&lt;C&gt;, int[])</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#bitsPerCompound()">bitsPerCompound()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Returns how many bits are used to represent a compound e.g.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">BufferedReaderBytesRead</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">Need to keep track of actual bytes read and take advantage of buffered reader
 performance.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#BufferedReaderBytesRead(java.io.Reader,%20int)">BufferedReaderBytesRead(Reader, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava.nbio.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>
<div class="block">Creates a buffering character-input stream that uses an input buffer of
 the specified size.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#BufferedReaderBytesRead(java.io.Reader)">BufferedReaderBytesRead(Reader)</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava.nbio.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>
<div class="block">Creates a buffering character-input stream that uses a default-sized
 input buffer.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#build()">build()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>
<div class="block">The method to finish any calls to the builder with which returns a
 transcription engine.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/RnaSequenceView.html#buildTranslators()">buildTranslators()</a></span> - Method in class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/RnaSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#BYTES_PER_INT">BYTES_PER_INT</a></span> - Static variable in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_C_">
<!--   -->
</a>
<h2 class="title">C</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/InputStreamProvider.html#CACHE_PROPERTY">CACHE_PROPERTY</a></span> - Static variable in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/InputStreamProvider.html" title="class in org.biojava.nbio.core.util">InputStreamProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#calculateIndirectAmbiguities()">calculateIndirectAmbiguities()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractNucleotideCompoundSet</a></dt>
<dd>
<div class="block">Loops through all known nucleotides and attempts to find which are
 equivalent to each other.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#canComplement(org.biojava.nbio.core.sequence.template.Sequence)">canComplement(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>
<div class="block">Uses the Sequence's CompoundSet to decide if a compound can
 be assgined to ComplementCompound meaning it can complement</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription"><span class="strong">CaseInsensitiveCompound</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/transcription/package-summary.html">org.biojava.nbio.core.sequence.transcription</a></dt>
<dd>
<div class="block">Attempts to wrap compounds so it is possible to view them
 in a case insensitive manner</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#CaseInsensitiveCompound(org.biojava.nbio.core.sequence.compound.NucleotideCompound)">CaseInsensitiveCompound(NucleotideCompound)</a></span> - Constructor for class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">CasePreservingProteinSequenceCreator</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">A sequence creator which preserves the case of its input string in
 the user collection of the returned ProteinSequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#CasePreservingProteinSequenceCreator(org.biojava.nbio.core.sequence.template.CompoundSet)">CasePreservingProteinSequenceCreator(CompoundSet&lt;AminoAcidCompound&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">CasePreservingProteinSequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/CDSComparator.html" title="class in org.biojava.nbio.core.sequence"><span class="strong">CDSComparator</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/CDSComparator.html#CDSComparator()">CDSComparator()</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/CDSComparator.html" title="class in org.biojava.nbio.core.sequence">CDSComparator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/CDSSequence.html" title="class in org.biojava.nbio.core.sequence"><span class="strong">CDSSequence</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">Represents a exon or coding sequence in a gene.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/CDSSequence.html#CDSSequence(org.biojava.nbio.core.sequence.TranscriptSequence,%20int,%20int,%20int)">CDSSequence(TranscriptSequence, int, int, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/CDSSequence.html" title="class in org.biojava.nbio.core.sequence">CDSSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#checksum(org.biojava.nbio.core.sequence.template.Sequence)">checksum(Sequence&lt;C&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Performs a simple CRC64 checksum on any given sequence.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/ChromosomeSequence.html" title="class in org.biojava.nbio.core.sequence"><span class="strong">ChromosomeSequence</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">A ChromosomeSequence is a DNASequence but keeps track of geneSequences</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/ChromosomeSequence.html#ChromosomeSequence()">ChromosomeSequence()</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/ChromosomeSequence.html" title="class in org.biojava.nbio.core.sequence">ChromosomeSequence</a></dt>
<dd>
<div class="block">Empty constructor used by tools that need a proper Bean that allows the actual
 sequence data to be set after construction.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/ChromosomeSequence.html#ChromosomeSequence(java.lang.String)">ChromosomeSequence(String)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/ChromosomeSequence.html" title="class in org.biojava.nbio.core.sequence">ChromosomeSequence</a></dt>
<dd>
<div class="block">String is king and assume DNA</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/ChromosomeSequence.html#ChromosomeSequence(org.biojava.nbio.core.sequence.template.SequenceReader)">ChromosomeSequence(SequenceReader&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/ChromosomeSequence.html" title="class in org.biojava.nbio.core.sequence">ChromosomeSequence</a></dt>
<dd>
<div class="block">Fairly important constructor given the size of a ChromsomeSequence where the
 ProxySequenceReader could load from disk via RandomAccessFile so that the sequence
 doesn't need to be kept in memory.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/ChromosomeSequence.html#ChromosomeSequence(java.lang.String,%20org.biojava.nbio.core.sequence.template.CompoundSet)">ChromosomeSequence(String, CompoundSet&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/ChromosomeSequence.html" title="class in org.biojava.nbio.core.sequence">ChromosomeSequence</a></dt>
<dd>
<div class="block">Allows the creation of a ChromosomeSequence using String for the sequence with a custom CompoundSet</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/ChromosomeSequence.html#ChromosomeSequence(org.biojava.nbio.core.sequence.template.SequenceReader,%20org.biojava.nbio.core.sequence.template.CompoundSet)">ChromosomeSequence(SequenceReader&lt;NucleotideCompound&gt;, CompoundSet&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/ChromosomeSequence.html" title="class in org.biojava.nbio.core.sequence">ChromosomeSequence</a></dt>
<dd>
<div class="block">Allows the creation of a ChromosomeSequence using a ProxyResequenceReader for the sequence with a custom CompoundSet</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#CIRCULAR_DNA">CIRCULAR_DNA</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriterHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html#circularLocation(int,%20int,%20org.biojava.nbio.core.sequence.Strand,%20int)">circularLocation(int, int, Strand, int)</a></span> - Static method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html" title="class in org.biojava.nbio.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Converts a location which defines the outer bounds of a circular
 location and splits it into the required portions.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Location.Tools.html#circularLocation(int,%20int,%20org.biojava.nbio.core.sequence.Strand,%20int)">circularLocation(int, int, Strand, int)</a></span> - Static method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Location.Tools.html" title="class in org.biojava.nbio.core.sequence.location.template">Location.Tools</a></dt>
<dd>
<div class="block">Converts a location which defines the outer bounds of a circular
 location and splits it into the required portions.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/Equals.html#classEqual(java.lang.Object,%20java.lang.Object)">classEqual(Object, Object)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/Equals.html" title="class in org.biojava.nbio.core.util">Equals</a></dt>
<dd>
<div class="block">This method should be called before beginning any equals methods.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/util/ClasspathResource.html" title="class in org.biojava.nbio.core.sequence.io.util"><span class="strong">ClasspathResource</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/io/util/package-summary.html">org.biojava.nbio.core.sequence.io.util</a></dt>
<dd>
<div class="block">This object represents a classpath resource on the local system.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#ClasspathResource(java.lang.String)">ClasspathResource(String)</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.util.<a href="./org/biojava/nbio/core/sequence/io/util/ClasspathResource.html" title="class in org.biojava.nbio.core.sequence.io.util">ClasspathResource</a></dt>
<dd>
<div class="block">Basic constructor only allowing you to specify where to find the file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#ClasspathResource(java.lang.String,%20boolean)">ClasspathResource(String, boolean)</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.util.<a href="./org/biojava/nbio/core/sequence/io/util/ClasspathResource.html" title="class in org.biojava.nbio.core.sequence.io.util">ClasspathResource</a></dt>
<dd>
<div class="block">Advanced constructor which allows you to optionally pre-cache the
 data</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#ClasspathResource(java.lang.String,%20boolean,%20boolean)">ClasspathResource(String, boolean, boolean)</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.util.<a href="./org/biojava/nbio/core/sequence/io/util/ClasspathResource.html" title="class in org.biojava.nbio.core.sequence.io.util">ClasspathResource</a></dt>
<dd>
<div class="block">Advanced constructor which lets you set the preCache variable and to
 force the type of file we are decompressing.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/FlatFileCache.html#clear()">clear()</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/FlatFileCache.html" title="class in org.biojava.nbio.core.util">FlatFileCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/SoftHashMap.html#clear()">clear()</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/SoftHashMap.html" title="class in org.biojava.nbio.core.util">SoftHashMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html#clonePoint()">clonePoint()</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Point.html#clonePoint()">clonePoint()</a></span> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Point.html" title="interface in org.biojava.nbio.core.sequence.location.template">Point</a></dt>
<dd>
<div class="block">Returns a copy of this point</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#close()">close()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava.nbio.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaReader.html#close()">close()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaReader.html" title="class in org.biojava.nbio.core.sequence.io">FastaReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankReader.html#close()">close()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankReader.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html#close(java.io.Closeable)">close(Closeable)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Closes any Object which implements the interface <a href="http://java.sun.com/javase/6/docs/api/java/io/Closeable.html?is-external=true" title="class or interface in java.io"><code>Closeable</code></a> and
 sending any error to the logger but not forcing any explicit catching of
 stream errors.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html#close()">close()</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/XMLWriter.html#close()">close()</a></span> - Method in interface org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/XMLWriter.html" title="interface in org.biojava.nbio.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Close this XMLWriter, and it's underlying stream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html#closeTag(java.lang.String,%20java.lang.String)">closeTag(String, String)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html#closeTag(java.lang.String)">closeTag(String)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/XMLWriter.html#closeTag(java.lang.String,%20java.lang.String)">closeTag(String, String)</a></span> - Method in interface org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/XMLWriter.html" title="interface in org.biojava.nbio.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Closes an element</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/XMLWriter.html#closeTag(java.lang.String)">closeTag(String)</a></span> - Method in interface org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/XMLWriter.html" title="interface in org.biojava.nbio.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Closes an un-qualified element.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound"><span class="strong">CodonCompound</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/compound/package-summary.html">org.biojava.nbio.core.sequence.compound</a></dt>
<dd>
<div class="block">Define a codon</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html#CodonCompound(org.biojava.nbio.core.sequence.compound.NucleotideCompound,%20org.biojava.nbio.core.sequence.compound.NucleotideCompound,%20org.biojava.nbio.core.sequence.compound.NucleotideCompound,%20boolean)">CodonCompound(NucleotideCompound, NucleotideCompound, NucleotideCompound, boolean)</a></span> - Constructor for class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#COMMENT_TAG">COMMENT_TAG</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/CDSComparator.html#compare(org.biojava.nbio.core.sequence.CDSSequence,%20org.biojava.nbio.core.sequence.CDSSequence)">compare(CDSSequence, CDSSequence)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/CDSComparator.html" title="class in org.biojava.nbio.core.sequence">CDSComparator</a></dt>
<dd>
<div class="block">Used to sort two CDSSequences where Negative Strand makes it tough</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/ExonComparator.html#compare(org.biojava.nbio.core.sequence.ExonSequence,%20org.biojava.nbio.core.sequence.ExonSequence)">compare(ExonSequence, ExonSequence)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/ExonComparator.html" title="class in org.biojava.nbio.core.sequence">ExonComparator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/SequenceComparator.html#compare(org.biojava.nbio.core.sequence.template.AbstractSequence,%20org.biojava.nbio.core.sequence.template.AbstractSequence)">compare(AbstractSequence&lt;?&gt;, AbstractSequence&lt;?&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/SequenceComparator.html" title="class in org.biojava.nbio.core.sequence">SequenceComparator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html#compareTo(org.biojava.nbio.core.sequence.location.template.Point)">compareTo(Point)</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html" title="class in org.biojava.nbio.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html#compareTo(org.biojava.nbio.core.sequence.location.template.Point)">compareTo(Point)</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/template/ComplementCompound.html" title="interface in org.biojava.nbio.core.sequence.template"><span class="strong">ComplementCompound</span></a> - Interface in <a href="./org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/views/ComplementSequenceView.html" title="class in org.biojava.nbio.core.sequence.views"><span class="strong">ComplementSequenceView</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/views/ComplementSequenceView.html" title="type parameter in ComplementSequenceView">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/ComplementCompound.html" title="interface in org.biojava.nbio.core.sequence.template">ComplementCompound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/views/package-summary.html">org.biojava.nbio.core.sequence.views</a></dt>
<dd>
<div class="block">For a given sequence this class will create a view over the top of it
 and for every request the code will return the complement of the underlying
 base e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/ComplementSequenceView.html#ComplementSequenceView(org.biojava.nbio.core.sequence.template.Sequence)">ComplementSequenceView(Sequence&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/ComplementSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">ComplementSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html#completeCircularPasses(int,%20int)">completeCircularPasses(int, int)</a></span> - Static method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html" title="class in org.biojava.nbio.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Works in a similar way to modulateCircularLocation but returns
 the number of complete passes over a Sequence length a circular
 location makes i.e.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Location.Tools.html#completeCircularPasses(int,%20int)">completeCircularPasses(int, int)</a></span> - Static method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Location.Tools.html" title="class in org.biojava.nbio.core.sequence.location.template">Location.Tools</a></dt>
<dd>
<div class="block">Works in a similar way to modulateCircularLocation but returns
 the number of complete passes over a Sequence length a circular
 location makes i.e.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template"><span class="strong">Compound</span></a> - Interface in <a href="./org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/exceptions/CompoundNotFoundException.html" title="class in org.biojava.nbio.core.exceptions"><span class="strong">CompoundNotFoundException</span></a> - Exception in <a href="./org/biojava/nbio/core/exceptions/package-summary.html">org.biojava.nbio.core.exceptions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/exceptions/CompoundNotFoundException.html#CompoundNotFoundException(java.lang.String)">CompoundNotFoundException(String)</a></span> - Constructor for exception org.biojava.nbio.core.exceptions.<a href="./org/biojava/nbio/core/exceptions/CompoundNotFoundException.html" title="class in org.biojava.nbio.core.exceptions">CompoundNotFoundException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#compoundsEqual(C,%20C)">compoundsEqual(C, C)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#compoundsEquivalent(org.biojava.nbio.core.sequence.compound.AminoAcidCompound,%20org.biojava.nbio.core.sequence.compound.AminoAcidCompound)">compoundsEquivalent(AminoAcidCompound, AminoAcidCompound)</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#compoundsEquivalent(C,%20C)">compoundsEquivalent(C, C)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/CompoundSet.html#compoundsEquivalent(C,%20C)">compoundsEquivalent(C, C)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template"><span class="strong">CompoundSet</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/template/CompoundSet.html" title="type parameter in CompoundSet">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#compoundsPerDatatype()">compoundsPerDatatype()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Should return the maximum amount of compounds we can encode per int</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html#compoundsPerDatatype()">compoundsPerDatatype()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader.FourBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html#compoundsPerDatatype()">compoundsPerDatatype()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader.TwoBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#compoundToInt(org.biojava.nbio.core.sequence.compound.NucleotideCompound)">compoundToInt(NucleotideCompound)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/template/CompoundTranslator.html" title="interface in org.biojava.nbio.core.sequence.template"><span class="strong">CompoundTranslator</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/template/CompoundTranslator.html" title="type parameter in CompoundTranslator">F</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>,<a href="./org/biojava/nbio/core/sequence/template/CompoundTranslator.html" title="type parameter in CompoundTranslator">T</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/util/ConcurrencyTools.html" title="class in org.biojava.nbio.core.util"><span class="strong">ConcurrencyTools</span></a> - Class in <a href="./org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>
<div class="block">Static utility to easily share a thread pool for concurrent/parallel/lazy execution.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html#consistentAccessions(java.util.List)">consistentAccessions(List&lt;Location&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html" title="class in org.biojava.nbio.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Scans a list of locations and returns true if all the given locations
 are linked to the same sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#CONSORTIUM_TAG">CONSORTIUM_TAG</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/StringManipulationHelper.html#convertStreamToString(java.io.InputStream)">convertStreamToString(InputStream)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/StringManipulationHelper.html" title="class in org.biojava.nbio.core.util">StringManipulationHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html#copy(java.io.InputStream,%20java.io.OutputStream)">copy(InputStream, OutputStream)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Moves the bytes from input to output using a 4KB byte array.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#countAT(org.biojava.nbio.core.sequence.template.Sequence)">countAT(Sequence&lt;NucleotideCompound&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Returns the count of AT in the given sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#countCompounds(C...)">countCompounds(C...)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#countCompounds(C...)">countCompounds(C...)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#countCompounds(C...)">countCompounds(C...)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#countCompounds(C...)">countCompounds(C...)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#countCompounds(C...)">countCompounds(C...)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Counts the number of times a compound appears in this sequence store</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#countCompounds(C...)">countCompounds(C...)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#countCompounds(C...)">countCompounds(C...)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Delegates to <code>SequenceMixin#countCompounds(org.biojava.nbio.core.sequence.template.Sequence, C[]) </code></div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#countCompounds(C...)">countCompounds(C...)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/Sequence.html#countCompounds(C...)">countCompounds(C...)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Returns the number of times we found a compound in the Sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#countCompounds(org.biojava.nbio.core.sequence.template.Sequence,%20C...)">countCompounds(Sequence&lt;C&gt;, C...)</a></span> - Static method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">For the given vargs of compounds this method counts the number of
 times those compounds appear in the given sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html#countCompounds(C...)">countCompounds(C...)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#countGC(org.biojava.nbio.core.sequence.template.Sequence)">countGC(Sequence&lt;NucleotideCompound&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Returns the count of GC in the given sequence</div>
</dd>
<dt><a href="./org/biojava/nbio/core/util/CRC64Checksum.html" title="class in org.biojava.nbio.core.util"><span class="strong">CRC64Checksum</span></a> - Class in <a href="./org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>
<div class="block">Utility class that calculates a CRC64 checksum on a stream of bytes.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/CRC64Checksum.html#CRC64Checksum()">CRC64Checksum()</a></span> - Constructor for class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/CRC64Checksum.html" title="class in org.biojava.nbio.core.util">CRC64Checksum</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#createIterator(org.biojava.nbio.core.sequence.template.Sequence)">createIterator(Sequence&lt;C&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Creates a simple sequence iterator which moves through a sequence going
 from 1 to the length of the Sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#createSequence(org.biojava.nbio.core.sequence.template.Sequence)">createSequence(Sequence&lt;F&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/CompoundTranslator.html#createSequence(org.biojava.nbio.core.sequence.template.Sequence)">createSequence(Sequence&lt;F&gt;)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/CompoundTranslator.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#createSequence(org.biojava.nbio.core.sequence.template.Sequence,%20org.biojava.nbio.core.sequence.transcription.Frame)">createSequence(Sequence&lt;NucleotideCompound&gt;, Frame)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html" title="class in org.biojava.nbio.core.sequence.transcription">DNAToRNATranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#createSequence(org.biojava.nbio.core.sequence.template.Sequence)">createSequence(Sequence&lt;NucleotideCompound&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html" title="class in org.biojava.nbio.core.sequence.transcription">DNAToRNATranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#createSequences(org.biojava.nbio.core.sequence.template.Sequence)">createSequences(Sequence&lt;F&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/CompoundTranslator.html#createSequences(org.biojava.nbio.core.sequence.template.Sequence)">createSequences(Sequence&lt;F&gt;)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/CompoundTranslator.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#createSequences(org.biojava.nbio.core.sequence.template.Sequence)">createSequences(Sequence&lt;NucleotideCompound&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html" title="class in org.biojava.nbio.core.sequence.transcription">DNAToRNATranslator</a></dt>
<dd>
<div class="block">Overloaded local version which delegates to an optional translator
 when told to (specified during construction).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#createSequences(org.biojava.nbio.core.sequence.template.Sequence)">createSequences(Sequence&lt;NucleotideCompound&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html" title="class in org.biojava.nbio.core.sequence.transcription">RNAToAminoAcidTranslator</a></dt>
<dd>
<div class="block">Performs the core conversion of RNA to Peptide.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#createSubSequence(org.biojava.nbio.core.sequence.template.Sequence,%20int,%20int)">createSubSequence(Sequence&lt;C&gt;, int, int)</a></span> - Static method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Creates a simple sub sequence view delimited by the given start and end.</div>
</dd>
</dl>
<a name="_D_">
<!--   -->
</a>
<h2 class="title">D</h2>
<dl>
<dt><a href="./org/biojava/nbio/core/sequence/features/DatabaseReferenceInterface.html" title="interface in org.biojava.nbio.core.sequence.features"><span class="strong">DatabaseReferenceInterface</span></a> - Interface in <a href="./org/biojava/nbio/core/sequence/features/package-summary.html">org.biojava.nbio.core.sequence.features</a></dt>
<dd>
<div class="block">If a SequenceProxyReader implements this interface then that external source
 has a list of cross reference id(s)</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/DataSource.html" title="enum in org.biojava.nbio.core.sequence"><span class="strong">DataSource</span></a> - Enum in <a href="./org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">
 GenBank                           gi|gi-number|gb|accession|locus
 ENA Data Library                  gi|gi-number|emb|accession|locus
 DDBJ, DNA Database of Japan       gi|gi-number|dbj|accession|locus
 NBRF PIR                          pir||entry
 Protein Research Foundation       prf||name
 SWISS-PROT UNIPROT                sp|accession|name
 Brookhaven Protein Data Bank (1)  pdb|entry|chain
 Brookhaven Protein Data Bank (2)  entry:chain|PDBID|CHAIN|SEQUENCE
 Patents                           pat|country|number
 GenInfo Backbone Id               bbs|number
 General database identifier       gnl|database|identifier
 NCBI Reference Sequence           ref|accession|locus
 Local Sequence identifier         lcl|identifier
 </pr></div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features"><span class="strong">DBReferenceInfo</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/features/package-summary.html">org.biojava.nbio.core.sequence.features</a></dt>
<dd>
<div class="block">If you have a uniprot ID then it is possible to get a collection
 of other id(s) that the protein is known by.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#DBReferenceInfo(java.lang.String,%20java.lang.String)">DBReferenceInfo(String, String)</a></span> - Constructor for class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">DBReferenceInfo</a></dt>
<dd>
<div class="block">The source database and id</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#dbxp">dbxp</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html#declareNamespace(java.lang.String,%20java.lang.String)">declareNamespace(String, String)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/XMLWriter.html#declareNamespace(java.lang.String,%20java.lang.String)">declareNamespace(String, String)</a></span> - Method in interface org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/XMLWriter.html" title="interface in org.biojava.nbio.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Hints that a namespace is going to be used in a sub-tree.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#decorateRna(boolean)">decorateRna(boolean)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>
<div class="block">Performs an optimisation where RNASequences are not translated into
 their own objects but are views onto the base DNA sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/SoftHashMap.html#DEFAULT_LIMIT">DEFAULT_LIMIT</a></span> - Static variable in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/SoftHashMap.html" title="class in org.biojava.nbio.core.util">SoftHashMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#DEFINITION_TAG">DEFINITION_TAG</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/FlatFileCache.html#destroy()">destroy()</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/FlatFileCache.html" title="class in org.biojava.nbio.core.util">FlatFileCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html#detectCicular(java.util.List)">detectCicular(List&lt;Location&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html" title="class in org.biojava.nbio.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Loops through the given list of locations and returns true if it looks
 like they represent a circular location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html#detectEnd(java.util.List,%20boolean)">detectEnd(List&lt;Location&gt;, boolean)</a></span> - Static method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html" title="class in org.biojava.nbio.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">This will attempt to find what the last point is and returns that
 position.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html#detectStart(java.util.List)">detectStart(List&lt;Location&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html" title="class in org.biojava.nbio.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Assumes that the first element is the start & clones it</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html#detectStrand(java.util.List)">detectStrand(List&lt;Location&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html" title="class in org.biojava.nbio.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Loops through the given list of locations and returns the consensus
 Strand class.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#dnaCompounds(org.biojava.nbio.core.sequence.template.CompoundSet)">dnaCompounds(CompoundSet&lt;NucleotideCompound&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/compound/DNACompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound"><span class="strong">DNACompoundSet</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/compound/package-summary.html">org.biojava.nbio.core.sequence.compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/DNACompoundSet.html#DNACompoundSet()">DNACompoundSet()</a></span> - Constructor for class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/DNACompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">DNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#dnaRnaTranslator(org.biojava.nbio.core.sequence.transcription.DNAToRNATranslator)">dnaRnaTranslator(DNAToRNATranslator)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence"><span class="strong">DNASequence</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">This is class should model the attributes associated with a DNA sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/DNASequence.html#DNASequence()">DNASequence()</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Shouldn't be used but makes it bean happy</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/DNASequence.html#DNASequence(java.lang.String)">DNASequence(String)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">String is king and create a sequence from DNA with default DNA compound set</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/DNASequence.html#DNASequence(org.biojava.nbio.core.sequence.template.SequenceReader)">DNASequence(SequenceReader&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Create a sequence where the actual storage of the sequence data is somewhere else</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/DNASequence.html#DNASequence(java.lang.String,%20org.biojava.nbio.core.sequence.template.CompoundSet)">DNASequence(String, CompoundSet&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Create a sequence from a string with user defined compound set</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/DNASequence.html#DNASequence(org.biojava.nbio.core.sequence.template.SequenceReader,%20org.biojava.nbio.core.sequence.template.CompoundSet)">DNASequence(SequenceReader&lt;NucleotideCompound&gt;, CompoundSet&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Create a sequence from a ProxySequencereader and user defined compound set</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/DNASequence.DNAType.html" title="enum in org.biojava.nbio.core.sequence"><span class="strong">DNASequence.DNAType</span></a> - Enum in <a href="./org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">The type of DNA sequence</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/DNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">DNASequenceCreator</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">A helper class that allows different ways to read a string and create a DNA sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#DNASequenceCreator(org.biojava.nbio.core.sequence.template.CompoundSet)">DNASequenceCreator(CompoundSet&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/DNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">DNASequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html" title="class in org.biojava.nbio.core.sequence.transcription"><span class="strong">DNAToRNATranslator</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/transcription/package-summary.html">org.biojava.nbio.core.sequence.transcription</a></dt>
<dd>
<div class="block">Performs the first stage of transcription by going from DNA to RNA.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#DNAToRNATranslator(org.biojava.nbio.core.sequence.io.template.SequenceCreatorInterface,%20org.biojava.nbio.core.sequence.template.CompoundSet,%20org.biojava.nbio.core.sequence.template.CompoundSet,%20boolean)">DNAToRNATranslator(SequenceCreatorInterface&lt;NucleotideCompound&gt;, CompoundSet&lt;NucleotideCompound&gt;, CompoundSet&lt;NucleotideCompound&gt;, boolean)</a></span> - Constructor for class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html" title="class in org.biojava.nbio.core.sequence.transcription">DNAToRNATranslator</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_E_">
<!--   -->
</a>
<h2 class="title">E</h2>
<dl>
<dt><a href="./org/biojava/nbio/core/sequence/edits/Edit.html" title="interface in org.biojava.nbio.core.sequence.edits"><span class="strong">Edit</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/edits/Edit.html" title="type parameter in Edit">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava/nbio/core/sequence/edits/package-summary.html">org.biojava.nbio.core.sequence.edits</a></dt>
<dd>
<div class="block">Interface for carrying out edit operations on a Sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html#edit(org.biojava.nbio.core.sequence.template.Sequence)">edit(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.html#edit(org.biojava.nbio.core.sequence.template.Sequence)">edit(Sequence&lt;C&gt;)</a></span> - Method in interface org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.html" title="interface in org.biojava.nbio.core.sequence.edits">Edit</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava.nbio.core.sequence.edits"><span class="strong">Edit.AbstractEdit</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html" title="type parameter in Edit.AbstractEdit">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/edits/package-summary.html">org.biojava.nbio.core.sequence.edits</a></dt>
<dd>
<div class="block">Abstract class which defines all edit operations as a call to discover
 what 5' and 3' ends of an editing Sequence should be joined together
 with a target Sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html#Edit.AbstractEdit(int)">Edit.AbstractEdit(int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html#Edit.AbstractEdit(int,%20int)">Edit.AbstractEdit(int, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/edits/Edit.Delete.html" title="class in org.biojava.nbio.core.sequence.edits"><span class="strong">Edit.Delete</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/edits/Edit.Delete.html" title="type parameter in Edit.Delete">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/edits/package-summary.html">org.biojava.nbio.core.sequence.edits</a></dt>
<dd>
<div class="block">Implementation which allows for the deletion of bases from a Sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.Delete.html#Edit.Delete(int)">Edit.Delete(int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.Delete.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Delete</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.Delete.html#Edit.Delete(int,%20int)">Edit.Delete(int, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.Delete.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Delete</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/edits/Edit.Insert.html" title="class in org.biojava.nbio.core.sequence.edits"><span class="strong">Edit.Insert</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/edits/Edit.Insert.html" title="type parameter in Edit.Insert">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/edits/package-summary.html">org.biojava.nbio.core.sequence.edits</a></dt>
<dd>
<div class="block">Edit implementation which allows us to insert a base at any position
 in a Sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.Insert.html#Edit.Insert(java.lang.String,%20int)">Edit.Insert(String, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.Insert.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Insert</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.Insert.html#Edit.Insert(org.biojava.nbio.core.sequence.template.Sequence,%20int)">Edit.Insert(Sequence&lt;C&gt;, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.Insert.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Insert</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.Insert.html#Edit.Insert(java.lang.String,%20int,%20int)">Edit.Insert(String, int, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.Insert.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Insert</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.Insert.html#Edit.Insert(org.biojava.nbio.core.sequence.template.Sequence,%20int,%20int)">Edit.Insert(Sequence&lt;C&gt;, int, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.Insert.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Insert</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/edits/Edit.Substitute.html" title="class in org.biojava.nbio.core.sequence.edits"><span class="strong">Edit.Substitute</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/edits/Edit.Substitute.html" title="type parameter in Edit.Substitute">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/edits/package-summary.html">org.biojava.nbio.core.sequence.edits</a></dt>
<dd>
<div class="block">Allows for the substitution of bases into an existing Sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.Substitute.html#Edit.Substitute(java.lang.String,%20int)">Edit.Substitute(String, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.Substitute.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Substitute</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.Substitute.html#Edit.Substitute(org.biojava.nbio.core.sequence.template.Sequence,%20int)">Edit.Substitute(Sequence&lt;C&gt;, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.Substitute.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Substitute</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Location.html#EMPTY">EMPTY</a></span> - Static variable in interface org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Basic location which is set to the minimum and maximum bounds of
 <a href="http://java.sun.com/javase/6/docs/api/java/lang/Integer.html?is-external=true" title="class or interface in java.lang"><code>Integer</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#END_SEQUENCE_TAG">END_SEQUENCE_TAG</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/SoftHashMap.html#entrySet()">entrySet()</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/SoftHashMap.html" title="class in org.biojava.nbio.core.util">SoftHashMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/Equals.html#equal(int,%20int)">equal(int, int)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/Equals.html" title="class in org.biojava.nbio.core.util">Equals</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/Equals.html#equal(long,%20long)">equal(long, long)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/Equals.html" title="class in org.biojava.nbio.core.util">Equals</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/Equals.html#equal(boolean,%20boolean)">equal(boolean, boolean)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/Equals.html" title="class in org.biojava.nbio.core.util">Equals</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/Equals.html#equal(java.lang.Object,%20java.lang.Object)">equal(Object, Object)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/Equals.html" title="class in org.biojava.nbio.core.util">Equals</a></dt>
<dd>
<div class="block">Does not compare class types.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/AccessionID.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html" title="class in org.biojava.nbio.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/util/Equals.html" title="class in org.biojava.nbio.core.util"><span class="strong">Equals</span></a> - Class in <a href="./org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>
<div class="block">A set of helper methods which return true if the two parameters are
 equal to each other.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/Equals.html#Equals()">Equals()</a></span> - Constructor for class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/Equals.html" title="class in org.biojava.nbio.core.util">Equals</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#equalsIgnoreCase(org.biojava.nbio.core.sequence.template.Compound)">equalsIgnoreCase(Compound)</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html#equalsIgnoreCase(org.biojava.nbio.core.sequence.template.Compound)">equalsIgnoreCase(Compound)</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#equalsIgnoreCase(org.biojava.nbio.core.sequence.template.Compound)">equalsIgnoreCase(Compound)</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html#equalsIgnoreCase(org.biojava.nbio.core.sequence.template.Compound)">equalsIgnoreCase(Compound)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/Compound.html#equalsIgnoreCase(org.biojava.nbio.core.sequence.template.Compound)">equalsIgnoreCase(Compound)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#equalsIgnoreCase(org.biojava.nbio.core.sequence.template.Compound)">equalsIgnoreCase(Compound)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html#equalsIgnoreCase(org.biojava.nbio.core.sequence.template.Compound)">equalsIgnoreCase(Compound)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/StringManipulationHelper.html#equalsToIgnoreEndline(java.lang.String,%20java.lang.String)">equalsToIgnoreEndline(String, String)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/StringManipulationHelper.html" title="class in org.biojava.nbio.core.util">StringManipulationHelper</a></dt>
<dd>
<div class="block">compares two strings for equality, line by line, ignoring any difference
 of end line delimiters contained within the 2 Strings.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/StringManipulationHelper.html#equalsToXml(java.lang.String,%20java.lang.String)">equalsToXml(String, String)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/StringManipulationHelper.html" title="class in org.biojava.nbio.core.util">StringManipulationHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/ExonComparator.html" title="class in org.biojava.nbio.core.sequence"><span class="strong">ExonComparator</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">Sort Exon where it is a little confusing if exons shoud always be ordered left to right
 where a negative stranded gene should go the other direction.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/ExonComparator.html#ExonComparator()">ExonComparator()</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/ExonComparator.html" title="class in org.biojava.nbio.core.sequence">ExonComparator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/ExonSequence.html" title="class in org.biojava.nbio.core.sequence"><span class="strong">ExonSequence</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">A gene contains a collection of Exon sequences</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/ExonSequence.html#ExonSequence(org.biojava.nbio.core.sequence.GeneSequence,%20int,%20int)">ExonSequence(GeneSequence, int, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/ExonSequence.html" title="class in org.biojava.nbio.core.sequence">ExonSequence</a></dt>
<dd>
<div class="block">Need a parent gene sequence and the bioBegin and bioEnd.</div>
</dd>
</dl>
<a name="_F_">
<!--   -->
</a>
<h2 class="title">F</h2>
<dl>
<dt><a href="./org/biojava/nbio/core/sequence/io/FastaGeneWriter.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">FastaGeneWriter</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">A Gene sequence has a Positive or Negative Strand where we want to write out to a stream the 5 to 3 prime version.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#FastaGeneWriter(java.io.OutputStream,%20java.util.Collection,%20org.biojava.nbio.core.sequence.io.template.FastaHeaderFormatInterface,%20boolean)">FastaGeneWriter(OutputStream, Collection&lt;GeneSequence&gt;, FastaHeaderFormatInterface&lt;GeneSequence, NucleotideCompound&gt;, boolean)</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaGeneWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaGeneWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#FastaGeneWriter(java.io.OutputStream,%20java.util.Collection,%20org.biojava.nbio.core.sequence.io.template.FastaHeaderFormatInterface,%20boolean,%20int)">FastaGeneWriter(OutputStream, Collection&lt;GeneSequence&gt;, FastaHeaderFormatInterface&lt;GeneSequence, NucleotideCompound&gt;, boolean, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaGeneWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaGeneWriter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/template/FastaHeaderFormatInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template"><span class="strong">FastaHeaderFormatInterface</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/io/template/FastaHeaderFormatInterface.html" title="type parameter in FastaHeaderFormatInterface">S</a> extends <a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;?&gt;,<a href="./org/biojava/nbio/core/sequence/io/template/FastaHeaderFormatInterface.html" title="type parameter in FastaHeaderFormatInterface">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava/nbio/core/sequence/io/template/package-summary.html">org.biojava.nbio.core.sequence.io.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/FastaReader.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">FastaReader</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/io/FastaReader.html" title="type parameter in FastaReader">S</a> extends <a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;?&gt;,<a href="./org/biojava/nbio/core/sequence/io/FastaReader.html" title="type parameter in FastaReader">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">Use FastaReaderHelper as an example of how to use this class where FastaReaderHelper should be the
 primary class used to read Fasta files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaReader.html#FastaReader(java.io.InputStream,%20org.biojava.nbio.core.sequence.io.template.SequenceHeaderParserInterface,%20org.biojava.nbio.core.sequence.io.template.SequenceCreatorInterface)">FastaReader(InputStream, SequenceHeaderParserInterface&lt;S, C&gt;, SequenceCreatorInterface&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaReader.html" title="class in org.biojava.nbio.core.sequence.io">FastaReader</a></dt>
<dd>
<div class="block">If you are going to use FileProxyProteinSequenceCreator then do not use this constructor because we need details about
 local file offsets for quick reads.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaReader.html#FastaReader(java.io.File,%20org.biojava.nbio.core.sequence.io.template.SequenceHeaderParserInterface,%20org.biojava.nbio.core.sequence.io.template.SequenceCreatorInterface)">FastaReader(File, SequenceHeaderParserInterface&lt;S, C&gt;, SequenceCreatorInterface&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaReader.html" title="class in org.biojava.nbio.core.sequence.io">FastaReader</a></dt>
<dd>
<div class="block">If you are going to use the FileProxyProteinSequenceCreator then you
 need to use this constructor because we need details about
 the location of the file.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/FastaReaderHelper.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">FastaReaderHelper</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#FastaReaderHelper()">FastaReaderHelper()</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaReaderHelper.html" title="class in org.biojava.nbio.core.sequence.io">FastaReaderHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/FastaSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">FastaSequenceParser</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">Used to parse a stream of a fasta file to get the sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaSequenceParser.html#FastaSequenceParser()">FastaSequenceParser()</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">FastaSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/FastaWriter.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">FastaWriter</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/io/FastaWriter.html" title="type parameter in FastaWriter">S</a> extends <a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;?&gt;,<a href="./org/biojava/nbio/core/sequence/io/FastaWriter.html" title="type parameter in FastaWriter">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">The FastaWriter writes a collection of sequences to an outputStream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaWriter.html#FastaWriter(java.io.OutputStream,%20java.util.Collection,%20org.biojava.nbio.core.sequence.io.template.FastaHeaderFormatInterface)">FastaWriter(OutputStream, Collection&lt;S&gt;, FastaHeaderFormatInterface&lt;S, C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriter</a></dt>
<dd>
<div class="block">Use default line length of 60</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaWriter.html#FastaWriter(java.io.OutputStream,%20java.util.Collection,%20org.biojava.nbio.core.sequence.io.template.FastaHeaderFormatInterface,%20int)">FastaWriter(OutputStream, Collection&lt;S&gt;, FastaHeaderFormatInterface&lt;S, C&gt;, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriter</a></dt>
<dd>
<div class="block">Set custom lineLength</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/FastaWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">FastaWriterHelper</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">The class that should be used to write out fasta file of a sequence collection</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#FastaWriterHelper()">FastaWriterHelper()</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriterHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#FEATURE_TAG">FEATURE_TAG</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features"><span class="strong">FeatureDbReferenceInfo</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="type parameter in FeatureDbReferenceInfo">S</a> extends <a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="type parameter in FeatureDbReferenceInfo">C</a>&gt;,<a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="type parameter in FeatureDbReferenceInfo">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/features/package-summary.html">org.biojava.nbio.core.sequence.features</a></dt>
<dd>
<div class="block">It is <a href="./org/biojava/nbio/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features"><code>DBReferenceInfo</code></a> which implements <a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features"><code>FeatureInterface</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#FeatureDbReferenceInfo(java.lang.String,%20java.lang.String)">FeatureDbReferenceInfo(String, String)</a></span> - Constructor for class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html#featureGlobalEnd">featureGlobalEnd</a></span> - Variable in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html" title="class in org.biojava.nbio.core.sequence.location">InsdcParser</a></dt>
<dd>
<div class="block">these variables are used to compute the global start and end of complex
 features</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html#featureGlobalStart">featureGlobalStart</a></span> - Variable in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html" title="class in org.biojava.nbio.core.sequence.location">InsdcParser</a></dt>
<dd>
<div class="block">these variables are used to compute the global start and end of complex
 features</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features"><span class="strong">FeatureInterface</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="type parameter in FeatureInterface">S</a> extends <a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="type parameter in FeatureInterface">C</a>&gt;,<a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="type parameter in FeatureInterface">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava/nbio/core/sequence/features/package-summary.html">org.biojava.nbio.core.sequence.features</a></dt>
<dd>
<div class="block">Interface class to handle describing arbitrary features.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/features/FeatureRetriever.html" title="interface in org.biojava.nbio.core.sequence.features"><span class="strong">FeatureRetriever</span></a> - Interface in <a href="./org/biojava/nbio/core/sequence/features/package-summary.html">org.biojava.nbio.core.sequence.features</a></dt>
<dd>
<div class="block">If a SequenceProxyReader implements this interface then that external source
 has a list features</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/features/FeaturesKeyWordInterface.html" title="interface in org.biojava.nbio.core.sequence.features"><span class="strong">FeaturesKeyWordInterface</span></a> - Interface in <a href="./org/biojava/nbio/core/sequence/features/package-summary.html">org.biojava.nbio.core.sequence.features</a></dt>
<dd>
<div class="block">Models the keywords that are annotated for a protein sequence at Uniprot.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">FileProxyDNASequenceCreator</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">This class is a good example of using the SequenceCreatorInterface where during parsing of the stream
 the sequence and the offset index are passed to create a Protein sequence that will be loaded in lazily.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#FileProxyDNASequenceCreator(java.io.File,%20org.biojava.nbio.core.sequence.template.CompoundSet,%20org.biojava.nbio.core.sequence.io.template.SequenceParserInterface)">FileProxyDNASequenceCreator(File, CompoundSet&lt;NucleotideCompound&gt;, SequenceParserInterface)</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">FileProxyDNASequenceCreator</a></dt>
<dd>
<div class="block">Need File so that we can store full path name in SequenceFileProxyLoader for Random File access as a quick read</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">FileProxyProteinSequenceCreator</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">This class is a good example of using the SequenceCreatorInterface where during parsing of the stream
 the sequence and the offset index are passed to create a Protein sequence that will be loaded in lazily.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#FileProxyProteinSequenceCreator(java.io.File,%20org.biojava.nbio.core.sequence.template.CompoundSet,%20org.biojava.nbio.core.sequence.io.template.SequenceParserInterface)">FileProxyProteinSequenceCreator(File, CompoundSet&lt;AminoAcidCompound&gt;, SequenceParserInterface)</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">FileProxyProteinSequenceCreator</a></dt>
<dd>
<div class="block">Need File so that we can store full path name in SequenceFileProxyLoader for Random File access as a quick read</div>
</dd>
<dt><a href="./org/biojava/nbio/core/util/FlatFileCache.html" title="class in org.biojava.nbio.core.util"><span class="strong">FlatFileCache</span></a> - Class in <a href="./org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>
<div class="block">Provides a cache for storing multiple small files in memory.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage"><span class="strong">FourBitSequenceReader</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html" title="type parameter in FourBitSequenceReader">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/storage/package-summary.html">org.biojava.nbio.core.sequence.storage</a></dt>
<dd>
<div class="block">Four bit encoding of the bit formats.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#FourBitSequenceReader(org.biojava.nbio.core.sequence.template.Sequence)">FourBitSequenceReader(Sequence&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#FourBitSequenceReader(java.lang.String,%20org.biojava.nbio.core.sequence.template.CompoundSet)">FourBitSequenceReader(String, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#FourBitSequenceReader(java.lang.String,%20org.biojava.nbio.core.sequence.template.CompoundSet,%20org.biojava.nbio.core.sequence.AccessionID)">FourBitSequenceReader(String, CompoundSet&lt;C&gt;, AccessionID)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#FourBitSequenceReader(org.biojava.nbio.core.sequence.storage.FourBitSequenceReader.FourBitArrayWorker)">FourBitSequenceReader(FourBitSequenceReader.FourBitArrayWorker&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#FourBitSequenceReader(org.biojava.nbio.core.sequence.storage.FourBitSequenceReader.FourBitArrayWorker,%20org.biojava.nbio.core.sequence.AccessionID)">FourBitSequenceReader(FourBitSequenceReader.FourBitArrayWorker&lt;C&gt;, AccessionID)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage"><span class="strong">FourBitSequenceReader.FourBitArrayWorker</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="type parameter in FourBitSequenceReader.FourBitArrayWorker">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/storage/package-summary.html">org.biojava.nbio.core.sequence.storage</a></dt>
<dd>
<div class="block">A four bit per compound implementation of the bit array worker code.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html#FourBitSequenceReader.FourBitArrayWorker(org.biojava.nbio.core.sequence.template.CompoundSet,%20int)">FourBitSequenceReader.FourBitArrayWorker(CompoundSet&lt;C&gt;, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader.FourBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html#FourBitSequenceReader.FourBitArrayWorker(org.biojava.nbio.core.sequence.template.CompoundSet,%20int[])">FourBitSequenceReader.FourBitArrayWorker(CompoundSet&lt;C&gt;, int[])</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader.FourBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html#FourBitSequenceReader.FourBitArrayWorker(org.biojava.nbio.core.sequence.template.Sequence)">FourBitSequenceReader.FourBitArrayWorker(Sequence&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader.FourBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html#FourBitSequenceReader.FourBitArrayWorker(java.lang.String,%20org.biojava.nbio.core.sequence.template.CompoundSet)">FourBitSequenceReader.FourBitArrayWorker(String, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader.FourBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/transcription/Frame.html" title="enum in org.biojava.nbio.core.sequence.transcription"><span class="strong">Frame</span></a> - Enum in <a href="./org/biojava/nbio/core/sequence/transcription/package-summary.html">org.biojava.nbio.core.sequence.transcription</a></dt>
<dd>
<div class="block">Indicates a way of translating a sequence.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html" title="class in org.biojava.nbio.core.sequence.location"><span class="strong">FuzzyPoint</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a></dt>
<dd>
<div class="block">Implementation for resolving fuzzy locations.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html#FuzzyPoint(int,%20int)">FuzzyPoint(int, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html" title="class in org.biojava.nbio.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html#FuzzyPoint(int,%20int,%20org.biojava.nbio.core.sequence.location.template.Point.Resolver)">FuzzyPoint(int, int, Point.Resolver&lt;FuzzyPoint&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html" title="class in org.biojava.nbio.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html#FuzzyPoint(int,%20int,%20org.biojava.nbio.core.sequence.location.template.Point.Resolver,%20boolean,%20boolean)">FuzzyPoint(int, int, Point.Resolver&lt;FuzzyPoint&gt;, boolean, boolean)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html" title="class in org.biojava.nbio.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_G_">
<!--   -->
</a>
<h2 class="title">G</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#GENBANK_FORMAT">GENBANK_FORMAT</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>
<div class="block">The name of this format</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template"><span class="strong">GenbankHeaderFormatInterface</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html" title="type parameter in GenbankHeaderFormatInterface">S</a> extends <a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;?&gt;,<a href="./org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html" title="type parameter in GenbankHeaderFormatInterface">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava/nbio/core/sequence/io/template/package-summary.html">org.biojava.nbio.core.sequence.io.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader"><span class="strong">GenbankProxySequenceReader</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="type parameter in GenbankProxySequenceReader">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/loader/package-summary.html">org.biojava.nbio.core.sequence.loader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#GenbankProxySequenceReader(java.lang.String,%20java.lang.String,%20org.biojava.nbio.core.sequence.template.CompoundSet)">GenbankProxySequenceReader(String, String, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/GenbankReader.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">GenbankReader</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/io/GenbankReader.html" title="type parameter in GenbankReader">S</a> extends <a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="./org/biojava/nbio/core/sequence/io/GenbankReader.html" title="type parameter in GenbankReader">C</a>&gt;,<a href="./org/biojava/nbio/core/sequence/io/GenbankReader.html" title="type parameter in GenbankReader">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">Use GenbankReaderHelper as an example of how to use this class where GenbankReaderHelper should be the
 primary class used to read Genbank files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankReader.html#GenbankReader(java.io.InputStream,%20org.biojava.nbio.core.sequence.io.template.SequenceHeaderParserInterface,%20org.biojava.nbio.core.sequence.io.template.SequenceCreatorInterface)">GenbankReader(InputStream, SequenceHeaderParserInterface&lt;S, C&gt;, SequenceCreatorInterface&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankReader.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReader</a></dt>
<dd>
<div class="block">If you are going to use FileProxyProteinSequenceCreator then do not use this constructor because we need details about
 local file offsets for quick reads.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankReader.html#GenbankReader(java.io.File,%20org.biojava.nbio.core.sequence.io.template.SequenceHeaderParserInterface,%20org.biojava.nbio.core.sequence.io.template.SequenceCreatorInterface)">GenbankReader(File, SequenceHeaderParserInterface&lt;S, C&gt;, SequenceCreatorInterface&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankReader.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReader</a></dt>
<dd>
<div class="block">If you are going to use the FileProxyProteinSequenceCreator then you
 need to use this constructor because we need details about
 the location of the file.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">GenbankReaderHelper</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#GenbankReaderHelper()">GenbankReaderHelper()</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReaderHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">GenbankSequenceParser</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="type parameter in GenbankSequenceParser">S</a> extends <a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="type parameter in GenbankSequenceParser">C</a>&gt;,<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="type parameter in GenbankSequenceParser">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#GenbankSequenceParser()">GenbankSequenceParser()</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html#genbankSplitPattern">genbankSplitPattern</a></span> - Static variable in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html" title="class in org.biojava.nbio.core.sequence.location">InsdcParser</a></dt>
<dd>
<div class="block">Decodes a split pattern.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/GenbankWriter.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">GenbankWriter</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/io/GenbankWriter.html" title="type parameter in GenbankWriter">S</a> extends <a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;?&gt;,<a href="./org/biojava/nbio/core/sequence/io/GenbankWriter.html" title="type parameter in GenbankWriter">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankWriter.html#GenbankWriter(java.io.OutputStream,%20java.util.Collection,%20org.biojava.nbio.core.sequence.io.template.GenbankHeaderFormatInterface)">GenbankWriter(OutputStream, Collection&lt;S&gt;, GenbankHeaderFormatInterface&lt;S, C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankWriter.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriter</a></dt>
<dd>
<div class="block">Use default line length of 60</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankWriter.html#GenbankWriter(java.io.OutputStream,%20java.util.Collection,%20org.biojava.nbio.core.sequence.io.template.GenbankHeaderFormatInterface,%20int)">GenbankWriter(OutputStream, Collection&lt;S&gt;, GenbankHeaderFormatInterface&lt;S, C&gt;, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankWriter.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriter</a></dt>
<dd>
<div class="block">Set custom lineLength</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">GenbankWriterHelper</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">The class that should be used to write out genbank file of a sequence
 collection</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#GenbankWriterHelper()">GenbankWriterHelper()</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriterHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#generateCompoundsToIndex()">generateCompoundsToIndex()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Returns what the value of a compound is in the backing bit storage i.e.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html#generateCompoundsToIndex()">generateCompoundsToIndex()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader.FourBitArrayWorker</a></dt>
<dd>
<div class="block">Returns a Map which encodes the contents of CompoundSet.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html#generateCompoundsToIndex()">generateCompoundsToIndex()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader.TwoBitArrayWorker</a></dt>
<dd>
<div class="block">Returns a Map which encodes TCAG into positions 0,1,2,3.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#generateIndexToCompounds()">generateIndexToCompounds()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Should return the inverse information that <a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#generateCompoundsToIndex()"><code>BitSequenceReader.BitArrayWorker.generateCompoundsToIndex()</code></a>
 returns i.e.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html#generateIndexToCompounds()">generateIndexToCompounds()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader.FourBitArrayWorker</a></dt>
<dd>
<div class="block">Returns a List which reverse encodes the Compound, Integer map</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html#generateIndexToCompounds()">generateIndexToCompounds()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader.TwoBitArrayWorker</a></dt>
<dd>
<div class="block">Returns a List which encodes TCAG into positions 0,1,2,3.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">GenericFastaHeaderFormat</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html" title="type parameter in GenericFastaHeaderFormat">S</a> extends <a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;?&gt;,<a href="./org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html" title="type parameter in GenericFastaHeaderFormat">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">We store the original header if the sequence is parsed from a fasta file and will use that exact
 sequence if we write out the sequences to a fasta file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html#GenericFastaHeaderFormat()">GenericFastaHeaderFormat()</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io">GenericFastaHeaderFormat</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">GenericFastaHeaderParser</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html" title="type parameter in GenericFastaHeaderParser">S</a> extends <a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="./org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html" title="type parameter in GenericFastaHeaderParser">C</a>&gt;,<a href="./org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html" title="type parameter in GenericFastaHeaderParser">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">The default fasta header parser where some headers are well defined based on the source
 database which allows us to set the source of the protein sequence and the identifier
 that can be used in future implementations to load features from external sources
 
 If the user has a custom header with local data then they can create their own implementation
 of a FastaHeaderParserInterface</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#GenericFastaHeaderParser()">GenericFastaHeaderParser()</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericFastaHeaderParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">GenericGenbankHeaderFormat</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html" title="type parameter in GenericGenbankHeaderFormat">S</a> extends <a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html" title="type parameter in GenericGenbankHeaderFormat">C</a>&gt;,<a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html" title="type parameter in GenericGenbankHeaderFormat">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#GenericGenbankHeaderFormat()">GenericGenbankHeaderFormat()</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderFormat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#GenericGenbankHeaderFormat(java.lang.String)">GenericGenbankHeaderFormat(String)</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderFormat</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">GenericGenbankHeaderParser</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="type parameter in GenericGenbankHeaderParser">S</a> extends <a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="type parameter in GenericGenbankHeaderParser">C</a>&gt;,<a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="type parameter in GenericGenbankHeaderParser">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#GenericGenbankHeaderParser()">GenericGenbankHeaderParser()</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">GenericInsdcHeaderFormat</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html" title="type parameter in GenericInsdcHeaderFormat">S</a> extends <a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="./org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html" title="type parameter in GenericInsdcHeaderFormat">C</a>&gt;,<a href="./org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html" title="type parameter in GenericInsdcHeaderFormat">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#GenericInsdcHeaderFormat()">GenericInsdcHeaderFormat()</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io">GenericInsdcHeaderFormat</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence"><span class="strong">GeneSequence</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/GeneSequence.html#GeneSequence(org.biojava.nbio.core.sequence.ChromosomeSequence,%20int,%20int,%20org.biojava.nbio.core.sequence.Strand)">GeneSequence(ChromosomeSequence, int, int, Strand)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">A class that keeps track of the details of a GeneSequence which is difficult to properly model.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/WindowedSequence.html#get(int)">get(int)</a></span> - Method in class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/WindowedSequence.html" title="class in org.biojava.nbio.core.sequence.views">WindowedSequence</a></dt>
<dd>
<div class="block">Returns the window specified at the given index in offsets i.e.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/SoftHashMap.html#get(java.lang.Object)">get(Object)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/SoftHashMap.html" title="class in org.biojava.nbio.core.util">SoftHashMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#getAccession()">getAccession()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#getAccession()">getAccession()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getAccession()">getAccession()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#getAccession()">getAccession()</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#getAccession()">getAccession()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#getAccession()">getAccession()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#getAccession()">getAccession()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#getAccession()">getAccession()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Unsupoorted</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getAccession()">getAccession()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/Accessioned.html#getAccession()">getAccession()</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/Accessioned.html" title="interface in org.biojava.nbio.core.sequence.template">Accessioned</a></dt>
<dd>
<div class="block">Returns the AccessionID this location is currently bound with</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getAccession()">getAccession()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getAccessions()">getAccessions()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">Pull uniprot accessions associated with this sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#getAlignedSequence(int)">getAlignedSequence(int)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava.nbio.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Uses bioIndex starting at 1 instead of 0</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.html#getAlignedSequence(int)">getAlignedSequence(int)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="interface in org.biojava.nbio.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns <a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> at given index.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#getAlignedSequences()">getAlignedSequences()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava.nbio.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Get the list of sequences</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.html#getAlignedSequences()">getAlignedSequences()</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="interface in org.biojava.nbio.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns a <a href="http://java.sun.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a> containing the individual <a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a>s of this alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#getAllCompounds()">getAllCompounds()</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#getAllCompounds()">getAllCompounds()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/CompoundSet.html#getAllCompounds()">getAllCompounds()</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Frame.html#getAllFrames()">getAllFrames()</a></span> - Static method in enum org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Frame.html" title="enum in org.biojava.nbio.core.sequence.transcription">Frame</a></dt>
<dd>
<div class="block">Delegates to <a href="./org/biojava/nbio/core/sequence/transcription/Frame.html#values()"><code>Frame.values()</code></a></div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#getAmbiguity(org.biojava.nbio.core.sequence.compound.NucleotideCompound...)">getAmbiguity(NucleotideCompound...)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractNucleotideCompoundSet</a></dt>
<dd>
<div class="block">Calculates the best symbol for a collection of compounds.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html#getAminoAcid()">getAminoAcid()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#getAminoAcidCompounds()">getAminoAcidCompounds()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#getAminoAcidCompoundSet()">getAminoAcidCompoundSet()</a></span> - Static method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getAnnotationType()">getAnnotationType()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#getAsList()">getAsList()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#getAsList()">getAsList()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getAsList()">getAsList()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#getAsList()">getAsList()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#getAsList()">getAsList()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Returns this Sequence store as a List</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#getAsList()">getAsList()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#getAsList()">getAsList()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Delegates to <a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#toList(org.biojava.nbio.core.sequence.template.Sequence)"><code>SequenceMixin.toList(org.biojava.nbio.core.sequence.template.Sequence)</code></a></div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getAsList()">getAsList()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/Sequence.html#getAsList()">getAsList()</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Returns the Sequence as a List of compounds</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getAsList()">getAsList()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/WindowedSequence.html#getBackingSequence()">getBackingSequence()</a></span> - Method in class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/WindowedSequence.html" title="class in org.biojava.nbio.core.sequence.views">WindowedSequence</a></dt>
<dd>
<div class="block">Access the sequence which backs this window</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html#getBase()">getBase()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getBioBegin()">getBioBegin()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getBioEnd()">getBioEnd()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getBioEnd()">getBioEnd()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceView.html#getBioEnd()">getBioEnd()</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceView.html" title="interface in org.biojava.nbio.core.sequence.template">SequenceView</a></dt>
<dd>
<div class="block">1-indexed, inclusive.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getBioStart()">getBioStart()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceView.html#getBioStart()">getBioStart()</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceView.html" title="interface in org.biojava.nbio.core.sequence.template">SequenceView</a></dt>
<dd>
<div class="block">1-indexed, inclusive.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#getBufferedReader()">getBufferedReader()</a></span> - Method in class org.biojava.nbio.core.sequence.io.util.<a href="./org/biojava/nbio/core/sequence/io/util/ClasspathResource.html" title="class in org.biojava.nbio.core.sequence.io.util">ClasspathResource</a></dt>
<dd>
<div class="block">Returns the reader representation of this classpath resource</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#getBytesRead()">getBytesRead()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava.nbio.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>
<div class="block">Keep track of bytesread via ReadLine to account for CR-LF in the stream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html#getCDSSequences()">getCDSSequences()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>
<div class="block">Get the CDS sequences that have been added to the TranscriptSequences</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html#getChildrenFeatures()">getChildrenFeatures()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">Get the children features</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#getChildrenFeatures()">getChildrenFeatures()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html#getChildrenFeatures()">getChildrenFeatures()</a></span> - Method in interface org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Get the features contained by this feature</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/ChromosomeSequence.html#getChromosomeNumber()">getChromosomeNumber()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/ChromosomeSequence.html" title="class in org.biojava.nbio.core.sequence">ChromosomeSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/SingleLinkageClusterer.html#getClusters(double)">getClusters(double)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/SingleLinkageClusterer.html" title="class in org.biojava.nbio.core.util">SingleLinkageClusterer</a></dt>
<dd>
<div class="block">Get the clusters by cutting the dendrogram at given cutoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/CDSSequence.html#getCodingSequence()">getCodingSequence()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/CDSSequence.html" title="class in org.biojava.nbio.core.sequence">CDSSequence</a></dt>
<dd>
<div class="block">A CDS sequence if negative stranded needs to be reverse complement
 to represent the actual coding sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html#getCodonCompoundSet(org.biojava.nbio.core.sequence.template.CompoundSet,%20org.biojava.nbio.core.sequence.template.CompoundSet)">getCodonCompoundSet(CompoundSet&lt;NucleotideCompound&gt;, CompoundSet&lt;AminoAcidCompound&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser.IUPACTable</a></dt>
<dd>
<div class="block">Returns the compound set of codons</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.html#getCodonCompoundSet(org.biojava.nbio.core.sequence.template.CompoundSet,%20org.biojava.nbio.core.sequence.template.CompoundSet)">getCodonCompoundSet(CompoundSet&lt;NucleotideCompound&gt;, CompoundSet&lt;AminoAcidCompound&gt;)</a></span> - Method in interface org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.html" title="interface in org.biojava.nbio.core.sequence.transcription">Table</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html#getCodons(org.biojava.nbio.core.sequence.template.CompoundSet,%20org.biojava.nbio.core.sequence.template.CompoundSet)">getCodons(CompoundSet&lt;NucleotideCompound&gt;, CompoundSet&lt;AminoAcidCompound&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser.IUPACTable</a></dt>
<dd>
<div class="block">Returns a list of codons where the source and target compounds
 are the same as those given by the parameters.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.html#getCodons(org.biojava.nbio.core.sequence.template.CompoundSet,%20org.biojava.nbio.core.sequence.template.CompoundSet)">getCodons(CompoundSet&lt;NucleotideCompound&gt;, CompoundSet&lt;AminoAcidCompound&gt;)</a></span> - Method in interface org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.html" title="interface in org.biojava.nbio.core.sequence.transcription">Table</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#getComplement()">getComplement()</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/DNASequence.html#getComplement()">getComplement()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Returns a Sequence which will complement every base</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/RNASequence.html#getComplement()">getComplement()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/RNASequence.html" title="class in org.biojava.nbio.core.sequence">RNASequence</a></dt>
<dd>
<div class="block">Get the complement view of the RNA sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/ComplementCompound.html#getComplement()">getComplement()</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/ComplementCompound.html" title="interface in org.biojava.nbio.core.sequence.template">ComplementCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#getComposition(org.biojava.nbio.core.sequence.template.Sequence)">getComposition(Sequence&lt;C&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Does a linear scan over the given Sequence and records the number of
 times each base appears.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Returns the compound at the specified biological index</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Returns the compound at the specified biological index</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Always returns the compound given at construction</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/Sequence.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Returns the Compound at the given biological index</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/ComplementSequenceView.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/ComplementSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">ComplementSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/ReversedSequenceView.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/ReversedSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">ReversedSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/RnaSequenceView.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/RnaSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#getCompoundForString(java.lang.String)">getCompoundForString(String)</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#getCompoundForString(java.lang.String)">getCompoundForString(String)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/CompoundSet.html#getCompoundForString(java.lang.String)">getCompoundForString(String)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundSet</a></dt>
<dd>
<div class="block">Return null if not recognised.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/WindowedSequence.html#getCompounds(int)">getCompounds(int)</a></span> - Method in class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/WindowedSequence.html" title="class in org.biojava.nbio.core.sequence.views">WindowedSequence</a></dt>
<dd>
<div class="block">For a given position into the windowed view this will return those
 compounds we can see in the window.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#getCompoundsAt(int)">getCompoundsAt(int)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava.nbio.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Get a list of compounds at a sequence position</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.html#getCompoundsAt(int)">getCompoundsAt(int)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="interface in org.biojava.nbio.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns the <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Compound</code></a> elements of the original <a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a>s at the given column.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava.nbio.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Get the Compounds defined in the first sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Returns the compound set backing this store</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Returns the compound set backing this store</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Returns the compound set given at construction</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.html#getCompoundSet()">getCompoundSet()</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="interface in org.biojava.nbio.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns <a href="./org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template"><code>CompoundSet</code></a> of all <a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a>s</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/Sequence.html#getCompoundSet()">getCompoundSet()</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Gets the compound set used to back this Sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#getCompoundsToIndexLookup()">getCompoundsToIndexLookup()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Returns a map which converts from compound to an integer representation</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#getCompoundType()">getCompoundType()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#getConsituents()">getConsituents()</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced with <a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#getConstituents()"><code>NucleotideCompound.getConstituents()</code></a> due to typographical error</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#getConstituents()">getConstituents()</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#getCreator()">getCreator()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#getDatabase()">getDatabase()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">DBReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/DatabaseReferenceInterface.html#getDatabaseReferences()">getDatabaseReferences()</a></span> - Method in interface org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/DatabaseReferenceInterface.html" title="interface in org.biojava.nbio.core.sequence.features">DatabaseReferenceInterface</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#getDatabaseReferences()">getDatabaseReferences()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#getDatabaseReferences()">getDatabaseReferences()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getDatabaseReferences()">getDatabaseReferences()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">The Uniprot mappings to other database identifiers for this sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getDatabaseReferences()">getDatabaseReferences()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/AccessionID.html#getDataSource()">getDataSource()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html#getDataSource()">getDataSource()</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html" title="class in org.biojava.nbio.core.sequence.location">InsdcParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/TaxonomyID.html#getDataSource()">getDataSource()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/TaxonomyID.html" title="class in org.biojava.nbio.core.sequence">TaxonomyID</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#getDefault()">getDefault()</a></span> - Static method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>
<div class="block">Default instance to use when Transcribing from DNA -&gt; RNA -&gt;
 Protein.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Frame.html#getDefaultFrame()">getDefaultFrame()</a></span> - Static method in enum org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Frame.html" title="enum in org.biojava.nbio.core.sequence.transcription">Frame</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/SingleLinkageClusterer.html#getDendrogram()">getDendrogram()</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/SingleLinkageClusterer.html" title="class in org.biojava.nbio.core.util">SingleLinkageClusterer</a></dt>
<dd>
<div class="block">Get the full dendrogram (size n-1) result of the hierarchical clustering</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html#getDescription()">getDescription()</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html#getDescription()">getDescription()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#getDescription()">getDescription()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html#getDescription()">getDescription()</a></span> - Method in interface org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Get the description that can be used to describe the feature</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html#getDescription()">getDescription()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getDescription()">getDescription()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/Compound.html#getDescription()">getDescription()</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#getDescription()">getDescription()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html#getDescription()">getDescription()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#getDistribution(org.biojava.nbio.core.sequence.template.Sequence)">getDistribution(Sequence&lt;C&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Analogous to <a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#getComposition(org.biojava.nbio.core.sequence.template.Sequence)"><code>SequenceMixin.getComposition(Sequence)</code></a> but returns the
 distribution of that <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Compound</code></a> over the given sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html#getDNACodingSequence()">getDNACodingSequence()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>
<div class="block">Get the stitched together CDS sequences then maps to the cDNA</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#getDnaCompounds()">getDnaCompounds()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundSet.html#getDNACompoundSet()">getDNACompoundSet()</a></span> - Static method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AmbiguityDNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/AmbiguityRNACompoundSet.html#getDNACompoundSet()">getDNACompoundSet()</a></span> - Static method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/AmbiguityRNACompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AmbiguityRNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/DNACompoundSet.html#getDNACompoundSet()">getDNACompoundSet()</a></span> - Static method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/DNACompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">DNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#getDnaRnaTranslator()">getDnaRnaTranslator()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/RnaSequenceView.html#getDnaToRna()">getDnaToRna()</a></span> - Method in class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/RnaSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/DNASequence.html#getDNAType()">getDNAType()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html#getEmptySequence(org.biojava.nbio.core.sequence.template.Sequence)">getEmptySequence(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>
<div class="block">Returns an empty sequence with the given compound set of the editing
 sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html#getEnd()">getEnd()</a></span> - Method in class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.Substitute.html#getEnd(org.biojava.nbio.core.sequence.template.Sequence)">getEnd(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.Substitute.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Substitute</a></dt>
<dd>
<div class="block">Must use this rather than the no-args getEnd as this can return
 -1 and the length of a sub is dependent on the length of the
 Sequence; we cannot assume 1:1 mapping between characters in a
 String and the number of compounds we will have to insert.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#getEnd()">getEnd()</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Location.html#getEnd()">getEnd()</a></span> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">End of the location</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#getEquivalentCompounds(org.biojava.nbio.core.sequence.compound.AminoAcidCompound)">getEquivalentCompounds(AminoAcidCompound)</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#getEquivalentCompounds(C)">getEquivalentCompounds(C)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/CompoundSet.html#getEquivalentCompounds(C)">getEquivalentCompounds(C)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/GeneSequence.html#getExonSequences()">getExonSequences()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Get the exons as an ArrayList</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getFeatureRetriever()">getFeatureRetriever()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureRetriever.html#getFeatures()">getFeatures()</a></span> - Method in interface org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureRetriever.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureRetriever</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#getFeatures(java.lang.String)">getFeatures(String)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#getFeatures()">getFeatures()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#getFeatures()">getFeatures()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getFeatures(java.lang.String,%20int)">getFeatures(String, int)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Return features at a sequence position by type</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getFeatures(int)">getFeatures(int)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Return features at a sequence position</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getFeatures()">getFeatures()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getFeaturesByType(java.lang.String)">getFeaturesByType(String)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getFeaturesKeyWord()">getFeaturesKeyWord()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html#getFivePrime(org.biojava.nbio.core.sequence.template.Sequence)">getFivePrime(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>
<div class="block">Should return the 5-prime end of the given Sequence according to
 the edit.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.Delete.html#getFivePrime(org.biojava.nbio.core.sequence.template.Sequence)">getFivePrime(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.Delete.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Delete</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.Insert.html#getFivePrime(org.biojava.nbio.core.sequence.template.Sequence)">getFivePrime(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.Insert.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Insert</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.Substitute.html#getFivePrime(org.biojava.nbio.core.sequence.template.Sequence)">getFivePrime(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.Substitute.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Substitute</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Frame.html#getForwardFrames()">getForwardFrames()</a></span> - Static method in enum org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Frame.html" title="enum in org.biojava.nbio.core.sequence.transcription">Frame</a></dt>
<dd>
<div class="block">Returns all frames in the forward orientation</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#getFromCompoundSet()">getFromCompoundSet()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/RNASequence.html#getGC()">getGC()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/RNASequence.html" title="class in org.biojava.nbio.core.sequence">RNASequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/DNASequence.html#getGCCount()">getGCCount()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Get the GC count in the DNA Sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html#getGCGChecksum(java.util.List)">getGCGChecksum(List&lt;S&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Calculates GCG checksum for entire list of sequences</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html#getGCGChecksum(S)">getGCGChecksum(S)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Calculates GCG checksum for a given sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html#getGCGHeader(java.util.List)">getGCGHeader(List&lt;S&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Assembles a GCG file header</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html#getGCGType(org.biojava.nbio.core.sequence.template.CompoundSet)">getGCGType(CompoundSet&lt;C&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Determines GCG type</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#getGenbankDirectoryCache()">getGenbankDirectoryCache()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a></dt>
<dd>
<div class="block">Local directory cache of Genbank that can be downloaded</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/ChromosomeSequence.html#getGene(java.lang.String)">getGene(String)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/ChromosomeSequence.html" title="class in org.biojava.nbio.core.sequence">ChromosomeSequence</a></dt>
<dd>
<div class="block">Get the gene based on accession.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getGeneName()">getGeneName()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">Get the gene name associated with this sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/ChromosomeSequence.html#getGeneSequences()">getGeneSequences()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/ChromosomeSequence.html" title="class in org.biojava.nbio.core.sequence">ChromosomeSequence</a></dt>
<dd>
<div class="block">Get the list of genes that have been added to the ChromosomeSequence where accession.toString is the key.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#getHeader()">getHeader()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html#getHeader(S)">getHeader(S)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io">GenericFastaHeaderFormat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#getHeader(S)">getHeader(S)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderFormat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/template/FastaHeaderFormatInterface.html#getHeader(S)">getHeader(S)</a></span> - Method in interface org.biojava.nbio.core.sequence.io.template.<a href="./org/biojava/nbio/core/sequence/io/template/FastaHeaderFormatInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template">FastaHeaderFormatInterface</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html#getHeader(S)">getHeader(S)</a></span> - Method in interface org.biojava.nbio.core.sequence.io.template.<a href="./org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template">GenbankHeaderFormatInterface</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#getHeader()">getHeader()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#getHeaderParser()">getHeaderParser()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/AccessionID.html#getID()">getID()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#getId()">getId()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">DBReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html#getId()">getId()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser.IUPACTable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/TaxonomyID.html#getID()">getID()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/TaxonomyID.html" title="class in org.biojava.nbio.core.sequence">TaxonomyID</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html#getIDFormat(java.util.List)">getIDFormat(List&lt;S&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Creates format String for accession IDs</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Returns the first occurrence of the given compound in this store; performs
 a linear search</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Returns 1 if the given compound is equal to the one given during
 construction; otherwise will return -1.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/Sequence.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Scans through the Sequence looking for the first occurrence of the given
 compound</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/ComplementSequenceView.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/ComplementSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">ComplementSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/RnaSequenceView.html#getIndexOf(org.biojava.nbio.core.sequence.compound.NucleotideCompound)">getIndexOf(NucleotideCompound)</a></span> - Method in class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/RnaSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#getIndexToCompoundsLookup()">getIndexToCompoundsLookup()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Returns a list of compounds the index position of which is used
 to translate from the byte representation into a compound.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#getInputStream()">getInputStream()</a></span> - Method in class org.biojava.nbio.core.sequence.io.util.<a href="./org/biojava/nbio/core/sequence/io/util/ClasspathResource.html" title="class in org.biojava.nbio.core.sequence.io.util">ClasspathResource</a></dt>
<dd>
<div class="block">Returns the InputStream instance of this classpath resource</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/FlatFileCache.html#getInputStream(java.lang.String)">getInputStream(String)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/FlatFileCache.html" title="class in org.biojava.nbio.core.util">FlatFileCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/InputStreamProvider.html#getInputStream(java.lang.String)">getInputStream(String)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/InputStreamProvider.html" title="class in org.biojava.nbio.core.util">InputStreamProvider</a></dt>
<dd>
<div class="block">get an InputStream for this file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/InputStreamProvider.html#getInputStream(java.net.URL)">getInputStream(URL)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/InputStreamProvider.html" title="class in org.biojava.nbio.core.util">InputStreamProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/InputStreamProvider.html#getInputStream(java.io.File)">getInputStream(File)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/InputStreamProvider.html" title="class in org.biojava.nbio.core.util">InputStreamProvider</a></dt>
<dd>
<div class="block">get an InputStream for the file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/IUPACParser.html#getInstance()">getInstance()</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/IUPACParser.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/FlatFileCache.html#getInstance()">getInstance()</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/FlatFileCache.html" title="class in org.biojava.nbio.core.util">FlatFileCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/GeneSequence.html#getIntronSequences()">getIntronSequences()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Get the introns as an ArrayList</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#getInverse()">getInverse()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#getInverse()">getInverse()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getInverse()">getInverse()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/RNASequence.html#getInverse()">getInverse()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/RNASequence.html" title="class in org.biojava.nbio.core.sequence">RNASequence</a></dt>
<dd>
<div class="block">Get the inverse view of the sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#getInverse()">getInverse()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#getInverse()">getInverse()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#getInverse()">getInverse()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#getInverse()">getInverse()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getInverse()">getInverse()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/Sequence.html#getInverse()">getInverse()</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Does the <em>right thing</em> to get the inverse of the current
 Sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getInverse()">getInverse()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeaturesKeyWordInterface.html#getKeyWords()">getKeyWords()</a></span> - Method in interface org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeaturesKeyWordInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeaturesKeyWordInterface</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#getKeyWords()">getKeyWords()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#getKeyWords()">getKeyWords()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getKeyWords()">getKeyWords()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">Pull UniProt key words which is a mixed bag of words associated with this sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Returns the last occurrence of the given compound in this store; performs
 a linear search</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Returns the length of the Sequence if the given compound was equal to
 the one given during construction.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/Sequence.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Scans through the Sequence looking for the last occurrence of the given
 compound</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/ComplementSequenceView.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/ComplementSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">ComplementSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/RnaSequenceView.html#getLastIndexOf(org.biojava.nbio.core.sequence.compound.NucleotideCompound)">getLastIndexOf(NucleotideCompound)</a></span> - Method in class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/RnaSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/CDSSequence.html#getLength()">getLength()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/CDSSequence.html" title="class in org.biojava.nbio.core.sequence">CDSSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/ExonSequence.html#getLength()">getLength()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/ExonSequence.html" title="class in org.biojava.nbio.core.sequence">ExonSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/GeneSequence.html#getLength()">getLength()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/IntronSequence.html#getLength()">getLength()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/IntronSequence.html" title="class in org.biojava.nbio.core.sequence">IntronSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#getLength()">getLength()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#getLength()">getLength()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getLength()">getLength()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">The sequence length</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#getLength()">getLength()</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Location.html#getLength()">getLength()</a></span> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Returns the length of the outer bounds of this location</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#getLength()">getLength()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava.nbio.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Get the length of the MSA where it is assumed that
 all sequence position</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/StartCodonSequence.html#getLength()">getLength()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/StartCodonSequence.html" title="class in org.biojava.nbio.core.sequence">StartCodonSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/StopCodonSequence.html#getLength()">getLength()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/StopCodonSequence.html" title="class in org.biojava.nbio.core.sequence">StopCodonSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#getLength()">getLength()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#getLength()">getLength()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#getLength()">getLength()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Returns the length of the sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#getLength()">getLength()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#getLength()">getLength()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Returns the length given during construction</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getLength()">getLength()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.html#getLength()">getLength()</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="interface in org.biojava.nbio.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns the number of columns in the alignment profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/Sequence.html#getLength()">getLength()</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Returns the length of the Sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getLength()">getLength()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html#getLength()">getLength()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/WindowedSequence.html#getLength()">getLength()</a></span> - Method in class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/WindowedSequence.html" title="class in org.biojava.nbio.core.sequence.views">WindowedSequence</a></dt>
<dd>
<div class="block">Returns the size of the windowed sequence which is the length by the
 window size.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#getLineLength()">getLineLength()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaGeneWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaGeneWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaWriter.html#getLineLength()">getLineLength()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankWriter.html#getLineLength()">getLineLength()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankWriter.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#getList()">getList()</a></span> - Method in class org.biojava.nbio.core.sequence.io.util.<a href="./org/biojava/nbio/core/sequence/io/util/ClasspathResource.html" title="class in org.biojava.nbio.core.sequence.io.util">ClasspathResource</a></dt>
<dd>
<div class="block">Returns this resource as a list of Strings</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html#getList(java.io.BufferedReader)">getList(BufferedReader)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Returns the contents of a buffered reader as a list of strings</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html#getList(java.io.InputStream)">getList(InputStream)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Delegates to <a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html#getList(java.io.BufferedReader)"><code>IOUtils.getList(BufferedReader)</code></a> by wrapping the InputStream
 in a valid reader.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html#getList(java.io.File)">getList(File)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Delegates to <a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html#getList(java.io.InputStream)"><code>IOUtils.getList(InputStream)</code></a> by wrapping the File
 in a valid stream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html#getLocations()">getLocations()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">A feature could be a single sequence position like a mutation or a post translational modification of an amino acid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#getLocations()">getLocations()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html#getLocations()">getLocations()</a></span> - Method in interface org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">The location(s) of this feature where the location should contain a reference to parent and sequence etc.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html#getLongName()">getLongName()</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html#getLongName()">getLongName()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/Compound.html#getLongName()">getLongName()</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#getLongName()">getLongName()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html#getLongName()">getLongName()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html#getMax()">getMax()</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html" title="class in org.biojava.nbio.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html#getMax(java.util.List)">getMax(List&lt;Location&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html" title="class in org.biojava.nbio.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Scans through a list of locations to find the Location with the
 highest end</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Location.Tools.html#getMax(java.util.List)">getMax(List&lt;Location&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Location.Tools.html" title="class in org.biojava.nbio.core.sequence.location.template">Location.Tools</a></dt>
<dd>
<div class="block">Scans through a list of locations to find the Location with the
 highest end</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#getMaxSingleCompoundStringLength()">getMaxSingleCompoundStringLength()</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#getMaxSingleCompoundStringLength()">getMaxSingleCompoundStringLength()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/CompoundSet.html#getMaxSingleCompoundStringLength()">getMaxSingleCompoundStringLength()</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundSet</a></dt>
<dd>
<div class="block">Returns the maximum size of a compound String this set holds</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html#getMin()">getMin()</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html" title="class in org.biojava.nbio.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html#getMin(java.util.List)">getMin(List&lt;Location&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html" title="class in org.biojava.nbio.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Scans through a list of locations to find the Location with the
 lowest start</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Location.Tools.html#getMin(java.util.List)">getMin(List&lt;Location&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Location.Tools.html" title="class in org.biojava.nbio.core.sequence.location.template">Location.Tools</a></dt>
<dd>
<div class="block">Scans through a list of locations to find the Location with the
 lowest start</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html#getMolecularWeight()">getMolecularWeight()</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html#getMolecularWeight()">getMolecularWeight()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/Compound.html#getMolecularWeight()">getMolecularWeight()</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#getMolecularWeight()">getMolecularWeight()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html#getMolecularWeight()">getMolecularWeight()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/Qualifier.html#getName()">getName()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/Qualifier.html" title="class in org.biojava.nbio.core.sequence.features">Qualifier</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html#getName()">getName()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser.IUPACTable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/XMLHelper.html#getNewDocument()">getNewDocument()</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/XMLHelper.html" title="class in org.biojava.nbio.core.util">XMLHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getNotesList()">getNotesList()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/Strand.html#getNumericRepresentation()">getNumericRepresentation()</a></span> - Method in enum org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/Strand.html" title="enum in org.biojava.nbio.core.sequence">Strand</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html#getOne()">getOne()</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#getOne()">getOne()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html#getOne()">getOne()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getOrganismName()">getOrganismName()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">Get the organism name assigned to this sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getOriginalHeader()">getOriginalHeader()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/GeneSequence.html#getParentChromosomeSequence()">getParentChromosomeSequence()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">The parent ChromosomeSequence which contains the actual DNA sequence data</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html#getParentFeature()">getParentFeature()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">Get the parent Feature</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#getParentFeature()">getParentFeature()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html#getParentFeature()">getParentFeature()</a></span> - Method in interface org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Get the parent feature</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getParentSequence()">getParentSequence()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html#getPDBCharacter(boolean,%20char,%20char,%20boolean,%20char)">getPDBCharacter(boolean, char, char, boolean, char)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Creates formatted String for a single character of PDB output</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html#getPDBConservation(boolean,%20char,%20char,%20boolean)">getPDBConservation(boolean, char, char, boolean)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Creates formatted String for displaying conservation in PDB output</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html#getPDBLegend()">getPDBLegend()</a></span> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Creates formatted String for displaying conservation legend in PDB output</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/CDSSequence.html#getPhase()">getPhase()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/CDSSequence.html" title="class in org.biojava.nbio.core.sequence">CDSSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html#getPosition()">getPosition()</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html" title="class in org.biojava.nbio.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html#getPosition()">getPosition()</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Point.html#getPosition()">getPosition()</a></span> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Point.html" title="interface in org.biojava.nbio.core.sequence.location.template">Point</a></dt>
<dd>
<div class="block">Returns the position held by this object</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#getProperties()">getProperties()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">DBReferenceInfo</a></dt>
<dd>
<div class="block">Get the properties</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html#getProteinCDSSequences()">getProteinCDSSequences()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>
<div class="block">Return a list of protein sequences based on each CDS sequence
 where the phase shift between two CDS sequences is assigned to the
 CDS sequence that starts the triplet.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/RNASequence.html#getProteinSequence()">getProteinSequence()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/RNASequence.html" title="class in org.biojava.nbio.core.sequence">RNASequence</a></dt>
<dd>
<div class="block">Get the ProteinSequence from the RNA sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/RNASequence.html#getProteinSequence(org.biojava.nbio.core.sequence.transcription.TranscriptionEngine)">getProteinSequence(TranscriptionEngine)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/RNASequence.html" title="class in org.biojava.nbio.core.sequence">RNASequence</a></dt>
<dd>
<div class="block">Get the ProteinSequene from the RNA sequence with user defined
 transcription engine</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html#getProteinSequence()">getProteinSequence()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>
<div class="block">Get the protein sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html#getProteinSequence(org.biojava.nbio.core.sequence.transcription.TranscriptionEngine)">getProteinSequence(TranscriptionEngine)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>
<div class="block">Get the protein sequence with user defined TranscriptEngine</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#getProteinSequenceCreator()">getProteinSequenceCreator()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AccesionedLocation.html#getProxySequenceReader()">getProxySequenceReader()</a></span> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AccesionedLocation.html" title="interface in org.biojava.nbio.core.sequence.location.template">AccesionedLocation</a></dt>
<dd>
<div class="block">Return the proxy reader used to get sequence for this location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getProxySequenceReader()">getProxySequenceReader()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html#getQualifiers()">getQualifiers()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#getQualifiers()">getQualifiers()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html#getQualifiers()">getQualifiers()</a></span> - Method in interface org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Get the qualifiers for this feature</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/QualityFeature.html#getQualities()">getQualities()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/QualityFeature.html" title="class in org.biojava.nbio.core.sequence.features">QualityFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/QualityFeature.html#getQualities(int,%20int)">getQualities(int, int)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/QualityFeature.html" title="class in org.biojava.nbio.core.sequence.features">QualityFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/QualityFeature.html#getQualityAt(int)">getQualityAt(int)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/QualityFeature.html" title="class in org.biojava.nbio.core.sequence.features">QualityFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/QuantityFeature.html#getQuantities()">getQuantities()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/QuantityFeature.html" title="class in org.biojava.nbio.core.sequence.features">QuantityFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.Delete.html#getRealEnd()">getRealEnd()</a></span> - Method in class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.Delete.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Delete</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.Delete.html#getRealStart()">getRealStart()</a></span> - Method in class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.Delete.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Delete</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#getRelevantSubLocations()">getRelevantSubLocations()</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>
<div class="block">Returns the normalised list of sub locations i.e.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Location.html#getRelevantSubLocations()">getRelevantSubLocations()</a></span> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">An extension to <a href="./org/biojava/nbio/core/sequence/location/template/Location.html#getSubLocations()"><code>Location.getSubLocations()</code></a> which returns sub-locations
 of sub-locations; this will continue until it runs out of those locations.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#getRelevantSubSequence(org.biojava.nbio.core.sequence.template.Sequence)">getRelevantSubSequence(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Location.html#getRelevantSubSequence(org.biojava.nbio.core.sequence.template.Sequence)">getRelevantSubSequence(Sequence&lt;C&gt;)</a></span> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Will return a SequenceReader object which offers a view of all resolved
 locations i.e.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html#getResolver()">getResolver()</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html" title="class in org.biojava.nbio.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/DNASequence.html#getReverse()">getReverse()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Returns a Sequence which runs in the current reverse order</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/Strand.html#getReverse()">getReverse()</a></span> - Method in enum org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/Strand.html" title="enum in org.biojava.nbio.core.sequence">Strand</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/DNASequence.html#getReverseComplement()">getReverseComplement()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Delegates to <a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getInverse()"><code>AbstractSequence.getInverse()</code></a> for the reverse complement</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/RNASequence.html#getReverseComplement()">getReverseComplement()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/RNASequence.html" title="class in org.biojava.nbio.core.sequence">RNASequence</a></dt>
<dd>
<div class="block">Get reverse complement view of the sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Frame.html#getReverseFrames()">getReverseFrames()</a></span> - Static method in enum org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Frame.html" title="enum in org.biojava.nbio.core.sequence.transcription">Frame</a></dt>
<dd>
<div class="block">Returns all frames which are in the reverse orientation</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#getRnaAminoAcidTranslator()">getRnaAminoAcidTranslator()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#getRnaCompounds()">getRnaCompounds()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/RNACompoundSet.html#getRNACompoundSet()">getRNACompoundSet()</a></span> - Static method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/RNACompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">RNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/DNASequence.html#getRNASequence()">getRNASequence()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Return the RNASequence equivalent of the DNASequence using default Transcription Engine.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/DNASequence.html#getRNASequence(org.biojava.nbio.core.sequence.transcription.TranscriptionEngine)">getRNASequence(TranscriptionEngine)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Allow a user to pass in a rules engine to do the DNA to RNA translation</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/DNASequence.html#getRNASequence(org.biojava.nbio.core.sequence.transcription.Frame)">getRNASequence(Frame)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Allows the user to pass in the Frame shift.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/DNASequence.html#getRNASequence(org.biojava.nbio.core.sequence.transcription.TranscriptionEngine,%20org.biojava.nbio.core.sequence.transcription.Frame)">getRNASequence(TranscriptionEngine, Frame)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#getRnaSequenceCreator()">getRnaSequenceCreator()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/RnaSequenceView.html#getRnaToDna()">getRnaToDna()</a></span> - Method in class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/RnaSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#getSequence(org.biojava.nbio.core.sequence.template.ProxySequenceReader,%20long)">getSequence(ProxySequenceReader&lt;AminoAcidCompound&gt;, long)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">CasePreservingProteinSequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#getSequence(java.lang.String,%20long)">getSequence(String, long)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">CasePreservingProteinSequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#getSequence(java.util.List)">getSequence(List&lt;AminoAcidCompound&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">CasePreservingProteinSequenceCreator</a></dt>
<dd>
<div class="block">Assumes all compounds were uppercase</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#getSequence(java.lang.String,%20long)">getSequence(String, long)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/DNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">DNASequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#getSequence(org.biojava.nbio.core.sequence.template.ProxySequenceReader,%20long)">getSequence(ProxySequenceReader&lt;NucleotideCompound&gt;, long)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/DNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">DNASequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#getSequence(java.util.List)">getSequence(List&lt;NucleotideCompound&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/DNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">DNASequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaSequenceParser.html#getSequence(java.io.BufferedReader,%20int)">getSequence(BufferedReader, int)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">FastaSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#getSequence(java.lang.String,%20long)">getSequence(String, long)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">FileProxyDNASequenceCreator</a></dt>
<dd>
<div class="block">Even though we are passing in the sequence we really only care about the length of the sequence and the offset
 index in the fasta file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#getSequence(org.biojava.nbio.core.sequence.template.ProxySequenceReader,%20long)">getSequence(ProxySequenceReader&lt;NucleotideCompound&gt;, long)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">FileProxyDNASequenceCreator</a></dt>
<dd>
<div class="block">Should be able to extend the same concept to a remote URL call or database connection.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#getSequence(java.util.List)">getSequence(List&lt;NucleotideCompound&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">FileProxyDNASequenceCreator</a></dt>
<dd>
<div class="block">Not sure of use case and currently not supported</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#getSequence(java.lang.String,%20long)">getSequence(String, long)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">FileProxyProteinSequenceCreator</a></dt>
<dd>
<div class="block">Even though we are passing in the sequence we really only care about the length of the sequence and the offset
 index in the fasta file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#getSequence(org.biojava.nbio.core.sequence.template.ProxySequenceReader,%20long)">getSequence(ProxySequenceReader&lt;AminoAcidCompound&gt;, long)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">FileProxyProteinSequenceCreator</a></dt>
<dd>
<div class="block">Should be able to extend the same concept to a remote URL call or database connection.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#getSequence(java.util.List)">getSequence(List&lt;AminoAcidCompound&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">FileProxyProteinSequenceCreator</a></dt>
<dd>
<div class="block">Not sure of use case and currently not supported</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#getSequence(java.io.BufferedReader,%20int)">getSequence(BufferedReader, int)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#getSequence(java.lang.String,%20long)">getSequence(String, long)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">ProteinSequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#getSequence(java.util.List)">getSequence(List&lt;AminoAcidCompound&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">ProteinSequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#getSequence(org.biojava.nbio.core.sequence.template.ProxySequenceReader,%20long)">getSequence(ProxySequenceReader&lt;AminoAcidCompound&gt;, long)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">ProteinSequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#getSequence(java.lang.String,%20long)">getSequence(String, long)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/RNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">RNASequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#getSequence(org.biojava.nbio.core.sequence.template.ProxySequenceReader,%20long)">getSequence(ProxySequenceReader&lt;NucleotideCompound&gt;, long)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/RNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">RNASequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#getSequence(java.util.List)">getSequence(List&lt;NucleotideCompound&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/RNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">RNASequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#getSequence(java.lang.String,%20long)">getSequence(String, long)</a></span> - Method in interface org.biojava.nbio.core.sequence.io.template.<a href="./org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template">SequenceCreatorInterface</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#getSequence(org.biojava.nbio.core.sequence.template.ProxySequenceReader,%20long)">getSequence(ProxySequenceReader&lt;C&gt;, long)</a></span> - Method in interface org.biojava.nbio.core.sequence.io.template.<a href="./org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template">SequenceCreatorInterface</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#getSequence(java.util.List)">getSequence(List&lt;C&gt;)</a></span> - Method in interface org.biojava.nbio.core.sequence.io.template.<a href="./org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template">SequenceCreatorInterface</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/template/SequenceParserInterface.html#getSequence(java.io.BufferedReader,%20int)">getSequence(BufferedReader, int)</a></span> - Method in interface org.biojava.nbio.core.sequence.io.template.<a href="./org/biojava/nbio/core/sequence/io/template/SequenceParserInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template">SequenceParserInterface</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SequenceLocation.html#getSequence()">getSequence()</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SequenceLocation.html" title="class in org.biojava.nbio.core.sequence.location">SequenceLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/GeneSequence.html#getSequence5PrimeTo3Prime()">getSequence5PrimeTo3Prime()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Try to give method clarity where you want a DNASequence coding in the 5' to 3' direction
 Returns the DNASequence representative of the 5' and 3' reading based on strand</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#getSequenceAsString(java.lang.Integer,%20java.lang.Integer,%20org.biojava.nbio.core.sequence.Strand)">getSequenceAsString(Integer, Integer, Strand)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#getSequenceAsString(java.lang.Integer,%20java.lang.Integer,%20org.biojava.nbio.core.sequence.Strand)">getSequenceAsString(Integer, Integer, Strand)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getSequenceAsString(java.lang.Integer,%20java.lang.Integer,%20org.biojava.nbio.core.sequence.Strand)">getSequenceAsString(Integer, Integer, Strand)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#getSequenceAsString(java.lang.Integer,%20java.lang.Integer,%20org.biojava.nbio.core.sequence.Strand)">getSequenceAsString(Integer, Integer, Strand)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Returns the sequence as a String</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#getSequenceAsString(java.util.List,%20org.biojava.nbio.core.sequence.template.CompoundSet,%20java.lang.Integer,%20java.lang.Integer,%20org.biojava.nbio.core.sequence.Strand)">getSequenceAsString(List&lt;C&gt;, CompoundSet&lt;C&gt;, Integer, Integer, Strand)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html" title="class in org.biojava.nbio.core.sequence.storage">SequenceAsStringHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Delegates to <a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#toList(org.biojava.nbio.core.sequence.template.Sequence)"><code>SequenceMixin.toList(org.biojava.nbio.core.sequence.template.Sequence)</code></a></div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getSequenceAsString(java.lang.Integer,%20java.lang.Integer,%20org.biojava.nbio.core.sequence.Strand)">getSequenceAsString(Integer, Integer, Strand)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Default case is to assume strand is positive because only CDSSequence can be either positive or negative Strand.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/Sequence.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Returns the String representation of the Sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/ComplementSequenceView.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/ComplementSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">ComplementSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/ReversedSequenceView.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/ReversedSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">ReversedSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/RnaSequenceView.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/RnaSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#getSequenceCollection()">getSequenceCollection()</a></span> - Static method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/SequenceOptimizationHints.html" title="class in org.biojava.nbio.core.sequence">SequenceOptimizationHints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/SequenceTools.html#getSequenceFromString(java.lang.String)">getSequenceFromString(String)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/SequenceTools.html" title="class in org.biojava.nbio.core.util">SequenceTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#getSequenceHeaderParser()">getSequenceHeaderParser()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getSequenceScore()">getSequenceScore()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Provide place holder for a metric that indicate a score associated with the sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#getSequenceUsage()">getSequenceUsage()</a></span> - Static method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/SequenceOptimizationHints.html" title="class in org.biojava.nbio.core.sequence">SequenceOptimizationHints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html#getShortDescription()">getShortDescription()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#getShortDescription()">getShortDescription()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html#getShortDescription()">getShortDescription()</a></span> - Method in interface org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Get the short description that can be used to describe the feature</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html#getShortName()">getShortName()</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#getShortName()">getShortName()</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html#getShortName()">getShortName()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/Compound.html#getShortName()">getShortName()</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#getShortName()">getShortName()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html#getShortName()">getShortName()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#getSize()">getSize()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava.nbio.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Get the number of sequences in the MSA</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.html#getSize()">getSize()</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="interface in org.biojava.nbio.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns the number of rows in this profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html#getSource()">getSource()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">The feature source</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#getSource()">getSource()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html#getSource()">getSource()</a></span> - Method in interface org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">The source of the feature.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getSource()">getSource()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Added support for the source of this sequence for GFF3 export
 If a sub sequence doesn't have  source then check for parent source</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html#getStart()">getStart()</a></span> - Method in class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#getStart()">getStart()</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Location.html#getStart()">getStart()</a></span> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Start of the location</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html#getStartCodonSequence()">getStartCodonSequence()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html#getStopCodonSequence()">getStopCodonSequence()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/CDSSequence.html#getStrand()">getStrand()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/CDSSequence.html" title="class in org.biojava.nbio.core.sequence">CDSSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/GeneSequence.html#getStrand()">getStrand()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">A gene should have Strand</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#getStrand()">getStrand()</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Location.html#getStrand()">getStrand()</a></span> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Strand which the location is located on</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html#getStrand()">getStrand()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#getStringForCompound(org.biojava.nbio.core.sequence.compound.AminoAcidCompound)">getStringForCompound(AminoAcidCompound)</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#getStringForCompound(C)">getStringForCompound(C)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/CompoundSet.html#getStringForCompound(C)">getStringForCompound(C)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/Strand.html#getStringRepresentation()">getStringRepresentation()</a></span> - Method in enum org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/Strand.html" title="enum in org.biojava.nbio.core.sequence">Strand</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#getSubLocations()">getSubLocations()</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Location.html#getSubLocations()">getSubLocations()</a></span> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Gives access to the sub locations for this location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#getSubSequence(java.lang.Integer,%20java.lang.Integer)">getSubSequence(Integer, Integer)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#getSubSequence(java.lang.Integer,%20java.lang.Integer)">getSubSequence(Integer, Integer)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getSubSequence(java.lang.Integer,%20java.lang.Integer)">getSubSequence(Integer, Integer)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#getSubSequence(org.biojava.nbio.core.sequence.template.Sequence)">getSubSequence(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>
<div class="block">If circular this will return the sequence represented by the sub
 locations joined.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Location.html#getSubSequence(org.biojava.nbio.core.sequence.template.Sequence)">getSubSequence(Sequence&lt;C&gt;)</a></span> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Will return a SequenceReader object which represents the outer bounds
 of this Location</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#getSubSequence(java.lang.Integer,%20java.lang.Integer)">getSubSequence(Integer, Integer)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#getSubSequence(int,%20int)">getSubSequence(int, int)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Returns a sub sequence view</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#getSubSequence(java.lang.Integer,%20java.lang.Integer)">getSubSequence(Integer, Integer)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#getSubSequence(java.lang.Integer,%20java.lang.Integer)">getSubSequence(Integer, Integer)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#getSubSequence(java.lang.Integer,%20java.lang.Integer)">getSubSequence(Integer, Integer)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Creates a <a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template"><code>SequenceProxyView</code></a> for the given coordinates</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getSubSequence(java.lang.Integer,%20java.lang.Integer)">getSubSequence(Integer, Integer)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/Sequence.html#getSubSequence(java.lang.Integer,%20java.lang.Integer)">getSubSequence(Integer, Integer)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Returns a portion of the sequence from the different positions.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getSubSequence(java.lang.Integer,%20java.lang.Integer)">getSubSequence(Integer, Integer)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/IUPACParser.html#getTable(java.lang.String)">getTable(String)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/IUPACParser.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser</a></dt>
<dd>
<div class="block">Returns a table by its name</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/IUPACParser.html#getTable(java.lang.Integer)">getTable(Integer)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/IUPACParser.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser</a></dt>
<dd>
<div class="block">Returns a table by its identifier i.e.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#getTable()">getTable()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/IUPACParser.html#getTables()">getTables()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/IUPACParser.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser</a></dt>
<dd>
<div class="block">Returns a list of all available IUPAC tables</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html#getTargetSequence(org.biojava.nbio.core.sequence.template.Sequence)">getTargetSequence(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>
<div class="block">Returns the Sequence which is our edit.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getTaxonomy()">getTaxonomy()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/ConcurrencyTools.html#getThreadPool()">getThreadPool()</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/ConcurrencyTools.html" title="class in org.biojava.nbio.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Returns current shared thread pool.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html#getThree()">getThree()</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#getThree()">getThree()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html#getThree()">getThree()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html#getThreePrime(org.biojava.nbio.core.sequence.template.Sequence)">getThreePrime(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>
<div class="block">Should return the 3-prime end of the given Sequence according to
 the edit.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.Delete.html#getThreePrime(org.biojava.nbio.core.sequence.template.Sequence)">getThreePrime(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.Delete.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Delete</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.Insert.html#getThreePrime(org.biojava.nbio.core.sequence.template.Sequence)">getThreePrime(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.Insert.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Insert</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.Substitute.html#getThreePrime(org.biojava.nbio.core.sequence.template.Sequence)">getThreePrime(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.Substitute.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Substitute</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#getToCompoundSet()">getToCompoundSet()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/GeneSequence.html#getTranscript(java.lang.String)">getTranscript(String)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Get the transcript sequence by accession</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/GeneSequence.html#getTranscripts()">getTranscripts()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Get the collection of transcription sequences assigned to this gene</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html#getTriplet()">getTriplet()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html#getTwo()">getTwo()</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#getTwo()">getTwo()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html#getTwo()">getTwo()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html#getType()">getType()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">The feature type</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#getType()">getType()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html#getType()">getType()</a></span> - Method in interface org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">The type of the feature.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#getUnderlyingCompound()">getUnderlyingCompound()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getUniprotbaseURL()">getUniprotbaseURL()</a></span> - Static method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">The current UniProt URL to deal with caching issues.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getUniprotDirectoryCache()">getUniprotDirectoryCache()</a></span> - Static method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">Local directory cache of XML that can be downloaded</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html#getUpperedBase()">getUpperedBase()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#getUserCollection()">getUserCollection()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Provided for convince if the developer needs to associate data with a sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html#getUserObject()">getUserObject()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#getUserObject()">getUserObject()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html#getUserObject()">getUserObject()</a></span> - Method in interface org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/Qualifier.html#getValue()">getValue()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/Qualifier.html" title="class in org.biojava.nbio.core.sequence.features">Qualifier</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/CRC64Checksum.html#getValue()">getValue()</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/CRC64Checksum.html" title="class in org.biojava.nbio.core.util">CRC64Checksum</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getViewedSequence()">getViewedSequence()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceView.html#getViewedSequence()">getViewedSequence()</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceView.html" title="interface in org.biojava.nbio.core.sequence.template">SequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/WindowedSequence.html#getWindowSize()">getWindowSize()</a></span> - Method in class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/WindowedSequence.html" title="class in org.biojava.nbio.core.sequence.views">WindowedSequence</a></dt>
<dd>
<div class="block">Access the current window size</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/InputStreamProvider.html#GZIP_MAGIC">GZIP_MAGIC</a></span> - Static variable in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/InputStreamProvider.html" title="class in org.biojava.nbio.core.util">InputStreamProvider</a></dt>
<dd>
<div class="block">The magic number found at the start of a GZIP stream.</div>
</dd>
</dl>
<a name="_H_">
<!--   -->
</a>
<h2 class="title">H</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#hasCompound(org.biojava.nbio.core.sequence.compound.AminoAcidCompound)">hasCompound(AminoAcidCompound)</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#hasCompound(C)">hasCompound(C)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/CompoundSet.html#hasCompound(C)">hasCompound(C)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/Hashcoder.html#hash(int,%20boolean)">hash(int, boolean)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/Hashcoder.html" title="class in org.biojava.nbio.core.util">Hashcoder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/Hashcoder.html#hash(int,%20char)">hash(int, char)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/Hashcoder.html" title="class in org.biojava.nbio.core.util">Hashcoder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/Hashcoder.html#hash(int,%20int)">hash(int, int)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/Hashcoder.html" title="class in org.biojava.nbio.core.util">Hashcoder</a></dt>
<dd>
<div class="block">Used for ints, bytes and shorts</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/Hashcoder.html#hash(int,%20long)">hash(int, long)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/Hashcoder.html" title="class in org.biojava.nbio.core.util">Hashcoder</a></dt>
<dd>
<div class="block">long support done by shifting by 32 (using unsigned shift)</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/Hashcoder.html#hash(int,%20float)">hash(int, float)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/Hashcoder.html" title="class in org.biojava.nbio.core.util">Hashcoder</a></dt>
<dd>
<div class="block">float support done via <a href="http://java.sun.com/javase/6/docs/api/java/lang/Float.html?is-external=true#floatToIntBits(float)" title="class or interface in java.lang"><code>Float.floatToIntBits(float)</code></a> which allows
 the encoding of a float as an int.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/Hashcoder.html#hash(int,%20double)">hash(int, double)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/Hashcoder.html" title="class in org.biojava.nbio.core.util">Hashcoder</a></dt>
<dd>
<div class="block">double support which is done using the same techinque as float hashing
 except we convert to a long not to an int.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/Hashcoder.html#hash(int,%20java.lang.Object)">hash(int, Object)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/Hashcoder.html" title="class in org.biojava.nbio.core.util">Hashcoder</a></dt>
<dd>
<div class="block"><code>o</code> is a possibly-null object field, and possibly an
 array.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/AccessionID.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html" title="class in org.biojava.nbio.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/util/Hashcoder.html" title="class in org.biojava.nbio.core.util"><span class="strong">Hashcoder</span></a> - Class in <a href="./org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>
<div class="block">Contains helper methods for generating a HashCode without having to resort to
 the commons lang hashcode builders.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/Hashcoder.html#Hashcoder()">Hashcoder()</a></span> - Constructor for class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/Hashcoder.html" title="class in org.biojava.nbio.core.util">Hashcoder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.SequenceIterator.html#hasNext()">hasNext()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.SequenceIterator.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin.SequenceIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#headerLine">headerLine</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_I_">
<!--   -->
</a>
<h2 class="title">I</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#indexOf(org.biojava.nbio.core.sequence.template.Sequence,%20C)">indexOf(Sequence&lt;C&gt;, C)</a></span> - Static method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Performs a linear search of the given Sequence for the given compound.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#initMet(boolean)">initMet(boolean)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/util/InputStreamProvider.html" title="class in org.biojava.nbio.core.util"><span class="strong">InputStreamProvider</span></a> - Class in <a href="./org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>
<div class="block">A class that provides an InputStream from a File.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/InputStreamProvider.html#InputStreamProvider()">InputStreamProvider()</a></span> - Constructor for class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/InputStreamProvider.html" title="class in org.biojava.nbio.core.util">InputStreamProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/XMLHelper.html#inputStreamToDocument(java.io.InputStream)">inputStreamToDocument(InputStream)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/XMLHelper.html" title="class in org.biojava.nbio.core.util">XMLHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.html" title="class in org.biojava.nbio.core.sequence.location"><span class="strong">InsdcLocations</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a></dt>
<dd>
<div class="block">A collection of locations which are used whenever we work with INSDC; some
 of which could be deprecated (from INSDC's point of view) yet appear
 in records.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.html#InsdcLocations()">InsdcLocations()</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.BondLocation.html" title="class in org.biojava.nbio.core.sequence.location"><span class="strong">InsdcLocations.BondLocation</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a></dt>
<dd>
<div class="block">Used to represent bond locations equivalent to bond(7,8) or bond(7).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.BondLocation.html#InsdcLocations.BondLocation(org.biojava.nbio.core.sequence.location.template.Location...)">InsdcLocations.BondLocation(Location...)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.BondLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.BondLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.BondLocation.html#InsdcLocations.BondLocation(java.util.List)">InsdcLocations.BondLocation(List&lt;Location&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.BondLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.BondLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava.nbio.core.sequence.location"><span class="strong">InsdcLocations.GroupLocation</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a></dt>
<dd>
<div class="block">Deprecated in INSDC yet still appears; equivalent to the order()
 directive except no 5' to 3' ordering is defined.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html#InsdcLocations.GroupLocation(java.util.List)">InsdcLocations.GroupLocation(List)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.GroupLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html#InsdcLocations.GroupLocation(org.biojava.nbio.core.sequence.location.template.Location...)">InsdcLocations.GroupLocation(Location...)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.GroupLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html#InsdcLocations.GroupLocation(org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.Strand,%20boolean,%20org.biojava.nbio.core.sequence.location.template.Location...)">InsdcLocations.GroupLocation(Point, Point, Strand, boolean, Location...)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.GroupLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html#InsdcLocations.GroupLocation(org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.Strand,%20org.biojava.nbio.core.sequence.location.template.Location...)">InsdcLocations.GroupLocation(Point, Point, Strand, Location...)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.GroupLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html#InsdcLocations.GroupLocation(int,%20int,%20org.biojava.nbio.core.sequence.Strand,%20org.biojava.nbio.core.sequence.location.template.Location...)">InsdcLocations.GroupLocation(int, int, Strand, Location...)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.GroupLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html#InsdcLocations.GroupLocation(org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.Strand,%20boolean,%20java.util.List)">InsdcLocations.GroupLocation(Point, Point, Strand, boolean, List&lt;Location&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.GroupLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html#InsdcLocations.GroupLocation(org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.Strand,%20java.util.List)">InsdcLocations.GroupLocation(Point, Point, Strand, List&lt;Location&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.GroupLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html#InsdcLocations.GroupLocation(int,%20int,%20org.biojava.nbio.core.sequence.Strand,%20java.util.List)">InsdcLocations.GroupLocation(int, int, Strand, List&lt;Location&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.GroupLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.OneOfLocation.html" title="class in org.biojava.nbio.core.sequence.location"><span class="strong">InsdcLocations.OneOfLocation</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a></dt>
<dd>
<div class="block">Deprecated in INSDC; refers to a set of locations of which one
 location could be valid e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.OneOfLocation.html#InsdcLocations.OneOfLocation(org.biojava.nbio.core.sequence.location.template.Location...)">InsdcLocations.OneOfLocation(Location...)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.OneOfLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.OneOfLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.OneOfLocation.html#InsdcLocations.OneOfLocation(java.util.List)">InsdcLocations.OneOfLocation(List&lt;Location&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.OneOfLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.OneOfLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.OrderLocation.html" title="class in org.biojava.nbio.core.sequence.location"><span class="strong">InsdcLocations.OrderLocation</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a></dt>
<dd>
<div class="block">Used to describe a 5' to 3' ordering but no firm assurance it is correct</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.OrderLocation.html#InsdcLocations.OrderLocation(java.util.List)">InsdcLocations.OrderLocation(List)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.OrderLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.OrderLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.OrderLocation.html#InsdcLocations.OrderLocation(org.biojava.nbio.core.sequence.location.template.Location...)">InsdcLocations.OrderLocation(Location...)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.OrderLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.OrderLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.OrderLocation.html#InsdcLocations.OrderLocation(org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.Strand,%20boolean,%20org.biojava.nbio.core.sequence.location.template.Location...)">InsdcLocations.OrderLocation(Point, Point, Strand, boolean, Location...)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.OrderLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.OrderLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.OrderLocation.html#InsdcLocations.OrderLocation(org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.Strand,%20org.biojava.nbio.core.sequence.location.template.Location...)">InsdcLocations.OrderLocation(Point, Point, Strand, Location...)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.OrderLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.OrderLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.OrderLocation.html#InsdcLocations.OrderLocation(int,%20int,%20org.biojava.nbio.core.sequence.Strand,%20org.biojava.nbio.core.sequence.location.template.Location...)">InsdcLocations.OrderLocation(int, int, Strand, Location...)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.OrderLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.OrderLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.OrderLocation.html#InsdcLocations.OrderLocation(org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.Strand,%20boolean,%20java.util.List)">InsdcLocations.OrderLocation(Point, Point, Strand, boolean, List&lt;Location&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.OrderLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.OrderLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.OrderLocation.html#InsdcLocations.OrderLocation(org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.Strand,%20java.util.List)">InsdcLocations.OrderLocation(Point, Point, Strand, List&lt;Location&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.OrderLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.OrderLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.OrderLocation.html#InsdcLocations.OrderLocation(int,%20int,%20org.biojava.nbio.core.sequence.Strand,%20java.util.List)">InsdcLocations.OrderLocation(int, int, Strand, List&lt;Location&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcLocations.OrderLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.OrderLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html" title="class in org.biojava.nbio.core.sequence.location"><span class="strong">InsdcParser</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html" title="type parameter in InsdcParser">S</a> extends <a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html" title="type parameter in InsdcParser">C</a>&gt;,<a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html" title="type parameter in InsdcParser">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a></dt>
<dd>
<div class="block">Parser for working with INSDC style locations.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html#InsdcParser()">InsdcParser()</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html" title="class in org.biojava.nbio.core.sequence.location">InsdcParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html#InsdcParser(org.biojava.nbio.core.sequence.DataSource)">InsdcParser(DataSource)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html" title="class in org.biojava.nbio.core.sequence.location">InsdcParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/IntronSequence.html" title="class in org.biojava.nbio.core.sequence"><span class="strong">IntronSequence</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/IntronSequence.html#IntronSequence(org.biojava.nbio.core.sequence.GeneSequence,%20int,%20int)">IntronSequence(GeneSequence, int, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/IntronSequence.html" title="class in org.biojava.nbio.core.sequence">IntronSequence</a></dt>
<dd>
<div class="block">Place holder for Intron sequence features</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#intValue()">intValue()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>
<div class="block">Attempts to provide an int version of this codon which multiplies
 each position by</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#inverse(org.biojava.nbio.core.sequence.template.Sequence)">inverse(Sequence&lt;C&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">A method which attempts to do the right thing when is comes to a
 reverse/reverse complement</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util"><span class="strong">IOUtils</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/io/util/package-summary.html">org.biojava.nbio.core.sequence.io.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html#IOUtils()">IOUtils()</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.util.<a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.ReaderProcessor.html" title="interface in org.biojava.nbio.core.sequence.io.util"><span class="strong">IOUtils.ReaderProcessor</span></a> - Interface in <a href="./org/biojava/nbio/core/sequence/io/util/package-summary.html">org.biojava.nbio.core.sequence.io.util</a></dt>
<dd>
<div class="block">Closure interface used when working with
 <code>IOUtils#processReader(String)</code>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#isAmbiguous()">isAmbiguous()</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#isBetweenCompounds()">isBetweenCompounds()</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Location.html#isBetweenCompounds()">isBetweenCompounds()</a></span> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Returns true if the position is meant to represent a point between
 two points such as 78^79.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#isCircular()">isCircular()</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Location.html#isCircular()">isCircular()</a></span> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Indicates if this location is circular.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#isComplementable()">isComplementable()</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#isComplementable()">isComplementable()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#isComplementable()">isComplementable()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractNucleotideCompoundSet</a></dt>
<dd>
<div class="block">NucleotideCompounds can always complement</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/CompoundSet.html#isComplementable()">isComplementable()</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#isComplex()">isComplex()</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Location.html#isComplex()">isComplex()</a></span> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Returns true if the location is considered to be complex meaning
 the location is actually composed of sub-locations.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#isCompoundStringLengthEqual()">isCompoundStringLengthEqual()</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#isCompoundStringLengthEqual()">isCompoundStringLengthEqual()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/CompoundSet.html#isCompoundStringLengthEqual()">isCompoundStringLengthEqual()</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundSet</a></dt>
<dd>
<div class="block">Returns true if all String representations of Compounds are of the
 same length.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html#isHigher(org.biojava.nbio.core.sequence.location.template.Point)">isHigher(Point)</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Point.html#isHigher(org.biojava.nbio.core.sequence.location.template.Point)">isHigher(Point)</a></span> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Point.html" title="interface in org.biojava.nbio.core.sequence.location.template">Point</a></dt>
<dd>
<div class="block">Returns true if the point is higher in value to the current point</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html#isLower(org.biojava.nbio.core.sequence.location.template.Point)">isLower(Point)</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Point.html#isLower(org.biojava.nbio.core.sequence.location.template.Point)">isLower(Point)</a></span> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Point.html" title="interface in org.biojava.nbio.core.sequence.location.template">Point</a></dt>
<dd>
<div class="block">Returns true if the current point is at a lower position than the
 point given.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/SequenceTools.html#isNucleotideSequence(java.lang.String)">isNucleotideSequence(String)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/SequenceTools.html" title="class in org.biojava.nbio.core.util">SequenceTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#isPartial()">isPartial()</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#isPartialOn3prime()">isPartialOn3prime()</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#isPartialOn5prime()">isPartialOn5prime()</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html#isStart()">isStart()</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html#isStart(org.biojava.nbio.core.sequence.compound.AminoAcidCompound)">isStart(AminoAcidCompound)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser.IUPACTable</a></dt>
<dd>
<div class="block">Returns true if the given compound was a start codon in this
 codon table.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html#isStart()">isStart()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.html#isStart(org.biojava.nbio.core.sequence.compound.AminoAcidCompound)">isStart(AminoAcidCompound)</a></span> - Method in interface org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.html" title="interface in org.biojava.nbio.core.sequence.transcription">Table</a></dt>
<dd>
<div class="block">Returns true if the given compound could have been a start amino acid;
 this does not assert if the codon that actually coded for the amino
 acid was a start codon.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html#isStop()">isStop()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html#isUncertain()">isUncertain()</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Point.html#isUncertain()">isUncertain()</a></span> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Point.html" title="interface in org.biojava.nbio.core.sequence.location.template">Point</a></dt>
<dd>
<div class="block">Returns a true if the exact point is unknown.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html#isUnknown()">isUnknown()</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Point.html#isUnknown()">isUnknown()</a></span> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Point.html" title="interface in org.biojava.nbio.core.sequence.location.template">Point</a></dt>
<dd>
<div class="block">Returns true if the current position is unknown but is
 beyond the position encoded for.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#isValidSequence(org.biojava.nbio.core.sequence.template.Sequence)">isValidSequence(Sequence&lt;AminoAcidCompound&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#isValidSequence(org.biojava.nbio.core.sequence.template.Sequence)">isValidSequence(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/CompoundSet.html#isValidSequence(org.biojava.nbio.core.sequence.template.Sequence)">isValidSequence(Sequence&lt;C&gt;)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#iterator()">iterator()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#iterator()">iterator()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#iterator()">iterator()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#iterator()">iterator()</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>
<div class="block">Iterates through all known sub-locations for this location but does
 not descend</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#iterator()">iterator()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#iterator()">iterator()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Provides basic iterable access to this class</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#iterator()">iterator()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>
<div class="block">Iterator implementation which attempts to move through the 2D structure
 attempting to skip onto the next sequence as & when it is asked to</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#iterator()">iterator()</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Returns an instance of <code>SequenceMixin.SequenceIterator</code></div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#iterator()">iterator()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html#iterator()">iterator()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/WindowedSequence.html#iterator()">iterator()</a></span> - Method in class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/WindowedSequence.html" title="class in org.biojava.nbio.core.sequence.views">WindowedSequence</a></dt>
<dd>
<div class="block">Returns an iterator which will return the windows in a sequence in
 sequential order.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/IUPACParser.html#IUPAC_LOCATION">IUPAC_LOCATION</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/IUPACParser.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/IUPACParser.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">IUPACParser</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">Available translations

 
 1 - UNIVERSAL
 2 - VERTEBRATE_MITOCHONDRIAL
 3 - YEAST_MITOCHONDRIAL
 4 - MOLD_MITOCHONDRIAL
 5 - INVERTEBRATE_MITOCHONDRIAL
 6 - CILIATE_NUCLEAR
 9 - ECHINODERM_MITOCHONDRIAL
 10 - EUPLOTID_NUCLEAR
 11 - BACTERIAL
 12 - ALTERNATIVE_YEAST_NUCLEAR
 13 - ASCIDIAN_MITOCHONDRIAL
 14 - FLATWORM_MITOCHONDRIAL
 15 - BLEPHARISMA_MACRONUCLEAR
 16 - 2CHLOROPHYCEAN_MITOCHONDRIAL
 21 - TREMATODE_MITOCHONDRIAL
 23 - SCENEDESMUS_MITOCHONDRIAL
 

 Taken from <a
 href="http://www.ncbi.nlm.nih.gov/Taxonomy/Utils/wprintgc.cgi?mode=c"
 >NCBI</a> with slight modification and put into the classpath resource.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/IUPACParser.html#IUPACParser()">IUPACParser()</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/IUPACParser.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser</a></dt>
<dd>
<div class="block">Default version and uses the classpath based IUPAC table</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/IUPACParser.html#IUPACParser(java.io.InputStream)">IUPACParser(InputStream)</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/IUPACParser.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser</a></dt>
<dd>
<div class="block">Allows you to specify a different IUPAC table.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">IUPACParser.IUPACTable</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">Holds the concept of a codon table from the IUPAC format</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html#IUPACParser.IUPACTable(java.lang.String,%20int,%20java.lang.String,%20java.lang.String,%20java.lang.String,%20java.lang.String,%20java.lang.String)">IUPACParser.IUPACTable(String, int, String, String, String, String, String)</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser.IUPACTable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html#IUPACParser.IUPACTable(java.lang.String,%20java.lang.Integer,%20java.lang.String,%20java.lang.String)">IUPACParser.IUPACTable(String, Integer, String, String)</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser.IUPACTable</a></dt>
<dd>
<div class="block">Constructor which uses the basic IUPAC codon table format.</div>
</dd>
</dl>
<a name="_J_">
<!--   -->
</a>
<h2 class="title">J</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/StringManipulationHelper.html#join(java.util.AbstractCollection,%20java.lang.String)">join(AbstractCollection&lt;String&gt;, String)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/StringManipulationHelper.html" title="class in org.biojava.nbio.core.util">StringManipulationHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage"><span class="strong">JoiningSequenceReader</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="type parameter in JoiningSequenceReader">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/storage/package-summary.html">org.biojava.nbio.core.sequence.storage</a></dt>
<dd>
<div class="block">This reader actually proxies onto multiple types of sequence in order
 to allow a number of sequence objects to act as if they are one sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#JoiningSequenceReader(org.biojava.nbio.core.sequence.template.Sequence...)">JoiningSequenceReader(Sequence&lt;C&gt;...)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>
<div class="block">Allows creation of the store from Vargs Sequence<C> objects.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#JoiningSequenceReader(java.util.List)">JoiningSequenceReader(List&lt;Sequence&lt;C&gt;&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>
<div class="block">Allows creation of the store from List<Sequence<C>>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#JoiningSequenceReader(org.biojava.nbio.core.sequence.template.CompoundSet,%20org.biojava.nbio.core.sequence.template.Sequence...)">JoiningSequenceReader(CompoundSet&lt;C&gt;, Sequence&lt;C&gt;...)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#JoiningSequenceReader(org.biojava.nbio.core.sequence.template.CompoundSet,%20java.util.List)">JoiningSequenceReader(CompoundSet&lt;C&gt;, List&lt;Sequence&lt;C&gt;&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#JOURNAL_TAG">JOURNAL_TAG</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_K_">
<!--   -->
</a>
<h2 class="title">K</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#KEYWORDS_TAG">KEYWORDS_TAG</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_L_">
<!--   -->
</a>
<h2 class="title">L</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#lastIndexOf(org.biojava.nbio.core.sequence.template.Sequence,%20C)">lastIndexOf(Sequence&lt;C&gt;, C)</a></span> - Static method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Performs a reversed linear search of the given Sequence by wrapping
 it in a <a href="./org/biojava/nbio/core/sequence/views/ReversedSequenceView.html" title="class in org.biojava.nbio.core.sequence.views"><code>ReversedSequenceView</code></a> and passing it into
 <a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#indexOf(org.biojava.nbio.core.sequence.template.Sequence,%20C)"><code>SequenceMixin.indexOf(Sequence, Compound)</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html#LENGTH">LENGTH</a></span> - Static variable in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">Sort features by length.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="interface in org.biojava.nbio.core.sequence.template"><span class="strong">LightweightProfile</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="type parameter in LightweightProfile">S</a> extends <a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="type parameter in LightweightProfile">C</a>&gt;,<a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="type parameter in LightweightProfile">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>
<div class="block">Defines a minimal data structure for reading and writing a sequence alignment.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.StringFormat.html" title="enum in org.biojava.nbio.core.sequence.template"><span class="strong">LightweightProfile.StringFormat</span></a> - Enum in <a href="./org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>
<div class="block">List of output formats.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#LINEAR_DNA">LINEAR_DNA</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriterHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/XMLHelper.html#loadXML(java.lang.String)">loadXML(String)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/XMLHelper.html" title="class in org.biojava.nbio.core.util">XMLHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html#location(java.util.List)">location(List&lt;Location&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html" title="class in org.biojava.nbio.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Used as a thin wrapper to the <a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html#location(java.util.List,%20java.lang.String)"><code>LocationHelper.location(java.util.List, java.lang.String)</code></a>
 method to bring the given location list together as a join (the default
 type)</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html#location(java.util.List,%20java.lang.String)">location(List&lt;Location&gt;, String)</a></span> - Static method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html" title="class in org.biojava.nbio.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Builds a location from a List of locations; this can be circular or
 linear joins.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html#location(int,%20int,%20org.biojava.nbio.core.sequence.Strand,%20int)">location(int, int, Strand, int)</a></span> - Static method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html" title="class in org.biojava.nbio.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Returns a location object which unlike the location constructors
 allows you to input reverse coordinates and will convert
 these into the right location on the positive strand.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template"><span class="strong">Location</span></a> - Interface in <a href="./org/biojava/nbio/core/sequence/location/template/package-summary.html">org.biojava.nbio.core.sequence.location.template</a></dt>
<dd>
<div class="block">Sets of integers used to represent the location of features on sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Location.Tools.html#location(java.util.List,%20java.lang.Integer,%20java.lang.String)">location(List&lt;Location&gt;, Integer, String)</a></span> - Static method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Location.Tools.html" title="class in org.biojava.nbio.core.sequence.location.template">Location.Tools</a></dt>
<dd>
<div class="block">Used for building a location from a series of sub-locations</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Location.Tools.html#location(int,%20int,%20org.biojava.nbio.core.sequence.Strand,%20int)">location(int, int, Strand, int)</a></span> - Static method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Location.Tools.html" title="class in org.biojava.nbio.core.sequence.location.template">Location.Tools</a></dt>
<dd>
<div class="block">Returns a location object which unlike the location constructors
 allows you to input reverse coordinates and will convert
 these into the right location on the positive strand.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/location/template/Location.Tools.html" title="class in org.biojava.nbio.core.sequence.location.template"><span class="strong">Location.Tools</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/location/template/package-summary.html">org.biojava.nbio.core.sequence.location.template</a></dt>
<dd>
<div class="block">Helper methods for use with the Location classes.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Location.Tools.html#Location.Tools()">Location.Tools()</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Location.Tools.html" title="class in org.biojava.nbio.core.sequence.location.template">Location.Tools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html#LOCATION_LENGTH">LOCATION_LENGTH</a></span> - Static variable in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">Sort features by start position and then longest length.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html" title="class in org.biojava.nbio.core.sequence.location"><span class="strong">LocationHelper</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a></dt>
<dd>
<div class="block">Helper methods for use with the Location classes.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html#LocationHelper()">LocationHelper()</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html" title="class in org.biojava.nbio.core.sequence.location">LocationHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#locationParser">locationParser</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html#locationSplitPattern">locationSplitPattern</a></span> - Static variable in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html" title="class in org.biojava.nbio.core.sequence.location">InsdcParser</a></dt>
<dd>
<div class="block">designed to recursively split a location string in tokens.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#LOCUS_TAG">LOCUS_TAG</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#lp">lp</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_M_">
<!--   -->
</a>
<h2 class="title">M</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/DNASequence.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">CasePreservingProteinSequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaGeneWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaGeneWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaReader.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaReader.html" title="class in org.biojava.nbio.core.sequence.io">FastaReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaReaderHelper.html" title="class in org.biojava.nbio.core.sequence.io">FastaReaderHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaWriter.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankReader.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankReader.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReaderHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericFastaHeaderParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html" title="class in org.biojava.nbio.core.sequence.location">InsdcParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/ProteinSequence.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/ProteinSequence.html" title="class in org.biojava.nbio.core.sequence">ProteinSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/UncompressInputStream.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/UncompressInputStream.html" title="class in org.biojava.nbio.core.util">UncompressInputStream</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#mapDB">mapDB</a></span> - Variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#mark(int)">mark(int)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava.nbio.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>
<div class="block">Marks the present position in the stream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#markSupported()">markSupported()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava.nbio.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>
<div class="block">Tells whether this stream supports the mark() operation, which it does.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/UncompressInputStream.html#markSupported()">markSupported()</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/UncompressInputStream.html" title="class in org.biojava.nbio.core.util">UncompressInputStream</a></dt>
<dd>
<div class="block">This stream does not support mark/reset on the stream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html#MAX_RESOLVER">MAX_RESOLVER</a></span> - Static variable in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html" title="class in org.biojava.nbio.core.sequence.location">FuzzyPoint</a></dt>
<dd>
<div class="block">Always uses the max point to resolve a location</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#MAX_WIDTH">MAX_WIDTH</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io">GenericInsdcHeaderFormat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html#MEAN_RESOLVER">MEAN_RESOLVER</a></span> - Static variable in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html" title="class in org.biojava.nbio.core.sequence.location">FuzzyPoint</a></dt>
<dd>
<div class="block">Combines min and max and then gets the mean of it</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#MEDLINE_TAG">MEDLINE_TAG</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html#MIN_RESOLVER">MIN_RESOLVER</a></span> - Static variable in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html" title="class in org.biojava.nbio.core.sequence.location">FuzzyPoint</a></dt>
<dd>
<div class="block">Always uses the min point to resolve a location</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html#modulateCircularIndex(int,%20int)">modulateCircularIndex(int, int)</a></span> - Static method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/LocationHelper.html" title="class in org.biojava.nbio.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Takes a point on a circular location and moves it left until it falls
 at the earliest possible point that represents the same base.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Location.Tools.html#modulateCircularIndex(int,%20int)">modulateCircularIndex(int, int)</a></span> - Static method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Location.Tools.html" title="class in org.biojava.nbio.core.sequence.location.template">Location.Tools</a></dt>
<dd>
<div class="block">Takes a point on a circular location and moves it left until it falls
 at the earliest possible point that represents the same base.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#multipleFrameTranslation(org.biojava.nbio.core.sequence.template.Sequence,%20org.biojava.nbio.core.sequence.transcription.Frame...)">multipleFrameTranslation(Sequence&lt;NucleotideCompound&gt;, Frame...)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>
<div class="block">A way of translating DNA in a number of frames</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava.nbio.core.sequence"><span class="strong">MultipleSequenceAlignment</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="type parameter in MultipleSequenceAlignment">S</a> extends <a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="type parameter in MultipleSequenceAlignment">C</a>&gt;,<a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="type parameter in MultipleSequenceAlignment">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">Implements a minimal data structure for reading and writing a sequence alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#MultipleSequenceAlignment()">MultipleSequenceAlignment()</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava.nbio.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_N_">
<!--   -->
</a>
<h2 class="title">N</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/Qualifier.html#needsQuotes()">needsQuotes()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/Qualifier.html" title="class in org.biojava.nbio.core.sequence.features">Qualifier</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/DNACompoundSet.html#newNucleotideCompound(java.lang.String,%20java.lang.String,%20java.lang.String...)">newNucleotideCompound(String, String, String...)</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/DNACompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">DNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/RNACompoundSet.html#newNucleotideCompound(java.lang.String,%20java.lang.String,%20java.lang.String...)">newNucleotideCompound(String, String, String...)</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/RNACompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">RNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#newNucleotideCompound(java.lang.String,%20java.lang.String,%20java.lang.String...)">newNucleotideCompound(String, String, String...)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractNucleotideCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.SequenceIterator.html#next()">next()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.SequenceIterator.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin.SequenceIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#nonOverlappingKmers(org.biojava.nbio.core.sequence.template.Sequence,%20int)">nonOverlappingKmers(Sequence&lt;C&gt;, int)</a></span> - Static method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Produces kmers of the specified size e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/SequenceTools.html#NUCLEOTIDE_LETTERS">NUCLEOTIDE_LETTERS</a></span> - Static variable in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/SequenceTools.html" title="class in org.biojava.nbio.core.util">SequenceTools</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound"><span class="strong">NucleotideCompound</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/compound/package-summary.html">org.biojava.nbio.core.sequence.compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#NucleotideCompound(java.lang.String,%20org.biojava.nbio.core.sequence.template.CompoundSet,%20java.lang.String)">NucleotideCompound(String, CompoundSet&lt;NucleotideCompound&gt;, String)</a></span> - Constructor for class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#NucleotideCompound(java.lang.String,%20org.biojava.nbio.core.sequence.template.CompoundSet,%20java.lang.String,%20org.biojava.nbio.core.sequence.compound.NucleotideCompound[])">NucleotideCompound(String, CompoundSet&lt;NucleotideCompound&gt;, String, NucleotideCompound[])</a></span> - Constructor for class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html#numericalEntity(char)">numericalEntity(char)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_O_">
<!--   -->
</a>
<h2 class="title">O</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html#offset(int)">offset(int)</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html" title="class in org.biojava.nbio.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html#offset(int)">offset(int)</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Point.html#offset(int)">offset(int)</a></span> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Point.html" title="interface in org.biojava.nbio.core.sequence.location.template">Point</a></dt>
<dd>
<div class="block">Returns a new point offset by the given distance</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html#openFile(java.io.File)">openFile(File)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">For a filename this code will check the extension of the file for a
 .gz extension.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html#openTag(java.lang.String,%20java.lang.String)">openTag(String, String)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html#openTag(java.lang.String)">openTag(String)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/XMLWriter.html#openTag(java.lang.String,%20java.lang.String)">openTag(String, String)</a></span> - Method in interface org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/XMLWriter.html" title="interface in org.biojava.nbio.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Open a new namespace-qualified XML tag.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/XMLWriter.html#openTag(java.lang.String)">openTag(String)</a></span> - Method in interface org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/XMLWriter.html" title="interface in org.biojava.nbio.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Open a new unqualified XML tag.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/exceptions/package-summary.html">org.biojava.nbio.core.exceptions</a> - package org.biojava.nbio.core.exceptions</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a> - package org.biojava.nbio.core.sequence</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/compound/package-summary.html">org.biojava.nbio.core.sequence.compound</a> - package org.biojava.nbio.core.sequence.compound</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/edits/package-summary.html">org.biojava.nbio.core.sequence.edits</a> - package org.biojava.nbio.core.sequence.edits</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/features/package-summary.html">org.biojava.nbio.core.sequence.features</a> - package org.biojava.nbio.core.sequence.features</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a> - package org.biojava.nbio.core.sequence.io</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/template/package-summary.html">org.biojava.nbio.core.sequence.io.template</a> - package org.biojava.nbio.core.sequence.io.template</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/util/package-summary.html">org.biojava.nbio.core.sequence.io.util</a> - package org.biojava.nbio.core.sequence.io.util</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/loader/package-summary.html">org.biojava.nbio.core.sequence.loader</a> - package org.biojava.nbio.core.sequence.loader</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a> - package org.biojava.nbio.core.sequence.location</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/location/template/package-summary.html">org.biojava.nbio.core.sequence.location.template</a> - package org.biojava.nbio.core.sequence.location.template</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/storage/package-summary.html">org.biojava.nbio.core.sequence.storage</a> - package org.biojava.nbio.core.sequence.storage</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a> - package org.biojava.nbio.core.sequence.template</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/transcription/package-summary.html">org.biojava.nbio.core.sequence.transcription</a> - package org.biojava.nbio.core.sequence.transcription</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/views/package-summary.html">org.biojava.nbio.core.sequence.views</a> - package org.biojava.nbio.core.sequence.views</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a> - package org.biojava.nbio.core.util</dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#ORGANISM_TAG">ORGANISM_TAG</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/XMLHelper.html#outputToStream(org.w3c.dom.Document,%20java.io.OutputStream)">outputToStream(Document, OutputStream)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/XMLHelper.html" title="class in org.biojava.nbio.core.util">XMLHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/XMLHelper.html#outputToStream(org.w3c.dom.Element,%20java.io.OutputStream)">outputToStream(Element, OutputStream)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/XMLHelper.html" title="class in org.biojava.nbio.core.util">XMLHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#overlappingKmers(org.biojava.nbio.core.sequence.template.Sequence,%20int)">overlappingKmers(Sequence&lt;C&gt;, int)</a></span> - Static method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Used to generate overlapping k-mers such i.e.</div>
</dd>
</dl>
<a name="_P_">
<!--   -->
</a>
<h2 class="title">P</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/StringManipulationHelper.html#padLeft(java.lang.String,%20int)">padLeft(String, int)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/StringManipulationHelper.html" title="class in org.biojava.nbio.core.util">StringManipulationHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/StringManipulationHelper.html#padRight(java.lang.String,%20int)">padRight(String, int)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/StringManipulationHelper.html" title="class in org.biojava.nbio.core.util">StringManipulationHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/StartCodonSequence.html#parentGeneSequence">parentGeneSequence</a></span> - Variable in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/StartCodonSequence.html" title="class in org.biojava.nbio.core.sequence">StartCodonSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/StopCodonSequence.html#parentGeneSequence">parentGeneSequence</a></span> - Variable in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/StopCodonSequence.html" title="class in org.biojava.nbio.core.sequence">StopCodonSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html#parse(java.lang.String)">parse(String)</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html" title="class in org.biojava.nbio.core.sequence.location">InsdcParser</a></dt>
<dd>
<div class="block">Main method for parsing a location from a String instance</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html#parse(java.io.Reader)">parse(Reader)</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html" title="class in org.biojava.nbio.core.sequence.location">InsdcParser</a></dt>
<dd>
<div class="block">Reader based version of the parse methods.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#parseFeatures(org.biojava.nbio.core.sequence.template.AbstractSequence)">parseFeatures(AbstractSequence&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#parseHeader(java.lang.String,%20S)">parseHeader(String, S)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericFastaHeaderParser</a></dt>
<dd>
<div class="block">Parse the header and set the values in the sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#parseHeader(java.lang.String,%20S)">parseHeader(String, S)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderParser</a></dt>
<dd>
<div class="block">Parse the header and set the values in the sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#parseHeader(java.lang.String,%20S)">parseHeader(String, S)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">PlainFastaHeaderParser</a></dt>
<dd>
<div class="block">Parse the header and set the values in the sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/template/SequenceHeaderParserInterface.html#parseHeader(java.lang.String,%20S)">parseHeader(String, S)</a></span> - Method in interface org.biojava.nbio.core.sequence.io.template.<a href="./org/biojava/nbio/core/sequence/io/template/SequenceHeaderParserInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template">SequenceHeaderParserInterface</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/exceptions/ParserException.html" title="class in org.biojava.nbio.core.exceptions"><span class="strong">ParserException</span></a> - Exception in <a href="./org/biojava/nbio/core/exceptions/package-summary.html">org.biojava.nbio.core.exceptions</a></dt>
<dd>
<div class="block">General abstraction of different parsing errors</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/exceptions/ParserException.html#ParserException(java.lang.String)">ParserException(String)</a></span> - Constructor for exception org.biojava.nbio.core.exceptions.<a href="./org/biojava/nbio/core/exceptions/ParserException.html" title="class in org.biojava.nbio.core.exceptions">ParserException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/exceptions/ParserException.html#ParserException(java.lang.Exception)">ParserException(Exception)</a></span> - Constructor for exception org.biojava.nbio.core.exceptions.<a href="./org/biojava/nbio/core/exceptions/ParserException.html" title="class in org.biojava.nbio.core.exceptions">ParserException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/exceptions/ParserException.html#ParserException(java.lang.String,%20java.lang.Exception)">ParserException(String, Exception)</a></span> - Constructor for exception org.biojava.nbio.core.exceptions.<a href="./org/biojava/nbio/core/exceptions/ParserException.html" title="class in org.biojava.nbio.core.exceptions">ParserException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#parseUniprotXMLString(java.lang.String,%20org.biojava.nbio.core.sequence.template.CompoundSet)">parseUniprotXMLString(String, CompoundSet&lt;C&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">The passed in xml is parsed as a DOM object so we know everything about the protein.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/SequenceTools.html#percentNucleotideSequence(java.lang.String)">percentNucleotideSequence(String)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/SequenceTools.html" title="class in org.biojava.nbio.core.util">SequenceTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/SequenceTools.html#permuteCyclic(java.lang.String,%20int)">permuteCyclic(String, int)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/SequenceTools.html" title="class in org.biojava.nbio.core.util">SequenceTools</a></dt>
<dd>
<div class="block">Cyclically permute the characters in <code>string</code> <em>forward</em> by <code>n</code> elements.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/SequenceTools.html#permuteCyclic(T[],%20T[],%20int)">permuteCyclic(T[], T[], int)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/SequenceTools.html" title="class in org.biojava.nbio.core.util">SequenceTools</a></dt>
<dd>
<div class="block">Cyclically permute <code>array</code> <em>forward</em> by <code>n</code> elements.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">PlainFastaHeaderParser</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html" title="type parameter in PlainFastaHeaderParser">S</a> extends <a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="./org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html" title="type parameter in PlainFastaHeaderParser">C</a>&gt;,<a href="./org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html" title="type parameter in PlainFastaHeaderParser">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">The plain fasta header takes everything in the header as a single entity.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#PlainFastaHeaderParser()">PlainFastaHeaderParser()</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">PlainFastaHeaderParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/location/template/Point.html" title="interface in org.biojava.nbio.core.sequence.location.template"><span class="strong">Point</span></a> - Interface in <a href="./org/biojava/nbio/core/sequence/location/template/package-summary.html">org.biojava.nbio.core.sequence.location.template</a></dt>
<dd>
<div class="block">Holds a single point part of a location</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/location/template/Point.Resolver.html" title="interface in org.biojava.nbio.core.sequence.location.template"><span class="strong">Point.Resolver</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/location/template/Point.Resolver.html" title="type parameter in Point.Resolver">T</a> extends <a href="./org/biojava/nbio/core/sequence/location/template/Point.html" title="interface in org.biojava.nbio.core.sequence.location.template">Point</a>&gt; - Interface in <a href="./org/biojava/nbio/core/sequence/location/template/package-summary.html">org.biojava.nbio.core.sequence.location.template</a></dt>
<dd>
<div class="block">Used to resolve a position about a point</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#populate(org.biojava.nbio.core.sequence.template.Sequence)">populate(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Loops through the Compounds in a Sequence and passes them onto
 <a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#setCompoundAt(C,%20int)"><code>BitSequenceReader.BitArrayWorker.setCompoundAt(Compound, int)</code></a></div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#populate(java.lang.String)">populate(String)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Loops through the chars in a String and passes them onto
 <a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#setCompoundAt(char,%20int)"><code>BitSequenceReader.BitArrayWorker.setCompoundAt(char, int)</code></a></div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#postProcessCompoundLists(java.util.List)">postProcessCompoundLists(List&lt;List&lt;T&gt;&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#postProcessCompoundLists(java.util.List)">postProcessCompoundLists(List&lt;List&lt;NucleotideCompound&gt;&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html" title="class in org.biojava.nbio.core.sequence.transcription">DNAToRNATranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#postProcessCompoundLists(java.util.List)">postProcessCompoundLists(List&lt;List&lt;AminoAcidCompound&gt;&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html" title="class in org.biojava.nbio.core.sequence.transcription">RNAToAminoAcidTranslator</a></dt>
<dd>
<div class="block">Performs the trimming of stop codons and the conversion of a valid start
 amino acid to M</div>
</dd>
<dt><a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util"><span class="strong">PrettyXMLWriter</span></a> - Class in <a href="./org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>
<div class="block">Implementation of XMLWriter which emits nicely formatted documents
 to a PrintWriter.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html#PrettyXMLWriter(java.io.PrintWriter)">PrettyXMLWriter(PrintWriter)</a></span> - Constructor for class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/Hashcoder.html#PRIME">PRIME</a></span> - Static variable in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/Hashcoder.html" title="class in org.biojava.nbio.core.util">Hashcoder</a></dt>
<dd>
<div class="block">The prime number used to multiply any calculated hashcode seed by

 i.e.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html#print(java.lang.String,%20java.io.File)">print(String, File)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Prints <code>string</code> to <code>file</code>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html#print(java.lang.String)">print(String)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/XMLWriter.html#print(java.lang.String)">print(String)</a></span> - Method in interface org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/XMLWriter.html" title="interface in org.biojava.nbio.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Prints some textual content in an element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html#printAttributeValue(java.lang.String)">printAttributeValue(String)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html#printChars(java.lang.String)">printChars(String)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html#println(java.lang.String)">println(String)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/XMLWriter.html#println(java.lang.String)">println(String)</a></span> - Method in interface org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/XMLWriter.html" title="interface in org.biojava.nbio.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Prints some textual content, terminated with a newline character.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html#printRaw(java.lang.String)">printRaw(String)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/XMLWriter.html#printRaw(java.lang.String)">printRaw(String)</a></span> - Method in interface org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/XMLWriter.html" title="interface in org.biojava.nbio.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Send raw data to the stream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#process()">process()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaGeneWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaGeneWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaReader.html#process()">process()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaReader.html" title="class in org.biojava.nbio.core.sequence.io">FastaReader</a></dt>
<dd>
<div class="block">The parsing is done in this method.<br>
 This method tries to process all the available fasta records 
 in the File or InputStream, closes the underlying resource, 
 and return the results in <a href="http://java.sun.com/javase/6/docs/api/java/util/LinkedHashMap.html?is-external=true" title="class or interface in java.util"><code>LinkedHashMap</code></a>.<br>
 You don't need to call <a href="./org/biojava/nbio/core/sequence/io/FastaReader.html#close()"><code>FastaReader.close()</code></a> after calling this method.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaReader.html#process(int)">process(int)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaReader.html" title="class in org.biojava.nbio.core.sequence.io">FastaReader</a></dt>
<dd>
<div class="block">This method tries to parse maximum <code>max</code> records from
 the open File or InputStream, and leaves the underlying resource open.<br>
 Subsequent calls to the same method continue parsing the rest of the file.<br>
 This is particularly useful when dealing with very big data files,
 (e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaWriter.html#process()">process()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankReader.html#process()">process()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankReader.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReader</a></dt>
<dd>
<div class="block">The parsing is done in this method.<br>
 This method tries to process all the available Genbank records 
 in the File or InputStream, closes the underlying resource, 
 and return the results in <a href="http://java.sun.com/javase/6/docs/api/java/util/LinkedHashMap.html?is-external=true" title="class or interface in java.util"><code>LinkedHashMap</code></a>.<br>
 You don't need to call <a href="./org/biojava/nbio/core/sequence/io/GenbankReader.html#close()"><code>GenbankReader.close()</code></a> after calling this method.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankReader.html#process(int)">process(int)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankReader.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReader</a></dt>
<dd>
<div class="block">This method tries to parse maximum <code>max</code> records from
 the open File or InputStream, and leaves the underlying resource open.<br>
 Subsequent calls to the same method continue parsing the rest of the file.<br>
 This is particularly useful when dealing with very big data files,
 (e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankWriter.html#process()">process()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankWriter.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriter</a></dt>
<dd>
<div class="block">Allow an override of operating system line separator for programs that
 needs a specific CRLF or CR or LF option</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.ReaderProcessor.html#process(java.lang.String)">process(String)</a></span> - Method in interface org.biojava.nbio.core.sequence.io.util.<a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.ReaderProcessor.html" title="interface in org.biojava.nbio.core.sequence.io.util">IOUtils.ReaderProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html#processReader(java.io.BufferedReader,%20org.biojava.nbio.core.sequence.io.util.IOUtils.ReaderProcessor)">processReader(BufferedReader, IOUtils.ReaderProcessor)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="./org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Takes in a reader and a processor, reads every line from the given
 file and then invokes the processor.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#processUnknownCompound(C,%20int)">processUnknownCompound(C, int)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Since bit encoding only supports a finite number of bases
 it is more than likely when processing sequence you will encounter a
 compound which is not covered by the encoding e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#proteinCreator(org.biojava.nbio.core.sequence.io.template.SequenceCreatorInterface)">proteinCreator(SequenceCreatorInterface&lt;AminoAcidCompound&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/ProteinSequence.html" title="class in org.biojava.nbio.core.sequence"><span class="strong">ProteinSequence</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">The representation of a ProteinSequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/ProteinSequence.html#ProteinSequence(java.lang.String)">ProteinSequence(String)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/ProteinSequence.html" title="class in org.biojava.nbio.core.sequence">ProteinSequence</a></dt>
<dd>
<div class="block">Create a protein from a string</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/ProteinSequence.html#ProteinSequence(java.lang.String,%20org.biojava.nbio.core.sequence.template.CompoundSet)">ProteinSequence(String, CompoundSet&lt;AminoAcidCompound&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/ProteinSequence.html" title="class in org.biojava.nbio.core.sequence">ProteinSequence</a></dt>
<dd>
<div class="block">Create a protein from a string with a user defined set of amino acids</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/ProteinSequence.html#ProteinSequence(org.biojava.nbio.core.sequence.template.ProxySequenceReader)">ProteinSequence(ProxySequenceReader&lt;AminoAcidCompound&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/ProteinSequence.html" title="class in org.biojava.nbio.core.sequence">ProteinSequence</a></dt>
<dd>
<div class="block">A protein sequence where the storage of the sequence is somewhere else.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/ProteinSequence.html#ProteinSequence(org.biojava.nbio.core.sequence.template.ProxySequenceReader,%20org.biojava.nbio.core.sequence.template.CompoundSet)">ProteinSequence(ProxySequenceReader&lt;AminoAcidCompound&gt;, CompoundSet&lt;AminoAcidCompound&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/ProteinSequence.html" title="class in org.biojava.nbio.core.sequence">ProteinSequence</a></dt>
<dd>
<div class="block">A protein sequence where the storage of the sequence is somewhere else
 with user defined set of amino acids.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">ProteinSequenceCreator</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">Used to create a ProteinSequence from a String to allow for details
 about the location of the sequence etc.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#ProteinSequenceCreator(org.biojava.nbio.core.sequence.template.CompoundSet)">ProteinSequenceCreator(CompoundSet&lt;AminoAcidCompound&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">ProteinSequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/template/ProxySequenceReader.html" title="interface in org.biojava.nbio.core.sequence.template"><span class="strong">ProxySequenceReader</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/template/ProxySequenceReader.html" title="type parameter in ProxySequenceReader">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#PUBMED_TAG">PUBMED_TAG</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/SoftHashMap.html#put(K,%20V)">put(K, V)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/SoftHashMap.html" title="class in org.biojava.nbio.core.util">SoftHashMap</a></dt>
<dd>
<div class="block">Here we put the key, value pair into the HashMap using a SoftValue

 object.</div>
</dd>
</dl>
<a name="_Q_">
<!--   -->
</a>
<h2 class="title">Q</h2>
<dl>
<dt><a href="./org/biojava/nbio/core/sequence/features/Qualifier.html" title="class in org.biojava.nbio.core.sequence.features"><span class="strong">Qualifier</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/features/package-summary.html">org.biojava.nbio.core.sequence.features</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/Qualifier.html#Qualifier(java.lang.String,%20java.lang.String)">Qualifier(String, String)</a></span> - Constructor for class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/Qualifier.html" title="class in org.biojava.nbio.core.sequence.features">Qualifier</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/Qualifier.html#Qualifier(java.lang.String,%20java.lang.String,%20boolean)">Qualifier(String, String, boolean)</a></span> - Constructor for class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/Qualifier.html" title="class in org.biojava.nbio.core.sequence.features">Qualifier</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#QUALIFIER_INDENT">QUALIFIER_INDENT</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io">GenericInsdcHeaderFormat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#QUALIFIER_INDENT_STR">QUALIFIER_INDENT_STR</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io">GenericInsdcHeaderFormat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#QUALIFIER_INDENT_TMP">QUALIFIER_INDENT_TMP</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io">GenericInsdcHeaderFormat</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/features/QualityFeature.html" title="class in org.biojava.nbio.core.sequence.features"><span class="strong">QualityFeature</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/features/QualityFeature.html" title="type parameter in QualityFeature">S</a> extends <a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="./org/biojava/nbio/core/sequence/features/QualityFeature.html" title="type parameter in QualityFeature">C</a>&gt;,<a href="./org/biojava/nbio/core/sequence/features/QualityFeature.html" title="type parameter in QualityFeature">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/features/package-summary.html">org.biojava.nbio.core.sequence.features</a></dt>
<dd>
<div class="block">DNA Sequences produced by modern sequencers usually have quality informaion
 attached to them.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/QualityFeature.html#QualityFeature(java.lang.String,%20java.lang.String)">QualityFeature(String, String)</a></span> - Constructor for class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/QualityFeature.html" title="class in org.biojava.nbio.core.sequence.features">QualityFeature</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/features/QuantityFeature.html" title="class in org.biojava.nbio.core.sequence.features"><span class="strong">QuantityFeature</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/features/QuantityFeature.html" title="type parameter in QuantityFeature">S</a> extends <a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="./org/biojava/nbio/core/sequence/features/QuantityFeature.html" title="type parameter in QuantityFeature">C</a>&gt;,<a href="./org/biojava/nbio/core/sequence/features/QuantityFeature.html" title="type parameter in QuantityFeature">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/features/package-summary.html">org.biojava.nbio.core.sequence.features</a></dt>
<dd>
<div class="block">It is common to have a numerical value or values associated with a feature.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/QuantityFeature.html#QuantityFeature(java.lang.String,%20java.lang.String)">QuantityFeature(String, String)</a></span> - Constructor for class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/QuantityFeature.html" title="class in org.biojava.nbio.core.sequence.features">QuantityFeature</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_R_">
<!--   -->
</a>
<h2 class="title">R</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#read()">read()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava.nbio.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>
<div class="block">Reads a single character.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#read(char[],%20int,%20int)">read(char[], int, int)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava.nbio.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>
<div class="block">Reads characters into a portion of an array.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/UncompressInputStream.html#read()">read()</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/UncompressInputStream.html" title="class in org.biojava.nbio.core.util">UncompressInputStream</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/UncompressInputStream.html#read(byte[],%20int,%20int)">read(byte[], int, int)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/UncompressInputStream.html" title="class in org.biojava.nbio.core.util">UncompressInputStream</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#readableFiles">readableFiles</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#readFastaDNASequence(java.io.File,%20boolean)">readFastaDNASequence(File, boolean)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaReaderHelper.html" title="class in org.biojava.nbio.core.sequence.io">FastaReaderHelper</a></dt>
<dd>
<div class="block">Selecting lazySequenceLoad=true will parse the FASTA file and figure out the accessionid and offsets and return sequence objects
 that can in the future read the sequence from the disk.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#readFastaDNASequence(java.io.InputStream)">readFastaDNASequence(InputStream)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaReaderHelper.html" title="class in org.biojava.nbio.core.sequence.io">FastaReaderHelper</a></dt>
<dd>
<div class="block">Read a fasta DNA sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#readFastaDNASequence(java.io.File)">readFastaDNASequence(File)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaReaderHelper.html" title="class in org.biojava.nbio.core.sequence.io">FastaReaderHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#readFastaProteinSequence(java.io.File)">readFastaProteinSequence(File)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaReaderHelper.html" title="class in org.biojava.nbio.core.sequence.io">FastaReaderHelper</a></dt>
<dd>
<div class="block">Read a fasta file containing amino acids with setup that would handle most
 cases.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#readFastaProteinSequence(java.io.InputStream)">readFastaProteinSequence(InputStream)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaReaderHelper.html" title="class in org.biojava.nbio.core.sequence.io">FastaReaderHelper</a></dt>
<dd>
<div class="block">Read a fasta file containing amino acids with setup that would handle most
 cases.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#readGenbankDNASequence(java.io.File,%20boolean)">readGenbankDNASequence(File, boolean)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReaderHelper</a></dt>
<dd>
<div class="block">Selecting lazySequenceLoad=true will parse the Genbank file and figure out the accessionid and offsets and return sequence objects
 that can in the future read the sequence from the disk.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#readGenbankDNASequence(java.io.InputStream)">readGenbankDNASequence(InputStream)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReaderHelper</a></dt>
<dd>
<div class="block">Read a Genbank DNA sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#readGenbankDNASequence(java.io.File)">readGenbankDNASequence(File)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReaderHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#readGenbankProteinSequence(java.io.File,%20boolean)">readGenbankProteinSequence(File, boolean)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReaderHelper</a></dt>
<dd>
<div class="block">Selecting lazySequenceLoad=true will parse the Genbank file and figure out the accessionid and offsets and return sequence objects
 that can in the future read the sequence from the disk.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#readGenbankProteinSequence(java.io.File)">readGenbankProteinSequence(File)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReaderHelper</a></dt>
<dd>
<div class="block">Read a Genbank file containing amino acids with setup that would handle most
 cases.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#readGenbankProteinSequence(java.io.InputStream)">readGenbankProteinSequence(InputStream)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReaderHelper</a></dt>
<dd>
<div class="block">Read a Genbank file containing amino acids with setup that would handle most
 cases.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#readLine()">readLine()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava.nbio.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>
<div class="block">Reads a line of text.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#ready()">ready()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava.nbio.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>
<div class="block">Tells whether this stream is ready to be read.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#REFERENCE_TAG">REFERENCE_TAG</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#refp">refp</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#refRange">refRange</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#REMARK_TAG">REMARK_TAG</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.SequenceIterator.html#remove()">remove()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.SequenceIterator.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin.SequenceIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/SoftHashMap.html#remove(java.lang.Object)">remove(Object)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/SoftHashMap.html" title="class in org.biojava.nbio.core.util">SoftHashMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#removeAlignedSequence(S)">removeAlignedSequence(S)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava.nbio.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Remove a sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html#removeCDS(java.lang.String)">removeCDS(String)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>
<div class="block">Remove a CDS or coding sequence from the transcript sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/GeneSequence.html#removeExon(java.lang.String)">removeExon(String)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Remove the exon sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#removeFeature(org.biojava.nbio.core.sequence.features.FeatureInterface)">removeFeature(FeatureInterface&lt;AbstractSequence&lt;C&gt;, C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Remove a feature from the sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/ChromosomeSequence.html#removeGeneSequence(java.lang.String)">removeGeneSequence(String)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/ChromosomeSequence.html" title="class in org.biojava.nbio.core.sequence">ChromosomeSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/GeneSequence.html#removeIntron(java.lang.String)">removeIntron(String)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Remove the intron by accession</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#removeNote(java.lang.String)">removeNote(String)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/GeneSequence.html#removeTranscript(java.lang.String)">removeTranscript(String)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Remove the transcript sequence from the gene</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#reset()">reset()</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava.nbio.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>
<div class="block">Resets the stream to the most recent mark.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/CRC64Checksum.html#reset()">reset()</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/CRC64Checksum.html" title="class in org.biojava.nbio.core.util">CRC64Checksum</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Point.Resolver.html#resolve(T)">resolve(T)</a></span> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Point.Resolver.html" title="interface in org.biojava.nbio.core.sequence.location.template">Point.Resolver</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html#reverse(int)">reverse(int)</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/FuzzyPoint.html" title="class in org.biojava.nbio.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html#reverse(int)">reverse(int)</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html#reverse(int,%20int)">reverse(int, int)</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/Point.html#reverse(int)">reverse(int)</a></span> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/Point.html" title="interface in org.biojava.nbio.core.sequence.location.template">Point</a></dt>
<dd>
<div class="block">Returns the equivalent position on the reverse strand</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/views/ReversedSequenceView.html" title="class in org.biojava.nbio.core.sequence.views"><span class="strong">ReversedSequenceView</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/views/ReversedSequenceView.html" title="type parameter in ReversedSequenceView">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/views/package-summary.html">org.biojava.nbio.core.sequence.views</a></dt>
<dd>
<div class="block">For a given sequence this class will return the base at the reversed
 position i.e.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/ReversedSequenceView.html#ReversedSequenceView(org.biojava.nbio.core.sequence.template.Sequence)">ReversedSequenceView(Sequence&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/ReversedSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">ReversedSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#reverseSequence(org.biojava.nbio.core.sequence.template.Sequence)">reverseSequence(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>
<div class="block">Reverses and (if possible) complements the Sequence so as to represent
 the reverse strand (if one exists).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#rnaAminoAcidTranslator(org.biojava.nbio.core.sequence.transcription.RNAToAminoAcidTranslator)">rnaAminoAcidTranslator(RNAToAminoAcidTranslator)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#rnaCompounds(org.biojava.nbio.core.sequence.template.CompoundSet)">rnaCompounds(CompoundSet&lt;NucleotideCompound&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/compound/RNACompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound"><span class="strong">RNACompoundSet</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/compound/package-summary.html">org.biojava.nbio.core.sequence.compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/RNACompoundSet.html#RNACompoundSet()">RNACompoundSet()</a></span> - Constructor for class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/RNACompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">RNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#rnaCreator(org.biojava.nbio.core.sequence.io.template.SequenceCreatorInterface)">rnaCreator(SequenceCreatorInterface&lt;NucleotideCompound&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/RNASequence.html" title="class in org.biojava.nbio.core.sequence"><span class="strong">RNASequence</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">RNASequence where RNACompoundSet are the allowed values</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/RNASequence.html#RNASequence(java.lang.String)">RNASequence(String)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/RNASequence.html" title="class in org.biojava.nbio.core.sequence">RNASequence</a></dt>
<dd>
<div class="block">Create a RNA sequence from a String</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/RNASequence.html#RNASequence(org.biojava.nbio.core.sequence.template.ProxySequenceReader)">RNASequence(ProxySequenceReader&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/RNASequence.html" title="class in org.biojava.nbio.core.sequence">RNASequence</a></dt>
<dd>
<div class="block">Create a RNA aequence from a proxy reader</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/RNASequence.html#RNASequence(java.lang.String,%20org.biojava.nbio.core.sequence.template.CompoundSet)">RNASequence(String, CompoundSet&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/RNASequence.html" title="class in org.biojava.nbio.core.sequence">RNASequence</a></dt>
<dd>
<div class="block">Create a RNA sequence from a string with a user defined RNA compound set</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/RNASequence.html#RNASequence(org.biojava.nbio.core.sequence.template.ProxySequenceReader,%20org.biojava.nbio.core.sequence.template.CompoundSet)">RNASequence(ProxySequenceReader&lt;NucleotideCompound&gt;, CompoundSet&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/RNASequence.html" title="class in org.biojava.nbio.core.sequence">RNASequence</a></dt>
<dd>
<div class="block">Create a RNA sequence from a proxy reader and user defined RNA compound set</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/RNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io"><span class="strong">RNASequenceCreator</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">Used to create a RNA sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#RNASequenceCreator(org.biojava.nbio.core.sequence.template.CompoundSet)">RNASequenceCreator(CompoundSet&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/RNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">RNASequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/views/RnaSequenceView.html" title="class in org.biojava.nbio.core.sequence.views"><span class="strong">RnaSequenceView</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/views/package-summary.html">org.biojava.nbio.core.sequence.views</a></dt>
<dd>
<div class="block">Attempts to do on the fly translation of RNA by not requesting the compounds
 until asked.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/RnaSequenceView.html#RnaSequenceView(org.biojava.nbio.core.sequence.template.Sequence)">RnaSequenceView(Sequence&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/RnaSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/RnaSequenceView.html#RnaSequenceView(org.biojava.nbio.core.sequence.template.Sequence,%20org.biojava.nbio.core.sequence.template.CompoundSet)">RnaSequenceView(Sequence&lt;NucleotideCompound&gt;, CompoundSet&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/RnaSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html" title="class in org.biojava.nbio.core.sequence.transcription"><span class="strong">RNAToAminoAcidTranslator</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/transcription/package-summary.html">org.biojava.nbio.core.sequence.transcription</a></dt>
<dd>
<div class="block">Takes a <a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> of <a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound"><code>NucleotideCompound</code></a> which should represent
 an RNA sequence (<a href="./org/biojava/nbio/core/sequence/RNASequence.html" title="class in org.biojava.nbio.core.sequence"><code>RNASequence</code></a> is good for this) and returns a list of
 <a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> which hold <a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava.nbio.core.sequence.compound"><code>AminoAcidCompound</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#RNAToAminoAcidTranslator(org.biojava.nbio.core.sequence.io.template.SequenceCreatorInterface,%20org.biojava.nbio.core.sequence.template.CompoundSet,%20org.biojava.nbio.core.sequence.template.CompoundSet,%20org.biojava.nbio.core.sequence.template.CompoundSet,%20org.biojava.nbio.core.sequence.transcription.Table,%20boolean,%20boolean,%20boolean)">RNAToAminoAcidTranslator(SequenceCreatorInterface&lt;AminoAcidCompound&gt;, CompoundSet&lt;NucleotideCompound&gt;, CompoundSet&lt;Table.Codon&gt;, CompoundSet&lt;AminoAcidCompound&gt;, Table, boolean, boolean, boolean)</a></span> - Constructor for class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html" title="class in org.biojava.nbio.core.sequence.transcription">RNAToAminoAcidTranslator</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>Retained for backwards compatability, setting
             <a href="./org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#stopAtStopCodons"><code>RNAToAminoAcidTranslator.stopAtStopCodons</code></a> to <code>false</code></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#RNAToAminoAcidTranslator(org.biojava.nbio.core.sequence.io.template.SequenceCreatorInterface,%20org.biojava.nbio.core.sequence.template.CompoundSet,%20org.biojava.nbio.core.sequence.template.CompoundSet,%20org.biojava.nbio.core.sequence.template.CompoundSet,%20org.biojava.nbio.core.sequence.transcription.Table,%20boolean,%20boolean,%20boolean,%20boolean)">RNAToAminoAcidTranslator(SequenceCreatorInterface&lt;AminoAcidCompound&gt;, CompoundSet&lt;NucleotideCompound&gt;, CompoundSet&lt;Table.Codon&gt;, CompoundSet&lt;AminoAcidCompound&gt;, Table, boolean, boolean, boolean, boolean)</a></span> - Constructor for class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html" title="class in org.biojava.nbio.core.sequence.transcription">RNAToAminoAcidTranslator</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#RNAToAminoAcidTranslator(org.biojava.nbio.core.sequence.io.template.SequenceCreatorInterface,%20org.biojava.nbio.core.sequence.template.CompoundSet,%20org.biojava.nbio.core.sequence.template.CompoundSet,%20org.biojava.nbio.core.sequence.template.CompoundSet,%20org.biojava.nbio.core.sequence.transcription.Table,%20boolean,%20boolean,%20boolean,%20boolean,%20boolean)">RNAToAminoAcidTranslator(SequenceCreatorInterface&lt;AminoAcidCompound&gt;, CompoundSet&lt;NucleotideCompound&gt;, CompoundSet&lt;Table.Codon&gt;, CompoundSet&lt;AminoAcidCompound&gt;, Table, boolean, boolean, boolean, boolean, boolean)</a></span> - Constructor for class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html" title="class in org.biojava.nbio.core.sequence.transcription">RNAToAminoAcidTranslator</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_S_">
<!--   -->
</a>
<h2 class="title">S</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#sectp">sectp</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/Hashcoder.html#SEED">SEED</a></span> - Static variable in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/Hashcoder.html" title="class in org.biojava.nbio.core.util">Hashcoder</a></dt>
<dd>
<div class="block">An initial value for a <code>hashCode</code>, to which we add
 contributions from fields.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/XMLHelper.html#selectElements(org.w3c.dom.Element,%20java.lang.String)">selectElements(Element, String)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/XMLHelper.html" title="class in org.biojava.nbio.core.util">XMLHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/XMLHelper.html#selectParentElement(org.w3c.dom.Element,%20java.lang.String)">selectParentElement(Element, String)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/XMLHelper.html" title="class in org.biojava.nbio.core.util">XMLHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/XMLHelper.html#selectSingleElement(org.w3c.dom.Element,%20java.lang.String)">selectSingleElement(Element, String)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/XMLHelper.html" title="class in org.biojava.nbio.core.util">XMLHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#seqArraySize(int)">seqArraySize(int)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><span class="strong">Sequence</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="type parameter in Sequence">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>
<div class="block">Main interface for defining a collection of Compounds and accessing them
 using biological indexes</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html" title="class in org.biojava.nbio.core.sequence.storage"><span class="strong">SequenceAsStringHelper</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html" title="type parameter in SequenceAsStringHelper">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/storage/package-summary.html">org.biojava.nbio.core.sequence.storage</a></dt>
<dd>
<div class="block">This is a common method that can be used across multiple storage/proxy implementations to
 handle Negative strand and other interesting elements of sequence data.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#SequenceAsStringHelper()">SequenceAsStringHelper()</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html" title="class in org.biojava.nbio.core.sequence.storage">SequenceAsStringHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/SequenceComparator.html" title="class in org.biojava.nbio.core.sequence"><span class="strong">SequenceComparator</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">Used to sort sequences</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/SequenceComparator.html#SequenceComparator()">SequenceComparator()</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/SequenceComparator.html" title="class in org.biojava.nbio.core.sequence">SequenceComparator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template"><span class="strong">SequenceCreatorInterface</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html" title="type parameter in SequenceCreatorInterface">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava/nbio/core/sequence/io/template/package-summary.html">org.biojava.nbio.core.sequence.io.template</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#sequenceEquality(org.biojava.nbio.core.sequence.template.Sequence,%20org.biojava.nbio.core.sequence.template.Sequence)">sequenceEquality(Sequence&lt;C&gt;, Sequence&lt;C&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">A case-sensitive manner of comparing two sequence objects together.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#sequenceEqualityIgnoreCase(org.biojava.nbio.core.sequence.template.Sequence,%20org.biojava.nbio.core.sequence.template.Sequence)">sequenceEqualityIgnoreCase(Sequence&lt;C&gt;, Sequence&lt;C&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">A case-insensitive manner of comparing two sequence objects together.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader"><span class="strong">SequenceFileProxyLoader</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="type parameter in SequenceFileProxyLoader">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/loader/package-summary.html">org.biojava.nbio.core.sequence.loader</a></dt>
<dd>
<div class="block">This class represents the storage container of a sequence stored in a fasta file where
 the initial parsing of the file we store the offset and length of the sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#SequenceFileProxyLoader(java.io.File,%20org.biojava.nbio.core.sequence.io.template.SequenceParserInterface,%20long,%20int,%20org.biojava.nbio.core.sequence.template.CompoundSet)">SequenceFileProxyLoader(File, SequenceParserInterface, long, int, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/template/SequenceHeaderParserInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template"><span class="strong">SequenceHeaderParserInterface</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/io/template/SequenceHeaderParserInterface.html" title="type parameter in SequenceHeaderParserInterface">S</a> extends <a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;?&gt;,<a href="./org/biojava/nbio/core/sequence/io/template/SequenceHeaderParserInterface.html" title="type parameter in SequenceHeaderParserInterface">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava/nbio/core/sequence/io/template/package-summary.html">org.biojava.nbio.core.sequence.io.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/location/SequenceLocation.html" title="class in org.biojava.nbio.core.sequence.location"><span class="strong">SequenceLocation</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/location/SequenceLocation.html" title="type parameter in SequenceLocation">S</a> extends <a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="./org/biojava/nbio/core/sequence/location/SequenceLocation.html" title="type parameter in SequenceLocation">C</a>&gt;,<a href="./org/biojava/nbio/core/sequence/location/SequenceLocation.html" title="type parameter in SequenceLocation">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a></dt>
<dd>
<div class="block">A location in a sequence that keeps a reference to its parent sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SequenceLocation.html#SequenceLocation(int,%20int,%20S)">SequenceLocation(int, int, S)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SequenceLocation.html" title="class in org.biojava.nbio.core.sequence.location">SequenceLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SequenceLocation.html#SequenceLocation(int,%20int,%20S,%20org.biojava.nbio.core.sequence.Strand,%20boolean,%20java.util.List)">SequenceLocation(int, int, S, Strand, boolean, List&lt;Location&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SequenceLocation.html" title="class in org.biojava.nbio.core.sequence.location">SequenceLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SequenceLocation.html#SequenceLocation(int,%20int,%20S,%20org.biojava.nbio.core.sequence.Strand)">SequenceLocation(int, int, S, Strand)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SequenceLocation.html" title="class in org.biojava.nbio.core.sequence.location">SequenceLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SequenceLocation.html#SequenceLocation(org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.location.template.Point,%20S,%20org.biojava.nbio.core.sequence.Strand)">SequenceLocation(Point, Point, S, Strand)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SequenceLocation.html" title="class in org.biojava.nbio.core.sequence.location">SequenceLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template"><span class="strong">SequenceMixin</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>
<div class="block">Provides a set of static methods to be used as static imports when needed
 across multiple Sequence implementations but inheritance gets in the way.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#SequenceMixin()">SequenceMixin()</a></span> - Constructor for class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.SequenceIterator.html" title="class in org.biojava.nbio.core.sequence.template"><span class="strong">SequenceMixin.SequenceIterator</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.SequenceIterator.html" title="type parameter in SequenceMixin.SequenceIterator">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>
<div class="block">A basic sequence iterator which iterates over the given Sequence by
 biological index.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.SequenceIterator.html#SequenceMixin.SequenceIterator(org.biojava.nbio.core.sequence.template.Sequence)">SequenceMixin.SequenceIterator(Sequence&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.SequenceIterator.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin.SequenceIterator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/SequenceOptimizationHints.html" title="class in org.biojava.nbio.core.sequence"><span class="strong">SequenceOptimizationHints</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">A static class that provides optimization hints for memory or performance handling of sequence data.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#SequenceOptimizationHints()">SequenceOptimizationHints()</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/SequenceOptimizationHints.html" title="class in org.biojava.nbio.core.sequence">SequenceOptimizationHints</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/SequenceOptimizationHints.SequenceCollection.html" title="enum in org.biojava.nbio.core.sequence"><span class="strong">SequenceOptimizationHints.SequenceCollection</span></a> - Enum in <a href="./org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/SequenceOptimizationHints.SequenceUsage.html" title="enum in org.biojava.nbio.core.sequence"><span class="strong">SequenceOptimizationHints.SequenceUsage</span></a> - Enum in <a href="./org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/io/template/SequenceParserInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template"><span class="strong">SequenceParserInterface</span></a> - Interface in <a href="./org/biojava/nbio/core/sequence/io/template/package-summary.html">org.biojava.nbio.core.sequence.io.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template"><span class="strong">SequenceProxyView</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="type parameter in SequenceProxyView">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html#SequenceProxyView()">SequenceProxyView()</a></span> - Constructor for class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html#SequenceProxyView(org.biojava.nbio.core.sequence.template.Sequence)">SequenceProxyView(Sequence&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html#SequenceProxyView(org.biojava.nbio.core.sequence.template.Sequence,%20java.lang.Integer,%20java.lang.Integer)">SequenceProxyView(Sequence&lt;C&gt;, Integer, Integer)</a></span> - Constructor for class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>
<div class="block">Main constructor for working with SequenceProxyViews</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/template/SequenceReader.html" title="interface in org.biojava.nbio.core.sequence.template"><span class="strong">SequenceReader</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/template/SequenceReader.html" title="type parameter in SequenceReader">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/util/SequenceTools.html" title="class in org.biojava.nbio.core.util"><span class="strong">SequenceTools</span></a> - Class in <a href="./org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/SequenceTools.html#SequenceTools()">SequenceTools()</a></span> - Constructor for class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/SequenceTools.html" title="class in org.biojava.nbio.core.util">SequenceTools</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/template/SequenceView.html" title="interface in org.biojava.nbio.core.sequence.template"><span class="strong">SequenceView</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/template/SequenceView.html" title="type parameter in SequenceView">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#setAccession(java.lang.String)">setAccession(String)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderParser</a></dt>
<dd>
<div class="block">
 The last accession passed to this routine will always be the one used.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#setAccession(org.biojava.nbio.core.sequence.AccessionID)">setAccession(AccessionID)</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#setAccession(org.biojava.nbio.core.sequence.AccessionID)">setAccession(AccessionID)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#setAnnotationType(org.biojava.nbio.core.sequence.template.AbstractSequence.AnnotationType)">setAnnotationType(AbstractSequence.AnnotationType)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#setBetweenCompounds(boolean)">setBetweenCompounds(boolean)</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#setBioBegin(java.lang.Integer)">setBioBegin(Integer)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#setBioEnd(java.lang.Integer)">setBioEnd(Integer)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html#setBioEnd(java.lang.Integer)">setBioEnd(Integer)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html#setBioStart(java.lang.Integer)">setBioStart(Integer)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html#setChildrenFeatures(java.util.List)">setChildrenFeatures(List&lt;FeatureInterface&lt;S, C&gt;&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">Set the children features</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#setChildrenFeatures(java.util.List)">setChildrenFeatures(List&lt;FeatureInterface&lt;S, C&gt;&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html#setChildrenFeatures(java.util.List)">setChildrenFeatures(List&lt;FeatureInterface&lt;S, C&gt;&gt;)</a></span> - Method in interface org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Set the children features</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/ChromosomeSequence.html#setChromosomeNumber(int)">setChromosomeNumber(int)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/ChromosomeSequence.html" title="class in org.biojava.nbio.core.sequence">ChromosomeSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#setCircular(boolean)">setCircular(boolean)</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#setComment(java.lang.String)">setComment(String)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderParser</a></dt>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html#setComplexFeaturesAppendMode(org.biojava.nbio.core.sequence.location.InsdcParser.complexFeaturesAppendEnum)">setComplexFeaturesAppendMode(InsdcParser.complexFeaturesAppendEnum)</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html" title="class in org.biojava.nbio.core.sequence.location">InsdcParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#setCompoundAt(char,%20int)">setCompoundAt(char, int)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Converts from char to Compound and sets it at the given biological index</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#setCompoundAt(C,%20int)">setCompoundAt(C, int)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Sets the compound at the specified biological index</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#setCompoundSet(org.biojava.nbio.core.sequence.template.CompoundSet)">setCompoundSet(CompoundSet&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#setCompoundSet(org.biojava.nbio.core.sequence.template.CompoundSet)">setCompoundSet(CompoundSet&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#setCompoundSet(org.biojava.nbio.core.sequence.template.CompoundSet)">setCompoundSet(CompoundSet&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#setCompoundSet(org.biojava.nbio.core.sequence.template.CompoundSet)">setCompoundSet(CompoundSet&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#setCompoundSet(org.biojava.nbio.core.sequence.template.CompoundSet)">setCompoundSet(CompoundSet&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Class is immutable & so this is unsupported</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#setCompoundSet(org.biojava.nbio.core.sequence.template.CompoundSet)">setCompoundSet(CompoundSet&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#setCompoundSet(org.biojava.nbio.core.sequence.template.CompoundSet)">setCompoundSet(CompoundSet&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Unsupported</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#setCompoundSet(org.biojava.nbio.core.sequence.template.CompoundSet)">setCompoundSet(CompoundSet&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceReader.html#setCompoundSet(org.biojava.nbio.core.sequence.template.CompoundSet)">setCompoundSet(CompoundSet&lt;C&gt;)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceReader.html" title="interface in org.biojava.nbio.core.sequence.template">SequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/RnaSequenceView.html#setCompoundSet(org.biojava.nbio.core.sequence.template.CompoundSet)">setCompoundSet(CompoundSet&lt;NucleotideCompound&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/RnaSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#setContents(java.lang.String)">setContents(String)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#setContents(java.lang.String)">setContents(String)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#setContents(java.lang.String,%20java.util.ArrayList)">setContents(String, ArrayList)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#setContents(java.lang.String)">setContents(String)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">Once the sequence is retrieved set the contents and make sure everything this is valid</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#setContents(java.lang.String)">setContents(String)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#setContents(java.util.List)">setContents(List&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#setContents(java.lang.String)">setContents(String)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Class is immutable & so this is unsupported</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#setContents(java.lang.String)">setContents(String)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#setContents(java.lang.String)">setContents(String)</a></span> - Method in class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Unsupported</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceReader.html#setContents(java.lang.String)">setContents(String)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceReader.html" title="interface in org.biojava.nbio.core.sequence.template">SequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/RnaSequenceView.html#setContents(java.lang.String)">setContents(String)</a></span> - Method in class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/RnaSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#setDatabase(java.lang.String)">setDatabase(String)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">DBReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#setDatabaseReferences(org.biojava.nbio.core.sequence.features.DatabaseReferenceInterface)">setDatabaseReferences(DatabaseReferenceInterface)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in interface org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Set the description that can be used to describe the feature</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderParser</a></dt>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/Compound.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/DNASequence.html#setDNAType(org.biojava.nbio.core.sequence.DNASequence.DNAType)">setDNAType(DNASequence.DNAType)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#setEnd(org.biojava.nbio.core.sequence.location.template.Point)">setEnd(Point)</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#setFeatureRetriever(org.biojava.nbio.core.sequence.features.FeatureRetriever)">setFeatureRetriever(FeatureRetriever)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#setFeaturesKeyWord(org.biojava.nbio.core.sequence.features.FeaturesKeyWordInterface)">setFeaturesKeyWord(FeaturesKeyWordInterface)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#setGenbankDirectoryCache(java.lang.String)">setGenbankDirectoryCache(String)</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#setId(java.lang.String)">setId(String)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">DBReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#setIdentifier(java.lang.String)">setIdentifier(String)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderParser</a></dt>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#setLineLength(int)">setLineLength(int)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaGeneWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaGeneWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaWriter.html#setLineLength(int)">setLineLength(int)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankWriter.html#setLineLength(int)">setLineLength(int)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankWriter.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaWriter.html#setLineSeparator(java.lang.String)">setLineSeparator(String)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriter</a></dt>
<dd>
<div class="block">Allow an override of operating system line separator for programs that needs a specific CRLF or CR or LF option</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html#setLocation(org.biojava.nbio.core.sequence.location.template.AbstractLocation)">setLocation(AbstractLocation)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">A feature could be a single sequence position like a mutation or a post translational modification of an amino acid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#setLocation(org.biojava.nbio.core.sequence.location.template.AbstractLocation)">setLocation(AbstractLocation)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html#setLocation(org.biojava.nbio.core.sequence.location.template.AbstractLocation)">setLocation(AbstractLocation)</a></span> - Method in interface org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">The new location for this feature.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html#setLongName(java.lang.String)">setLongName(String)</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html#setLongName(java.lang.String)">setLongName(String)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/Compound.html#setLongName(java.lang.String)">setLongName(String)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#setLongName(java.lang.String)">setLongName(String)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html#setLongName(java.lang.String)">setLongName(String)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#setLowercaseToNull(org.biojava.nbio.core.sequence.ProteinSequence,%20java.lang.Object[])">setLowercaseToNull(ProteinSequence, Object[])</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">CasePreservingProteinSequenceCreator</a></dt>
<dd>
<div class="block">Takes a <a href="./org/biojava/nbio/core/sequence/ProteinSequence.html" title="class in org.biojava.nbio.core.sequence"><code>ProteinSequence</code></a> which was created by a 
 <a href="./org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io"><code>CasePreservingProteinSequenceCreator</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html#setMolecularWeight(java.lang.Float)">setMolecularWeight(Float)</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html#setMolecularWeight(java.lang.Float)">setMolecularWeight(Float)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/Compound.html#setMolecularWeight(java.lang.Float)">setMolecularWeight(Float)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#setMolecularWeight(java.lang.Float)">setMolecularWeight(Float)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html#setMolecularWeight(java.lang.Float)">setMolecularWeight(Float)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/Qualifier.html#setName(java.lang.String)">setName(String)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/Qualifier.html" title="class in org.biojava.nbio.core.sequence.features">Qualifier</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#setName(java.lang.String)">setName(String)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderParser</a></dt>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/Qualifier.html#setNeedsQuotes(boolean)">setNeedsQuotes(boolean)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/Qualifier.html" title="class in org.biojava.nbio.core.sequence.features">Qualifier</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#setNotesList(java.util.ArrayList)">setNotesList(ArrayList&lt;String&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#setOriginalHeader(java.lang.String)">setOriginalHeader(String)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/ProteinSequence.html#setParentDNASequence(org.biojava.nbio.core.sequence.template.AbstractSequence,%20java.lang.Integer,%20java.lang.Integer)">setParentDNASequence(AbstractSequence&lt;NucleotideCompound&gt;, Integer, Integer)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/ProteinSequence.html" title="class in org.biojava.nbio.core.sequence">ProteinSequence</a></dt>
<dd>
<div class="block">A Protein sequence can be stand alone or loaded from a transcript
 sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html#setParentFeature(org.biojava.nbio.core.sequence.features.FeatureInterface)">setParentFeature(FeatureInterface&lt;S, C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">A feature can be the child or contained by a parent feature.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#setParentFeature(org.biojava.nbio.core.sequence.features.FeatureInterface)">setParentFeature(FeatureInterface&lt;S, C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html#setParentFeature(org.biojava.nbio.core.sequence.features.FeatureInterface)">setParentFeature(FeatureInterface&lt;S, C&gt;)</a></span> - Method in interface org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Set the parent feature</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#setParentSequence(org.biojava.nbio.core.sequence.template.AbstractSequence)">setParentSequence(AbstractSequence&lt;?&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#setPartialOn3prime(boolean)">setPartialOn3prime(boolean)</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#setPartialOn5prime(boolean)">setPartialOn5prime(boolean)</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html#setPosition(int)">setPosition(int)</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#setProperties(java.util.LinkedHashMap)">setProperties(LinkedHashMap&lt;String, String&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">DBReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#setProxySequenceReader(org.biojava.nbio.core.sequence.template.SequenceReader)">setProxySequenceReader(SequenceReader&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Very important method that allows external mappings of sequence data and features.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html#setQualifiers(java.util.HashMap)">setQualifiers(HashMap&lt;String, Qualifier&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#setQualifiers(java.util.HashMap)">setQualifiers(HashMap&lt;String, Qualifier&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html#setQualifiers(java.util.HashMap)">setQualifiers(HashMap&lt;String, Qualifier&gt;)</a></span> - Method in interface org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Set the qualifiers</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/QualityFeature.html#setQualities(java.util.List)">setQualities(List&lt;Number&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/QualityFeature.html" title="class in org.biojava.nbio.core.sequence.features">QualityFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/QuantityFeature.html#setQuantities(java.util.List)">setQuantities(List&lt;Number&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/QuantityFeature.html" title="class in org.biojava.nbio.core.sequence.features">QuantityFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html#setSequence(org.biojava.nbio.core.sequence.template.Sequence)">setSequence(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SequenceLocation.html#setSequence(S)">setSequence(S)</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SequenceLocation.html" title="class in org.biojava.nbio.core.sequence.location">SequenceLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#setSequenceCollection(org.biojava.nbio.core.sequence.SequenceOptimizationHints.SequenceCollection)">setSequenceCollection(SequenceOptimizationHints.SequenceCollection)</a></span> - Static method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/SequenceOptimizationHints.html" title="class in org.biojava.nbio.core.sequence">SequenceOptimizationHints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#setSequenceScore(java.lang.Double)">setSequenceScore(Double)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#setSequenceUsage(org.biojava.nbio.core.sequence.SequenceOptimizationHints.SequenceUsage)">setSequenceUsage(SequenceOptimizationHints.SequenceUsage)</a></span> - Static method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/SequenceOptimizationHints.html" title="class in org.biojava.nbio.core.sequence">SequenceOptimizationHints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html#setShortDescription(java.lang.String)">setShortDescription(String)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#setShortDescription(java.lang.String)">setShortDescription(String)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html#setShortDescription(java.lang.String)">setShortDescription(String)</a></span> - Method in interface org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Set the short description that can be used to describe the feature</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html#setShortName(java.lang.String)">setShortName(String)</a></span> - Method in class org.biojava.nbio.core.sequence.compound.<a href="./org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html#setShortName(java.lang.String)">setShortName(String)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/Compound.html#setShortName(java.lang.String)">setShortName(String)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#setShortName(java.lang.String)">setShortName(String)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html#setShortName(java.lang.String)">setShortName(String)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html#setSource(java.lang.String)">setSource(String)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">Set the feature source</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#setSource(java.lang.String)">setSource(String)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html#setSource(java.lang.String)">setSource(String)</a></span> - Method in interface org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Change the source of the FeatureInterface.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#setSource(java.lang.String)">setSource(String)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Added support for the source of this sequence for GFF3 export</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#setStart(org.biojava.nbio.core.sequence.location.template.Point)">setStart(Point)</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/GeneSequence.html#setStrand(org.biojava.nbio.core.sequence.Strand)">setStrand(Strand)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#setStrand(org.biojava.nbio.core.sequence.Strand)">setStrand(Strand)</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html#setStringSequence(java.lang.String)">setStringSequence(String)</a></span> - Method in class org.biojava.nbio.core.sequence.edits.<a href="./org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#setSubLocations(java.util.List)">setSubLocations(List&lt;Location&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#setTaxonomy(org.biojava.nbio.core.sequence.TaxonomyID)">setTaxonomy(TaxonomyID)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/ConcurrencyTools.html#setThreadPool(java.util.concurrent.ThreadPoolExecutor)">setThreadPool(ThreadPoolExecutor)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/ConcurrencyTools.html" title="class in org.biojava.nbio.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Sets thread pool to any given <a href="http://java.sun.com/javase/6/docs/api/java/util/concurrent/ThreadPoolExecutor.html?is-external=true" title="class or interface in java.util.concurrent"><code>ThreadPoolExecutor</code></a> to allow use of an alternative execution style.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/ConcurrencyTools.html#setThreadPoolCPUsAvailable(int)">setThreadPoolCPUsAvailable(int)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/ConcurrencyTools.html" title="class in org.biojava.nbio.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Sets thread pool to reserve a given number of processor cores for foreground or other use.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/ConcurrencyTools.html#setThreadPoolCPUsFraction(float)">setThreadPoolCPUsFraction(float)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/ConcurrencyTools.html" title="class in org.biojava.nbio.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Sets thread pool to a given fraction of the available processors.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/ConcurrencyTools.html#setThreadPoolDefault()">setThreadPoolDefault()</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/ConcurrencyTools.html" title="class in org.biojava.nbio.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Sets thread pool to default of 1 background thread for each processor core.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/ConcurrencyTools.html#setThreadPoolSingle()">setThreadPoolSingle()</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/ConcurrencyTools.html" title="class in org.biojava.nbio.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Sets thread pool to a single background thread.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/ConcurrencyTools.html#setThreadPoolSize(int)">setThreadPoolSize(int)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/ConcurrencyTools.html" title="class in org.biojava.nbio.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Sets thread pool to given size.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html#setType(java.lang.String)">setType(String)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">Set the feature type</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#setType(java.lang.String)">setType(String)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html#setType(java.lang.String)">setType(String)</a></span> - Method in interface org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Change the type of this feature.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html#setUncertain(boolean)">setUncertain(boolean)</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#setUniprotbaseURL(java.lang.String)">setUniprotbaseURL(String)</a></span> - Static method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#setUniprotDirectoryCache(java.lang.String)">setUniprotDirectoryCache(String)</a></span> - Static method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html#setUnknown(boolean)">setUnknown(boolean)</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#setUserCollection(java.util.Collection)">setUserCollection(Collection&lt;Object&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html#setUserObject(java.lang.Object)">setUserObject(Object)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">Allow the user to associate an object with the feature.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#setUserObject(java.lang.Object)">setUserObject(Object)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html#setUserObject(java.lang.Object)">setUserObject(Object)</a></span> - Method in interface org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/Qualifier.html#setValue(java.lang.String)">setValue(String)</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/Qualifier.html" title="class in org.biojava.nbio.core.sequence.features">Qualifier</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#setVersion(int)">setVersion(int)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderParser</a></dt>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#shuffle(org.biojava.nbio.core.sequence.template.Sequence)">shuffle(Sequence&lt;C&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Implements sequence shuffling by first materializing the given
 <a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> into a <a href="http://java.sun.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a>, applying
 <a href="http://java.sun.com/javase/6/docs/api/java/util/Collections.html?is-external=true#shuffle(java.util.List)" title="class or interface in java.util"><code>Collections.shuffle(List)</code></a> and then returning the shuffled
 elements in a new instance of <code>SequenceBackingStore</code> which behaves
 as a <a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/ConcurrencyTools.html#shutdown()">shutdown()</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/ConcurrencyTools.html" title="class in org.biojava.nbio.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Disables new tasks from being submitted and closes the thread pool cleanly.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/ConcurrencyTools.html#shutdownAndAwaitTermination()">shutdownAndAwaitTermination()</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/ConcurrencyTools.html" title="class in org.biojava.nbio.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Closes the thread pool.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location"><span class="strong">SimpleLocation</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a></dt>
<dd>
<div class="block">Very basic implementation of the Location interface which defines a series
 of simple constructors.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html#SimpleLocation(int,%20int)">SimpleLocation(int, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html#SimpleLocation(org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.location.template.Point)">SimpleLocation(Point, Point)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html#SimpleLocation(int,%20int,%20org.biojava.nbio.core.sequence.Strand)">SimpleLocation(int, int, Strand)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html#SimpleLocation(int,%20int,%20org.biojava.nbio.core.sequence.Strand,%20java.util.List)">SimpleLocation(int, int, Strand, List&lt;Location&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html#SimpleLocation(org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.Strand)">SimpleLocation(Point, Point, Strand)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html#SimpleLocation(org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.Strand,%20org.biojava.nbio.core.sequence.AccessionID)">SimpleLocation(Point, Point, Strand, AccessionID)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html#SimpleLocation(org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.Strand,%20boolean,%20org.biojava.nbio.core.sequence.AccessionID)">SimpleLocation(Point, Point, Strand, boolean, AccessionID)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html#SimpleLocation(org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.Strand,%20boolean,%20boolean)">SimpleLocation(Point, Point, Strand, boolean, boolean)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html#SimpleLocation(int,%20int,%20org.biojava.nbio.core.sequence.Strand,%20org.biojava.nbio.core.sequence.location.template.Location...)">SimpleLocation(int, int, Strand, Location...)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html#SimpleLocation(org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.Strand,%20org.biojava.nbio.core.sequence.location.template.Location...)">SimpleLocation(Point, Point, Strand, Location...)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html#SimpleLocation(org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.Strand,%20boolean,%20org.biojava.nbio.core.sequence.location.template.Location...)">SimpleLocation(Point, Point, Strand, boolean, Location...)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html#SimpleLocation(org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.Strand,%20boolean,%20java.util.List)">SimpleLocation(Point, Point, Strand, boolean, List&lt;Location&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html#SimpleLocation(org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.Strand,%20java.util.List)">SimpleLocation(Point, Point, Strand, List&lt;Location&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html#SimpleLocation(org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.location.template.Point,%20org.biojava.nbio.core.sequence.Strand,%20boolean,%20boolean,%20java.util.List)">SimpleLocation(Point, Point, Strand, boolean, boolean, List&lt;Location&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location"><span class="strong">SimplePoint</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a></dt>
<dd>
<div class="block">Basic implementation of the Point interface.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html#SimplePoint()">SimplePoint()</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html#SimplePoint(int)">SimplePoint(int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html#SimplePoint(int,%20boolean,%20boolean)">SimplePoint(int, boolean, boolean)</a></span> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage"><span class="strong">SingleCompoundSequenceReader</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="type parameter in SingleCompoundSequenceReader">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/storage/package-summary.html">org.biojava.nbio.core.sequence.storage</a></dt>
<dd>
<div class="block">An implementation of the SequenceReader interface which for every
 call will return only 1 compound (given to it during construction; a String
 is also valid but will require a CompoundSet).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#SingleCompoundSequenceReader(java.lang.String,%20org.biojava.nbio.core.sequence.template.CompoundSet,%20int)">SingleCompoundSequenceReader(String, CompoundSet&lt;C&gt;, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Public constructor to be used with String based constructor</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#SingleCompoundSequenceReader(C,%20org.biojava.nbio.core.sequence.template.CompoundSet,%20int)">SingleCompoundSequenceReader(C, CompoundSet&lt;C&gt;, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Build the object with a compound rather than a String</div>
</dd>
<dt><a href="./org/biojava/nbio/core/util/SingleLinkageClusterer.html" title="class in org.biojava.nbio.core.util"><span class="strong">SingleLinkageClusterer</span></a> - Class in <a href="./org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>
<div class="block">An implementation of a single linkage clusterer
 
 See http://en.wikipedia.org/wiki/Single-linkage_clustering</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/SingleLinkageClusterer.html#SingleLinkageClusterer(double[][],%20boolean)">SingleLinkageClusterer(double[][], boolean)</a></span> - Constructor for class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/SingleLinkageClusterer.html" title="class in org.biojava.nbio.core.util">SingleLinkageClusterer</a></dt>
<dd>
<div class="block">Constructs a new SingleLinkageClusterer
 Subsequently use <a href="./org/biojava/nbio/core/util/SingleLinkageClusterer.html#getDendrogram()"><code>SingleLinkageClusterer.getDendrogram()</code></a> to get the full tree  
 or <a href="./org/biojava/nbio/core/util/SingleLinkageClusterer.html#getClusters(double)"><code>SingleLinkageClusterer.getClusters(double)</code></a> to get the clusters at a certain cutoff in the tree
 Please note that the matrix will be altered during the clustering procedure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html#singleLocationPattern">singleLocationPattern</a></span> - Static variable in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/InsdcParser.html" title="class in org.biojava.nbio.core.sequence.location">InsdcParser</a></dt>
<dd>
<div class="block">parse a location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/FlatFileCache.html#size()">size()</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/FlatFileCache.html" title="class in org.biojava.nbio.core.util">FlatFileCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/SoftHashMap.html#size()">size()</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/SoftHashMap.html" title="class in org.biojava.nbio.core.util">SoftHashMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#skip(long)">skip(long)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava.nbio.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>
<div class="block">Skips characters.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/UncompressInputStream.html#skip(long)">skip(long)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/UncompressInputStream.html" title="class in org.biojava.nbio.core.util">UncompressInputStream</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/util/SoftHashMap.html" title="class in org.biojava.nbio.core.util"><span class="strong">SoftHashMap</span></a>&lt;<a href="./org/biojava/nbio/core/util/SoftHashMap.html" title="type parameter in SoftHashMap">K</a>,<a href="./org/biojava/nbio/core/util/SoftHashMap.html" title="type parameter in SoftHashMap">V</a>&gt; - Class in <a href="./org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>
<div class="block">A in memory cache using soft references.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/SoftHashMap.html#SoftHashMap()">SoftHashMap()</a></span> - Constructor for class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/SoftHashMap.html" title="class in org.biojava.nbio.core.util">SoftHashMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/SoftHashMap.html#SoftHashMap(int)">SoftHashMap(int)</a></span> - Constructor for class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/SoftHashMap.html" title="class in org.biojava.nbio.core.util">SoftHashMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#SOURCE_TAG">SOURCE_TAG</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#START_SEQUENCE_TAG">START_SEQUENCE_TAG</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/StartCodonSequence.html" title="class in org.biojava.nbio.core.sequence"><span class="strong">StartCodonSequence</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">Used to map the start codon feature on a gene</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/StartCodonSequence.html#StartCodonSequence(org.biojava.nbio.core.sequence.TranscriptSequence,%20int,%20int)">StartCodonSequence(TranscriptSequence, int, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/StartCodonSequence.html" title="class in org.biojava.nbio.core.sequence">StartCodonSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#stopAtStopCodons(boolean)">stopAtStopCodons(boolean)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>
<div class="block">If set, then the last codon translated in the resulting peptide
 sequence will be the stop codon</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/StopCodonSequence.html" title="class in org.biojava.nbio.core.sequence"><span class="strong">StopCodonSequence</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">Used to map the stop codon sequence on a gene</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/StopCodonSequence.html#StopCodonSequence(org.biojava.nbio.core.sequence.TranscriptSequence,%20int,%20int)">StopCodonSequence(TranscriptSequence, int, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/StopCodonSequence.html" title="class in org.biojava.nbio.core.sequence">StopCodonSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/Strand.html" title="enum in org.biojava.nbio.core.sequence"><span class="strong">Strand</span></a> - Enum in <a href="./org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">Provides a way of representing the strand of a sequence, location
 hit or feature.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/util/StringManipulationHelper.html" title="class in org.biojava.nbio.core.util"><span class="strong">StringManipulationHelper</span></a> - Class in <a href="./org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>
<div class="block">A utility class for common <a href="http://java.sun.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang"><code>String</code></a> manipulation tasks.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader"><span class="strong">StringProxySequenceReader</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="type parameter in StringProxySequenceReader">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/loader/package-summary.html">org.biojava.nbio.core.sequence.loader</a></dt>
<dd>
<div class="block">An example of a ProxySequenceReader that is created from a String.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#StringProxySequenceReader()">StringProxySequenceReader()</a></span> - Constructor for class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#StringProxySequenceReader(java.lang.String,%20org.biojava.nbio.core.sequence.template.CompoundSet)">StringProxySequenceReader(String, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/ConcurrencyTools.html#submit(java.util.concurrent.Callable,%20java.lang.String)">submit(Callable&lt;T&gt;, String)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/ConcurrencyTools.html" title="class in org.biojava.nbio.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Queues up a task and adds a log entry.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/ConcurrencyTools.html#submit(java.util.concurrent.Callable)">submit(Callable&lt;T&gt;)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/ConcurrencyTools.html" title="class in org.biojava.nbio.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Queues up a task and adds a default log entry.</div>
</dd>
</dl>
<a name="_T_">
<!--   -->
</a>
<h2 class="title">T</h2>
<dl>
<dt><a href="./org/biojava/nbio/core/sequence/transcription/Table.html" title="interface in org.biojava.nbio.core.sequence.transcription"><span class="strong">Table</span></a> - Interface in <a href="./org/biojava/nbio/core/sequence/transcription/package-summary.html">org.biojava.nbio.core.sequence.transcription</a></dt>
<dd>
<div class="block">Provides a way of separating us from the specific <a href="./org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava.nbio.core.sequence.io"><code>IUPACParser.IUPACTable</code></a> even
 though this is the only implementing class for the interface.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#table(java.lang.Integer)">table(Integer)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>
<div class="block">Uses the static instance of <a href="./org/biojava/nbio/core/sequence/io/IUPACParser.html" title="class in org.biojava.nbio.core.sequence.io"><code>IUPACParser</code></a> to find instances of
 <a href="./org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava.nbio.core.sequence.io"><code>IUPACParser.IUPACTable</code></a>s by ID.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#table(java.lang.String)">table(String)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>
<div class="block">Uses the static instance of <a href="./org/biojava/nbio/core/sequence/io/IUPACParser.html" title="class in org.biojava.nbio.core.sequence.io"><code>IUPACParser</code></a> to find instances of
 <a href="./org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava.nbio.core.sequence.io"><code>IUPACParser.IUPACTable</code></a>s by its String name</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#table(org.biojava.nbio.core.sequence.transcription.Table)">table(Table)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava.nbio.core.sequence.transcription"><span class="strong">Table.CaseInsensitiveTriplet</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/transcription/package-summary.html">org.biojava.nbio.core.sequence.transcription</a></dt>
<dd>
<div class="block">Class used to hold three nucleotides together and allow for equality
 to be assessed in a case insensitive manner.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#Table.CaseInsensitiveTriplet(org.biojava.nbio.core.sequence.compound.NucleotideCompound,%20org.biojava.nbio.core.sequence.compound.NucleotideCompound,%20org.biojava.nbio.core.sequence.compound.NucleotideCompound)">Table.CaseInsensitiveTriplet(NucleotideCompound, NucleotideCompound, NucleotideCompound)</a></span> - Constructor for class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription"><span class="strong">Table.Codon</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/transcription/package-summary.html">org.biojava.nbio.core.sequence.transcription</a></dt>
<dd>
<div class="block">Instance of a Codon which is 3 <a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound"><code>NucleotideCompound</code></a>s, its
 corresponding <a href="./org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava.nbio.core.sequence.compound"><code>AminoAcidCompound</code></a> and if it is a start or stop codon.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html#Table.Codon(org.biojava.nbio.core.sequence.transcription.Table.CaseInsensitiveTriplet,%20org.biojava.nbio.core.sequence.compound.AminoAcidCompound,%20boolean,%20boolean)">Table.Codon(Table.CaseInsensitiveTriplet, AminoAcidCompound, boolean, boolean)</a></span> - Constructor for class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html#Table.Codon(org.biojava.nbio.core.sequence.transcription.Table.CaseInsensitiveTriplet)">Table.Codon(Table.CaseInsensitiveTriplet)</a></span> - Constructor for class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/TaxonomyID.html" title="class in org.biojava.nbio.core.sequence"><span class="strong">TaxonomyID</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">A sequence can be associated with a species or Taxonomy ID</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/TaxonomyID.html#TaxonomyID(java.lang.String,%20org.biojava.nbio.core.sequence.DataSource)">TaxonomyID(String, DataSource)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/TaxonomyID.html" title="class in org.biojava.nbio.core.sequence">TaxonomyID</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/features/TextFeature.html" title="class in org.biojava.nbio.core.sequence.features"><span class="strong">TextFeature</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/features/TextFeature.html" title="type parameter in TextFeature">S</a> extends <a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="./org/biojava/nbio/core/sequence/features/TextFeature.html" title="type parameter in TextFeature">C</a>&gt;,<a href="./org/biojava/nbio/core/sequence/features/TextFeature.html" title="type parameter in TextFeature">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/features/package-summary.html">org.biojava.nbio.core.sequence.features</a></dt>
<dd>
<div class="block">A implmentation of AbstractFeature</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/TextFeature.html#TextFeature(java.lang.String,%20java.lang.String,%20java.lang.String,%20java.lang.String)">TextFeature(String, String, String, String)</a></span> - Constructor for class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/TextFeature.html" title="class in org.biojava.nbio.core.sequence.features">TextFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#TITLE_TAG">TITLE_TAG</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/ReversedSequenceView.html#toIndex(int)">toIndex(int)</a></span> - Method in class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/ReversedSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">ReversedSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#toList(org.biojava.nbio.core.sequence.template.Sequence)">toList(Sequence&lt;C&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">For the given <a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> this will return a <a href="http://java.sun.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a> filled with
 the Compounds of that <a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/WindowedSequence.html#toStartIndex(int)">toStartIndex(int)</a></span> - Method in class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/WindowedSequence.html" title="class in org.biojava.nbio.core.sequence.views">WindowedSequence</a></dt>
<dd>
<div class="block">Calculates start index according to the equation start = ( (index-1) -
 windowSize) +1</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/AccessionID.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">DBReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.core.sequence.location.<a href="./org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="./org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#toString(int)">toString(int)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava.nbio.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Get a string representation of the MSA with a fixed width</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#toString(org.biojava.nbio.core.sequence.template.LightweightProfile.StringFormat)">toString(LightweightProfile.StringFormat)</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava.nbio.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Support for different MSA formats</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava.nbio.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">String representation of the MSA</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.html#toString()">toString()</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="interface in org.biojava.nbio.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns a simple view of the alignment profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.html#toString(int)">toString(int)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="interface in org.biojava.nbio.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns a formatted view of the alignment profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.html#toString(org.biojava.nbio.core.sequence.template.LightweightProfile.StringFormat)">toString(LightweightProfile.StringFormat)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="interface in org.biojava.nbio.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns a formatted view of the alignment profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#toString(org.biojava.nbio.core.sequence.template.Sequence)">toString(Sequence&lt;C&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Shortcut to <a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#toStringBuilder(org.biojava.nbio.core.sequence.template.Sequence)"><code>SequenceMixin.toStringBuilder(org.biojava.nbio.core.sequence.template.Sequence)</code></a>
 which calls toString() on the resulting object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/CRC64Checksum.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/CRC64Checksum.html" title="class in org.biojava.nbio.core.util">CRC64Checksum</a></dt>
<dd>
<div class="block">Returns a zero-padded 16 character wide string containing the current
 value of this checksum in uppercase hexadecimal format.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#toStringBuilder(org.biojava.nbio.core.sequence.template.Sequence)">toStringBuilder(Sequence&lt;C&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">For the given Sequence this will return a <a href="http://java.sun.com/javase/6/docs/api/java/lang/StringBuilder.html?is-external=true" title="class or interface in java.lang"><code>StringBuilder</code></a> object
 filled with the results of <code>Compound#toString()</code>.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava.nbio.core.sequence.transcription"><span class="strong">TranscriptionEngine</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/transcription/package-summary.html">org.biojava.nbio.core.sequence.transcription</a></dt>
<dd>
<div class="block">Used as a way of encapsulating the data structures required to parse DNA to a
 Protein sequence.</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription"><span class="strong">TranscriptionEngine.Builder</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/transcription/package-summary.html">org.biojava.nbio.core.sequence.transcription</a></dt>
<dd>
<div class="block">This class is the way to create a <code>TranslationEngine</code>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#TranscriptionEngine.Builder()">TranscriptionEngine.Builder()</a></span> - Constructor for class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence"><span class="strong">TranscriptSequence</span></a> - Class in <a href="./org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">This is the sequence if you want to go from a gene sequence to a protein sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html#TranscriptSequence(org.biojava.nbio.core.sequence.GeneSequence,%20int,%20int)">TranscriptSequence(GeneSequence, int, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#translate(F)">translate(F)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/CompoundTranslator.html#translate(F)">translate(F)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/CompoundTranslator.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#translate(org.biojava.nbio.core.sequence.template.Sequence)">translate(Sequence&lt;NucleotideCompound&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>
<div class="block">Quick method to let you go from a CDS to a Peptide quickly.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#translateMany(F)">translateMany(F)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/CompoundTranslator.html#translateMany(F)">translateMany(F)</a></span> - Method in interface org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/CompoundTranslator.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#translateNCodons()">translateNCodons()</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html" title="class in org.biojava.nbio.core.sequence.transcription">RNAToAminoAcidTranslator</a></dt>
<dd>
<div class="block">Indicates if we want to force exact translation of compounds or not i.e.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#translateNCodons(boolean)">translateNCodons(boolean)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/exceptions/TranslationException.html" title="class in org.biojava.nbio.core.exceptions"><span class="strong">TranslationException</span></a> - Exception in <a href="./org/biojava/nbio/core/exceptions/package-summary.html">org.biojava.nbio.core.exceptions</a></dt>
<dd>
<div class="block">Thrown from AbstractCompundTranslator</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/exceptions/TranslationException.html#TranslationException(java.lang.String)">TranslationException(String)</a></span> - Constructor for exception org.biojava.nbio.core.exceptions.<a href="./org/biojava/nbio/core/exceptions/TranslationException.html" title="class in org.biojava.nbio.core.exceptions">TranslationException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/exceptions/TranslationException.html#TranslationException(java.lang.Exception)">TranslationException(Exception)</a></span> - Constructor for exception org.biojava.nbio.core.exceptions.<a href="./org/biojava/nbio/core/exceptions/TranslationException.html" title="class in org.biojava.nbio.core.exceptions">TranslationException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/exceptions/TranslationException.html#TranslationException(java.lang.String,%20java.lang.Exception)">TranslationException(String, Exception)</a></span> - Constructor for exception org.biojava.nbio.core.exceptions.<a href="./org/biojava/nbio/core/exceptions/TranslationException.html" title="class in org.biojava.nbio.core.exceptions">TranslationException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#trimStop(java.util.List)">trimStop(List&lt;AminoAcidCompound&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html" title="class in org.biojava.nbio.core.sequence.transcription">RNAToAminoAcidTranslator</a></dt>
<dd>
<div class="block">Imperfect code.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#trimStop(boolean)">trimStop(boolean)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage"><span class="strong">TwoBitSequenceReader</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html" title="type parameter in TwoBitSequenceReader">C</a> extends <a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/storage/package-summary.html">org.biojava.nbio.core.sequence.storage</a></dt>
<dd>
<div class="block">Implementation of the 2bit encoding.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#TwoBitSequenceReader(org.biojava.nbio.core.sequence.template.Sequence)">TwoBitSequenceReader(Sequence&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#TwoBitSequenceReader(java.lang.String,%20org.biojava.nbio.core.sequence.template.CompoundSet)">TwoBitSequenceReader(String, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#TwoBitSequenceReader(java.lang.String,%20org.biojava.nbio.core.sequence.template.CompoundSet,%20org.biojava.nbio.core.sequence.AccessionID)">TwoBitSequenceReader(String, CompoundSet&lt;C&gt;, AccessionID)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#TwoBitSequenceReader(org.biojava.nbio.core.sequence.storage.TwoBitSequenceReader.TwoBitArrayWorker)">TwoBitSequenceReader(TwoBitSequenceReader.TwoBitArrayWorker&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#TwoBitSequenceReader(org.biojava.nbio.core.sequence.storage.TwoBitSequenceReader.TwoBitArrayWorker,%20org.biojava.nbio.core.sequence.AccessionID)">TwoBitSequenceReader(TwoBitSequenceReader.TwoBitArrayWorker&lt;C&gt;, AccessionID)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage"><span class="strong">TwoBitSequenceReader.TwoBitArrayWorker</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="type parameter in TwoBitSequenceReader.TwoBitArrayWorker">C</a> extends <a href="./org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/storage/package-summary.html">org.biojava.nbio.core.sequence.storage</a></dt>
<dd>
<div class="block">Extension of the BitArrayWorker which provides the 2bit implementation
 code.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html#TwoBitSequenceReader.TwoBitArrayWorker(org.biojava.nbio.core.sequence.template.CompoundSet,%20int)">TwoBitSequenceReader.TwoBitArrayWorker(CompoundSet&lt;C&gt;, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader.TwoBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html#TwoBitSequenceReader.TwoBitArrayWorker(org.biojava.nbio.core.sequence.template.CompoundSet,%20int[])">TwoBitSequenceReader.TwoBitArrayWorker(CompoundSet&lt;C&gt;, int[])</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader.TwoBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html#TwoBitSequenceReader.TwoBitArrayWorker(org.biojava.nbio.core.sequence.template.Sequence)">TwoBitSequenceReader.TwoBitArrayWorker(Sequence&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader.TwoBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html#TwoBitSequenceReader.TwoBitArrayWorker(java.lang.String,%20org.biojava.nbio.core.sequence.template.CompoundSet)">TwoBitSequenceReader.TwoBitArrayWorker(String, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="./org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader.TwoBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html#TYPE">TYPE</a></span> - Static variable in class org.biojava.nbio.core.sequence.features.<a href="./org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">Sort features by type</div>
</dd>
</dl>
<a name="_U_">
<!--   -->
</a>
<h2 class="title">U</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/UncompressInputStream.html#uncompress(java.lang.String,%20java.io.FileOutputStream)">uncompress(String, FileOutputStream)</a></span> - Static method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/UncompressInputStream.html" title="class in org.biojava.nbio.core.util">UncompressInputStream</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/util/UncompressInputStream.html" title="class in org.biojava.nbio.core.util"><span class="strong">UncompressInputStream</span></a> - Class in <a href="./org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>
<div class="block">This class decompresses an input stream containing data compressed with
 the unix "compress" utility (LZC, a LZW variant).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/UncompressInputStream.html#UncompressInputStream(java.io.InputStream)">UncompressInputStream(InputStream)</a></span> - Constructor for class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/UncompressInputStream.html" title="class in org.biojava.nbio.core.util">UncompressInputStream</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader"><span class="strong">UniprotProxySequenceReader</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="type parameter in UniprotProxySequenceReader">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/loader/package-summary.html">org.biojava.nbio.core.sequence.loader</a></dt>
<dd>
<div class="block">Pass in a Uniprot ID and this ProxySequenceReader when passed to a ProteinSequence will get the sequence data and other data elements
 associated with the ProteinSequence by Uniprot.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#UniprotProxySequenceReader(java.lang.String,%20org.biojava.nbio.core.sequence.template.CompoundSet)">UniprotProxySequenceReader(String, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">The UniProt id is used to retrieve the UniProt XML which is then parsed as a DOM object
 so we know everything about the protein.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#UniprotProxySequenceReader(org.w3c.dom.Document,%20org.biojava.nbio.core.sequence.template.CompoundSet)">UniprotProxySequenceReader(Document, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">The xml is passed in as a DOM object so we know everything about the protein.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html#UNKNOWN_DNA">UNKNOWN_DNA</a></span> - Static variable in interface org.biojava.nbio.core.sequence.io.template.<a href="./org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template">GenbankHeaderFormatInterface</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#UP_AC_PATTERN">UP_AC_PATTERN</a></span> - Static variable in class org.biojava.nbio.core.sequence.loader.<a href="./org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/CRC64Checksum.html#update(int)">update(int)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/CRC64Checksum.html" title="class in org.biojava.nbio.core.util">CRC64Checksum</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/CRC64Checksum.html#update(byte[],%20int,%20int)">update(byte[], int, int)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/CRC64Checksum.html" title="class in org.biojava.nbio.core.util">CRC64Checksum</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/CRC64Checksum.html#update(java.lang.String)">update(String)</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/CRC64Checksum.html" title="class in org.biojava.nbio.core.util">CRC64Checksum</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_V_">
<!--   -->
</a>
<h2 class="title">V</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/DataSource.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/DataSource.html" title="enum in org.biojava.nbio.core.sequence">DataSource</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/DNASequence.DNAType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/DNASequence.DNAType.html" title="enum in org.biojava.nbio.core.sequence">DNASequence.DNAType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/SequenceOptimizationHints.SequenceCollection.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/SequenceOptimizationHints.SequenceCollection.html" title="enum in org.biojava.nbio.core.sequence">SequenceOptimizationHints.SequenceCollection</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/SequenceOptimizationHints.SequenceUsage.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/SequenceOptimizationHints.SequenceUsage.html" title="enum in org.biojava.nbio.core.sequence">SequenceOptimizationHints.SequenceUsage</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/Strand.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/Strand.html" title="enum in org.biojava.nbio.core.sequence">Strand</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.AnnotationType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.AnnotationType.html" title="enum in org.biojava.nbio.core.sequence.template">AbstractSequence.AnnotationType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.StringFormat.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.StringFormat.html" title="enum in org.biojava.nbio.core.sequence.template">LightweightProfile.StringFormat</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Frame.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Frame.html" title="enum in org.biojava.nbio.core.sequence.transcription">Frame</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/DataSource.html#values()">values()</a></span> - Static method in enum org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/DataSource.html" title="enum in org.biojava.nbio.core.sequence">DataSource</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/DNASequence.DNAType.html#values()">values()</a></span> - Static method in enum org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/DNASequence.DNAType.html" title="enum in org.biojava.nbio.core.sequence">DNASequence.DNAType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/SequenceOptimizationHints.SequenceCollection.html#values()">values()</a></span> - Static method in enum org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/SequenceOptimizationHints.SequenceCollection.html" title="enum in org.biojava.nbio.core.sequence">SequenceOptimizationHints.SequenceCollection</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/SequenceOptimizationHints.SequenceUsage.html#values()">values()</a></span> - Static method in enum org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/SequenceOptimizationHints.SequenceUsage.html" title="enum in org.biojava.nbio.core.sequence">SequenceOptimizationHints.SequenceUsage</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/Strand.html#values()">values()</a></span> - Static method in enum org.biojava.nbio.core.sequence.<a href="./org/biojava/nbio/core/sequence/Strand.html" title="enum in org.biojava.nbio.core.sequence">Strand</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.AnnotationType.html#values()">values()</a></span> - Static method in enum org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractSequence.AnnotationType.html" title="enum in org.biojava.nbio.core.sequence.template">AbstractSequence.AnnotationType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.StringFormat.html#values()">values()</a></span> - Static method in enum org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/LightweightProfile.StringFormat.html" title="enum in org.biojava.nbio.core.sequence.template">LightweightProfile.StringFormat</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Frame.html#values()">values()</a></span> - Static method in enum org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Frame.html" title="enum in org.biojava.nbio.core.sequence.transcription">Frame</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#VERSION_TAG">VERSION_TAG</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#vp">vp</a></span> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_W_">
<!--   -->
</a>
<h2 class="title">W</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#waitForStartCodon(boolean)">waitForStartCodon(boolean)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>
<div class="block">If set, then translation will not start until a start codon is
 encountered</div>
</dd>
<dt><a href="./org/biojava/nbio/core/sequence/views/WindowedSequence.html" title="class in org.biojava.nbio.core.sequence.views"><span class="strong">WindowedSequence</span></a>&lt;<a href="./org/biojava/nbio/core/sequence/views/WindowedSequence.html" title="type parameter in WindowedSequence">C</a> extends <a href="./org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava/nbio/core/sequence/views/package-summary.html">org.biojava.nbio.core.sequence.views</a></dt>
<dd>
<div class="block">A sliding window view of a sequence which does not implement any
 interfaces like <a href="./org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> because they do not fit how this works.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/views/WindowedSequence.html#WindowedSequence(org.biojava.nbio.core.sequence.template.Sequence,%20int)">WindowedSequence(Sequence&lt;C&gt;, int)</a></span> - Constructor for class org.biojava.nbio.core.sequence.views.<a href="./org/biojava/nbio/core/sequence/views/WindowedSequence.html" title="class in org.biojava.nbio.core.sequence.views">WindowedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#workingListToSequences(java.util.List)">workingListToSequences(List&lt;List&lt;T&gt;&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/Frame.html#wrap(org.biojava.nbio.core.sequence.template.Sequence)">wrap(Sequence&lt;C&gt;)</a></span> - Method in enum org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/Frame.html" title="enum in org.biojava.nbio.core.sequence.transcription">Frame</a></dt>
<dd>
<div class="block">Optionally wraps a Sequence in a reverse complementing view (if the
 frame is on the reverse strand) and creates a sub sequence view if
 it is required.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#wrapToRna(org.biojava.nbio.core.sequence.template.Sequence)">wrapToRna(Sequence&lt;NucleotideCompound&gt;)</a></span> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="./org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html" title="class in org.biojava.nbio.core.sequence.transcription">DNAToRNATranslator</a></dt>
<dd>
<div class="block">Takes in the given DNA Sequence and returns an instance of RNASequence
 which is using <a href="./org/biojava/nbio/core/sequence/views/RnaSequenceView.html" title="class in org.biojava.nbio.core.sequence.views"><code>RnaSequenceView</code></a> as a
 <a href="./org/biojava/nbio/core/sequence/template/ProxySequenceReader.html" title="interface in org.biojava.nbio.core.sequence.template"><code>ProxySequenceReader</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html#write(java.lang.Appendable,%20org.biojava.nbio.core.sequence.template.Sequence)">write(Appendable, Sequence&lt;C&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.template.<a href="./org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Used as a way of sending a Sequence to a writer without the cost of
 converting to a full length String and then writing the data out</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#writeGeneSequence(java.io.File,%20java.util.Collection,%20boolean)">writeGeneSequence(File, Collection&lt;GeneSequence&gt;, boolean)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriterHelper</a></dt>
<dd>
<div class="block">Write a collection of GeneSequences to a file where if the gene is negative strand it will flip and complement the sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#writeGeneSequence(java.io.OutputStream,%20java.util.Collection,%20boolean)">writeGeneSequence(OutputStream, Collection&lt;GeneSequence&gt;, boolean)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriterHelper</a></dt>
<dd>
<div class="block">Write a collection of GeneSequences to a file where if the gene is negative strand it will flip and complement the sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html#writeIndent()">writeIndent()</a></span> - Method in class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#writeNucleotideSequence(java.io.File,%20java.util.Collection)">writeNucleotideSequence(File, Collection&lt;DNASequence&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriterHelper</a></dt>
<dd>
<div class="block">Write a collection of NucleotideSequences to a file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#writeNucleotideSequence(java.io.OutputStream,%20java.util.Collection)">writeNucleotideSequence(OutputStream, Collection&lt;DNASequence&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriterHelper</a></dt>
<dd>
<div class="block">Write a collection of NucleotideSequences to a file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#writeNucleotideSequence(java.io.File,%20java.util.Collection)">writeNucleotideSequence(File, Collection&lt;DNASequence&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriterHelper</a></dt>
<dd>
<div class="block">Write a collection of NucleotideSequences to a file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#writeNucleotideSequence(java.io.OutputStream,%20java.util.Collection)">writeNucleotideSequence(OutputStream, Collection&lt;DNASequence&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriterHelper</a></dt>
<dd>
<div class="block">Write a collection of NucleotideSequences to a file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#writeNucleotideSequence(java.io.OutputStream,%20java.util.Collection,%20java.lang.String)">writeNucleotideSequence(OutputStream, Collection&lt;DNASequence&gt;, String)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriterHelper</a></dt>
<dd>
<div class="block">Write a collection of NucleotideSequences to a file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#writeProteinSequence(java.io.File,%20java.util.Collection)">writeProteinSequence(File, Collection&lt;ProteinSequence&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriterHelper</a></dt>
<dd>
<div class="block">Write collection of protein sequences to a file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#writeProteinSequence(java.io.OutputStream,%20java.util.Collection)">writeProteinSequence(OutputStream, Collection&lt;ProteinSequence&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriterHelper</a></dt>
<dd>
<div class="block">Write collection of protein sequences to a stream</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#writeProteinSequence(java.io.File,%20java.util.Collection)">writeProteinSequence(File, Collection&lt;ProteinSequence&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriterHelper</a></dt>
<dd>
<div class="block">Write collection of protein sequences to a file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#writeProteinSequence(java.io.OutputStream,%20java.util.Collection)">writeProteinSequence(OutputStream, Collection&lt;ProteinSequence&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriterHelper</a></dt>
<dd>
<div class="block">Write collection of protein sequences to a stream</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#writeSequence(java.io.File,%20org.biojava.nbio.core.sequence.template.Sequence)">writeSequence(File, Sequence&lt;?&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriterHelper</a></dt>
<dd>
<div class="block">Write a sequence to a file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#writeSequence(java.io.OutputStream,%20org.biojava.nbio.core.sequence.template.Sequence)">writeSequence(OutputStream, Sequence&lt;?&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriterHelper</a></dt>
<dd>
<div class="block">Write a sequence to OutputStream</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#writeSequence(java.io.File,%20org.biojava.nbio.core.sequence.template.Sequence)">writeSequence(File, Sequence&lt;?&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriterHelper</a></dt>
<dd>
<div class="block">Write a sequence to a file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#writeSequence(java.io.OutputStream,%20org.biojava.nbio.core.sequence.template.Sequence)">writeSequence(OutputStream, Sequence&lt;?&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriterHelper</a></dt>
<dd>
<div class="block">Write a sequence to OutputStream</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#writeSequences(java.io.OutputStream,%20java.util.Collection)">writeSequences(OutputStream, Collection&lt;Sequence&lt;?&gt;&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/FastaWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriterHelper</a></dt>
<dd>
<div class="block">Method which will write your given Sequences to the specified
 <a href="http://java.sun.com/javase/6/docs/api/java/io/OutputStream.html?is-external=true" title="class or interface in java.io"><code>OutputStream</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#writeSequences(java.io.OutputStream,%20java.util.Collection)">writeSequences(OutputStream, Collection&lt;Sequence&lt;?&gt;&gt;)</a></span> - Static method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriterHelper</a></dt>
<dd>
<div class="block">Method which will write your given Sequences to the specified
 <a href="http://java.sun.com/javase/6/docs/api/java/io/OutputStream.html?is-external=true" title="class or interface in java.io"><code>OutputStream</code></a>.</div>
</dd>
</dl>
<a name="_X_">
<!--   -->
</a>
<h2 class="title">X</h2>
<dl>
<dt><a href="./org/biojava/nbio/core/util/XMLHelper.html" title="class in org.biojava.nbio.core.util"><span class="strong">XMLHelper</span></a> - Class in <a href="./org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/util/XMLHelper.html#XMLHelper()">XMLHelper()</a></span> - Constructor for class org.biojava.nbio.core.util.<a href="./org/biojava/nbio/core/util/XMLHelper.html" title="class in org.biojava.nbio.core.util">XMLHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/core/util/XMLWriter.html" title="interface in org.biojava.nbio.core.util"><span class="strong">XMLWriter</span></a> - Interface in <a href="./org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>
<div class="block">Simple interface for building XML documents.</div>
</dd>
</dl>
<a name="___">
<!--   -->
</a>
<h2 class="title">_</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#_split_multi_line(java.lang.String,%20int)">_split_multi_line(String, int)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io">GenericInsdcHeaderFormat</a></dt>
<dd>
<div class="block">Returns a list of strings.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#_write_feature(org.biojava.nbio.core.sequence.features.FeatureInterface,%20int)">_write_feature(FeatureInterface&lt;AbstractSequence&lt;C&gt;, C&gt;, int)</a></span> - Method in class org.biojava.nbio.core.sequence.io.<a href="./org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io">GenericInsdcHeaderFormat</a></dt>
<dd>
<div class="block">Write a single SeqFeature object to features table.</div>
</dd>
</dl>
<a href="#_A_">A</a>&nbsp;<a href="#_B_">B</a>&nbsp;<a href="#_C_">C</a>&nbsp;<a href="#_D_">D</a>&nbsp;<a href="#_E_">E</a>&nbsp;<a href="#_F_">F</a>&nbsp;<a href="#_G_">G</a>&nbsp;<a href="#_H_">H</a>&nbsp;<a href="#_I_">I</a>&nbsp;<a href="#_J_">J</a>&nbsp;<a href="#_K_">K</a>&nbsp;<a href="#_L_">L</a>&nbsp;<a href="#_M_">M</a>&nbsp;<a href="#_N_">N</a>&nbsp;<a href="#_O_">O</a>&nbsp;<a href="#_P_">P</a>&nbsp;<a href="#_Q_">Q</a>&nbsp;<a href="#_R_">R</a>&nbsp;<a href="#_S_">S</a>&nbsp;<a href="#_T_">T</a>&nbsp;<a href="#_U_">U</a>&nbsp;<a href="#_V_">V</a>&nbsp;<a href="#_W_">W</a>&nbsp;<a href="#_X_">X</a>&nbsp;<a href="#___">_</a>&nbsp;</div>
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar_bottom">
<!--   -->
</a><a href="#skip-navbar_bottom" title="Skip navigation links"></a><a name="navbar_bottom_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="./overview-summary.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li>Use</li>
<li><a href="./overview-tree.html">Tree</a></li>
<li><a href="./deprecated-list.html">Deprecated</a></li>
<li class="navBarCell1Rev">Index</li>
<li><a href="./help-doc.html">Help</a></li>
</ul>
<div class="aboutLanguage"><em><script src="http://www.google-analytics.com/urchin.js" 							type="text/javascript"></script><script 							type="text/javascript">_uacct = 							"UA-1326640-1";urchinTracker();</script></em></div>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="./index.html?index-all.html" target="_top">Frames</a></li>
<li><a href="index-all.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="./allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
<p class="legalCopy"><small>Copyright &#169; 2000-2015 <a href="http://www.biojava.org">BioJava</a>. All Rights Reserved.</small></p>
</body>
</html>
