<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_71) on Fri Jan 30 08:14:21 PST 2015 -->
<meta http-equiv="Content-Type" content="text/html" charset="UTF-8">
<title>Index (biojava-alignment 4.0.0 API)</title>
<meta name="date" content="2015-01-30">
<link rel="stylesheet" type="text/css" href="./stylesheet.css" title="Style">
</head>
<body>
<script type="text/javascript"><!--
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="Index (biojava-alignment 4.0.0 API)";
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar_top">
<!--   -->
</a><a href="#skip-navbar_top" title="Skip navigation links"></a><a name="navbar_top_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="./overview-summary.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li>Use</li>
<li><a href="./overview-tree.html">Tree</a></li>
<li><a href="./deprecated-list.html">Deprecated</a></li>
<li class="navBarCell1Rev">Index</li>
<li><a href="./help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="./index.html?index-all.html" target="_top">Frames</a></li>
<li><a href="index-all.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="./allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<div class="contentContainer"><a href="#_A_">A</a>&nbsp;<a href="#_C_">C</a>&nbsp;<a href="#_D_">D</a>&nbsp;<a href="#_E_">E</a>&nbsp;<a href="#_F_">F</a>&nbsp;<a href="#_G_">G</a>&nbsp;<a href="#_H_">H</a>&nbsp;<a href="#_I_">I</a>&nbsp;<a href="#_L_">L</a>&nbsp;<a href="#_M_">M</a>&nbsp;<a href="#_N_">N</a>&nbsp;<a href="#_O_">O</a>&nbsp;<a href="#_P_">P</a>&nbsp;<a href="#_R_">R</a>&nbsp;<a href="#_S_">S</a>&nbsp;<a href="#_T_">T</a>&nbsp;<a href="#_U_">U</a>&nbsp;<a href="#_V_">V</a>&nbsp;<a href="#_X_">X</a>&nbsp;<a name="_A_">
<!--   -->
</a>
<h2 class="title">A</h2>
<dl>
<dt><a href="./org/biojava/nbio/alignment/aaindex/AAindexFactory.html" title="class in org.biojava.nbio.alignment.aaindex"><span class="strong">AAindexFactory</span></a> - Class in <a href="./org/biojava/nbio/alignment/aaindex/package-summary.html">org.biojava.nbio.alignment.aaindex</a></dt>
<dd>
<div class="block">Factory class to get Providers for substitution matrices the are provided by the AAINDEX database.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/AAindexFactory.html#AAindexFactory()">AAindexFactory()</a></span> - Constructor for class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/AAindexFactory.html" title="class in org.biojava.nbio.alignment.aaindex">AAindexFactory</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/aaindex/AAIndexFileParser.html" title="class in org.biojava.nbio.alignment.aaindex"><span class="strong">AAIndexFileParser</span></a> - Class in <a href="./org/biojava/nbio/alignment/aaindex/package-summary.html">org.biojava.nbio.alignment.aaindex</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/AAIndexFileParser.html#AAIndexFileParser()">AAIndexFileParser()</a></span> - Constructor for class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/AAIndexFileParser.html" title="class in org.biojava.nbio.alignment.aaindex">AAIndexFileParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/aaindex/AAIndexProvider.html" title="interface in org.biojava.nbio.alignment.aaindex"><span class="strong">AAIndexProvider</span></a> - Interface in <a href="./org/biojava/nbio/alignment/aaindex/package-summary.html">org.biojava.nbio.alignment.aaindex</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template"><span class="strong">AbstractMatrixAligner</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="type parameter in AbstractMatrixAligner">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="type parameter in AbstractMatrixAligner">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="type parameter in AbstractMatrixAligner">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Class in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Implements common code for an <a href="./org/biojava/nbio/alignment/template/Aligner.html" title="interface in org.biojava.nbio.alignment.template"><code>Aligner</code></a> which builds a score matrix during computation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#AbstractMatrixAligner()">AbstractMatrixAligner()</a></span> - Constructor for class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Before running an alignment, data must be sent in via calls to <a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#setGapPenalty(org.biojava.nbio.alignment.template.GapPenalty)"><code>setGapPenalty(GapPenalty)</code></a> and
 <a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#setSubstitutionMatrix(org.biojava.nbio.alignment.template.SubstitutionMatrix)"><code>setSubstitutionMatrix(SubstitutionMatrix)</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#AbstractMatrixAligner(org.biojava.nbio.alignment.template.GapPenalty,%20org.biojava.nbio.alignment.template.SubstitutionMatrix)">AbstractMatrixAligner(GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Prepares for an alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#AbstractMatrixAligner(org.biojava.nbio.alignment.template.GapPenalty,%20org.biojava.nbio.alignment.template.SubstitutionMatrix,%20boolean)">AbstractMatrixAligner(GapPenalty, SubstitutionMatrix&lt;C&gt;, boolean)</a></span> - Constructor for class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Prepares for an alignment.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.template"><span class="strong">AbstractPairwiseSequenceAligner</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html" title="type parameter in AbstractPairwiseSequenceAligner">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html" title="type parameter in AbstractPairwiseSequenceAligner">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html" title="type parameter in AbstractPairwiseSequenceAligner">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Class in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Implements common code for an <a href="./org/biojava/nbio/alignment/template/Aligner.html" title="interface in org.biojava.nbio.alignment.template"><code>Aligner</code></a> for a pair of <code>Sequence</code>s.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#AbstractPairwiseSequenceAligner()">AbstractPairwiseSequenceAligner()</a></span> - Constructor for class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Before running a pairwise global sequence alignment, data must be sent in via calls to
 <a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#setQuery(S)"><code>AbstractPairwiseSequenceAligner.setQuery(Sequence)</code></a>, <a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#setTarget(S)"><code>AbstractPairwiseSequenceAligner.setTarget(Sequence)</code></a>, <a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#setGapPenalty(org.biojava.nbio.alignment.template.GapPenalty)"><code>setGapPenalty(GapPenalty)</code></a>, and
 <a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#setSubstitutionMatrix(org.biojava.nbio.alignment.template.SubstitutionMatrix)"><code>setSubstitutionMatrix(SubstitutionMatrix)</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#AbstractPairwiseSequenceAligner(S,%20S,%20org.biojava.nbio.alignment.template.GapPenalty,%20org.biojava.nbio.alignment.template.SubstitutionMatrix)">AbstractPairwiseSequenceAligner(S, S, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Prepares for a pairwise global sequence alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#AbstractPairwiseSequenceAligner(S,%20S,%20org.biojava.nbio.alignment.template.GapPenalty,%20org.biojava.nbio.alignment.template.SubstitutionMatrix,%20boolean)">AbstractPairwiseSequenceAligner(S, S, GapPenalty, SubstitutionMatrix&lt;C&gt;, boolean)</a></span> - Constructor for class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Prepares for a pairwise sequence alignment.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava.nbio.alignment.template"><span class="strong">AbstractProfileProfileAligner</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html" title="type parameter in AbstractProfileProfileAligner">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html" title="type parameter in AbstractProfileProfileAligner">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html" title="type parameter in AbstractProfileProfileAligner">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Class in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Implements common code for an <a href="./org/biojava/nbio/alignment/template/Aligner.html" title="interface in org.biojava.nbio.alignment.template"><code>Aligner</code></a> for a pair of <a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template"><code>Profile</code></a>s.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html#AbstractProfileProfileAligner()">AbstractProfileProfileAligner()</a></span> - Constructor for class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>
<div class="block">Before running a profile-profile alignment, data must be sent in via calls to
 <a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html#setQuery(org.biojava.nbio.alignment.template.Profile)"><code>AbstractProfileProfileAligner.setQuery(Profile)</code></a>, <a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html#setTarget(org.biojava.nbio.alignment.template.Profile)"><code>AbstractProfileProfileAligner.setTarget(Profile)</code></a>, <a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#setGapPenalty(org.biojava.nbio.alignment.template.GapPenalty)"><code>setGapPenalty(GapPenalty)</code></a>, and
 <a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#setSubstitutionMatrix(org.biojava.nbio.alignment.template.SubstitutionMatrix)"><code>setSubstitutionMatrix(SubstitutionMatrix)</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html#AbstractProfileProfileAligner(org.biojava.nbio.alignment.template.Profile,%20org.biojava.nbio.alignment.template.Profile,%20org.biojava.nbio.alignment.template.GapPenalty,%20org.biojava.nbio.alignment.template.SubstitutionMatrix)">AbstractProfileProfileAligner(Profile&lt;S, C&gt;, Profile&lt;S, C&gt;, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>
<div class="block">Prepares for a profile-profile alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html#AbstractProfileProfileAligner(java.util.concurrent.Future,%20java.util.concurrent.Future,%20org.biojava.nbio.alignment.template.GapPenalty,%20org.biojava.nbio.alignment.template.SubstitutionMatrix)">AbstractProfileProfileAligner(Future&lt;ProfilePair&lt;S, C&gt;&gt;, Future&lt;ProfilePair&lt;S, C&gt;&gt;, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>
<div class="block">Prepares for a profile-profile alignment run concurrently.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html#AbstractProfileProfileAligner(org.biojava.nbio.alignment.template.Profile,%20java.util.concurrent.Future,%20org.biojava.nbio.alignment.template.GapPenalty,%20org.biojava.nbio.alignment.template.SubstitutionMatrix)">AbstractProfileProfileAligner(Profile&lt;S, C&gt;, Future&lt;ProfilePair&lt;S, C&gt;&gt;, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>
<div class="block">Prepares for a profile-profile alignment run concurrently.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html#AbstractProfileProfileAligner(java.util.concurrent.Future,%20org.biojava.nbio.alignment.template.Profile,%20org.biojava.nbio.alignment.template.GapPenalty,%20org.biojava.nbio.alignment.template.SubstitutionMatrix)">AbstractProfileProfileAligner(Future&lt;ProfilePair&lt;S, C&gt;&gt;, Profile&lt;S, C&gt;, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>
<div class="block">Prepares for a profile-profile alignment run concurrently.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/template/AbstractScorer.html" title="class in org.biojava.nbio.alignment.template"><span class="strong">AbstractScorer</span></a> - Class in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Implements common code for algorithms which compute a score.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractScorer.html#AbstractScorer()">AbstractScorer()</a></span> - Constructor for class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractScorer.html" title="class in org.biojava.nbio.alignment.template">AbstractScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#addActiveSite(java.lang.String,%20java.lang.String)">addActiveSite(String, String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AnchoredPairwiseSequenceAligner.html#addAnchor(int,%20int)">addAnchor(int, int)</a></span> - Method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.routines">AnchoredPairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Adds an additional anchor to the set of anchored compounds</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html#addAnchors(org.biojava.nbio.alignment.routines.AlignerHelper.Cut[],%20int[],%20boolean,%20int[])">addAnchors(AlignerHelper.Cut[], int[], boolean, int[])</a></span> - Static method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#addASPFamPredicted(java.lang.String,%20java.lang.String)">addASPFamPredicted(String, String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#addASSwissProt(java.lang.String,%20java.lang.String)">addASSwissProt(String, String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#addDBReference(java.lang.String)">addDBReference(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#addGFBuildMethod(java.lang.String)">addGFBuildMethod(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#addGFFalseDiscoveryRate(java.lang.String)">addGFFalseDiscoveryRate(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#addGFNewHampshire(java.lang.String)">addGFNewHampshire(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#addGFTreeID(java.lang.String)">addGFTreeID(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#addGSAccessionNumber(java.lang.String,%20java.lang.String)">addGSAccessionNumber(String, String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#addGSdbReference(java.lang.String,%20java.lang.String)">addGSdbReference(String, String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#addGSDescription(java.lang.String,%20java.lang.String)">addGSDescription(String, String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#addGSLook(java.lang.String,%20java.lang.String)">addGSLook(String, String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#addGSOrganismClassification(java.lang.String,%20java.lang.String)">addGSOrganismClassification(String, String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#addGSOrganismSpecies(java.lang.String,%20java.lang.String)">addGSOrganismSpecies(String, String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#addIntron(java.lang.String,%20java.lang.String)">addIntron(String, String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#addLigandBinding(java.lang.String,%20java.lang.String)">addLigandBinding(String, String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#addPosteriorProbability(java.lang.String,%20java.lang.String)">addPosteriorProbability(String, String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#addSecondaryStructure(java.lang.String,%20java.lang.String)">addSecondaryStructure(String, String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#addSequence(java.lang.String,%20java.lang.String)">addSequence(String, String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>Use <a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#appendToSequence(java.lang.String,%20java.lang.String)"><code>StockholmStructure.appendToSequence(String,String)</code></a> instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#addSurfaceAccessibility(java.lang.String,%20java.lang.String)">addSurfaceAccessibility(String, String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#addToGFComment(java.lang.String)">addToGFComment(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.StockholmFileAnnotationReference.html#addToRefAuthor(java.lang.String)">addToRefAuthor(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.StockholmFileAnnotationReference.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation.StockholmFileAnnotationReference</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.StockholmFileAnnotationReference.html#addToRefTitle(java.lang.String)">addToRefTitle(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.StockholmFileAnnotationReference.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation.StockholmFileAnnotationReference</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#addTransMembrane(java.lang.String,%20java.lang.String)">addTransMembrane(String, String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#align()">align()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Performs alignment</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template"><span class="strong">AlignedSequence</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="type parameter in AlignedSequence">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="type parameter in AlignedSequence">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="type parameter in AlignedSequence">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Interface in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Defines a data structure for a <code>Sequence</code> within an alignment.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/template/AlignedSequence.Step.html" title="enum in org.biojava.nbio.alignment.template"><span class="strong">AlignedSequence.Step</span></a> - Enum in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Defines an alignment step in order to pass alignment information from an <a href="./org/biojava/nbio/alignment/template/Aligner.html" title="interface in org.biojava.nbio.alignment.template"><code>Aligner</code></a> to a constructor.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/template/Aligner.html" title="interface in org.biojava.nbio.alignment.template"><span class="strong">Aligner</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/Aligner.html" title="type parameter in Aligner">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/Aligner.html" title="type parameter in Aligner">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/Aligner.html" title="type parameter in Aligner">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Interface in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Defines an algorithm which computes an alignment <a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template"><code>Profile</code></a> from a list of <code>Sequence</code>s.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html" title="class in org.biojava.nbio.alignment.routines"><span class="strong">AlignerHelper</span></a> - Class in <a href="./org/biojava/nbio/alignment/routines/package-summary.html">org.biojava.nbio.alignment.routines</a></dt>
<dd>
<div class="block">Static utility to construct alignment routines from a common library of methods.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html#AlignerHelper()">AlignerHelper()</a></span> - Constructor for class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Anchor.html" title="class in org.biojava.nbio.alignment.routines"><span class="strong">AlignerHelper.Anchor</span></a> - Class in <a href="./org/biojava/nbio/alignment/routines/package-summary.html">org.biojava.nbio.alignment.routines</a></dt>
<dd>
<div class="block">Compounds in query and target sequences that must align</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Anchor.html#AlignerHelper.Anchor(int,%20int)">AlignerHelper.Anchor(int, int)</a></span> - Constructor for class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Anchor.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper.Anchor</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Anchor.QueryIndexComparator.html" title="class in org.biojava.nbio.alignment.routines"><span class="strong">AlignerHelper.Anchor.QueryIndexComparator</span></a> - Class in <a href="./org/biojava/nbio/alignment/routines/package-summary.html">org.biojava.nbio.alignment.routines</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Anchor.QueryIndexComparator.html#AlignerHelper.Anchor.QueryIndexComparator()">AlignerHelper.Anchor.QueryIndexComparator()</a></span> - Constructor for class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Anchor.QueryIndexComparator.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper.Anchor.QueryIndexComparator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Cut.html" title="class in org.biojava.nbio.alignment.routines"><span class="strong">AlignerHelper.Cut</span></a> - Class in <a href="./org/biojava/nbio/alignment/routines/package-summary.html">org.biojava.nbio.alignment.routines</a></dt>
<dd>
<div class="block">Defines a 'cut' row for divide-and-conquer alignment in which a new anchor is found.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Cut.html#AlignerHelper.Cut(int,%20int[])">AlignerHelper.Cut(int, int[])</a></span> - Constructor for class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Cut.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper.Cut</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Last.html" title="enum in org.biojava.nbio.alignment.routines"><span class="strong">AlignerHelper.Last</span></a> - Enum in <a href="./org/biojava/nbio/alignment/routines/package-summary.html">org.biojava.nbio.alignment.routines</a></dt>
<dd>
<div class="block">Define a traceback pointer for the three edit operations: substitution (match/replacement of a query compound
 with a target compound), deletion (removal of a query compound leaving a gap in the target sequence), and
 insertion (addition of a target compound opening a gap in the query sequence).</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Subproblem.html" title="class in org.biojava.nbio.alignment.routines"><span class="strong">AlignerHelper.Subproblem</span></a> - Class in <a href="./org/biojava/nbio/alignment/routines/package-summary.html">org.biojava.nbio.alignment.routines</a></dt>
<dd>
<div class="block">Alignment subproblem.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Subproblem.html#AlignerHelper.Subproblem(int,%20int,%20int,%20int)">AlignerHelper.Subproblem(int, int, int, int)</a></span> - Constructor for class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Subproblem.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper.Subproblem</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Subproblem.html#AlignerHelper.Subproblem(int,%20int,%20int,%20int,%20boolean)">AlignerHelper.Subproblem(int, int, int, int, boolean)</a></span> - Constructor for class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Subproblem.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper.Subproblem</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/Alignments.html" title="class in org.biojava.nbio.alignment"><span class="strong">Alignments</span></a> - Class in <a href="./org/biojava/nbio/alignment/package-summary.html">org.biojava.nbio.alignment</a></dt>
<dd>
<div class="block">Static utility to easily run alignment routines.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/Alignments.PairInProfileScorerType.html" title="enum in org.biojava.nbio.alignment"><span class="strong">Alignments.PairInProfileScorerType</span></a> - Enum in <a href="./org/biojava/nbio/alignment/package-summary.html">org.biojava.nbio.alignment</a></dt>
<dd>
<div class="block">List of implemented sequence pair in a profile scoring routines.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/Alignments.PairwiseSequenceAlignerType.html" title="enum in org.biojava.nbio.alignment"><span class="strong">Alignments.PairwiseSequenceAlignerType</span></a> - Enum in <a href="./org/biojava/nbio/alignment/package-summary.html">org.biojava.nbio.alignment</a></dt>
<dd>
<div class="block">List of implemented pairwise sequence alignment routines.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/Alignments.PairwiseSequenceScorerType.html" title="enum in org.biojava.nbio.alignment"><span class="strong">Alignments.PairwiseSequenceScorerType</span></a> - Enum in <a href="./org/biojava/nbio/alignment/package-summary.html">org.biojava.nbio.alignment</a></dt>
<dd>
<div class="block">List of implemented pairwise sequence scoring routines.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/Alignments.ProfileProfileAlignerType.html" title="enum in org.biojava.nbio.alignment"><span class="strong">Alignments.ProfileProfileAlignerType</span></a> - Enum in <a href="./org/biojava/nbio/alignment/package-summary.html">org.biojava.nbio.alignment</a></dt>
<dd>
<div class="block">List of implemented profile-profile alignment routines.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/Alignments.RefinerType.html" title="enum in org.biojava.nbio.alignment"><span class="strong">Alignments.RefinerType</span></a> - Enum in <a href="./org/biojava/nbio/alignment/package-summary.html">org.biojava.nbio.alignment</a></dt>
<dd>
<div class="block">List of implemented profile refinement routines.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.routines"><span class="strong">AnchoredPairwiseSequenceAligner</span></a>&lt;<a href="./org/biojava/nbio/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="type parameter in AnchoredPairwiseSequenceAligner">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="type parameter in AnchoredPairwiseSequenceAligner">C</a>&gt;,<a href="./org/biojava/nbio/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="type parameter in AnchoredPairwiseSequenceAligner">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Class in <a href="./org/biojava/nbio/alignment/routines/package-summary.html">org.biojava.nbio.alignment.routines</a></dt>
<dd>
<div class="block">This algorithm uses a divide-and-conquer approach to find optimal pairwise global sequence alignments (from the
 first until the last <code>Compound</code> of each <code>Sequence</code>) with the restriction that any alignment produced
 will connect the query sequence to the target sequence at the <em>anchors</em>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AnchoredPairwiseSequenceAligner.html#AnchoredPairwiseSequenceAligner()">AnchoredPairwiseSequenceAligner()</a></span> - Constructor for class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.routines">AnchoredPairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Before running a pairwise global sequence alignment, data must be sent in via calls to
 <a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#setQuery(S)"><code>AbstractPairwiseSequenceAligner.setQuery(Sequence)</code></a>, <a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#setTarget(S)"><code>AbstractPairwiseSequenceAligner.setTarget(Sequence)</code></a>, <a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#setGapPenalty(org.biojava.nbio.alignment.template.GapPenalty)"><code>setGapPenalty(GapPenalty)</code></a>, and
 <a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#setSubstitutionMatrix(org.biojava.nbio.alignment.template.SubstitutionMatrix)"><code>setSubstitutionMatrix(SubstitutionMatrix)</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AnchoredPairwiseSequenceAligner.html#AnchoredPairwiseSequenceAligner(S,%20S,%20org.biojava.nbio.alignment.template.GapPenalty,%20org.biojava.nbio.alignment.template.SubstitutionMatrix)">AnchoredPairwiseSequenceAligner(S, S, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.routines">AnchoredPairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Prepares for a pairwise global sequence alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AnchoredPairwiseSequenceAligner.html#AnchoredPairwiseSequenceAligner(S,%20S,%20org.biojava.nbio.alignment.template.GapPenalty,%20org.biojava.nbio.alignment.template.SubstitutionMatrix,%20int[])">AnchoredPairwiseSequenceAligner(S, S, GapPenalty, SubstitutionMatrix&lt;C&gt;, int[])</a></span> - Constructor for class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.routines">AnchoredPairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Prepares for a pairwise global sequence alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#anchors">anchors</a></span> - Variable in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#appendToSequence(java.lang.String,%20java.lang.String)">appendToSequence(String, String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_C_">
<!--   -->
</a>
<h2 class="title">C</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/CallablePairwiseSequenceAligner.html#call()">call()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/CallablePairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.template">CallablePairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/CallablePairwiseSequenceScorer.html#call()">call()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/CallablePairwiseSequenceScorer.html" title="class in org.biojava.nbio.alignment.template">CallablePairwiseSequenceScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/CallableProfileProfileAligner.html#call()">call()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/CallableProfileProfileAligner.html" title="class in org.biojava.nbio.alignment.template">CallableProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/template/CallablePairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.template"><span class="strong">CallablePairwiseSequenceAligner</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/CallablePairwiseSequenceAligner.html" title="type parameter in CallablePairwiseSequenceAligner">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/CallablePairwiseSequenceAligner.html" title="type parameter in CallablePairwiseSequenceAligner">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/CallablePairwiseSequenceAligner.html" title="type parameter in CallablePairwiseSequenceAligner">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Class in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Implements a concurrency wrapper for a <a href="./org/biojava/nbio/alignment/template/PairwiseSequenceAligner.html" title="interface in org.biojava.nbio.alignment.template"><code>PairwiseSequenceAligner</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/CallablePairwiseSequenceAligner.html#CallablePairwiseSequenceAligner(org.biojava.nbio.alignment.template.PairwiseSequenceAligner)">CallablePairwiseSequenceAligner(PairwiseSequenceAligner&lt;S, C&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/CallablePairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.template">CallablePairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Creates a pairwise sequence alignment task for simplified parallel execution.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/template/CallablePairwiseSequenceScorer.html" title="class in org.biojava.nbio.alignment.template"><span class="strong">CallablePairwiseSequenceScorer</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/CallablePairwiseSequenceScorer.html" title="type parameter in CallablePairwiseSequenceScorer">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/CallablePairwiseSequenceScorer.html" title="type parameter in CallablePairwiseSequenceScorer">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/CallablePairwiseSequenceScorer.html" title="type parameter in CallablePairwiseSequenceScorer">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Class in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Implements a concurrency wrapper for a <a href="./org/biojava/nbio/alignment/template/PairwiseSequenceScorer.html" title="interface in org.biojava.nbio.alignment.template"><code>PairwiseSequenceScorer</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/CallablePairwiseSequenceScorer.html#CallablePairwiseSequenceScorer(org.biojava.nbio.alignment.template.PairwiseSequenceScorer)">CallablePairwiseSequenceScorer(PairwiseSequenceScorer&lt;S, C&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/CallablePairwiseSequenceScorer.html" title="class in org.biojava.nbio.alignment.template">CallablePairwiseSequenceScorer</a></dt>
<dd>
<div class="block">Creates a pairwise sequence scoring task for simplified parallel execution.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/template/CallableProfileProfileAligner.html" title="class in org.biojava.nbio.alignment.template"><span class="strong">CallableProfileProfileAligner</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/CallableProfileProfileAligner.html" title="type parameter in CallableProfileProfileAligner">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/CallableProfileProfileAligner.html" title="type parameter in CallableProfileProfileAligner">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/CallableProfileProfileAligner.html" title="type parameter in CallableProfileProfileAligner">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Class in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Implements a concurrency wrapper for a <a href="./org/biojava/nbio/alignment/template/ProfileProfileAligner.html" title="interface in org.biojava.nbio.alignment.template"><code>ProfileProfileAligner</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/CallableProfileProfileAligner.html#CallableProfileProfileAligner(org.biojava.nbio.alignment.template.ProfileProfileAligner)">CallableProfileProfileAligner(ProfileProfileAligner&lt;S, C&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/CallableProfileProfileAligner.html" title="class in org.biojava.nbio.alignment.template">CallableProfileProfileAligner</a></dt>
<dd>
<div class="block">Creates a profile-profile alignment task for simplified parallel execution.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/GuideTree.Node.html#children()">children()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/GuideTree.Node.html" title="class in org.biojava.nbio.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#clearCache()">clearCache()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AlignedSequence.html#clearCache()">clearCache()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Nullifies cached arrays/objects.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Anchor.QueryIndexComparator.html#compare(org.biojava.nbio.alignment.routines.AlignerHelper.Anchor,%20org.biojava.nbio.alignment.routines.AlignerHelper.Anchor)">compare(AlignerHelper.Anchor, AlignerHelper.Anchor)</a></span> - Method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Anchor.QueryIndexComparator.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper.Anchor.QueryIndexComparator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/CookbookMSA.html" title="class in demo"><span class="strong">CookbookMSA</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/CookbookMSA.html#CookbookMSA()">CookbookMSA()</a></span> - Constructor for class demo.<a href="./demo/CookbookMSA.html" title="class in demo">CookbookMSA</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#countCompounds(C...)">countCompounds(C...)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#cutsPerSection">cutsPerSection</a></span> - Variable in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_D_">
<!--   -->
</a>
<h2 class="title">D</h2>
<dl>
<dt><a href="./org/biojava/nbio/alignment/aaindex/DefaultAAIndexProvider.html" title="class in org.biojava.nbio.alignment.aaindex"><span class="strong">DefaultAAIndexProvider</span></a> - Class in <a href="./org/biojava/nbio/alignment/aaindex/package-summary.html">org.biojava.nbio.alignment.aaindex</a></dt>
<dd>
<div class="block">The default provider for AAINDEX loads substitution matrices from the AAINDEX file in the resources directory</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/DefaultAAIndexProvider.html#DefaultAAIndexProvider()">DefaultAAIndexProvider()</a></span> - Constructor for class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/DefaultAAIndexProvider.html" title="class in org.biojava.nbio.alignment.aaindex">DefaultAAIndexProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/package-summary.html">demo</a> - package demo</dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/DemoAlignProteins.html" title="class in demo"><span class="strong">DemoAlignProteins</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoAlignProteins.html#DemoAlignProteins()">DemoAlignProteins()</a></span> - Constructor for class demo.<a href="./demo/DemoAlignProteins.html" title="class in demo">DemoAlignProteins</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/DemoLoadSubstMax.html" title="class in demo"><span class="strong">DemoLoadSubstMax</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoLoadSubstMax.html#DemoLoadSubstMax()">DemoLoadSubstMax()</a></span> - Constructor for class demo.<a href="./demo/DemoLoadSubstMax.html" title="class in demo">DemoLoadSubstMax</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_E_">
<!--   -->
</a>
<h2 class="title">E</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html#EXPERT">EXPERT</a></span> - Static variable in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure.DatabaseReference</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_F_">
<!--   -->
</a>
<h2 class="title">F</h2>
<dl>
<dt><a href="./org/biojava/nbio/alignment/FractionalIdentityInProfileScorer.html" title="class in org.biojava.nbio.alignment"><span class="strong">FractionalIdentityInProfileScorer</span></a>&lt;<a href="./org/biojava/nbio/alignment/FractionalIdentityInProfileScorer.html" title="type parameter in FractionalIdentityInProfileScorer">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/FractionalIdentityInProfileScorer.html" title="type parameter in FractionalIdentityInProfileScorer">C</a>&gt;,<a href="./org/biojava/nbio/alignment/FractionalIdentityInProfileScorer.html" title="type parameter in FractionalIdentityInProfileScorer">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Class in <a href="./org/biojava/nbio/alignment/package-summary.html">org.biojava.nbio.alignment</a></dt>
<dd>
<div class="block">Implements an algorithm which computes a score for a sequence alignment pair picked from an alignment
 <a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template"><code>Profile</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/FractionalIdentityInProfileScorer.html#FractionalIdentityInProfileScorer(org.biojava.nbio.alignment.template.Profile,%20int,%20int)">FractionalIdentityInProfileScorer(Profile&lt;S, C&gt;, int, int)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/FractionalIdentityInProfileScorer.html" title="class in org.biojava.nbio.alignment">FractionalIdentityInProfileScorer</a></dt>
<dd>
<div class="block">Creates a fractional identity scorer for an aligned pair of sequences in the given alignment profile.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/FractionalIdentityScorer.html" title="class in org.biojava.nbio.alignment"><span class="strong">FractionalIdentityScorer</span></a>&lt;<a href="./org/biojava/nbio/alignment/FractionalIdentityScorer.html" title="type parameter in FractionalIdentityScorer">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/FractionalIdentityScorer.html" title="type parameter in FractionalIdentityScorer">C</a>&gt;,<a href="./org/biojava/nbio/alignment/FractionalIdentityScorer.html" title="type parameter in FractionalIdentityScorer">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Class in <a href="./org/biojava/nbio/alignment/package-summary.html">org.biojava.nbio.alignment</a></dt>
<dd>
<div class="block">Implements an algorithm which computes a score for a sequence alignment pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/FractionalIdentityScorer.html#FractionalIdentityScorer(org.biojava.nbio.alignment.template.PairwiseSequenceAligner)">FractionalIdentityScorer(PairwiseSequenceAligner&lt;S, C&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/FractionalIdentityScorer.html" title="class in org.biojava.nbio.alignment">FractionalIdentityScorer</a></dt>
<dd>
<div class="block">Creates a fractional identity scorer for a pair of sequences aligned by the given pairwise sequence aligner.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/FractionalIdentityScorer.html#FractionalIdentityScorer(org.biojava.nbio.alignment.template.SequencePair)">FractionalIdentityScorer(SequencePair&lt;S, C&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/FractionalIdentityScorer.html" title="class in org.biojava.nbio.alignment">FractionalIdentityScorer</a></dt>
<dd>
<div class="block">Creates a fractional identity scorer for an aligned pair of sequences.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/FractionalSimilarityInProfileScorer.html" title="class in org.biojava.nbio.alignment"><span class="strong">FractionalSimilarityInProfileScorer</span></a>&lt;<a href="./org/biojava/nbio/alignment/FractionalSimilarityInProfileScorer.html" title="type parameter in FractionalSimilarityInProfileScorer">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/FractionalSimilarityInProfileScorer.html" title="type parameter in FractionalSimilarityInProfileScorer">C</a>&gt;,<a href="./org/biojava/nbio/alignment/FractionalSimilarityInProfileScorer.html" title="type parameter in FractionalSimilarityInProfileScorer">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Class in <a href="./org/biojava/nbio/alignment/package-summary.html">org.biojava.nbio.alignment</a></dt>
<dd>
<div class="block">Implements an algorithm which computes a score for a sequence alignment pair picked from an alignment
 <a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template"><code>Profile</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/FractionalSimilarityInProfileScorer.html#FractionalSimilarityInProfileScorer(org.biojava.nbio.alignment.template.Profile,%20int,%20int)">FractionalSimilarityInProfileScorer(Profile&lt;S, C&gt;, int, int)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/FractionalSimilarityInProfileScorer.html" title="class in org.biojava.nbio.alignment">FractionalSimilarityInProfileScorer</a></dt>
<dd>
<div class="block">Creates a fractional similarity scorer for an aligned pair of sequences in the given alignment profile.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/FractionalSimilarityScorer.html" title="class in org.biojava.nbio.alignment"><span class="strong">FractionalSimilarityScorer</span></a>&lt;<a href="./org/biojava/nbio/alignment/FractionalSimilarityScorer.html" title="type parameter in FractionalSimilarityScorer">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/FractionalSimilarityScorer.html" title="type parameter in FractionalSimilarityScorer">C</a>&gt;,<a href="./org/biojava/nbio/alignment/FractionalSimilarityScorer.html" title="type parameter in FractionalSimilarityScorer">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Class in <a href="./org/biojava/nbio/alignment/package-summary.html">org.biojava.nbio.alignment</a></dt>
<dd>
<div class="block">Implements an algorithm which computes a score for a sequence alignment pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/FractionalSimilarityScorer.html#FractionalSimilarityScorer(org.biojava.nbio.alignment.template.PairwiseSequenceAligner)">FractionalSimilarityScorer(PairwiseSequenceAligner&lt;S, C&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/FractionalSimilarityScorer.html" title="class in org.biojava.nbio.alignment">FractionalSimilarityScorer</a></dt>
<dd>
<div class="block">Creates a fractional similarity scorer for a pair of sequences aligned by the given pairwise sequence aligner.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/FractionalSimilarityScorer.html#FractionalSimilarityScorer(org.biojava.nbio.alignment.template.SequencePair)">FractionalSimilarityScorer(SequencePair&lt;S, C&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/FractionalSimilarityScorer.html" title="class in org.biojava.nbio.alignment">FractionalSimilarityScorer</a></dt>
<dd>
<div class="block">Creates a fractional similarity scorer for an aligned pair of sequences.</div>
</dd>
</dl>
<a name="_G_">
<!--   -->
</a>
<h2 class="title">G</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#gapPenalty">gapPenalty</a></span> - Variable in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/template/GapPenalty.html" title="interface in org.biojava.nbio.alignment.template"><span class="strong">GapPenalty</span></a> - Interface in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Defines a data structure for the gap penalties used during a sequence alignment routine.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/template/GapPenalty.Type.html" title="enum in org.biojava.nbio.alignment.template"><span class="strong">GapPenalty.Type</span></a> - Enum in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Defines the possible types of gap penalties.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/AAindexFactory.html#getAAIndexProvider()">getAAIndexProvider()</a></span> - Static method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/AAindexFactory.html" title="class in org.biojava.nbio.alignment.aaindex">AAindexFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#getAccession()">getAccession()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getAccessionNumber()">getAccessionNumber()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#getAlignedSequence(int)">getAlignedSequence(int)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#getAlignedSequence(S)">getAlignedSequence(S)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.html#getAlignedSequence(int)">getAlignedSequence(int)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns <a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template"><code>AlignedSequence</code></a> at given index.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.html#getAlignedSequence(S)">getAlignedSequence(S)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template">Profile</a></dt>
<dd>
<div class="block">Searches for the given <code>Sequence</code> within this alignment profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#getAlignedSequences()">getAlignedSequences()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#getAlignedSequences(int...)">getAlignedSequences(int...)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#getAlignedSequences(S...)">getAlignedSequences(S...)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.html#getAlignedSequences()">getAlignedSequences()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns a <a href="http://java.sun.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a> containing the individual <a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template"><code>AlignedSequence</code></a>s of this alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.html#getAlignedSequences(int...)">getAlignedSequences(int...)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns a <a href="http://java.sun.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a> containing some of the individual <a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template"><code>AlignedSequence</code></a>s of this alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.html#getAlignedSequences(S...)">getAlignedSequences(S...)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns a <a href="http://java.sun.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a> containing some of the individual <a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template"><code>AlignedSequence</code></a>s of this alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#getAlignmentIndexAt(int)">getAlignmentIndexAt(int)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AlignedSequence.html#getAlignmentIndexAt(int)">getAlignmentIndexAt(int)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns the column index within an alignment corresponding to the given index in the original <code>Sequence</code>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getAlignmentMethod()">getAlignmentMethod()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/GuideTree.Node.html#getAllowsChildren()">getAllowsChildren()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/GuideTree.Node.html" title="class in org.biojava.nbio.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/Alignments.html#getAllPairsAlignments(java.util.List,%20org.biojava.nbio.alignment.Alignments.PairwiseSequenceAlignerType,%20org.biojava.nbio.alignment.template.GapPenalty,%20org.biojava.nbio.alignment.template.SubstitutionMatrix)">getAllPairsAlignments(List&lt;S&gt;, Alignments.PairwiseSequenceAlignerType, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/Alignments.html" title="class in org.biojava.nbio.alignment">Alignments</a></dt>
<dd>
<div class="block">Factory method which computes a sequence alignment for all <code>Sequence</code> pairs in the given <a href="http://java.sun.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/Alignments.html#getAllPairsScorers(java.util.List,%20org.biojava.nbio.alignment.Alignments.PairwiseSequenceScorerType,%20org.biojava.nbio.alignment.template.GapPenalty,%20org.biojava.nbio.alignment.template.SubstitutionMatrix)">getAllPairsScorers(List&lt;S&gt;, Alignments.PairwiseSequenceScorerType, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/Alignments.html" title="class in org.biojava.nbio.alignment">Alignments</a></dt>
<dd>
<div class="block">Factory method which sets up a sequence pair scorer for all <code>Sequence</code> pairs in the given <a href="http://java.sun.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/Alignments.html#getAllPairsScores(java.util.List,%20org.biojava.nbio.alignment.Alignments.PairwiseSequenceScorerType,%20org.biojava.nbio.alignment.template.GapPenalty,%20org.biojava.nbio.alignment.template.SubstitutionMatrix)">getAllPairsScores(List&lt;S&gt;, Alignments.PairwiseSequenceScorerType, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/Alignments.html" title="class in org.biojava.nbio.alignment">Alignments</a></dt>
<dd>
<div class="block">Factory method which computes a sequence pair score for all <code>Sequence</code> pairs in the given <a href="http://java.sun.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/GuideTree.html#getAllPairsScores()">getAllPairsScores()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/GuideTree.html" title="class in org.biojava.nbio.alignment">GuideTree</a></dt>
<dd>
<div class="block">Returns a sequence pair score for all <code>Sequence</code> pairs in the given <a href="http://java.sun.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html#getAminoAcidSubstitutionMatrix(java.lang.String)">getAminoAcidSubstitutionMatrix(String)</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns a substitution matrix for <code>amino acids</code> given by the name <code>name</code>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AnchoredPairwiseSequenceAligner.html#getAnchors()">getAnchors()</a></span> - Method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.routines">AnchoredPairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Returns the list of anchors.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#getAsList()">getAsList()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getAuthors()">getAuthors()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#getBioSequences()">getBioSequences()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>
<div class="block">used to retrieve sequences from the structure</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#getBioSequences(boolean,%20java.lang.String)">getBioSequences(boolean, String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>
<div class="block">This function tolerates mixed case letters, and allows for forcing the output biosequence type (PFAM/RFAM).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#getBioSequences(boolean)">getBioSequences(boolean)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>
<div class="block">Because some database files have incorrectly small letters (e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html#getBlosum100()">getBlosum100()</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 100 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html#getBlosum30()">getBlosum30()</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 30 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html#getBlosum35()">getBlosum35()</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 35 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html#getBlosum40()">getBlosum40()</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 40 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html#getBlosum45()">getBlosum45()</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 45 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html#getBlosum50()">getBlosum50()</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 50 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html#getBlosum55()">getBlosum55()</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 55 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html#getBlosum60()">getBlosum60()</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 60 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html#getBlosum62()">getBlosum62()</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html#getBlosum62()">getBlosum62()</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 62 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html#getBlosum65()">getBlosum65()</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 65 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html#getBlosum70()">getBlosum70()</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 70 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html#getBlosum75()">getBlosum75()</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 75 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html#getBlosum80()">getBlosum80()</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 80 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html#getBlosum85()">getBlosum85()</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 85 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html#getBlosum90()">getBlosum90()</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 90 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getBuildMethod()">getBuildMethod()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/GuideTree.Node.html#getChild1()">getChild1()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/GuideTree.Node.html" title="class in org.biojava.nbio.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/GuideTreeNode.html#getChild1()">getChild1()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/GuideTreeNode.html" title="interface in org.biojava.nbio.alignment.template">GuideTreeNode</a></dt>
<dd>
<div class="block">Returns the first child node of this node.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/GuideTree.Node.html#getChild2()">getChild2()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/GuideTree.Node.html" title="class in org.biojava.nbio.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/GuideTreeNode.html#getChild2()">getChild2()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/GuideTreeNode.html" title="interface in org.biojava.nbio.alignment.template">GuideTreeNode</a></dt>
<dd>
<div class="block">Returns the second child node of this node.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/GuideTree.Node.html#getChildAt(int)">getChildAt(int)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/GuideTree.Node.html" title="class in org.biojava.nbio.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/GuideTree.Node.html#getChildCount()">getChildCount()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/GuideTree.Node.html" title="class in org.biojava.nbio.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getClan()">getClan()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#getCols()">getCols()</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#getColumn(org.biojava.nbio.core.sequence.compound.AminoAcidCompound)">getColumn(AminoAcidCompound)</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html#getColumn(C)">getColumn(C)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html#getColumn(C)">getColumn(C)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.alignment.template">SubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#getComment()">getComment()</a></span> - Static method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getComment()">getComment()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#getCompoundAt(int,%20int)">getCompoundAt(int, int)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#getCompoundAt(S,%20int)">getCompoundAt(S, int)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.html#getCompoundAt(int,%20int)">getCompoundAt(int, int)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the <code>Compound</code> at row of given sequence and column of alignment index.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.html#getCompoundAt(S,%20int)">getCompoundAt(S, int)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the <code>Compound</code> at row of given sequence and column of alignment index.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#getCompoundCountsAt(int)">getCompoundCountsAt(int)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#getCompoundCountsAt(int,%20java.util.List)">getCompoundCountsAt(int, List&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.html#getCompoundCountsAt(int)">getCompoundCountsAt(int)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the number of each <code>Compound</code> in the given column for all compounds in <code>CompoundSet</code>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.html#getCompoundCountsAt(int,%20java.util.List)">getCompoundCountsAt(int, List&lt;C&gt;)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the number of each <code>Compound</code> in the given column only for compounds in the given list.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSequencePair.html#getCompoundInQueryAt(int)">getCompoundInQueryAt(int)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/SequencePair.html#getCompoundInQueryAt(int)">getCompoundInQueryAt(int)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/SequencePair.html" title="interface in org.biojava.nbio.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the <code>Compound</code> in query sequence at given column index in alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSequencePair.html#getCompoundInTargetAt(int)">getCompoundInTargetAt(int)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/SequencePair.html#getCompoundInTargetAt(int)">getCompoundInTargetAt(int)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/SequencePair.html" title="interface in org.biojava.nbio.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the <code>Compound</code> in target sequence at given column index in alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#getCompoundsAt(int)">getCompoundsAt(int)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.html#getCompoundsAt(int)">getCompoundsAt(int)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the <code>Compound</code> elements of the original <code>Sequence</code>s at the given column.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.html#getCompoundSet()">getCompoundSet()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns <code>CompoundSet</code> of all <a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template"><code>AlignedSequence</code></a>s</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html#getCompoundSet()">getCompoundSet()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns the <code>CompoundSet</code> on which the matrix is defined.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#getCompoundsOfQuery()">getCompoundsOfQuery()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#getCompoundsOfQuery()">getCompoundsOfQuery()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html#getCompoundsOfQuery()">getCompoundsOfQuery()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#getCompoundsOfTarget()">getCompoundsOfTarget()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#getCompoundsOfTarget()">getCompoundsOfTarget()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html#getCompoundsOfTarget()">getCompoundsOfTarget()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#getCompoundWeightsAt(int)">getCompoundWeightsAt(int)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#getCompoundWeightsAt(int,%20java.util.List)">getCompoundWeightsAt(int, List&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.html#getCompoundWeightsAt(int)">getCompoundWeightsAt(int)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the fraction of each <code>Compound</code> in the given column for all compounds in <code>CompoundSet</code>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.html#getCompoundWeightsAt(int,%20java.util.List)">getCompoundWeightsAt(int, List&lt;C&gt;)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the fraction of each <code>Compound</code> in the given column only for compounds in the given list.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/StandardRescoreRefiner.html#getComputationTime()">getComputationTime()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/StandardRescoreRefiner.html" title="class in org.biojava.nbio.alignment">StandardRescoreRefiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#getComputationTime()">getComputationTime()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Aligner.html#getComputationTime()">getComputationTime()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Aligner.html" title="interface in org.biojava.nbio.alignment.template">Aligner</a></dt>
<dd>
<div class="block">Returns the computation time needed for an alignment computed in nanoseconds.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#getConsAnnotation()">getConsAnnotation()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html#getCuts(int,%20org.biojava.nbio.alignment.routines.AlignerHelper.Subproblem,%20int[],%20boolean)">getCuts(int, AlignerHelper.Subproblem, int[], boolean)</a></span> - Static method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/GuanUberbacher.html#getCutsPerSection()">getCutsPerSection()</a></span> - Method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/GuanUberbacher.html" title="class in org.biojava.nbio.alignment.routines">GuanUberbacher</a></dt>
<dd>
<div class="block">Returns the number of cuts added to each section during each pass.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html#getDatabase()">getDatabase()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure.DatabaseReference</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getDbComment()">getDbComment()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getDbReferences()">getDbReferences()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getDefinition()">getDefinition()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#getDescription()">getDescription()</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html#getDescription()">getDescription()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html#getDescription()">getDescription()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns the description of this matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractScorer.html#getDistance()">getDistance()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractScorer.html" title="class in org.biojava.nbio.alignment.template">AbstractScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractScorer.html#getDistance(double)">getDistance(double)</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractScorer.html" title="class in org.biojava.nbio.alignment.template">AbstractScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Scorer.html#getDistance()">getDistance()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Scorer.html" title="interface in org.biojava.nbio.alignment.template">Scorer</a></dt>
<dd>
<div class="block">Returns score as a distance between 0.0 and 1.0.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Scorer.html#getDistance(double)">getDistance(double)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Scorer.html" title="interface in org.biojava.nbio.alignment.template">Scorer</a></dt>
<dd>
<div class="block">Returns score as a distance between 0.0 and scale.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/GuideTree.html#getDistanceMatrix()">getDistanceMatrix()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/GuideTree.html" title="class in org.biojava.nbio.alignment">GuideTree</a></dt>
<dd>
<div class="block">Returns the distance matrix used to construct this guide tree.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/HierarchicalClusterer.html#getDistanceMatrix()">getDistanceMatrix()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/HierarchicalClusterer.html" title="interface in org.biojava.nbio.alignment.template">HierarchicalClusterer</a></dt>
<dd>
<div class="block">Returns the distance matrix used in clustering.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/GuideTree.Node.html#getDistanceToParent()">getDistanceToParent()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/GuideTree.Node.html" title="class in org.biojava.nbio.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/GuideTreeNode.html#getDistanceToParent()">getDistanceToParent()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/GuideTreeNode.html" title="interface in org.biojava.nbio.alignment.template">GuideTreeNode</a></dt>
<dd>
<div class="block">Returns the difference in height of this node and it's parent node.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getEmbTrees()">getEmbTrees()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#getEnd()">getEnd()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AlignedSequence.html#getEnd()">getEnd()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns the <code>Point</code> within an alignment of the last element of the original <code>Sequence</code>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/ProfileView.html#getEnd()">getEnd()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/ProfileView.html" title="interface in org.biojava.nbio.alignment.template">ProfileView</a></dt>
<dd>
<div class="block">Returns the column index of the viewed <a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template"><code>Profile</code></a> corresponding to the final element in this view</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleGapPenalty.html#getExtensionPenalty()">getExtensionPenalty()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleGapPenalty.html" title="class in org.biojava.nbio.alignment">SimpleGapPenalty</a></dt>
<dd>
<div class="block"><strong>Returns the negative of the extension penalty passed to the constructor.</strong></div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/GapPenalty.html#getExtensionPenalty()">getExtensionPenalty()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/GapPenalty.html" title="interface in org.biojava.nbio.alignment.template">GapPenalty</a></dt>
<dd>
<div class="block">Returns penalty given when an already open gap elongates by a single element</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getFalseDiscoveryRate()">getFalseDiscoveryRate()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#getFileAnnotation()">getFileAnnotation()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getFormat()">getFormat()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#getGapPenalty()">getGapPenalty()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Returns the gap penalties.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getGatheringThreshs()">getGatheringThreshs()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>
<div class="block">Not always 2.<br>
 It may undergo further change.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html#getGonnet250()">getGonnet250()</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns PAM 250 matrix by Gonnet, Cohen & Benner</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getIdentification()">getIdentification()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html#getIdentity()">getIdentity()</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/GuideTree.Node.html#getIndex(javax.swing.tree.TreeNode)">getIndex(TreeNode)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/GuideTree.Node.html" title="class in org.biojava.nbio.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSequencePair.html#getIndexInQueryAt(int)">getIndexInQueryAt(int)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/SequencePair.html#getIndexInQueryAt(int)">getIndexInQueryAt(int)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/SequencePair.html" title="interface in org.biojava.nbio.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the query index corresponding to a given alignment column.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSequencePair.html#getIndexInQueryForTargetAt(int)">getIndexInQueryForTargetAt(int)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/SequencePair.html#getIndexInQueryForTargetAt(int)">getIndexInQueryForTargetAt(int)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/SequencePair.html" title="interface in org.biojava.nbio.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the query index corresponding to a given target index.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSequencePair.html#getIndexInTargetAt(int)">getIndexInTargetAt(int)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/SequencePair.html#getIndexInTargetAt(int)">getIndexInTargetAt(int)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/SequencePair.html" title="interface in org.biojava.nbio.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the target index corresponding to a given alignment column.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSequencePair.html#getIndexInTargetForQueryAt(int)">getIndexInTargetForQueryAt(int)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/SequencePair.html#getIndexInTargetForQueryAt(int)">getIndexInTargetForQueryAt(int)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/SequencePair.html" title="interface in org.biojava.nbio.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the target index corresponding to a given query index.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template">Profile</a></dt>
<dd>
<div class="block">Searches for the given <code>Compound</code> within this alignment profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#getIndicesAt(int)">getIndicesAt(int)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.html#getIndicesAt(int)">getIndicesAt(int)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the indices in the original <code>Sequence</code>s corresponding to the given column.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/DefaultAAIndexProvider.html#getInputStreamToAAindexFile()">getInputStreamToAAindexFile()</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/DefaultAAIndexProvider.html" title="class in org.biojava.nbio.alignment.aaindex">DefaultAAIndexProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#getInverse()">getInverse()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getKeywords()">getKeywords()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template">Profile</a></dt>
<dd>
<div class="block">Searches for the given <code>Compound</code> within this alignment profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#getLength()">getLength()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#getLength()">getLength()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.html#getLength()">getLength()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the number of columns in the alignment profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getLocation()">getLocation()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#getLocationInAlignment()">getLocationInAlignment()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AlignedSequence.html#getLocationInAlignment()">getLocationInAlignment()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns the <code>Location</code> of the original <code>Sequence</code> within an alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/AAIndexFileParser.html#getMatrices()">getMatrices()</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/AAIndexFileParser.html" title="class in org.biojava.nbio.alignment.aaindex">AAIndexFileParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/AAIndexProvider.html#getMatrix(java.lang.String)">getMatrix(String)</a></span> - Method in interface org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/AAIndexProvider.html" title="interface in org.biojava.nbio.alignment.aaindex">AAIndexProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/DefaultAAIndexProvider.html#getMatrix(java.lang.String)">getMatrix(String)</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/DefaultAAIndexProvider.html" title="class in org.biojava.nbio.alignment.aaindex">DefaultAAIndexProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#getMatrix()">getMatrix()</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html#getMatrix()">getMatrix()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html#getMatrix()">getMatrix()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns entire matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#getMatrixAsString()">getMatrixAsString()</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html#getMatrixAsString()">getMatrixAsString()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html#getMatrixAsString()">getMatrixAsString()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns this matrix as a formatted String with <code>Compound</code> labels along the axes.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html#getMatrixFromAAINDEX(java.lang.String)">getMatrixFromAAINDEX(String)</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns any matrix from the AAINDEX database file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#getMax()">getMax()</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/FractionalIdentityScorer.html#getMaxScore()">getMaxScore()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/FractionalIdentityScorer.html" title="class in org.biojava.nbio.alignment">FractionalIdentityScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/FractionalSimilarityScorer.html#getMaxScore()">getMaxScore()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/FractionalSimilarityScorer.html" title="class in org.biojava.nbio.alignment">FractionalSimilarityScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/StandardRescoreRefiner.html#getMaxScore()">getMaxScore()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/StandardRescoreRefiner.html" title="class in org.biojava.nbio.alignment">StandardRescoreRefiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixScorer.html#getMaxScore()">getMaxScore()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixScorer.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#getMaxScore()">getMaxScore()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Scorer.html#getMaxScore()">getMaxScore()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Scorer.html" title="interface in org.biojava.nbio.alignment.template">Scorer</a></dt>
<dd>
<div class="block">Returns maximum possible score.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#getMaxValue()">getMaxValue()</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html#getMaxValue()">getMaxValue()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html#getMaxValue()">getMaxValue()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns the maximum value in this matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getMembership()">getMembership()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#getMin()">getMin()</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/FractionalIdentityScorer.html#getMinScore()">getMinScore()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/FractionalIdentityScorer.html" title="class in org.biojava.nbio.alignment">FractionalIdentityScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/FractionalSimilarityScorer.html#getMinScore()">getMinScore()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/FractionalSimilarityScorer.html" title="class in org.biojava.nbio.alignment">FractionalSimilarityScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/StandardRescoreRefiner.html#getMinScore()">getMinScore()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/StandardRescoreRefiner.html" title="class in org.biojava.nbio.alignment">StandardRescoreRefiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixScorer.html#getMinScore()">getMinScore()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixScorer.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#getMinScore()">getMinScore()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Scorer.html#getMinScore()">getMinScore()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Scorer.html" title="interface in org.biojava.nbio.alignment.template">Scorer</a></dt>
<dd>
<div class="block">Returns minimum possible score.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#getMinValue()">getMinValue()</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html#getMinValue()">getMinValue()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html#getMinValue()">getMinValue()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns the minimum value in this matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/Alignments.html#getMultipleSequenceAlignment(java.util.List,%20java.lang.Object...)">getMultipleSequenceAlignment(List&lt;S&gt;, Object...)</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/Alignments.html" title="class in org.biojava.nbio.alignment">Alignments</a></dt>
<dd>
<div class="block">Factory method which computes a multiple sequence alignment for the given <a href="http://java.sun.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a> of <code>Sequence</code>s.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#getName()">getName()</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/GuideTree.Node.html#getName()">getName()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/GuideTree.Node.html" title="class in org.biojava.nbio.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html#getName()">getName()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/GuideTreeNode.html#getName()">getName()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/GuideTreeNode.html" title="interface in org.biojava.nbio.alignment.template">GuideTreeNode</a></dt>
<dd>
<div class="block">Returns the name of this node.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html#getName()">getName()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns the name (short description) of this matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getNoiseCutoffs()">getNoiseCutoffs()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>
<div class="block">Not always 2.<br>
 It may undergo further change.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html#getNuc4_2()">getNuc4_2()</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Nuc 4.2 matrix by Lowe
 Only the first nucleotide sequence to align can contain ambiguous nucleotides</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html#getNuc4_4()">getNuc4_4()</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Nuc 4.4 matrix by Lowe
 Both of the nucleotide sequences to align can contain ambiguous nucleotides</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#getNumGaps()">getNumGaps()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AlignedSequence.html#getNumGaps()">getNumGaps()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns number of gaps in the sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSequencePair.html#getNumIdenticals()">getNumIdenticals()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/SequencePair.html#getNumIdenticals()">getNumIdenticals()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/SequencePair.html" title="interface in org.biojava.nbio.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the number of indices for which both the query and target sequences have an identical <code>Compound</code>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getNumSequences()">getNumSequences()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSequencePair.html#getNumSimilars()">getNumSimilars()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/SequencePair.html#getNumSimilars()">getNumSimilars()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/SequencePair.html" title="interface in org.biojava.nbio.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the number of indices for which both the query and target sequences have a similar <code>Compound</code>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleGapPenalty.html#getOpenPenalty()">getOpenPenalty()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleGapPenalty.html" title="class in org.biojava.nbio.alignment">SimpleGapPenalty</a></dt>
<dd>
<div class="block"><strong>Returns the negative of the opening penalty passed to the constructor.</strong></div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/GapPenalty.html#getOpenPenalty()">getOpenPenalty()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/GapPenalty.html" title="interface in org.biojava.nbio.alignment.template">GapPenalty</a></dt>
<dd>
<div class="block">Returns penalty given when a deletion or insertion gap first opens</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#getOriginalSequence()">getOriginalSequence()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AlignedSequence.html#getOriginalSequence()">getOriginalSequence()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns the original <code>Sequence</code> before alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#getOriginalSequences()">getOriginalSequences()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.html#getOriginalSequences()">getOriginalSequences()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns a <a href="http://java.sun.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a> containing the original <code>Sequence</code>s used for alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#getOverlapCount()">getOverlapCount()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AlignedSequence.html#getOverlapCount()">getOverlapCount()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns the maximum number of elements contributed to a column of an alignment by this <code>Sequence</code>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#getPair()">getPair()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html#getPair()">getPair()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/PairwiseSequenceAligner.html#getPair()">getPair()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/PairwiseSequenceAligner.html" title="interface in org.biojava.nbio.alignment.template">PairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Returns sequence alignment pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/ProfileProfileAligner.html#getPair()">getPair()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/ProfileProfileAligner.html" title="interface in org.biojava.nbio.alignment.template">ProfileProfileAligner</a></dt>
<dd>
<div class="block">Returns alignment profile pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/StandardRescoreRefiner.html#getPairInProfileScorer()">getPairInProfileScorer()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/StandardRescoreRefiner.html" title="class in org.biojava.nbio.alignment">StandardRescoreRefiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/RescoreRefiner.html#getPairInProfileScorer()">getPairInProfileScorer()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/RescoreRefiner.html" title="interface in org.biojava.nbio.alignment.template">RescoreRefiner</a></dt>
<dd>
<div class="block">Returns class type of pairwise scoring subroutine.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/Alignments.html#getPairwiseAligner(S,%20S,%20org.biojava.nbio.alignment.Alignments.PairwiseSequenceAlignerType,%20org.biojava.nbio.alignment.template.GapPenalty,%20org.biojava.nbio.alignment.template.SubstitutionMatrix)">getPairwiseAligner(S, S, Alignments.PairwiseSequenceAlignerType, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/Alignments.html" title="class in org.biojava.nbio.alignment">Alignments</a></dt>
<dd>
<div class="block">Factory method which constructs a pairwise sequence aligner.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/Alignments.html#getPairwiseAlignment(S,%20S,%20org.biojava.nbio.alignment.Alignments.PairwiseSequenceAlignerType,%20org.biojava.nbio.alignment.template.GapPenalty,%20org.biojava.nbio.alignment.template.SubstitutionMatrix)">getPairwiseAlignment(S, S, Alignments.PairwiseSequenceAlignerType, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/Alignments.html" title="class in org.biojava.nbio.alignment">Alignments</a></dt>
<dd>
<div class="block">Factory method which computes a sequence alignment for the given <code>Sequence</code> pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html#getPAM250()">getPAM250()</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns PAM 250 matrix by Dayhoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/GuideTree.Node.html#getParent()">getParent()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/GuideTree.Node.html" title="class in org.biojava.nbio.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getPfamAccession()">getPfamAccession()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getPreviousIDs()">getPreviousIDs()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/FractionalIdentityInProfileScorer.html#getProfile()">getProfile()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/FractionalIdentityInProfileScorer.html" title="class in org.biojava.nbio.alignment">FractionalIdentityInProfileScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/FractionalSimilarityInProfileScorer.html#getProfile()">getProfile()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/FractionalSimilarityInProfileScorer.html" title="class in org.biojava.nbio.alignment">FractionalSimilarityInProfileScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/GuideTree.Node.html#getProfile()">getProfile()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/GuideTree.Node.html" title="class in org.biojava.nbio.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/StandardRescoreRefiner.html#getProfile()">getProfile()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/StandardRescoreRefiner.html" title="class in org.biojava.nbio.alignment">StandardRescoreRefiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#getProfile()">getProfile()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Aligner.html#getProfile()">getProfile()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Aligner.html" title="interface in org.biojava.nbio.alignment.template">Aligner</a></dt>
<dd>
<div class="block">Returns the alignment <a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template"><code>Profile</code></a> produced by this alignment algorithm.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/GuideTreeNode.html#getProfile()">getProfile()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/GuideTreeNode.html" title="interface in org.biojava.nbio.alignment.template">GuideTreeNode</a></dt>
<dd>
<div class="block">Returns the profile stored at this node.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/PairInProfileScorer.html#getProfile()">getProfile()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/PairInProfileScorer.html" title="interface in org.biojava.nbio.alignment.template">PairInProfileScorer</a></dt>
<dd>
<div class="block">Returns the <a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template"><code>Profile</code></a> from which the aligned pair originated.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/GuideTree.Node.html#getProfileFuture()">getProfileFuture()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/GuideTree.Node.html" title="class in org.biojava.nbio.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/GuideTreeNode.html#getProfileFuture()">getProfileFuture()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/GuideTreeNode.html" title="interface in org.biojava.nbio.alignment.template">GuideTreeNode</a></dt>
<dd>
<div class="block">Returns the profile future stored at this node, but does not force the calculation, yet.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/StandardRescoreRefiner.html#getProfileProfileAligner()">getProfileProfileAligner()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/StandardRescoreRefiner.html" title="class in org.biojava.nbio.alignment">StandardRescoreRefiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/PartitionRefiner.html#getProfileProfileAligner()">getProfileProfileAligner()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/PartitionRefiner.html" title="interface in org.biojava.nbio.alignment.template">PartitionRefiner</a></dt>
<dd>
<div class="block">Returns class type of profile-profile alignment subroutine.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/RescoreRefiner.html#getProfileProfileAligner()">getProfileProfileAligner()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/RescoreRefiner.html" title="interface in org.biojava.nbio.alignment.template">RescoreRefiner</a></dt>
<dd>
<div class="block">Returns class type of profile-profile alignment subroutine.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/Alignments.html#getProgressiveAlignment(org.biojava.nbio.alignment.GuideTree,%20org.biojava.nbio.alignment.Alignments.ProfileProfileAlignerType,%20org.biojava.nbio.alignment.template.GapPenalty,%20org.biojava.nbio.alignment.template.SubstitutionMatrix)">getProgressiveAlignment(GuideTree&lt;S, C&gt;, Alignments.ProfileProfileAlignerType, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/Alignments.html" title="class in org.biojava.nbio.alignment">Alignments</a></dt>
<dd>
<div class="block">Factory method to run the profile-profile alignments of a progressive multiple sequence alignment concurrently.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/FractionalIdentityScorer.html#getQuery()">getQuery()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/FractionalIdentityScorer.html" title="class in org.biojava.nbio.alignment">FractionalIdentityScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/FractionalSimilarityScorer.html#getQuery()">getQuery()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/FractionalSimilarityScorer.html" title="class in org.biojava.nbio.alignment">FractionalSimilarityScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfilePair.html#getQuery()">getQuery()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfilePair.html" title="class in org.biojava.nbio.alignment">SimpleProfilePair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSequencePair.html#getQuery()">getQuery()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixScorer.html#getQuery()">getQuery()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixScorer.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#getQuery()">getQuery()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html#getQuery()">getQuery()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/PairwiseSequenceScorer.html#getQuery()">getQuery()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/PairwiseSequenceScorer.html" title="interface in org.biojava.nbio.alignment.template">PairwiseSequenceScorer</a></dt>
<dd>
<div class="block">Returns the first sequence of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/ProfilePair.html#getQuery()">getQuery()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/ProfilePair.html" title="interface in org.biojava.nbio.alignment.template">ProfilePair</a></dt>
<dd>
<div class="block">Returns the first <a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template"><code>Profile</code></a> of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/ProfileProfileScorer.html#getQuery()">getQuery()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/ProfileProfileScorer.html" title="interface in org.biojava.nbio.alignment.template">ProfileProfileScorer</a></dt>
<dd>
<div class="block">Returns the first profile of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/SequencePair.html#getQuery()">getQuery()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/SequencePair.html" title="interface in org.biojava.nbio.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the first <a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template"><code>AlignedSequence</code></a> of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Subproblem.html#getQueryEndIndex()">getQueryEndIndex()</a></span> - Method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Subproblem.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper.Subproblem</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Anchor.html#getQueryIndex()">getQueryIndex()</a></span> - Method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Anchor.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper.Anchor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Cut.html#getQueryIndex()">getQueryIndex()</a></span> - Method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Cut.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper.Cut</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Subproblem.html#getQueryStartIndex()">getQueryStartIndex()</a></span> - Method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Subproblem.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper.Subproblem</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.StockholmFileAnnotationReference.html#getRefAuthor()">getRefAuthor()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.StockholmFileAnnotationReference.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation.StockholmFileAnnotationReference</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getRefComment()">getRefComment()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html#getReference()">getReference()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure.DatabaseReference</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getReferences()">getReferences()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.StockholmFileAnnotationReference.html#getRefLocation()">getRefLocation()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.StockholmFileAnnotationReference.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation.StockholmFileAnnotationReference</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.StockholmFileAnnotationReference.html#getRefMedline()">getRefMedline()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.StockholmFileAnnotationReference.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation.StockholmFileAnnotationReference</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.StockholmFileAnnotationReference.html#getRefTitle()">getRefTitle()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.StockholmFileAnnotationReference.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation.StockholmFileAnnotationReference</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/GuideTree.html#getRoot()">getRoot()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/GuideTree.html" title="class in org.biojava.nbio.alignment">GuideTree</a></dt>
<dd>
<div class="block">Returns the root <a href="./org/biojava/nbio/alignment/GuideTree.Node.html" title="class in org.biojava.nbio.alignment"><code>GuideTree.Node</code></a> which corresponds to the full multiple sequence alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/HierarchicalClusterer.html#getRoot()">getRoot()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/HierarchicalClusterer.html" title="interface in org.biojava.nbio.alignment.template">HierarchicalClusterer</a></dt>
<dd>
<div class="block">Returns the root node of the tree resulting from this clustering algorithm.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#getRow(org.biojava.nbio.core.sequence.compound.AminoAcidCompound)">getRow(AminoAcidCompound)</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html#getRow(C)">getRow(C)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html#getRow(C)">getRow(C)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.alignment.template">SubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#getRows()">getRows()</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#getScale()">getScale()</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/FractionalIdentityScorer.html#getScore()">getScore()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/FractionalIdentityScorer.html" title="class in org.biojava.nbio.alignment">FractionalIdentityScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/FractionalSimilarityScorer.html#getScore()">getScore()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/FractionalSimilarityScorer.html" title="class in org.biojava.nbio.alignment">FractionalSimilarityScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/StandardRescoreRefiner.html#getScore()">getScore()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/StandardRescoreRefiner.html" title="class in org.biojava.nbio.alignment">StandardRescoreRefiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixScorer.html#getScore()">getScore()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixScorer.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#getScore()">getScore()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Scorer.html#getScore()">getScore()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Scorer.html" title="interface in org.biojava.nbio.alignment.template">Scorer</a></dt>
<dd>
<div class="block">Returns score resulting from algorithm.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/GuideTree.html#getScoreMatrix()">getScoreMatrix()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/GuideTree.html" title="class in org.biojava.nbio.alignment">GuideTree</a></dt>
<dd>
<div class="block">Returns the similarity matrix used to construct this guide tree.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#getScoreMatrix()">getScoreMatrix()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/MatrixAligner.html#getScoreMatrix()">getScoreMatrix()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/MatrixAligner.html" title="interface in org.biojava.nbio.alignment.template">MatrixAligner</a></dt>
<dd>
<div class="block">Returns the entire score matrix built during alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#getScoreMatrixAsString()">getScoreMatrixAsString()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/MatrixAligner.html#getScoreMatrixAsString()">getScoreMatrixAsString()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/MatrixAligner.html" title="interface in org.biojava.nbio.alignment.template">MatrixAligner</a></dt>
<dd>
<div class="block">Returns a depiction of the score matrix as a <a href="http://java.sun.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang"><code>String</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#getScoreMatrixDimensions()">getScoreMatrixDimensions()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#getScoreMatrixDimensions()">getScoreMatrixDimensions()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html#getScoreMatrixDimensions()">getScoreMatrixDimensions()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getSearchMethod()">getSearchMethod()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#getSequenceIndexAt(int)">getSequenceIndexAt(int)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AlignedSequence.html#getSequenceIndexAt(int)">getSequenceIndexAt(int)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns the index in the original <code>Sequence</code> corresponding to the given index within an alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/GuideTree.html#getSequences()">getSequences()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/GuideTree.html" title="class in org.biojava.nbio.alignment">GuideTree</a></dt>
<dd>
<div class="block">Returns the <code>Sequence</code>s which make up the leaves of this tree.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#getSequences()">getSequences()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractScorer.html#getSimilarity()">getSimilarity()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractScorer.html" title="class in org.biojava.nbio.alignment.template">AbstractScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractScorer.html#getSimilarity(double)">getSimilarity(double)</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractScorer.html" title="class in org.biojava.nbio.alignment.template">AbstractScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Scorer.html#getSimilarity()">getSimilarity()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Scorer.html" title="interface in org.biojava.nbio.alignment.template">Scorer</a></dt>
<dd>
<div class="block">Returns score as a similarity between 0.0 and 1.0.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Scorer.html#getSimilarity(double)">getSimilarity(double)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Scorer.html" title="interface in org.biojava.nbio.alignment.template">Scorer</a></dt>
<dd>
<div class="block">Returns score as a similarity between 0.0 and scale.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#getSize()">getSize()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.html#getSize()">getSize()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the number of rows in this profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getSourceSeed()">getSourceSeed()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getSourceStructure()">getSourceStructure()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#getStart()">getStart()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AlignedSequence.html#getStart()">getStart()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns the <code>Point</code> within an alignment of the first element of the original <code>Sequence</code>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/ProfileView.html#getStart()">getStart()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/ProfileView.html" title="interface in org.biojava.nbio.alignment.template">ProfileView</a></dt>
<dd>
<div class="block">Returns the column index of the viewed <a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template"><code>Profile</code></a> corresponding to the first element in this view</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Subproblem.html#getSubproblems(java.util.List,%20int,%20int)">getSubproblems(List&lt;AlignerHelper.Anchor&gt;, int, int)</a></span> - Static method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Subproblem.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper.Subproblem</a></dt>
<dd>
<div class="block">Convert a list of anchors into a subproblem list.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#getSubProfile(org.biojava.nbio.core.sequence.location.template.Location)">getSubProfile(Location)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.html#getSubProfile(org.biojava.nbio.core.sequence.location.template.Location)">getSubProfile(Location)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns a <a href="./org/biojava/nbio/alignment/template/ProfileView.html" title="interface in org.biojava.nbio.alignment.template"><code>ProfileView</code></a> windowed to contain only the given <code>Location</code>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#getSubSequence(java.lang.Integer,%20java.lang.Integer)">getSubSequence(Integer, Integer)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#getSubstitutionMatrix()">getSubstitutionMatrix()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Returns the substitution matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#getSubstitutionScore(int,%20int)">getSubstitutionScore(int, int)</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#getSubstitutionScore(int,%20int)">getSubstitutionScore(int, int)</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html#getSubstitutionScore(int,%20int)">getSubstitutionScore(int, int)</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#getSubstitutionScoreVector(int)">getSubstitutionScoreVector(int)</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Returns score for the alignment of the query column to all target columns</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#getSubstitutionScoreVector(int,%20org.biojava.nbio.alignment.routines.AlignerHelper.Subproblem)">getSubstitutionScoreVector(int, AlignerHelper.Subproblem)</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Returns score for the alignment of the query column to all target columns</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/FractionalIdentityScorer.html#getTarget()">getTarget()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/FractionalIdentityScorer.html" title="class in org.biojava.nbio.alignment">FractionalIdentityScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/FractionalSimilarityScorer.html#getTarget()">getTarget()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/FractionalSimilarityScorer.html" title="class in org.biojava.nbio.alignment">FractionalSimilarityScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfilePair.html#getTarget()">getTarget()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfilePair.html" title="class in org.biojava.nbio.alignment">SimpleProfilePair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSequencePair.html#getTarget()">getTarget()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixScorer.html#getTarget()">getTarget()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixScorer.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#getTarget()">getTarget()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html#getTarget()">getTarget()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/PairwiseSequenceScorer.html#getTarget()">getTarget()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/PairwiseSequenceScorer.html" title="interface in org.biojava.nbio.alignment.template">PairwiseSequenceScorer</a></dt>
<dd>
<div class="block">Returns the second sequence of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/ProfilePair.html#getTarget()">getTarget()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/ProfilePair.html" title="interface in org.biojava.nbio.alignment.template">ProfilePair</a></dt>
<dd>
<div class="block">Returns the second <a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template"><code>Profile</code></a> of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/ProfileProfileScorer.html#getTarget()">getTarget()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/ProfileProfileScorer.html" title="interface in org.biojava.nbio.alignment.template">ProfileProfileScorer</a></dt>
<dd>
<div class="block">Returns the second profile of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/SequencePair.html#getTarget()">getTarget()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/SequencePair.html" title="interface in org.biojava.nbio.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the second <a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template"><code>AlignedSequence</code></a> of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Subproblem.html#getTargetEndIndex()">getTargetEndIndex()</a></span> - Method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Subproblem.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper.Subproblem</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Anchor.html#getTargetIndex()">getTargetIndex()</a></span> - Method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Anchor.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper.Anchor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Cut.html#getTargetIndex(int)">getTargetIndex(int)</a></span> - Method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Cut.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper.Cut</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Subproblem.html#getTargetStartIndex()">getTargetStartIndex()</a></span> - Method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Subproblem.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper.Subproblem</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getTrustedCutoffs()">getTrustedCutoffs()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>
<div class="block">Not always 2.<br>
 It may undergo further change.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleGapPenalty.html#getType()">getType()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleGapPenalty.html" title="class in org.biojava.nbio.alignment">SimpleGapPenalty</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/GapPenalty.html#getType()">getType()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/GapPenalty.html" title="interface in org.biojava.nbio.alignment.template">GapPenalty</a></dt>
<dd>
<div class="block">Returns <a href="./org/biojava/nbio/alignment/template/GapPenalty.Type.html" title="enum in org.biojava.nbio.alignment.template"><code>GapPenalty.Type</code></a> stored.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getTypeField()">getTypeField()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#getValue(org.biojava.nbio.core.sequence.compound.AminoAcidCompound,%20org.biojava.nbio.core.sequence.compound.AminoAcidCompound)">getValue(AminoAcidCompound, AminoAcidCompound)</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html#getValue(C,%20C)">getValue(C, C)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html#getValue(C,%20C)">getValue(C, C)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns value in matrix for conversion from first <code>Compound</code> to the second.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getVersion()">getVersion()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/ProfileView.html#getViewedProfile()">getViewedProfile()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/ProfileView.html" title="interface in org.biojava.nbio.alignment.template">ProfileView</a></dt>
<dd>
<div class="block">Returns the entire <a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template"><code>Profile</code></a> being viewed</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#getWikipediaLink()">getWikipediaLink()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/routines/GuanUberbacher.html" title="class in org.biojava.nbio.alignment.routines"><span class="strong">GuanUberbacher</span></a>&lt;<a href="./org/biojava/nbio/alignment/routines/GuanUberbacher.html" title="type parameter in GuanUberbacher">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/routines/GuanUberbacher.html" title="type parameter in GuanUberbacher">C</a>&gt;,<a href="./org/biojava/nbio/alignment/routines/GuanUberbacher.html" title="type parameter in GuanUberbacher">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Class in <a href="./org/biojava/nbio/alignment/routines/package-summary.html">org.biojava.nbio.alignment.routines</a></dt>
<dd>
<div class="block">Guan and Uberbacher defined an algorithm for pairwise global sequence alignments (from the first until the last
 <code>Compound</code> of each <code>Sequence</code>).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/GuanUberbacher.html#GuanUberbacher()">GuanUberbacher()</a></span> - Constructor for class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/GuanUberbacher.html" title="class in org.biojava.nbio.alignment.routines">GuanUberbacher</a></dt>
<dd>
<div class="block">Before running a pairwise global sequence alignment, data must be sent in via calls to
 <a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#setQuery(S)"><code>AbstractPairwiseSequenceAligner.setQuery(Sequence)</code></a>, <a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#setTarget(S)"><code>AbstractPairwiseSequenceAligner.setTarget(Sequence)</code></a>, <a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#setGapPenalty(org.biojava.nbio.alignment.template.GapPenalty)"><code>setGapPenalty(GapPenalty)</code></a>, and
 <a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#setSubstitutionMatrix(org.biojava.nbio.alignment.template.SubstitutionMatrix)"><code>setSubstitutionMatrix(SubstitutionMatrix)</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/GuanUberbacher.html#GuanUberbacher(S,%20S,%20org.biojava.nbio.alignment.template.GapPenalty,%20org.biojava.nbio.alignment.template.SubstitutionMatrix)">GuanUberbacher(S, S, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/GuanUberbacher.html" title="class in org.biojava.nbio.alignment.routines">GuanUberbacher</a></dt>
<dd>
<div class="block">Prepares for a pairwise global sequence alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/GuanUberbacher.html#GuanUberbacher(S,%20S,%20org.biojava.nbio.alignment.template.GapPenalty,%20org.biojava.nbio.alignment.template.SubstitutionMatrix,%20int)">GuanUberbacher(S, S, GapPenalty, SubstitutionMatrix&lt;C&gt;, int)</a></span> - Constructor for class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/GuanUberbacher.html" title="class in org.biojava.nbio.alignment.routines">GuanUberbacher</a></dt>
<dd>
<div class="block">Prepares for a pairwise global sequence alignment.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/GuideTree.html" title="class in org.biojava.nbio.alignment"><span class="strong">GuideTree</span></a>&lt;<a href="./org/biojava/nbio/alignment/GuideTree.html" title="type parameter in GuideTree">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/GuideTree.html" title="type parameter in GuideTree">C</a>&gt;,<a href="./org/biojava/nbio/alignment/GuideTree.html" title="type parameter in GuideTree">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Class in <a href="./org/biojava/nbio/alignment/package-summary.html">org.biojava.nbio.alignment</a></dt>
<dd>
<div class="block">Implements a data structure for a guide tree used during progressive multiple sequence alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/GuideTree.html#GuideTree(java.util.List,%20java.util.List)">GuideTree(List&lt;S&gt;, List&lt;PairwiseSequenceScorer&lt;S, C&gt;&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/GuideTree.html" title="class in org.biojava.nbio.alignment">GuideTree</a></dt>
<dd>
<div class="block">Creates a guide tree for use during progressive multiple sequence alignment.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/GuideTree.Node.html" title="class in org.biojava.nbio.alignment"><span class="strong">GuideTree.Node</span></a> - Class in <a href="./org/biojava/nbio/alignment/package-summary.html">org.biojava.nbio.alignment</a></dt>
<dd>
<div class="block">Implements a data structure for the node in a guide tree used during progressive multiple sequence alignment.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/template/GuideTreeNode.html" title="interface in org.biojava.nbio.alignment.template"><span class="strong">GuideTreeNode</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/GuideTreeNode.html" title="type parameter in GuideTreeNode">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/GuideTreeNode.html" title="type parameter in GuideTreeNode">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/GuideTreeNode.html" title="type parameter in GuideTreeNode">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Interface in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Defines a data structure for the node in a guide tree used during progressive multiple sequence alignment.</div>
</dd>
</dl>
<a name="_H_">
<!--   -->
</a>
<h2 class="title">H</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#hasGap(int)">hasGap(int)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.html#hasGap(int)">hasGap(int)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns true if any <a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template"><code>AlignedSequence</code></a> has a gap at the given index.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/template/HierarchicalClusterer.html" title="interface in org.biojava.nbio.alignment.template"><span class="strong">HierarchicalClusterer</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/HierarchicalClusterer.html" title="type parameter in HierarchicalClusterer">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/HierarchicalClusterer.html" title="type parameter in HierarchicalClusterer">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/HierarchicalClusterer.html" title="type parameter in HierarchicalClusterer">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Interface in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Defines a clustering algorithm that converts a distance matrix into a tree.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html#HOMSTRAD">HOMSTRAD</a></span> - Static variable in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure.DatabaseReference</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_I_">
<!--   -->
</a>
<h2 class="title">I</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileParser.html#INFINITY">INFINITY</a></span> - Static variable in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileParser.html" title="class in org.biojava.nbio.alignment.io">StockholmFileParser</a></dt>
<dd>
<div class="block">indicates reading as much as possible, without limits</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html#INTERPRO">INTERPRO</a></span> - Static variable in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure.DatabaseReference</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#isCircular()">isCircular()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#isCircular()">isCircular()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AlignedSequence.html#isCircular()">isCircular()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns true if this <code>Sequence</code> wraps around from the last alignment column back to the first.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.html#isCircular()">isCircular()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns true if any <a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template"><code>AlignedSequence</code></a> is circular.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#isGap(int)">isGap(int)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AlignedSequence.html#isGap(int)">isGap(int)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns true if this <code>Sequence</code> has a gap at a particular alignment column.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/GuideTree.Node.html#isLeaf()">isLeaf()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/GuideTree.Node.html" title="class in org.biojava.nbio.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#isLocal()">isLocal()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Returns whether alignment finds a region of similarity rather than aligning every compound.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#isPFam()">isPFam()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#isReady()">isReady()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#isReady()">isReady()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html#isReady()">isReady()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#isRFam()">isRFam()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Subproblem.html#isStartAnchored()">isStartAnchored()</a></span> - Method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Subproblem.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper.Subproblem</a></dt>
<dd>
<div class="block">Indicates whether the start query and start target index compounds
 are anchored to each other</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#isStoringScoreMatrix()">isStoringScoreMatrix()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Returns choice to cache the score matrix or to save memory by deleting score matrix after alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/GuideTree.html#iterator()">iterator()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/GuideTree.html" title="class in org.biojava.nbio.alignment">GuideTree</a></dt>
<dd>
<div class="block">Returns a post-order <a href="http://java.sun.com/javase/6/docs/api/java/util/Iterator.html?is-external=true" title="class or interface in java.util"><code>Iterator</code></a> that traverses the tree from leaves to root.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#iterator()">iterator()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#iterator()">iterator()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_L_">
<!--   -->
</a>
<h2 class="title">L</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html#LOAD">LOAD</a></span> - Static variable in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure.DatabaseReference</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_M_">
<!--   -->
</a>
<h2 class="title">M</h2>
<dl>
<dt><span class="strong"><a href="./demo/CookbookMSA.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/CookbookMSA.html" title="class in demo">CookbookMSA</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoAlignProteins.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/DemoAlignProteins.html" title="class in demo">DemoAlignProteins</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoLoadSubstMax.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/DemoLoadSubstMax.html" title="class in demo">DemoLoadSubstMax</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/TestDNANeedlemanWunsch.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/TestDNANeedlemanWunsch.html" title="class in demo">TestDNANeedlemanWunsch</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#matrix">matrix</a></span> - Static variable in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/template/MatrixAligner.html" title="interface in org.biojava.nbio.alignment.template"><span class="strong">MatrixAligner</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/MatrixAligner.html" title="type parameter in MatrixAligner">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/MatrixAligner.html" title="type parameter in MatrixAligner">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/MatrixAligner.html" title="type parameter in MatrixAligner">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Interface in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Defines an <a href="./org/biojava/nbio/alignment/template/Aligner.html" title="interface in org.biojava.nbio.alignment.template"><code>Aligner</code></a> which builds a score matrix during computation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#max">max</a></span> - Variable in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html#MIM">MIM</a></span> - Static variable in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure.DatabaseReference</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#min">min</a></span> - Variable in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/template/MutableAlignedSequence.html" title="interface in org.biojava.nbio.alignment.template"><span class="strong">MutableAlignedSequence</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/MutableAlignedSequence.html" title="type parameter in MutableAlignedSequence">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/MutableAlignedSequence.html" title="type parameter in MutableAlignedSequence">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/MutableAlignedSequence.html" title="type parameter in MutableAlignedSequence">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Interface in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Defines a mutable (editable) data structure for an <a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template"><code>AlignedSequence</code></a>.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/template/MutableProfile.html" title="interface in org.biojava.nbio.alignment.template"><span class="strong">MutableProfile</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/MutableProfile.html" title="type parameter in MutableProfile">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/MutableProfile.html" title="type parameter in MutableProfile">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/MutableProfile.html" title="type parameter in MutableProfile">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Interface in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Defines a mutable (editable) data structure for a <a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template"><code>Profile</code></a>.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/template/MutableProfilePair.html" title="interface in org.biojava.nbio.alignment.template"><span class="strong">MutableProfilePair</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/MutableProfilePair.html" title="type parameter in MutableProfilePair">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/MutableProfilePair.html" title="type parameter in MutableProfilePair">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/MutableProfilePair.html" title="type parameter in MutableProfilePair">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Interface in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Defines a mutable (editable) data structure for a <a href="./org/biojava/nbio/alignment/template/ProfilePair.html" title="interface in org.biojava.nbio.alignment.template"><code>ProfilePair</code></a>.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/template/MutableSequencePair.html" title="interface in org.biojava.nbio.alignment.template"><span class="strong">MutableSequencePair</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/MutableSequencePair.html" title="type parameter in MutableSequencePair">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/MutableSequencePair.html" title="type parameter in MutableSequencePair">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/MutableSequencePair.html" title="type parameter in MutableSequencePair">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Interface in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Defines a mutable (editable) data structure for the results of pairwise sequence alignment.</div>
</dd>
</dl>
<a name="_N_">
<!--   -->
</a>
<h2 class="title">N</h2>
<dl>
<dt><a href="./org/biojava/nbio/alignment/NeedlemanWunsch.html" title="class in org.biojava.nbio.alignment"><span class="strong">NeedlemanWunsch</span></a>&lt;<a href="./org/biojava/nbio/alignment/NeedlemanWunsch.html" title="type parameter in NeedlemanWunsch">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/NeedlemanWunsch.html" title="type parameter in NeedlemanWunsch">C</a>&gt;,<a href="./org/biojava/nbio/alignment/NeedlemanWunsch.html" title="type parameter in NeedlemanWunsch">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Class in <a href="./org/biojava/nbio/alignment/package-summary.html">org.biojava.nbio.alignment</a></dt>
<dd>
<div class="block">Needleman and Wunsch defined an algorithm for pairwise global sequence alignments (from the first until the last
 <code>Compound</code> of each <code>Sequence</code>).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/NeedlemanWunsch.html#NeedlemanWunsch()">NeedlemanWunsch()</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/NeedlemanWunsch.html" title="class in org.biojava.nbio.alignment">NeedlemanWunsch</a></dt>
<dd>
<div class="block">Before running a pairwise global sequence alignment, data must be sent in via calls to
 <a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#setQuery(S)"><code>AbstractPairwiseSequenceAligner.setQuery(Sequence)</code></a>, <a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#setTarget(S)"><code>AbstractPairwiseSequenceAligner.setTarget(Sequence)</code></a>, <a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#setGapPenalty(org.biojava.nbio.alignment.template.GapPenalty)"><code>setGapPenalty(GapPenalty)</code></a>, and
 <a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#setSubstitutionMatrix(org.biojava.nbio.alignment.template.SubstitutionMatrix)"><code>setSubstitutionMatrix(SubstitutionMatrix)</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/NeedlemanWunsch.html#NeedlemanWunsch(S,%20S,%20org.biojava.nbio.alignment.template.GapPenalty,%20org.biojava.nbio.alignment.template.SubstitutionMatrix)">NeedlemanWunsch(S, S, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/NeedlemanWunsch.html" title="class in org.biojava.nbio.alignment">NeedlemanWunsch</a></dt>
<dd>
<div class="block">Prepares for a pairwise global sequence alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#normalizeMatrix(short)">normalizeMatrix(short)</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html#normalizeMatrix(short)">normalizeMatrix(short)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html#normalizeMatrix(short)">normalizeMatrix(short)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Rescales the matrix so that to <a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html#getMaxValue()"><code>SubstitutionMatrix.getMaxValue()</code></a> - <a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html#getMinValue()"><code>SubstitutionMatrix.getMinValue()</code></a> = scale.</div>
</dd>
</dl>
<a name="_O_">
<!--   -->
</a>
<h2 class="title">O</h2>
<dl>
<dt><a href="./org/biojava/nbio/alignment/package-summary.html">org.biojava.nbio.alignment</a> - package org.biojava.nbio.alignment</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/aaindex/package-summary.html">org.biojava.nbio.alignment.aaindex</a> - package org.biojava.nbio.alignment.aaindex</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/io/package-summary.html">org.biojava.nbio.alignment.io</a> - package org.biojava.nbio.alignment.io</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/routines/package-summary.html">org.biojava.nbio.alignment.routines</a> - package org.biojava.nbio.alignment.routines</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a> - package org.biojava.nbio.alignment.template</dt>
<dd>&nbsp;</dd>
</dl>
<a name="_P_">
<!--   -->
</a>
<h2 class="title">P</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#pair">pair</a></span> - Variable in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html#pair">pair</a></span> - Variable in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/template/PairInProfileScorer.html" title="interface in org.biojava.nbio.alignment.template"><span class="strong">PairInProfileScorer</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/PairInProfileScorer.html" title="type parameter in PairInProfileScorer">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/PairInProfileScorer.html" title="type parameter in PairInProfileScorer">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/PairInProfileScorer.html" title="type parameter in PairInProfileScorer">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Interface in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Defines an algorithm which computes a score for a sequence alignment pair picked from an alignment <a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template"><code>Profile</code></a>.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/template/PairwiseSequenceAligner.html" title="interface in org.biojava.nbio.alignment.template"><span class="strong">PairwiseSequenceAligner</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/PairwiseSequenceAligner.html" title="type parameter in PairwiseSequenceAligner">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/PairwiseSequenceAligner.html" title="type parameter in PairwiseSequenceAligner">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/PairwiseSequenceAligner.html" title="type parameter in PairwiseSequenceAligner">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Interface in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Defines an <a href="./org/biojava/nbio/alignment/template/Aligner.html" title="interface in org.biojava.nbio.alignment.template"><code>Aligner</code></a> for a pair of <code>Sequence</code>s.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/template/PairwiseSequenceScorer.html" title="interface in org.biojava.nbio.alignment.template"><span class="strong">PairwiseSequenceScorer</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/PairwiseSequenceScorer.html" title="type parameter in PairwiseSequenceScorer">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/PairwiseSequenceScorer.html" title="type parameter in PairwiseSequenceScorer">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/PairwiseSequenceScorer.html" title="type parameter in PairwiseSequenceScorer">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Interface in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Defines an algorithm which computes a score for a pair of sequences.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/AAIndexFileParser.html#parse(java.io.InputStream)">parse(InputStream)</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/AAIndexFileParser.html" title="class in org.biojava.nbio.alignment.aaindex">AAIndexFileParser</a></dt>
<dd>
<div class="block">parse an inputStream that points to an AAINDEX database file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileParser.html#parse(java.lang.String)">parse(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileParser.html" title="class in org.biojava.nbio.alignment.io">StockholmFileParser</a></dt>
<dd>
<div class="block">Parses a Stockholm file and returns a <a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io"><code>StockholmStructure</code></a> object with its content.<br>
 This function is meant to be used for single access to specific file and it closes the file after doing its
 assigned job.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileParser.html#parse(java.lang.String,%20int)">parse(String, int)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileParser.html" title="class in org.biojava.nbio.alignment.io">StockholmFileParser</a></dt>
<dd>
<div class="block">Parses a Stockholm file and returns a <a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io"><code>StockholmStructure</code></a> object with its content.<br>
 This function doesn't close the file after doing its assigned job; to allow for further calls of
 <a href="./org/biojava/nbio/alignment/io/StockholmFileParser.html#parseNext(int)"><code>StockholmFileParser.parseNext(int)</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileParser.html#parse(java.io.InputStream)">parse(InputStream)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileParser.html" title="class in org.biojava.nbio.alignment.io">StockholmFileParser</a></dt>
<dd>
<div class="block">parses <a href="http://java.sun.com/javase/6/docs/api/java/io/InputStream.html?is-external=true" title="class or interface in java.io"><code>InputStream</code></a> and returns a the first contained alignment in a <a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io"><code>StockholmStructure</code></a> object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileParser.html#parse(java.io.InputStream,%20int)">parse(InputStream, int)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileParser.html" title="class in org.biojava.nbio.alignment.io">StockholmFileParser</a></dt>
<dd>
<div class="block">parses an <a href="http://java.sun.com/javase/6/docs/api/java/io/InputStream.html?is-external=true" title="class or interface in java.io"><code>InputStream</code></a> and returns at maximum <code>max</code> objects contained in that file.<br>
 This method leaves the stream open for further calls of <a href="./org/biojava/nbio/alignment/io/StockholmFileParser.html#parse(java.io.InputStream,%20int)"><code>StockholmFileParser.parse(InputStream, int)</code></a> (same function) or
 <a href="./org/biojava/nbio/alignment/io/StockholmFileParser.html#parseNext(int)"><code>StockholmFileParser.parseNext(int)</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileParser.html#parseNext(int)">parseNext(int)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileParser.html" title="class in org.biojava.nbio.alignment.io">StockholmFileParser</a></dt>
<dd>
<div class="block">Tries to parse and return as maximum as <code>max</code> structures in the last used file or input stream.<br>
 Please consider calling either <a href="./org/biojava/nbio/alignment/io/StockholmFileParser.html#parse(java.io.InputStream)"><code>StockholmFileParser.parse(InputStream)</code></a>, <a href="./org/biojava/nbio/alignment/io/StockholmFileParser.html#parse(java.io.InputStream,%20int)"><code>StockholmFileParser.parse(InputStream, int)</code></a>, or
 <a href="./org/biojava/nbio/alignment/io/StockholmFileParser.html#parse(java.lang.String,%20int)"><code>StockholmFileParser.parse(String, int)</code></a> before calling this function.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/template/PartitionRefiner.html" title="interface in org.biojava.nbio.alignment.template"><span class="strong">PartitionRefiner</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/PartitionRefiner.html" title="type parameter in PartitionRefiner">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/PartitionRefiner.html" title="type parameter in PartitionRefiner">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/PartitionRefiner.html" title="type parameter in PartitionRefiner">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Interface in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Defines an algorithm which computes a new alignment <a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template"><code>Profile</code></a> by splitting a current alignment and realigning.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html#PDB">PDB</a></span> - Static variable in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure.DatabaseReference</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#PFAM">PFAM</a></span> - Static variable in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html#PFAMB">PFAMB</a></span> - Static variable in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure.DatabaseReference</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html#PRINTS">PRINTS</a></span> - Static variable in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure.DatabaseReference</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#profile">profile</a></span> - Variable in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template"><span class="strong">Profile</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/Profile.html" title="type parameter in Profile">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/Profile.html" title="type parameter in Profile">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/Profile.html" title="type parameter in Profile">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Interface in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Defines a data structure for the results of sequence alignment.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/template/Profile.StringFormat.html" title="enum in org.biojava.nbio.alignment.template"><span class="strong">Profile.StringFormat</span></a> - Enum in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">List of output formats.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/template/ProfilePair.html" title="interface in org.biojava.nbio.alignment.template"><span class="strong">ProfilePair</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/ProfilePair.html" title="type parameter in ProfilePair">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/ProfilePair.html" title="type parameter in ProfilePair">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/ProfilePair.html" title="type parameter in ProfilePair">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Interface in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Defines a data structure for the results of the alignment of a pair of <a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template"><code>Profile</code></a>s.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/template/ProfileProfileAligner.html" title="interface in org.biojava.nbio.alignment.template"><span class="strong">ProfileProfileAligner</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/ProfileProfileAligner.html" title="type parameter in ProfileProfileAligner">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/ProfileProfileAligner.html" title="type parameter in ProfileProfileAligner">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/ProfileProfileAligner.html" title="type parameter in ProfileProfileAligner">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Interface in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Defines an <a href="./org/biojava/nbio/alignment/template/Aligner.html" title="interface in org.biojava.nbio.alignment.template"><code>Aligner</code></a> for a pair of <a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template"><code>Profile</code></a>s.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/template/ProfileProfileScorer.html" title="interface in org.biojava.nbio.alignment.template"><span class="strong">ProfileProfileScorer</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/ProfileProfileScorer.html" title="type parameter in ProfileProfileScorer">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/ProfileProfileScorer.html" title="type parameter in ProfileProfileScorer">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/ProfileProfileScorer.html" title="type parameter in ProfileProfileScorer">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Interface in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Defines an algorithm which computes a score for a pairing of alignment profiles.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/template/ProfileView.html" title="interface in org.biojava.nbio.alignment.template"><span class="strong">ProfileView</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/ProfileView.html" title="type parameter in ProfileView">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/ProfileView.html" title="type parameter in ProfileView">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/ProfileView.html" title="type parameter in ProfileView">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Interface in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Defines a data structure for a view of sequence alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html#PROSITE">PROSITE</a></span> - Static variable in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure.DatabaseReference</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html#PROSITE_PROFILE">PROSITE_PROFILE</a></span> - Static variable in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure.DatabaseReference</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_R_">
<!--   -->
</a>
<h2 class="title">R</h2>
<dl>
<dt><a href="./org/biojava/nbio/alignment/template/RescoreRefiner.html" title="interface in org.biojava.nbio.alignment.template"><span class="strong">RescoreRefiner</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/RescoreRefiner.html" title="type parameter in RescoreRefiner">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/RescoreRefiner.html" title="type parameter in RescoreRefiner">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/RescoreRefiner.html" title="type parameter in RescoreRefiner">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Interface in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Defines an algorithm which computes a new alignment <a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template"><code>Profile</code></a> by rescoring all pairs in an alignment and
 realigning.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#reset()">reset()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Resets output fields; should be overridden to set max and min</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#reset()">reset()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html#reset()">reset()</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#RFAM">RFAM</a></span> - Static variable in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/Alignments.html#runPairwiseScorers(java.util.List)">runPairwiseScorers(List&lt;PairwiseSequenceScorer&lt;S, C&gt;&gt;)</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/Alignments.html" title="class in org.biojava.nbio.alignment">Alignments</a></dt>
<dd>
<div class="block">Factory method to run a list of scorers concurrently.</div>
</dd>
</dl>
<a name="_S_">
<!--   -->
</a>
<h2 class="title">S</h2>
<dl>
<dt><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex"><span class="strong">ScaledSubstitutionMatrix</span></a> - Class in <a href="./org/biojava/nbio/alignment/aaindex/package-summary.html">org.biojava.nbio.alignment.aaindex</a></dt>
<dd>
<div class="block">The biojava-alignment module represents substitution matrices with short
 values.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#ScaledSubstitutionMatrix()">ScaledSubstitutionMatrix()</a></span> - Constructor for class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html#SCOP">SCOP</a></span> - Static variable in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure.DatabaseReference</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#score">score</a></span> - Variable in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/template/Scorer.html" title="interface in org.biojava.nbio.alignment.template"><span class="strong">Scorer</span></a> - Interface in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Defines an algorithm which computes a score.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#scores">scores</a></span> - Variable in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Dynamic programming score matrix
 The first dimension has the length of the first (query) sequence + 1
 The second has the length of the second (target) sequence + 1
 The third has length 1 for linear gap penalty and 3 for affine/constant gap
 (one each for match/substitution, deletion, insertion)</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/template/SequencePair.html" title="interface in org.biojava.nbio.alignment.template"><span class="strong">SequencePair</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/SequencePair.html" title="type parameter in SequencePair">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/template/SequencePair.html" title="type parameter in SequencePair">C</a>&gt;,<a href="./org/biojava/nbio/alignment/template/SequencePair.html" title="type parameter in SequencePair">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Interface in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Defines a data structure for the results of pairwise sequence alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/AAindexFactory.html#setAAIndexProvider(org.biojava.nbio.alignment.aaindex.AAIndexProvider)">setAAIndexProvider(AAIndexProvider)</a></span> - Static method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/AAindexFactory.html" title="class in org.biojava.nbio.alignment.aaindex">AAindexFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setAlignmentMethod(java.lang.String)">setAlignmentMethod(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AnchoredPairwiseSequenceAligner.html#setAnchors(int[])">setAnchors(int[])</a></span> - Method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.routines">AnchoredPairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Sets the starting list of anchors before running the alignment routine.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#setCols(java.util.List)">setCols(List&lt;AminoAcidCompound&gt;)</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html#setCuts(int,%20org.biojava.nbio.alignment.routines.AlignerHelper.Subproblem,%20org.biojava.nbio.alignment.routines.AlignerHelper.Last[][],%20org.biojava.nbio.alignment.routines.AlignerHelper.Cut[])">setCuts(int, AlignerHelper.Subproblem, AlignerHelper.Last[][], AlignerHelper.Cut[])</a></span> - Static method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/GuanUberbacher.html#setCutsPerSection(int)">setCutsPerSection(int)</a></span> - Method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/GuanUberbacher.html" title="class in org.biojava.nbio.alignment.routines">GuanUberbacher</a></dt>
<dd>
<div class="block">Sets the number of cuts added to each section during each pass.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setDbComment(java.lang.String)">setDbComment(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setDbReferences(java.util.Set)">setDbReferences(Set&lt;StockholmStructure.DatabaseReference&gt;)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/GuanUberbacher.html#setDefaultCutsPerSection(int)">setDefaultCutsPerSection(int)</a></span> - Static method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/GuanUberbacher.html" title="class in org.biojava.nbio.alignment.routines">GuanUberbacher</a></dt>
<dd>
<div class="block">Sets the default number of cuts added to each section during each pass.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleGapPenalty.html#setDefaultExtensionPenalty(int)">setDefaultExtensionPenalty(int)</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleGapPenalty.html" title="class in org.biojava.nbio.alignment">SimpleGapPenalty</a></dt>
<dd>
<div class="block">Sets the default gap extension penalty.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleGapPenalty.html#setDefaultOpenPenalty(int)">setDefaultOpenPenalty(int)</a></span> - Static method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleGapPenalty.html" title="class in org.biojava.nbio.alignment">SimpleGapPenalty</a></dt>
<dd>
<div class="block">Sets the default gap open penalty.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Sets the description of this matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleGapPenalty.html#setExtensionPenalty(int)">setExtensionPenalty(int)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleGapPenalty.html" title="class in org.biojava.nbio.alignment">SimpleGapPenalty</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/GapPenalty.html#setExtensionPenalty(int)">setExtensionPenalty(int)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/GapPenalty.html" title="interface in org.biojava.nbio.alignment.template">GapPenalty</a></dt>
<dd>
<div class="block">Sets penalty given when an already open gap elongates by a single element</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setFalseDiscoveryRate(float)">setFalseDiscoveryRate(float)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setFormat(java.lang.String)">setFormat(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#setGapPenalty(org.biojava.nbio.alignment.template.GapPenalty)">setGapPenalty(GapPenalty)</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Sets the gap penalties.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setGFAccessionNumber(java.lang.String)">setGFAccessionNumber(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setGFAuthors(java.lang.String)">setGFAuthors(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setGFClan(java.lang.String)">setGFClan(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setGFDBComment(java.lang.String)">setGFDBComment(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setGFDefinition(java.lang.String)">setGFDefinition(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setGFGatheringThreshs(java.lang.String)">setGFGatheringThreshs(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setGFIdentification(java.lang.String)">setGFIdentification(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setGFKeywords(java.lang.String)">setGFKeywords(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setGFLocation(java.lang.String)">setGFLocation(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setGFMembership(java.lang.String)">setGFMembership(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setGFNoiseCutoffs(java.lang.String)">setGFNoiseCutoffs(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setGFNumSequences(java.lang.String)">setGFNumSequences(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setGFPfamAccession(java.lang.String)">setGFPfamAccession(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setGFPreviousIDs(java.lang.String)">setGFPreviousIDs(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setGFRefComment(java.lang.String)">setGFRefComment(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setGFSearchMethod(java.lang.String)">setGFSearchMethod(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setGFSourceSeed(java.lang.String)">setGFSourceSeed(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setGFSourceStructure(java.lang.String)">setGFSourceStructure(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setGFTrustedCutoffs(java.lang.String)">setGFTrustedCutoffs(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setGFTypeField(java.lang.String)">setGFTypeField(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setGFWikipediaLink(java.lang.String)">setGFWikipediaLink(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/MutableAlignedSequence.html#setLocationInAlignment(org.biojava.nbio.core.sequence.location.template.Location)">setLocationInAlignment(Location)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/MutableAlignedSequence.html" title="interface in org.biojava.nbio.alignment.template">MutableAlignedSequence</a></dt>
<dd>
<div class="block">Sets the position of the <a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template"><code>AlignedSequence</code></a> to the given <code>Location</code> (start, gaps, end).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#setMatrix(short[][])">setMatrix(short[][])</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#setMax(short)">setMax(short)</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#setMin(short)">setMin(short)</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#setName(java.lang.String)">setName(String)</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html#setName(java.lang.String)">setName(String)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html#setName(java.lang.String)">setName(String)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Sets the name (short description) of this matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setNumSequences(int)">setNumSequences(int)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleGapPenalty.html#setOpenPenalty(int)">setOpenPenalty(int)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleGapPenalty.html" title="class in org.biojava.nbio.alignment">SimpleGapPenalty</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/GapPenalty.html#setOpenPenalty(int)">setOpenPenalty(int)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/GapPenalty.html" title="interface in org.biojava.nbio.alignment.template">GapPenalty</a></dt>
<dd>
<div class="block">Sets penalty given when a deletion or insertion gap first opens</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/MutableProfilePair.html#setPair(org.biojava.nbio.alignment.template.Profile,%20org.biojava.nbio.alignment.template.Profile)">setPair(Profile&lt;S, C&gt;, Profile&lt;S, C&gt;)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/MutableProfilePair.html" title="interface in org.biojava.nbio.alignment.template">MutableProfilePair</a></dt>
<dd>
<div class="block">Sets both <a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template"><code>Profile</code></a>s of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/MutableSequencePair.html#setPair(org.biojava.nbio.alignment.template.AlignedSequence,%20org.biojava.nbio.alignment.template.AlignedSequence)">setPair(AlignedSequence&lt;S, C&gt;, AlignedSequence&lt;S, C&gt;)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/MutableSequencePair.html" title="interface in org.biojava.nbio.alignment.template">MutableSequencePair</a></dt>
<dd>
<div class="block">Sets both <a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template"><code>AlignedSequence</code></a>s of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/GuideTree.Node.html#setProfile(org.biojava.nbio.alignment.template.Profile)">setProfile(Profile&lt;S, C&gt;)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/GuideTree.Node.html" title="class in org.biojava.nbio.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AnchoredPairwiseSequenceAligner.html#setProfile(java.util.List,%20java.util.List)">setProfile(List&lt;AlignedSequence.Step&gt;, List&lt;AlignedSequence.Step&gt;)</a></span> - Method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.routines">AnchoredPairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfileProfileAligner.html#setProfile(java.util.List,%20java.util.List)">setProfile(List&lt;AlignedSequence.Step&gt;, List&lt;AlignedSequence.Step&gt;)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfileProfileAligner.html" title="class in org.biojava.nbio.alignment">SimpleProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SmithWaterman.html#setProfile(java.util.List,%20java.util.List)">setProfile(List&lt;AlignedSequence.Step&gt;, List&lt;AlignedSequence.Step&gt;)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SmithWaterman.html" title="class in org.biojava.nbio.alignment">SmithWaterman</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#setProfile(java.util.List,%20java.util.List)">setProfile(List&lt;AlignedSequence.Step&gt;, List&lt;AlignedSequence.Step&gt;)</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/GuideTreeNode.html#setProfile(org.biojava.nbio.alignment.template.Profile)">setProfile(Profile&lt;S, C&gt;)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/GuideTreeNode.html" title="interface in org.biojava.nbio.alignment.template">GuideTreeNode</a></dt>
<dd>
<div class="block">Stores the given profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/GuideTree.Node.html#setProfileFuture(java.util.concurrent.Future)">setProfileFuture(Future&lt;ProfilePair&lt;S, C&gt;&gt;)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/GuideTree.Node.html" title="class in org.biojava.nbio.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/GuideTreeNode.html#setProfileFuture(java.util.concurrent.Future)">setProfileFuture(Future&lt;ProfilePair&lt;S, C&gt;&gt;)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/GuideTreeNode.html" title="interface in org.biojava.nbio.alignment.template">GuideTreeNode</a></dt>
<dd>
<div class="block">Stores the given profile future.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#setQuery(S)">setQuery(S)</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Sets the query <code>Sequence</code>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html#setQuery(org.biojava.nbio.alignment.template.Profile)">setQuery(Profile&lt;S, C&gt;)</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>
<div class="block">Sets the query <a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template"><code>Profile</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/MutableProfilePair.html#setQuery(org.biojava.nbio.alignment.template.Profile)">setQuery(Profile&lt;S, C&gt;)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/MutableProfilePair.html" title="interface in org.biojava.nbio.alignment.template">MutableProfilePair</a></dt>
<dd>
<div class="block">Sets the first <a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template"><code>Profile</code></a> of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/MutableSequencePair.html#setQuery(org.biojava.nbio.alignment.template.AlignedSequence)">setQuery(AlignedSequence&lt;S, C&gt;)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/MutableSequencePair.html" title="interface in org.biojava.nbio.alignment.template">MutableSequencePair</a></dt>
<dd>
<div class="block">Sets the first <a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template"><code>AlignedSequence</code></a> of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.StockholmFileAnnotationReference.html#setRefAuthor(java.lang.StringBuffer)">setRefAuthor(StringBuffer)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.StockholmFileAnnotationReference.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation.StockholmFileAnnotationReference</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setReferences(java.util.Vector)">setReferences(Vector&lt;StockholmFileAnnotation.StockholmFileAnnotationReference&gt;)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.StockholmFileAnnotationReference.html#setRefLocation(java.lang.String)">setRefLocation(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.StockholmFileAnnotationReference.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation.StockholmFileAnnotationReference</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.StockholmFileAnnotationReference.html#setRefMedline(java.lang.String)">setRefMedline(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.StockholmFileAnnotationReference.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation.StockholmFileAnnotationReference</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.StockholmFileAnnotationReference.html#setRefTitle(java.lang.String)">setRefTitle(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.StockholmFileAnnotationReference.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation.StockholmFileAnnotationReference</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#setRows(java.util.List)">setRows(List&lt;AminoAcidCompound&gt;)</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#setScale(int)">setScale(int)</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html#setScorePoint(int,%20int,%20int,%20int,%20int,%20int[][][])">setScorePoint(int, int, int, int, int, int[][][])</a></span> - Static method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper</a></dt>
<dd>
<div class="block">Calculate the optimal alignment score for the given sequence positions with an affine or constant gap penalty</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html#setScorePoint(int,%20int,%20int,%20int,%20int[][][])">setScorePoint(int, int, int, int, int[][][])</a></span> - Static method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper</a></dt>
<dd>
<div class="block">Calculates the optimal alignment score for the given sequence positions and a linear gap penalty</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html#setScoreVector(int,%20org.biojava.nbio.alignment.routines.AlignerHelper.Subproblem,%20int,%20int,%20int[],%20boolean,%20int[][][])">setScoreVector(int, AlignerHelper.Subproblem, int, int, int[], boolean, int[][][])</a></span> - Static method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper</a></dt>
<dd>
<div class="block">Score global alignment for a given position in the query sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html#setScoreVector(int,%20int,%20int,%20int,%20int,%20int,%20int[],%20boolean,%20int[][][],%20boolean)">setScoreVector(int, int, int, int, int, int, int[], boolean, int[][][], boolean)</a></span> - Static method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper</a></dt>
<dd>
<div class="block">Score global alignment for a given position in the query sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html#setScoreVector(int,%20org.biojava.nbio.alignment.routines.AlignerHelper.Subproblem,%20int,%20int[],%20boolean,%20int[][][])">setScoreVector(int, AlignerHelper.Subproblem, int, int[], boolean, int[][][])</a></span> - Static method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper</a></dt>
<dd>
<div class="block">Score global alignment for a given position in the query sequence for a linear gap penalty</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html#setScoreVector(int,%20int,%20int,%20int,%20int,%20int[],%20boolean,%20int[][][],%20boolean)">setScoreVector(int, int, int, int, int, int[], boolean, int[][][], boolean)</a></span> - Static method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper</a></dt>
<dd>
<div class="block">Score global alignment for a given position in the query sequence for a linear gap penalty</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html#setScoreVector(int,%20int,%20int,%20int[],%20boolean,%20int[][][],%20int[],%20int)">setScoreVector(int, int, int, int[], boolean, int[][][], int[], int)</a></span> - Static method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper</a></dt>
<dd>
<div class="block">Score local alignment for a given position in the query sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html#setScoreVector(int,%20int,%20int,%20int,%20int,%20int,%20int[],%20boolean,%20int[][][],%20int[],%20int)">setScoreVector(int, int, int, int, int, int, int[], boolean, int[][][], int[], int)</a></span> - Static method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper</a></dt>
<dd>
<div class="block">Score local alignment for a given position in the query sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html#setScoreVector(int,%20int,%20int[],%20boolean,%20int[][][],%20int[],%20int)">setScoreVector(int, int, int[], boolean, int[][][], int[], int)</a></span> - Static method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper</a></dt>
<dd>
<div class="block">Score local alignment for a given position in the query sequence for a linear gap penalty</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html#setScoreVector(int,%20int,%20int,%20int,%20int,%20int[],%20boolean,%20int[][][],%20int[],%20int)">setScoreVector(int, int, int, int, int, int[], boolean, int[][][], int[], int)</a></span> - Static method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper</a></dt>
<dd>
<div class="block">Score local alignment for a given position in the query sequence for a linear gap penalty</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/MutableProfile.html#setSequences(java.util.List)">setSequences(List&lt;AlignedSequence&lt;S, C&gt;&gt;)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/MutableProfile.html" title="interface in org.biojava.nbio.alignment.template">MutableProfile</a></dt>
<dd>
<div class="block">Sets entirely new set of <a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template"><code>AlignedSequence</code></a>s.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html#setSteps(org.biojava.nbio.alignment.routines.AlignerHelper.Last[][][],%20boolean,%20int[],%20org.biojava.nbio.alignment.routines.AlignerHelper.Last,%20java.util.List,%20java.util.List)">setSteps(AlignerHelper.Last[][][], boolean, int[], AlignerHelper.Last, List&lt;AlignedSequence.Step&gt;, List&lt;AlignedSequence.Step&gt;)</a></span> - Static method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper</a></dt>
<dd>
<div class="block">Find alignment path through traceback matrix</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html#setSteps(org.biojava.nbio.alignment.routines.AlignerHelper.Last[][][],%20int[][][],%20java.util.List,%20java.util.List)">setSteps(AlignerHelper.Last[][][], int[][][], List&lt;AlignedSequence.Step&gt;, List&lt;AlignedSequence.Step&gt;)</a></span> - Static method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper</a></dt>
<dd>
<div class="block">Find global alignment path through traceback matrix</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html#setSteps(org.biojava.nbio.alignment.routines.AlignerHelper.Last[][][],%20int[],%20java.util.List,%20java.util.List)">setSteps(AlignerHelper.Last[][][], int[], List&lt;AlignedSequence.Step&gt;, List&lt;AlignedSequence.Step&gt;)</a></span> - Static method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper</a></dt>
<dd>
<div class="block">Find local alignment path through traceback matrix</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#setStoringScoreMatrix(boolean)">setStoringScoreMatrix(boolean)</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Sets choice to cache the score matrix or to save memory by deleting score matrix after alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#setSubstitutionMatrix(org.biojava.nbio.alignment.template.SubstitutionMatrix)">setSubstitutionMatrix(SubstitutionMatrix&lt;C&gt;)</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Sets the substitution matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#setTarget(S)">setTarget(S)</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Sets the target <code>Sequence</code>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html#setTarget(org.biojava.nbio.alignment.template.Profile)">setTarget(Profile&lt;S, C&gt;)</a></span> - Method in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>
<div class="block">Sets the target <a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template"><code>Profile</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/MutableProfilePair.html#setTarget(org.biojava.nbio.alignment.template.Profile)">setTarget(Profile&lt;S, C&gt;)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/MutableProfilePair.html" title="interface in org.biojava.nbio.alignment.template">MutableProfilePair</a></dt>
<dd>
<div class="block">Sets the second <a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template"><code>Profile</code></a> of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/MutableSequencePair.html#setTarget(org.biojava.nbio.alignment.template.AlignedSequence)">setTarget(AlignedSequence&lt;S, C&gt;)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/MutableSequencePair.html" title="interface in org.biojava.nbio.alignment.template">MutableSequencePair</a></dt>
<dd>
<div class="block">Sets the second <a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template"><code>AlignedSequence</code></a> of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#setVersion(java.lang.String)">setVersion(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/MutableAlignedSequence.html#shiftAtAlignmentLocation(org.biojava.nbio.core.sequence.location.template.Location,%20int)">shiftAtAlignmentLocation(Location, int)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/MutableAlignedSequence.html" title="interface in org.biojava.nbio.alignment.template">MutableAlignedSequence</a></dt>
<dd>
<div class="block">Slides a part of the <a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template"><code>AlignedSequence</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/MutableAlignedSequence.html#shiftAtSequenceLocation(org.biojava.nbio.core.sequence.location.template.Location,%20int)">shiftAtSequenceLocation(Location, int)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/MutableAlignedSequence.html" title="interface in org.biojava.nbio.alignment.template">MutableAlignedSequence</a></dt>
<dd>
<div class="block">Slides a part of the <a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template"><code>AlignedSequence</code></a>.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment"><span class="strong">SimpleAlignedSequence</span></a>&lt;<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="type parameter in SimpleAlignedSequence">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="type parameter in SimpleAlignedSequence">C</a>&gt;,<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="type parameter in SimpleAlignedSequence">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Class in <a href="./org/biojava/nbio/alignment/package-summary.html">org.biojava.nbio.alignment</a></dt>
<dd>
<div class="block">Implements a data structure for a <code>Sequence</code> within an alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#SimpleAlignedSequence(S,%20java.util.List)">SimpleAlignedSequence(S, List&lt;AlignedSequence.Step&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>
<div class="block">Creates an <a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template"><code>AlignedSequence</code></a> for the given <code>Sequence</code> in a global alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#SimpleAlignedSequence(S,%20java.util.List,%20int,%20int)">SimpleAlignedSequence(S, List&lt;AlignedSequence.Step&gt;, int, int)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>
<div class="block">Creates an <a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template"><code>AlignedSequence</code></a> for the given <code>Sequence</code> in a local alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#SimpleAlignedSequence(org.biojava.nbio.alignment.template.AlignedSequence,%20java.util.List)">SimpleAlignedSequence(AlignedSequence&lt;S, C&gt;, List&lt;AlignedSequence.Step&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>
<div class="block">Creates a new <a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template"><code>AlignedSequence</code></a> for the given <a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template"><code>AlignedSequence</code></a> in a global alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#SimpleAlignedSequence(org.biojava.nbio.alignment.template.AlignedSequence,%20java.util.List,%20int,%20int)">SimpleAlignedSequence(AlignedSequence&lt;S, C&gt;, List&lt;AlignedSequence.Step&gt;, int, int)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>
<div class="block">Creates a new <a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template"><code>AlignedSequence</code></a> for the given <a href="./org/biojava/nbio/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.alignment.template"><code>AlignedSequence</code></a> in a local alignment.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/SimpleGapPenalty.html" title="class in org.biojava.nbio.alignment"><span class="strong">SimpleGapPenalty</span></a> - Class in <a href="./org/biojava/nbio/alignment/package-summary.html">org.biojava.nbio.alignment</a></dt>
<dd>
<div class="block">Implements a data structure for the gap penalties used during a sequence alignment routine.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleGapPenalty.html#SimpleGapPenalty()">SimpleGapPenalty()</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleGapPenalty.html" title="class in org.biojava.nbio.alignment">SimpleGapPenalty</a></dt>
<dd>
<div class="block">Creates a new set of gap penalties using the defaults.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleGapPenalty.html#SimpleGapPenalty(int,%20int)">SimpleGapPenalty(int, int)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleGapPenalty.html" title="class in org.biojava.nbio.alignment">SimpleGapPenalty</a></dt>
<dd>
<div class="block">Creates a new set of gap penalties.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment"><span class="strong">SimpleProfile</span></a>&lt;<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="type parameter in SimpleProfile">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="type parameter in SimpleProfile">C</a>&gt;,<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="type parameter in SimpleProfile">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Class in <a href="./org/biojava/nbio/alignment/package-summary.html">org.biojava.nbio.alignment</a></dt>
<dd>
<div class="block">Implements a data structure for the results of sequence alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#SimpleProfile(org.biojava.nbio.alignment.template.AlignedSequence,%20org.biojava.nbio.alignment.template.AlignedSequence)">SimpleProfile(AlignedSequence&lt;S, C&gt;, AlignedSequence&lt;S, C&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>
<div class="block">Creates a pair profile for the given already aligned sequences.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#SimpleProfile(S)">SimpleProfile(S)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>
<div class="block">Creates a profile from a single sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#SimpleProfile(S,%20S,%20java.util.List,%20int,%20int,%20java.util.List,%20int,%20int)">SimpleProfile(S, S, List&lt;AlignedSequence.Step&gt;, int, int, List&lt;AlignedSequence.Step&gt;, int, int)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>
<div class="block">Creates a pair profile for the given sequences.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#SimpleProfile(org.biojava.nbio.alignment.template.Profile,%20org.biojava.nbio.alignment.template.Profile,%20java.util.List,%20java.util.List)">SimpleProfile(Profile&lt;S, C&gt;, Profile&lt;S, C&gt;, List&lt;AlignedSequence.Step&gt;, List&lt;AlignedSequence.Step&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>
<div class="block">Creates a pair profile for the given profiles.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#SimpleProfile(java.util.Collection)">SimpleProfile(Collection&lt;AlignedSequence&lt;S, C&gt;&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>
<div class="block">Creates a profile for the already aligned sequences.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/SimpleProfilePair.html" title="class in org.biojava.nbio.alignment"><span class="strong">SimpleProfilePair</span></a>&lt;<a href="./org/biojava/nbio/alignment/SimpleProfilePair.html" title="type parameter in SimpleProfilePair">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/SimpleProfilePair.html" title="type parameter in SimpleProfilePair">C</a>&gt;,<a href="./org/biojava/nbio/alignment/SimpleProfilePair.html" title="type parameter in SimpleProfilePair">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Class in <a href="./org/biojava/nbio/alignment/package-summary.html">org.biojava.nbio.alignment</a></dt>
<dd>
<div class="block">Implements a data structure for the results of the alignment of a pair of <a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template"><code>Profile</code></a>s.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfilePair.html#SimpleProfilePair(org.biojava.nbio.alignment.template.Profile,%20org.biojava.nbio.alignment.template.Profile,%20java.util.List,%20java.util.List)">SimpleProfilePair(Profile&lt;S, C&gt;, Profile&lt;S, C&gt;, List&lt;AlignedSequence.Step&gt;, List&lt;AlignedSequence.Step&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfilePair.html" title="class in org.biojava.nbio.alignment">SimpleProfilePair</a></dt>
<dd>
<div class="block">Creates a pair profile for the given profiles.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/SimpleProfileProfileAligner.html" title="class in org.biojava.nbio.alignment"><span class="strong">SimpleProfileProfileAligner</span></a>&lt;<a href="./org/biojava/nbio/alignment/SimpleProfileProfileAligner.html" title="type parameter in SimpleProfileProfileAligner">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/SimpleProfileProfileAligner.html" title="type parameter in SimpleProfileProfileAligner">C</a>&gt;,<a href="./org/biojava/nbio/alignment/SimpleProfileProfileAligner.html" title="type parameter in SimpleProfileProfileAligner">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Class in <a href="./org/biojava/nbio/alignment/package-summary.html">org.biojava.nbio.alignment</a></dt>
<dd>
<div class="block">Implements a simple (naive) <a href="./org/biojava/nbio/alignment/template/Aligner.html" title="interface in org.biojava.nbio.alignment.template"><code>Aligner</code></a> for a pair of <a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template"><code>Profile</code></a>s.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfileProfileAligner.html#SimpleProfileProfileAligner()">SimpleProfileProfileAligner()</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfileProfileAligner.html" title="class in org.biojava.nbio.alignment">SimpleProfileProfileAligner</a></dt>
<dd>
<div class="block">Before running a profile-profile alignment, data must be sent in via calls to
 <a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html#setQuery(org.biojava.nbio.alignment.template.Profile)"><code>AbstractProfileProfileAligner.setQuery(Profile)</code></a>, <a href="./org/biojava/nbio/alignment/template/AbstractProfileProfileAligner.html#setTarget(org.biojava.nbio.alignment.template.Profile)"><code>AbstractProfileProfileAligner.setTarget(Profile)</code></a>, <a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#setGapPenalty(org.biojava.nbio.alignment.template.GapPenalty)"><code>setGapPenalty(GapPenalty)</code></a>, and
 <a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#setSubstitutionMatrix(org.biojava.nbio.alignment.template.SubstitutionMatrix)"><code>setSubstitutionMatrix(SubstitutionMatrix)</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfileProfileAligner.html#SimpleProfileProfileAligner(org.biojava.nbio.alignment.template.Profile,%20org.biojava.nbio.alignment.template.Profile,%20org.biojava.nbio.alignment.template.GapPenalty,%20org.biojava.nbio.alignment.template.SubstitutionMatrix)">SimpleProfileProfileAligner(Profile&lt;S, C&gt;, Profile&lt;S, C&gt;, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfileProfileAligner.html" title="class in org.biojava.nbio.alignment">SimpleProfileProfileAligner</a></dt>
<dd>
<div class="block">Prepares for a profile-profile alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfileProfileAligner.html#SimpleProfileProfileAligner(java.util.concurrent.Future,%20java.util.concurrent.Future,%20org.biojava.nbio.alignment.template.GapPenalty,%20org.biojava.nbio.alignment.template.SubstitutionMatrix)">SimpleProfileProfileAligner(Future&lt;ProfilePair&lt;S, C&gt;&gt;, Future&lt;ProfilePair&lt;S, C&gt;&gt;, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfileProfileAligner.html" title="class in org.biojava.nbio.alignment">SimpleProfileProfileAligner</a></dt>
<dd>
<div class="block">Prepares for a profile-profile alignment run concurrently.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfileProfileAligner.html#SimpleProfileProfileAligner(org.biojava.nbio.alignment.template.Profile,%20java.util.concurrent.Future,%20org.biojava.nbio.alignment.template.GapPenalty,%20org.biojava.nbio.alignment.template.SubstitutionMatrix)">SimpleProfileProfileAligner(Profile&lt;S, C&gt;, Future&lt;ProfilePair&lt;S, C&gt;&gt;, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfileProfileAligner.html" title="class in org.biojava.nbio.alignment">SimpleProfileProfileAligner</a></dt>
<dd>
<div class="block">Prepares for a profile-profile alignment run concurrently.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfileProfileAligner.html#SimpleProfileProfileAligner(java.util.concurrent.Future,%20org.biojava.nbio.alignment.template.Profile,%20org.biojava.nbio.alignment.template.GapPenalty,%20org.biojava.nbio.alignment.template.SubstitutionMatrix)">SimpleProfileProfileAligner(Future&lt;ProfilePair&lt;S, C&gt;&gt;, Profile&lt;S, C&gt;, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfileProfileAligner.html" title="class in org.biojava.nbio.alignment">SimpleProfileProfileAligner</a></dt>
<dd>
<div class="block">Prepares for a profile-profile alignment run concurrently.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.alignment"><span class="strong">SimpleSequencePair</span></a>&lt;<a href="./org/biojava/nbio/alignment/SimpleSequencePair.html" title="type parameter in SimpleSequencePair">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/SimpleSequencePair.html" title="type parameter in SimpleSequencePair">C</a>&gt;,<a href="./org/biojava/nbio/alignment/SimpleSequencePair.html" title="type parameter in SimpleSequencePair">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Class in <a href="./org/biojava/nbio/alignment/package-summary.html">org.biojava.nbio.alignment</a></dt>
<dd>
<div class="block">Implements a data structure for the results of pairwise sequence alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSequencePair.html#SimpleSequencePair(org.biojava.nbio.alignment.template.AlignedSequence,%20org.biojava.nbio.alignment.template.AlignedSequence)">SimpleSequencePair(AlignedSequence&lt;S, C&gt;, AlignedSequence&lt;S, C&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.alignment">SimpleSequencePair</a></dt>
<dd>
<div class="block">Creates a pair profile for the given already aligned sequences.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSequencePair.html#SimpleSequencePair(S,%20S,%20java.util.List,%20java.util.List)">SimpleSequencePair(S, S, List&lt;AlignedSequence.Step&gt;, List&lt;AlignedSequence.Step&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.alignment">SimpleSequencePair</a></dt>
<dd>
<div class="block">Creates a pair profile for the given sequences with a global alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSequencePair.html#SimpleSequencePair(S,%20S,%20java.util.List,%20int,%20int,%20java.util.List,%20int,%20int)">SimpleSequencePair(S, S, List&lt;AlignedSequence.Step&gt;, int, int, List&lt;AlignedSequence.Step&gt;, int, int)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.alignment">SimpleSequencePair</a></dt>
<dd>
<div class="block">Creates a pair profile for the given sequences with a local alignment.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment"><span class="strong">SimpleSubstitutionMatrix</span></a>&lt;<a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html" title="type parameter in SimpleSubstitutionMatrix">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Class in <a href="./org/biojava/nbio/alignment/package-summary.html">org.biojava.nbio.alignment</a></dt>
<dd>
<div class="block">Implements a data structure which holds the score (penalty or bonus) given during alignment for the exchange of one
 <code>Compound</code> in a sequence for another.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html#SimpleSubstitutionMatrix(org.biojava.nbio.core.sequence.template.CompoundSet,%20java.io.File)">SimpleSubstitutionMatrix(CompoundSet&lt;C&gt;, File)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>
<div class="block">Creates a substitution matrix by reading in a file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html#SimpleSubstitutionMatrix(org.biojava.nbio.core.sequence.template.CompoundSet,%20java.io.Reader,%20java.lang.String)">SimpleSubstitutionMatrix(CompoundSet&lt;C&gt;, Reader, String)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>
<div class="block">Creates a substitution matrix by parsing some input.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html#SimpleSubstitutionMatrix(org.biojava.nbio.core.sequence.template.CompoundSet,%20java.lang.String,%20java.lang.String)">SimpleSubstitutionMatrix(CompoundSet&lt;C&gt;, String, String)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>
<div class="block">Creates a substitution matrix by parsing a String.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html#SimpleSubstitutionMatrix(org.biojava.nbio.core.sequence.template.CompoundSet,%20short,%20short)">SimpleSubstitutionMatrix(CompoundSet&lt;C&gt;, short, short)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>
<div class="block">Creates an identity substitution matrix from match and replace values.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html#SMART">SMART</a></span> - Static variable in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure.DatabaseReference</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/SmithWaterman.html" title="class in org.biojava.nbio.alignment"><span class="strong">SmithWaterman</span></a>&lt;<a href="./org/biojava/nbio/alignment/SmithWaterman.html" title="type parameter in SmithWaterman">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/SmithWaterman.html" title="type parameter in SmithWaterman">C</a>&gt;,<a href="./org/biojava/nbio/alignment/SmithWaterman.html" title="type parameter in SmithWaterman">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Class in <a href="./org/biojava/nbio/alignment/package-summary.html">org.biojava.nbio.alignment</a></dt>
<dd>
<div class="block">Smith and Waterman defined an algorithm for pairwise local sequence alignments (best match of sections from each
 <code>Sequence</code>).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SmithWaterman.html#SmithWaterman()">SmithWaterman()</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SmithWaterman.html" title="class in org.biojava.nbio.alignment">SmithWaterman</a></dt>
<dd>
<div class="block">Before running a pairwise local sequence alignment, data must be sent in via calls to
 <a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#setQuery(S)"><code>AbstractPairwiseSequenceAligner.setQuery(Sequence)</code></a>, <a href="./org/biojava/nbio/alignment/template/AbstractPairwiseSequenceAligner.html#setTarget(S)"><code>AbstractPairwiseSequenceAligner.setTarget(Sequence)</code></a>, <a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#setGapPenalty(org.biojava.nbio.alignment.template.GapPenalty)"><code>setGapPenalty(GapPenalty)</code></a>, and
 <a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#setSubstitutionMatrix(org.biojava.nbio.alignment.template.SubstitutionMatrix)"><code>setSubstitutionMatrix(SubstitutionMatrix)</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SmithWaterman.html#SmithWaterman(S,%20S,%20org.biojava.nbio.alignment.template.GapPenalty,%20org.biojava.nbio.alignment.template.SubstitutionMatrix)">SmithWaterman(S, S, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SmithWaterman.html" title="class in org.biojava.nbio.alignment">SmithWaterman</a></dt>
<dd>
<div class="block">Prepares for a pairwise local sequence alignment.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/StandardRescoreRefiner.html" title="class in org.biojava.nbio.alignment"><span class="strong">StandardRescoreRefiner</span></a>&lt;<a href="./org/biojava/nbio/alignment/StandardRescoreRefiner.html" title="type parameter in StandardRescoreRefiner">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/StandardRescoreRefiner.html" title="type parameter in StandardRescoreRefiner">C</a>&gt;,<a href="./org/biojava/nbio/alignment/StandardRescoreRefiner.html" title="type parameter in StandardRescoreRefiner">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Class in <a href="./org/biojava/nbio/alignment/package-summary.html">org.biojava.nbio.alignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/StandardRescoreRefiner.html#StandardRescoreRefiner(org.biojava.nbio.alignment.Alignments.PairInProfileScorerType,%20org.biojava.nbio.alignment.Alignments.ProfileProfileAlignerType)">StandardRescoreRefiner(Alignments.PairInProfileScorerType, Alignments.ProfileProfileAlignerType)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/StandardRescoreRefiner.html" title="class in org.biojava.nbio.alignment">StandardRescoreRefiner</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io"><span class="strong">StockholmFileAnnotation</span></a> - Class in <a href="./org/biojava/nbio/alignment/io/package-summary.html">org.biojava.nbio.alignment.io</a></dt>
<dd>
<div class="block">Stores all the content parsed from the #=GF lines</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#StockholmFileAnnotation()">StockholmFileAnnotation()</a></span> - Constructor for class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.StockholmFileAnnotationReference.html" title="class in org.biojava.nbio.alignment.io"><span class="strong">StockholmFileAnnotation.StockholmFileAnnotationReference</span></a> - Class in <a href="./org/biojava/nbio/alignment/io/package-summary.html">org.biojava.nbio.alignment.io</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.StockholmFileAnnotationReference.html#StockholmFileAnnotation.StockholmFileAnnotationReference()">StockholmFileAnnotation.StockholmFileAnnotationReference()</a></span> - Constructor for class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.StockholmFileAnnotationReference.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation.StockholmFileAnnotationReference</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/io/StockholmFileParser.html" title="class in org.biojava.nbio.alignment.io"><span class="strong">StockholmFileParser</span></a> - Class in <a href="./org/biojava/nbio/alignment/io/package-summary.html">org.biojava.nbio.alignment.io</a></dt>
<dd>
<div class="block">Stockholm file parser.<br>
 for more information about the format refer to
 
 <a
 href="./org/biojava/nbio/alignment/io/ftp://ftp.sanger.ac.uk/pub/databases/Pfam/current_release/userman.txt">ftp://ftp.sanger.ac.uk/pub/databases
 /Pfam/current_release/userman.txt</a>.
 <a
 href="./org/biojava/nbio/alignment/io/ftp://ftp.sanger.ac.uk/pub/databases/Rfam/CURRENT/USERMAN">ftp://ftp.sanger.ac.uk/pub/databases/Rfam/CURRENT
 /USERMAN</a>.
 <a href="http://sonnhammer.sbc.su.se/Stockholm.html">http://sonnhammer.sbc.su.se/Stockholm.html</a>.
 </div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileParser.html#StockholmFileParser()">StockholmFileParser()</a></span> - Constructor for class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileParser.html" title="class in org.biojava.nbio.alignment.io">StockholmFileParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io"><span class="strong">StockholmStructure</span></a> - Class in <a href="./org/biojava/nbio/alignment/io/package-summary.html">org.biojava.nbio.alignment.io</a></dt>
<dd>
<div class="block">Stores all the content of a Stockholm file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#StockholmStructure()">StockholmStructure()</a></span> - Constructor for class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html" title="class in org.biojava.nbio.alignment.io"><span class="strong">StockholmStructure.DatabaseReference</span></a> - Class in <a href="./org/biojava/nbio/alignment/io/package-summary.html">org.biojava.nbio.alignment.io</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html#StockholmStructure.DatabaseReference(java.lang.String,%20java.lang.String)">StockholmStructure.DatabaseReference(String, String)</a></span> - Constructor for class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure.DatabaseReference</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html#StockholmStructure.DatabaseReference(java.lang.String)">StockholmStructure.DatabaseReference(String)</a></span> - Constructor for class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure.DatabaseReference</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html#stringToFloats(java.lang.String)">stringToFloats(String)</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmFileAnnotation.html" title="class in org.biojava.nbio.alignment.io">StockholmFileAnnotation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.alignment.template"><span class="strong">SubstitutionMatrix</span></a>&lt;<a href="./org/biojava/nbio/alignment/template/SubstitutionMatrix.html" title="type parameter in SubstitutionMatrix">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Interface in <a href="./org/biojava/nbio/alignment/template/package-summary.html">org.biojava.nbio.alignment.template</a></dt>
<dd>
<div class="block">Defines a data structure which holds the score (penalty or bonus) given during alignment for the exchange of one
 <code>Compound</code> in a sequence for another.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.alignment"><span class="strong">SubstitutionMatrixHelper</span></a> - Class in <a href="./org/biojava/nbio/alignment/package-summary.html">org.biojava.nbio.alignment</a></dt>
<dd>
<div class="block">Static utility to access substitution matrices that come bundled with BioJava.</div>
</dd>
<dt><a href="./org/biojava/nbio/alignment/SubstitutionMatrixScorer.html" title="class in org.biojava.nbio.alignment"><span class="strong">SubstitutionMatrixScorer</span></a>&lt;<a href="./org/biojava/nbio/alignment/SubstitutionMatrixScorer.html" title="type parameter in SubstitutionMatrixScorer">S</a> extends org.biojava.nbio.core.sequence.template.Sequence&lt;<a href="./org/biojava/nbio/alignment/SubstitutionMatrixScorer.html" title="type parameter in SubstitutionMatrixScorer">C</a>&gt;,<a href="./org/biojava/nbio/alignment/SubstitutionMatrixScorer.html" title="type parameter in SubstitutionMatrixScorer">C</a> extends org.biojava.nbio.core.sequence.template.Compound&gt; - Class in <a href="./org/biojava/nbio/alignment/package-summary.html">org.biojava.nbio.alignment</a></dt>
<dd>
<div class="block">Scores using a substitution matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SubstitutionMatrixScorer.html#SubstitutionMatrixScorer(org.biojava.nbio.alignment.template.SequencePair,%20org.biojava.nbio.alignment.template.SubstitutionMatrix)">SubstitutionMatrixScorer(SequencePair&lt;S, C&gt;, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SubstitutionMatrixScorer.html" title="class in org.biojava.nbio.alignment">SubstitutionMatrixScorer</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_T_">
<!--   -->
</a>
<h2 class="title">T</h2>
<dl>
<dt><a href="./demo/TestDNANeedlemanWunsch.html" title="class in demo"><span class="strong">TestDNANeedlemanWunsch</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/TestDNANeedlemanWunsch.html#TestDNANeedlemanWunsch()">TestDNANeedlemanWunsch()</a></span> - Constructor for class demo.<a href="./demo/TestDNANeedlemanWunsch.html" title="class in demo">TestDNANeedlemanWunsch</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#time">time</a></span> - Variable in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.alignment.aaindex.<a href="./org/biojava/nbio/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns in a format similar to the standard NCBI files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/GuideTree.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/GuideTree.html" title="class in org.biojava.nbio.alignment">GuideTree</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure.DatabaseReference</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.alignment">SimpleAlignedSequence</a></dt>
<dd>
<div class="block">Provides standard Java language access to results of <a href="./org/biojava/nbio/alignment/SimpleAlignedSequence.html#getSequenceAsString()"><code>SimpleAlignedSequence.getSequenceAsString()</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#toString(int)">toString(int)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#toString(org.biojava.nbio.alignment.template.Profile.StringFormat)">toString(Profile.StringFormat)</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleProfile.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleProfile.html" title="class in org.biojava.nbio.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns in a format similar to the standard NCBI files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.html#toString()">toString()</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns a simple view of the alignment profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.html#toString(int)">toString(int)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns a formatted view of the alignment profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.html#toString(org.biojava.nbio.alignment.template.Profile.StringFormat)">toString(Profile.StringFormat)</a></span> - Method in interface org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.html" title="interface in org.biojava.nbio.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns a formatted view of the alignment profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html#tracebackToString(org.biojava.nbio.alignment.routines.AlignerHelper.Last[][][])">tracebackToString(AlignerHelper.Last[][][])</a></span> - Static method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_U_">
<!--   -->
</a>
<h2 class="title">U</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Cut.html#update(int,%20org.biojava.nbio.alignment.routines.AlignerHelper.Subproblem,%20org.biojava.nbio.alignment.routines.AlignerHelper.Last[][])">update(int, AlignerHelper.Subproblem, AlignerHelper.Last[][])</a></span> - Method in class org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Cut.html" title="class in org.biojava.nbio.alignment.routines">AlignerHelper.Cut</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html#URL">URL</a></span> - Static variable in class org.biojava.nbio.alignment.io.<a href="./org/biojava/nbio/alignment/io/StockholmStructure.DatabaseReference.html" title="class in org.biojava.nbio.alignment.io">StockholmStructure.DatabaseReference</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_V_">
<!--   -->
</a>
<h2 class="title">V</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/Alignments.PairInProfileScorerType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/Alignments.PairInProfileScorerType.html" title="enum in org.biojava.nbio.alignment">Alignments.PairInProfileScorerType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/Alignments.PairwiseSequenceAlignerType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/Alignments.PairwiseSequenceAlignerType.html" title="enum in org.biojava.nbio.alignment">Alignments.PairwiseSequenceAlignerType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/Alignments.PairwiseSequenceScorerType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/Alignments.PairwiseSequenceScorerType.html" title="enum in org.biojava.nbio.alignment">Alignments.PairwiseSequenceScorerType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/Alignments.ProfileProfileAlignerType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/Alignments.ProfileProfileAlignerType.html" title="enum in org.biojava.nbio.alignment">Alignments.ProfileProfileAlignerType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/Alignments.RefinerType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/Alignments.RefinerType.html" title="enum in org.biojava.nbio.alignment">Alignments.RefinerType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Last.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Last.html" title="enum in org.biojava.nbio.alignment.routines">AlignerHelper.Last</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AlignedSequence.Step.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AlignedSequence.Step.html" title="enum in org.biojava.nbio.alignment.template">AlignedSequence.Step</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/GapPenalty.Type.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/GapPenalty.Type.html" title="enum in org.biojava.nbio.alignment.template">GapPenalty.Type</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.StringFormat.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.StringFormat.html" title="enum in org.biojava.nbio.alignment.template">Profile.StringFormat</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/Alignments.PairInProfileScorerType.html#values()">values()</a></span> - Static method in enum org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/Alignments.PairInProfileScorerType.html" title="enum in org.biojava.nbio.alignment">Alignments.PairInProfileScorerType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/Alignments.PairwiseSequenceAlignerType.html#values()">values()</a></span> - Static method in enum org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/Alignments.PairwiseSequenceAlignerType.html" title="enum in org.biojava.nbio.alignment">Alignments.PairwiseSequenceAlignerType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/Alignments.PairwiseSequenceScorerType.html#values()">values()</a></span> - Static method in enum org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/Alignments.PairwiseSequenceScorerType.html" title="enum in org.biojava.nbio.alignment">Alignments.PairwiseSequenceScorerType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/Alignments.ProfileProfileAlignerType.html#values()">values()</a></span> - Static method in enum org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/Alignments.ProfileProfileAlignerType.html" title="enum in org.biojava.nbio.alignment">Alignments.ProfileProfileAlignerType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/Alignments.RefinerType.html#values()">values()</a></span> - Static method in enum org.biojava.nbio.alignment.<a href="./org/biojava/nbio/alignment/Alignments.RefinerType.html" title="enum in org.biojava.nbio.alignment">Alignments.RefinerType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Last.html#values()">values()</a></span> - Static method in enum org.biojava.nbio.alignment.routines.<a href="./org/biojava/nbio/alignment/routines/AlignerHelper.Last.html" title="enum in org.biojava.nbio.alignment.routines">AlignerHelper.Last</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AlignedSequence.Step.html#values()">values()</a></span> - Static method in enum org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AlignedSequence.Step.html" title="enum in org.biojava.nbio.alignment.template">AlignedSequence.Step</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/GapPenalty.Type.html#values()">values()</a></span> - Static method in enum org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/GapPenalty.Type.html" title="enum in org.biojava.nbio.alignment.template">GapPenalty.Type</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/Profile.StringFormat.html#values()">values()</a></span> - Static method in enum org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/Profile.StringFormat.html" title="enum in org.biojava.nbio.alignment.template">Profile.StringFormat</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
</dl>
<a name="_X_">
<!--   -->
</a>
<h2 class="title">X</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#xyMax">xyMax</a></span> - Variable in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">End position of the aligned sequence in the query and target respectively</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html#xyStart">xyStart</a></span> - Variable in class org.biojava.nbio.alignment.template.<a href="./org/biojava/nbio/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava.nbio.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Start position of the aligned sequence in the query and target respectively</div>
</dd>
</dl>
<a href="#_A_">A</a>&nbsp;<a href="#_C_">C</a>&nbsp;<a href="#_D_">D</a>&nbsp;<a href="#_E_">E</a>&nbsp;<a href="#_F_">F</a>&nbsp;<a href="#_G_">G</a>&nbsp;<a href="#_H_">H</a>&nbsp;<a href="#_I_">I</a>&nbsp;<a href="#_L_">L</a>&nbsp;<a href="#_M_">M</a>&nbsp;<a href="#_N_">N</a>&nbsp;<a href="#_O_">O</a>&nbsp;<a href="#_P_">P</a>&nbsp;<a href="#_R_">R</a>&nbsp;<a href="#_S_">S</a>&nbsp;<a href="#_T_">T</a>&nbsp;<a href="#_U_">U</a>&nbsp;<a href="#_V_">V</a>&nbsp;<a href="#_X_">X</a>&nbsp;</div>
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar_bottom">
<!--   -->
</a><a href="#skip-navbar_bottom" title="Skip navigation links"></a><a name="navbar_bottom_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="./overview-summary.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li>Use</li>
<li><a href="./overview-tree.html">Tree</a></li>
<li><a href="./deprecated-list.html">Deprecated</a></li>
<li class="navBarCell1Rev">Index</li>
<li><a href="./help-doc.html">Help</a></li>
</ul>
<div class="aboutLanguage"><em><script src="http://www.google-analytics.com/urchin.js" 							type="text/javascript"></script><script 							type="text/javascript">_uacct = 							"UA-1326640-1";urchinTracker();</script></em></div>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="./index.html?index-all.html" target="_top">Frames</a></li>
<li><a href="index-all.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="./allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
<p class="legalCopy"><small>Copyright &#169; 2000-2015 <a href="http://www.biojava.org">BioJava</a>. All Rights Reserved.</small></p>
</body>
</html>
