<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_71) on Fri Jan 30 08:16:45 PST 2015 -->
<meta http-equiv="Content-Type" content="text/html" charset="UTF-8">
<title>Uses of Interface org.biojava.nbio.core.sequence.template.Compound (biojava-core 4.0.0 API)</title>
<meta name="date" content="2015-01-30">
<link rel="stylesheet" type="text/css" href="../../../../../../../stylesheet.css" title="Style">
</head>
<body>
<script type="text/javascript"><!--
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="Uses of Interface org.biojava.nbio.core.sequence.template.Compound (biojava-core 4.0.0 API)";
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar_top">
<!--   -->
</a><a href="#skip-navbar_top" title="Skip navigation links"></a><a name="navbar_top_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../../overview-summary.html">Overview</a></li>
<li><a href="../package-summary.html">Package</a></li>
<li><a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Class</a></li>
<li class="navBarCell1Rev">Use</li>
<li><a href="../package-tree.html">Tree</a></li>
<li><a href="../../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../../index-all.html">Index</a></li>
<li><a href="../../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="../../../../../../../index.html?org/biojava/nbio/core/sequence/template/class-use/Compound.html" target="_top">Frames</a></li>
<li><a href="Compound.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="../../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<div class="header">
<h2 title="Uses of Interface org.biojava.nbio.core.sequence.template.Compound" class="title">Uses of Interface<br>org.biojava.nbio.core.sequence.template.Compound</h2>
</div>
<div class="classUseContainer">
<ul class="blockList">
<li class="blockList">
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing packages, and an explanation">
<caption><span>Packages that use <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Package</th>
<th class="colLast" scope="col">Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava.nbio.core.sequence">org.biojava.nbio.core.sequence</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.biojava.nbio.core.sequence.compound">org.biojava.nbio.core.sequence.compound</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava.nbio.core.sequence.edits">org.biojava.nbio.core.sequence.edits</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.biojava.nbio.core.sequence.features">org.biojava.nbio.core.sequence.features</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava.nbio.core.sequence.io">org.biojava.nbio.core.sequence.io</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.biojava.nbio.core.sequence.io.template">org.biojava.nbio.core.sequence.io.template</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava.nbio.core.sequence.io.util">org.biojava.nbio.core.sequence.io.util</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.biojava.nbio.core.sequence.loader">org.biojava.nbio.core.sequence.loader</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava.nbio.core.sequence.location">org.biojava.nbio.core.sequence.location</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.biojava.nbio.core.sequence.location.template">org.biojava.nbio.core.sequence.location.template</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava.nbio.core.sequence.storage">org.biojava.nbio.core.sequence.storage</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.biojava.nbio.core.sequence.template">org.biojava.nbio.core.sequence.template</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava.nbio.core.sequence.transcription">org.biojava.nbio.core.sequence.transcription</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.biojava.nbio.core.sequence.views">org.biojava.nbio.core.sequence.views</a></td>
<td class="colLast">&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList">
<ul class="blockList">
<li class="blockList"><a name="org.biojava.nbio.core.sequence">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a> in <a href="../../../../../../../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../../../../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a> with type parameters of type <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/BasicSequence.html" title="class in org.biojava.nbio.core.sequence">BasicSequence</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">Bare bones version of the Sequence object to be used sparingly.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava.nbio.core.sequence">MultipleSequenceAlignment</a>&lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">Implements a minimal data structure for reading and writing a sequence alignment.</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava.nbio.core.sequence.compound">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a> in <a href="../../../../../../../org/biojava/nbio/core/sequence/compound/package-summary.html">org.biojava.nbio.core.sequence.compound</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../../../../org/biojava/nbio/core/sequence/compound/package-summary.html">org.biojava.nbio.core.sequence.compound</a> that implement <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompound</a></strong></code>
<div class="block">Used to describe an Amino Acid.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></strong></code>
<div class="block">Define a codon</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a></strong></code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../../org/biojava/nbio/core/sequence/compound/package-summary.html">org.biojava.nbio.core.sequence.compound</a> with parameters of type <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>boolean</code></td>
<td class="colLast"><span class="strong">CodonCompound.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/compound/CodonCompound.html#equalsIgnoreCase(org.biojava.nbio.core.sequence.template.Compound)">equalsIgnoreCase</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&nbsp;compound)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>boolean</code></td>
<td class="colLast"><span class="strong">AminoAcidCompound.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#equalsIgnoreCase(org.biojava.nbio.core.sequence.template.Compound)">equalsIgnoreCase</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&nbsp;compound)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>boolean</code></td>
<td class="colLast"><span class="strong">NucleotideCompound.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#equalsIgnoreCase(org.biojava.nbio.core.sequence.template.Compound)">equalsIgnoreCase</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&nbsp;compound)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava.nbio.core.sequence.edits">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a> in <a href="../../../../../../../org/biojava/nbio/core/sequence/edits/package-summary.html">org.biojava.nbio.core.sequence.edits</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../../../../org/biojava/nbio/core/sequence/edits/package-summary.html">org.biojava.nbio.core.sequence.edits</a> with type parameters of type <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Interface and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>interface&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/edits/Edit.html" title="interface in org.biojava.nbio.core.sequence.edits">Edit</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">Interface for carrying out edit operations on a Sequence.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>static class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.AbstractEdit</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">Abstract class which defines all edit operations as a call to discover
 what 5' and 3' ends of an editing Sequence should be joined together
 with a target Sequence.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/edits/Edit.Delete.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Delete</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">Implementation which allows for the deletion of bases from a Sequence</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>static class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/edits/Edit.Insert.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Insert</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">Edit implementation which allows us to insert a base at any position
 in a Sequence.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/edits/Edit.Substitute.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Substitute</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">Allows for the substitution of bases into an existing Sequence.</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava.nbio.core.sequence.features">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a> in <a href="../../../../../../../org/biojava/nbio/core/sequence/features/package-summary.html">org.biojava.nbio.core.sequence.features</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../../../../org/biojava/nbio/core/sequence/features/package-summary.html">org.biojava.nbio.core.sequence.features</a> with type parameters of type <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a>&lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;C&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">A feature is currently any descriptive item that can be associated with a sequence position(s)
 A feature has a type and a source which is currently a string to allow flexibility for the user
 Ideally well defined features should have a class to describe attributes of that feature</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a>&lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;C&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">It is <a href="../../../../../../../org/biojava/nbio/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features"><code>DBReferenceInfo</code></a> which implements <a href="../../../../../../../org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features"><code>FeatureInterface</code></a>.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>interface&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a>&lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;C&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">Interface class to handle describing arbitrary features.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/features/QualityFeature.html" title="class in org.biojava.nbio.core.sequence.features">QualityFeature</a>&lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;C&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">DNA Sequences produced by modern sequencers usually have quality informaion
 attached to them.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/features/QuantityFeature.html" title="class in org.biojava.nbio.core.sequence.features">QuantityFeature</a>&lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;C&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">It is common to have a numerical value or values associated with a feature.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/features/TextFeature.html" title="class in org.biojava.nbio.core.sequence.features">TextFeature</a>&lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;C&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">A implmentation of AbstractFeature</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava.nbio.core.sequence.io">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a> in <a href="../../../../../../../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../../../../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a> with type parameters of type <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/io/FastaReader.html" title="class in org.biojava.nbio.core.sequence.io">FastaReader</a>&lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;?&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">Use FastaReaderHelper as an example of how to use this class where FastaReaderHelper should be the
 primary class used to read Fasta files</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/io/FastaWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriter</a>&lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;?&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">The FastaWriter writes a collection of sequences to an outputStream.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/io/GenbankReader.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReader</a>&lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;C&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">Use GenbankReaderHelper as an example of how to use this class where GenbankReaderHelper should be the
 primary class used to read Genbank files</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a>&lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;C&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/io/GenbankWriter.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriter</a>&lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;?&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io">GenericFastaHeaderFormat</a>&lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;?&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">We store the original header if the sequence is parsed from a fasta file and will use that exact
 sequence if we write out the sequences to a fasta file.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericFastaHeaderParser</a>&lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;C&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">The default fasta header parser where some headers are well defined based on the source
 database which allows us to set the source of the protein sequence and the identifier
 that can be used in future implementations to load features from external sources
 
 If the user has a custom header with local data then they can create their own implementation
 of a FastaHeaderParserInterface</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderFormat</a>&lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;C&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderParser</a>&lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;C&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io">GenericInsdcHeaderFormat</a>&lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;C&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">PlainFastaHeaderParser</a>&lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;C&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">The plain fasta header takes everything in the header as a single entity.</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava.nbio.core.sequence.io.template">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a> in <a href="../../../../../../../org/biojava/nbio/core/sequence/io/template/package-summary.html">org.biojava.nbio.core.sequence.io.template</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../../../../org/biojava/nbio/core/sequence/io/template/package-summary.html">org.biojava.nbio.core.sequence.io.template</a> with type parameters of type <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Interface and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>interface&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/io/template/FastaHeaderFormatInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template">FastaHeaderFormatInterface</a>&lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;?&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>interface&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template">GenbankHeaderFormatInterface</a>&lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;?&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>interface&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template">SequenceCreatorInterface</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>interface&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/io/template/SequenceHeaderParserInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template">SequenceHeaderParserInterface</a>&lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;?&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava.nbio.core.sequence.io.util">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a> in <a href="../../../../../../../org/biojava/nbio/core/sequence/io/util/package-summary.html">org.biojava.nbio.core.sequence.io.util</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../../org/biojava/nbio/core/sequence/io/util/package-summary.html">org.biojava.nbio.core.sequence.io.util</a> with type parameters of type <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>static &lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br>int</code></td>
<td class="colLast"><span class="strong">IOUtils.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/io/util/IOUtils.html#getGCGChecksum(java.util.List)">getGCGChecksum</a></strong>(<a href="http://java.sun.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;S&gt;&nbsp;sequences)</code>
<div class="block">Calculates GCG checksum for entire list of sequences</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>static &lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br>int</code></td>
<td class="colLast"><span class="strong">IOUtils.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/io/util/IOUtils.html#getGCGChecksum(S)">getGCGChecksum</a></strong>(S&nbsp;sequence)</code>
<div class="block">Calculates GCG checksum for a given sequence</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static &lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br><a href="http://java.sun.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a></code></td>
<td class="colLast"><span class="strong">IOUtils.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/io/util/IOUtils.html#getGCGHeader(java.util.List)">getGCGHeader</a></strong>(<a href="http://java.sun.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;S&gt;&nbsp;sequences)</code>
<div class="block">Assembles a GCG file header</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>static &lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br><a href="http://java.sun.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a></code></td>
<td class="colLast"><span class="strong">IOUtils.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/io/util/IOUtils.html#getGCGType(org.biojava.nbio.core.sequence.template.CompoundSet)">getGCGType</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundSet</a>&lt;C&gt;&nbsp;cs)</code>
<div class="block">Determines GCG type</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static &lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br><a href="http://java.sun.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a></code></td>
<td class="colLast"><span class="strong">IOUtils.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/io/util/IOUtils.html#getIDFormat(java.util.List)">getIDFormat</a></strong>(<a href="http://java.sun.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;S&gt;&nbsp;sequences)</code>
<div class="block">Creates format String for accession IDs</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava.nbio.core.sequence.loader">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a> in <a href="../../../../../../../org/biojava/nbio/core/sequence/loader/package-summary.html">org.biojava.nbio.core.sequence.loader</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../../../../org/biojava/nbio/core/sequence/loader/package-summary.html">org.biojava.nbio.core.sequence.loader</a> with type parameters of type <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/loader/ArrayListProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">ArrayListProxySequenceReader</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">This class represents the storage container of a sequence stored in a fasta file where
 the initial parsing of the file we store the offset and length of the sequence.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">An example of a ProxySequenceReader that is created from a String.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">Pass in a Uniprot ID and this ProxySequenceReader when passed to a ProteinSequence will get the sequence data and other data elements
 associated with the ProteinSequence by Uniprot.</div>
</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../../org/biojava/nbio/core/sequence/loader/package-summary.html">org.biojava.nbio.core.sequence.loader</a> with type parameters of type <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>static &lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br><a href="../../../../../../../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a>&lt;C&gt;</code></td>
<td class="colLast"><span class="strong">UniprotProxySequenceReader.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#parseUniprotXMLString(java.lang.String,%20org.biojava.nbio.core.sequence.template.CompoundSet)">parseUniprotXMLString</a></strong>(<a href="http://java.sun.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;xml,
                     <a href="../../../../../../../org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundSet</a>&lt;C&gt;&nbsp;compoundSet)</code>
<div class="block">The passed in xml is parsed as a DOM object so we know everything about the protein.</div>
</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../../org/biojava/nbio/core/sequence/loader/package-summary.html">org.biojava.nbio.core.sequence.loader</a> with parameters of type <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>int</code></td>
<td class="colLast"><span class="strong">UniprotProxySequenceReader.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#countCompounds(C...)">countCompounds</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="type parameter in UniprotProxySequenceReader">C</a>...&nbsp;compounds)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>int</code></td>
<td class="colLast"><span class="strong">StringProxySequenceReader.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#countCompounds(C...)">countCompounds</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="type parameter in StringProxySequenceReader">C</a>...&nbsp;compounds)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>int</code></td>
<td class="colLast"><span class="strong">SequenceFileProxyLoader.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#countCompounds(C...)">countCompounds</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="type parameter in SequenceFileProxyLoader">C</a>...&nbsp;compounds)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava.nbio.core.sequence.location">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a> in <a href="../../../../../../../org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../../../../org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a> with type parameters of type <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/location/InsdcParser.html" title="class in org.biojava.nbio.core.sequence.location">InsdcParser</a>&lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;C&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">Parser for working with INSDC style locations.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/location/SequenceLocation.html" title="class in org.biojava.nbio.core.sequence.location">SequenceLocation</a>&lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;C&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">A location in a sequence that keeps a reference to its parent sequence</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava.nbio.core.sequence.location.template">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a> in <a href="../../../../../../../org/biojava/nbio/core/sequence/location/template/package-summary.html">org.biojava.nbio.core.sequence.location.template</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../../org/biojava/nbio/core/sequence/location/template/package-summary.html">org.biojava.nbio.core.sequence.location.template</a> with type parameters of type <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>protected &lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br>boolean</code></td>
<td class="colLast"><span class="strong">AbstractLocation.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#canComplement(org.biojava.nbio.core.sequence.template.Sequence)">canComplement</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;sequence)</code>
<div class="block">Uses the Sequence's CompoundSet to decide if a compound can
 be assgined to ComplementCompound meaning it can complement</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br><a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;</code></td>
<td class="colLast"><span class="strong">AbstractLocation.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#getRelevantSubSequence(org.biojava.nbio.core.sequence.template.Sequence)">getRelevantSubSequence</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;sequence)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br><a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;</code></td>
<td class="colLast"><span class="strong">Location.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/location/template/Location.html#getRelevantSubSequence(org.biojava.nbio.core.sequence.template.Sequence)">getRelevantSubSequence</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;sequence)</code>
<div class="block">Will return a SequenceReader object which offers a view of all resolved
 locations i.e.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br><a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;</code></td>
<td class="colLast"><span class="strong">AbstractLocation.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#getSubSequence(org.biojava.nbio.core.sequence.template.Sequence)">getSubSequence</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;sequence)</code>
<div class="block">If circular this will return the sequence represented by the sub
 locations joined.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br><a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;</code></td>
<td class="colLast"><span class="strong">Location.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/location/template/Location.html#getSubSequence(org.biojava.nbio.core.sequence.template.Sequence)">getSubSequence</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;sequence)</code>
<div class="block">Will return a SequenceReader object which represents the outer bounds
 of this Location</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>protected &lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br><a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;</code></td>
<td class="colLast"><span class="strong">AbstractLocation.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#reverseSequence(org.biojava.nbio.core.sequence.template.Sequence)">reverseSequence</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;sequence)</code>
<div class="block">Reverses and (if possible) complements the Sequence so as to represent
 the reverse strand (if one exists).</div>
</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../../org/biojava/nbio/core/sequence/location/template/package-summary.html">org.biojava.nbio.core.sequence.location.template</a> that return types with arguments of type <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../../org/biojava/nbio/core/sequence/template/ProxySequenceReader.html" title="interface in org.biojava.nbio.core.sequence.template">ProxySequenceReader</a>&lt;? extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</code></td>
<td class="colLast"><span class="strong">AccesionedLocation.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/location/template/AccesionedLocation.html#getProxySequenceReader()">getProxySequenceReader</a></strong>()</code>
<div class="block">Return the proxy reader used to get sequence for this location.</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava.nbio.core.sequence.storage">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a> in <a href="../../../../../../../org/biojava/nbio/core/sequence/storage/package-summary.html">org.biojava.nbio.core.sequence.storage</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../../../../org/biojava/nbio/core/sequence/storage/package-summary.html">org.biojava.nbio.core.sequence.storage</a> with type parameters of type <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">Stores a Sequence as a collection of compounds in an ArrayList</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">An implementation of the popular bit encodings.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">The logic of working with a bit has been separated out into this class
 to help developers create the bit data structures without having to
 put the code into an intermediate format and to also use the format
 without the need to copy this code.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">Four bit encoding of the bit formats.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader.FourBitArrayWorker</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">A four bit per compound implementation of the bit array worker code.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">This reader actually proxies onto multiple types of sequence in order
 to allow a number of sequence objects to act as if they are one sequence.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html" title="class in org.biojava.nbio.core.sequence.storage">SequenceAsStringHelper</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">This is a common method that can be used across multiple storage/proxy implementations to
 handle Negative strand and other interesting elements of sequence data.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">An implementation of the SequenceReader interface which for every
 call will return only 1 compound (given to it during construction; a String
 is also valid but will require a CompoundSet).</div>
</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../../org/biojava/nbio/core/sequence/storage/package-summary.html">org.biojava.nbio.core.sequence.storage</a> with parameters of type <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>int</code></td>
<td class="colLast"><span class="strong">SingleCompoundSequenceReader.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#countCompounds(C...)">countCompounds</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="type parameter in SingleCompoundSequenceReader">C</a>...&nbsp;compounds)</code>
<div class="block">Delegates to <code>SequenceMixin#countCompounds(org.biojava.nbio.core.sequence.template.Sequence, C[]) </code></div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>int</code></td>
<td class="colLast"><span class="strong">JoiningSequenceReader.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#countCompounds(C...)">countCompounds</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="type parameter in JoiningSequenceReader">C</a>...&nbsp;compounds)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>int</code></td>
<td class="colLast"><span class="strong">ArrayListSequenceReader.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#countCompounds(C...)">countCompounds</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="type parameter in ArrayListSequenceReader">C</a>...&nbsp;compounds)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>int</code></td>
<td class="colLast"><span class="strong">BitSequenceReader.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#countCompounds(C...)">countCompounds</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="type parameter in BitSequenceReader">C</a>...&nbsp;compounds)</code>
<div class="block">Counts the number of times a compound appears in this sequence store</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava.nbio.core.sequence.template">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a> in <a href="../../../../../../../org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../../../../org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a> with type parameters of type <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a>&lt;F extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>,T extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a>&lt;F extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>,T extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">The base class for DNA, RNA and Protein sequences.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>interface&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundSet</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>interface&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/CompoundTranslator.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundTranslator</a>&lt;F extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>,T extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>interface&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/CompoundTranslator.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundTranslator</a>&lt;F extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>,T extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>interface&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="interface in org.biojava.nbio.core.sequence.template">LightweightProfile</a>&lt;S extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;,C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">Defines a minimal data structure for reading and writing a sequence alignment.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>interface&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/ProxySequenceReader.html" title="interface in org.biojava.nbio.core.sequence.template">ProxySequenceReader</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>interface&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">Main interface for defining a collection of Compounds and accessing them
 using biological indexes</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceMixin.SequenceIterator.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin.SequenceIterator</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">A basic sequence iterator which iterates over the given Sequence by
 biological index.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>interface&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceReader.html" title="interface in org.biojava.nbio.core.sequence.template">SequenceReader</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>interface&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceView.html" title="interface in org.biojava.nbio.core.sequence.template">SequenceView</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing subinterfaces, and an explanation">
<caption><span>Subinterfaces of <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a> in <a href="../../../../../../../org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Interface and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>interface&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/ComplementCompound.html" title="interface in org.biojava.nbio.core.sequence.template">ComplementCompound</a></strong></code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../../../../org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a> that implement <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></strong></code>
<div class="block">The details of a Compound</div>
</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../../org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a> with type parameters of type <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>static &lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br><a href="http://java.sun.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a></code></td>
<td class="colLast"><span class="strong">SequenceMixin.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceMixin.html#checksum(org.biojava.nbio.core.sequence.template.Sequence)">checksum</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;sequence)</code>
<div class="block">Performs a simple CRC64 checksum on any given sequence.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>static &lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br>int</code></td>
<td class="colLast"><span class="strong">SequenceMixin.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceMixin.html#countCompounds(org.biojava.nbio.core.sequence.template.Sequence,%20C...)">countCompounds</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;sequence,
              C...&nbsp;compounds)</code>
<div class="block">For the given vargs of compounds this method counts the number of
 times those compounds appear in the given sequence</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static &lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br><a href="http://java.sun.com/javase/6/docs/api/java/util/Iterator.html?is-external=true" title="class or interface in java.util">Iterator</a>&lt;C&gt;</code></td>
<td class="colLast"><span class="strong">SequenceMixin.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceMixin.html#createIterator(org.biojava.nbio.core.sequence.template.Sequence)">createIterator</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;sequence)</code>
<div class="block">Creates a simple sequence iterator which moves through a sequence going
 from 1 to the length of the Sequence.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>static &lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceView.html" title="interface in org.biojava.nbio.core.sequence.template">SequenceView</a>&lt;C&gt;</code></td>
<td class="colLast"><span class="strong">SequenceMixin.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceMixin.html#createSubSequence(org.biojava.nbio.core.sequence.template.Sequence,%20int,%20int)">createSubSequence</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;sequence,
                 int&nbsp;start,
                 int&nbsp;end)</code>
<div class="block">Creates a simple sub sequence view delimited by the given start and end.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static &lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br><a href="http://java.sun.com/javase/6/docs/api/java/util/Map.html?is-external=true" title="class or interface in java.util">Map</a>&lt;C,<a href="http://java.sun.com/javase/6/docs/api/java/lang/Integer.html?is-external=true" title="class or interface in java.lang">Integer</a>&gt;</code></td>
<td class="colLast"><span class="strong">SequenceMixin.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceMixin.html#getComposition(org.biojava.nbio.core.sequence.template.Sequence)">getComposition</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;sequence)</code>
<div class="block">Does a linear scan over the given Sequence and records the number of
 times each base appears.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>static &lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br><a href="http://java.sun.com/javase/6/docs/api/java/util/Map.html?is-external=true" title="class or interface in java.util">Map</a>&lt;C,<a href="http://java.sun.com/javase/6/docs/api/java/lang/Double.html?is-external=true" title="class or interface in java.lang">Double</a>&gt;</code></td>
<td class="colLast"><span class="strong">SequenceMixin.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceMixin.html#getDistribution(org.biojava.nbio.core.sequence.template.Sequence)">getDistribution</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;sequence)</code>
<div class="block">Analogous to <a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceMixin.html#getComposition(org.biojava.nbio.core.sequence.template.Sequence)"><code>SequenceMixin.getComposition(Sequence)</code></a> but returns the
 distribution of that <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Compound</code></a> over the given sequence.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static &lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br>int</code></td>
<td class="colLast"><span class="strong">SequenceMixin.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceMixin.html#indexOf(org.biojava.nbio.core.sequence.template.Sequence,%20C)">indexOf</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;sequence,
       C&nbsp;compound)</code>
<div class="block">Performs a linear search of the given Sequence for the given compound.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>static &lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceView.html" title="interface in org.biojava.nbio.core.sequence.template">SequenceView</a>&lt;C&gt;</code></td>
<td class="colLast"><span class="strong">SequenceMixin.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceMixin.html#inverse(org.biojava.nbio.core.sequence.template.Sequence)">inverse</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;sequence)</code>
<div class="block">A method which attempts to do the right thing when is comes to a
 reverse/reverse complement</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static &lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br>int</code></td>
<td class="colLast"><span class="strong">SequenceMixin.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceMixin.html#lastIndexOf(org.biojava.nbio.core.sequence.template.Sequence,%20C)">lastIndexOf</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;sequence,
           C&nbsp;compound)</code>
<div class="block">Performs a reversed linear search of the given Sequence by wrapping
 it in a <a href="../../../../../../../org/biojava/nbio/core/sequence/views/ReversedSequenceView.html" title="class in org.biojava.nbio.core.sequence.views"><code>ReversedSequenceView</code></a> and passing it into
 <a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceMixin.html#indexOf(org.biojava.nbio.core.sequence.template.Sequence,%20C)"><code>SequenceMixin.indexOf(Sequence, Compound)</code></a>.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>static &lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br><a href="http://java.sun.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceView.html" title="interface in org.biojava.nbio.core.sequence.template">SequenceView</a>&lt;C&gt;&gt;</code></td>
<td class="colLast"><span class="strong">SequenceMixin.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceMixin.html#nonOverlappingKmers(org.biojava.nbio.core.sequence.template.Sequence,%20int)">nonOverlappingKmers</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;sequence,
                   int&nbsp;kmer)</code>
<div class="block">Produces kmers of the specified size e.g.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static &lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br><a href="http://java.sun.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceView.html" title="interface in org.biojava.nbio.core.sequence.template">SequenceView</a>&lt;C&gt;&gt;</code></td>
<td class="colLast"><span class="strong">SequenceMixin.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceMixin.html#overlappingKmers(org.biojava.nbio.core.sequence.template.Sequence,%20int)">overlappingKmers</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;sequence,
                int&nbsp;kmer)</code>
<div class="block">Used to generate overlapping k-mers such i.e.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>static &lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br>boolean</code></td>
<td class="colLast"><span class="strong">SequenceMixin.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceMixin.html#sequenceEquality(org.biojava.nbio.core.sequence.template.Sequence,%20org.biojava.nbio.core.sequence.template.Sequence)">sequenceEquality</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;source,
                <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;target)</code>
<div class="block">A case-sensitive manner of comparing two sequence objects together.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static &lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br>boolean</code></td>
<td class="colLast"><span class="strong">SequenceMixin.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceMixin.html#sequenceEqualityIgnoreCase(org.biojava.nbio.core.sequence.template.Sequence,%20org.biojava.nbio.core.sequence.template.Sequence)">sequenceEqualityIgnoreCase</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;source,
                          <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;target)</code>
<div class="block">A case-insensitive manner of comparing two sequence objects together.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>static &lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br><a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;</code></td>
<td class="colLast"><span class="strong">SequenceMixin.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceMixin.html#shuffle(org.biojava.nbio.core.sequence.template.Sequence)">shuffle</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;sequence)</code>
<div class="block">Implements sequence shuffling by first materializing the given
 <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> into a <a href="http://java.sun.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a>, applying
 <a href="http://java.sun.com/javase/6/docs/api/java/util/Collections.html?is-external=true#shuffle(java.util.List)" title="class or interface in java.util"><code>Collections.shuffle(List)</code></a> and then returning the shuffled
 elements in a new instance of <code>SequenceBackingStore</code> which behaves
 as a <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a>.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static &lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br><a href="http://java.sun.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;C&gt;</code></td>
<td class="colLast"><span class="strong">SequenceMixin.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceMixin.html#toList(org.biojava.nbio.core.sequence.template.Sequence)">toList</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;sequence)</code>
<div class="block">For the given <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> this will return a <a href="http://java.sun.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a> filled with
 the Compounds of that <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a>.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>static &lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br><a href="http://java.sun.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a></code></td>
<td class="colLast"><span class="strong">SequenceMixin.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceMixin.html#toString(org.biojava.nbio.core.sequence.template.Sequence)">toString</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;sequence)</code>
<div class="block">Shortcut to <a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceMixin.html#toStringBuilder(org.biojava.nbio.core.sequence.template.Sequence)"><code>SequenceMixin.toStringBuilder(org.biojava.nbio.core.sequence.template.Sequence)</code></a>
 which calls toString() on the resulting object.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static &lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br><a href="http://java.sun.com/javase/6/docs/api/java/lang/StringBuilder.html?is-external=true" title="class or interface in java.lang">StringBuilder</a></code></td>
<td class="colLast"><span class="strong">SequenceMixin.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceMixin.html#toStringBuilder(org.biojava.nbio.core.sequence.template.Sequence)">toStringBuilder</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;sequence)</code>
<div class="block">For the given Sequence this will return a <a href="http://java.sun.com/javase/6/docs/api/java/lang/StringBuilder.html?is-external=true" title="class or interface in java.lang"><code>StringBuilder</code></a> object
 filled with the results of <code>Compound#toString()</code>.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>static &lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br>void</code></td>
<td class="colLast"><span class="strong">SequenceMixin.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceMixin.html#write(java.lang.Appendable,%20org.biojava.nbio.core.sequence.template.Sequence)">write</a></strong>(<a href="http://java.sun.com/javase/6/docs/api/java/lang/Appendable.html?is-external=true" title="class or interface in java.lang">Appendable</a>&nbsp;appendable,
     <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;sequence)</code>
<div class="block">Used as a way of sending a Sequence to a writer without the cost of
 converting to a full length String and then writing the data out</div>
</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../../org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a> with parameters of type <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><span class="strong">AbstractCompoundSet.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#addCompound(C,%20C,%20C...)">addCompound</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="type parameter in AbstractCompoundSet">C</a>&nbsp;compound,
           <a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="type parameter in AbstractCompoundSet">C</a>&nbsp;lowerCasedCompound,
           <a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="type parameter in AbstractCompoundSet">C</a>...&nbsp;equivalents)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><span class="strong">AbstractCompoundTranslator.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#addCompounds(F,%20T...)">addCompounds</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="type parameter in AbstractCompoundTranslator">F</a>&nbsp;source,
            <a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="type parameter in AbstractCompoundTranslator">T</a>...&nbsp;targets)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>int</code></td>
<td class="colLast"><span class="strong">SequenceProxyView.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceProxyView.html#countCompounds(C...)">countCompounds</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="type parameter in SequenceProxyView">C</a>...&nbsp;compounds)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>int</code></td>
<td class="colLast"><span class="strong">AbstractSequence.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractSequence.html#countCompounds(C...)">countCompounds</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="type parameter in AbstractSequence">C</a>...&nbsp;compounds)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>int</code></td>
<td class="colLast"><span class="strong">Sequence.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html#countCompounds(C...)">countCompounds</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="type parameter in Sequence">C</a>...&nbsp;compounds)</code>
<div class="block">Returns the number of times we found a compound in the Sequence</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>static &lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;&nbsp;<br>int</code></td>
<td class="colLast"><span class="strong">SequenceMixin.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/SequenceMixin.html#countCompounds(org.biojava.nbio.core.sequence.template.Sequence,%20C...)">countCompounds</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;C&gt;&nbsp;sequence,
              C...&nbsp;compounds)</code>
<div class="block">For the given vargs of compounds this method counts the number of
 times those compounds appear in the given sequence</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>boolean</code></td>
<td class="colLast"><span class="strong">AbstractCompound.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/AbstractCompound.html#equalsIgnoreCase(org.biojava.nbio.core.sequence.template.Compound)">equalsIgnoreCase</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&nbsp;compound)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>boolean</code></td>
<td class="colLast"><span class="strong">Compound.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html#equalsIgnoreCase(org.biojava.nbio.core.sequence.template.Compound)">equalsIgnoreCase</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&nbsp;compound)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava.nbio.core.sequence.transcription">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a> in <a href="../../../../../../../org/biojava/nbio/core/sequence/transcription/package-summary.html">org.biojava.nbio.core.sequence.transcription</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../../../../org/biojava/nbio/core/sequence/transcription/package-summary.html">org.biojava.nbio.core.sequence.transcription</a> that implement <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></strong></code>
<div class="block">Attempts to wrap compounds so it is possible to view them
 in a case insensitive manner</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>static class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></strong></code>
<div class="block">Instance of a Codon which is 3 <a href="../../../../../../../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound"><code>NucleotideCompound</code></a>s, its
 corresponding <a href="../../../../../../../org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava.nbio.core.sequence.compound"><code>AminoAcidCompound</code></a> and if it is a start or stop codon.</div>
</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../../org/biojava/nbio/core/sequence/transcription/package-summary.html">org.biojava.nbio.core.sequence.transcription</a> with parameters of type <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>boolean</code></td>
<td class="colLast"><span class="strong">CaseInsensitiveCompound.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#equalsIgnoreCase(org.biojava.nbio.core.sequence.template.Compound)">equalsIgnoreCase</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&nbsp;compound)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>boolean</code></td>
<td class="colLast"><span class="strong">Table.Codon.</span><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/transcription/Table.Codon.html#equalsIgnoreCase(org.biojava.nbio.core.sequence.template.Compound)">equalsIgnoreCase</a></strong>(<a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&nbsp;compound)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava.nbio.core.sequence.views">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a> in <a href="../../../../../../../org/biojava/nbio/core/sequence/views/package-summary.html">org.biojava.nbio.core.sequence.views</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../../../../org/biojava/nbio/core/sequence/views/package-summary.html">org.biojava.nbio.core.sequence.views</a> with type parameters of type <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/views/ReversedSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">ReversedSequenceView</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">For a given sequence this class will return the base at the reversed
 position i.e.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/nbio/core/sequence/views/WindowedSequence.html" title="class in org.biojava.nbio.core.sequence.views">WindowedSequence</a>&lt;C extends <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">A sliding window view of a sequence which does not implement any
 interfaces like <a href="../../../../../../../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> because they do not fit how this works.</div>
</td>
</tr>
</tbody>
</table>
</li>
</ul>
</li>
</ul>
</div>
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar_bottom">
<!--   -->
</a><a href="#skip-navbar_bottom" title="Skip navigation links"></a><a name="navbar_bottom_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../../overview-summary.html">Overview</a></li>
<li><a href="../package-summary.html">Package</a></li>
<li><a href="../../../../../../../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Class</a></li>
<li class="navBarCell1Rev">Use</li>
<li><a href="../package-tree.html">Tree</a></li>
<li><a href="../../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../../index-all.html">Index</a></li>
<li><a href="../../../../../../../help-doc.html">Help</a></li>
</ul>
<div class="aboutLanguage"><em><script src="http://www.google-analytics.com/urchin.js" 							type="text/javascript"></script><script 							type="text/javascript">_uacct = 							"UA-1326640-1";urchinTracker();</script></em></div>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="../../../../../../../index.html?org/biojava/nbio/core/sequence/template/class-use/Compound.html" target="_top">Frames</a></li>
<li><a href="Compound.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="../../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
<p class="legalCopy"><small>Copyright &#169; 2000-2015 <a href="http://www.biojava.org">BioJava</a>. All Rights Reserved.</small></p>
</body>
</html>
