<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!-- Generated by Apache Maven Doxia at Jan 30, 2015 -->
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
    <title>Checkstyle Results</title>
    <style type="text/css" media="all">
      @import url("./css/maven-base.css");
      @import url("./css/maven-theme.css");
      @import url("./css/site.css");
    </style>
    <link rel="stylesheet" href="./css/print.css" type="text/css" media="print" />
    <meta name="Date-Revision-yyyymmdd" content="20150130" />
    <meta http-equiv="Content-Language" content="en" />
        
        </head>
  <body class="composite">
    <div id="banner">
                    <div id="bannerLeft">
                biojava-core
                </div>
                    <div class="clear">
        <hr/>
      </div>
    </div>
    <div id="breadcrumbs">
            
        
                <div class="xleft">
        <span id="publishDate">Last Published: 2015-01-30</span>
                  &nbsp;| <span id="projectVersion">Version: 4.0.0</span>
                      </div>
            <div class="xright">                    <a href="./" title="biojava-core">biojava-core</a>
              
        
      </div>
      <div class="clear">
        <hr/>
      </div>
    </div>
    <div id="leftColumn">
      <div id="navcolumn">
             
        
                                <h5>Parent Project</h5>
                  <ul>
                  <li class="none">
                          <a href="../index.html" title="biojava">biojava</a>
            </li>
          </ul>
                       <h5>Project Documentation</h5>
                  <ul>
                                                                                                                                                                                <li class="collapsed">
                          <a href="project-info.html" title="Project Information">Project Information</a>
                  </li>
                                                                                                                                    <li class="expanded">
                          <a href="project-reports.html" title="Project Reports">Project Reports</a>
                    <ul>
                      <li class="none">
                          <a href="findbugs.html" title="FindBugs Report">FindBugs Report</a>
            </li>
                      <li class="none">
            <strong>Checkstyle</strong>
          </li>
                      <li class="none">
                          <a href="xref-test/index.html" title="Test Source Xref">Test Source Xref</a>
            </li>
                      <li class="none">
                          <a href="xref/index.html" title="Source Xref">Source Xref</a>
            </li>
              </ul>
        </li>
          </ul>
                             <a href="http://maven.apache.org/" title="Built by Maven" class="poweredBy">
        <img class="poweredBy" alt="Built by Maven" src="./images/logos/maven-feather.png" />
      </a>
                   
        
            </div>
    </div>
    <div id="bodyColumn">
      <div id="contentBox">
        <div class="section"><h2>Checkstyle Results<a name="Checkstyle_Results"></a></h2><p>The following document contains the results of <a class="externalLink" href="http://checkstyle.sourceforge.net/">Checkstyle</a>.&#160;<a href="checkstyle.rss"><img alt="rss feed" src="images/rss.png" /></a></p></div><div class="section"><h2>Summary<a name="Summary"></a></h2><table border="0" class="bodyTable"><tr class="a"><th>Files</th><th><img src="images/icon_info_sml.gif" alt="" />&#160;Info</th><th><img src="images/icon_warning_sml.gif" alt="" />&#160;Warnings</th><th><img src="images/icon_error_sml.gif" alt="" />&#160;Errors</th></tr><tr class="b"><td>140</td><td>0</td><td>0</td><td>11481</td></tr></table></div><div class="section"><h2>Files<a name="Files"></a></h2><table border="0" class="bodyTable"><tr class="a"><th>File</th><th><img src="images/icon_info_sml.gif" alt="" />&#160;I</th><th><img src="images/icon_warning_sml.gif" alt="" />&#160;W</th><th><img src="images/icon_error_sml.gif" alt="" />&#160;E</th></tr><tr class="b"><td><a href="#org.biojava.nbio.core.exceptions.CompoundNotFoundException.java">org/biojava/nbio/core/exceptions/CompoundNotFoundException.java</a></td><td>0</td><td>0</td><td>12</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.exceptions.ParserException.java">org/biojava/nbio/core/exceptions/ParserException.java</a></td><td>0</td><td>0</td><td>9</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.exceptions.TranslationException.java">org/biojava/nbio/core/exceptions/TranslationException.java</a></td><td>0</td><td>0</td><td>9</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.AccessionID.java">org/biojava/nbio/core/sequence/AccessionID.java</a></td><td>0</td><td>0</td><td>22</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.BasicSequence.java">org/biojava/nbio/core/sequence/BasicSequence.java</a></td><td>0</td><td>0</td><td>8</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.CDSComparator.java">org/biojava/nbio/core/sequence/CDSComparator.java</a></td><td>0</td><td>0</td><td>13</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.CDSSequence.java">org/biojava/nbio/core/sequence/CDSSequence.java</a></td><td>0</td><td>0</td><td>26</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.ChromosomeSequence.java">org/biojava/nbio/core/sequence/ChromosomeSequence.java</a></td><td>0</td><td>0</td><td>62</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.DNASequence.java">org/biojava/nbio/core/sequence/DNASequence.java</a></td><td>0</td><td>0</td><td>77</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.DataSource.java">org/biojava/nbio/core/sequence/DataSource.java</a></td><td>0</td><td>0</td><td>20</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.ExonComparator.java">org/biojava/nbio/core/sequence/ExonComparator.java</a></td><td>0</td><td>0</td><td>8</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.ExonSequence.java">org/biojava/nbio/core/sequence/ExonSequence.java</a></td><td>0</td><td>0</td><td>15</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.GeneSequence.java">org/biojava/nbio/core/sequence/GeneSequence.java</a></td><td>0</td><td>0</td><td>130</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.IntronSequence.java">org/biojava/nbio/core/sequence/IntronSequence.java</a></td><td>0</td><td>0</td><td>10</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.MultipleSequenceAlignment.java">org/biojava/nbio/core/sequence/MultipleSequenceAlignment.java</a></td><td>0</td><td>0</td><td>112</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.ProteinSequence.java">org/biojava/nbio/core/sequence/ProteinSequence.java</a></td><td>0</td><td>0</td><td>73</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.RNASequence.java">org/biojava/nbio/core/sequence/RNASequence.java</a></td><td>0</td><td>0</td><td>41</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.SequenceComparator.java">org/biojava/nbio/core/sequence/SequenceComparator.java</a></td><td>0</td><td>0</td><td>7</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.SequenceOptimizationHints.java">org/biojava/nbio/core/sequence/SequenceOptimizationHints.java</a></td><td>0</td><td>0</td><td>27</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.StartCodonSequence.java">org/biojava/nbio/core/sequence/StartCodonSequence.java</a></td><td>0</td><td>0</td><td>11</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.StopCodonSequence.java">org/biojava/nbio/core/sequence/StopCodonSequence.java</a></td><td>0</td><td>0</td><td>11</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.Strand.java">org/biojava/nbio/core/sequence/Strand.java</a></td><td>0</td><td>0</td><td>13</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.TaxonomyID.java">org/biojava/nbio/core/sequence/TaxonomyID.java</a></td><td>0</td><td>0</td><td>11</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.TranscriptSequence.java">org/biojava/nbio/core/sequence/TranscriptSequence.java</a></td><td>0</td><td>0</td><td>111</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.compound.AmbiguityDNACompoundSet.java">org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundSet.java</a></td><td>0</td><td>0</td><td>7</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.compound.AmbiguityRNACompoundSet.java">org/biojava/nbio/core/sequence/compound/AmbiguityRNACompoundSet.java</a></td><td>0</td><td>0</td><td>5</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.compound.AminoAcidCompound.java">org/biojava/nbio/core/sequence/compound/AminoAcidCompound.java</a></td><td>0</td><td>0</td><td>15</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.compound.AminoAcidCompoundSet.java">org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.java</a></td><td>0</td><td>0</td><td>128</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.compound.CodonCompound.java">org/biojava/nbio/core/sequence/compound/CodonCompound.java</a></td><td>0</td><td>0</td><td>40</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.compound.DNACompoundSet.java">org/biojava/nbio/core/sequence/compound/DNACompoundSet.java</a></td><td>0</td><td>0</td><td>18</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.compound.NucleotideCompound.java">org/biojava/nbio/core/sequence/compound/NucleotideCompound.java</a></td><td>0</td><td>0</td><td>43</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.compound.RNACompoundSet.java">org/biojava/nbio/core/sequence/compound/RNACompoundSet.java</a></td><td>0</td><td>0</td><td>11</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.edits.Edit.java">org/biojava/nbio/core/sequence/edits/Edit.java</a></td><td>0</td><td>0</td><td>124</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.features.AbstractFeature.java">org/biojava/nbio/core/sequence/features/AbstractFeature.java</a></td><td>0</td><td>0</td><td>152</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.features.DBReferenceInfo.java">org/biojava/nbio/core/sequence/features/DBReferenceInfo.java</a></td><td>0</td><td>0</td><td>36</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.features.DatabaseReferenceInterface.java">org/biojava/nbio/core/sequence/features/DatabaseReferenceInterface.java</a></td><td>0</td><td>0</td><td>5</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.features.FeatureDbReferenceInfo.java">org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.java</a></td><td>0</td><td>0</td><td>62</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.features.FeatureInterface.java">org/biojava/nbio/core/sequence/features/FeatureInterface.java</a></td><td>0</td><td>0</td><td>53</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.features.FeatureRetriever.java">org/biojava/nbio/core/sequence/features/FeatureRetriever.java</a></td><td>0</td><td>0</td><td>2</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.features.FeaturesKeyWordInterface.java">org/biojava/nbio/core/sequence/features/FeaturesKeyWordInterface.java</a></td><td>0</td><td>0</td><td>11</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.features.Qualifier.java">org/biojava/nbio/core/sequence/features/Qualifier.java</a></td><td>0</td><td>0</td><td>101</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.features.QualityFeature.java">org/biojava/nbio/core/sequence/features/QualityFeature.java</a></td><td>0</td><td>0</td><td>19</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.features.QuantityFeature.java">org/biojava/nbio/core/sequence/features/QuantityFeature.java</a></td><td>0</td><td>0</td><td>17</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.features.TextFeature.java">org/biojava/nbio/core/sequence/features/TextFeature.java</a></td><td>0</td><td>0</td><td>12</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.io.BufferedReaderBytesRead.java">org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.java</a></td><td>0</td><td>0</td><td>78</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.io.CasePreservingProteinSequenceCreator.java">org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.java</a></td><td>0</td><td>0</td><td>179</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.io.DNASequenceCreator.java">org/biojava/nbio/core/sequence/io/DNASequenceCreator.java</a></td><td>0</td><td>0</td><td>19</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.io.FastaGeneWriter.java">org/biojava/nbio/core/sequence/io/FastaGeneWriter.java</a></td><td>0</td><td>0</td><td>74</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.io.FastaReader.java">org/biojava/nbio/core/sequence/io/FastaReader.java</a></td><td>0</td><td>0</td><td>189</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.io.FastaReaderHelper.java">org/biojava/nbio/core/sequence/io/FastaReaderHelper.java</a></td><td>0</td><td>0</td><td>59</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.io.FastaSequenceParser.java">org/biojava/nbio/core/sequence/io/FastaSequenceParser.java</a></td><td>0</td><td>0</td><td>6</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.io.FastaWriter.java">org/biojava/nbio/core/sequence/io/FastaWriter.java</a></td><td>0</td><td>0</td><td>70</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.io.FastaWriterHelper.java">org/biojava/nbio/core/sequence/io/FastaWriterHelper.java</a></td><td>0</td><td>0</td><td>83</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.io.FileProxyDNASequenceCreator.java">org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.java</a></td><td>0</td><td>0</td><td>54</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.io.FileProxyProteinSequenceCreator.java">org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.java</a></td><td>0</td><td>0</td><td>64</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.io.GenbankReader.java">org/biojava/nbio/core/sequence/io/GenbankReader.java</a></td><td>0</td><td>0</td><td>112</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.io.GenbankReaderHelper.java">org/biojava/nbio/core/sequence/io/GenbankReaderHelper.java</a></td><td>0</td><td>0</td><td>96</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.io.GenbankSequenceParser.java">org/biojava/nbio/core/sequence/io/GenbankSequenceParser.java</a></td><td>0</td><td>0</td><td>143</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.io.GenbankWriter.java">org/biojava/nbio/core/sequence/io/GenbankWriter.java</a></td><td>0</td><td>0</td><td>206</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.io.GenbankWriterHelper.java">org/biojava/nbio/core/sequence/io/GenbankWriterHelper.java</a></td><td>0</td><td>0</td><td>220</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.io.GenericFastaHeaderFormat.java">org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.java</a></td><td>0</td><td>0</td><td>11</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.io.GenericFastaHeaderParser.java">org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.java</a></td><td>0</td><td>0</td><td>61</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.io.GenericGenbankHeaderFormat.java">org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.java</a></td><td>0</td><td>0</td><td>432</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.io.GenericGenbankHeaderParser.java">org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.java</a></td><td>0</td><td>0</td><td>86</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.io.GenericInsdcHeaderFormat.java">org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.java</a></td><td>0</td><td>0</td><td>428</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.io.IUPACParser.java">org/biojava/nbio/core/sequence/io/IUPACParser.java</a></td><td>0</td><td>0</td><td>129</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.io.PlainFastaHeaderParser.java">org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.java</a></td><td>0</td><td>0</td><td>46</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.io.ProteinSequenceCreator.java">org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.java</a></td><td>0</td><td>0</td><td>20</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.io.RNASequenceCreator.java">org/biojava/nbio/core/sequence/io/RNASequenceCreator.java</a></td><td>0</td><td>0</td><td>25</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.io.template.FastaHeaderFormatInterface.java">org/biojava/nbio/core/sequence/io/template/FastaHeaderFormatInterface.java</a></td><td>0</td><td>0</td><td>8</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.io.template.GenbankHeaderFormatInterface.java">org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.java</a></td><td>0</td><td>0</td><td>16</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.io.template.SequenceCreatorInterface.java">org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.java</a></td><td>0</td><td>0</td><td>44</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.io.template.SequenceHeaderParserInterface.java">org/biojava/nbio/core/sequence/io/template/SequenceHeaderParserInterface.java</a></td><td>0</td><td>0</td><td>6</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.io.template.SequenceParserInterface.java">org/biojava/nbio/core/sequence/io/template/SequenceParserInterface.java</a></td><td>0</td><td>0</td><td>7</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.io.util.ClasspathResource.java">org/biojava/nbio/core/sequence/io/util/ClasspathResource.java</a></td><td>0</td><td>0</td><td>42</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.io.util.IOUtils.java">org/biojava/nbio/core/sequence/io/util/IOUtils.java</a></td><td>0</td><td>0</td><td>440</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.loader.ArrayListProxySequenceReader.java">org/biojava/nbio/core/sequence/loader/ArrayListProxySequenceReader.java</a></td><td>0</td><td>0</td><td>2</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.loader.GenbankProxySequenceReader.java">org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.java</a></td><td>0</td><td>0</td><td>144</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.loader.SequenceFileProxyLoader.java">org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.java</a></td><td>0</td><td>0</td><td>129</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.loader.StringProxySequenceReader.java">org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.java</a></td><td>0</td><td>0</td><td>86</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.loader.UniprotProxySequenceReader.java">org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.java</a></td><td>0</td><td>0</td><td>398</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.location.FuzzyPoint.java">org/biojava/nbio/core/sequence/location/FuzzyPoint.java</a></td><td>0</td><td>0</td><td>53</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.location.InsdcLocations.java">org/biojava/nbio/core/sequence/location/InsdcLocations.java</a></td><td>0</td><td>0</td><td>92</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.location.InsdcParser.java">org/biojava/nbio/core/sequence/location/InsdcParser.java</a></td><td>0</td><td>0</td><td>108</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.location.LocationHelper.java">org/biojava/nbio/core/sequence/location/LocationHelper.java</a></td><td>0</td><td>0</td><td>96</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.location.SequenceLocation.java">org/biojava/nbio/core/sequence/location/SequenceLocation.java</a></td><td>0</td><td>0</td><td>45</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.location.SimpleLocation.java">org/biojava/nbio/core/sequence/location/SimpleLocation.java</a></td><td>0</td><td>0</td><td>89</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.location.SimplePoint.java">org/biojava/nbio/core/sequence/location/SimplePoint.java</a></td><td>0</td><td>0</td><td>49</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.location.template.AbstractLocation.java">org/biojava/nbio/core/sequence/location/template/AbstractLocation.java</a></td><td>0</td><td>0</td><td>195</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.location.template.AccesionedLocation.java">org/biojava/nbio/core/sequence/location/template/AccesionedLocation.java</a></td><td>0</td><td>0</td><td>2</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.location.template.Location.java">org/biojava/nbio/core/sequence/location/template/Location.java</a></td><td>0</td><td>0</td><td>91</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.location.template.Point.java">org/biojava/nbio/core/sequence/location/template/Point.java</a></td><td>0</td><td>0</td><td>19</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.storage.ArrayListSequenceReader.java">org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.java</a></td><td>0</td><td>0</td><td>104</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.storage.BitSequenceReader.java">org/biojava/nbio/core/sequence/storage/BitSequenceReader.java</a></td><td>0</td><td>0</td><td>211</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.storage.FourBitSequenceReader.java">org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.java</a></td><td>0</td><td>0</td><td>64</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.storage.JoiningSequenceReader.java">org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.java</a></td><td>0</td><td>0</td><td>115</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.storage.SequenceAsStringHelper.java">org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.java</a></td><td>0</td><td>0</td><td>29</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.storage.SingleCompoundSequenceReader.java">org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.java</a></td><td>0</td><td>0</td><td>124</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.storage.TwoBitSequenceReader.java">org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.java</a></td><td>0</td><td>0</td><td>137</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.template.AbstractCompound.java">org/biojava/nbio/core/sequence/template/AbstractCompound.java</a></td><td>0</td><td>0</td><td>37</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.template.AbstractCompoundSet.java">org/biojava/nbio/core/sequence/template/AbstractCompoundSet.java</a></td><td>0</td><td>0</td><td>89</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.template.AbstractCompoundTranslator.java">org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.java</a></td><td>0</td><td>0</td><td>71</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.template.AbstractNucleotideCompoundSet.java">org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.java</a></td><td>0</td><td>0</td><td>84</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.template.AbstractSequence.java">org/biojava/nbio/core/sequence/template/AbstractSequence.java</a></td><td>0</td><td>0</td><td>301</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.template.Accessioned.java">org/biojava/nbio/core/sequence/template/Accessioned.java</a></td><td>0</td><td>0</td><td>3</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.template.ComplementCompound.java">org/biojava/nbio/core/sequence/template/ComplementCompound.java</a></td><td>0</td><td>0</td><td>3</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.template.Compound.java">org/biojava/nbio/core/sequence/template/Compound.java</a></td><td>0</td><td>0</td><td>19</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.template.CompoundSet.java">org/biojava/nbio/core/sequence/template/CompoundSet.java</a></td><td>0</td><td>0</td><td>36</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.template.CompoundTranslator.java">org/biojava/nbio/core/sequence/template/CompoundTranslator.java</a></td><td>0</td><td>0</td><td>5</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.template.LightweightProfile.java">org/biojava/nbio/core/sequence/template/LightweightProfile.java</a></td><td>0</td><td>0</td><td>22</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.template.ProxySequenceReader.java">org/biojava/nbio/core/sequence/template/ProxySequenceReader.java</a></td><td>0</td><td>0</td><td>4</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.template.Sequence.java">org/biojava/nbio/core/sequence/template/Sequence.java</a></td><td>0</td><td>0</td><td>26</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.template.SequenceMixin.java">org/biojava/nbio/core/sequence/template/SequenceMixin.java</a></td><td>0</td><td>0</td><td>144</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.template.SequenceProxyView.java">org/biojava/nbio/core/sequence/template/SequenceProxyView.java</a></td><td>0</td><td>0</td><td>63</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.template.SequenceReader.java">org/biojava/nbio/core/sequence/template/SequenceReader.java</a></td><td>0</td><td>0</td><td>12</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.template.SequenceView.java">org/biojava/nbio/core/sequence/template/SequenceView.java</a></td><td>0</td><td>0</td><td>7</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.transcription.CaseInsensitiveCompound.java">org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.java</a></td><td>0</td><td>0</td><td>25</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.transcription.DNAToRNATranslator.java">org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.java</a></td><td>0</td><td>0</td><td>33</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.transcription.Frame.java">org/biojava/nbio/core/sequence/transcription/Frame.java</a></td><td>0</td><td>0</td><td>34</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.transcription.RNAToAminoAcidTranslator.java">org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.java</a></td><td>0</td><td>0</td><td>328</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.transcription.Table.java">org/biojava/nbio/core/sequence/transcription/Table.java</a></td><td>0</td><td>0</td><td>100</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.transcription.TranscriptionEngine.java">org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.java</a></td><td>0</td><td>0</td><td>469</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.views.ComplementSequenceView.java">org/biojava/nbio/core/sequence/views/ComplementSequenceView.java</a></td><td>0</td><td>0</td><td>11</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.views.ReversedSequenceView.java">org/biojava/nbio/core/sequence/views/ReversedSequenceView.java</a></td><td>0</td><td>0</td><td>10</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.views.RnaSequenceView.java">org/biojava/nbio/core/sequence/views/RnaSequenceView.java</a></td><td>0</td><td>0</td><td>35</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.views.WindowedSequence.java">org/biojava/nbio/core/sequence/views/WindowedSequence.java</a></td><td>0</td><td>0</td><td>42</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.util.CRC64Checksum.java">org/biojava/nbio/core/util/CRC64Checksum.java</a></td><td>0</td><td>0</td><td>82</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.util.ConcurrencyTools.java">org/biojava/nbio/core/util/ConcurrencyTools.java</a></td><td>0</td><td>0</td><td>36</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.util.Equals.java">org/biojava/nbio/core/util/Equals.java</a></td><td>0</td><td>0</td><td>20</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.util.FlatFileCache.java">org/biojava/nbio/core/util/FlatFileCache.java</a></td><td>0</td><td>0</td><td>135</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.util.Hashcoder.java">org/biojava/nbio/core/util/Hashcoder.java</a></td><td>0</td><td>0</td><td>40</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.util.InputStreamProvider.java">org/biojava/nbio/core/util/InputStreamProvider.java</a></td><td>0</td><td>0</td><td>120</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.util.PrettyXMLWriter.java">org/biojava/nbio/core/util/PrettyXMLWriter.java</a></td><td>0</td><td>0</td><td>159</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.util.SequenceTools.java">org/biojava/nbio/core/util/SequenceTools.java</a></td><td>0</td><td>0</td><td>92</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.util.SingleLinkageClusterer.java">org/biojava/nbio/core/util/SingleLinkageClusterer.java</a></td><td>0</td><td>0</td><td>717</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.util.SoftHashMap.java">org/biojava/nbio/core/util/SoftHashMap.java</a></td><td>0</td><td>0</td><td>55</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.util.StringManipulationHelper.java">org/biojava/nbio/core/util/StringManipulationHelper.java</a></td><td>0</td><td>0</td><td>214</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.util.UncompressInputStream.java">org/biojava/nbio/core/util/UncompressInputStream.java</a></td><td>0</td><td>0</td><td>159</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.util.XMLHelper.java">org/biojava/nbio/core/util/XMLHelper.java</a></td><td>0</td><td>0</td><td>49</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.util.XMLWriter.java">org/biojava/nbio/core/util/XMLWriter.java</a></td><td>0</td><td>0</td><td>56</td></tr></table></div><div class="section"><h2>Rules<a name="Rules"></a></h2><table border="0" class="bodyTable"><tr class="b"><th>Rule</th><th>Violations</th><th>Severity</th></tr><tr class="a"><td>JavadocPackage<ul><li><b>allowLegacy</b>: <tt>&quot;true&quot;</tt></li></ul></td><td>16</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>NewlineAtEndOfFile</td><td>7</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>Translation</td><td>0</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>FileLength</td><td>0</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>FileTabCharacter<ul><li><b>eachLine</b>: <tt>&quot;true&quot;</tt></li></ul></td><td>3230</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>RegexpSingleline<ul><li><b>message</b>: <tt>&quot;Line has trailing spaces.&quot;</tt></li><li><b>format</b>: <tt>&quot;\s+$&quot;</tt></li></ul></td><td>608</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>JavadocMethod</td><td>1378</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>JavadocType</td><td>76</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>JavadocVariable</td><td>483</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>JavadocStyle</td><td>309</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>ConstantName</td><td>39</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>LocalFinalVariableName</td><td>0</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>LocalVariableName</td><td>34</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>MemberName</td><td>9</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>MethodName</td><td>17</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>PackageName</td><td>0</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>ParameterName</td><td>5</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>StaticVariableName</td><td>0</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>TypeName</td><td>0</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>AvoidStarImport</td><td>41</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>IllegalImport</td><td>0</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>RedundantImport</td><td>0</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>UnusedImports</td><td>9</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>LineLength</td><td>1509</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>MethodLength</td><td>2</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>ParameterNumber</td><td>5</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>EmptyForIteratorPad</td><td>2</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>MethodParamPad</td><td>2</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>NoWhitespaceAfter</td><td>6</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>NoWhitespaceBefore</td><td>8</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>OperatorWrap</td><td>35</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>ParenPad</td><td>44</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>TypecastParenPad</td><td>0</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>WhitespaceAfter</td><td>157</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>WhitespaceAround</td><td>487</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>ModifierOrder</td><td>38</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>RedundantModifier</td><td>80</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>AvoidNestedBlocks</td><td>0</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>EmptyBlock</td><td>7</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>LeftCurly</td><td>27</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>NeedBraces</td><td>76</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>RightCurly</td><td>48</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>AvoidInlineConditionals</td><td>36</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>EmptyStatement</td><td>0</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>EqualsHashCode</td><td>0</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>HiddenField</td><td>306</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>IllegalInstantiation</td><td>0</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>InnerAssignment</td><td>21</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>MagicNumber</td><td>138</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>MissingSwitchDefault</td><td>1</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>RedundantThrows</td><td>19</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>SimplifyBooleanExpression</td><td>1</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>SimplifyBooleanReturn</td><td>2</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>DesignForExtension</td><td>725</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>FinalClass</td><td>5</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>HideUtilityClassConstructor</td><td>11</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>InterfaceIsType</td><td>0</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>VisibilityModifier</td><td>53</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>ArrayTypeStyle</td><td>7</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>FinalParameters</td><td>1368</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="a"><td>TodoComment</td><td>0</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr><tr class="b"><td>UpperEll</td><td>0</td><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td></tr></table></div><div class="section"><h2>Details<a name="Details"></a></h2><div class="section"><h3 id="org.biojava.nbio.core.exceptions.CompoundNotFoundException.java">org/biojava/nbio/core/exceptions/CompoundNotFoundException.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing package-info.java file.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/CompoundNotFoundException.html#L0"></a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/CompoundNotFoundException.html#L28">28</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/CompoundNotFoundException.html#L29">29</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/CompoundNotFoundException.html#L30">30</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/CompoundNotFoundException.html#L30">30</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/CompoundNotFoundException.html#L31">31</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/CompoundNotFoundException.html#L32">32</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/CompoundNotFoundException.html#L34">34</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/CompoundNotFoundException.html#L34">34</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter msg should be final.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/CompoundNotFoundException.html#L34">34</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/CompoundNotFoundException.html#L35">35</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/CompoundNotFoundException.html#L36">36</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.exceptions.ParserException.java">org/biojava/nbio/core/exceptions/ParserException.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/ParserException.html#L24">24</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/ParserException.html#L30">30</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/ParserException.html#L32">32</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter message should be final.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/ParserException.html#L32">32</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/ParserException.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter e should be final.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/ParserException.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/ParserException.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter message should be final.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/ParserException.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter e should be final.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/ParserException.html#L40">40</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.exceptions.TranslationException.java">org/biojava/nbio/core/exceptions/TranslationException.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/TranslationException.html#L25">25</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/TranslationException.html#L33">33</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/TranslationException.html#L35">35</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter m should be final.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/TranslationException.html#L35">35</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/TranslationException.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter t should be final.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/TranslationException.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/TranslationException.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter m should be final.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/TranslationException.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter t should be final.</td><td><a href="./xref/org/biojava/nbio/core/exceptions/TranslationException.html#L43">43</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.AccessionID.java">org/biojava/nbio/core/sequence/AccessionID.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing package-info.java file.</td><td><a href="./xref/org/biojava/nbio/core/sequence/AccessionID.html#L0"></a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 99).</td><td><a href="./xref/org/biojava/nbio/core/sequence/AccessionID.html#L29">29</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/AccessionID.html#L30">30</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/AccessionID.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/AccessionID.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/AccessionID.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/AccessionID.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter id should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/AccessionID.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'id' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/AccessionID.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'id'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/AccessionID.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter id should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/AccessionID.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'id' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/AccessionID.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'id'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/AccessionID.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter source should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/AccessionID.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'source' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/AccessionID.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'source'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/AccessionID.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getID' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/AccessionID.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getDataSource' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/AccessionID.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'equals' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/AccessionID.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter o should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/AccessionID.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'hashCode' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/AccessionID.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'toString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/AccessionID.html#L106">106</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.BasicSequence.java">org/biojava/nbio/core/sequence/BasicSequence.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - org.biojava.nbio.core.sequence.template.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/BasicSequence.html#L26">26</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/BasicSequence.html#L33">33</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/BasicSequence.html#L35">35</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/BasicSequence.html#L35">35</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/BasicSequence.html#L35">35</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/BasicSequence.html#L35">35</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/BasicSequence.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter reader should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/BasicSequence.html#L39">39</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.CDSComparator.java">org/biojava/nbio/core/sequence/CDSComparator.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSComparator.html#L30">30</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSComparator.html#L30">30</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSComparator.html#L32">32</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'compare' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSComparator.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSComparator.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter o1 should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSComparator.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'o1'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSComparator.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter o2 should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSComparator.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'o2'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSComparator.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSComparator.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSComparator.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSComparator.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSComparator.html#L43">43</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.CDSSequence.java">org/biojava/nbio/core/sequence/CDSSequence.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L31">31</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L33">33</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'phase' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'parentTranscriptSequence' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter parentSequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'parentSequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioBegin should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioBegin'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioEnd should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioEnd'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter phase should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'phase' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'phase'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getPhase' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getStrand' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCodingSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L90">90</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L91">91</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 111).</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L93">93</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 101).</td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSSequence.html#L98">98</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.ChromosomeSequence.java">org/biojava/nbio/core/sequence/ChromosomeSequence.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L33">33</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 112).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter seqString should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'seqString'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'CompoundNotFoundException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter proxyLoader should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L66">66</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'proxyLoader'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 106).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 127).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter seqString should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'seqString'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'CompoundNotFoundException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 123).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 124).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter proxyLoader should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'proxyLoader'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getChromosomeNumber' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setChromosomeNumber' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter chromosomeNumber should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'chromosomeNumber' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 112).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L104">104</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getGeneSequences' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L118">118</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'removeGeneSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L118">118</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'accession'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L123">123</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L124">124</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L126">126</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 97).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addGene' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'accession'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioBegin should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioBegin'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioEnd should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioEnd'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'strand'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L137">137</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L148">148</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getGene' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L148">148</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L148">148</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'accession'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ChromosomeSequence.html#L148">148</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.DNASequence.java">org/biojava/nbio/core/sequence/DNASequence.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - org.biojava.nbio.core.sequence.template.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L29">29</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'static' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'logger' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L51">51</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter seqString should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'seqString'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'CompoundNotFoundException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L70">70</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter proxyLoader should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L73">73</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'proxyLoader'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 120).</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter seqString should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'seqString'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'CompoundNotFoundException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 117).</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter proxyLoader should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'proxyLoader'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 103).</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getRNASequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L101">101</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getRNASequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L110">110</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter engine should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'engine'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L110">110</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getRNASequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L119">119</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter frame should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'frame'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L119">119</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getRNASequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L123">123</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L123">123</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter engine should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L123">123</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter frame should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L123">123</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L124">124</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getGCCount' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L131">131</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L135">135</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L138">138</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getReverse' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L138">138</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L142">142</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L145">145</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getComplement' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L145">145</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L149">149</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L152">152</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getReverseComplement' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L152">152</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getDNAType' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L159">159</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setDNAType' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter dnaType should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'dnaType' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L170">170</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter args should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L170">170</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L174">174</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 110).</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L175">175</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L176">176</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/DNASequence.html#L177">177</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.DataSource.java">org/biojava/nbio/core/sequence/DataSource.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DataSource.html#L27">27</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unclosed HTML tag found: &lt;pre&gt;</td><td><a href="./xref/org/biojava/nbio/core/sequence/DataSource.html#L28">28</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Extra HTML tag found: &lt;/pr&gt;</td><td><a href="./xref/org/biojava/nbio/core/sequence/DataSource.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 115).</td><td><a href="./xref/org/biojava/nbio/core/sequence/DataSource.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DataSource.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DataSource.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DataSource.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DataSource.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DataSource.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DataSource.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DataSource.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DataSource.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DataSource.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DataSource.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DataSource.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DataSource.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DataSource.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DataSource.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DataSource.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/DataSource.html#L48">48</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.ExonComparator.java">org/biojava/nbio/core/sequence/ExonComparator.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ExonComparator.html#L30">30</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ExonComparator.html#L31">31</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ExonComparator.html#L34">34</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ExonComparator.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'compare' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ExonComparator.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ExonComparator.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter o1 should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ExonComparator.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter o2 should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ExonComparator.html#L38">38</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.ExonSequence.java">org/biojava/nbio/core/sequence/ExonSequence.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ExonSequence.html#L28">28</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ExonSequence.html#L34">34</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 112).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ExonSequence.html#L37">37</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 99).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ExonSequence.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ExonSequence.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 106).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ExonSequence.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 116).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ExonSequence.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ExonSequence.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter parentGeneSequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ExonSequence.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'parentGeneSequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ExonSequence.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioBegin should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ExonSequence.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioBegin'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ExonSequence.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioEnd should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ExonSequence.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioEnd'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ExonSequence.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ExonSequence.html#L54">54</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.GeneSequence.java">org/biojava/nbio/core/sequence/GeneSequence.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L42">42</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'static' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L42">42</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'logger' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 140).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 124).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 118).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 97).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 186).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'intronAdded' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 143).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 141).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 119).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 115).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 97).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter parentSequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'parentSequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter begin should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'begin'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'strand' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getParentChromosomeSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 114).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 111).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L93">93</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addIntronsUsingExons' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L112">112</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 162).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L113">113</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 160).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L123">123</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L124">124</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 157).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L125">125</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L129">129</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L133">133</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getStrand' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L137">137</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setStrand' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L144">144</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L144">144</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'strand' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L144">144</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L148">148</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getTranscript' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L153">153</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L153">153</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'accession'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L153">153</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L157">157</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getTranscripts' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L161">161</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L165">165</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'removeTranscript' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L170">170</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L170">170</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'accession'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L170">170</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L176">176</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L184">184</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addTranscript' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L184">184</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L184">184</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'accession'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L184">184</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter begin should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L184">184</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'begin'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L184">184</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L184">184</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'end'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L184">184</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L188">188</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L194">194</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'removeIntron' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L199">199</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L199">199</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'accession'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L199">199</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L210">210</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 97).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L217">217</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addIntron' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L217">217</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L217">217</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'accession'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L217">217</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter begin should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L217">217</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'begin'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L217">217</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L217">217</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'end'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L217">217</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L217">217</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 180).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L222">222</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L229">229</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'removeExon' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L234">234</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L234">234</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'accession'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L234">234</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'try' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L243">243</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L243">243</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'catch' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L245">245</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L245">245</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L246">246</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L254">254</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L261">261</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addExon' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L261">261</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L261">261</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'accession'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L261">261</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter begin should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L261">261</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'begin'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L261">261</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L261">261</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'end'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L261">261</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L261">261</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 108).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L266">266</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L273">273</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getExonSequences' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L277">277</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L281">281</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L283">283</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getIntronSequences' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L285">285</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L289">289</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L290">290</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L291">291</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequence5PrimeTo3Prime' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L294">294</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L295">295</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 136).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L297">297</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 101).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L302">302</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L309">309</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 110).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L311">311</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L311">311</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 123).</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L312">312</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L312">312</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#L312">312</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.IntronSequence.java">org/biojava/nbio/core/sequence/IntronSequence.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/IntronSequence.html#L33">33</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/IntronSequence.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter parentGeneSequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/IntronSequence.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'parentGeneSequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/IntronSequence.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter begin should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/IntronSequence.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'begin'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/IntronSequence.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/IntronSequence.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'end'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/IntronSequence.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/IntronSequence.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/IntronSequence.html#L50">50</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.MultipleSequenceAlignment.java">org/biojava/nbio/core/sequence/MultipleSequenceAlignment.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 115).</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L37">37</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 119).</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;S&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addAlignedSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 110).</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'removeAlignedSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAlignedSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter listIndex should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'listIndex'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAlignedSequences' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundsAt' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter alignmentIndex should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L100">100</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'alignmentIndex'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L100">100</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundSet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L112">112</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L117">117</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L122">122</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSize' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L131">131</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'toString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L141">141</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter width should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L142">142</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'width'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L142">142</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 94).</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L143">143</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L146">146</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'toString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L151">151</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter format should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L152">152</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'format'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L152">152</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 113).</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L157">157</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'60' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L157">157</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L157">157</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'60' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L160">160</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 116).</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L163">163</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'50' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L163">163</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'60' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L170">170</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'toString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L175">175</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L182">182</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 111).</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L194">194</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter width should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L194">194</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'width'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L194">194</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter header should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L194">194</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'header'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L194">194</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter idFormat should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L194">194</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'idFormat'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L194">194</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter interlaced should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L194">194</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'interlaced'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L194">194</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter aligIndices should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L194">194</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'aligIndices'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L194">194</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter aligConservation should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L195">195</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'aligConservation'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L195">195</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter webDisplay should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L195">195</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'webDisplay'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L195">195</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L198">198</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L198">198</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 113).</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L206">206</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L206">206</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 115).</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L216">216</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L216">216</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L228">228</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L231">231</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L233">233</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L236">236</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L248">248</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 111).</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L268">268</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter s should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L268">268</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 's'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L268">268</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter counter should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L268">268</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'counter'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L268">268</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter idFormat should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L268">268</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'idFormat'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L268">268</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L268">268</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'start'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L268">268</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L268">268</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'end'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L268">268</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L269">269</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L281">281</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 104).</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L286">286</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 110).</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L301">301</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter s should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L301">301</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 's'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L301">301</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter idFormat should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L301">301</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'idFormat'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L301">301</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L301">301</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'start'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L301">301</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L301">301</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'end'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L301">301</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter webDisplay should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L301">301</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'webDisplay'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L301">301</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L306">306</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L316">316</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 113).</td><td><a href="./xref/org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#L320">320</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.ProteinSequence.java">org/biojava/nbio/core/sequence/ProteinSequence.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - org.biojava.nbio.core.sequence.compound.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L26">26</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'static' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'logger' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 104).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 104).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 134).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 127).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L62">62</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter seqString should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'seqString'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 123).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter seqString should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'seqString'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter proxyLoader should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L90">90</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'proxyLoader'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L90">90</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 124).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L101">101</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter proxyLoader should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L101">101</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'proxyLoader'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L101">101</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L101">101</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L101">101</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 119).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L120">120</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 113).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L121">121</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 122).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L122">122</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setParentDNASequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter parentDNASequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L122">122</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'parentDNASequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter begin should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L122">122</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'begin'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L122">122</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'end'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L133">133</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addFeature' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L135">135</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 110).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter feature should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 112).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L146">146</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L146">146</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 113).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L147">147</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L147">147</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L149">149</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 150).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L150">150</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L150">150</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L150">150</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L154">154</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L155">155</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L155">155</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accessId should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L155">155</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 135).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L156">156</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L158">158</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L160">160</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L165">165</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L178">178</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter cdna should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L178">178</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 143).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L186">186</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L186">186</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L195">195</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L202">202</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter args should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L202">202</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L203">203</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 206).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L206">206</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L207">207</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/ProteinSequence.html#L208">208</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.RNASequence.java">org/biojava/nbio/core/sequence/RNASequence.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter seqString should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'seqString'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter proxyLoader should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'proxyLoader'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 118).</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter seqString should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L70">70</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'seqString'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L70">70</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'CompoundNotFoundException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L70">70</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter proxyLoader should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'proxyLoader'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getReverseComplement' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L88">88</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L98">98</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getInverse' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L98">98</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L103">103</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getComplement' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L111">111</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getProteinSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L126">126</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getProteinSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter engine should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L126">126</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'engine'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'cast' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getGC' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L130">130</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/RNASequence.html#L130">130</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.SequenceComparator.java">org/biojava/nbio/core/sequence/SequenceComparator.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceComparator.html#L29">29</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceComparator.html#L33">33</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceComparator.html#L35">35</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'compare' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceComparator.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceComparator.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter o1 should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceComparator.html#L37">37</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter o2 should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceComparator.html#L37">37</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.SequenceOptimizationHints.java">org/biojava/nbio/core/sequence/SequenceOptimizationHints.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 103).</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L27">27</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 112).</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L28">28</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 108).</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L29">29</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 121).</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L30">30</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 114).</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L31">31</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 117).</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L32">32</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 118).</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L33">33</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 120).</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L34">34</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Utility classes should not have a public or default constructor.</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter aSequenceUsage should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter aSequenceColection should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L71">71</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'private' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'private' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#L84">84</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.StartCodonSequence.java">org/biojava/nbio/core/sequence/StartCodonSequence.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/StartCodonSequence.html#L27">27</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/StartCodonSequence.html#L32">32</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'parentGeneSequence' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/StartCodonSequence.html#L32">32</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/StartCodonSequence.html#L35">35</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/StartCodonSequence.html#L35">35</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter parentGeneSequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/StartCodonSequence.html#L35">35</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'parentGeneSequence' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/StartCodonSequence.html#L35">35</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter begin should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/StartCodonSequence.html#L35">35</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/StartCodonSequence.html#L35">35</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/StartCodonSequence.html#L35">35</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/StartCodonSequence.html#L41">41</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.StopCodonSequence.java">org/biojava/nbio/core/sequence/StopCodonSequence.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/StopCodonSequence.html#L27">27</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/StopCodonSequence.html#L33">33</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'parentGeneSequence' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/StopCodonSequence.html#L33">33</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/StopCodonSequence.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/StopCodonSequence.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter parentGeneSequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/StopCodonSequence.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'parentGeneSequence' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/StopCodonSequence.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter begin should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/StopCodonSequence.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/StopCodonSequence.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/StopCodonSequence.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/StopCodonSequence.html#L43">43</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.Strand.java">org/biojava/nbio/core/sequence/Strand.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/Strand.html#L32">32</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/Strand.html#L32">32</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/Strand.html#L32">32</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/Strand.html#L33">33</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/Strand.html#L34">34</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/Strand.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter stringRepresentation should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/Strand.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'stringRepresentation' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/Strand.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter numericRepresentation should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/Strand.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'numericRepresentation' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/Strand.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/Strand.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/Strand.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/Strand.html#L49">49</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.TaxonomyID.java">org/biojava/nbio/core/sequence/TaxonomyID.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TaxonomyID.html#L26">26</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TaxonomyID.html#L33">33</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TaxonomyID.html#L34">34</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'dataSource' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TaxonomyID.html#L34">34</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TaxonomyID.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter id should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TaxonomyID.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'id' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TaxonomyID.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter dataSource should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TaxonomyID.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'dataSource' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TaxonomyID.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getID' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TaxonomyID.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getDataSource' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TaxonomyID.html#L51">51</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.TranscriptSequence.java">org/biojava/nbio/core/sequence/TranscriptSequence.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 106).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L42">42</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'static' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L42">42</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'logger' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 115).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter parentDNASequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'parentDNASequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter begin should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'begin'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getStrand' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L73">73</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'removeCDS' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'accession'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L93">93</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCDSSequences' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L101">101</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addCDS' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'accession'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter begin should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'begin'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'end'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter phase should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 112).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L113">113</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L139">139</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getProteinCDSSequences' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L139">139</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L140">140</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 108).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L144">144</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L147">147</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L149">149</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L154">154</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L155">155</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L157">157</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L158">158</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L163">163</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L165">165</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L170">170</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L171">171</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L173">173</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L174">174</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L181">181</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L184">184</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L184">184</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L186">186</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L186">186</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 117).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L187">187</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L187">187</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 111).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L189">189</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 117).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L190">190</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 94).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L191">191</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L192">192</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L198">198</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L202">202</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getDNACodingSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L202">202</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L205">205</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L207">207</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L210">210</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L212">212</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L212">212</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 117).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L213">213</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L213">213</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L219">219</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L223">223</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getProteinSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L223">223</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L227">227</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L232">232</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getProteinSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L232">232</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter engine should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L232">232</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'engine'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L232">232</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L234">234</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L235">235</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L236">236</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L237">237</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getStartCodonSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L244">244</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused @param tag for 'startCodonSequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L249">249</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L251">251</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addStartCodonSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L251">251</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L251">251</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'accession'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L251">251</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter begin should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L251">251</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'begin'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L251">251</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L251">251</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'end'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L251">251</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getStopCodonSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L259">259</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused @param tag for 'stopCodonSequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L264">264</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L266">266</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addStopCodonSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L266">266</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L266">266</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'accession'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L266">266</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter begin should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L266">266</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'begin'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L266">266</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L266">266</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'end'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#L266">266</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.compound.AmbiguityDNACompoundSet.java">org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundSet.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing package-info.java file.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundSet.html#L0"></a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundSet.html#L31">31</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundSet.html#L32">32</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundSet.html#L32">32</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundSet.html#L35">35</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundSet.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundSet.html#L66">66</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.compound.AmbiguityRNACompoundSet.java">org/biojava/nbio/core/sequence/compound/AmbiguityRNACompoundSet.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AmbiguityRNACompoundSet.html#L30">30</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AmbiguityRNACompoundSet.html#L31">31</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AmbiguityRNACompoundSet.html#L31">31</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AmbiguityRNACompoundSet.html#L34">34</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AmbiguityRNACompoundSet.html#L38">38</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.compound.AminoAcidCompound.java">org/biojava/nbio/core/sequence/compound/AminoAcidCompound.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter shortName should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter longName should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter description should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter molecularWeight should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'equals' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#L51">51</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter obj should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'hashCode' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'equalsIgnoreCase' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#L73">73</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundSet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#L87">87</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.compound.AminoAcidCompoundSet.java">org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.util.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L28">28</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 117).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L31">31</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 118).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L32">32</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L34">34</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 115).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L42">42</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 122).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'71.0788f' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 104).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'156.1875f' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 106).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'114.1039f' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L51">51</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 109).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'115.0886f' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 104).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'103.1388f' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 109).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'129.1155f' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'128.1307f' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'57.0519f' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'137.1411f' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 106).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'113.1594f' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 103).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'113.1594f' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'128.1741f' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 106).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'131.1986f' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 109).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'147.1766f' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L62">62</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'97.1167f' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 101).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'87.0782f' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'101.1051f' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 106).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'186.2132f' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L66">66</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 104).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'163.1760f' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 101).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'99.1326f' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 118).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 117).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 112).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 110).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'150.0388f' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 107).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L86">86</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'255.3172f' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L90">90</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getStringForCompound' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundForString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L100">100</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L101">101</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter string should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L101">101</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'3' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L106">106</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L106">106</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 135).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getMaxSingleCompoundStringLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L114">114</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'isCompoundStringLengthEqual' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L120">120</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L121">121</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L125">125</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'static' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'aminoAcidCompoundSet' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L125">125</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'compoundsEquivalent' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L131">131</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 106).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L132">132</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L132">132</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundOne should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L132">132</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundTwo should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L132">132</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L133">133</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getEquivalentCompounds' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L137">137</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L138">138</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L138">138</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L138">138</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Inner assignments should be avoided.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L153">153</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Inner assignments should be avoided.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L154">154</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Inner assignments should be avoided.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L155">155</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 100).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L160">160</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 110).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L165">165</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter one should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter two should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter either should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Inner assignments should be avoided.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L171">171</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Inner assignments should be avoided.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L172">172</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Inner assignments should be avoided.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L173">173</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'hasCompound' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L187">187</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L188">188</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L188">188</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'isValidSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L192">192</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L193">193</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAllCompounds' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L202">202</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L203">203</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L204">204</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'isComplementable' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L208">208</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#L209">209</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.compound.CodonCompound.java">org/biojava/nbio/core/sequence/compound/CodonCompound.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L26">26</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L32">32</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L33">33</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L34">34</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L35">35</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter one should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L37">37</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'one' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter two should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L37">37</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'two' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter three should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'three' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'start' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'equalsIgnoreCase' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'equals' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter obj should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'cast' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'hashCode' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getOne' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getTwo' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getThree' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'isStart' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getDescription' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLongName' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getMolecularWeight' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L103">103</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getShortName' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter description should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L116">116</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter longName should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter molecularWeight should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L128">128</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter shortName should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/CodonCompound.html#L134">134</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.compound.DNACompoundSet.java">org/biojava/nbio/core/sequence/compound/DNACompoundSet.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/DNACompoundSet.html#L29">29</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/DNACompoundSet.html#L31">31</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/DNACompoundSet.html#L32">32</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/DNACompoundSet.html#L35">35</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/DNACompoundSet.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'newNucleotideCompound' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/DNACompoundSet.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 104).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/DNACompoundSet.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter base should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/DNACompoundSet.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter complement should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/DNACompoundSet.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter equivalents should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/DNACompoundSet.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/DNACompoundSet.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/DNACompoundSet.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/DNACompoundSet.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/DNACompoundSet.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/DNACompoundSet.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/DNACompoundSet.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/DNACompoundSet.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/DNACompoundSet.html#L55">55</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.compound.NucleotideCompound.java">org/biojava/nbio/core/sequence/compound/NucleotideCompound.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 111).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter base should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter complementStr should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'complementStr' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 146).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter base should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter complementStr should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'complementStr' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter constituents should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'constituents' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 99).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getShortName' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getComplement' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L70">70</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'equals' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter obj should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'hashCode' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L86">86</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'equalsIgnoreCase' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getConstituents' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L103">103</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L103">103</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L106">106</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getConsituents' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'isAmbiguous' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L113">113</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#L113">113</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.compound.RNACompoundSet.java">org/biojava/nbio/core/sequence/compound/RNACompoundSet.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>File does not end with a newline.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/RNACompoundSet.html#L0"></a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/RNACompoundSet.html#L29">29</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/RNACompoundSet.html#L31">31</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/RNACompoundSet.html#L32">32</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/RNACompoundSet.html#L34">34</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/RNACompoundSet.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'newNucleotideCompound' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/RNACompoundSet.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 104).</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/RNACompoundSet.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter base should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/RNACompoundSet.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter complement should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/RNACompoundSet.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter equivalents should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/compound/RNACompoundSet.html#L48">48</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.edits.Edit.java">org/biojava/nbio/core/sequence/edits/Edit.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing package-info.java file.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L0"></a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'abstract' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'static' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'logger' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'editingSequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'editingSequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'edit' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter editingSequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L90">90</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'start' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L98">98</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L98">98</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'start' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L98">98</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L98">98</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'end' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L98">98</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setStringSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L103">103</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L103">103</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter stringSequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L103">103</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'stringSequence' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L103">103</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequence' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getTargetSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L119">119</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter editingSequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L121">121</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L123">123</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L123">123</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L124">124</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 114).</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L126">126</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L132">132</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getEmptySequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter editingSequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'editingSequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L137">137</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L138">138</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L139">139</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L139">139</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L140">140</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L141">141</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 123).</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L142">142</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L142">142</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L143">143</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getStart' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L147">147</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L147">147</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getEnd' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L151">151</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L151">151</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L156">156</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L161">161</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L161">161</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L165">165</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L165">165</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L165">165</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getRealStart' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L170">170</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L170">170</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getRealEnd' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L174">174</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L174">174</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getFivePrime' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L178">178</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter editingSequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L179">179</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getThreePrime' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L187">187</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter editingSequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L188">188</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L193">193</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L214">214</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L216">216</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L216">216</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L216">216</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L222">222</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L222">222</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L222">222</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L228">228</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L228">228</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L228">228</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter stop should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L228">228</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L234">234</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L234">234</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L234">234</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter stop should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L234">234</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getFivePrime' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L240">240</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter editingSequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L241">241</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getThreePrime' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L256">256</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter editingSequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L257">257</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L269">269</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L290">290</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L290">290</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L290">290</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L295">295</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L295">295</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L295">295</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L306">306</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getEnd' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L306">306</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L306">306</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L306">306</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getFivePrime' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L315">315</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter editingSequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L316">316</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getThreePrime' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L324">324</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter editingSequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L325">325</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L328">328</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L329">329</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L330">330</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/edits/Edit.html#L335">335</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.features.AbstractFeature.java">org/biojava/nbio/core/sequence/features/AbstractFeature.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing package-info.java file.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L0"></a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L34">34</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 97).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L35">35</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;S&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L42">42</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'childrenFeatures' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L42">42</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'parentFeature' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'sequenceLocation' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'type' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'source' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'Qualifiers' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter type should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'type' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'type'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter source should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'source' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'source'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 123).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 127).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 110).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLocations' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 124).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 127).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 110).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setLocation' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter loc should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'loc'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getType' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setType' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L98">98</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter type should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'type' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'type'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L103">103</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSource' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L113">113</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setSource' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L117">117</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter source should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'source' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L118">118</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'source'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 111).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L123">123</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setParentFeature' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter feature should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L128">128</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'feature'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L128">128</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L132">132</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getParentFeature' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L141">141</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L145">145</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getChildrenFeatures' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L145">145</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L150">150</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setChildrenFeatures' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L154">154</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter features should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L155">155</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'features'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L155">155</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getDescription' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L163">163</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L164">164</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setDescription' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L171">171</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L172">172</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter description should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L172">172</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'description' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L172">172</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getShortDescription' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L179">179</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L180">180</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setShortDescription' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L187">187</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L188">188</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter shortDescription should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L188">188</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'shortDescription' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L188">188</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L193">193</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L194">194</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 119).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L198">198</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L201">201</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L201">201</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter e1 should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L201">201</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter e2 should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L201">201</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L211">211</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L211">211</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L213">213</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L213">213</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'else' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L215">215</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 110).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L223">223</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 110).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L227">227</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'public' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L227">227</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L230">230</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L230">230</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter e1 should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L230">230</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter e2 should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L230">230</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 119).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L231">231</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'-' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L231">231</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 121).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L232">232</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L243">243</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L244">244</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 108).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L247">247</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L250">250</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter o1 should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L250">250</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter o2 should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L250">250</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getUserObject' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L258">258</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L259">259</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 112).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L264">264</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 100).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L265">265</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setUserObject' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L268">268</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L269">269</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter userObject should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L269">269</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'userObject' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L269">269</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L272">272</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L273">273</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getQualifiers' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L273">273</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L274">274</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L275">275</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L276">276</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L277">277</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L279">279</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setQualifiers' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L279">279</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L280">280</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter qualifiers should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L280">280</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L281">281</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L282">282</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L283">283</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L283">283</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L284">284</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L286">286</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addQualifier' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L286">286</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L287">287</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter key should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L287">287</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter qualifier should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L287">287</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L288">288</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L289">289</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L290">290</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L290">290</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/AbstractFeature.html#L291">291</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.features.DBReferenceInfo.java">org/biojava/nbio/core/sequence/features/DBReferenceInfo.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused import - org.biojava.nbio.core.sequence.loader.UniprotProxySequenceReader.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L25">25</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter database should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'database' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'database'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter id should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'id' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'id'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addProperty' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter type should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'type'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter value should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'value'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getProperties' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setProperties' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter properties should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'properties' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getDatabase' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setDatabase' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L90">90</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter database should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L90">90</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'database' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L90">90</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getId' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setId' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L104">104</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter id should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L104">104</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'id' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L104">104</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'toString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#L108">108</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.features.DatabaseReferenceInterface.java">org/biojava/nbio/core/sequence/features/DatabaseReferenceInterface.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DatabaseReferenceInterface.html#L28">28</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DatabaseReferenceInterface.html#L35">35</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DatabaseReferenceInterface.html#L35">35</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DatabaseReferenceInterface.html#L35">35</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/DatabaseReferenceInterface.html#L35">35</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.features.FeatureDbReferenceInfo.java">org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L13">13</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 125).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L37">37</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 145).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 100).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter database should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter id should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getShortDescription' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setShortDescription' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter shortDescription should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'shortDescription' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getDescription' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setDescription' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter description should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'description' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLocations' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setLocation' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter loc should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getType' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setType' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter type should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSource' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setSource' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L104">104</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter source should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setParentFeature' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter feature should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L110">110</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getParentFeature' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L114">114</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getChildrenFeatures' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setChildrenFeatures' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L124">124</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter features should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getUserObject' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L129">129</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setUserObject' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L134">134</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter userObject should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L135">135</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'userObject' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L135">135</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getQualifiers' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L139">139</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setQualifiers' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L144">144</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter qualifiers should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L145">145</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'qualifiers' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L145">145</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addQualifier' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L149">149</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter key should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L150">150</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter qualifier should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L150">150</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L154">154</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#L157">157</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.features.FeatureInterface.java">org/biojava/nbio/core/sequence/features/FeatureInterface.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 126).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 126).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 127).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 129).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;S&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'shortDescription'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L71">71</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'description'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 112).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L100">100</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L123">123</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L131">131</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L133">133</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L138">138</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'feature'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L138">138</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L140">140</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L145">145</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L145">145</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L147">147</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L152">152</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L152">152</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L154">154</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L159">159</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'features'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L159">159</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L165">165</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L170">170</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L172">172</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L173">173</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L173">173</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L178">178</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L178">178</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L180">180</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L185">185</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'qualifiers'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L185">185</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L186">186</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L191">191</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'key'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L191">191</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'qualifier'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureInterface.html#L191">191</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.features.FeatureRetriever.java">org/biojava/nbio/core/sequence/features/FeatureRetriever.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureRetriever.html#L27">27</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeatureRetriever.html#L33">33</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.features.FeaturesKeyWordInterface.java">org/biojava/nbio/core/sequence/features/FeaturesKeyWordInterface.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 101).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeaturesKeyWordInterface.html#L28">28</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeaturesKeyWordInterface.html#L34">34</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeaturesKeyWordInterface.html#L34">34</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeaturesKeyWordInterface.html#L35">35</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeaturesKeyWordInterface.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeaturesKeyWordInterface.html#L37">37</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeaturesKeyWordInterface.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeaturesKeyWordInterface.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeaturesKeyWordInterface.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeaturesKeyWordInterface.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/FeaturesKeyWordInterface.html#L39">39</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.features.Qualifier.java">org/biojava/nbio/core/sequence/features/Qualifier.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L22">22</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L31">31</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L31">31</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'value' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L31">31</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L32">32</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L32">32</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'name' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L32">32</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L34">34</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L34">34</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'needsQuotes' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L34">34</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L37">37</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter name should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'name' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'name'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter value should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'value' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'value'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L42">42</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter name should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'name' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'name'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter value should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'value' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'value'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter needsQuotes should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'needsQuotes' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'needsQuotes'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L51">51</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getName' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getValue' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L66">66</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L70">70</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L71">71</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L73">73</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'needsQuotes' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setName' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter name should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'name' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setNeedsQuotes' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter needsQuotes should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'needsQuotes' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L90">90</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L91">91</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setValue' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter value should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'value' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L93">93</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/Qualifier.html#L94">94</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.features.QualityFeature.java">org/biojava/nbio/core/sequence/features/QualityFeature.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>File does not end with a newline.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QualityFeature.html#L0"></a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 110).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QualityFeature.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QualityFeature.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;S&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QualityFeature.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QualityFeature.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter type should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QualityFeature.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'type'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QualityFeature.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter source should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QualityFeature.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'source'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QualityFeature.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getQualities' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QualityFeature.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setQualities' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QualityFeature.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter qualities should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QualityFeature.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'qualities' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QualityFeature.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getQualityAt' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QualityFeature.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioindex should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QualityFeature.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QualityFeature.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getQualities' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QualityFeature.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter biostart should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QualityFeature.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioend should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QualityFeature.html#L77">77</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.features.QuantityFeature.java">org/biojava/nbio/core/sequence/features/QuantityFeature.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QuantityFeature.html#L31">31</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QuantityFeature.html#L32">32</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 111).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QuantityFeature.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QuantityFeature.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;S&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QuantityFeature.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QuantityFeature.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter type should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QuantityFeature.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'type'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QuantityFeature.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter source should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QuantityFeature.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'source'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QuantityFeature.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addQuantity' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QuantityFeature.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter value should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QuantityFeature.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'value'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QuantityFeature.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getQuantities' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QuantityFeature.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setQuantities' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QuantityFeature.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter quantities should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QuantityFeature.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'quantities' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/QuantityFeature.html#L67">67</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.features.TextFeature.java">org/biojava/nbio/core/sequence/features/TextFeature.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/TextFeature.html#L27">27</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 107).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/TextFeature.html#L31">31</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/TextFeature.html#L31">31</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;S&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/TextFeature.html#L31">31</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/TextFeature.html#L33">33</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/TextFeature.html#L33">33</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter type should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/TextFeature.html#L33">33</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter source should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/TextFeature.html#L33">33</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/TextFeature.html#L33">33</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter shortDescription should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/TextFeature.html#L33">33</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/TextFeature.html#L33">33</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter description should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/features/TextFeature.html#L33">33</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.BufferedReaderBytesRead.java">org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing package-info.java file.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L0"></a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused import - java.io.FileReader.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L27">27</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused import - java.io.InputStreamReader.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L29">29</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L34">34</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Array brackets at illegal position.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L73">73</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'8192' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'80' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'bytesRead' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused @throws tag for 'IllegalArgumentException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter in should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'in' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sz should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Inner assignments should be avoided.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L103">103</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter in should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L112">112</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'in' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L112">112</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 114).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L117">117</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L119">119</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getBytesRead' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L122">122</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Array brackets at illegal position.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L157">157</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Inner assignments should be avoided.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L163">163</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'read' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L185">185</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L186">186</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L214">214</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter cbuf should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L214">214</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'cbuf'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L214">214</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter off should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L214">214</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'off'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L214">214</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter len should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L214">214</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'len'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L214">214</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L214">214</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'read' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L292">292</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L293">293</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter cbuf should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L293">293</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Array brackets at illegal position.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L293">293</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter off should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L293">293</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter len should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L293">293</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter ignoreLF should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L335">335</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'readLine' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L419">419</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused @throws tag for 'IllegalArgumentException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L434">434</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'skip' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L437">437</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L438">438</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter n should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L438">438</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L480">480</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'ready' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L480">480</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L481">481</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L510">510</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'markSupported' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L510">510</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L511">511</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused @throws tag for 'IllegalArgumentException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L528">528</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'mark' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L531">531</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L532">532</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter readAheadLimit should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L532">532</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'readAheadLimit' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L532">532</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'reset' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L550">550</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L551">551</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L556">556</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'close' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L564">564</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#L565">565</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.CasePreservingProteinSequenceCreator.java">org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unclosed HTML tag found: &lt;pre&gt;CasePreservingProteinSequenceCreator creator =</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Extra HTML tag found: &lt;/pre&gt;</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 113).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'static' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'logger' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 151).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter proxyLoader should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'proxyLoader'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter index should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'index'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L70">70</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L71">71</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 108).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter index should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 121).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L88">88</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 100).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L90">90</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L91">91</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L93">93</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter list should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L93">93</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'list'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L93">93</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L98">98</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L100">100</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L101">101</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L103">103</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L104">104</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L104">104</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L105">105</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L106">106</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L106">106</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L110">110</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L111">111</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter str should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L111">111</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L112">112</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L113">113</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L113">113</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L113">113</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L113">113</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L113">113</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L113">113</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L113">113</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L113">113</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L114">114</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L116">116</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L118">118</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L119">119</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter args should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 152).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L120">120</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L120">120</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L121">121</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L121">121</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L121">121</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 107).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L123">123</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L123">123</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L124">124</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L125">125</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L128">128</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L129">129</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L129">129</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L130">130</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L130">130</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L131">131</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L131">131</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L132">132</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L133">133</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L133">133</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L134">134</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L135">135</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L135">135</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L137">137</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L137">137</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter seq should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L137">137</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L138">138</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter out should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L138">138</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L139">139</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L140">140</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L141">141</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L141">141</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L141">141</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 103).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L142">142</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L142">142</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L143">143</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L143">143</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L143">143</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 111).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L144">144</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L144">144</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L145">145</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L145">145</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L146">146</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L147">147</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L147">147</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L148">148</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'assert' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L148">148</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L149">149</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L149">149</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'cast' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L149">149</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L150">150</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L151">151</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L152">152</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L153">153</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#L154">154</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.DNASequenceCreator.java">org/biojava/nbio/core/sequence/io/DNASequenceCreator.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 114).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#L51">51</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#L51">51</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter index should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'CompoundNotFoundException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter proxyLoader should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter index should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter list should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'list'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 116).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#L86">86</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.FastaGeneWriter.java">org/biojava/nbio/core/sequence/io/FastaGeneWriter.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - org.biojava.nbio.core.sequence.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L25">25</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 117).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 138).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'static' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'logger' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'showExonUppercase' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'os' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'sequences' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'headerFormat' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'60' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 183).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter os should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'os' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'os'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequences' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequences'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter headerFormat should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'headerFormat' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'headerFormat'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter showExonUppercase should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'showExonUppercase' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'showExonUppercase'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'60' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 199).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter os should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'os' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'os'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequences' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequences'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter headerFormat should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'headerFormat' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'headerFormat'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter showExonUppercase should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'showExonUppercase' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'showExonUppercase'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter lineLength should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'lineLength' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'lineLength'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'process' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L90">90</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L101">101</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 131).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 285).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 107).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L112">112</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLineLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L145">145</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setLineLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L152">152</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter lineLength should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L152">152</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'lineLength' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L152">152</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L156">156</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter args should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L156">156</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 273).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L160">160</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L161">161</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'20' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L161">161</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'10' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L163">163</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'12' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L164">164</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'15' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L164">164</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L165">165</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'20' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L165">165</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'10' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L167">167</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'12' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L168">168</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'15' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L168">168</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 157).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#L173">173</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.FastaReader.java">org/biojava/nbio/core/sequence/io/FastaReader.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.io.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L35">35</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused import - java.util.HashMap.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 101).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;S&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'static' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'logger' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'sequenceCreator' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'headerParser' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'br' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'isr' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L51">51</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'fi' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'fileIndex' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'sequenceIndex' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'line' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'header' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 125).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 139).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter is should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L66">66</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'is'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter headerParser should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L66">66</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'headerParser' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L66">66</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'headerParser'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequenceCreator should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequenceCreator' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequenceCreator'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused @throws tag for 'SecurityException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter file should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'file'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter headerParser should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'headerParser' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'headerParser'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L88">88</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequenceCreator should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequenceCreator' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L88">88</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequenceCreator'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L98">98</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L103">103</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'process' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L116">116</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L123">123</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L128">128</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L129">129</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L129">129</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L133">133</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'process' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L133">133</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L133">133</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter max should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L133">133</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L134">134</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L134">134</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'line' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L137">137</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L137">137</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L138">138</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L138">138</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L138">138</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'header' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L140">140</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L141">141</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L141">141</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L142">142</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L142">142</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L142">142</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L144">144</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L146">146</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L146">146</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L152">152</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L153">153</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L153">153</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L155">155</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L155">155</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L156">156</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L156">156</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L157">157</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 106).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L158">158</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L158">158</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'cast' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L158">158</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L160">160</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L160">160</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L163">163</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L164">164</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 125).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L164">164</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L164">164</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L165">165</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L171">171</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Must have at least one statement.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L174">174</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L176">176</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L177">177</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L177">177</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L185">185</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L185">185</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L187">187</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 101).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L188">188</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L191">191</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L193">193</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L194">194</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L194">194</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'cast' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L194">194</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L195">195</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L196">196</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L196">196</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L197">197</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L197">197</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L199">199</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 117).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L199">199</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L199">199</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L200">200</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L201">201</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L204">204</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&gt;=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L204">204</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&gt;=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L204">204</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L205">205</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L205">205</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L205">205</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L206">206</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L209">209</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L213">213</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'close' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L213">213</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L213">213</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L214">214</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L220">220</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L220">220</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Inner assignments should be avoided.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L220">220</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L221">221</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L223">223</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter args should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L223">223</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 273).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L228">228</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L228">228</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L229">229</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L229">229</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L236">236</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L236">236</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L237">237</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L237">237</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L238">238</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L238">238</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 106).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L239">239</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L239">239</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L239">239</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L240">240</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L241">241</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L241">241</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L242">242</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 104).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L242">242</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L242">242</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L243">243</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L243">243</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L244">244</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L244">244</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L245">245</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 101).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L246">246</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L246">246</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L248">248</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L248">248</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#L249">249</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.FastaReaderHelper.java">org/biojava/nbio/core/sequence/io/FastaReaderHelper.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Utility classes should not have a public or default constructor.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 135).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 137).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 131).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter file should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'file'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter lazySequenceLoad should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'lazySequenceLoad'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L62">62</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter file should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'file'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 101).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L88">88</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter inStream should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'inStream'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 122).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L100">100</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L104">104</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter inStream should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L111">111</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'inStream'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L111">111</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L111">111</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 116).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L112">112</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter file should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L126">126</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'file'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L126">126</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L128">128</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L133">133</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter args should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L133">133</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Array brackets at illegal position.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L133">133</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 120).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L135">135</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#L137">137</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.FastaSequenceParser.java">org/biojava/nbio/core/sequence/io/FastaSequenceParser.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaSequenceParser.html#L29">29</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaSequenceParser.html#L35">35</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaSequenceParser.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaSequenceParser.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bufferedReader should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaSequenceParser.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequenceLength should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaSequenceParser.html#L36">36</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.FastaWriter.java">org/biojava/nbio/core/sequence/io/FastaWriter.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 99).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 104).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;S&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'static' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'logger' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'os' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'sequences' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'headerFormat' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'60' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'lineSep' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 113).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter os should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'os' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'os'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequences' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequences'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter headerFormat should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'headerFormat' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'headerFormat'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L70">70</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 129).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter os should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'os' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'os'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequences' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequences'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter headerFormat should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'headerFormat' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'headerFormat'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter lineLength should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'lineLength' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'lineLength'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 118).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setLineSeparator' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter lineSeparator should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'lineSeparator'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'process' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L93">93</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L93">93</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L130">130</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter args should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L130">130</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 106).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L132">132</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 274).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L135">135</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 127).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L142">142</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L143">143</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L144">144</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 225).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L146">146</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L151">151</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLineLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L163">163</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setLineLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L170">170</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter lineLength should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L170">170</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'lineLength' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#L170">170</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.FastaWriterHelper.java">org/biojava/nbio/core/sequence/io/FastaWriterHelper.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Utility classes should not have a public or default constructor.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter file should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'file'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter proteinSequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'proteinSequences'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter outputStream should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'outputStream'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L71">71</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter proteinSequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'proteinSequences'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 122).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 130).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 136).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter file should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L88">88</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'file'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter geneSequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L88">88</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'geneSequences'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L88">88</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter showExonUppercase should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'showExonUppercase'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L88">88</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 130).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 152).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L103">103</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter outputStream should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L103">103</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'outputStream'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L103">103</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter geneSequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L103">103</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'geneSequences'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L103">103</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L103">103</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter showExonUppercase should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L103">103</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'showExonUppercase'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L103">103</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L103">103</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 100).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L106">106</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L106">106</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L112">112</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 114).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter file should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L119">119</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'file'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter dnaSequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L119">119</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'dnaSequences'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L119">119</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 130).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L134">134</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter outputStream should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L134">134</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'outputStream'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L134">134</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter dnaSequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L134">134</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'dnaSequences'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L134">134</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L134">134</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 116).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L135">135</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L137">137</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L142">142</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L148">148</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter file should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L148">148</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'file'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L148">148</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L148">148</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L148">148</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L148">148</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L156">156</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 104).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L162">162</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter outputStream should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L162">162</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'outputStream'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L162">162</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L162">162</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L162">162</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L162">162</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L167">167</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L172">172</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L172">172</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L172">172</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L172">172</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter outputStream should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L187">187</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L188">188</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L194">194</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L194">194</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L194">194</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#L198">198</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.FileProxyDNASequenceCreator.java">org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 104).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 107).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 109).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 100).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L42">42</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 109).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'compoundSet' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'file' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'sequenceParser' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 118).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter file should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'file' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L62">62</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'file'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequenceParser should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequenceParser' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequenceParser'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 117).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 141).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter index should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'index'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>')' is preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 126).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 111).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter proxyLoader should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L98">98</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'proxyLoader'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L98">98</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter index should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L98">98</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'index'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L98">98</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L102">102</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter list should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'list'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#L109">109</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.FileProxyProteinSequenceCreator.java">org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 104).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 107).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 109).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 104).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 109).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 101).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'compoundSet' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'file' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'sequenceParser' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 118).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 140).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter file should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'file' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'file'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequenceParser should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequenceParser' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequenceParser'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>')' is preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 117).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L73">73</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 139).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter index should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'index'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 111).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L90">90</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter proxyLoader should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'proxyLoader'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter index should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'index'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L101">101</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L106">106</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L106">106</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter list should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'list'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#L108">108</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.GenbankReader.java">org/biojava/nbio/core/sequence/io/GenbankReader.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L13">13</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L17">17</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.io.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused import - java.util.HashMap.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;S&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 125).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 139).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 136).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter is should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'is'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter headerParser should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'headerParser'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequenceCreator should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequenceCreator' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequenceCreator'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L71">71</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused @throws tag for 'SecurityException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L88">88</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter file should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'file'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L88">88</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter headerParser should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'headerParser'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L90">90</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequenceCreator should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L90">90</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequenceCreator' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L90">90</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequenceCreator'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L90">90</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L100">100</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L101">101</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L110">110</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'process' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L110">110</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L111">111</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L111">111</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L112">112</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L118">118</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L124">124</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L130">130</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L131">131</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L131">131</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L134">134</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 99).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'process' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter max should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L137">137</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L137">137</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 142).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L139">139</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 97).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L140">140</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L141">141</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L143">143</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L144">144</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 148).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L145">145</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L149">149</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L151">151</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L152">152</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 100).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L154">154</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L154">154</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L154">154</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L154">154</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L154">154</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L156">156</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L157">157</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L158">158</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L162">162</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'close' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L162">162</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L162">162</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L163">163</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L164">164</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter args should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 277).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L170">170</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L170">170</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L171">171</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L171">171</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 248).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L176">176</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L176">176</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L177">177</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L179">179</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 250).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L182">182</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L182">182</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L183">183</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L183">183</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#L184">184</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.GenbankReaderHelper.java">org/biojava/nbio/core/sequence/io/GenbankReaderHelper.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Utility classes should not have a public or default constructor.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'static' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'logger' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 137).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 139).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 131).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter file should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'file'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter lazySequenceLoad should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'lazySequenceLoad'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'GenbankProxyReader' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 107).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L66">66</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 141).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 137).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 139).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 139).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter file should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'file'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter lazySequenceLoad should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'lazySequenceLoad'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'GenbankProxyReader' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L90">90</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L91">91</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 109).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L93">93</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 103).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 139).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L98">98</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L103">103</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L110">110</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter file should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L111">111</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'file'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L111">111</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L111">111</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 103).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L113">113</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L119">119</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L126">126</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter inStream should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'inStream'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 128).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L128">128</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'GenbankReader' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L128">128</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L130">130</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L131">131</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L135">135</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L141">141</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter inStream should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L142">142</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'inStream'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L142">142</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L142">142</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 122).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L143">143</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'GenbankReader' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L143">143</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L145">145</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L156">156</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter file should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L157">157</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'file'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L157">157</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L157">157</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L159">159</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L164">164</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter args should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L164">164</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Array brackets at illegal position.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L164">164</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 152).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 114).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L168">168</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 166).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L171">171</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L173">173</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#L173">173</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.GenbankSequenceParser.java">org/biojava/nbio/core/sequence/io/GenbankSequenceParser.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L13">13</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L22">22</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 121).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L62">62</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L66">66</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'mapDB' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L86">86</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L88">88</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L90">90</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L93">93</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L98">98</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L100">100</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L101">101</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L102">102</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L103">103</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L104">104</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L104">104</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 158).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L110">110</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'lp' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 97).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L112">112</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L112">112</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'vp' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L112">112</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 94).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L114">114</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L114">114</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'refRange' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L114">114</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 170).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'refp' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'dbxp' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L117">117</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L119">119</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'locationParser' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L120">120</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 129).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L126">126</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'sectp' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L128">128</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L128">128</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'readableFiles' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L128">128</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L129">129</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'headerLine' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L129">129</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L132">132</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L133">133</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method length is 151 lines (max allowed is 150).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L135">135</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L135">135</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bufferedReader should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L135">135</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L152">152</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'5' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L155">155</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L158">158</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L162">162</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L164">164</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L171">171</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'3' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L195">195</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'3' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L196">196</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'5' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L198">198</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'5' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L199">199</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Must have at least one statement.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L204">204</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Must have at least one statement.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L205">205</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Must have at least one statement.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L207">207</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L212">212</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 116).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L220">220</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L225">225</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L233">233</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 106).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L236">236</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L243">243</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L259">259</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'cast' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L261">261</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Must have at least one statement.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L269">269</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L281">281</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L287">287</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L289">289</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L289">289</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L295">295</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bufferedReader should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L295">295</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'320' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L305">305</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L307">307</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L310">310</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L310">310</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L315">315</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L332">332</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'4' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L332">332</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L332">332</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'6' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L333">333</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'4' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L333">333</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L335">335</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L337">337</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L337">337</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'4' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L337">337</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L337">337</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'5' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L338">338</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'3' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L339">339</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L348">348</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'21' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L349">349</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'12' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L349">349</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L362">362</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 101).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L363">363</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bufferedReader should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L363">363</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequenceLength should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L363">363</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getHeader' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L373">373</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L373">373</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequenceHeaderParser' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L377">377</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L377">377</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L381">381</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getDatabaseReferences' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L381">381</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L381">381</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getKeyWords' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L385">385</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L385">385</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L388">388</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getFeatures' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L389">389</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L389">389</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter keyword should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L389">389</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getFeatures' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L392">392</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L392">392</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L395">395</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'parseFeatures' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L396">396</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L396">396</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L396">396</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L397">397</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L398">398</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundType' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L402">402</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#L402">402</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.GenbankWriter.java">org/biojava/nbio/core/sequence/io/GenbankWriter.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L22">22</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;S&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'SEQUENCE_INDENT' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'SEQUENCE_INDENT' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'9' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'os' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'sequences' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'headerFormat' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'60' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L51">51</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter os should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'os' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'os'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequences' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequences'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter headerFormat should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'headerFormat' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'headerFormat'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L62">62</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L66">66</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L70">70</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L71">71</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter os should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L73">73</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'os' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'os'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L73">73</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequences' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L73">73</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequences'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter headerFormat should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'headerFormat' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'headerFormat'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter lineLength should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'lineLength' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'lineLength'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L86">86</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'process' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L88">88</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L90">90</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L93">93</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L100">100</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L100">100</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L101">101</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L101">101</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L102">102</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L102">102</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L103">103</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L103">103</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L104">104</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'seq_len' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L110">110</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 101).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L112">112</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L112">112</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'line_number' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L112">112</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L113">113</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 101).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L114">114</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L114">114</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L116">116</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L116">116</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'10' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L116">116</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L117">117</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'10' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L118">118</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 104).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L120">120</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L120">120</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'10' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L120">120</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L121">121</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L122">122</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L123">123</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L124">124</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L129">129</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L131">131</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L133">133</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L135">135</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L137">137</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L138">138</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L138">138</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L139">139</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L139">139</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L140">140</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L141">141</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L142">142</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L143">143</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L143">143</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L144">144</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L145">145</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L146">146</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L146">146</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L147">147</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L147">147</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L148">148</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L149">149</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L149">149</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L150">150</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L151">151</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L151">151</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L152">152</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L153">153</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L153">153</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L154">154</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L155">155</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L156">156</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L157">157</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L158">158</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L159">159</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L159">159</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L160">160</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L161">161</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L161">161</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L162">162</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L163">163</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L163">163</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L164">164</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L164">164</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L165">165</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L167">167</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L168">168</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L169">169</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L170">170</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLineLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L170">170</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L171">171</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L172">172</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L174">174</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L175">175</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L176">176</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L177">177</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L178">178</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setLineLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L178">178</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter lineLength should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L178">178</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'lineLength' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L178">178</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L179">179</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L180">180</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L181">181</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#L181">181</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.GenbankWriterHelper.java">org/biojava/nbio/core/sequence/io/GenbankWriterHelper.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L22">22</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Utility classes should not have a public or default constructor.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter file should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'file'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter proteinSequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'proteinSequences'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L62">62</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L66">66</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L70">70</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L71">71</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L73">73</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter outputStream should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'outputStream'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter proteinSequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'proteinSequences'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 136).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L90">90</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L91">91</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L93">93</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter file should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'file'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter dnaSequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'dnaSequences'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L98">98</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L98">98</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L100">100</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L101">101</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L102">102</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L104">104</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L104">104</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L105">105</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L106">106</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L106">106</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L112">112</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter outputStream should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L112">112</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'outputStream'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L112">112</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L113">113</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter dnaSequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L113">113</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'dnaSequences'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L113">113</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L113">113</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L114">114</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L117">117</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L119">119</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L120">120</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L121">121</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L123">123</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L124">124</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L126">126</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter outputStream should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'outputStream'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L126">126</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter dnaSequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'dnaSequences'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter seqType should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'seqType'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L128">128</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L128">128</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 169).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L129">129</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L129">129</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L130">130</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L131">131</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 130).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L132">132</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L132">132</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L133">133</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L133">133</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L134">134</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L135">135</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L138">138</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L138">138</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L139">139</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L140">140</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L140">140</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L141">141</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L142">142</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L143">143</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L144">144</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L145">145</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter file should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L145">145</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'file'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L145">145</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L145">145</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L145">145</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L146">146</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L146">146</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L147">147</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L148">148</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L148">148</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L149">149</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L150">150</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L151">151</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L152">152</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L154">154</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L154">154</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L155">155</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L156">156</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L156">156</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L157">157</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L158">158</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L159">159</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L160">160</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L161">161</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter outputStream should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L161">161</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'outputStream'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L161">161</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L162">162</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L162">162</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L162">162</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'Exception'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L162">162</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L163">163</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L164">164</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L167">167</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L167">167</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L168">168</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L169">169</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L170">170</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L172">172</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L172">172</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L173">173</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L173">173</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L173">173</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L174">174</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L174">174</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L175">175</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L176">176</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L177">177</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L179">179</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L180">180</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L181">181</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L181">181</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L182">182</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L183">183</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L183">183</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L184">184</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L185">185</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L186">186</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L187">187</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L188">188</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L189">189</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L190">190</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L191">191</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter outputStream should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L191">191</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L192">192</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L192">192</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 134).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L194">194</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L194">194</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L196">196</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L197">197</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L197">197</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L198">198</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L199">199</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L201">201</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L201">201</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L202">202</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 110).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L204">204</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L204">204</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L205">205</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L207">207</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#L208">208</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.GenericFastaHeaderFormat.java">org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 99).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html#L30">30</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html#L31">31</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html#L32">32</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html#L34">34</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 134).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html#L37">37</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;S&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html#L37">37</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getHeader' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html#L43">43</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.GenericFastaHeaderParser.java">org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 136).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;S&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 101).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'static' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'logger' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter header should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'header'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 123).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L88">88</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'parseHeader' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L111">111</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter header should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L111">111</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'header'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L111">111</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L111">111</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L111">111</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 139).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L113">113</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'3' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L133">133</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'3' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L141">141</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L150">150</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L154">154</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 94).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L155">155</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 156).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L168">168</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L175">175</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter args should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L178">178</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'args'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L178">178</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L184">184</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L186">186</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 126).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L187">187</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L187">187</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L187">187</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L189">189</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L190">190</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L195">195</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 101).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L200">200</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L205">205</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L210">210</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L215">215</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L220">220</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L225">225</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L230">230</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L231">231</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L235">235</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 100).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L240">240</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L245">245</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L250">250</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L255">255</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 103).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#L260">260</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.GenericGenbankHeaderFormat.java">org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>File does not end with a newline.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L0"></a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.util.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L32">32</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L34">34</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L34">34</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L35">35</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L37">37</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'lineSep' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L42">42</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter seqType should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'seqType' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L51">51</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name '_write_single_line' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter tag should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'tag'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter text should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'text'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L66">66</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name '_write_multi_line' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter tag should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'tag'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter text should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'text'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'max_len' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L71">71</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L73">73</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L90">90</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L90">90</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name '_get_date' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 117).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L98">98</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L98">98</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'default_date' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L100">100</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L101">101</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L102">102</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L103">103</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L103">103</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L104">104</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L104">104</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L105">105</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L106">106</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L106">106</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L110">110</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L111">111</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L112">112</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L113">113</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name '_get_data_division' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L116">116</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L118">118</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L119">119</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L120">120</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L120">120</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L121">121</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L121">121</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L122">122</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L123">123</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L123">123</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L124">124</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L124">124</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L125">125</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L128">128</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L128">128</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L129">129</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L129">129</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L130">130</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L130">130</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L131">131</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L131">131</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L132">132</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L132">132</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L133">133</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L134">134</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L134">134</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L135">135</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L135">135</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L137">137</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L137">137</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L138">138</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L138">138</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L139">139</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L139">139</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L140">140</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L140">140</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L141">141</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L141">141</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L142">142</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L142">142</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L143">143</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L144">144</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L144">144</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L145">145</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L146">146</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L147">147</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L149">149</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L150">150</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L151">151</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L151">151</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L152">152</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L153">153</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L154">154</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L155">155</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L155">155</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name '_write_the_first_line' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L155">155</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L155">155</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L155">155</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L156">156</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L157">157</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L157">157</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L158">158</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L159">159</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L159">159</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L160">160</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L161">161</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L162">162</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L163">163</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L164">164</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L165">165</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L167">167</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'16' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L167">167</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L168">168</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L169">169</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L170">170</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L172">172</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L173">173</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'mol_type' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L173">173</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L174">174</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L175">175</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L176">176</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L177">177</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L177">177</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L178">178</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L179">179</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L180">180</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L180">180</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L181">181</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L182">182</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L183">183</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L184">184</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 109).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L185">185</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L185">185</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L186">186</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L188">188</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L189">189</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L189">189</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L190">190</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L191">191</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L192">192</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L193">193</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L194">194</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L194">194</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L195">195</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L196">196</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L197">197</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L198">198</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L198">198</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L199">199</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L200">200</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L201">201</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L202">202</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L202">202</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L203">203</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L203">203</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'16' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L203">203</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L204">204</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L204">204</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 108).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L205">205</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L205">205</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'11' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L205">205</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 112).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L206">206</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L206">206</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'6' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L206">206</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L207">207</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L208">208</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L209">209</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L210">210</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L211">211</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L212">212</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L213">213</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L213">213</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L214">214</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L214">214</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L215">215</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L215">215</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L216">216</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L216">216</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L217">217</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L217">217</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L218">218</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L218">218</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L219">219</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L219">219</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L220">220</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L220">220</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L221">221</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L221">221</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L222">222</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L222">222</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L223">223</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L224">224</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L225">225</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L225">225</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L226">226</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L226">226</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L227">227</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L227">227</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L228">228</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L228">228</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L229">229</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L229">229</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L230">230</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L230">230</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L231">231</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L231">231</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L232">232</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L232">232</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L233">233</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L233">233</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L234">234</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L234">234</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L235">235</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L236">236</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L237">237</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L239">239</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L240">240</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L240">240</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L241">241</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L241">241</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L242">242</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L242">242</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L243">243</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L243">243</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L244">244</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L245">245</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L245">245</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L246">246</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L247">247</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L248">248</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L248">248</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name '_write_comment' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L248">248</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L248">248</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L248">248</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L249">249</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L250">250</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L250">250</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L251">251</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L252">252</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L253">253</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L255">255</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L256">256</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L258">258</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getHeader' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L258">258</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L259">259</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L259">259</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L260">260</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L261">261</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'acc_with_version' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L261">261</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L262">262</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L263">263</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L264">264</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L265">265</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L266">266</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L267">267</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L268">268</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L269">269</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L270">270</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L271">271</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L271">271</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L272">272</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L273">273</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L274">274</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L275">275</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L276">276</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L278">278</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L279">279</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L280">280</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L280">280</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L281">281</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L282">282</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L282">282</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L283">283</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L284">284</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L285">285</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L285">285</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L286">286</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L286">286</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L287">287</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L287">287</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L288">288</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L288">288</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L289">289</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L289">289</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L290">290</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L290">290</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L291">291</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L291">291</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L292">292</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L292">292</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L293">293</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L293">293</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L294">294</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L295">295</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L297">297</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L299">299</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L300">300</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L300">300</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L301">301</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L302">302</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L302">302</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L303">303</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L304">304</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L305">305</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L305">305</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L306">306</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L306">306</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L307">307</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L308">308</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L310">310</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L312">312</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L313">313</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L313">313</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L314">314</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L314">314</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L315">315</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L316">316</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L317">317</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L317">317</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L318">318</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L318">318</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L319">319</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L319">319</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L320">320</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L321">321</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L322">322</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L322">322</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L323">323</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L323">323</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L324">324</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L325">325</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L326">326</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L327">327</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L329">329</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L330">330</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'rec_length' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L330">330</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L331">331</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L332">332</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L333">333</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L334">334</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L336">336</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#L337">337</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.GenericGenbankHeaderParser.java">org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L15">15</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 138).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L34">34</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L34">34</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L34">34</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L37">37</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'parseHeader' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter header should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'header'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setVersion' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter version should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'version' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant throws: 'ParserException' is unchecked exception.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L86">86</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setAccession' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L91">91</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'accession' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L91">91</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant throws: 'ParserException' is unchecked exception.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setDescription' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter description should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'description' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant throws: 'ParserException' is unchecked exception.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L100">100</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 108).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L100">100</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'!=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L100">100</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'!=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L100">100</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L103">103</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L104">104</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setIdentifier' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter identifier should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'identifier' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant throws: 'ParserException' is unchecked exception.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 106).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L110">110</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'!=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'!=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L110">110</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L113">113</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setName' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L117">117</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter name should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'name' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L117">117</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant throws: 'ParserException' is unchecked exception.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L118">118</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L118">118</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L119">119</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 94).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'!=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L119">119</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'!=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L122">122</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setComment' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter comment should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L126">126</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant throws: 'ParserException' is unchecked exception.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#L127">127</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.GenericInsdcHeaderFormat.java">org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L22">22</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;S&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'lineSep' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L51">51</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name '_write_feature_qualifier' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter key should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'key'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter value should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'value'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter quote should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'quote'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L62">62</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'while' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L71">71</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 111).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'-' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'-' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L86">86</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L88">88</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 100).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L90">90</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L91">91</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L93">93</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name '_wrap_location' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter location should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'location'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L98">98</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L100">100</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L101">101</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L102">102</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L103">103</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L103">103</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L104">104</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L106">106</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 132).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L111">111</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L112">112</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L113">113</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L114">114</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 110).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L114">114</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L114">114</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method '_write_feature' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L114">114</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name '_write_feature' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L114">114</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter feature should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L114">114</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'feature'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L114">114</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter record_length should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L114">114</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'record_length'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L114">114</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'record_length' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L114">114</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L116">116</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'f_type' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L116">116</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L118">118</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 119).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L120">120</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L120">120</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L121">121</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L122">122</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L123">123</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L124">124</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L124">124</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 101).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L125">125</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L126">126</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L128">128</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L141">141</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L142">142</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L143">143</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L144">144</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L155">155</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L163">163</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L164">164</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L165">165</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 124).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name '_insdc_feature_location_string' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter feature should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'feature'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter record_length should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'record_length'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'record_length' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L167">167</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L167">167</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L168">168</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 136).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L169">169</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L169">169</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L170">170</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L170">170</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L170">170</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L171">171</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L172">172</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L172">172</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L172">172</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L173">173</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L174">174</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L175">175</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L176">176</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L177">177</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L178">178</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L179">179</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L180">180</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L180">180</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L181">181</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L181">181</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L182">182</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L182">182</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L182">182</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L183">183</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L183">183</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L183">183</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L184">184</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L185">185</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L185">185</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L185">185</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 176).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L186">186</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L186">186</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L187">187</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L188">188</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L189">189</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L190">190</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L191">191</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L192">192</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L193">193</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L193">193</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L194">194</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 106).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L195">195</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L195">195</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L195">195</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 135).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L196">196</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L196">196</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L197">197</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L198">198</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L198">198</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 111).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L199">199</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L199">199</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L200">200</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L201">201</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L202">202</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L203">203</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L204">204</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L204">204</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L205">205</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L206">206</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L206">206</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L207">207</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L208">208</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L208">208</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L208">208</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 127).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L209">209</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L209">209</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L210">210</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L211">211</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L211">211</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L212">212</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L212">212</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L213">213</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L214">214</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L215">215</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L216">216</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L218">218</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L218">218</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name '_insdc_location_string_ignoring_strand_and_subfeatures' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L218">218</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L219">219</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L219">219</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 107).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L220">220</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequenceLocation should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L220">220</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L221">221</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter record_length should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L221">221</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'record_length' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L221">221</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L222">222</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L223">223</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L229">229</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 168).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L230">230</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L230">230</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L230">230</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L231">231</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L232">232</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L233">233</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L233">233</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L233">233</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L234">234</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L235">235</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L236">236</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L237">237</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L238">238</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L238">238</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L238">238</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L239">239</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L240">240</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L241">241</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L242">242</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L243">243</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L244">244</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L245">245</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L245">245</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L245">245</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 149).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L246">246</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L246">246</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L247">247</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L248">248</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L249">249</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L249">249</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L250">250</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L251">251</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 200).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L252">252</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L252">252</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L252">252</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L253">253</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L254">254</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L255">255</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L256">256</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L256">256</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L257">257</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L257">257</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L258">258</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L259">259</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L260">260</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L260">260</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L260">260</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 109).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L261">261</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L261">261</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L261">261</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L262">262</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 110).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L263">263</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L263">263</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L263">263</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L264">264</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L264">264</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L265">265</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L265">265</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L266">266</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L266">266</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L267">267</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L268">268</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L268">268</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L268">268</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 167).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L269">269</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L269">269</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L270">270</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L271">271</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L272">272</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L272">272</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L273">273</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L274">274</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L274">274</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L275">275</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L276">276</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L276">276</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L276">276</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 171).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L277">277</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L277">277</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L278">278</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L279">279</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L280">280</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 104).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L280">280</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L280">280</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L281">281</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L282">282</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L283">283</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 167).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L284">284</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L284">284</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L285">285</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L286">286</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L287">287</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L287">287</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name '_insdc_feature_position_string' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L287">287</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter location should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L287">287</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L288">288</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L289">289</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L290">290</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L291">291</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L291">291</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L292">292</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L293">293</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L294">294</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L295">295</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L296">296</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L297">297</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L297">297</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L297">297</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name '_insdc_feature_position_string' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L297">297</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter location should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L297">297</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'location'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L297">297</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter increment should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L297">297</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'increment'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L297">297</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L298">298</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L299">299</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L299">299</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L300">300</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L300">300</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L301">301</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L302">302</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L303">303</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L305">305</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L306">306</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L326">326</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L327">327</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L328">328</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L328">328</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L329">329</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L330">330</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L331">331</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L334">334</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L335">335</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L336">336</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L337">337</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L337">337</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L337">337</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method '_split_multi_line' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L337">337</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name '_split_multi_line' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L337">337</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter text should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L337">337</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'text'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L337">337</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter max_len should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L337">337</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'max_len'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L337">337</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'max_len' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L337">337</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L338">338</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L339">339</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L340">340</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L341">341</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L341">341</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L342">342</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L343">343</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L344">344</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L345">345</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L345">345</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L346">346</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L347">347</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L348">348</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'while' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L348">348</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L349">349</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 107).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L350">350</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L350">350</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'while' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L350">350</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L351">351</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L352">352</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L353">353</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L354">354</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L355">355</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L356">356</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L357">357</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#L358">358</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.IUPACParser.java">org/biojava/nbio/core/sequence/io/IUPACParser.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>File does not end with a newline.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L0"></a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.util.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L34">34</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 94).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L90">90</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter is should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'is' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L105">105</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'is'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L112">112</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getTables' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L112">112</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L122">122</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getTable' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter name should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L122">122</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'name'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L130">130</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getTable' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L130">130</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter id should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L130">130</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'id'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L130">130</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L135">135</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L139">139</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L146">146</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Inner assignments should be avoided.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L151">151</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Inner assignments should be avoided.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L151">151</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Inner assignments should be avoided.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L151">151</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Inner assignments should be avoided.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L151">151</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Inner assignments should be avoided.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L151">151</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Inner assignments should be avoided.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L156">156</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Inner assignments should be avoided.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L156">156</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Inner assignments should be avoided.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L156">156</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Inner assignments should be avoided.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L156">156</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Inner assignments should be avoided.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L156">156</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L158">158</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L163">163</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L169">169</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L172">172</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L175">175</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L186">186</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L193">193</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L194">194</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L195">195</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L196">196</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L197">197</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L198">198</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L199">199</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L201">201</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L202">202</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L204">204</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter name should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L204">204</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'name' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L204">204</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter id should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L204">204</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'id' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L204">204</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter aminoAcidString should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L204">204</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'aminoAcidString' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L204">204</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter startCodons should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L205">205</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'startCodons' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L205">205</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter baseOne should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L205">205</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'baseOne' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L205">205</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter baseTwo should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L205">205</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'baseTwo' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L205">205</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter baseThree should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L205">205</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'baseThree' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L205">205</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter name should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L220">220</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'name' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L220">220</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'name'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L220">220</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter id should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L220">220</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'id' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L220">220</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'id'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L220">220</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter aminoAcidString should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L220">220</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'aminoAcidString' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L220">220</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'aminoAcidString'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L220">220</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter startCodons should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L221">221</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'startCodons' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L221">221</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'startCodons'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L221">221</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getId' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L228">228</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L228">228</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getName' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L232">232</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L232">232</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L244">244</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'isStart' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L244">244</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L245">245</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L245">245</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compound'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L245">245</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant throws: 'IllegalStateException' is unchecked exception.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L245">245</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L246">246</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 112).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L247">247</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L249">249</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L250">250</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L251">251</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L252">252</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L264">264</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused @param tag for 'nucleotides'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L264">264</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L268">268</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCodons' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L268">268</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter nucelotides should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L269">269</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'nucelotides'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L269">269</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter aminoAcids should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L270">270</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L280">280</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L281">281</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L282">282</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 101).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L287">287</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L294">294</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L294">294</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compounds' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L294">294</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L295">295</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter nucelotides should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L295">295</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L302">302</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L311">311</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L314">314</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCodonCompoundSet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L314">314</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L315">315</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'rnaCompounds'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L316">316</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'aminoAcidCompounds'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L317">317</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L330">330</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'codons' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L331">331</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L342">342</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L346">346</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L350">350</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter string should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/IUPACParser.html#L350">350</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.PlainFastaHeaderParser.java">org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L31">31</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L35">35</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;S&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L42">42</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter header should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'header'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'parseHeader' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter header should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'header'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 97).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#L69">69</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.ProteinSequenceCreator.java">org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter index should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'CompoundNotFoundException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#L62">62</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#L70">70</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter list should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'list'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 114).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter proxyLoader should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'proxyLoader'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#L86">86</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter index should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#L86">86</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.RNASequenceCreator.java">org/biojava/nbio/core/sequence/io/RNASequenceCreator.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#L62">62</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 119).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter index should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'index'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'CompoundNotFoundException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter proxyLoader should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'proxyLoader'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter index should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'index'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter list should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'list'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#L83">83</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.template.FastaHeaderFormatInterface.java">org/biojava/nbio/core/sequence/io/template/FastaHeaderFormatInterface.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing package-info.java file.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/FastaHeaderFormatInterface.html#L0"></a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/FastaHeaderFormatInterface.html#L32">32</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/FastaHeaderFormatInterface.html#L32">32</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;S&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/FastaHeaderFormatInterface.html#L32">32</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/FastaHeaderFormatInterface.html#L34">34</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/FastaHeaderFormatInterface.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/FastaHeaderFormatInterface.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/FastaHeaderFormatInterface.html#L38">38</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.template.GenbankHeaderFormatInterface.java">org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html#L22">22</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html#L31">31</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html#L33">33</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html#L33">33</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;S&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html#L33">33</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html#L34">34</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html#L35">35</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html#L35">35</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html#L41">41</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.template.SequenceCreatorInterface.java">org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 122).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'index'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'CompoundNotFoundException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'proxyLoader'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'index'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'list'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#L61">61</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.template.SequenceHeaderParserInterface.java">org/biojava/nbio/core/sequence/io/template/SequenceHeaderParserInterface.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceHeaderParserInterface.html#L32">32</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceHeaderParserInterface.html#L32">32</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;S&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceHeaderParserInterface.html#L32">32</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceHeaderParserInterface.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'header'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceHeaderParserInterface.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceHeaderParserInterface.html#L38">38</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.template.SequenceParserInterface.java">org/biojava/nbio/core/sequence/io/template/SequenceParserInterface.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceParserInterface.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 99).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceParserInterface.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceParserInterface.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bufferedReader'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceParserInterface.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceParserInterface.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequenceLength'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceParserInterface.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/template/SequenceParserInterface.html#L40">40</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.util.ClasspathResource.java">org/biojava/nbio/core/sequence/io/util/ClasspathResource.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing package-info.java file.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L0"></a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.io.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L26">26</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter location should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'location' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter location should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'location' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter preCache should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'preCache' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter location should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'location' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter preCache should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'preCache' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter isGzip should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'isGzip' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L91">91</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getInputStream' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L98">98</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L101">101</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getBufferedReader' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L101">101</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getList' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L112">112</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 94).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L114">114</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L116">116</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L116">116</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L117">117</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L125">125</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L130">130</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L135">135</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L138">138</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L151">151</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L152">152</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#L154">154</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.io.util.IOUtils.java">org/biojava/nbio/core/sequence/io/util/IOUtils.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.io.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L33">33</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Utility classes should not have a public or default constructor.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L42">42</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter c should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L51">51</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter input should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter output should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L86">86</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 111).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter br should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter processor should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant throws: 'ParserException' is unchecked exception.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L90">90</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'while' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>')' is preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L93">93</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'catch' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L98">98</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L98">98</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L100">100</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L101">101</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L102">102</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L104">104</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L104">104</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L106">106</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L111">111</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L111">111</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter br should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L111">111</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant throws: 'ParserException' is unchecked exception.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L111">111</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L112">112</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L113">113</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L114">114</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter line should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L116">116</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L117">117</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L120">120</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L123">123</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L123">123</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L124">124</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L126">126</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L128">128</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L129">129</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L129">129</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L130">130</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L131">131</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L132">132</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L132">132</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter is should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L132">132</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant throws: 'ParserException' is unchecked exception.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L132">132</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L133">133</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L134">134</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L137">137</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L138">138</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L139">139</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L140">140</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L141">141</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L142">142</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L143">143</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L144">144</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L145">145</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L145">145</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L146">146</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L147">147</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L148">148</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter file should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L148">148</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant throws: 'ParserException' is unchecked exception.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L148">148</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L149">149</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L150">150</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L152">152</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L153">153</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L154">154</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L155">155</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L156">156</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L157">157</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L158">158</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L159">159</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L160">160</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L160">160</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L161">161</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L162">162</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L163">163</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter file should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L163">163</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant throws: 'ParserException' is unchecked exception.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L163">163</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L164">164</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L165">165</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L165">165</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L167">167</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L168">168</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L169">169</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L170">170</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L170">170</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L171">171</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L171">171</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L172">172</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L172">172</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L173">173</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L174">174</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L175">175</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L176">176</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L176">176</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L177">177</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'catch' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L177">177</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L178">178</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L178">178</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L178">178</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L178">178</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L178">178</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L178">178</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L179">179</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L180">180</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L181">181</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L183">183</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L184">184</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L185">185</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L185">185</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L186">186</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L187">187</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L188">188</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L189">189</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L190">190</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'static' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L190">190</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L191">191</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L191">191</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L192">192</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L194">194</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L194">194</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L195">195</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L196">196</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L197">197</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L198">198</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L199">199</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L200">200</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L200">200</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;S&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L200">200</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;C&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L200">200</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L200">200</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L201">201</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L202">202</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L203">203</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L204">204</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L205">205</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'10000' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L205">205</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L206">206</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L208">208</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L208">208</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L209">209</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L210">210</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L211">211</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L212">212</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L213">213</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L214">214</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L214">214</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;S&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L214">214</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;C&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L214">214</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L214">214</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L215">215</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L216">216</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L217">217</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L218">218</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L219">219</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L220">220</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'57' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L220">220</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L221">221</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L222">222</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L223">223</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L224">224</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'10000' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L224">224</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L225">225</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L227">227</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L227">227</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L228">228</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L229">229</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L230">230</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L231">231</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L232">232</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 106).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L233">233</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L233">233</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;S&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L233">233</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;C&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L233">233</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L233">233</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L234">234</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L235">235</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 103).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L236">236</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L236">236</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 109).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L237">237</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L237">237</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 127).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L238">238</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L238">238</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L239">239</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 100).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L240">240</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L240">240</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L241">241</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L242">242</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L243">243</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L244">244</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L245">245</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L246">246</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L248">248</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L248">248</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L249">249</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L250">250</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L251">251</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L252">252</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L253">253</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L253">253</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;C&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L253">253</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter cs should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L253">253</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 126).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L254">254</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L254">254</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L254">254</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>':' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L254">254</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 132).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L255">255</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L255">255</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L255">255</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L256">256</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L258">258</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L258">258</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L259">259</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L260">260</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L261">261</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L262">262</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L263">263</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L264">264</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L264">264</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;S&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L264">264</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;C&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L264">264</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L264">264</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L265">265</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L266">266</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 123).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L267">267</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L267">267</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L267">267</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L268">268</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L269">269</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L269">269</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L270">270</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L272">272</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L272">272</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L273">273</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L274">274</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L275">275</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L276">276</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L277">277</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L278">278</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L279">279</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L280">280</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L281">281</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L282">282</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L282">282</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter web should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L282">282</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter c1 should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L282">282</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter c2 should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L282">282</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter similar should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L282">282</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter c should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L282">282</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L283">283</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L284">284</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L285">285</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L287">287</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L287">287</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L288">288</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L289">289</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L290">290</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L291">291</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L292">292</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L293">293</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L294">294</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L295">295</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 97).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L296">296</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L296">296</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter web should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L296">296</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter c1 should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L296">296</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter c2 should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L296">296</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter similar should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L296">296</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L297">297</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L297">297</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L297">297</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L298">298</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L300">300</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 122).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L301">301</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L301">301</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L301">301</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>More than 7 parameters (found 8).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L301">301</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter web should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L301">301</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter c1 should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L301">301</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter c2 should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L301">301</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter similar should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L301">301</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter m should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L301">301</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sm should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L301">301</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter dm should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L301">301</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L302">302</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter qg should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L302">302</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L303">303</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L303">303</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L304">304</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 107).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L304">304</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L304">304</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L304">304</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L305">305</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L305">305</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L306">306</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L306">306</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L306">306</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L307">307</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L307">307</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L308">308</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L308">308</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L308">308</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'else' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L309">309</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L309">309</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L310">310</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L310">310</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L310">310</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L311">311</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L313">313</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L313">313</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L314">314</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L314">314</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L315">315</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L316">316</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L317">317</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L318">318</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L319">319</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L320">320</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L321">321</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L322">322</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L323">323</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L323">323</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L323">323</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L324">324</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L324">324</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L325">325</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L325">325</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L326">326</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 111).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L326">326</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L326">326</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L327">327</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L328">328</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L329">329</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L330">330</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L332">332</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L333">333</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 116).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L334">334</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L334">334</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L335">335</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L336">336</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter string should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L336">336</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'string'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L336">336</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter file should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L336">336</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'file'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L336">336</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L337">337</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L338">338</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L339">339</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L339">339</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L340">340</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L341">341</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L342">342</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L343">343</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L344">344</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L344">344</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L345">345</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#L346">346</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.loader.ArrayListProxySequenceReader.java">org/biojava/nbio/core/sequence/loader/ArrayListProxySequenceReader.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing package-info.java file.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/ArrayListProxySequenceReader.html#L0"></a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/ArrayListProxySequenceReader.html#L34">34</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.loader.GenbankProxySequenceReader.java">org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L13">13</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - org.biojava.nbio.core.sequence.features.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L35">35</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.io.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 180).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 103).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'static' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'logger' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'eutilBaseURL' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L70">70</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L71">71</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter genbankDirectoryCache should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'genbankDirectoryCache' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'genbankDirectoryCache'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accessionID should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'accessionID'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 110).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>')' is preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 103).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 153).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L93">93</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 128).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L101">101</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L101">101</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accessionID should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L101">101</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter db should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L101">101</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L103">103</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L104">104</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L114">114</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L119">119</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter in should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L119">119</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter f should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'1024' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L121">121</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accessionID should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter db should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 118).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L137">137</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L144">144</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getGenbankDirectoryCache' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L149">149</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setGenbankDirectoryCache' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L156">156</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter genbankDirectoryCache should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L156">156</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'genbankDirectoryCache' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L156">156</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'genbankDirectoryCache'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L156">156</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getHeader' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L170">170</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getHeaderParser' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L170">170</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L170">170</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getFeatures' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L173">173</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L173">173</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getDatabaseReferences' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L177">177</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L178">178</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getKeyWords' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L182">182</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L187">187</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter args should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L187">187</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 137).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L190">190</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L191">191</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 110).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L192">192</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 163).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L193">193</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'10' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L193">193</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L195">195</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 126).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L199">199</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L201">201</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 151).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L203">203</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'10' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L203">203</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L205">205</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 126).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L208">208</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L210">210</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 151).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L211">211</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'10' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L211">211</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L213">213</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 125).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L216">216</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L218">218</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 151).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L219">219</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'10' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L219">219</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L221">221</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 127).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L224">224</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L226">226</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 151).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L227">227</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'10' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L227">227</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L229">229</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 123).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L232">232</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L234">234</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 151).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L235">235</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'10' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L235">235</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L237">237</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 127).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L240">240</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L242">242</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 151).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L243">243</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'10' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L243">243</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L245">245</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 125).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L248">248</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L250">250</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 151).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L251">251</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'10' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L251">251</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L253">253</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L256">256</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L259">259</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 134).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L260">260</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L262">262</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 134).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L263">263</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L264">264</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L264">264</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L265">265</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 134).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L266">266</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L267">267</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L267">267</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L268">268</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 134).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L269">269</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L270">270</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L270">270</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L271">271</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 134).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L272">272</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L273">273</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L273">273</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L274">274</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 134).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L275">275</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L276">276</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L276">276</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L277">277</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 134).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L278">278</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L279">279</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L279">279</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L280">280</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 134).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#L281">281</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.loader.SequenceFileProxyLoader.java">org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - org.biojava.nbio.core.sequence.template.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L33">33</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 94).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'sequenceParser' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'file' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'sequenceStartIndex' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L62">62</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'sequenceLength' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 110).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L73">73</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 159).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter file should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'file' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequenceParser should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequenceParser' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequenceStartIndex should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequenceStartIndex' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequenceLength should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequenceLength' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setCompoundSet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L90">90</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L91">91</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L91">91</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'CompoundNotFoundException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L101">101</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L102">102</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L103">103</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L104">104</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L105">105</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setContents' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L114">114</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'CompoundNotFoundException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 155).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L140">140</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L140">140</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L141">141</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L150">150</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundAt' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L150">150</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L151">151</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L151">151</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'position'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L151">151</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L152">152</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L161">161</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getIndexOf' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L161">161</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L162">162</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L162">162</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compound'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L162">162</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L163">163</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L172">172</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLastIndexOf' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L172">172</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L173">173</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L173">173</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compound'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L173">173</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L174">174</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L182">182</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'toString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L182">182</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L183">183</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L184">184</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L192">192</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequenceAsString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L192">192</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L193">193</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L204">204</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L204">204</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequenceAsString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L204">204</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioBegin should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L204">204</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioBegin'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L204">204</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioEnd should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L204">204</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioEnd'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L204">204</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L204">204</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'strand'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L204">204</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L205">205</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L206">206</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 119).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L207">207</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L214">214</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAsList' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L214">214</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L215">215</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L216">216</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L227">227</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSubSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L227">227</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L228">228</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L228">228</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioBegin'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L228">228</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioEnd'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L228">228</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L229">229</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L230">230</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L237">237</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'iterator' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L237">237</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L238">238</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L239">239</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L247">247</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundSet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L247">247</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L248">248</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L256">256</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAccession' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L256">256</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L257">257</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L266">266</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'countCompounds' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L266">266</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L267">267</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L267">267</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compounds'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L267">267</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L275">275</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getInverse' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#L275">275</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.loader.StringProxySequenceReader.java">org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - org.biojava.nbio.core.sequence.template.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L32">32</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 94).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 117).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequence' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setCompoundSet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setContents' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequence' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 155).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setContents' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequence' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter features should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L90">90</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundAt' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getIndexOf' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L100">100</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L100">100</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLastIndexOf' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L104">104</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L105">105</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'toString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L110">110</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L111">111</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequenceAsString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L116">116</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAsList' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L120">120</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L121">121</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequenceAsString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioBegin should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioEnd should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L128">128</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 119).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L129">129</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSubSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L132">132</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L133">133</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L133">133</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L134">134</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L134">134</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L134">134</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'iterator' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L137">137</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L138">138</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundSet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L142">142</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L143">143</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L147">147</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAccession' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L148">148</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L149">149</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L153">153</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'countCompounds' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L154">154</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L155">155</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L155">155</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getInverse' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#L159">159</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.loader.UniprotProxySequenceReader.java">org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - org.biojava.nbio.core.sequence.template.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.io.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 136).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 143).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 130).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 149).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 103).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'static' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'logger' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 99).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 119).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'uniprotDoc' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 94).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 131).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'accession'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'CompoundNotFoundException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 141).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L105">105</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 119).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L111">111</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter document should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L111">111</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'document'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L111">111</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L111">111</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L111">111</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L111">111</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'CompoundNotFoundException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L111">111</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L112">112</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L113">113</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L114">114</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L124">124</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 132).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L125">125</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;C&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter xml should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L125">125</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'xml'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L125">125</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 106).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setCompoundSet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L135">135</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L140">140</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L141">141</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L143">143</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setContents' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L145">145</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L146">146</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L146">146</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L146">146</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L146">146</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequence' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L146">146</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L146">146</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 155).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L154">154</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L159">159</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L159">159</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L159">159</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L159">159</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L159">159</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L167">167</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L171">171</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L171">171</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L172">172</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L181">181</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundAt' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L181">181</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L182">182</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L182">182</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'position'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L182">182</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L191">191</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getIndexOf' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L191">191</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L192">192</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L192">192</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compound'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L192">192</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L201">201</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLastIndexOf' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L201">201</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L202">202</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L202">202</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compound'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L202">202</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L210">210</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'toString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L210">210</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L219">219</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequenceAsString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L219">219</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L220">220</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L228">228</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAsList' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L228">228</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L229">229</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L237">237</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getInverse' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L237">237</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L249">249</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L249">249</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequenceAsString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L249">249</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioBegin should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L249">249</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioBegin'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L249">249</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioEnd should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L249">249</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioEnd'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L249">249</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L249">249</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'strand'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L249">249</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L250">250</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 119).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L251">251</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L260">260</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSubSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L260">260</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L261">261</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L261">261</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioBegin'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L261">261</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioEnd'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L261">261</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L262">262</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L269">269</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'iterator' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L269">269</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L270">270</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L278">278</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundSet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L278">278</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L279">279</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L287">287</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAccession' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L287">287</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L288">288</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L295">295</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L296">296</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L297">297</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L304">304</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L307">307</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L309">309</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L309">309</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L309">309</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAccessions' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L309">309</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L315">315</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 100).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L316">316</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 100).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L318">318</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L330">330</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'countCompounds' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L330">330</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L331">331</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L331">331</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compounds'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L331">331</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L341">341</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 100).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L341">341</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L341">341</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'accession'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L341">341</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L341">341</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'CompoundNotFoundException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L341">341</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L342">342</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L343">343</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L344">344</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L347">347</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L350">350</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L350">350</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L353">353</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 101).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L354">354</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L359">359</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L359">359</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L359">359</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L360">360</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L360">360</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 116).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L366">366</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L376">376</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L376">376</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L376">376</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sb should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L376">376</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L376">376</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 99).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L377">377</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L377">377</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L378">378</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L381">381</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L383">383</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L383">383</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter uniprotURL should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L383">383</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L384">384</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L385">385</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L385">385</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L386">386</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L387">387</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L388">388</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'5' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L388">388</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L389">389</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L390">390</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'while' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L390">390</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 107).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L391">391</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L391">391</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L392">392</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L392">392</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L393">393</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L394">394</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L395">395</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'200' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L395">395</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L396">396</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L397">397</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L398">398</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L399">399</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L401">401</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L402">402</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L403">403</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L404">404</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L405">405</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L406">406</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L407">407</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L408">408</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L409">409</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 157).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L410">410</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L410">410</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L411">411</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L413">413</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L414">414</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 118).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L415">415</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L415">415</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L416">416</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L417">417</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L418">418</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L419">419</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter key should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L419">419</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'key'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L419">419</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L420">420</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'FileNotFoundException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L420">420</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant throws: 'FileNotFoundException' is subclass of 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L420">420</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L420">420</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L421">421</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L422">422</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L422">422</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L423">423</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L424">424</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L425">425</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L426">426</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L427">427</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L428">428</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L429">429</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L430">430</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L431">431</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L431">431</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L432">432</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L433">433</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L434">434</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L435">435</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L436">436</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L437">437</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L438">438</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L445">445</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter uniprotDoc should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L445">445</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'uniprotDoc' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L445">445</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'uniprotDoc'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L445">445</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L446">446</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L446">446</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L447">447</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L448">448</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 94).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L449">449</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L449">449</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L450">450</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L450">450</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L452">452</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L454">454</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L455">455</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 124).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L456">456</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L456">456</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L457">457</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L458">458</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L462">462</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter aUniprotbaseURL should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L473">473</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L477">477</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter aUniprotDirectoryCache should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L488">488</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L496">496</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter args should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L496">496</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L497">497</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L497">497</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 189).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L499">499</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L500">500</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L501">501</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L510">510</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L513">513</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L513">513</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getGeneName' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L513">513</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L514">514</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L515">515</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L516">516</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L517">517</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L518">518</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 94).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L519">519</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L519">519</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L520">520</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L520">520</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L521">521</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L522">522</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L523">523</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L524">524</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L524">524</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L525">525</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L526">526</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L527">527</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L528">528</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L529">529</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 125).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L530">530</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L530">530</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L530">530</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L531">531</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L532">532</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L535">535</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L539">539</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L539">539</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getOrganismName' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L539">539</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L544">544</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 94).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L545">545</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L545">545</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L546">546</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L546">546</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L547">547</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L548">548</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L549">549</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L550">550</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L550">550</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L551">551</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L552">552</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L553">553</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 133).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L556">556</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L556">556</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L556">556</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L557">557</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L562">562</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L563">563</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L566">566</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getKeyWords' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L566">566</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L567">567</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L567">567</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L572">572</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L573">573</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L575">575</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L575">575</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 114).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L576">576</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L576">576</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L577">577</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L578">578</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L579">579</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L580">580</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 138).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L581">581</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L581">581</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L581">581</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L582">582</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L583">583</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L588">588</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L592">592</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getDatabaseReferences' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L592">592</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L593">593</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L593">593</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 142).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L594">594</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L600">600</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 94).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L601">601</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L601">601</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 114).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L602">602</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L602">602</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L603">603</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L604">604</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L605">605</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L606">606</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L606">606</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L607">607</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L608">608</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L609">609</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L610">610</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L611">611</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L611">611</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 111).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L612">612</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L612">612</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L613">613</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L614">614</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L614">614</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L615">615</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L615">615</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L616">616</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L616">616</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L617">617</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L619">619</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L620">620</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 140).</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L622">622</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L622">622</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L622">622</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#L623">623</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.location.FuzzyPoint.java">org/biojava/nbio/core/sequence/location/FuzzyPoint.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing package-info.java file.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L0"></a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter point should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter point should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L51">51</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter point should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L66">66</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L70">70</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter minPoint should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter maxPoint should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L70">70</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter minPoint should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter maxPoint should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter resolver should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'resolver' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 118).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter minPoint should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter maxPoint should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter resolver should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'resolver' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter unknown should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter uncertain should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getPosition' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getMax' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getMin' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getResolver' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L103">103</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L103">103</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'reverse' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter length should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L111">111</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'offset' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L114">114</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter distance should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'equals' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter obj should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L123">123</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'hashCode' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'compareTo' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L146">146</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter point should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L147">147</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L149">149</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'cast' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L150">150</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L152">152</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#L152">152</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.location.InsdcLocations.java">org/biojava/nbio/core/sequence/location/InsdcLocations.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L93">93</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L93">93</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter circular should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L98">98</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L98">98</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L102">102</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L102">102</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L102">102</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L102">102</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L103">103</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L112">112</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L112">112</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L112">112</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L112">112</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter circular should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L113">113</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L113">113</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L117">117</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L117">117</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L122">122</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L122">122</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L123">123</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L124">124</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L135">135</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L135">135</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L135">135</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L140">140</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L145">145</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L146">146</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L146">146</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L150">150</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L150">150</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L150">150</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L150">150</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter circular should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L151">151</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L151">151</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L155">155</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L155">155</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L155">155</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L155">155</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L156">156</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L160">160</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L160">160</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L160">160</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L160">160</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L161">161</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L162">162</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L165">165</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L165">165</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L165">165</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L165">165</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter circular should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L170">170</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L170">170</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L170">170</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L170">170</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L171">171</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L175">175</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L175">175</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L175">175</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L175">175</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L176">176</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L177">177</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L192">192</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter locations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L192">192</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L196">196</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter locations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L196">196</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 94).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcLocations.html#L199">199</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.location.InsdcParser.java">org/biojava/nbio/core/sequence/location/InsdcParser.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;S&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 148).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'singleLocationPattern' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 130).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 123).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'genbankSplitPattern' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 106).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'locationSplitPattern' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L88">88</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'featureGlobalStart' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'featureGlobalEnd' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L93">93</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L101">101</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 99).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setComplexFeaturesAppendMode' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter complexFeaturesAppendMode should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'complexFeaturesAppendMode' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L113">113</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter dataSource should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L117">117</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'dataSource' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getDataSource' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L121">121</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L121">121</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'parse' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L134">134</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter locationString should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L134">134</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant throws: 'ParserException' is unchecked exception.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L134">134</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L137">137</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L140">140</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L145">145</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L149">149</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L162">162</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'parse' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L162">162</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter reader should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L162">162</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant throws: 'ParserException' is unchecked exception.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L162">162</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L167">167</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L167">167</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter string should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L167">167</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter versus should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L167">167</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant throws: 'ParserException' is unchecked exception.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L167">167</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L184">184</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L184">184</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 101).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L185">185</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>switch without &quot;default&quot; clause.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L187">187</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L195">195</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L196">196</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L199">199</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L204">204</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L207">207</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L210">210</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L212">212</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L221">221</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L226">226</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'3' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L227">227</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'6' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L228">228</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L228">228</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'6' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L228">228</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L242">242</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L243">243</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L243">243</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L243">243</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'4' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L243">243</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'4' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L243">243</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L248">248</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'5' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L248">248</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'5' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L248">248</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'7' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L248">248</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L251">251</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L252">252</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 109).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L252">252</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L261">261</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L262">262</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L263">263</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter input should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L263">263</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L284">284</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L285">285</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter ll should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L285">285</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L285">285</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L287">287</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L289">289</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L290">290</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L294">294</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L295">295</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter args should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L295">295</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Array brackets at illegal position.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L295">295</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L295">295</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L302">302</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 106).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L303">303</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 117).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L304">304</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L305">305</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L309">309</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L310">310</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L314">314</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#L316">316</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.location.LocationHelper.java">org/biojava/nbio/core/sequence/location/LocationHelper.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Utility classes should not have a public or default constructor.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L42">42</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'subLocations'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter type should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'start'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'end'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'strand'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter length should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'length'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 101).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused @param tag for 'location'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L116">116</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L123">123</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L123">123</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'start'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L123">123</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L123">123</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'end'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L123">123</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L123">123</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'strand'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L123">123</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter length should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L123">123</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L144">144</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L147">147</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L148">148</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'static' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L167">167</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L170">170</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L174">174</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter locations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L174">174</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'locations'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L174">174</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter previous should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L177">177</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter current should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L177">177</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L184">184</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L188">188</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter locations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L188">188</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'locations'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L188">188</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter previous should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L191">191</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter current should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L191">191</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L198">198</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L203">203</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L203">203</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter locations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L203">203</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'locations'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L203">203</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter predicate should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L203">203</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'predicate'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L203">203</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L208">208</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter index should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L226">226</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter seqLength should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L226">226</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L245">245</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter index should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L245">245</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'index'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L245">245</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter seqLength should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L245">245</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'seqLength'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L245">245</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L259">259</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L259">259</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'subLocations'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L259">259</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L261">261</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L261">261</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'!' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L261">261</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L268">268</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L285">285</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L287">287</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L298">298</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L298">298</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'subLocations'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L298">298</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L309">309</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L312">312</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L312">312</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'subLocations'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L312">312</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L322">322</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L322">322</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L322">322</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'subLocations'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L322">322</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter isCircular should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L322">322</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'isCircular'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L322">322</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L325">325</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L330">330</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'-' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L332">332</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'-' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L332">332</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/LocationHelper.html#L335">335</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.location.SequenceLocation.java">org/biojava/nbio/core/sequence/location/SequenceLocation.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L32">32</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L37">37</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;S&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequence' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 123).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequence' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter circular should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequence' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequence' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L73">73</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L73">73</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequence' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SequenceLocation.html#L73">73</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.location.SimpleLocation.java">org/biojava/nbio/core/sequence/location/SimpleLocation.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L62">62</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L62">62</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 115).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L71">71</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L71">71</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L71">71</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter betweenCompounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L71">71</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 106).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter circular should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter betweenBases should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 110).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter circular should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 113).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L91">91</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L91">91</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter circular should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L91">91</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 135).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter circular should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter betweenBases should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimpleLocation.html#L101">101</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.location.SimplePoint.java">org/biojava/nbio/core/sequence/location/SimplePoint.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L35">35</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L37">37</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'position' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'position' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter unknown should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'unknown' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter uncertain should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'uncertain' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getPosition' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setPosition' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'position' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'isUnknown' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L62">62</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setUnknown' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter unknown should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'unknown' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'isUncertain' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setUncertain' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter uncertain should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'uncertain' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'reverse' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter length should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'offset' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter distance should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'reverse' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'position' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter length should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'equals' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter obj should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'hashCode' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'toString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L117">117</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'compareTo' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L122">122</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter o should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L123">123</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'isLower' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter point should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L128">128</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'isHigher' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L132">132</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter point should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L133">133</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'clonePoint' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#L137">137</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.location.template.AbstractLocation.java">org/biojava/nbio/core/sequence/location/template/AbstractLocation.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing package-info.java file.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L0"></a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L62">62</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L66">66</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L73">73</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'start' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L86">86</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'end' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L86">86</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'strand' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L86">86</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter circular should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'circular' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter betweenCompounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'betweenCompounds' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'subLocations' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L88">88</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L100">100</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L106">106</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'start' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L106">106</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L106">106</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'end' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L106">106</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L106">106</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'strand' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L106">106</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter circular should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'circular' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter betweenCompounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'betweenCompounds' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'accession' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'subLocations' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'?' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'assertLocation' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L119">119</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L135">135</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L140">140</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L147">147</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getEnd' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L148">148</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L149">149</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L153">153</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getStart' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L154">154</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L155">155</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L159">159</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L160">160</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L161">161</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L165">165</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getStrand' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L167">167</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L171">171</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSubLocations' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L172">172</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L173">173</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L174">174</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L180">180</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'isComplex' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L181">181</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L182">182</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L186">186</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAccession' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L187">187</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L188">188</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L191">191</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'isPartialOn5prime' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L192">192</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L192">192</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setPartialOn5prime' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L196">196</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L196">196</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter partialOn5prime should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L196">196</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'partialOn5prime' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L196">196</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'isPartialOn3prime' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L200">200</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L200">200</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setPartialOn3prime' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L204">204</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L204">204</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter partialOn3prime should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L204">204</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'partialOn3prime' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L204">204</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L207">207</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'isPartial' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L208">208</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L208">208</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L212">212</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L216">216</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L217">217</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'iterator' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L217">217</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L218">218</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L220">220</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L222">222</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L235">235</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L236">236</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getRelevantSubLocations' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L236">236</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L237">237</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L241">241</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L244">244</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter location should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L244">244</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'location'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L244">244</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L249">249</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L257">257</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'equals' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L258">258</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L259">259</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter obj should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L259">259</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L274">274</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'hashCode' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L275">275</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L276">276</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L288">288</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'isCircular' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L289">289</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L290">290</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L294">294</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'isBetweenCompounds' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L295">295</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L296">296</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L300">300</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L307">307</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L308">308</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSubSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L308">308</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L309">309</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L309">309</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;C&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L309">309</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L309">309</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L309">309</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L310">310</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L312">312</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L315">315</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L322">322</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L324">324</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L325">325</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getRelevantSubSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L325">325</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 94).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L326">326</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L326">326</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;C&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L326">326</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L326">326</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L326">326</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L328">328</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L331">331</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L339">339</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'reverseSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L339">339</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L340">340</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;C&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L340">340</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L340">340</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L340">340</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L341">341</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L347">347</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L348">348</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'cast' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L351">351</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L356">356</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L360">360</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'canComplement' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L360">360</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;C&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L360">360</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L360">360</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L360">360</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L366">366</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'toString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L367">367</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L368">368</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L369">369</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L370">370</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L371">371</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setCircular' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L375">375</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L375">375</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter circular should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L375">375</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'circular' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L375">375</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setEnd' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L379">379</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L379">379</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L379">379</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'end' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L379">379</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setStart' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L383">383</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L383">383</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L383">383</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'start' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L383">383</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setStrand' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L387">387</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L387">387</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L387">387</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'strand' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L387">387</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setBetweenCompounds' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L391">391</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L391">391</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter betweenCompounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L391">391</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'betweenCompounds' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L391">391</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setSubLocations' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L395">395</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L395">395</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter subLocations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L395">395</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'subLocations' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L395">395</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setAccession' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L399">399</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L399">399</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L399">399</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'accession' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#L399">399</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.location.template.AccesionedLocation.java">org/biojava/nbio/core/sequence/location/template/AccesionedLocation.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AccesionedLocation.html#L29">29</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AccesionedLocation.html#L42">42</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.location.template.Location.java">org/biojava/nbio/core/sequence/location/template/Location.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L104">104</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L106">106</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused @param tag for '&amp;lt;C&amp;gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;C&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L114">114</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused @param tag for '&amp;lt;C&amp;gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L121">121</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;C&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L125">125</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L133">133</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 104).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter locations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'locations'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequenceLength should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequenceLength'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter type should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'type'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L137">137</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L138">138</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L150">150</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L150">150</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L150">150</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'start'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L150">150</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L150">150</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'end'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L150">150</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L150">150</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'strand'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L150">150</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter length should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L150">150</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'length'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L150">150</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L152">152</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused @param tag for 'location'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L169">169</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L176">176</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L176">176</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'start'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L176">176</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L176">176</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'end'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L176">176</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L176">176</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'strand'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L176">176</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter length should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L176">176</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L197">197</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L200">200</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L201">201</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L219">219</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'static' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L219">219</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L221">221</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L224">224</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L228">228</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter locations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L228">228</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'locations'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L228">228</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L230">230</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L232">232</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L232">232</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter previous should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L232">232</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter current should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L232">232</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L239">239</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L243">243</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter locations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L243">243</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'locations'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L243">243</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L245">245</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L247">247</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L247">247</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter previous should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L247">247</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter current should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L247">247</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L254">254</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L259">259</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L259">259</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter locations should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L259">259</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'locations'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L259">259</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter predicate should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L259">259</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'predicate'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L259">259</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter index should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L281">281</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter seqLength should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L281">281</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L300">300</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter index should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L300">300</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'index'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L300">300</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter seqLength should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L300">300</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'seqLength'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#L300">300</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.location.template.Point.java">org/biojava/nbio/core/sequence/location/template/Point.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Point.html#L24">24</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Point.html#L29">29</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Point.html#L33">33</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Point.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Point.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Point.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Point.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Point.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Point.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Point.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Point.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'distance'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Point.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Point.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'point'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Point.html#L70">70</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Point.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Point.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'point'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Point.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Point.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Point.html#L80">80</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.storage.ArrayListSequenceReader.java">org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing package-info.java file.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L0"></a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - org.biojava.nbio.core.sequence.template.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L31">31</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compounds'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 115).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequenceAsString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequenceAsString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter begin should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'begin'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'end'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'strand'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 113).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L98">98</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L105">105</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAsList' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L106">106</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundAt' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L116">116</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L116">116</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'position'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L116">116</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L125">125</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getIndexOf' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L126">126</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compound'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L126">126</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L135">135</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLastIndexOf' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L135">135</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compound'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L144">144</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L144">144</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L145">145</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L153">153</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'iterator' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L153">153</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L154">154</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setCompoundSet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L162">162</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L163">163</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L163">163</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L163">163</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L163">163</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L171">171</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundSet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L171">171</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L172">172</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setContents' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L180">180</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L181">181</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L181">181</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L181">181</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L181">181</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'CompoundNotFoundException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L181">181</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 132).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L196">196</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 130).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L200">200</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L206">206</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L212">212</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setContents' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L219">219</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter list should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L219">219</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'list'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L219">219</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L232">232</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSubSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L232">232</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L233">233</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L233">233</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioBegin'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L233">233</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioEnd'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L233">233</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L234">234</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L241">241</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAccession' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L241">241</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L242">242</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L251">251</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'countCompounds' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L251">251</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L252">252</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L252">252</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compounds'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L252">252</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L260">260</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getInverse' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L260">260</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'hashCode' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L265">265</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L267">267</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'equals' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L276">276</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter o should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L278">278</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L279">279</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'cast' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L280">280</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&amp;&amp;' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#L281">281</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.storage.BitSequenceReader.java">org/biojava/nbio/core/sequence/storage/BitSequenceReader.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused import - org.biojava.nbio.core.sequence.compound.NucleotideCompound.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L25">25</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - org.biojava.nbio.core.sequence.template.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L26">26</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 97).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter worker should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'worker' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'worker'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'accession' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'accession'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setCompoundSet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L70">70</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 101).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setContents' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'CompoundNotFoundException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 135).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'countCompounds' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L86">86</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compounds'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L86">86</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L90">90</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAccession' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L98">98</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAsList' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L100">100</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L104">104</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L106">106</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundAt' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'position'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L112">112</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L114">114</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundSet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L116">116</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L120">120</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L121">121</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L124">124</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L125">125</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getIndexOf' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L126">126</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compound'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L126">126</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L130">130</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L134">134</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L135">135</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLastIndexOf' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L135">135</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compound'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L140">140</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L143">143</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L144">144</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L144">144</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L145">145</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L149">149</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L152">152</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L153">153</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequenceAsString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L153">153</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L154">154</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L158">158</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L161">161</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSubSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L161">161</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'start'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L161">161</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'end'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L161">161</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L165">165</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L168">168</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'iterator' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L168">168</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L172">172</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSubSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L173">173</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L174">174</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L174">174</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L174">174</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getInverse' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L178">178</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'hashCode' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L183">183</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'equals' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L191">191</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter o should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L192">192</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L193">193</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'cast' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L195">195</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&amp;&amp;' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L196">196</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'abstract' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L214">214</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L216">216</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L217">217</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L218">218</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L219">219</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L220">220</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L221">221</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L223">223</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L225">225</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L225">225</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequence' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L225">225</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L230">230</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L230">230</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequence' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L230">230</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L230">230</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L230">230</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L235">235</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L235">235</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L235">235</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter length should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L235">235</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'length' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L235">235</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L241">241</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L241">241</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L241">241</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L241">241</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequence' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L241">241</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L252">252</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L254">254</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L257">257</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L260">260</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L261">261</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L264">264</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L267">267</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L270">270</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L273">273</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L276">276</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'bitsPerCompound' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L276">276</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'seqArraySize' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L280">280</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L280">280</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter length should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L280">280</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'length' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L280">280</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L281">281</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L284">284</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'populate' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L288">288</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L288">288</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequence' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L288">288</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L288">288</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L295">295</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'populate' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L299">299</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L299">299</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequence' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L299">299</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L299">299</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L305">305</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L306">306</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setCompoundAt' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L308">308</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter base should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L308">308</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'base'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L308">308</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L308">308</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'position'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L308">308</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L309">309</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L313">313</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setCompoundAt' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L316">316</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L316">316</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compound'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L316">316</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L316">316</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'position'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L316">316</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L333">333</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L336">336</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundAt' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L336">336</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L336">336</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'position'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L336">336</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 118).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L339">339</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 133).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L343">343</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 159).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L354">354</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L362">362</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L364">364</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L371">371</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'processUnknownCompound' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L371">371</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L371">371</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L371">371</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'position'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L371">371</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant throws: 'IllegalStateException' is unchecked exception.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L371">371</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 155).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L372">372</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L379">379</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getIndexToCompoundsLookup' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L379">379</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L386">386</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L387">387</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L389">389</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundsToIndexLookup' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L389">389</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L415">415</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter index should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L415">415</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'index'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L415">415</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L419">419</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L431">431</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter index should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L431">431</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'index'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L431">431</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L432">432</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L435">435</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L438">438</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundSet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L438">438</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L442">442</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L442">442</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'hashCode' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L446">446</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L448">448</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'equals' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L458">458</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter o should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L460">460</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L461">461</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'cast' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L462">462</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&amp;&amp;' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L463">463</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L464">464</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&amp;&amp;' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#L464">464</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.storage.FourBitSequenceReader.java">org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.util.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L32">32</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter worker should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter worker should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter length should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L91">91</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 140).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'static' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'3' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'bitMask' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L100">100</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L101">101</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L105">105</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'compoundsPerDatatype' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L106">106</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'8' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L118">118</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'generateCompoundsToIndex' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L124">124</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L138">138</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L138">138</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter currentCompound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L138">138</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter cs should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L138">138</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L140">140</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L149">149</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L153">153</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L155">155</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter o1 should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L155">155</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter o2 should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L155">155</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L164">164</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L167">167</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L168">168</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'generateIndexToCompounds' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L168">168</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L169">169</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L175">175</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#L180">180</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.storage.JoiningSequenceReader.java">org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - org.biojava.nbio.core.sequence.template.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L25">25</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.util.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L27">27</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequences' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequences'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequences' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L73">73</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequences'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequences' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequences' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequences' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 126).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L104">104</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundAt' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L106">106</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L106">106</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L113">113</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundSet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L114">114</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L119">119</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L120">120</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L121">121</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L129">129</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L132">132</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L132">132</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'position'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L132">132</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L140">140</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L147">147</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L154">154</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L173">173</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L173">173</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'position'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L173">173</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 111).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L182">182</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L185">185</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L188">188</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L188">188</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'position'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L188">188</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L206">206</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 111).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L214">214</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L217">217</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L221">221</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L222">222</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'iterator' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L222">222</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L223">223</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L230">230</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L232">232</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L233">233</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L246">246</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L248">248</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 133).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L251">251</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L253">253</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L257">257</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L263">263</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L265">265</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L265">265</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant throws: 'UnsupportedOperationException' is unchecked exception.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L265">265</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L266">266</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L271">271</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setCompoundSet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L272">272</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L273">273</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L273">273</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L273">273</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L277">277</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setContents' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L278">278</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L279">279</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L279">279</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L279">279</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L283">283</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'countCompounds' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L284">284</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L285">285</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L285">285</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L289">289</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAccession' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L290">290</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L291">291</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant throws: 'UnsupportedOperationException' is unchecked exception.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L291">291</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L295">295</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAsList' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L296">296</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L297">297</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L301">301</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getIndexOf' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L302">302</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L303">303</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L303">303</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L307">307</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLastIndexOf' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L308">308</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L309">309</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L309">309</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L313">313</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequenceAsString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L314">314</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L315">315</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L319">319</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSubSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L320">320</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L321">321</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L321">321</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L321">321</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getInverse' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#L325">325</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.storage.SequenceAsStringHelper.java">org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L34">34</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 141).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequenceAsString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter parsedCompounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'parsedCompounds'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioBegin should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioBegin'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioEnd should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioEnd'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'strand'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L51">51</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'else' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'else' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'else' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'else' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#L81">81</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.storage.SingleCompoundSequenceReader.java">org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - org.biojava.nbio.core.sequence.template.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L25">25</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 97).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compound' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compound'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter length should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'length' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'length'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compound' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compound'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter length should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'length' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'length'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setCompoundSet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setContents' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'CompoundNotFoundException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L86">86</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L90">90</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L93">93</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundAt' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'position'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L103">103</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L104">104</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getIndexOf' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L104">104</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L105">105</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compound' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compound'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L105">105</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L106">106</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L116">116</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L117">117</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLastIndexOf' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L118">118</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compound' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L118">118</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compound'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L125">125</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 99).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L128">128</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L129">129</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequenceAsString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L129">129</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L130">130</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L134">134</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 99).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L135">135</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L137">137</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L138">138</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAsList' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L138">138</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L139">139</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L143">143</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L146">146</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L147">147</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSubSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L147">147</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L148">148</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L148">148</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'start'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L148">148</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L148">148</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'end'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L148">148</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L152">152</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L155">155</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L156">156</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundSet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L156">156</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L157">157</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L161">161</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L164">164</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L165">165</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAccession' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L165">165</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L170">170</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 112).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L171">171</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L173">173</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L174">174</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'countCompounds' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L174">174</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L175">175</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L175">175</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compounds'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L175">175</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L179">179</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L182">182</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L183">183</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'iterator' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L183">183</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L184">184</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getInverse' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L188">188</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'hashCode' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L193">193</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'equals' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L202">202</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter o should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L204">204</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L205">205</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L206">206</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'cast' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L206">206</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&amp;&amp;' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L207">207</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&amp;&amp;' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#L208">208</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.storage.TwoBitSequenceReader.java">org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 94).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;C&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L62">62</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter worker should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter worker should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L73">73</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 103).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter length should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L86">86</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L86">86</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L90">90</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L90">90</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L90">90</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L91">91</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L98">98</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L98">98</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L100">100</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L101">101</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L101">101</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'static' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L101">101</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L103">103</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'bitMask' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L103">103</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L104">104</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L106">106</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'compoundsPerDatatype' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'16' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L110">110</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L111">111</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L113">113</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L114">114</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L116">116</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L116">116</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'generateCompoundsToIndex' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L116">116</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L117">117</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L120">120</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L123">123</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L124">124</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L125">125</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'3' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L126">126</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L128">128</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L129">129</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L130">130</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'3' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L130">130</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L131">131</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L132">132</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L133">133</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L135">135</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L137">137</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L138">138</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L138">138</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'generateIndexToCompounds' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L138">138</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L139">139</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L140">140</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L141">141</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L142">142</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L142">142</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L145">145</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L145">145</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L146">146</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L146">146</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L147">147</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L147">147</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L148">148</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L149">149</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#L150">150</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.template.AbstractCompound.java">org/biojava/nbio/core/sequence/template/AbstractCompound.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing package-info.java file.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L0"></a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L25">25</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L31">31</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L32">32</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L33">33</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L34">34</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L35">35</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter base should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'base' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getBase' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getUpperedBase' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getDescription' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L51">51</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setDescription' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter description should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'description' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getShortName' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setShortName' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter shortName should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'shortName' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLongName' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L71">71</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setLongName' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter longName should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'longName' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getMolecularWeight' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setMolecularWeight' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L86">86</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter molecularWeight should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'molecularWeight' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'toString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L91">91</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'equals' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter obj should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'hashCode' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'equalsIgnoreCase' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L113">113</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompound.html#L114">114</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.template.AbstractCompoundSet.java">org/biojava/nbio/core/sequence/template/AbstractCompoundSet.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.util.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L29">29</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'static' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'logger' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L42">42</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'equivalentsMap' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addCompound' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter lowerCasedCompound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter equivalents should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addCompound' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter lowerCasedCompound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter equivalents should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addEquivalent' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter equivalent should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L71">71</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addCompound' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getStringForCompound' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundForString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L90">90</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter string should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L91">91</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L93">93</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getMaxSingleCompoundStringLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L112">112</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'isCompoundStringLengthEqual' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L120">120</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L125">125</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'hasCompound' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L135">135</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'compoundsEquivalent' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L141">141</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundOne should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L142">142</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundTwo should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L142">142</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 100).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L145">145</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getEquivalentCompounds' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L148">148</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L149">149</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'compoundsEqual' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L153">153</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L153">153</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundOne should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L153">153</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundTwo should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L153">153</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'isValidSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L159">159</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L160">160</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAllCompounds' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L171">171</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L176">176</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L176">176</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L178">178</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L178">178</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L179">179</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 116).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L180">180</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L180">180</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'isComplementable' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L187">187</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'hashCode' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L192">192</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'equals' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L200">200</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter o should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L202">202</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L203">203</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'!' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L203">203</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L204">204</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'cast' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L205">205</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&amp;&amp;' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#L206">206</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.template.AbstractCompoundTranslator.java">org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.util.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L26">26</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L29">29</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L29">29</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L32">32</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L33">33</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L34">34</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L35">35</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter creator should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L37">37</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'creator' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter fromCompoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'fromCompoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter toCompoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'toCompoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCreator' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getFromCompoundSet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getToCompoundSet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addStrings' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter source should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter targets should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addCompounds' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter source should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter targets should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L66">66</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L70">70</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L71">71</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'translateMany' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter fromCompound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'translate' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter fromCompound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'createSequences' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L98">98</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter originalSequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addCompoundsToList' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter workingList should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L139">139</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L149">149</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'workingListToSequences' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L149">149</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L149">149</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter workingList should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L149">149</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L157">157</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter incoming should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L157">157</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addCompoundToLists' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L165">165</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L165">165</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter list should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L165">165</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L165">165</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'createSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L176">176</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter originalSequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L177">177</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#L182">182</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.template.AbstractNucleotideCompoundSet.java">org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.util.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L25">25</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L33">33</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addNucleotideCompound' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter base should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter complement should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter equivalents should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 100).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 100).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 100).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'calculateIndirectAmbiguities' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 131).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L88">88</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L91">91</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L103">103</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L104">104</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L104">104</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L106">106</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L106">106</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'cast' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'cast' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'cast' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'cast' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L113">113</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L114">114</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L114">114</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter equivalentsMap should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L114">114</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter key should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L116">116</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter value should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L116">116</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L118">118</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L120">120</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L120">120</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L121">121</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L122">122</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L126">126</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L129">129</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L129">129</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAmbiguity' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L141">141</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L141">141</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L143">143</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L144">144</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L145">145</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L148">148</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L149">149</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L156">156</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L159">159</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'isComplementable' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#L159">159</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.template.AbstractSequence.java">org/biojava/nbio/core/sequence/template/AbstractSequence.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - org.biojava.nbio.core.sequence.features.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L34">34</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.util.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'static' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'logger' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L62">62</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L66">66</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 104).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L73">73</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 108).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter seqString should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'seqString'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L93">93</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 132).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 117).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 124).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L102">102</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter proxyLoader should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L102">102</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'proxyLoader'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L102">102</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L102">102</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L102">102</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundSet'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L102">102</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 101).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 97).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L111">111</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setProxySequenceReader' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L114">114</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter proxyLoader should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L114">114</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'proxyLoader'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L114">114</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L120">120</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L122">122</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 97).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L126">126</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L131">131</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 114).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L132">132</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'cast' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L132">132</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L133">133</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 141).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L133">133</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L133">133</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L133">133</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L133">133</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L133">133</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L135">135</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 180).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getProxySequenceReader' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L141">141</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L141">141</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getBioBegin' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L148">148</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused @param tag for 'bioBegin'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L157">157</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setBioBegin' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L159">159</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter begin should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L159">159</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'begin'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L159">159</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getBioEnd' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused @param tag for 'bioEnd'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L175">175</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setBioEnd' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L177">177</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L177">177</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'end'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L177">177</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L181">181</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L182">182</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L186">186</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getUserCollection' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L186">186</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setUserCollection' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L195">195</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter userCollection should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L195">195</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'userCollection' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L195">195</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'userCollection'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L195">195</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAnnotationType' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L202">202</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused @param tag for 'annotation'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L207">207</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setAnnotationType' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L209">209</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter annotationType should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L209">209</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'annotationType' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L209">209</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'annotationType'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L209">209</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getDescription' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L216">216</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setDescription' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L223">223</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter description should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L223">223</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'description' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L223">223</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getOriginalHeader' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L230">230</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setOriginalHeader' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L237">237</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter originalHeader should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L237">237</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'originalHeader' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L237">237</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getParentSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L244">244</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setParentSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L251">251</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter parentSequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L251">251</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'parentSequence' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L251">251</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L255">255</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSource' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L260">260</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L270">270</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setSource' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L274">274</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter source should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L274">274</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'source' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L274">274</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L279">279</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addNote' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L283">283</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter note should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L283">283</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'note'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L283">283</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'removeNote' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L287">287</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L287">287</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter note should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L287">287</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getNotesList' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L294">294</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setNotesList' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L301">301</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter notesList should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L301">301</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'notesList' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L301">301</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L305">305</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L306">306</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequenceScore' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L309">309</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setSequenceScore' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L316">316</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequenceScore should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L316">316</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequenceScore' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L316">316</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L320">320</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L326">326</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 116).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L326">326</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getFeatures' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L326">326</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter featureType should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L326">326</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'featureType'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L326">326</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioSequencePosition should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L326">326</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioSequencePosition'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L326">326</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 97).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L329">329</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'features' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L329">329</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 165).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L332">332</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L340">340</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L346">346</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L346">346</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getFeatures' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L346">346</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioSequencePosition should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L346">346</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioSequencePosition'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L346">346</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 165).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L351">351</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L363">363</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getFeatures' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L363">363</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L367">367</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L368">368</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 104).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L374">374</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addFeature' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L374">374</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioStart should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L374">374</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioStart'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L374">374</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioEnd should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L374">374</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'bioEnd' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L374">374</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioEnd'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L374">374</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter feature should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L374">374</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'feature'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L374">374</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L376">376</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 133).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L382">382</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 133).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L383">383</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'addFeature' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L387">387</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter feature should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L387">387</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'feature'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L387">387</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 113).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L389">389</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L391">391</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L399">399</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L403">403</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'removeFeature' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L403">403</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter feature should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L403">403</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'feature'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L403">403</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 113).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L405">405</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L419">419</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L419">419</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getFeaturesByType' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L419">419</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter type should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L419">419</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'type'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L419">419</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L420">420</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'features' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L420">420</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L422">422</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getFeaturesKeyWord' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L430">430</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setFeaturesKeyWord' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L437">437</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter featuresKeyWord should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L437">437</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'featuresKeyWord' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L437">437</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getDatabaseReferences' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L444">444</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L451">451</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setDatabaseReferences' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L451">451</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter databaseReferences should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L451">451</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'databaseReferences' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L451">451</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getFeatureRetriever' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L455">455</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L455">455</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setFeatureRetriever' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L459">459</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L459">459</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter featureRetriever should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L459">459</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'featureRetriever' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L459">459</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L462">462</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L463">463</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L465">465</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L467">467</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L467">467</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L467">467</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAccession' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L473">473</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L474">474</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setAccession' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L481">481</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter accession should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L481">481</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'accession' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L481">481</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getTaxonomy' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L488">488</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused @param tag for 'species'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L493">493</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setTaxonomy' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L495">495</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter taxonomy should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L495">495</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'taxonomy' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L495">495</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'taxonomy'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L495">495</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundSet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L499">499</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L500">500</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 116).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L504">504</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setCompoundSet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L516">516</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L516">516</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L516">516</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compoundSet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L516">516</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'toString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L520">520</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L525">525</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L530">530</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L531">531</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L532">532</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L532">532</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L533">533</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L533">533</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L534">534</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L537">537</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L537">537</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L539">539</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L540">540</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 152).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L540">540</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L540">540</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L540">540</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L544">544</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L546">546</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L554">554</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L557">557</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L557">557</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequenceAsString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L557">557</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioStart should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L557">557</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioStart'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L557">557</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioEnd should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L557">557</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'bioEnd' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L557">557</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioEnd'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L557">557</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter strand should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L557">557</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L558">558</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L558">558</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 119).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L564">564</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L567">567</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequenceAsString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L567">567</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L568">568</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L577">577</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAsList' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L577">577</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L578">578</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L587">587</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundAt' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L587">587</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L588">588</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L588">588</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L589">589</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L589">589</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getIndexOf' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L598">598</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L599">599</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L599">599</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compound'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L599">599</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLastIndexOf' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L608">608</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L609">609</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L609">609</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compound'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L609">609</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L617">617</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L617">617</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L618">618</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L628">628</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSubSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L628">628</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L629">629</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L629">629</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioStart'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L629">629</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'bioEnd' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L629">629</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bioEnd'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L629">629</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L637">637</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'iterator' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L637">637</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L638">638</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L647">647</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'countCompounds' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L647">647</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L648">648</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L648">648</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compounds'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L648">648</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L656">656</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getInverse' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L656">656</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#L660">660</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.template.Accessioned.java">org/biojava/nbio/core/sequence/template/Accessioned.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Accessioned.html#L25">25</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Accessioned.html#L32">32</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Accessioned.html#L35">35</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.template.ComplementCompound.java">org/biojava/nbio/core/sequence/template/ComplementCompound.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/ComplementCompound.html#L30">30</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/ComplementCompound.html#L32">32</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/ComplementCompound.html#L32">32</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.template.Compound.java">org/biojava/nbio/core/sequence/template/Compound.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Compound.html#L29">29</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Compound.html#L31">31</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Compound.html#L31">31</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Compound.html#L33">33</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Compound.html#L33">33</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Compound.html#L35">35</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Compound.html#L35">35</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Compound.html#L37">37</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Compound.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Compound.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Compound.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Compound.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Compound.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Compound.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Compound.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Compound.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Compound.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Compound.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Compound.html#L47">47</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.template.CompoundSet.java">org/biojava/nbio/core/sequence/template/CompoundSet.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L32">32</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L34">34</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L37">37</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'string'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L51">51</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundSet.html#L63">63</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.template.CompoundTranslator.java">org/biojava/nbio/core/sequence/template/CompoundTranslator.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundTranslator.html#L25">25</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundTranslator.html#L27">27</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundTranslator.html#L29">29</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundTranslator.html#L31">31</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/CompoundTranslator.html#L33">33</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.template.LightweightProfile.java">org/biojava/nbio/core/sequence/template/LightweightProfile.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 112).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/LightweightProfile.html#L29">29</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/LightweightProfile.html#L34">34</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/LightweightProfile.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/LightweightProfile.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/LightweightProfile.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/LightweightProfile.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/LightweightProfile.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/LightweightProfile.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/LightweightProfile.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused @throws tag for 'IndexOutOfBoundsException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/LightweightProfile.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/LightweightProfile.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 99).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/LightweightProfile.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 103).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/LightweightProfile.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused @throws tag for 'IndexOutOfBoundsException'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/LightweightProfile.html#L71">71</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/LightweightProfile.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 112).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/LightweightProfile.html#L90">90</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 118).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/LightweightProfile.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 119).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/LightweightProfile.html#L98">98</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 111).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/LightweightProfile.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/LightweightProfile.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 119).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/LightweightProfile.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 94).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/LightweightProfile.html#L117">117</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.template.ProxySequenceReader.java">org/biojava/nbio/core/sequence/template/ProxySequenceReader.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/ProxySequenceReader.html#L29">29</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/ProxySequenceReader.html#L29">29</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/ProxySequenceReader.html#L30">30</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/ProxySequenceReader.html#L30">30</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.template.Sequence.java">org/biojava/nbio/core/sequence/template/Sequence.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L30">30</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L90">90</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L103">103</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/Sequence.html#L110">110</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.template.SequenceMixin.java">org/biojava/nbio/core/sequence/template/SequenceMixin.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.util.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L31">31</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Utility classes should not have a public or default constructor.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'G' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'C' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L90">90</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L93">93</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'A' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'T' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L100">100</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L135">135</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L135">135</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L140">140</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L140">140</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L141">141</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L148">148</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused @param tag for 'writer'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L153">153</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 115).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L157">157</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter appendable should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L157">157</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'appendable'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L157">157</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L157">157</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L158">158</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 99).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L169">169</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L169">169</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;C&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L169">169</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L169">169</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L169">169</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L178">178</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L181">181</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;C&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L181">181</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L181">181</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L181">181</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L186">186</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L189">189</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;C&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L189">189</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L189">189</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L189">189</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L201">201</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;C&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L201">201</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L201">201</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L201">201</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L202">202</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compound'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L202">202</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L219">219</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;C&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L219">219</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L219">219</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L219">219</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L220">220</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compound'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L220">220</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L222">222</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L222">222</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L230">230</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;C&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L230">230</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L231">231</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L231">231</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L238">238</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;C&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L238">238</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L239">239</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L239">239</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L239">239</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'start'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L239">239</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter end should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L239">239</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'end'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L239">239</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L250">250</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L250">250</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;C&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L250">250</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L250">250</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L250">250</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L260">260</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;C&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L260">260</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L260">260</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L260">260</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 114).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L277">277</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L277">277</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter kmer should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L277">277</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L280">280</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 111).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L295">295</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L295">295</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter kmer should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L295">295</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L300">300</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L300">300</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L300">300</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L300">300</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L300">300</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L301">301</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L303">303</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L305">305</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'while' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L310">310</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L311">311</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L311">311</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L311">311</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L311">311</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L311">311</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L313">313</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L313">313</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L314">314</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L318">318</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L326">326</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L335">335</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L335">335</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L337">337</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 115).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L355">355</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter source should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L355">355</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter target should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L355">355</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L371">371</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter source should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L371">371</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter target should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L371">371</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 130).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L375">375</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L375">375</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter source should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L375">375</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter target should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L375">375</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter ignoreCase should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L375">375</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L377">377</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&amp;&amp;' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L378">378</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'while' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L382">382</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L385">385</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L385">385</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L386">386</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L391">391</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L410">410</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L411">411</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L412">412</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L414">414</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L414">414</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequence' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L414">414</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'hasNext' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L420">420</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L421">421</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L425">425</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'next' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L426">426</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L427">427</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L428">428</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L429">429</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'remove' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L434">434</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L435">435</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceMixin.html#L436">436</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.template.SequenceProxyView.java">org/biojava/nbio/core/sequence/template/SequenceProxyView.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L33">33</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L35">35</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L42">42</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequence' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L42">42</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequence' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioStart should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'bioStart' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioEnd should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'bioEnd' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getViewedSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequenceAsString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAsList' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundAt' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getIndexOf' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLastIndexOf' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L90">90</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundSet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSubSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L100">100</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L100">100</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'bioStart' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L100">100</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'bioEnd' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L100">100</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'iterator' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L104">104</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L105">105</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAccession' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getBioStart' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L117">117</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setBioStart' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioStart should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L125">125</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'bioStart' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getBioEnd' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L137">137</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setBioEnd' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L144">144</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bioEnd should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L144">144</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'bioEnd' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L144">144</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L145">145</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 114).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L146">146</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 129).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L148">148</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 119).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L149">149</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'countCompounds' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L159">159</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L160">160</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L160">160</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getInverse' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceProxyView.html#L164">164</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.template.SequenceReader.java">org/biojava/nbio/core/sequence/template/SequenceReader.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceReader.html#L30">30</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceReader.html#L31">31</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceReader.html#L31">31</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceReader.html#L32">32</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceReader.html#L32">32</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceReader.html#L32">32</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceReader.html#L33">33</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceReader.html#L33">33</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceReader.html#L34">34</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceReader.html#L34">34</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceReader.html#L34">34</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceReader.html#L34">34</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.template.SequenceView.java">org/biojava/nbio/core/sequence/template/SequenceView.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceView.html#L29">29</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceView.html#L31">31</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceView.html#L31">31</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceView.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceView.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceView.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/sequence/template/SequenceView.html#L41">41</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.transcription.CaseInsensitiveCompound.java">org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing package-info.java file.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#L0"></a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#L27">27</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#L33">33</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#L35">35</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#L35">35</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compound' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#L35">35</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'equalsIgnoreCase' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'compound' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'equals' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter obj should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'cast' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'hashCode' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#L62">62</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getUnderlyingCompound' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getDescription' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLongName' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getMolecularWeight' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getShortName' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#L90">90</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'toString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#L91">91</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter description should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter longName should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#L102">102</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter molecularWeight should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter shortName should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#L112">112</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.transcription.DNAToRNATranslator.java">org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 108).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter rnaCreator should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter dna should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter rna should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter shortCutTranslation should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'shortCutTranslation' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L51">51</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'createSequences' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 110).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L66">66</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter originalSequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 103).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L71">71</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'wrapToRna' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter dna should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'dna'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L90">90</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L100">100</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 114).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'createSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter originalSequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter frame should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'createSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L112">112</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 101).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L113">113</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter originalSequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L113">113</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundLists should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#L119">119</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.transcription.Frame.java">org/biojava/nbio/core/sequence/transcription/Frame.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>File does not end with a newline.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L0"></a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L35">35</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L37">37</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'start' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter reverse should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'reverse' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L71">71</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;C&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter incoming should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'incoming'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Frame.html#L96">96</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.transcription.RNAToAminoAcidTranslator.java">org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused import - org.biojava.nbio.core.sequence.RNASequence.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L24">24</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.util.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L35">35</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L51">51</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'64000' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L66">66</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>More than 7 parameters (found 8).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter creator should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'creator'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter nucleotides should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'nucleotides'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter codons should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'codons'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter aminoAcids should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'aminoAcids'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter table should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'table'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter trimStops should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'trimStops' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'trimStops'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter initMetOnly should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'initMetOnly' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'initMetOnly'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter translateNCodons should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'translateNCodons' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'translateNCodons'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L91">91</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L93">93</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L98">98</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L100">100</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L101">101</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L103">103</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L104">104</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L105">105</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L106">106</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L111">111</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L111">111</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L112">112</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>More than 7 parameters (found 9).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L112">112</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L113">113</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter creator should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L113">113</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L114">114</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter nucleotides should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L114">114</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter codons should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L116">116</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter aminoAcids should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L116">116</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter table should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L116">116</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L117">117</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter trimStops should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'trimStops' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L117">117</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter initMetOnly should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'initMetOnly' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L117">117</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter translateNCodons should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'translateNCodons' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L117">117</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter stopAtStopCodons should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L118">118</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'stopAtStopCodons' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L120">120</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L121">121</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L123">123</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L125">125</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L129">129</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L129">129</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L130">130</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L131">131</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L132">132</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L134">134</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L134">134</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L135">135</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L137">137</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L137">137</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L138">138</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L139">139</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L141">141</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L142">142</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L143">143</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L143">143</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L144">144</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L145">145</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L146">146</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L147">147</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L149">149</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L149">149</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>More than 7 parameters (found 10).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L149">149</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L150">150</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter creator should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L150">150</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L151">151</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter nucleotides should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L151">151</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L152">152</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter codons should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L152">152</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L153">153</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter aminoAcids should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L153">153</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter table should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L153">153</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L154">154</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L154">154</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter trimStops should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L154">154</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'trimStops' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L154">154</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter initMetOnly should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L154">154</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'initMetOnly' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L154">154</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter translateNCodons should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L154">154</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'translateNCodons' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L154">154</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L155">155</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter stopAtStopCodons should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L155">155</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'stopAtStopCodons' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L155">155</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter waitForStartCodon should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L155">155</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'waitForStartCodon' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L155">155</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L157">157</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L158">158</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L159">159</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L160">160</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L162">162</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L162">162</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L163">163</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L164">164</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L164">164</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L167">167</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L168">168</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L169">169</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L171">171</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L171">171</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L172">172</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L173">173</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L174">174</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L174">174</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L175">175</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L176">176</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L178">178</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L179">179</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L180">180</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L180">180</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L181">181</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L182">182</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L183">183</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L185">185</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L186">186</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L186">186</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L187">187</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L187">187</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L188">188</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L189">189</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L191">191</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L191">191</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'createSequences' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L191">191</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L192">192</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L193">193</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter originalSequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L193">193</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'originalSequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L193">193</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 101).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L195">195</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L195">195</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 107).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L197">197</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L197">197</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L198">198</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'3' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L198">198</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L200">200</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L202">202</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L202">202</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L203">203</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L205">205</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L206">206</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L208">208</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L209">209</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L209">209</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L210">210</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L210">210</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L211">211</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L213">213</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L215">215</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L217">217</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L218">218</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expression can be simplified.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L218">218</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L219">219</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L220">220</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L221">221</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L221">221</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L222">222</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L223">223</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L223">223</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L224">224</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L225">225</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L226">226</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L227">227</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L228">228</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L228">228</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L229">229</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L229">229</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L230">230</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L231">231</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L233">233</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L233">233</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L234">234</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L236">236</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L236">236</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L237">237</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L237">237</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L238">238</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L239">239</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L241">241</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L242">242</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L243">243</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L245">245</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L246">246</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L248">248</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L248">248</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L249">249</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L249">249</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L250">250</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L251">251</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L252">252</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'postProcessCompoundLists' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L252">252</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L253">253</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L254">254</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundLists should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L254">254</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compoundLists'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L254">254</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L255">255</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L256">256</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L257">257</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L258">258</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L259">259</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L260">260</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L262">262</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L263">263</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L263">263</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L264">264</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L265">265</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L266">266</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'trimStop' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L266">266</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L266">266</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'sequence'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L266">266</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L267">267</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L268">268</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L269">269</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L270">270</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L271">271</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L272">272</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L273">273</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L274">274</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L275">275</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L276">276</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L278">278</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L279">279</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L280">280</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L281">281</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L283">283</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L284">284</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L284">284</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L285">285</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L285">285</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L286">286</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L287">287</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L288">288</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L288">288</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'translateNCodons' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L288">288</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L289">289</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#L290">290</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.transcription.Table.java">org/biojava/nbio/core/sequence/transcription/Table.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused import - org.biojava.nbio.core.sequence.io.IUPACParser.IUPACTable.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L26">26</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'compound'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L70">70</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L71">71</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L73">73</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter triplet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'triplet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter aminoAcid should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'aminoAcid' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter start should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'start' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter stop should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'stop' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter triplet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'triplet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getOne' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L88">88</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getTwo' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getThree' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'isStart' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L100">100</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L100">100</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'isStop' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L104">104</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L104">104</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getAminoAcid' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getTriplet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L112">112</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L112">112</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'equals' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L116">116</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter obj should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&amp;&amp;' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L121">121</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&amp;&amp;' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&amp;&amp;' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L123">123</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'hashCode' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L129">129</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'toString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L139">139</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'equalsIgnoreCase' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L144">144</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L145">145</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getDescription' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L149">149</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLongName' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L154">154</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getMolecularWeight' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L159">159</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getShortName' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L164">164</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setDescription' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L169">169</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter description should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L170">170</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setLongName' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L174">174</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter longName should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L175">175</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setMolecularWeight' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L179">179</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter molecularWeight should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L180">180</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setShortName' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L184">184</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter shortName should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L185">185</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L196">196</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L197">197</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L198">198</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L200">200</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L201">201</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L202">202</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L203">203</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L205">205</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter one should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L205">205</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'one' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L205">205</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter two should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L206">206</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'two' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L206">206</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter three should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L206">206</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'three' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L206">206</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getOne' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L213">213</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L213">213</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getTwo' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L217">217</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L217">217</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getThree' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L221">221</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L221">221</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'equals' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L225">225</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter obj should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L226">226</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L228">228</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'hashCode' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L235">235</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L237">237</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'toString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L244">244</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L246">246</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L247">247</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L248">248</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L254">254</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L258">258</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'intValue' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L258">258</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'16' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L259">259</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L259">259</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'4' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L260">260</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L260">260</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'compoundToInt' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L264">264</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L264">264</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter c should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L264">264</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'cast' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#L266">266</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.transcription.TranscriptionEngine.java">org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - org.biojava.nbio.core.sequence.compound.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L24">24</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused import - org.biojava.nbio.core.sequence.io.IUPACParser.IUPACTable.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L26">26</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Class TranscriptionEngine should be declared as final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 100).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L66">66</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L70">70</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>More than 7 parameters (found 8).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter table should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'table' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter rnaAminoAcidTranslator should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'rnaAminoAcidTranslator' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter dnaRnaTranslator should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'dnaRnaTranslator' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L86">86</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter proteinSequenceCreator should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'proteinSequenceCreator' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L86">86</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter rnaSequenceCreator should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'rnaSequenceCreator' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter dnaCompounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L88">88</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'dnaCompounds' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter rnaCompounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'rnaCompounds' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L90">90</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter aminoAcidCompounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L90">90</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'aminoAcidCompounds' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L90">90</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L93">93</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L98">98</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L101">101</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L102">102</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L102">102</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L103">103</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L104">104</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L105">105</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L106">106</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L110">110</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter dna should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L111">111</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L111">111</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L112">112</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L113">113</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L114">114</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L116">116</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L116">116</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L118">118</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L119">119</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L120">120</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L121">121</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L123">123</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L124">124</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L125">125</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter dna should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter frames should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 114).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L128">128</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L128">128</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L129">129</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L130">130</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L131">131</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L132">132</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L133">133</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L133">133</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L134">134</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L135">135</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L137">137</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L138">138</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L140">140</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L140">140</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L141">141</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L142">142</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L144">144</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L144">144</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L145">145</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L146">146</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L148">148</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L148">148</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L149">149</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L150">150</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L152">152</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L152">152</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L152">152</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L153">153</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L154">154</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L156">156</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L156">156</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L156">156</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L157">157</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L158">158</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L160">160</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L160">160</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L161">161</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L162">162</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L164">164</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L164">164</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L165">165</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L168">168</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L168">168</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L169">169</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L170">170</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L172">172</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L173">173</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L174">174</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L175">175</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L177">177</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L177">177</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L178">178</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L178">178</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L179">179</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L179">179</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L180">180</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L180">180</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L180">180</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L181">181</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L181">181</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L181">181</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L182">182</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L182">182</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L183">183</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L183">183</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L184">184</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L184">184</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L185">185</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L185">185</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L186">186</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L186">186</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L187">187</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L187">187</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L188">188</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L188">188</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L189">189</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L190">190</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L190">190</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L191">191</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L191">191</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L193">193</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L194">194</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L194">194</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L195">195</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L195">195</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L196">196</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L197">197</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L198">198</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L199">199</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L199">199</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'build' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L199">199</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L200">200</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L201">201</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L201">201</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L202">202</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L202">202</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L203">203</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L203">203</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L204">204</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L206">206</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L207">207</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L208">208</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L208">208</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L209">209</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L210">210</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L211">211</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L211">211</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'table' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L211">211</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter id should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L211">211</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'id'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L211">211</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L212">212</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L213">213</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L214">214</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L216">216</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L216">216</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L217">217</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L217">217</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L218">218</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L219">219</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L220">220</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L220">220</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'table' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L220">220</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter name should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L220">220</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'name'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L220">220</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L221">221</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L222">222</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L223">223</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L225">225</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'table' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L225">225</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L225">225</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter table should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L225">225</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'table' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L225">225</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L226">226</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L227">227</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L228">228</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L230">230</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L230">230</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'dnaCompounds' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L230">230</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L230">230</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L230">230</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L231">231</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L232">232</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L233">233</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L235">235</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L235">235</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'rnaCompounds' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L235">235</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L235">235</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L235">235</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L236">236</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L237">237</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L238">238</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L240">240</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'aminoAcidsCompounds' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L240">240</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L240">240</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L241">241</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L241">241</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L242">242</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L243">243</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L244">244</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L246">246</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'dnaRnaTranslator' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L246">246</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L246">246</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter translator should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L246">246</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L247">247</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L248">248</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L249">249</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L251">251</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'rnaAminoAcidTranslator' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L251">251</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L251">251</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L252">252</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter translator should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L252">252</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L253">253</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L254">254</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L255">255</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L257">257</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'proteinCreator' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L257">257</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L257">257</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L258">258</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L258">258</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter creator should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L258">258</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L259">259</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L260">260</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L261">261</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L263">263</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'rnaCreator' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L263">263</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L263">263</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L264">264</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L264">264</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter creator should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L264">264</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L265">265</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L266">266</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L267">267</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L269">269</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'initMet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L269">269</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L269">269</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter initMet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L269">269</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'initMet' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L269">269</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L270">270</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L271">271</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L272">272</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L274">274</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'trimStop' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L274">274</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L274">274</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter trimStop should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L274">274</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'trimStop' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L274">274</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L275">275</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L276">276</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L277">277</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L279">279</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'translateNCodons' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L279">279</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L279">279</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter translateNCodons should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L279">279</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'translateNCodons' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L279">279</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L280">280</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L281">281</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L282">282</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L284">284</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L284">284</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L285">285</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L285">285</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L286">286</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L287">287</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L288">288</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L288">288</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'stopAtStopCodons' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L288">288</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter stopAtStopCodons should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L288">288</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'stopAtStopCodons' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L288">288</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'stopAtStopCodons'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L288">288</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L289">289</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L290">290</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L291">291</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L293">293</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L293">293</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L294">294</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L294">294</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L295">295</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L296">296</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L297">297</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L297">297</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'waitForStartCodon' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L297">297</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter waitForStartCodon should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L297">297</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'waitForStartCodon' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L297">297</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'waitForStartCodon'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L297">297</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L298">298</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L299">299</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L300">300</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L302">302</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L303">303</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L303">303</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L304">304</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L305">305</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L306">306</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L306">306</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'decorateRna' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L306">306</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter decorateRna should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L306">306</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'decorateRna' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L306">306</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'decorateRna'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L306">306</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L307">307</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L308">308</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L309">309</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L311">311</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L312">312</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L312">312</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L313">313</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L314">314</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L315">315</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L316">316</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L317">317</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L319">319</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L319">319</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L320">320</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L321">321</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L322">322</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L323">323</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L324">324</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L326">326</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L326">326</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L327">327</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L328">328</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L329">329</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L330">330</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L331">331</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L333">333</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L333">333</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L334">334</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L335">335</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L336">336</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L337">337</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 97).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L338">338</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L338">338</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L339">339</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L340">340</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L342">342</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L342">342</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L343">343</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L344">344</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L345">345</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L346">346</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L347">347</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L347">347</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L348">348</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L349">349</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L349">349</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L350">350</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L351">351</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L353">353</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L353">353</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L354">354</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L355">355</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L356">356</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L358">358</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L358">358</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L358">358</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L359">359</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L360">360</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L361">361</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L362">362</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L362">362</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L363">363</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L365">365</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L365">365</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L365">365</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L366">366</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L367">367</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L368">368</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L369">369</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L370">370</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L372">372</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L372">372</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L373">373</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L374">374</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L375">375</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L376">376</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L377">377</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L378">378</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L380">380</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L380">380</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L381">381</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L382">382</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L384">384</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L384">384</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L385">385</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L386">386</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L388">388</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L388">388</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L389">389</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L390">390</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L392">392</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L392">392</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L393">393</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L394">394</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L396">396</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L396">396</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L397">397</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L398">398</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L400">400</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L400">400</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L401">401</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L402">402</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#L403">403</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.views.ComplementSequenceView.java">org/biojava/nbio/core/sequence/views/ComplementSequenceView.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing package-info.java file.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/ComplementSequenceView.html#L0"></a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/ComplementSequenceView.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/ComplementSequenceView.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/ComplementSequenceView.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequenceAsString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/ComplementSequenceView.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundAt' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/ComplementSequenceView.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/ComplementSequenceView.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getIndexOf' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/ComplementSequenceView.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/ComplementSequenceView.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLastIndexOf' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/ComplementSequenceView.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/ComplementSequenceView.html#L63">63</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.views.ReversedSequenceView.java">org/biojava/nbio/core/sequence/views/ReversedSequenceView.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/ReversedSequenceView.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/ReversedSequenceView.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/ReversedSequenceView.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/ReversedSequenceView.html#L42">42</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequenceAsString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/ReversedSequenceView.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'toIndex' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/ReversedSequenceView.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/ReversedSequenceView.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter index should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/ReversedSequenceView.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundAt' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/ReversedSequenceView.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/ReversedSequenceView.html#L57">57</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.views.RnaSequenceView.java">org/biojava/nbio/core/sequence/views/RnaSequenceView.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - org.biojava.nbio.core.sequence.template.*.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L29">29</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 127).</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L42">42</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sourceDna should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sourceDna should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter rnaCompounds should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'rnaCompounds' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L51">51</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequenceAsString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompoundAt' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter position should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getIndexOf' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLastIndexOf' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compound should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getRnaToDna' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getDnaToRna' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'buildTranslators' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L91">91</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 99).</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L102">102</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L106">106</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setCompoundSet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L121">121</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter compoundSet should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 103).</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L123">123</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'setContents' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 103).</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/RnaSequenceView.html#L128">128</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.sequence.views.WindowedSequence.java">org/biojava/nbio/core/sequence/views/WindowedSequence.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L51">51</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'sequence' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter windowSize should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'windowSize' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getWindowSize' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getBackingSequence' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'toStartIndex' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter index should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'index'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getLength' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L90">90</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getCompounds' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter index should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'get' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L105">105</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter index should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'index'.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L105">105</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'iterator' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 108).</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L123">123</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L125">125</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L128">128</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L129">129</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L131">131</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L131">131</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L140">140</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L140">140</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L145">145</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/sequence/views/WindowedSequence.html#L152">152</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.util.CRC64Checksum.java">org/biojava/nbio/core/util/CRC64Checksum.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing package-info.java file.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L0"></a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 103).</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L34">34</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L37">37</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'crcTable' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'256' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'256' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'8' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'update' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter b should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'8' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'0xFF' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'update' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter b should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter offset should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter length should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L62">62</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'update' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter s should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L66">66</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L73">73</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getValue' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'toString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'4' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'0xF' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L86">86</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'16' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L88">88</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L90">90</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'reset' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L93">93</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/CRC64Checksum.html#L95">95</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.util.ConcurrencyTools.java">org/biojava/nbio/core/util/ConcurrencyTools.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.util.concurrent.*.</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L29">29</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L32">32</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 116).</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L33">33</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Class ConcurrencyTools should be declared as final.</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L37">37</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'static' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'logger' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 97).</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter cpus should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L66">66</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter fraction should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 106).</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter threads should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L98">98</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L102">102</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L102">102</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L103">103</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 111).</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter pool should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L111">111</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'pool' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L111">111</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 117).</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L128">128</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'60L' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L135">135</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'60L' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L138">138</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 94).</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L139">139</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L143">143</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter task should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L157">157</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter message should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L157">157</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L158">158</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter task should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/ConcurrencyTools.html#L169">169</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.util.Equals.java">org/biojava/nbio/core/util/Equals.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Utility classes should not have a public or default constructor.</td><td><a href="./xref/org/biojava/nbio/core/util/Equals.html#L31">31</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/Equals.html#L33">33</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter one should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/Equals.html#L33">33</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter two should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/Equals.html#L33">33</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/Equals.html#L37">37</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter one should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/Equals.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter two should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/Equals.html#L37">37</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/Equals.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter one should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/Equals.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter two should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/Equals.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/util/Equals.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter one should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/Equals.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'one'.</td><td><a href="./xref/org/biojava/nbio/core/util/Equals.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter two should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/Equals.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'two'.</td><td><a href="./xref/org/biojava/nbio/core/util/Equals.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 106).</td><td><a href="./xref/org/biojava/nbio/core/util/Equals.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/util/Equals.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter one should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/Equals.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter two should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/Equals.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/util/Equals.html#L85">85</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.util.FlatFileCache.java">org/biojava/nbio/core/util/FlatFileCache.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L21">21</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.io.*.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L30">30</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 149).</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L32">32</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L33">33</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Class FlatFileCache should be declared as final.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'static' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'logger' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 94).</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter key should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter fileToCache should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 103).</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L70">70</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L71">71</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Must have at least one statement.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'cast' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 104).</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Inner assignments should be avoided.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'-' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'-' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 111).</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L90">90</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L93">93</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L98">98</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L98">98</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L100">100</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L102">102</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L102">102</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter key should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L102">102</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L102">102</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 107).</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L103">103</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L103">103</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L104">104</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L105">105</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L105">105</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L106">106</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L110">110</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L112">112</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L112">112</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L113">113</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L113">113</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L113">113</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L114">114</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'else' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L116">116</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L117">117</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L118">118</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L119">119</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L119">119</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L120">120</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L120">120</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L121">121</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L122">122</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L123">123</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L123">123</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L123">123</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L124">124</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L125">125</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#L127">127</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.util.Hashcoder.java">org/biojava/nbio/core/util/Hashcoder.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>File does not end with a newline.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L0"></a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Utility classes should not have a public or default constructor.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter seed should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter b should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter seed should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter c should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter seed should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'seed'.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter i should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'i'.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter seed should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'seed'.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter l should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'l'.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'32' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter seed should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'seed'.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter f should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'f'.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter seed should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'seed'.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter d should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'd'.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L106">106</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter seed should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L106">106</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'seed'.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L106">106</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter o should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L106">106</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'o'.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L106">106</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L111">111</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/Hashcoder.html#L121">121</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.util.InputStreamProvider.java">org/biojava/nbio/core/util/InputStreamProvider.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L19">19</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.io.*.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L25">25</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 103).</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L66">66</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'cache' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L68">68</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getInputStream' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter pathToFile should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L88">88</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L102">102</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter in should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L102">102</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L103">103</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'0xff' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'8' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'0xff' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L111">111</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getInputStream' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L117">117</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter u should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L118">118</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L122">122</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L123">123</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L126">126</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>')' is preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L128">128</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>')' is preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L131">131</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L132">132</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L133">133</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L135">135</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L147">147</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L148">148</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L153">153</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getInputStream' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L153">153</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter f should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L153">153</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L154">154</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L155">155</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L157">157</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L158">158</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>')' is preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L172">172</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L175">175</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>')' is preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L177">177</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L178">178</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L179">179</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L181">181</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L183">183</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L183">183</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L185">185</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L185">185</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L188">188</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L193">193</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L193">193</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L197">197</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L200">200</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L200">200</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L202">202</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L205">205</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L210">210</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L210">210</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L214">214</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L216">216</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L216">216</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L218">218</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L219">219</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L222">222</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L226">226</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 128).</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L234">234</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L235">235</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L240">240</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L240">240</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter f should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L240">240</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'f'.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L240">240</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'FileNotFoundException'.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L240">240</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L240">240</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L245">245</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>')' is preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L245">245</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L245">245</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L248">248</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L248">248</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L249">249</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L254">254</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L254">254</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L255">255</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L255">255</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L256">256</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L261">261</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter f should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L261">261</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L262">262</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L269">269</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter u should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L269">269</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L270">270</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L278">278</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L278">278</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter f should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L278">278</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L279">279</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L286">286</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L286">286</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter u should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L286">286</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#L287">287</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.util.PrettyXMLWriter.java">org/biojava/nbio/core/util/PrettyXMLWriter.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.util.*.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L26">26</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L37">37</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter writer should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'writer' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'declareNamespace' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter nsURI should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter prefixHint should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L66">66</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L71">71</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'writeIndent' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L90">90</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name '_openTag' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L90">90</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L105">105</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L106">106</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter nsURI should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L106">106</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L118">118</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'openTag' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L119">119</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L120">120</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter nsURI should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L120">120</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter localName should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L120">120</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L122">122</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L123">123</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L124">124</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L124">124</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L125">125</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L125">125</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L143">143</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'openTag' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L144">144</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L145">145</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter qName should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L145">145</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L147">147</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'attribute' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L154">154</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L155">155</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter nsURI should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L155">155</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter localName should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L155">155</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter value should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L155">155</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L157">157</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'!' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L158">158</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L167">167</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L176">176</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'attribute' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L177">177</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L178">178</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter qName should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L178">178</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter value should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L178">178</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L180">180</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'!' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L181">181</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L192">192</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name '_closeTag' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L192">192</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L197">197</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L197">197</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L202">202</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'closeTag' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L203">203</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L204">204</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter nsURI should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L204">204</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter localName should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L204">204</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L206">206</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L209">209</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L227">227</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'closeTag' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L228">228</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L229">229</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L229">229</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter qName should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L229">229</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L231">231</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'println' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L247">247</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L248">248</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter data should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L248">248</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L250">250</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L251">251</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L252">252</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L253">253</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L254">254</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L255">255</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L256">256</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L257">257</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'print' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L260">260</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L261">261</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter data should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L261">261</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L263">263</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L264">264</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L265">265</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L266">266</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L267">267</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L268">268</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L269">269</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'printRaw' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L272">272</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L273">273</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter data should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L273">273</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L275">275</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L276">276</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L279">279</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'printChars' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L279">279</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L279">279</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter data should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L279">279</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L281">281</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L282">282</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L283">283</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L284">284</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L285">285</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L287">287</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L288">288</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L289">289</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L290">290</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L291">291</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L292">292</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L293">293</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L294">294</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L297">297</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'printAttributeValue' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L297">297</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L297">297</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter data should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L297">297</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L299">299</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L300">300</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L301">301</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L302">302</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L303">303</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L305">305</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L306">306</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L307">307</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L308">308</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L309">309</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L310">310</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L311">311</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L312">312</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'numericalEntity' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L315">315</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L315">315</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter c should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L315">315</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L317">317</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L318">318</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L319">319</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L320">320</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L322">322</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'close' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L323">323</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L324">324</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/PrettyXMLWriter.html#L326">326</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.util.SequenceTools.java">org/biojava/nbio/core/util/SequenceTools.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L28">28</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L30">30</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L30">30</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L32">32</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L33">33</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L33">33</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L34">34</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 147).</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L35">35</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L35">35</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L37">37</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter string should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter n should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L37">37</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L42">42</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 145).</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for '&lt;T&gt;'.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter array should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter fill should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter n should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 108).</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L62">62</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L66">66</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L73">73</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'100' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L93">93</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L98">98</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L100">100</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L106">106</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getSequenceFromString' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter sequence should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L110">110</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L111">111</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L112">112</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L113">113</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L114">114</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L117">117</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SequenceTools.html#L117">117</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.util.SingleLinkageClusterer.java">org/biojava/nbio/core/util/SingleLinkageClusterer.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.util.*.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L26">26</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L30">30</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L32">32</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L34">34</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 99).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L39">39</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'logger' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L39">39</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L42">42</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L43">43</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L43">43</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L44">44</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L45">45</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L46">46</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter first should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'first' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter second should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'second' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L47">47</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter minDistance should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L47">47</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L48">48</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L49">49</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L52">52</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L52">52</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L56">56</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L60">60</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L62">62</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L66">66</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L68">68</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L72">72</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L76">76</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L77">77</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L78">78</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L79">79</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L82">82</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L82">82</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L83">83</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L86">86</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L88">88</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L89">89</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L90">90</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L90">90</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L91">91</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L93">93</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L95">95</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 94).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 103).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L97">97</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L98">98</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L98">98</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 111).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 135).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L100">100</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L100">100</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 107).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L101">101</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L101">101</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L102">102</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L102">102</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Unused @throws tag for 'IllegalArgumentException'.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L102">102</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L103">103</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L104">104</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L104">104</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter matrix should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L104">104</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'matrix' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L104">104</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter isScoreMatrix should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L104">104</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'isScoreMatrix' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L104">104</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L105">105</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L106">106</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'!=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'!=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L108">108</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 117).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L110">110</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L112">112</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L113">113</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L113">113</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L114">114</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L116">116</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L116">116</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L117">117</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L119">119</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L120">120</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L120">120</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getDendrogram' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L120">120</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L121">121</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L121">121</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L121">121</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L122">122</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L123">123</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L124">124</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L124">124</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L125">125</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L126">126</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L128">128</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L128">128</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L129">129</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L129">129</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 106).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L130">130</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L130">130</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L131">131</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L132">132</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L133">133</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L134">134</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L134">134</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L135">135</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'-' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L135">135</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'-' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L135">135</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L137">137</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L137">137</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L138">138</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L138">138</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L138">138</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L138">138</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L138">138</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L139">139</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L139">139</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L140">140</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L140">140</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L141">141</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L141">141</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L141">141</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L141">141</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L141">141</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L141">141</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'-' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L141">141</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'-' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L141">141</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L141">141</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L142">142</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L142">142</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L143">143</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L144">144</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L145">145</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L146">146</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L147">147</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L147">147</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L148">148</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 133).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L149">149</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L149">149</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L150">150</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L151">151</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L152">152</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L153">153</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L153">153</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L154">154</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L155">155</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L155">155</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L156">156</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L156">156</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 104).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L157">157</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L157">157</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L158">158</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L159">159</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L160">160</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter closestPair should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L160">160</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'closestPair'.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L160">160</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L161">161</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L161">161</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L162">162</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L162">162</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L163">163</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L164">164</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L165">165</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L165">165</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 140).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L167">167</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L167">167</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L167">167</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L168">168</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L169">169</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L169">169</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L170">170</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L171">171</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L171">171</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L172">172</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 92).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L173">173</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L173">173</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L174">174</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L175">175</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L176">176</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L177">177</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L178">178</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L178">178</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter d1 should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L178">178</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'd1'.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L178">178</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter d2 should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L178">178</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'd2'.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L178">178</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L179">179</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L180">180</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L180">180</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L181">181</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L182">182</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L182">182</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L183">183</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L184">184</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L185">185</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L185">185</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L186">186</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L186">186</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter first should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L186">186</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter second should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L186">186</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L187">187</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L188">188</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L189">189</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L189">189</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L190">190</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L190">190</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter m should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L190">190</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L191">191</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L191">191</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L192">192</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L192">192</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L192">192</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L193">193</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L195">195</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L195">195</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L195">195</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L195">195</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L195">195</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L195">195</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L195">195</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L196">196</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L197">197</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L197">197</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L198">198</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L199">199</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L199">199</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L200">200</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L200">200</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L200">200</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L200">200</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L201">201</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L201">201</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L202">202</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'-' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L202">202</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'-' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L202">202</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L203">203</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L204">204</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L204">204</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L205">205</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L205">205</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L205">205</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L206">206</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L206">206</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L207">207</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L208">208</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L208">208</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L209">209</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L210">210</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L211">211</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L212">212</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L212">212</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L213">213</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 96).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L213">213</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L213">213</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L214">214</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L214">214</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L214">214</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L214">214</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L215">215</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L215">215</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L216">216</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&gt;=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L216">216</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&gt;=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L216">216</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L217">217</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L218">218</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L218">218</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L218">218</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L218">218</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L219">219</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L221">221</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L222">222</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L223">223</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L224">224</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L225">225</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L226">226</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L226">226</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L227">227</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L228">228</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L228">228</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L228">228</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L228">228</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L229">229</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L229">229</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L230">230</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L230">230</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L230">230</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L231">231</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L232">232</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L232">232</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L232">232</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L232">232</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L233">233</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L235">235</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L236">236</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L237">237</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L238">238</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L238">238</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L239">239</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L240">240</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L241">241</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L241">241</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L242">242</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L242">242</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L243">243</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L244">244</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L245">245</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L246">246</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L247">247</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L247">247</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'getClusters' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L247">247</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter cutoff should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L247">247</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'cutoff'.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L247">247</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L248">248</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L248">248</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L249">249</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L249">249</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L249">249</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L250">250</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L251">251</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L252">252</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L252">252</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L253">253</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L253">253</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L254">254</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L254">254</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L255">255</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L256">256</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L256">256</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L257">257</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L257">257</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L257">257</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L257">257</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L257">257</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L257">257</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'-' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L257">257</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'-' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L257">257</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L257">257</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L258">258</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L258">258</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L259">259</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L260">260</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L260">260</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 106).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L261">261</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L261">261</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L262">262</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L262">262</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L263">263</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L264">264</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L265">265</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L266">266</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L266">266</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L267">267</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L267">267</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L268">268</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L268">268</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L269">269</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L270">270</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L271">271</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L271">271</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L271">271</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L272">272</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L273">273</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L274">274</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L275">275</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L275">275</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L276">276</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L276">276</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L277">277</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L277">277</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L277">277</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L277">277</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L277">277</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 126).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L278">278</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L278">278</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L279">279</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L279">279</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L280">280</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L281">281</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L282">282</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L283">283</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L284">284</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L284">284</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'!=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L284">284</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'!=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L284">284</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L284">284</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L284">284</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L285">285</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 176).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L285">285</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L285">285</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 100).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L286">286</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L286">286</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L287">287</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L287">287</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'!=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L287">287</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'!=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L287">287</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L287">287</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L287">287</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L288">288</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 176).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L288">288</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L288">288</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L289">289</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 136).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L289">289</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L289">289</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L290">290</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L291">291</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L291">291</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 123).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L292">292</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L292">292</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 97).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L293">293</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L293">293</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 99).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L294">294</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L294">294</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L295">295</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L295">295</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L295">295</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L295">295</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 127).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L296">296</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L296">296</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L296">296</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L296">296</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L296">296</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L296">296</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L296">296</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L296">296</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L297">297</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L298">298</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L298">298</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L299">299</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L299">299</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L300">300</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L301">301</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 128).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L301">301</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L301">301</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L302">302</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 127).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L303">303</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L303">303</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L303">303</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L303">303</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L303">303</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L303">303</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L303">303</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L303">303</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L304">304</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L305">305</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L305">305</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L306">306</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L306">306</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L307">307</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L308">308</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L308">308</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L309">309</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L310">310</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L311">311</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L311">311</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L312">312</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L312">312</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L312">312</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L312">312</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L313">313</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L313">313</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L314">314</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L315">315</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L315">315</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L316">316</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L316">316</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L316">316</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L316">316</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L317">317</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L317">317</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L318">318</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L319">319</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L320">320</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L320">320</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 133).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L321">321</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L321">321</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 95).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L322">322</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L322">322</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L322">322</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L323">323</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L324">324</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L324">324</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L325">325</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L325">325</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L326">326</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L327">327</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L328">328</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L328">328</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L329">329</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L329">329</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L330">330</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L330">330</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L330">330</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L330">330</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L330">330</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L330">330</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L330">330</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L331">331</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L332">332</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L332">332</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L333">333</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L334">334</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L335">335</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L336">336</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L337">337</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L338">338</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L338">338</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L339">339</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L340">340</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L341">341</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L342">342</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L343">343</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L345">345</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L347">347</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L347">347</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L347">347</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L349">349</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L350">350</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L351">351</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L351">351</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L352">352</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L352">352</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter i should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L352">352</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter cutoff should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L352">352</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L353">353</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L354">354</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Conditional logic can be removed.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L354">354</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&gt;' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L354">354</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&gt;' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L354">354</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L355">355</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L356">356</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L357">357</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L358">358</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L359">359</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L360">360</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Conditional logic can be removed.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L360">360</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L360">360</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L360">360</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L361">361</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L362">362</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L363">363</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L364">364</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L364">364</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L365">365</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L366">366</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L367">367</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L367">367</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L368">368</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L368">368</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L368">368</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter finalClusters should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L368">368</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L368">368</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L369">369</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L370">370</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L371">371</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L371">371</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L371">371</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L372">372</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L373">373</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L373">373</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L373">373</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L374">374</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L375">375</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L376">376</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L377">377</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L378">378</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L379">379</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L379">379</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L380">380</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L380">380</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L381">381</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L382">382</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L382">382</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L382">382</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L382">382</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L382">382</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L382">382</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L382">382</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L383">383</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L383">383</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L383">383</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L383">383</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L383">383</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L383">383</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>';' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L383">383</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L384">384</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L384">384</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L384">384</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L385">385</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L385">385</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L386">386</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L386">386</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L387">387</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L387">387</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'&lt;' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L387">387</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L388">388</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 115).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L388">388</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L388">388</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L388">388</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L388">388</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L388">388</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'else' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L389">389</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L389">389</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L389">389</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L389">389</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L390">390</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' should be on the same line.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L390">390</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L391">391</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L392">392</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 115).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L392">392</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L392">392</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L392">392</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L392">392</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L392">392</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'else' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L393">393</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L393">393</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L393">393</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>',' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L393">393</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L394">394</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L395">395</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L396">396</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L397">397</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L398">398</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L399">399</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#L400">400</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.util.SoftHashMap.java">org/biojava/nbio/core/util/SoftHashMap.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>File does not end with a newline.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L0"></a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.util.*.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L27">27</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L31">31</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L32">32</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 115).</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L32">32</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;K&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L36">36</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Type Javadoc comment is missing an @param &lt;V&gt; tag.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L36">36</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 88).</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'static' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L38">38</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'logger' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L38">38</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L40">40</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L44">44</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'max_limit' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L48">48</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'1000' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L64">64</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter hardSize should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L64">64</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'get' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter key should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'soft_ref' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L77">77</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L111">111</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L124">124</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L125">125</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Class SoftValue should be declared as final.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L146">146</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L148">148</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter k should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'k'.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter key should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'key' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'key'.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter q should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L166">166</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'q'.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L188">188</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L211">211</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'put' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L211">211</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter key should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L212">212</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'key'.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L212">212</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter value should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L212">212</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'value'.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L212">212</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L217">217</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L219">219</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L226">226</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'remove' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L226">226</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L227">227</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter key should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L227">227</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L228">228</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L229">229</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L230">230</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L231">231</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'clear' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L235">235</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L238">238</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'size' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L250">250</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'entrySet' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/SoftHashMap.html#L261">261</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.util.StringManipulationHelper.java">org/biojava/nbio/core/util/StringManipulationHelper.java</h3><table border="0" class="bodyTable"><tr class="b"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L21">21</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.io.*.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L37">37</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L46">46</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Class StringManipulationHelper should be declared as final.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 101).</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L51">51</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'static' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L51">51</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'logger' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L53">53</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L53">53</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L54">54</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L54">54</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L55">55</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L56">56</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L61">61</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L61">61</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L62">62</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L65">65</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L65">65</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L67">67</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L67">67</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L69">69</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L71">71</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L72">72</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter stream should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'stream'.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 90).</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L75">75</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L75">75</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L76">76</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L78">78</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L84">84</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Must have at least one statement.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L84">84</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L85">85</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L88">88</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L89">89</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L90">90</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L94">94</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L96">96</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L97">97</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L98">98</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L98">98</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 84).</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L99">99</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L100">100</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L100">100</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L101">101</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L102">102</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L103">103</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L103">103</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L104">104</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L105">105</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L106">106</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter expected should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'expected'.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter actual should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L107">107</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'actual'.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L110">110</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L111">111</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L112">112</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L113">113</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L114">114</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L116">116</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L118">118</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L119">119</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L120">120</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'!' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L120">120</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L121">121</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L122">122</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L123">123</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L124">124</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L126">126</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L128">128</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L129">129</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L130">130</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L132">132</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L133">133</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L134">134</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L135">135</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L136">136</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L137">137</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L137">137</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L138">138</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L138">138</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter expected should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L138">138</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter actual should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L138">138</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L139">139</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L139">139</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L139">139</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L140">140</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L140">140</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L140">140</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L141">141</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 109).</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L142">142</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L142">142</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 102).</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L143">143</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L143">143</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 112).</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L144">144</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L144">144</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 108).</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L145">145</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L145">145</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L146">146</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L147">147</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L148">148</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L149">149</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L150">150</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L151">151</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L152">152</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L153">153</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L154">154</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L155">155</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L156">156</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L157">157</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L158">158</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L158">158</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'==' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L158">158</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L159">159</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L160">160</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'else' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L160">160</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L160">160</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'!=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L160">160</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L161">161</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L161">161</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L162">162</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L163">163</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L163">163</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L164">164</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L164">164</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L165">165</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'=' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L165">165</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L166">166</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L167">167</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L168">168</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'else' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L168">168</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'}' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L168">168</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'else' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L168">168</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' is not preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L168">168</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L169">169</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L170">170</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L171">171</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L171">171</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L172">172</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L173">173</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L173">173</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L174">174</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L175">175</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L176">176</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L176">176</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L177">177</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L177">177</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter s should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L177">177</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter n should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L177">177</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L178">178</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L178">178</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L179">179</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L180">180</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L180">180</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L181">181</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L181">181</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter s should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L181">181</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter n should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L181">181</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L182">182</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L182">182</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L183">183</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L184">184</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L184">184</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L185">185</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L185">185</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L185">185</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter s should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L185">185</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter delimiter should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L185">185</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L186">186</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L186">186</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L187">187</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L188">188</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L189">189</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'while' is not followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L189">189</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L189">189</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>')' is preceded with whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L189">189</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L190">190</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'{' should be on the previous line.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L190">190</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L191">191</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L192">192</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L193">193</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L194">194</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L195">195</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#L195">195</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.util.UncompressInputStream.java">org/biojava/nbio/core/util/UncompressInputStream.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.io.*.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Extra HTML tag found: &lt;/var&gt; code in &lt;var&gt;gzip-1.2.4&lt;/var&gt; (written</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Extra HTML tag found: &lt;/var&gt; (written</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 98).</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'static' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'logger' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line contains a tab character.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L83">83</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter is should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L87">87</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L93">93</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Variable 'one' must be private and have accessor methods.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L93">93</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'read' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L95">95</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L98">98</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'0xff' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L99">99</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'else' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L100">100</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L106">106</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L107">107</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'tab_prefix' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L110">110</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'tab_suffix' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L111">111</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'256' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L111">111</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L112">112</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'block_mode' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L115">115</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L116">116</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'n_bits' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L116">116</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L118">118</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L119">119</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L120">120</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L121">121</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L122">122</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L123">123</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L124">124</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'free_ent' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L124">124</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L127">127</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'10000' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L127">127</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L128">128</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'bit_pos' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L128">128</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L129">129</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L130">130</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'read' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L133">133</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method length is 208 lines (max allowed is 150).</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L133">133</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter buf should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L134">134</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter off should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L134">134</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter len should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L134">134</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L136">136</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'l_tab_prefix' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L142">142</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'l_tab_suffix' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L143">143</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'l_stack' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L144">144</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'l_n_bits' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L145">145</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'l_maxcode' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L146">146</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'l_maxmaxcode' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L147">147</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'l_bitmask' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L148">148</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'l_oldcode' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L149">149</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'l_finchar' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L150">150</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'l_stackp' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L151">151</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'l_free_ent' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L152">152</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'l_data' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L153">153</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'l_bit_pos' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L154">154</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 's_size' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L159">159</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L161">161</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L177">177</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'bit_in' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L179">179</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L179">179</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'3' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L179">179</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>':' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L179">179</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'3' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L180">180</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'n_bytes' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L201">201</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'3' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L201">201</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L202">202</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L206">206</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>':' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L206">206</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'3' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L219">219</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'0xFF' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L220">220</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'0xFF' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L220">220</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'8' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L220">220</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'|' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L220">220</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'0xFF' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L221">221</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'16' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L221">221</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'0x7' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L222">222</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L229">229</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'256' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L229">229</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L230">230</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Inner assignments should be avoided.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L232">232</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'n_bytes' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L245">245</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'3' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L245">245</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L246">246</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L268">268</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L269">269</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'256' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L279">279</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L291">291</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected an @return tag.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L349">349</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'final' modifier.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L349">349</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter bit_pos should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L349">349</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'bit_pos' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L349">349</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'bit_pos'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L349">349</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'bit_pos' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L349">349</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'3' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L350">350</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L357">357</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'final' modifier.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L357">357</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L359">359</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'skip' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L363">363</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter num should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L364">364</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'got' hides a field.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L366">366</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L368">368</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'else' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L370">370</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'available' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L375">375</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L377">377</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L383">383</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L384">384</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L385">385</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L386">386</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L387">387</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L388">388</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L389">389</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L391">391</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'parse_header' must match pattern '^[a-z][a-zA-Z0-9]*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L391">391</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L395">395</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'0xff' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L396">396</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'8' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L396">396</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L398">398</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'0xff' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L399">399</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L400">400</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L401">401</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L402">402</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L409">409</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L414">414</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L415">415</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'+' should be on a new line.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L416">416</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L419">419</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L422">422</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Avoid inline conditionals.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L437">437</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'256' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L437">437</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'for' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L444">444</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'255' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L444">444</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Method 'markSupported' is not designed for extension - needs to be abstract, final or empty.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L453">453</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 94).</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L458">458</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L458">458</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'public' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L458">458</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L458">458</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter fileInName should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L458">458</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter out should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L458">458</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L461">461</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'(' is followed by whitespace.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L461">461</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'100000' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L464">464</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L467">467</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'1000' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L477">477</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L482">482</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Name 'debugTiming' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L482">482</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L484">484</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter args should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L484">484</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Array brackets at illegal position.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L484">484</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'100000' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L493">493</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'if' construct must use '{}'s.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L499">499</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'1000' is a magic number.</td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#L506">506</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.util.XMLHelper.java">org/biojava/nbio/core/util/XMLHelper.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Using the '.*' form of import should be avoided - java.io.*.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L45">45</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Utility classes should not have a public or default constructor.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L55">55</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'public' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter parentElement should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L57">57</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter elementName should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L57">57</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L58">58</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 82).</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L63">63</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'public' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L63">63</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 117).</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'public' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L74">74</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter fileName should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L74">74</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 139).</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'public' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter inputStream should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L81">81</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 113).</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'public' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter document should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter outputStream should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L96">96</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 113).</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L105">105</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'public' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L105">105</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter document should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L105">105</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter outputStream should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L105">105</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 83).</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L118">118</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'public' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L118">118</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter element should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L118">118</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter parentName should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L118">118</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 120).</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L129">129</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L129">129</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'public' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L129">129</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter element should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L129">129</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter xpathExpression should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L129">129</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 108).</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L134">134</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 99).</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L146">146</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 126).</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L151">151</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Missing a Javadoc comment.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L151">151</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>'public' modifier out of order with the JLS suggestions.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L151">151</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter element should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L151">151</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Parameter xpathExpression should be final.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L151">151</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 108).</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L160">160</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 105).</td><td><a href="./xref/org/biojava/nbio/core/util/XMLHelper.html#L166">166</a></td></tr></table></div><div class="section"><h3 id="org.biojava.nbio.core.util.XMLWriter.java">org/biojava/nbio/core/util/XMLWriter.java</h3><table border="0" class="bodyTable"><tr class="a"><th>Severity</th><th>Message</th><th>Line</th></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L40">40</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L41">41</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L41">41</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L42">42</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L49">49</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L50">50</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L50">50</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L51">51</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L58">58</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L59">59</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L59">59</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L60">60</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L69">69</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 91).</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L70">70</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L70">70</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L70">70</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L71">71</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 93).</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L73">73</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L79">79</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L80">80</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L80">80</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L81">81</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L85">85</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L86">86</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'data'.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L86">86</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L86">86</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L87">87</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L91">91</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @param tag for 'data'.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L92">92</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L92">92</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L93">93</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>First sentence should end with a period.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L94">94</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L100">100</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L101">101</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L101">101</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L102">102</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L105">105</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L108">108</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L109">109</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L109">109</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L110">110</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 81).</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L112">112</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 86).</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L113">113</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L114">114</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 87).</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L115">115</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 89).</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L116">116</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L117">117</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L123">123</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line is longer than 80 characters (found 85).</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L124">124</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L124">124</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L124">124</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L125">125</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Line has trailing spaces.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L131">131</a></td></tr><tr class="b"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Redundant 'public' modifier.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L132">132</a></td></tr><tr class="a"><td><img src="images/icon_error_sml.gif" alt="" />&#160;Error</td><td>Expected @throws tag for 'IOException'.</td><td><a href="./xref/org/biojava/nbio/core/util/XMLWriter.html#L132">132</a></td></tr></table></div></div>
      </div>
    </div>
    <div class="clear">
      <hr/>
    </div>
    <div id="footer">
      <div class="xright">
              Copyright &#169;                    2000-2015
                        <a href="http://www.biojava.org">BioJava</a>.
            All Rights Reserved.      
        
                  </div>
      <div class="clear">
        <hr/>
      </div>
    </div>
  </body>
</html>
