<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!-- Generated by Apache Maven Doxia at Jan 30, 2015 -->
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
    <title>FindBugs Bug Detector Report</title>
    <style type="text/css" media="all">
      @import url("./css/maven-base.css");
      @import url("./css/maven-theme.css");
      @import url("./css/site.css");
    </style>
    <link rel="stylesheet" href="./css/print.css" type="text/css" media="print" />
    <meta name="Date-Revision-yyyymmdd" content="20150130" />
    <meta http-equiv="Content-Language" content="en" />
        
        </head>
  <body class="composite">
    <div id="banner">
                    <div id="bannerLeft">
                biojava-core
                </div>
                    <div class="clear">
        <hr/>
      </div>
    </div>
    <div id="breadcrumbs">
            
        
                <div class="xleft">
        <span id="publishDate">Last Published: 2015-01-30</span>
                  &nbsp;| <span id="projectVersion">Version: 4.0.0</span>
                      </div>
            <div class="xright">                    <a href="./" title="biojava-core">biojava-core</a>
              
        
      </div>
      <div class="clear">
        <hr/>
      </div>
    </div>
    <div id="leftColumn">
      <div id="navcolumn">
             
        
                                <h5>Parent Project</h5>
                  <ul>
                  <li class="none">
                          <a href="../index.html" title="biojava">biojava</a>
            </li>
          </ul>
                       <h5>Project Documentation</h5>
                  <ul>
                                                                                                                                                                                <li class="collapsed">
                          <a href="project-info.html" title="Project Information">Project Information</a>
                  </li>
                                                                                                                                    <li class="expanded">
                          <a href="project-reports.html" title="Project Reports">Project Reports</a>
                    <ul>
                      <li class="none">
            <strong>FindBugs Report</strong>
          </li>
                      <li class="none">
                          <a href="checkstyle.html" title="Checkstyle">Checkstyle</a>
            </li>
                      <li class="none">
                          <a href="xref-test/index.html" title="Test Source Xref">Test Source Xref</a>
            </li>
                      <li class="none">
                          <a href="xref/index.html" title="Source Xref">Source Xref</a>
            </li>
              </ul>
        </li>
          </ul>
                             <a href="http://maven.apache.org/" title="Built by Maven" class="poweredBy">
        <img class="poweredBy" alt="Built by Maven" src="./images/logos/maven-feather.png" />
      </a>
                   
        
            </div>
    </div>
    <div id="bodyColumn">
      <div id="contentBox">
        <div class="section"><h2>FindBugs Bug Detector Report<a name="FindBugs_Bug_Detector_Report"></a></h2><p>The following document contains the results of <a class="externalLink" href="http://findbugs.sourceforge.net">FindBugs Report</a></p><p>FindBugs Version is <i>2.0.2</i></p><p>Threshold is <i>medium</i></p><p>Effort is <i>min</i></p></div><div class="section"><h2>Summary<a name="Summary"></a></h2><table border="0" class="bodyTable"><tr class="a"><th>Classes</th><th>Bugs</th><th>Errors</th><th>Missing Classes</th></tr><tr class="b"><td>198</td><td>79</td><td>0</td><td>0</td></tr></table></div><div class="section"><h2>Files<a name="Files"></a></h2><table border="0" class="bodyTable"><tr class="a"><th>Class</th><th>Bugs</th></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.AccessionID">org.biojava.nbio.core.sequence.AccessionID</a></td><td>1</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.CDSComparator">org.biojava.nbio.core.sequence.CDSComparator</a></td><td>1</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.ExonComparator">org.biojava.nbio.core.sequence.ExonComparator</a></td><td>1</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.GeneSequence">org.biojava.nbio.core.sequence.GeneSequence</a></td><td>1</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.SequenceComparator">org.biojava.nbio.core.sequence.SequenceComparator</a></td><td>1</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.StartCodonSequence">org.biojava.nbio.core.sequence.StartCodonSequence</a></td><td>1</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.StopCodonSequence">org.biojava.nbio.core.sequence.StopCodonSequence</a></td><td>1</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.TranscriptSequence">org.biojava.nbio.core.sequence.TranscriptSequence</a></td><td>2</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.io.FastaGeneWriter">org.biojava.nbio.core.sequence.io.FastaGeneWriter</a></td><td>1</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.io.FastaReader">org.biojava.nbio.core.sequence.io.FastaReader</a></td><td>3</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.io.FastaWriter">org.biojava.nbio.core.sequence.io.FastaWriter</a></td><td>4</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.io.GenbankReader">org.biojava.nbio.core.sequence.io.GenbankReader</a></td><td>1</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.io.GenbankWriter">org.biojava.nbio.core.sequence.io.GenbankWriter</a></td><td>1</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.io.GenericFastaHeaderParser">org.biojava.nbio.core.sequence.io.GenericFastaHeaderParser</a></td><td>1</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.io.GenericGenbankHeaderFormat">org.biojava.nbio.core.sequence.io.GenericGenbankHeaderFormat</a></td><td>3</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.io.GenericGenbankHeaderParser">org.biojava.nbio.core.sequence.io.GenericGenbankHeaderParser</a></td><td>1</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.io.GenericInsdcHeaderFormat">org.biojava.nbio.core.sequence.io.GenericInsdcHeaderFormat</a></td><td>2</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.io.util.ClasspathResource">org.biojava.nbio.core.sequence.io.util.ClasspathResource</a></td><td>1</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.io.util.IOUtils">org.biojava.nbio.core.sequence.io.util.IOUtils</a></td><td>2</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.loader.GenbankProxySequenceReader">org.biojava.nbio.core.sequence.loader.GenbankProxySequenceReader</a></td><td>4</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.loader.SequenceFileProxyLoader">org.biojava.nbio.core.sequence.loader.SequenceFileProxyLoader</a></td><td>2</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.loader.UniprotProxySequenceReader">org.biojava.nbio.core.sequence.loader.UniprotProxySequenceReader</a></td><td>8</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.location.FuzzyPoint">org.biojava.nbio.core.sequence.location.FuzzyPoint</a></td><td>1</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.location.InsdcParser">org.biojava.nbio.core.sequence.location.InsdcParser</a></td><td>1</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.location.SimplePoint">org.biojava.nbio.core.sequence.location.SimplePoint</a></td><td>1</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.location.template.AbstractLocation">org.biojava.nbio.core.sequence.location.template.AbstractLocation</a></td><td>1</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.location.template.Location$Tools">org.biojava.nbio.core.sequence.location.template.Location$Tools</a></td><td>3</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.storage.ArrayListSequenceReader">org.biojava.nbio.core.sequence.storage.ArrayListSequenceReader</a></td><td>1</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.storage.BitSequenceReader">org.biojava.nbio.core.sequence.storage.BitSequenceReader</a></td><td>1</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.storage.BitSequenceReader$BitArrayWorker">org.biojava.nbio.core.sequence.storage.BitSequenceReader$BitArrayWorker</a></td><td>3</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.storage.SingleCompoundSequenceReader">org.biojava.nbio.core.sequence.storage.SingleCompoundSequenceReader</a></td><td>1</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.template.AbstractNucleotideCompoundSet">org.biojava.nbio.core.sequence.template.AbstractNucleotideCompoundSet</a></td><td>1</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.template.AbstractSequence">org.biojava.nbio.core.sequence.template.AbstractSequence</a></td><td>1</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.sequence.transcription.Table$CaseInsensitiveTriplet">org.biojava.nbio.core.sequence.transcription.Table$CaseInsensitiveTriplet</a></td><td>1</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.sequence.transcription.Table$Codon">org.biojava.nbio.core.sequence.transcription.Table$Codon</a></td><td>1</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.util.FlatFileCache">org.biojava.nbio.core.util.FlatFileCache</a></td><td>1</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.util.InputStreamProvider">org.biojava.nbio.core.util.InputStreamProvider</a></td><td>2</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.util.SingleLinkageClusterer">org.biojava.nbio.core.util.SingleLinkageClusterer</a></td><td>3</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.util.SingleLinkageClusterer$LinkedPair">org.biojava.nbio.core.util.SingleLinkageClusterer$LinkedPair</a></td><td>1</td></tr><tr class="a"><td><a href="#org.biojava.nbio.core.util.StringManipulationHelper">org.biojava.nbio.core.util.StringManipulationHelper</a></td><td>2</td></tr><tr class="b"><td><a href="#org.biojava.nbio.core.util.UncompressInputStream">org.biojava.nbio.core.util.UncompressInputStream</a></td><td>10</td></tr></table></div><a name="org.biojava.nbio.core.sequence.AccessionID"></a><div class="section"><h3>org.biojava.nbio.core.sequence.AccessionID<a name="org.biojava.nbio.core.sequence.AccessionID"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>Equals method for org.biojava.nbio.core.sequence.AccessionID assumes the argument is of type AccessionID</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#BC_EQUALS_METHOD_SHOULD_WORK_FOR_ALL_OBJECTS">BC_EQUALS_METHOD_SHOULD_WORK_FOR_ALL_OBJECTS</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/AccessionID.html#87">87</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.sequence.CDSComparator"></a><div class="section"><h3>org.biojava.nbio.core.sequence.CDSComparator<a name="org.biojava.nbio.core.sequence.CDSComparator"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>org.biojava.nbio.core.sequence.CDSComparator implements Comparator but not Serializable</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#SE_COMPARATOR_SHOULD_BE_SERIALIZABLE">SE_COMPARATOR_SHOULD_BE_SERIALIZABLE</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/CDSComparator.html#30">30-47</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.sequence.ExonComparator"></a><div class="section"><h3>org.biojava.nbio.core.sequence.ExonComparator<a name="org.biojava.nbio.core.sequence.ExonComparator"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>org.biojava.nbio.core.sequence.ExonComparator implements Comparator but not Serializable</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#SE_COMPARATOR_SHOULD_BE_SERIALIZABLE">SE_COMPARATOR_SHOULD_BE_SERIALIZABLE</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/ExonComparator.html#34">34-40</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.sequence.GeneSequence"></a><div class="section"><h3>org.biojava.nbio.core.sequence.GeneSequence<a name="org.biojava.nbio.core.sequence.GeneSequence"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>Possible null pointer dereference of dnaSequence in org.biojava.nbio.core.sequence.GeneSequence.getSequence5PrimeTo3Prime() on exception path</td><td>CORRECTNESS</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#NP_NULL_ON_SOME_PATH_EXCEPTION">NP_NULL_ON_SOME_PATH_EXCEPTION</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/GeneSequence.html#314">314</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.sequence.SequenceComparator"></a><div class="section"><h3>org.biojava.nbio.core.sequence.SequenceComparator<a name="org.biojava.nbio.core.sequence.SequenceComparator"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>org.biojava.nbio.core.sequence.SequenceComparator implements Comparator but not Serializable</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#SE_COMPARATOR_SHOULD_BE_SERIALIZABLE">SE_COMPARATOR_SHOULD_BE_SERIALIZABLE</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/SequenceComparator.html#33">33-38</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.sequence.StartCodonSequence"></a><div class="section"><h3>org.biojava.nbio.core.sequence.StartCodonSequence<a name="org.biojava.nbio.core.sequence.StartCodonSequence"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>Unread public/protected field: org.biojava.nbio.core.sequence.StartCodonSequence.parentGeneSequence</td><td>STYLE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD">URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/StartCodonSequence.html#32">32</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.sequence.StopCodonSequence"></a><div class="section"><h3>org.biojava.nbio.core.sequence.StopCodonSequence<a name="org.biojava.nbio.core.sequence.StopCodonSequence"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>Unread public/protected field: org.biojava.nbio.core.sequence.StopCodonSequence.parentGeneSequence</td><td>STYLE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD">URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/StopCodonSequence.html#33">33</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.sequence.TranscriptSequence"></a><div class="section"><h3>org.biojava.nbio.core.sequence.TranscriptSequence<a name="org.biojava.nbio.core.sequence.TranscriptSequence"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>Possible null pointer dereference of dnaSequence in org.biojava.nbio.core.sequence.TranscriptSequence.getDNACodingSequence() on exception path</td><td>CORRECTNESS</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#NP_NULL_ON_SOME_PATH_EXCEPTION">NP_NULL_ON_SOME_PATH_EXCEPTION</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#215">215</a></td><td>Medium</td></tr><tr class="a"><td>Possible null pointer dereference of dnaCodingSequence in org.biojava.nbio.core.sequence.TranscriptSequence.getProteinCDSSequences() on exception path</td><td>CORRECTNESS</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#NP_NULL_ON_SOME_PATH_EXCEPTION">NP_NULL_ON_SOME_PATH_EXCEPTION</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/TranscriptSequence.html#189">189</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.sequence.io.FastaGeneWriter"></a><div class="section"><h3>org.biojava.nbio.core.sequence.io.FastaGeneWriter<a name="org.biojava.nbio.core.sequence.io.FastaGeneWriter"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Found reliance on default encoding in org.biojava.nbio.core.sequence.io.FastaGeneWriter.process(): String.getBytes()</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#80">80</a></td><td>High</td></tr></table></div><a name="org.biojava.nbio.core.sequence.io.FastaReader"></a><div class="section"><h3>org.biojava.nbio.core.sequence.io.FastaReader<a name="org.biojava.nbio.core.sequence.io.FastaReader"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Found reliance on default encoding in new org.biojava.nbio.core.sequence.io.FastaReader(File, SequenceHeaderParserInterface, SequenceCreatorInterface): new java.io.InputStreamReader(InputStream)</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#91">91</a></td><td>High</td></tr><tr class="b"><td>Found reliance on default encoding in new org.biojava.nbio.core.sequence.io.FastaReader(InputStream, SequenceHeaderParserInterface, SequenceCreatorInterface): new java.io.InputStreamReader(InputStream)</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#69">69</a></td><td>High</td></tr><tr class="a"><td>org.biojava.nbio.core.sequence.io.FastaReader.main(String[]) makes inefficient use of keySet iterator instead of entrySet iterator</td><td>PERFORMANCE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#WMI_WRONG_MAP_ITERATOR">WMI_WRONG_MAP_ITERATOR</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaReader.html#249">249</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.sequence.io.FastaWriter"></a><div class="section"><h3>org.biojava.nbio.core.sequence.io.FastaWriter<a name="org.biojava.nbio.core.sequence.io.FastaWriter"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Found reliance on default encoding in new org.biojava.nbio.core.sequence.io.FastaWriter(OutputStream, Collection, FastaHeaderFormatInterface): String.getBytes()</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#56">56</a></td><td>High</td></tr><tr class="b"><td>Found reliance on default encoding in new org.biojava.nbio.core.sequence.io.FastaWriter(OutputStream, Collection, FastaHeaderFormatInterface, int): String.getBytes()</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#56">56</a></td><td>High</td></tr><tr class="a"><td>Found reliance on default encoding in org.biojava.nbio.core.sequence.io.FastaWriter.process(): String.getBytes()</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#101">101</a></td><td>High</td></tr><tr class="b"><td>Found reliance on default encoding in org.biojava.nbio.core.sequence.io.FastaWriter.setLineSeparator(String): String.getBytes()</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/io/FastaWriter.html#90">90</a></td><td>High</td></tr></table></div><a name="org.biojava.nbio.core.sequence.io.GenbankReader"></a><div class="section"><h3>org.biojava.nbio.core.sequence.io.GenbankReader<a name="org.biojava.nbio.core.sequence.io.GenbankReader"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>Found reliance on default encoding in org.biojava.nbio.core.sequence.io.GenbankReader.process(int): new java.io.InputStreamReader(InputStream)</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankReader.html#139">139</a></td><td>High</td></tr></table></div><a name="org.biojava.nbio.core.sequence.io.GenbankWriter"></a><div class="section"><h3>org.biojava.nbio.core.sequence.io.GenbankWriter<a name="org.biojava.nbio.core.sequence.io.GenbankWriter"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>Found reliance on default encoding in org.biojava.nbio.core.sequence.io.GenbankWriter.process(): new java.io.PrintWriter(OutputStream)</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenbankWriter.html#92">92</a></td><td>High</td></tr></table></div><a name="org.biojava.nbio.core.sequence.io.GenericFastaHeaderParser"></a><div class="section"><h3>org.biojava.nbio.core.sequence.io.GenericFastaHeaderParser<a name="org.biojava.nbio.core.sequence.io.GenericFastaHeaderParser"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>Null passed for nonnull parameter of parseHeader(String, AbstractSequence) in org.biojava.nbio.core.sequence.io.GenericFastaHeaderParser.main(String[])</td><td>CORRECTNESS</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#NP_NULL_PARAM_DEREF">NP_NULL_PARAM_DEREF</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#193">193</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.sequence.io.GenericGenbankHeaderFormat"></a><div class="section"><h3>org.biojava.nbio.core.sequence.io.GenericGenbankHeaderFormat<a name="org.biojava.nbio.core.sequence.io.GenericGenbankHeaderFormat"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>org.biojava.nbio.core.sequence.io.GenericGenbankHeaderFormat._write_comment(AbstractSequence) concatenates strings using + in a loop</td><td>PERFORMANCE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#SBSC_USE_STRINGBUFFER_CONCATENATION">SBSC_USE_STRINGBUFFER_CONCATENATION</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#252">252</a></td><td>Medium</td></tr><tr class="a"><td>org.biojava.nbio.core.sequence.io.GenericGenbankHeaderFormat._write_multi_line(String, String) concatenates strings using + in a loop</td><td>PERFORMANCE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#SBSC_USE_STRINGBUFFER_CONCATENATION">SBSC_USE_STRINGBUFFER_CONCATENATION</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#75">75</a></td><td>Medium</td></tr><tr class="b"><td>org.biojava.nbio.core.sequence.io.GenericGenbankHeaderFormat.getHeader(AbstractSequence) concatenates strings using + in a loop</td><td>PERFORMANCE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#SBSC_USE_STRINGBUFFER_CONCATENATION">SBSC_USE_STRINGBUFFER_CONCATENATION</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#333">333</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.sequence.io.GenericGenbankHeaderParser"></a><div class="section"><h3>org.biojava.nbio.core.sequence.io.GenericGenbankHeaderParser<a name="org.biojava.nbio.core.sequence.io.GenericGenbankHeaderParser"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>Unread field: org.biojava.nbio.core.sequence.io.GenericGenbankHeaderParser.version</td><td>PERFORMANCE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#URF_UNREAD_FIELD">URF_UNREAD_FIELD</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#62">62</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.sequence.io.GenericInsdcHeaderFormat"></a><div class="section"><h3>org.biojava.nbio.core.sequence.io.GenericInsdcHeaderFormat<a name="org.biojava.nbio.core.sequence.io.GenericInsdcHeaderFormat"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>org.biojava.nbio.core.sequence.io.GenericInsdcHeaderFormat._write_feature(FeatureInterface, int) concatenates strings using + in a loop</td><td>PERFORMANCE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#SBSC_USE_STRINGBUFFER_CONCATENATION">SBSC_USE_STRINGBUFFER_CONCATENATION</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#125">125</a></td><td>Medium</td></tr><tr class="a"><td>Illegal format string &quot;Inconsistent strands: %r for parent, %r for child&quot; in org.biojava.nbio.core.sequence.io.GenericInsdcHeaderFormat._insdc_feature_location_string(FeatureInterface, int)</td><td>CORRECTNESS</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#VA_FORMAT_STRING_ILLEGAL">VA_FORMAT_STRING_ILLEGAL</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#186">186</a></td><td>High</td></tr></table></div><a name="org.biojava.nbio.core.sequence.io.util.ClasspathResource"></a><div class="section"><h3>org.biojava.nbio.core.sequence.io.util.ClasspathResource<a name="org.biojava.nbio.core.sequence.io.util.ClasspathResource"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Found reliance on default encoding in org.biojava.nbio.core.sequence.io.util.ClasspathResource.getBufferedReader(): new java.io.InputStreamReader(InputStream)</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#102">102</a></td><td>High</td></tr></table></div><a name="org.biojava.nbio.core.sequence.io.util.IOUtils"></a><div class="section"><h3>org.biojava.nbio.core.sequence.io.util.IOUtils<a name="org.biojava.nbio.core.sequence.io.util.IOUtils"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Found reliance on default encoding in org.biojava.nbio.core.sequence.io.util.IOUtils.getList(InputStream): new java.io.InputStreamReader(InputStream)</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#133">133</a></td><td>High</td></tr><tr class="b"><td>Found reliance on default encoding in org.biojava.nbio.core.sequence.io.util.IOUtils.print(String, File): new java.io.FileWriter(File)</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/io/util/IOUtils.html#339">339</a></td><td>High</td></tr></table></div><a name="org.biojava.nbio.core.sequence.loader.GenbankProxySequenceReader"></a><div class="section"><h3>org.biojava.nbio.core.sequence.loader.GenbankProxySequenceReader<a name="org.biojava.nbio.core.sequence.loader.GenbankProxySequenceReader"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>Found reliance on default encoding in new org.biojava.nbio.core.sequence.loader.GenbankProxySequenceReader(String, String, CompoundSet): new java.io.InputStreamReader(InputStream)</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#86">86</a></td><td>High</td></tr><tr class="a"><td>Call to org.biojava.nbio.core.sequence.template.CompoundSet.equals(Class) in new org.biojava.nbio.core.sequence.loader.GenbankProxySequenceReader(String, String, CompoundSet)</td><td>CORRECTNESS</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#EC_UNRELATED_CLASS_AND_INTERFACE">EC_UNRELATED_CLASS_AND_INTERFACE</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#91">91</a></td><td>High</td></tr><tr class="b"><td>org.biojava.nbio.core.sequence.loader.GenbankProxySequenceReader.copyInputStreamToFile(InputStream, File) may fail to clean up java.io.OutputStream on checked exception</td><td>EXPERIMENTAL</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#OBL_UNSATISFIED_OBLIGATION_EXCEPTION_EDGE">OBL_UNSATISFIED_OBLIGATION_EXCEPTION_EDGE</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#120">120</a></td><td>Medium</td></tr><tr class="a"><td>exceptional return value of java.io.File.mkdirs() ignored in org.biojava.nbio.core.sequence.loader.GenbankProxySequenceReader.setGenbankDirectoryCache(String)</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#RV_RETURN_VALUE_IGNORED_BAD_PRACTICE">RV_RETURN_VALUE_IGNORED_BAD_PRACTICE</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#160">160</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.sequence.loader.SequenceFileProxyLoader"></a><div class="section"><h3>org.biojava.nbio.core.sequence.loader.SequenceFileProxyLoader<a name="org.biojava.nbio.core.sequence.loader.SequenceFileProxyLoader"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Found reliance on default encoding in org.biojava.nbio.core.sequence.loader.SequenceFileProxyLoader.init(): new java.io.FileReader(File)</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#101">101</a></td><td>High</td></tr><tr class="b"><td>org.biojava.nbio.core.sequence.loader.SequenceFileProxyLoader.init() ignores result of java.io.BufferedReader.skip(long)</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#SR_NOT_CHECKED">SR_NOT_CHECKED</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#102">102</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.sequence.loader.UniprotProxySequenceReader"></a><div class="section"><h3>org.biojava.nbio.core.sequence.loader.UniprotProxySequenceReader<a name="org.biojava.nbio.core.sequence.loader.UniprotProxySequenceReader"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>Found reliance on default encoding in org.biojava.nbio.core.sequence.loader.UniprotProxySequenceReader.fetchFromCache(String): new java.io.FileReader(File)</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#425">425</a></td><td>High</td></tr><tr class="a"><td>Found reliance on default encoding in org.biojava.nbio.core.sequence.loader.UniprotProxySequenceReader.fetchUniprotXML(String): new java.io.InputStreamReader(InputStream)</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#396">396</a></td><td>High</td></tr><tr class="b"><td>Found reliance on default encoding in org.biojava.nbio.core.sequence.loader.UniprotProxySequenceReader.getUniprotXML(String): String.getBytes()</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#366">366</a></td><td>High</td></tr><tr class="a"><td>Found reliance on default encoding in org.biojava.nbio.core.sequence.loader.UniprotProxySequenceReader.parseUniprotXMLString(String, CompoundSet): String.getBytes()</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#127">127</a></td><td>High</td></tr><tr class="b"><td>Found reliance on default encoding in org.biojava.nbio.core.sequence.loader.UniprotProxySequenceReader.writeCache(StringBuilder, String): new java.io.FileWriter(File)</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#378">378</a></td><td>High</td></tr><tr class="a"><td>org.biojava.nbio.core.sequence.loader.UniprotProxySequenceReader.fetchFromCache(String) may fail to clean up java.io.Reader on checked exception</td><td>EXPERIMENTAL</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#OBL_UNSATISFIED_OBLIGATION_EXCEPTION_EDGE">OBL_UNSATISFIED_OBLIGATION_EXCEPTION_EDGE</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#425">425</a></td><td>Medium</td></tr><tr class="b"><td>org.biojava.nbio.core.sequence.loader.UniprotProxySequenceReader.writeCache(StringBuilder, String) may fail to clean up java.io.Writer on checked exception</td><td>EXPERIMENTAL</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#OBL_UNSATISFIED_OBLIGATION_EXCEPTION_EDGE">OBL_UNSATISFIED_OBLIGATION_EXCEPTION_EDGE</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#378">378</a></td><td>Medium</td></tr><tr class="a"><td>exceptional return value of java.io.File.mkdirs() ignored in org.biojava.nbio.core.sequence.loader.UniprotProxySequenceReader.setUniprotDirectoryCache(String)</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#RV_RETURN_VALUE_IGNORED_BAD_PRACTICE">RV_RETURN_VALUE_IGNORED_BAD_PRACTICE</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#491">491</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.sequence.location.FuzzyPoint"></a><div class="section"><h3>org.biojava.nbio.core.sequence.location.FuzzyPoint<a name="org.biojava.nbio.core.sequence.location.FuzzyPoint"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Equals method for org.biojava.nbio.core.sequence.location.FuzzyPoint assumes the argument is of type FuzzyPoint</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#BC_EQUALS_METHOD_SHOULD_WORK_FOR_ALL_OBJECTS">BC_EQUALS_METHOD_SHOULD_WORK_FOR_ALL_OBJECTS</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/location/FuzzyPoint.html#126">126</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.sequence.location.InsdcParser"></a><div class="section"><h3>org.biojava.nbio.core.sequence.location.InsdcParser<a name="org.biojava.nbio.core.sequence.location.InsdcParser"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Unread field: org.biojava.nbio.core.sequence.location.InsdcParser.referenceSequence</td><td>PERFORMANCE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#URF_UNREAD_FIELD">URF_UNREAD_FIELD</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/location/InsdcParser.html#95">95</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.sequence.location.SimplePoint"></a><div class="section"><h3>org.biojava.nbio.core.sequence.location.SimplePoint<a name="org.biojava.nbio.core.sequence.location.SimplePoint"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Equals method for org.biojava.nbio.core.sequence.location.SimplePoint assumes the argument is of type SimplePoint</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#BC_EQUALS_METHOD_SHOULD_WORK_FOR_ALL_OBJECTS">BC_EQUALS_METHOD_SHOULD_WORK_FOR_ALL_OBJECTS</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/location/SimplePoint.html#100">100</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.sequence.location.template.AbstractLocation"></a><div class="section"><h3>org.biojava.nbio.core.sequence.location.template.AbstractLocation<a name="org.biojava.nbio.core.sequence.location.template.AbstractLocation"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Equals method for org.biojava.nbio.core.sequence.location.template.AbstractLocation assumes the argument is of type AbstractLocation</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#BC_EQUALS_METHOD_SHOULD_WORK_FOR_ALL_OBJECTS">BC_EQUALS_METHOD_SHOULD_WORK_FOR_ALL_OBJECTS</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#262">262</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.sequence.location.template.LocationTools"></a><div class="section"><h3>org.biojava.nbio.core.sequence.location.template.Location$Tools<a name="org.biojava.nbio.core.sequence.location.template.LocationTools"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Boxed value is unboxed and then immediately reboxed in org.biojava.nbio.core.sequence.location.template.Location$Tools.location(List, Integer, String)</td><td>PERFORMANCE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#BX_UNBOXING_IMMEDIATELY_REBOXED">BX_UNBOXING_IMMEDIATELY_REBOXED</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#138">138</a></td><td>Medium</td></tr><tr class="b"><td>Dead store to sequenceLength in org.biojava.nbio.core.sequence.location.template.Location$Tools.location(List, Integer, String)</td><td>STYLE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DLS_DEAD_LOCAL_STORE">DLS_DEAD_LOCAL_STORE</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#138">138</a></td><td>High</td></tr><tr class="a"><td>Dead store to type in org.biojava.nbio.core.sequence.location.template.Location$Tools.location(List, Integer, String)</td><td>STYLE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DLS_DEAD_LOCAL_STORE">DLS_DEAD_LOCAL_STORE</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/location/template/Location.html#137">137</a></td><td>High</td></tr></table></div><a name="org.biojava.nbio.core.sequence.storage.ArrayListSequenceReader"></a><div class="section"><h3>org.biojava.nbio.core.sequence.storage.ArrayListSequenceReader<a name="org.biojava.nbio.core.sequence.storage.ArrayListSequenceReader"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Equals method for org.biojava.nbio.core.sequence.storage.ArrayListSequenceReader assumes the argument is of type ArrayListSequenceReader</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#BC_EQUALS_METHOD_SHOULD_WORK_FOR_ALL_OBJECTS">BC_EQUALS_METHOD_SHOULD_WORK_FOR_ALL_OBJECTS</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#280">280</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.sequence.storage.BitSequenceReader"></a><div class="section"><h3>org.biojava.nbio.core.sequence.storage.BitSequenceReader<a name="org.biojava.nbio.core.sequence.storage.BitSequenceReader"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Equals method for org.biojava.nbio.core.sequence.storage.BitSequenceReader assumes the argument is of type BitSequenceReader</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#BC_EQUALS_METHOD_SHOULD_WORK_FOR_ALL_OBJECTS">BC_EQUALS_METHOD_SHOULD_WORK_FOR_ALL_OBJECTS</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#195">195</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.sequence.storage.BitSequenceReaderBitArrayWorker"></a><div class="section"><h3>org.biojava.nbio.core.sequence.storage.BitSequenceReader$BitArrayWorker<a name="org.biojava.nbio.core.sequence.storage.BitSequenceReaderBitArrayWorker"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Equals method for org.biojava.nbio.core.sequence.storage.BitSequenceReader$BitArrayWorker assumes the argument is of type BitSequenceReader$BitArrayWorker</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#BC_EQUALS_METHOD_SHOULD_WORK_FOR_ALL_OBJECTS">BC_EQUALS_METHOD_SHOULD_WORK_FOR_ALL_OBJECTS</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#462">462</a></td><td>Medium</td></tr><tr class="b"><td>new org.biojava.nbio.core.sequence.storage.BitSequenceReader$BitArrayWorker(CompoundSet, int[]) may expose internal representation by storing an externally mutable object into BitSequenceReader$BitArrayWorker.sequence</td><td>MALICIOUS_CODE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#EI_EXPOSE_REP2">EI_EXPOSE_REP2</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#243">243</a></td><td>Medium</td></tr><tr class="a"><td>Possible null pointer dereference of integerValue in org.biojava.nbio.core.sequence.storage.BitSequenceReader$BitArrayWorker.setCompoundAt(Compound, int)</td><td>CORRECTNESS</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#NP_NULL_ON_SOME_PATH">NP_NULL_ON_SOME_PATH</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#328">328</a></td><td>High</td></tr></table></div><a name="org.biojava.nbio.core.sequence.storage.SingleCompoundSequenceReader"></a><div class="section"><h3>org.biojava.nbio.core.sequence.storage.SingleCompoundSequenceReader<a name="org.biojava.nbio.core.sequence.storage.SingleCompoundSequenceReader"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Equals method for org.biojava.nbio.core.sequence.storage.SingleCompoundSequenceReader assumes the argument is of type SingleCompoundSequenceReader</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#BC_EQUALS_METHOD_SHOULD_WORK_FOR_ALL_OBJECTS">BC_EQUALS_METHOD_SHOULD_WORK_FOR_ALL_OBJECTS</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#206">206</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.sequence.template.AbstractNucleotideCompoundSet"></a><div class="section"><h3>org.biojava.nbio.core.sequence.template.AbstractNucleotideCompoundSet<a name="org.biojava.nbio.core.sequence.template.AbstractNucleotideCompoundSet"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>org.biojava.nbio.core.sequence.template.AbstractNucleotideCompoundSet.calculateIndirectAmbiguities() makes inefficient use of keySet iterator instead of entrySet iterator</td><td>PERFORMANCE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#WMI_WRONG_MAP_ITERATOR">WMI_WRONG_MAP_ITERATOR</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#105">105</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.sequence.template.AbstractSequence"></a><div class="section"><h3>org.biojava.nbio.core.sequence.template.AbstractSequence<a name="org.biojava.nbio.core.sequence.template.AbstractSequence"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>org.biojava.nbio.core.sequence.template.AbstractSequence.setProxySequenceReader(SequenceReader) makes inefficient use of keySet iterator instead of entrySet iterator</td><td>PERFORMANCE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#WMI_WRONG_MAP_ITERATOR">WMI_WRONG_MAP_ITERATOR</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/template/AbstractSequence.html#127">127</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.sequence.transcription.TableCaseInsensitiveTriplet"></a><div class="section"><h3>org.biojava.nbio.core.sequence.transcription.Table$CaseInsensitiveTriplet<a name="org.biojava.nbio.core.sequence.transcription.TableCaseInsensitiveTriplet"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Equals method for org.biojava.nbio.core.sequence.transcription.Table$CaseInsensitiveTriplet assumes the argument is of type Table$CaseInsensitiveTriplet</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#BC_EQUALS_METHOD_SHOULD_WORK_FOR_ALL_OBJECTS">BC_EQUALS_METHOD_SHOULD_WORK_FOR_ALL_OBJECTS</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#229">229</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.sequence.transcription.TableCodon"></a><div class="section"><h3>org.biojava.nbio.core.sequence.transcription.Table$Codon<a name="org.biojava.nbio.core.sequence.transcription.TableCodon"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Equals method for org.biojava.nbio.core.sequence.transcription.Table$Codon assumes the argument is of type Table$Codon</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#BC_EQUALS_METHOD_SHOULD_WORK_FOR_ALL_OBJECTS">BC_EQUALS_METHOD_SHOULD_WORK_FOR_ALL_OBJECTS</a></td><td><a href="./xref/org/biojava/nbio/core/sequence/transcription/Table.html#120">120</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.util.FlatFileCache"></a><div class="section"><h3>org.biojava.nbio.core.util.FlatFileCache<a name="org.biojava.nbio.core.util.FlatFileCache"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>org.biojava.nbio.core.util.FlatFileCache.addToCache(String, File) may fail to clean up java.io.InputStream on checked exception</td><td>EXPERIMENTAL</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#OBL_UNSATISFIED_OBLIGATION_EXCEPTION_EDGE">OBL_UNSATISFIED_OBLIGATION_EXCEPTION_EDGE</a></td><td><a href="./xref/org/biojava/nbio/core/util/FlatFileCache.html#63">63</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.util.InputStreamProvider"></a><div class="section"><h3>org.biojava.nbio.core.util.InputStreamProvider<a name="org.biojava.nbio.core.util.InputStreamProvider"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>org.biojava.nbio.core.util.InputStreamProvider.getInputStream(File) may fail to close stream</td><td>BAD_PRACTICE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#OS_OPEN_STREAM">OS_OPEN_STREAM</a></td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#188">188</a></td><td>Medium</td></tr><tr class="b"><td>Unread field: org.biojava.nbio.core.util.InputStreamProvider.cache</td><td>PERFORMANCE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#URF_UNREAD_FIELD">URF_UNREAD_FIELD</a></td><td><a href="./xref/org/biojava/nbio/core/util/InputStreamProvider.html#76">76</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.util.SingleLinkageClusterer"></a><div class="section"><h3>org.biojava.nbio.core.util.SingleLinkageClusterer<a name="org.biojava.nbio.core.util.SingleLinkageClusterer"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>org.biojava.nbio.core.util.SingleLinkageClusterer.updateIndicesToCheck(int) invokes inefficient new Integer(int) constructor; use Integer.valueOf(int) instead</td><td>PERFORMANCE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_NUMBER_CTOR">DM_NUMBER_CTOR</a></td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#202">202</a></td><td>Medium</td></tr><tr class="a"><td>org.biojava.nbio.core.util.SingleLinkageClusterer.getDendrogram() may expose internal representation by returning SingleLinkageClusterer.dendrogram</td><td>MALICIOUS_CODE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#EI_EXPOSE_REP">EI_EXPOSE_REP</a></td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#125">125</a></td><td>Medium</td></tr><tr class="b"><td>new org.biojava.nbio.core.util.SingleLinkageClusterer(double[][], boolean) may expose internal representation by storing an externally mutable object into SingleLinkageClusterer.matrix</td><td>MALICIOUS_CODE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#EI_EXPOSE_REP2">EI_EXPOSE_REP2</a></td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#105">105</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.util.SingleLinkageClustererLinkedPair"></a><div class="section"><h3>org.biojava.nbio.core.util.SingleLinkageClusterer$LinkedPair<a name="org.biojava.nbio.core.util.SingleLinkageClustererLinkedPair"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>Should org.biojava.nbio.core.util.SingleLinkageClusterer$LinkedPair be a _static_ inner class?</td><td>PERFORMANCE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#SIC_INNER_SHOULD_BE_STATIC">SIC_INNER_SHOULD_BE_STATIC</a></td><td><a href="./xref/org/biojava/nbio/core/util/SingleLinkageClusterer.html#47">47-75</a></td><td>Medium</td></tr></table></div><a name="org.biojava.nbio.core.util.StringManipulationHelper"></a><div class="section"><h3>org.biojava.nbio.core.util.StringManipulationHelper<a name="org.biojava.nbio.core.util.StringManipulationHelper"></a></h3><table border="0" class="bodyTable"><tr class="a"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="b"><td>Found reliance on default encoding in org.biojava.nbio.core.util.StringManipulationHelper.convertStreamToString(InputStream): new java.io.InputStreamReader(InputStream)</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#75">75</a></td><td>High</td></tr><tr class="a"><td>Found reliance on default encoding in org.biojava.nbio.core.util.StringManipulationHelper.equalsToXml(String, String): String.getBytes()</td><td>I18N</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING">DM_DEFAULT_ENCODING</a></td><td><a href="./xref/org/biojava/nbio/core/util/StringManipulationHelper.html#144">144</a></td><td>High</td></tr></table></div><a name="org.biojava.nbio.core.util.UncompressInputStream"></a><div class="section"><h3>org.biojava.nbio.core.util.UncompressInputStream<a name="org.biojava.nbio.core.util.UncompressInputStream"></a></h3><table border="0" class="bodyTable"><tr class="b"><th>Bug</th><th>Category</th><th>Details</th><th>Line</th><th>Priority</th></tr><tr class="a"><td>Dead store to start in org.biojava.nbio.core.util.UncompressInputStream.uncompress(String, FileOutputStream)</td><td>STYLE</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#DLS_DEAD_LOCAL_STORE">DLS_DEAD_LOCAL_STORE</a></td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#459">459</a></td><td>Medium</td></tr><tr class="b"><td>Inconsistent synchronization of org.biojava.nbio.core.util.UncompressInputStream.bitmask; locked 80% of time</td><td>MT_CORRECTNESS</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#IS2_INCONSISTENT_SYNC">IS2_INCONSISTENT_SYNC</a></td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#434">434</a></td><td>Medium</td></tr><tr class="a"><td>Inconsistent synchronization of org.biojava.nbio.core.util.UncompressInputStream.finchar; locked 80% of time</td><td>MT_CORRECTNESS</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#IS2_INCONSISTENT_SYNC">IS2_INCONSISTENT_SYNC</a></td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#436">436</a></td><td>Medium</td></tr><tr class="b"><td>Inconsistent synchronization of org.biojava.nbio.core.util.UncompressInputStream.free_ent; locked 80% of time</td><td>MT_CORRECTNESS</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#IS2_INCONSISTENT_SYNC">IS2_INCONSISTENT_SYNC</a></td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#437">437</a></td><td>Medium</td></tr><tr class="a"><td>Inconsistent synchronization of org.biojava.nbio.core.util.UncompressInputStream.maxcode; locked 66% of time</td><td>MT_CORRECTNESS</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#IS2_INCONSISTENT_SYNC">IS2_INCONSISTENT_SYNC</a></td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#433">433</a></td><td>Medium</td></tr><tr class="b"><td>Inconsistent synchronization of org.biojava.nbio.core.util.UncompressInputStream.maxmaxcode; locked 66% of time</td><td>MT_CORRECTNESS</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#IS2_INCONSISTENT_SYNC">IS2_INCONSISTENT_SYNC</a></td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#431">431</a></td><td>Medium</td></tr><tr class="a"><td>Inconsistent synchronization of org.biojava.nbio.core.util.UncompressInputStream.n_bits; locked 66% of time</td><td>MT_CORRECTNESS</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#IS2_INCONSISTENT_SYNC">IS2_INCONSISTENT_SYNC</a></td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#432">432</a></td><td>Medium</td></tr><tr class="b"><td>Inconsistent synchronization of org.biojava.nbio.core.util.UncompressInputStream.oldcode; locked 80% of time</td><td>MT_CORRECTNESS</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#IS2_INCONSISTENT_SYNC">IS2_INCONSISTENT_SYNC</a></td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#435">435</a></td><td>Medium</td></tr><tr class="a"><td>Inconsistent synchronization of org.biojava.nbio.core.util.UncompressInputStream.stackp; locked 83% of time</td><td>MT_CORRECTNESS</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#IS2_INCONSISTENT_SYNC">IS2_INCONSISTENT_SYNC</a></td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#442">442</a></td><td>Medium</td></tr><tr class="b"><td>Inconsistent synchronization of org.biojava.nbio.core.util.UncompressInputStream.tab_prefix; locked 50% of time</td><td>MT_CORRECTNESS</td><td><a class="externalLink" href="http://findbugs.sourceforge.net/bugDescriptions.html#IS2_INCONSISTENT_SYNC">IS2_INCONSISTENT_SYNC</a></td><td><a href="./xref/org/biojava/nbio/core/util/UncompressInputStream.html#439">439</a></td><td>Medium</td></tr></table></div>
      </div>
    </div>
    <div class="clear">
      <hr/>
    </div>
    <div id="footer">
      <div class="xright">
              Copyright &#169;                    2000-2015
                        <a href="http://www.biojava.org">BioJava</a>.
            All Rights Reserved.      
        
                  </div>
      <div class="clear">
        <hr/>
      </div>
    </div>
  </body>
</html>
