<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head><meta http-equiv="content-type" content="text/html; charset=UTF-8" />
<title>Jronn xref</title>
<link type="text/css" rel="stylesheet" href="../../../../stylesheet.css" />
</head>
<body>
<div id="overview"><a href="../../../../../apidocs/org/biojava/nbio/ronn/Jronn.html">View Javadoc</a></div><pre>
<a class="jxr_linenumber" name="L1" href="#L1">1</a>    <em class="jxr_comment">/*        BioJava development code</em>
<a class="jxr_linenumber" name="L2" href="#L2">2</a>   <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L3" href="#L3">3</a>   <em class="jxr_comment"> * This code may be freely distributed and modified under the</em>
<a class="jxr_linenumber" name="L4" href="#L4">4</a>   <em class="jxr_comment"> * terms of the GNU Lesser General Public Licence.  This should</em>
<a class="jxr_linenumber" name="L5" href="#L5">5</a>   <em class="jxr_comment"> * be distributed with the code.  If you do not have a copy,</em>
<a class="jxr_linenumber" name="L6" href="#L6">6</a>   <em class="jxr_comment"> * see:</em>
<a class="jxr_linenumber" name="L7" href="#L7">7</a>   <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L8" href="#L8">8</a>   <em class="jxr_comment"> *      <a href="http://www.gnu.org/copyleft/lesser.htm" target="alexandria_uri">http://www.gnu.org/copyleft/lesser.htm</a>l</em>
<a class="jxr_linenumber" name="L9" href="#L9">9</a>   <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L10" href="#L10">10</a>  <em class="jxr_comment"> * Copyright for this code is held jointly by the individual</em>
<a class="jxr_linenumber" name="L11" href="#L11">11</a>  <em class="jxr_comment"> * authors.  These should be listed in @author doc comments.</em>
<a class="jxr_linenumber" name="L12" href="#L12">12</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L13" href="#L13">13</a>  <em class="jxr_comment"> * For more information on the BioJava project and its aims,</em>
<a class="jxr_linenumber" name="L14" href="#L14">14</a>  <em class="jxr_comment"> * or to join the biojava-l mailing list, visit the home page</em>
<a class="jxr_linenumber" name="L15" href="#L15">15</a>  <em class="jxr_comment"> * at:</em>
<a class="jxr_linenumber" name="L16" href="#L16">16</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L17" href="#L17">17</a>  <em class="jxr_comment"> *      <a href="http://www.biojava.org" target="alexandria_uri">http://www.biojava.org</a>/</em>
<a class="jxr_linenumber" name="L18" href="#L18">18</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L19" href="#L19">19</a>  <em class="jxr_comment"> */</em>
<a class="jxr_linenumber" name="L20" href="#L20">20</a>  <strong class="jxr_keyword">package</strong> org.biojava.nbio.ronn;
<a class="jxr_linenumber" name="L21" href="#L21">21</a>  
<a class="jxr_linenumber" name="L22" href="#L22">22</a>   <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.ProteinSequence;
<a class="jxr_linenumber" name="L23" href="#L23">23</a>   <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.compound.AminoAcidCompound;
<a class="jxr_linenumber" name="L24" href="#L24">24</a>   <strong class="jxr_keyword">import</strong> org.biojava.nbio.data.sequence.FastaSequence;
<a class="jxr_linenumber" name="L25" href="#L25">25</a>   <strong class="jxr_keyword">import</strong> org.biojava.nbio.data.sequence.SequenceUtil;
<a class="jxr_linenumber" name="L26" href="#L26">26</a>  
<a class="jxr_linenumber" name="L27" href="#L27">27</a>   <strong class="jxr_keyword">import</strong> java.io.FileInputStream;
<a class="jxr_linenumber" name="L28" href="#L28">28</a>   <strong class="jxr_keyword">import</strong> java.io.FileNotFoundException;
<a class="jxr_linenumber" name="L29" href="#L29">29</a>   <strong class="jxr_keyword">import</strong> java.io.IOException;
<a class="jxr_linenumber" name="L30" href="#L30">30</a>   <strong class="jxr_keyword">import</strong> java.util.ArrayList;
<a class="jxr_linenumber" name="L31" href="#L31">31</a>   <strong class="jxr_keyword">import</strong> java.util.List;
<a class="jxr_linenumber" name="L32" href="#L32">32</a>   <strong class="jxr_keyword">import</strong> java.util.Map;
<a class="jxr_linenumber" name="L33" href="#L33">33</a>   <strong class="jxr_keyword">import</strong> java.util.TreeMap;
<a class="jxr_linenumber" name="L34" href="#L34">34</a>  
<a class="jxr_linenumber" name="L35" href="#L35">35</a>  
<a class="jxr_linenumber" name="L36" href="#L36">36</a>  <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L37" href="#L37">37</a>  <em class="jxr_javadoccomment"> * This class gives public API to RONN functions. </em>
<a class="jxr_linenumber" name="L38" href="#L38">38</a>  <em class="jxr_javadoccomment"> * It is build on top of the command line client. Due to this fact a few things </em>
<a class="jxr_linenumber" name="L39" href="#L39">39</a>  <em class="jxr_javadoccomment"> * could be improved and extended pending the refactoring of the command line client.  </em>
<a class="jxr_linenumber" name="L40" href="#L40">40</a>  <em class="jxr_javadoccomment"> *</em>
<a class="jxr_linenumber" name="L41" href="#L41">41</a>  <em class="jxr_javadoccomment"> * The input sequence limitations - the input sequence must not contain any ambiguous characters, </em>
<a class="jxr_linenumber" name="L42" href="#L42">42</a>  <em class="jxr_javadoccomment"> * and have a minimum length of 19 amino acids. </em>
<a class="jxr_linenumber" name="L43" href="#L43">43</a>  <em class="jxr_javadoccomment"> * </em>
<a class="jxr_linenumber" name="L44" href="#L44">44</a>  <em class="jxr_javadoccomment"> * @author Peter Troshin</em>
<a class="jxr_linenumber" name="L45" href="#L45">45</a>  <em class="jxr_javadoccomment"> * @version 1.0</em>
<a class="jxr_linenumber" name="L46" href="#L46">46</a>  <em class="jxr_javadoccomment"> * @since 3.0.2</em>
<a class="jxr_linenumber" name="L47" href="#L47">47</a>  <em class="jxr_javadoccomment"> * </em>
<a class="jxr_linenumber" name="L48" href="#L48">48</a>  <em class="jxr_javadoccomment"> */</em>
<a class="jxr_linenumber" name="L49" href="#L49">49</a>  <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">class</strong> <a href="../../../../org/biojava/nbio/ronn/Jronn.html">Jronn</a> {
<a class="jxr_linenumber" name="L50" href="#L50">50</a>  	
<a class="jxr_linenumber" name="L51" href="#L51">51</a>  	<em class="jxr_comment">// Load models</em>
<a class="jxr_linenumber" name="L52" href="#L52">52</a>  	<strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">final</strong> <a href="../../../../org/biojava/nbio/ronn/ModelLoader.html">ModelLoader</a> loader = <strong class="jxr_keyword">new</strong> <a href="../../../../org/biojava/nbio/ronn/ModelLoader.html">ModelLoader</a>();  
<a class="jxr_linenumber" name="L53" href="#L53">53</a>  	<strong class="jxr_keyword">static</strong> {
<a class="jxr_linenumber" name="L54" href="#L54">54</a>  		<strong class="jxr_keyword">try</strong> {
<a class="jxr_linenumber" name="L55" href="#L55">55</a>  			loader.loadModels();
<a class="jxr_linenumber" name="L56" href="#L56">56</a>  		} <strong class="jxr_keyword">catch</strong> (NumberFormatException e) {
<a class="jxr_linenumber" name="L57" href="#L57">57</a>  			<strong class="jxr_keyword">throw</strong> <strong class="jxr_keyword">new</strong> RuntimeException(<span class="jxr_string">"Fails to load models!"</span> + e.getMessage(), e); 
<a class="jxr_linenumber" name="L58" href="#L58">58</a>  		} <strong class="jxr_keyword">catch</strong> (IOException e) {
<a class="jxr_linenumber" name="L59" href="#L59">59</a>  			<strong class="jxr_keyword">throw</strong> <strong class="jxr_keyword">new</strong> RuntimeException(<span class="jxr_string">"Fails to load models!"</span> + e.getMessage(), e);
<a class="jxr_linenumber" name="L60" href="#L60">60</a>  		}
<a class="jxr_linenumber" name="L61" href="#L61">61</a>  	}
<a class="jxr_linenumber" name="L62" href="#L62">62</a>  	
<a class="jxr_linenumber" name="L63" href="#L63">63</a>  	
<a class="jxr_linenumber" name="L64" href="#L64">64</a>  	<em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L65" href="#L65">65</a>  <em class="jxr_javadoccomment">	 * Holder for the ranges, contain pointers to starting and ending position </em>
<a class="jxr_linenumber" name="L66" href="#L66">66</a>  <em class="jxr_javadoccomment">	 * on the sequence which comprises a disordered region. Immutable. </em>
<a class="jxr_linenumber" name="L67" href="#L67">67</a>  <em class="jxr_javadoccomment">	 * @author pvtroshin</em>
<a class="jxr_linenumber" name="L68" href="#L68">68</a>  <em class="jxr_javadoccomment">	 */</em>
<a class="jxr_linenumber" name="L69" href="#L69">69</a>  	<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">class</strong> <a href="../../../../org/biojava/nbio/ronn/Jronn.html">Range</a> {
<a class="jxr_linenumber" name="L70" href="#L70">70</a>  		<em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L71" href="#L71">71</a>  <em class="jxr_javadoccomment">		 * Range starting position counts from 1 (the first position on the sequence is 1)</em>
<a class="jxr_linenumber" name="L72" href="#L72">72</a>  <em class="jxr_javadoccomment">		 */</em>
<a class="jxr_linenumber" name="L73" href="#L73">73</a>  		<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">final</strong> <strong class="jxr_keyword">int</strong> from; 
<a class="jxr_linenumber" name="L74" href="#L74">74</a>  		<em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L75" href="#L75">75</a>  <em class="jxr_javadoccomment">		 * The range ending position includes the last residue. </em>
<a class="jxr_linenumber" name="L76" href="#L76">76</a>  <em class="jxr_javadoccomment">		 */</em>
<a class="jxr_linenumber" name="L77" href="#L77">77</a>  		<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">final</strong> <strong class="jxr_keyword">int</strong> to; 
<a class="jxr_linenumber" name="L78" href="#L78">78</a>  	
<a class="jxr_linenumber" name="L79" href="#L79">79</a>  		<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">final</strong> <strong class="jxr_keyword">float</strong> score;
<a class="jxr_linenumber" name="L80" href="#L80">80</a>  		<strong class="jxr_keyword">public</strong> <a href="../../../../org/biojava/nbio/ronn/Jronn.html">Range</a>(<strong class="jxr_keyword">int</strong> from, <strong class="jxr_keyword">int</strong> to, <strong class="jxr_keyword">float</strong> score) {
<a class="jxr_linenumber" name="L81" href="#L81">81</a>  			assert from&gt;=0; 
<a class="jxr_linenumber" name="L82" href="#L82">82</a>  			assert from&lt;to; 
<a class="jxr_linenumber" name="L83" href="#L83">83</a>  			<strong class="jxr_keyword">this</strong>.from = from; 
<a class="jxr_linenumber" name="L84" href="#L84">84</a>  			<strong class="jxr_keyword">this</strong>.to = to; 
<a class="jxr_linenumber" name="L85" href="#L85">85</a>  			<strong class="jxr_keyword">this</strong>.score = score;
<a class="jxr_linenumber" name="L86" href="#L86">86</a>  		}
<a class="jxr_linenumber" name="L87" href="#L87">87</a>  
<a class="jxr_linenumber" name="L88" href="#L88">88</a>  		@Override
<a class="jxr_linenumber" name="L89" href="#L89">89</a>  		<strong class="jxr_keyword">public</strong> String toString() { 
<a class="jxr_linenumber" name="L90" href="#L90">90</a>  			<strong class="jxr_keyword">return</strong> <span class="jxr_string">"Range"</span> + <span class="jxr_string">" From:"</span> + from + <span class="jxr_string">"\t"</span> + <span class="jxr_string">"to: "</span> + to + <span class="jxr_string">"\n"</span>;
<a class="jxr_linenumber" name="L91" href="#L91">91</a>  		}
<a class="jxr_linenumber" name="L92" href="#L92">92</a>  		
<a class="jxr_linenumber" name="L93" href="#L93">93</a>  		@Override
<a class="jxr_linenumber" name="L94" href="#L94">94</a>  		<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">int</strong> hashCode() {
<a class="jxr_linenumber" name="L95" href="#L95">95</a>  			<strong class="jxr_keyword">final</strong> <strong class="jxr_keyword">int</strong> prime = 31;
<a class="jxr_linenumber" name="L96" href="#L96">96</a>  			<strong class="jxr_keyword">int</strong> result = 1;
<a class="jxr_linenumber" name="L97" href="#L97">97</a>  			result = prime * result + from;
<a class="jxr_linenumber" name="L98" href="#L98">98</a>  			result = prime * result + to;
<a class="jxr_linenumber" name="L99" href="#L99">99</a>  			<strong class="jxr_keyword">return</strong> result;
<a class="jxr_linenumber" name="L100" href="#L100">100</a> 		}
<a class="jxr_linenumber" name="L101" href="#L101">101</a> 
<a class="jxr_linenumber" name="L102" href="#L102">102</a> 		@Override
<a class="jxr_linenumber" name="L103" href="#L103">103</a> 		<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">boolean</strong> equals(Object obj) {
<a class="jxr_linenumber" name="L104" href="#L104">104</a> 			<strong class="jxr_keyword">if</strong> (<strong class="jxr_keyword">this</strong> == obj)
<a class="jxr_linenumber" name="L105" href="#L105">105</a> 				<strong class="jxr_keyword">return</strong> <strong class="jxr_keyword">true</strong>;
<a class="jxr_linenumber" name="L106" href="#L106">106</a> 			<strong class="jxr_keyword">if</strong> (obj == <strong class="jxr_keyword">null</strong>)
<a class="jxr_linenumber" name="L107" href="#L107">107</a> 				<strong class="jxr_keyword">return</strong> false;
<a class="jxr_linenumber" name="L108" href="#L108">108</a> 			<strong class="jxr_keyword">if</strong> (getClass() != obj.getClass())
<a class="jxr_linenumber" name="L109" href="#L109">109</a> 				<strong class="jxr_keyword">return</strong> false;
<a class="jxr_linenumber" name="L110" href="#L110">110</a> 			Range other = (Range) obj;
<a class="jxr_linenumber" name="L111" href="#L111">111</a> 			<strong class="jxr_keyword">if</strong> (from != other.from)
<a class="jxr_linenumber" name="L112" href="#L112">112</a> 				<strong class="jxr_keyword">return</strong> false;
<a class="jxr_linenumber" name="L113" href="#L113">113</a> 			<strong class="jxr_keyword">if</strong> (to != other.to)
<a class="jxr_linenumber" name="L114" href="#L114">114</a> 				<strong class="jxr_keyword">return</strong> false;
<a class="jxr_linenumber" name="L115" href="#L115">115</a> 			<strong class="jxr_keyword">return</strong> <strong class="jxr_keyword">true</strong>;
<a class="jxr_linenumber" name="L116" href="#L116">116</a> 		}
<a class="jxr_linenumber" name="L117" href="#L117">117</a> 		
<a class="jxr_linenumber" name="L118" href="#L118">118</a> 		
<a class="jxr_linenumber" name="L119" href="#L119">119</a> 	}
<a class="jxr_linenumber" name="L120" href="#L120">120</a> 	
<a class="jxr_linenumber" name="L121" href="#L121">121</a> 	<em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L122" href="#L122">122</a> <em class="jxr_javadoccomment">	 * Calculates the probability value for each residue in the protein sequence, </em>
<a class="jxr_linenumber" name="L123" href="#L123">123</a> <em class="jxr_javadoccomment">	 * telling the probability that the residue belongs to disordered region. </em>
<a class="jxr_linenumber" name="L124" href="#L124">124</a> <em class="jxr_javadoccomment">	 * In general, values greater than 0.5 considered to be in the disordered regions. </em>
<a class="jxr_linenumber" name="L125" href="#L125">125</a> <em class="jxr_javadoccomment">	 *   </em>
<a class="jxr_linenumber" name="L126" href="#L126">126</a> <em class="jxr_javadoccomment">	 * @param sequence an instance of FastaSequence object, holding the name and the sequence. </em>
<a class="jxr_linenumber" name="L127" href="#L127">127</a> <em class="jxr_javadoccomment">	 * @return the probability scores for each residue in the sequence</em>
<a class="jxr_linenumber" name="L128" href="#L128">128</a> <em class="jxr_javadoccomment">	 */</em>
<a class="jxr_linenumber" name="L129" href="#L129">129</a> 	<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">float</strong>[] getDisorderScores(<a href="../../../../org/biojava/nbio/data/sequence/FastaSequence.html">FastaSequence</a> sequence) {
<a class="jxr_linenumber" name="L130" href="#L130">130</a> 		    <strong class="jxr_keyword">return</strong> predictSerial(sequence);
<a class="jxr_linenumber" name="L131" href="#L131">131</a> 	}
<a class="jxr_linenumber" name="L132" href="#L132">132</a> 	
<a class="jxr_linenumber" name="L133" href="#L133">133</a> 	<em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L134" href="#L134">134</a> <em class="jxr_javadoccomment">	 * Calculates the probability value for each residue in the protein sequence, </em>
<a class="jxr_linenumber" name="L135" href="#L135">135</a> <em class="jxr_javadoccomment">	 * telling the probability that the residue belongs to disordered region. </em>
<a class="jxr_linenumber" name="L136" href="#L136">136</a> <em class="jxr_javadoccomment">	 * In general, values greater than 0.5 considered to be in the disordered regions. </em>
<a class="jxr_linenumber" name="L137" href="#L137">137</a> <em class="jxr_javadoccomment">	 *   </em>
<a class="jxr_linenumber" name="L138" href="#L138">138</a> <em class="jxr_javadoccomment">	 * @param sequence an instance of FastaSequence object, holding the name and the sequence. </em>
<a class="jxr_linenumber" name="L139" href="#L139">139</a> <em class="jxr_javadoccomment">	 * @return the probability scores for each residue in the sequence</em>
<a class="jxr_linenumber" name="L140" href="#L140">140</a> <em class="jxr_javadoccomment">	 */</em>
<a class="jxr_linenumber" name="L141" href="#L141">141</a> 	<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">float</strong>[] getDisorderScores(ProteinSequence sequence) {
<a class="jxr_linenumber" name="L142" href="#L142">142</a> 		
<a class="jxr_linenumber" name="L143" href="#L143">143</a> 		FastaSequence seq = convertProteinSequencetoFasta(sequence);
<a class="jxr_linenumber" name="L144" href="#L144">144</a> 		
<a class="jxr_linenumber" name="L145" href="#L145">145</a> 		<strong class="jxr_keyword">return</strong> predictSerial(seq);
<a class="jxr_linenumber" name="L146" href="#L146">146</a> 	}
<a class="jxr_linenumber" name="L147" href="#L147">147</a> 	
<a class="jxr_linenumber" name="L148" href="#L148">148</a> 	<em class="jxr_javadoccomment">/** Utility method to convert a BioJava ProteinSequence object to the FastaSequence </em>
<a class="jxr_linenumber" name="L149" href="#L149">149</a> <em class="jxr_javadoccomment">	 *  object used internally in JRonn.</em>
<a class="jxr_linenumber" name="L150" href="#L150">150</a> <em class="jxr_javadoccomment">	 * </em>
<a class="jxr_linenumber" name="L151" href="#L151">151</a> <em class="jxr_javadoccomment">	 * @param sequence</em>
<a class="jxr_linenumber" name="L152" href="#L152">152</a> <em class="jxr_javadoccomment">	 * @return</em>
<a class="jxr_linenumber" name="L153" href="#L153">153</a> <em class="jxr_javadoccomment">	 */</em>
<a class="jxr_linenumber" name="L154" href="#L154">154</a> 	<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">static</strong> <a href="../../../../org/biojava/nbio/data/sequence/FastaSequence.html">FastaSequence</a> convertProteinSequencetoFasta(ProteinSequence sequence){
<a class="jxr_linenumber" name="L155" href="#L155">155</a> 		StringBuffer buf = <strong class="jxr_keyword">new</strong> StringBuffer();
<a class="jxr_linenumber" name="L156" href="#L156">156</a> 		<strong class="jxr_keyword">for</strong> (AminoAcidCompound compound : sequence) {
<a class="jxr_linenumber" name="L157" href="#L157">157</a> 			
<a class="jxr_linenumber" name="L158" href="#L158">158</a> 			String c = compound.getShortName();
<a class="jxr_linenumber" name="L159" href="#L159">159</a> 			
<a class="jxr_linenumber" name="L160" href="#L160">160</a> 			<strong class="jxr_keyword">if</strong> (! SequenceUtil.NON_AA.matcher(c).find()) {
<a class="jxr_linenumber" name="L161" href="#L161">161</a> 				buf.append(c);
<a class="jxr_linenumber" name="L162" href="#L162">162</a> 			} <strong class="jxr_keyword">else</strong> {				
<a class="jxr_linenumber" name="L163" href="#L163">163</a> 				buf.append(<span class="jxr_string">"X"</span>);
<a class="jxr_linenumber" name="L164" href="#L164">164</a> 			}									
<a class="jxr_linenumber" name="L165" href="#L165">165</a> 		}
<a class="jxr_linenumber" name="L166" href="#L166">166</a> 		
<a class="jxr_linenumber" name="L167" href="#L167">167</a> 		<strong class="jxr_keyword">return</strong> <strong class="jxr_keyword">new</strong> <a href="../../../../org/biojava/nbio/data/sequence/FastaSequence.html">FastaSequence</a>(sequence.getAccession().getID(),buf.toString());
<a class="jxr_linenumber" name="L168" href="#L168">168</a> 	}
<a class="jxr_linenumber" name="L169" href="#L169">169</a> 
<a class="jxr_linenumber" name="L170" href="#L170">170</a> 	<strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">float</strong>[] predictSerial(<a href="../../../../org/biojava/nbio/data/sequence/FastaSequence.html">FastaSequence</a> fsequence) {
<a class="jxr_linenumber" name="L171" href="#L171">171</a> 		ORonn.validateSequenceForRonn(fsequence);
<a class="jxr_linenumber" name="L172" href="#L172">172</a> 		ORonn ronn;
<a class="jxr_linenumber" name="L173" href="#L173">173</a> 		<strong class="jxr_keyword">float</strong>[] disorder = <strong class="jxr_keyword">null</strong>; 
<a class="jxr_linenumber" name="L174" href="#L174">174</a> 		<strong class="jxr_keyword">try</strong> {
<a class="jxr_linenumber" name="L175" href="#L175">175</a> 			ronn = <strong class="jxr_keyword">new</strong> <a href="../../../../org/biojava/nbio/ronn/ORonn.html">ORonn</a>(fsequence, loader);
<a class="jxr_linenumber" name="L176" href="#L176">176</a> 			disorder = ronn.call().getMeanScores();
<a class="jxr_linenumber" name="L177" href="#L177">177</a> 		} <strong class="jxr_keyword">catch</strong> (NumberFormatException e) {
<a class="jxr_linenumber" name="L178" href="#L178">178</a> 			<strong class="jxr_keyword">throw</strong> <strong class="jxr_keyword">new</strong> RuntimeException(<span class="jxr_string">"Jronn fails to load models "</span> + e.getLocalizedMessage(), e);
<a class="jxr_linenumber" name="L179" href="#L179">179</a> 		} <strong class="jxr_keyword">catch</strong> (IOException e) {
<a class="jxr_linenumber" name="L180" href="#L180">180</a> 			<strong class="jxr_keyword">throw</strong> <strong class="jxr_keyword">new</strong> RuntimeException(<span class="jxr_string">"Jronn fails to load models "</span> + e.getLocalizedMessage(), e);
<a class="jxr_linenumber" name="L181" href="#L181">181</a> 		}
<a class="jxr_linenumber" name="L182" href="#L182">182</a> 		<strong class="jxr_keyword">return</strong> disorder;  
<a class="jxr_linenumber" name="L183" href="#L183">183</a> 	}
<a class="jxr_linenumber" name="L184" href="#L184">184</a> 
<a class="jxr_linenumber" name="L185" href="#L185">185</a> 	<em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L186" href="#L186">186</a> <em class="jxr_javadoccomment">	 * Calculates the disordered regions of the sequence. More formally, the regions for which the </em>
<a class="jxr_linenumber" name="L187" href="#L187">187</a> <em class="jxr_javadoccomment">	 * probability of disorder is greater then 0.50.  </em>
<a class="jxr_linenumber" name="L188" href="#L188">188</a> <em class="jxr_javadoccomment">	 *  </em>
<a class="jxr_linenumber" name="L189" href="#L189">189</a> <em class="jxr_javadoccomment">	 *   </em>
<a class="jxr_linenumber" name="L190" href="#L190">190</a> <em class="jxr_javadoccomment">	 * @param sequence an instance of FastaSequence object, holding the name and the sequence.</em>
<a class="jxr_linenumber" name="L191" href="#L191">191</a> <em class="jxr_javadoccomment">	 * @return the array of ranges if there are any residues predicted to have the </em>
<a class="jxr_linenumber" name="L192" href="#L192">192</a> <em class="jxr_javadoccomment">	 * probability of disorder greater then 0.5, null otherwise. </em>
<a class="jxr_linenumber" name="L193" href="#L193">193</a> <em class="jxr_javadoccomment">	 *</em>
<a class="jxr_linenumber" name="L194" href="#L194">194</a> <em class="jxr_javadoccomment">	 */</em>
<a class="jxr_linenumber" name="L195" href="#L195">195</a> 	<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">static</strong> <a href="../../../../org/biojava/nbio/ronn/Jronn.html">Range</a>[] getDisorder(<a href="../../../../org/biojava/nbio/data/sequence/FastaSequence.html">FastaSequence</a> sequence) {
<a class="jxr_linenumber" name="L196" href="#L196">196</a> 		<strong class="jxr_keyword">float</strong>[] scores = getDisorderScores(sequence);
<a class="jxr_linenumber" name="L197" href="#L197">197</a> 		<strong class="jxr_keyword">return</strong> scoresToRanges(scores, RonnConstraint.DEFAULT_RANGE_PROBABILITY_THRESHOLD);
<a class="jxr_linenumber" name="L198" href="#L198">198</a> 	}
<a class="jxr_linenumber" name="L199" href="#L199">199</a> 
<a class="jxr_linenumber" name="L200" href="#L200">200</a> 	<em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L201" href="#L201">201</a> <em class="jxr_javadoccomment">	 * Convert raw scores to ranges. Gives ranges for given probability of disorder value </em>
<a class="jxr_linenumber" name="L202" href="#L202">202</a> <em class="jxr_javadoccomment">	 * @param scores the raw probability of disorder scores for each residue in the sequence.  </em>
<a class="jxr_linenumber" name="L203" href="#L203">203</a> <em class="jxr_javadoccomment">	 * @param probability the cut off threshold. Include all residues with the probability of disorder greater then this value</em>
<a class="jxr_linenumber" name="L204" href="#L204">204</a> <em class="jxr_javadoccomment">	 * @return the array of ranges if there are any residues predicted to have the </em>
<a class="jxr_linenumber" name="L205" href="#L205">205</a> <em class="jxr_javadoccomment">	 * probability of disorder greater then {@code probability}, null otherwise.</em>
<a class="jxr_linenumber" name="L206" href="#L206">206</a> <em class="jxr_javadoccomment">	 */</em>
<a class="jxr_linenumber" name="L207" href="#L207">207</a> 	<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">static</strong> <a href="../../../../org/biojava/nbio/ronn/Jronn.html">Range</a>[] scoresToRanges(<strong class="jxr_keyword">float</strong>[] scores, <strong class="jxr_keyword">float</strong> probability)  {
<a class="jxr_linenumber" name="L208" href="#L208">208</a> 		assert scores!=<strong class="jxr_keyword">null</strong> &amp;&amp; scores.length&gt;0;
<a class="jxr_linenumber" name="L209" href="#L209">209</a> 		assert probability&gt;0 &amp;&amp; probability&lt;1;
<a class="jxr_linenumber" name="L210" href="#L210">210</a> 		
<a class="jxr_linenumber" name="L211" href="#L211">211</a> 		<strong class="jxr_keyword">int</strong> count=0;
<a class="jxr_linenumber" name="L212" href="#L212">212</a> 		<strong class="jxr_keyword">int</strong> regionLen=0;
<a class="jxr_linenumber" name="L213" href="#L213">213</a> 		List&lt;Range&gt; ranges = <strong class="jxr_keyword">new</strong> ArrayList&lt;Range&gt;();
<a class="jxr_linenumber" name="L214" href="#L214">214</a> 		<strong class="jxr_keyword">for</strong>(<strong class="jxr_keyword">float</strong> score: scores) { 
<a class="jxr_linenumber" name="L215" href="#L215">215</a> 			count++;
<a class="jxr_linenumber" name="L216" href="#L216">216</a> 			<em class="jxr_comment">// Round to 2 decimal points before comparison </em>
<a class="jxr_linenumber" name="L217" href="#L217">217</a> 			score = (<strong class="jxr_keyword">float</strong>) (Math.round(score*100.0)/100.0);
<a class="jxr_linenumber" name="L218" href="#L218">218</a> 			<strong class="jxr_keyword">if</strong>(score&gt;probability) {
<a class="jxr_linenumber" name="L219" href="#L219">219</a> 				regionLen++;
<a class="jxr_linenumber" name="L220" href="#L220">220</a> 			} <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L221" href="#L221">221</a> 				<strong class="jxr_keyword">if</strong>(regionLen&gt;0) {
<a class="jxr_linenumber" name="L222" href="#L222">222</a> 					ranges.add(<strong class="jxr_keyword">new</strong> <a href="../../../../org/biojava/nbio/ronn/Jronn.html">Range</a>(count-regionLen, count-1,score));
<a class="jxr_linenumber" name="L223" href="#L223">223</a> 				}
<a class="jxr_linenumber" name="L224" href="#L224">224</a> 				regionLen=0;
<a class="jxr_linenumber" name="L225" href="#L225">225</a> 			}
<a class="jxr_linenumber" name="L226" href="#L226">226</a> 		}
<a class="jxr_linenumber" name="L227" href="#L227">227</a> 		<em class="jxr_comment">// In case of the range to boundary runs to the very end of the sequence </em>
<a class="jxr_linenumber" name="L228" href="#L228">228</a> 		<strong class="jxr_keyword">if</strong>(regionLen&gt;1) {
<a class="jxr_linenumber" name="L229" href="#L229">229</a> 			ranges.add(<strong class="jxr_keyword">new</strong> <a href="../../../../org/biojava/nbio/ronn/Jronn.html">Range</a>(count-regionLen+1, count,scores[scores.length-1]));
<a class="jxr_linenumber" name="L230" href="#L230">230</a> 		}
<a class="jxr_linenumber" name="L231" href="#L231">231</a> 		<strong class="jxr_keyword">return</strong> ranges.toArray(<strong class="jxr_keyword">new</strong> <a href="../../../../org/biojava/nbio/ronn/Jronn.html">Range</a>[ranges.size()]); 		
<a class="jxr_linenumber" name="L232" href="#L232">232</a> 
<a class="jxr_linenumber" name="L233" href="#L233">233</a> 	}
<a class="jxr_linenumber" name="L234" href="#L234">234</a> 	
<a class="jxr_linenumber" name="L235" href="#L235">235</a> 	<em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L236" href="#L236">236</a> <em class="jxr_javadoccomment">	 * Calculates the probability of disorder scores for each residue in the sequence for </em>
<a class="jxr_linenumber" name="L237" href="#L237">237</a> <em class="jxr_javadoccomment">	 * many sequences in the input.</em>
<a class="jxr_linenumber" name="L238" href="#L238">238</a> <em class="jxr_javadoccomment">	 * </em>
<a class="jxr_linenumber" name="L239" href="#L239">239</a> <em class="jxr_javadoccomment">	 * @param sequences the list of the FastaSequence objects </em>
<a class="jxr_linenumber" name="L240" href="#L240">240</a> <em class="jxr_javadoccomment">	 * @return the Map with key-&gt;FastaSequence, value-&gt;probability of disorder for each residue</em>
<a class="jxr_linenumber" name="L241" href="#L241">241</a> <em class="jxr_javadoccomment">	 * @see #getDisorder(FastaSequence)</em>
<a class="jxr_linenumber" name="L242" href="#L242">242</a> <em class="jxr_javadoccomment">	 */</em>
<a class="jxr_linenumber" name="L243" href="#L243">243</a> 	<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">static</strong> Map&lt;FastaSequence,<strong class="jxr_keyword">float</strong>[]&gt; getDisorderScores(List&lt;FastaSequence&gt; sequences) {
<a class="jxr_linenumber" name="L244" href="#L244">244</a> 		Map&lt;FastaSequence,<strong class="jxr_keyword">float</strong>[]&gt; results = <strong class="jxr_keyword">new</strong> TreeMap&lt;FastaSequence, <strong class="jxr_keyword">float</strong>[]&gt;();
<a class="jxr_linenumber" name="L245" href="#L245">245</a> 		<strong class="jxr_keyword">for</strong>(<a href="../../../../org/biojava/nbio/data/sequence/FastaSequence.html">FastaSequence</a> fsequence : sequences) {
<a class="jxr_linenumber" name="L246" href="#L246">246</a> 			results.put(fsequence, predictSerial(fsequence));
<a class="jxr_linenumber" name="L247" href="#L247">247</a> 		}
<a class="jxr_linenumber" name="L248" href="#L248">248</a> 		<strong class="jxr_keyword">return</strong> results; 
<a class="jxr_linenumber" name="L249" href="#L249">249</a> 	}
<a class="jxr_linenumber" name="L250" href="#L250">250</a> 	
<a class="jxr_linenumber" name="L251" href="#L251">251</a> 	<em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L252" href="#L252">252</a> <em class="jxr_javadoccomment">	 * Calculates the disordered regions of the sequence for many sequences in the input.</em>
<a class="jxr_linenumber" name="L253" href="#L253">253</a> <em class="jxr_javadoccomment">	 * </em>
<a class="jxr_linenumber" name="L254" href="#L254">254</a> <em class="jxr_javadoccomment">	 * @param sequences sequences the list of the FastaSequence objects</em>
<a class="jxr_linenumber" name="L255" href="#L255">255</a> <em class="jxr_javadoccomment">	 * @return</em>
<a class="jxr_linenumber" name="L256" href="#L256">256</a> <em class="jxr_javadoccomment">	 * @see #getDisorder(FastaSequence)</em>
<a class="jxr_linenumber" name="L257" href="#L257">257</a> <em class="jxr_javadoccomment">	 */</em>
<a class="jxr_linenumber" name="L258" href="#L258">258</a> 	<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">static</strong> Map&lt;FastaSequence,Range[]&gt; getDisorder(List&lt;FastaSequence&gt; sequences) {
<a class="jxr_linenumber" name="L259" href="#L259">259</a> 		Map&lt;FastaSequence,Range[]&gt; disorderRanges = <strong class="jxr_keyword">new</strong> TreeMap&lt;FastaSequence,Range[]&gt;();
<a class="jxr_linenumber" name="L260" href="#L260">260</a> 		<strong class="jxr_keyword">for</strong>(<a href="../../../../org/biojava/nbio/data/sequence/FastaSequence.html">FastaSequence</a> fs: sequences) {
<a class="jxr_linenumber" name="L261" href="#L261">261</a> 			disorderRanges.put(fs, getDisorder(fs));
<a class="jxr_linenumber" name="L262" href="#L262">262</a> 		}
<a class="jxr_linenumber" name="L263" href="#L263">263</a> 		<strong class="jxr_keyword">return</strong> disorderRanges; 
<a class="jxr_linenumber" name="L264" href="#L264">264</a> 	}
<a class="jxr_linenumber" name="L265" href="#L265">265</a> 	
<a class="jxr_linenumber" name="L266" href="#L266">266</a> 	<em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L267" href="#L267">267</a> <em class="jxr_javadoccomment">	 * Calculates the disordered regions of the protein sequence.</em>
<a class="jxr_linenumber" name="L268" href="#L268">268</a> <em class="jxr_javadoccomment">	 * @param fastaFile input file name containing the sequence in FASTA</em>
<a class="jxr_linenumber" name="L269" href="#L269">269</a> <em class="jxr_javadoccomment">	 * @return the Map with key-&gt;FastaSequence, value-&gt;the list of disordered regions for each sequence</em>
<a class="jxr_linenumber" name="L270" href="#L270">270</a> <em class="jxr_javadoccomment">	 * @throws FileNotFoundException if the input file cannot be found</em>
<a class="jxr_linenumber" name="L271" href="#L271">271</a> <em class="jxr_javadoccomment">	 * @throws IOException of the system cannot access or read from the input file </em>
<a class="jxr_linenumber" name="L272" href="#L272">272</a> <em class="jxr_javadoccomment">	 * @see #getDisorder(FastaSequence)</em>
<a class="jxr_linenumber" name="L273" href="#L273">273</a> <em class="jxr_javadoccomment">	 * @see #Jronn.Range</em>
<a class="jxr_linenumber" name="L274" href="#L274">274</a> <em class="jxr_javadoccomment">	 */</em>
<a class="jxr_linenumber" name="L275" href="#L275">275</a> 	<strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">static</strong> Map&lt;FastaSequence,Range[]&gt; getDisorder(String fastaFile) <strong class="jxr_keyword">throws</strong> FileNotFoundException, IOException {
<a class="jxr_linenumber" name="L276" href="#L276">276</a> 		<strong class="jxr_keyword">final</strong> List&lt;FastaSequence&gt; sequences = SequenceUtil.readFasta(<strong class="jxr_keyword">new</strong> FileInputStream(fastaFile));
<a class="jxr_linenumber" name="L277" href="#L277">277</a> 		<strong class="jxr_keyword">return</strong> getDisorder(sequences);
<a class="jxr_linenumber" name="L278" href="#L278">278</a> 	}
<a class="jxr_linenumber" name="L279" href="#L279">279</a> 	
<a class="jxr_linenumber" name="L280" href="#L280">280</a> 	<em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L281" href="#L281">281</a> <em class="jxr_javadoccomment">	 * TODO </em>
<a class="jxr_linenumber" name="L282" href="#L282">282</a> <em class="jxr_javadoccomment">	 * </em>
<a class="jxr_linenumber" name="L283" href="#L283">283</a> <em class="jxr_javadoccomment">	 * High performance method for calculating disorder. Use multiple threads to achieve the speedup.</em>
<a class="jxr_linenumber" name="L284" href="#L284">284</a> <em class="jxr_javadoccomment">	 *  </em>
<a class="jxr_linenumber" name="L285" href="#L285">285</a> <em class="jxr_javadoccomment">	 * @param fastaFile  fully qualified path to the input FASTA file  </em>
<a class="jxr_linenumber" name="L286" href="#L286">286</a> <em class="jxr_javadoccomment">	 * @param outputFile file name of the file for the results </em>
<a class="jxr_linenumber" name="L287" href="#L287">287</a> <em class="jxr_javadoccomment">	 * @param threadNumber the number of threads to use, default</em>
<a class="jxr_linenumber" name="L288" href="#L288">288</a> <em class="jxr_javadoccomment">	 * @param controls the format of the result file </em>
<a class="jxr_linenumber" name="L289" href="#L289">289</a> <em class="jxr_javadoccomment">	 * @throws FileNotFoundException if input file in not found </em>
<a class="jxr_linenumber" name="L290" href="#L290">290</a> <em class="jxr_javadoccomment">	 * @throws IOException if the input or the output files cannot be accessed  </em>
<a class="jxr_linenumber" name="L291" href="#L291">291</a> <em class="jxr_javadoccomment">	 * @see ORonn.ResultLayout</em>
<a class="jxr_linenumber" name="L292" href="#L292">292</a> <em class="jxr_javadoccomment">	 </em>
<a class="jxr_linenumber" name="L293" href="#L293">293</a> <em class="jxr_javadoccomment">	public static void calculateDisorder(String fastaFile, String outputFile, int threadNumber, ResultLayout layout) throws FileNotFoundException, IOException {</em>
<a class="jxr_linenumber" name="L294" href="#L294">294</a> <em class="jxr_javadoccomment">		final List&lt;FastaSequence&gt; sequences = SequenceUtil.readFasta(new FileInputStream(fastaFile));</em>
<a class="jxr_linenumber" name="L295" href="#L295">295</a> <em class="jxr_javadoccomment">		InputParameters in = new InputParameters(); </em>
<a class="jxr_linenumber" name="L296" href="#L296">296</a> <em class="jxr_javadoccomment">		in.setFilePrm(fastaFile, InputParameters.inputKey);</em>
<a class="jxr_linenumber" name="L297" href="#L297">297</a> <em class="jxr_javadoccomment">		in.setFilePrm(outputFile, InputParameters.outputKey);</em>
<a class="jxr_linenumber" name="L298" href="#L298">298</a> <em class="jxr_javadoccomment">		//in.setThreadNum(Integer.toString(threadNumber)); </em>
<a class="jxr_linenumber" name="L299" href="#L299">299</a> <em class="jxr_javadoccomment">		ORonn.predictParallel(sequences, in, loader); </em>
<a class="jxr_linenumber" name="L300" href="#L300">300</a> <em class="jxr_javadoccomment">	}</em>
<a class="jxr_linenumber" name="L301" href="#L301">301</a> <em class="jxr_javadoccomment">	*/</em> 
<a class="jxr_linenumber" name="L302" href="#L302">302</a> }
</pre>
<hr/>
<div id="footer">Copyright &#169; 2000&#x2013;2015 <a href="http://www.biojava.org">BioJava</a>. All rights reserved.</div>
</body>
</html>
