<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head><meta http-equiv="content-type" content="text/html; charset=UTF-8" />
<title>GeneFeatureHelper xref</title>
<link type="text/css" rel="stylesheet" href="../../../../stylesheet.css" />
</head>
<body>
<div id="overview"><a href="../../../../../apidocs/org/biojava/nbio/genome/GeneFeatureHelper.html">View Javadoc</a></div><pre>
<a class="jxr_linenumber" name="L1" href="#L1">1</a>   <em class="jxr_comment">/*</em>
<a class="jxr_linenumber" name="L2" href="#L2">2</a>   <em class="jxr_comment"> *                    BioJava development code</em>
<a class="jxr_linenumber" name="L3" href="#L3">3</a>   <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L4" href="#L4">4</a>   <em class="jxr_comment"> * This code may be freely distributed and modified under the</em>
<a class="jxr_linenumber" name="L5" href="#L5">5</a>   <em class="jxr_comment"> * terms of the GNU Lesser General Public Licence.  This should</em>
<a class="jxr_linenumber" name="L6" href="#L6">6</a>   <em class="jxr_comment"> * be distributed with the code.  If you do not have a copy,</em>
<a class="jxr_linenumber" name="L7" href="#L7">7</a>   <em class="jxr_comment"> * see:</em>
<a class="jxr_linenumber" name="L8" href="#L8">8</a>   <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L9" href="#L9">9</a>   <em class="jxr_comment"> *      <a href="http://www.gnu.org/copyleft/lesser.htm" target="alexandria_uri">http://www.gnu.org/copyleft/lesser.htm</a>l</em>
<a class="jxr_linenumber" name="L10" href="#L10">10</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L11" href="#L11">11</a>  <em class="jxr_comment"> * Copyright for this code is held jointly by the individual</em>
<a class="jxr_linenumber" name="L12" href="#L12">12</a>  <em class="jxr_comment"> * authors.  These should be listed in @author doc comments.</em>
<a class="jxr_linenumber" name="L13" href="#L13">13</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L14" href="#L14">14</a>  <em class="jxr_comment"> * For more information on the BioJava project and its aims,</em>
<a class="jxr_linenumber" name="L15" href="#L15">15</a>  <em class="jxr_comment"> * or to join the biojava-l mailing list, visit the home page</em>
<a class="jxr_linenumber" name="L16" href="#L16">16</a>  <em class="jxr_comment"> * at:</em>
<a class="jxr_linenumber" name="L17" href="#L17">17</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L18" href="#L18">18</a>  <em class="jxr_comment"> *      <a href="http://www.biojava.org" target="alexandria_uri">http://www.biojava.org</a>/</em>
<a class="jxr_linenumber" name="L19" href="#L19">19</a>  <em class="jxr_comment"> *</em>
<a class="jxr_linenumber" name="L20" href="#L20">20</a>  <em class="jxr_comment"> */</em>
<a class="jxr_linenumber" name="L21" href="#L21">21</a>  <em class="jxr_comment">/*</em>
<a class="jxr_linenumber" name="L22" href="#L22">22</a>  <em class="jxr_comment"> * To change this template, choose Tools | Templates</em>
<a class="jxr_linenumber" name="L23" href="#L23">23</a>  <em class="jxr_comment"> * and open the template in the editor.</em>
<a class="jxr_linenumber" name="L24" href="#L24">24</a>  <em class="jxr_comment"> */</em>
<a class="jxr_linenumber" name="L25" href="#L25">25</a>  <strong class="jxr_keyword">package</strong> org.biojava.nbio.genome;
<a class="jxr_linenumber" name="L26" href="#L26">26</a>  
<a class="jxr_linenumber" name="L27" href="#L27">27</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.genome.parsers.gff.*;
<a class="jxr_linenumber" name="L28" href="#L28">28</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.*;
<a class="jxr_linenumber" name="L29" href="#L29">29</a>  <strong class="jxr_keyword">import</strong> org.biojava.nbio.core.sequence.io.FastaReaderHelper;
<a class="jxr_linenumber" name="L30" href="#L30">30</a>  <strong class="jxr_keyword">import</strong> org.slf4j.Logger;
<a class="jxr_linenumber" name="L31" href="#L31">31</a>  <strong class="jxr_keyword">import</strong> org.slf4j.LoggerFactory;
<a class="jxr_linenumber" name="L32" href="#L32">32</a>  
<a class="jxr_linenumber" name="L33" href="#L33">33</a>  <strong class="jxr_keyword">import</strong> java.io.File;
<a class="jxr_linenumber" name="L34" href="#L34">34</a>  <strong class="jxr_keyword">import</strong> java.io.FileWriter;
<a class="jxr_linenumber" name="L35" href="#L35">35</a>  <strong class="jxr_keyword">import</strong> java.util.ArrayList;
<a class="jxr_linenumber" name="L36" href="#L36">36</a>  <strong class="jxr_keyword">import</strong> java.util.Collection;
<a class="jxr_linenumber" name="L37" href="#L37">37</a>  <strong class="jxr_keyword">import</strong> java.util.Collections;
<a class="jxr_linenumber" name="L38" href="#L38">38</a>  <strong class="jxr_keyword">import</strong> java.util.LinkedHashMap;
<a class="jxr_linenumber" name="L39" href="#L39">39</a>  
<a class="jxr_linenumber" name="L40" href="#L40">40</a>  <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L41" href="#L41">41</a>  <em class="jxr_javadoccomment"> *</em>
<a class="jxr_linenumber" name="L42" href="#L42">42</a>  <em class="jxr_javadoccomment"> * @author Scooter Willis &lt;willishf at gmail dot com&gt;</em>
<a class="jxr_linenumber" name="L43" href="#L43">43</a>  <em class="jxr_javadoccomment"> */</em>
<a class="jxr_linenumber" name="L44" href="#L44">44</a>  <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">class</strong> <a href="../../../../org/biojava/nbio/genome/GeneFeatureHelper.html">GeneFeatureHelper</a> {
<a class="jxr_linenumber" name="L45" href="#L45">45</a>  
<a class="jxr_linenumber" name="L46" href="#L46">46</a>  	<strong class="jxr_keyword">private</strong> <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">final</strong> Logger logger = LoggerFactory.getLogger(GeneFeatureHelper.<strong class="jxr_keyword">class</strong>);
<a class="jxr_linenumber" name="L47" href="#L47">47</a>  
<a class="jxr_linenumber" name="L48" href="#L48">48</a>      <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">public</strong> LinkedHashMap&lt;String, ChromosomeSequence&gt; loadFastaAddGeneFeaturesFromUpperCaseExonFastaFile(File fastaSequenceFile, File uppercaseFastaFile, <strong class="jxr_keyword">boolean</strong> throwExceptionGeneNotFound) <strong class="jxr_keyword">throws</strong> Exception {
<a class="jxr_linenumber" name="L49" href="#L49">49</a>          LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList = <strong class="jxr_keyword">new</strong> LinkedHashMap&lt;String, ChromosomeSequence&gt;();
<a class="jxr_linenumber" name="L50" href="#L50">50</a>          LinkedHashMap&lt;String, DNASequence&gt; dnaSequenceList = FastaReaderHelper.readFastaDNASequence(fastaSequenceFile);
<a class="jxr_linenumber" name="L51" href="#L51">51</a>          <strong class="jxr_keyword">for</strong> (String accession : dnaSequenceList.keySet()) {
<a class="jxr_linenumber" name="L52" href="#L52">52</a>              DNASequence contigSequence = dnaSequenceList.get(accession);
<a class="jxr_linenumber" name="L53" href="#L53">53</a>              ChromosomeSequence chromsomeSequence = <strong class="jxr_keyword">new</strong> ChromosomeSequence(contigSequence.getSequenceAsString());
<a class="jxr_linenumber" name="L54" href="#L54">54</a>              chromsomeSequence.setAccession(contigSequence.getAccession());
<a class="jxr_linenumber" name="L55" href="#L55">55</a>              chromosomeSequenceList.put(accession, chromsomeSequence);
<a class="jxr_linenumber" name="L56" href="#L56">56</a>          }
<a class="jxr_linenumber" name="L57" href="#L57">57</a>  
<a class="jxr_linenumber" name="L58" href="#L58">58</a>  
<a class="jxr_linenumber" name="L59" href="#L59">59</a>          LinkedHashMap&lt;String, DNASequence&gt; geneSequenceList = FastaReaderHelper.readFastaDNASequence(uppercaseFastaFile);
<a class="jxr_linenumber" name="L60" href="#L60">60</a>          <strong class="jxr_keyword">for</strong> (DNASequence dnaSequence : geneSequenceList.values()) {
<a class="jxr_linenumber" name="L61" href="#L61">61</a>              String geneSequence = dnaSequence.getSequenceAsString();
<a class="jxr_linenumber" name="L62" href="#L62">62</a>              String lcGeneSequence = geneSequence.toLowerCase();
<a class="jxr_linenumber" name="L63" href="#L63">63</a>              String reverseGeneSequence = dnaSequence.getReverse().getSequenceAsString();
<a class="jxr_linenumber" name="L64" href="#L64">64</a>              String lcReverseGeneSequence = reverseGeneSequence.toLowerCase();
<a class="jxr_linenumber" name="L65" href="#L65">65</a>              Integer bioStart = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L66" href="#L66">66</a>              Integer bioEnd = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L67" href="#L67">67</a>              Strand strand = Strand.POSITIVE;
<a class="jxr_linenumber" name="L68" href="#L68">68</a>              <strong class="jxr_keyword">boolean</strong> geneFound = false;
<a class="jxr_linenumber" name="L69" href="#L69">69</a>              String accession = <span class="jxr_string">""</span>;
<a class="jxr_linenumber" name="L70" href="#L70">70</a>              DNASequence contigDNASequence = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L71" href="#L71">71</a>              <strong class="jxr_keyword">for</strong> (String id : dnaSequenceList.keySet()) {
<a class="jxr_linenumber" name="L72" href="#L72">72</a>                  accession = id;
<a class="jxr_linenumber" name="L73" href="#L73">73</a>                  contigDNASequence = dnaSequenceList.get(id);
<a class="jxr_linenumber" name="L74" href="#L74">74</a>                  String contigSequence = contigDNASequence.getSequenceAsString().toLowerCase();
<a class="jxr_linenumber" name="L75" href="#L75">75</a>                  bioStart = contigSequence.indexOf(lcGeneSequence);
<a class="jxr_linenumber" name="L76" href="#L76">76</a>                  <strong class="jxr_keyword">if</strong> (bioStart != -1) {
<a class="jxr_linenumber" name="L77" href="#L77">77</a>                      bioStart = bioStart + 1;
<a class="jxr_linenumber" name="L78" href="#L78">78</a>                      bioEnd = bioStart + geneSequence.length() - 1;
<a class="jxr_linenumber" name="L79" href="#L79">79</a>                      geneFound = <strong class="jxr_keyword">true</strong>;
<a class="jxr_linenumber" name="L80" href="#L80">80</a>                      <strong class="jxr_keyword">break</strong>;
<a class="jxr_linenumber" name="L81" href="#L81">81</a>                  } <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L82" href="#L82">82</a>                      bioStart = contigSequence.indexOf(lcReverseGeneSequence);
<a class="jxr_linenumber" name="L83" href="#L83">83</a>                      <strong class="jxr_keyword">if</strong> (bioStart != -1) {
<a class="jxr_linenumber" name="L84" href="#L84">84</a>                          bioStart = bioStart + 1;
<a class="jxr_linenumber" name="L85" href="#L85">85</a>                          bioEnd = bioStart - geneSequence.length() - 1;
<a class="jxr_linenumber" name="L86" href="#L86">86</a>                          strand = Strand.NEGATIVE;
<a class="jxr_linenumber" name="L87" href="#L87">87</a>                          geneFound = <strong class="jxr_keyword">true</strong>;
<a class="jxr_linenumber" name="L88" href="#L88">88</a>                          <strong class="jxr_keyword">break</strong>;
<a class="jxr_linenumber" name="L89" href="#L89">89</a>                      }
<a class="jxr_linenumber" name="L90" href="#L90">90</a>                  }
<a class="jxr_linenumber" name="L91" href="#L91">91</a>              }
<a class="jxr_linenumber" name="L92" href="#L92">92</a>  
<a class="jxr_linenumber" name="L93" href="#L93">93</a>              <strong class="jxr_keyword">if</strong> (geneFound) {
<a class="jxr_linenumber" name="L94" href="#L94">94</a>                  logger.info(<span class="jxr_string">"Gene {} found at {} {} {} {}"</span>,
<a class="jxr_linenumber" name="L95" href="#L95">95</a>                  		dnaSequence.getAccession().toString(), contigDNASequence.getAccession().toString(), bioStart, bioEnd, strand);
<a class="jxr_linenumber" name="L96" href="#L96">96</a>                  ChromosomeSequence chromosomeSequence = chromosomeSequenceList.get(accession);
<a class="jxr_linenumber" name="L97" href="#L97">97</a>  
<a class="jxr_linenumber" name="L98" href="#L98">98</a>                  ArrayList&lt;Integer&gt; exonBoundries = <strong class="jxr_keyword">new</strong> ArrayList&lt;Integer&gt;();
<a class="jxr_linenumber" name="L99" href="#L99">99</a>  
<a class="jxr_linenumber" name="L100" href="#L100">100</a>                 <em class="jxr_comment">//look for transitions from lowercase to upper case</em>
<a class="jxr_linenumber" name="L101" href="#L101">101</a>                 <strong class="jxr_keyword">for</strong> (<strong class="jxr_keyword">int</strong> i = 0; i &lt; geneSequence.length(); i++) {
<a class="jxr_linenumber" name="L102" href="#L102">102</a>                     <strong class="jxr_keyword">if</strong> (i == 0 &amp;&amp; Character.isUpperCase(geneSequence.charAt(i))) {
<a class="jxr_linenumber" name="L103" href="#L103">103</a>                         exonBoundries.add(i);
<a class="jxr_linenumber" name="L104" href="#L104">104</a>                     } <strong class="jxr_keyword">else</strong> <strong class="jxr_keyword">if</strong> (i == geneSequence.length() - 1) {
<a class="jxr_linenumber" name="L105" href="#L105">105</a>                         exonBoundries.add(i);
<a class="jxr_linenumber" name="L106" href="#L106">106</a>                     } <strong class="jxr_keyword">else</strong> <strong class="jxr_keyword">if</strong> (Character.isUpperCase(geneSequence.charAt(i)) &amp;&amp; Character.isLowerCase(geneSequence.charAt(i - 1))) {
<a class="jxr_linenumber" name="L107" href="#L107">107</a>                         exonBoundries.add(i);
<a class="jxr_linenumber" name="L108" href="#L108">108</a>                     } <strong class="jxr_keyword">else</strong> <strong class="jxr_keyword">if</strong> (Character.isUpperCase(geneSequence.charAt(i)) &amp;&amp; Character.isLowerCase(geneSequence.charAt(i + 1))) {
<a class="jxr_linenumber" name="L109" href="#L109">109</a>                         exonBoundries.add(i);
<a class="jxr_linenumber" name="L110" href="#L110">110</a>                     }
<a class="jxr_linenumber" name="L111" href="#L111">111</a>                 }
<a class="jxr_linenumber" name="L112" href="#L112">112</a>                 <strong class="jxr_keyword">if</strong> (strand == Strand.NEGATIVE) {
<a class="jxr_linenumber" name="L113" href="#L113">113</a>                     Collections.reverse(exonBoundries);
<a class="jxr_linenumber" name="L114" href="#L114">114</a>                 }
<a class="jxr_linenumber" name="L115" href="#L115">115</a> 
<a class="jxr_linenumber" name="L116" href="#L116">116</a> 
<a class="jxr_linenumber" name="L117" href="#L117">117</a>                 String geneaccession = dnaSequence.getAccession().getID();
<a class="jxr_linenumber" name="L118" href="#L118">118</a>                 String note = geneaccession;
<a class="jxr_linenumber" name="L119" href="#L119">119</a>                 String[] values = geneaccession.split(<span class="jxr_string">" "</span>);
<a class="jxr_linenumber" name="L120" href="#L120">120</a>                 geneaccession = values[0];
<a class="jxr_linenumber" name="L121" href="#L121">121</a> 
<a class="jxr_linenumber" name="L122" href="#L122">122</a> 
<a class="jxr_linenumber" name="L123" href="#L123">123</a> 
<a class="jxr_linenumber" name="L124" href="#L124">124</a>                 GeneSequence geneSeq = chromosomeSequence.addGene(<strong class="jxr_keyword">new</strong> AccessionID(geneaccession), bioStart, bioEnd, strand);
<a class="jxr_linenumber" name="L125" href="#L125">125</a>                 geneSeq.addNote(note);
<a class="jxr_linenumber" name="L126" href="#L126">126</a>                 geneSeq.setSource(uppercaseFastaFile.getName());
<a class="jxr_linenumber" name="L127" href="#L127">127</a>                 <em class="jxr_comment">//String transcriptName = geneaccession + "-transcript";</em>
<a class="jxr_linenumber" name="L128" href="#L128">128</a>                 <em class="jxr_comment">//TranscriptSequence transcriptSequence = geneSeq.addTranscript(new AccessionID(transcriptName), bioStart, bioEnd);</em>
<a class="jxr_linenumber" name="L129" href="#L129">129</a> 
<a class="jxr_linenumber" name="L130" href="#L130">130</a>                 <strong class="jxr_keyword">int</strong> runningFrameLength = 0;
<a class="jxr_linenumber" name="L131" href="#L131">131</a>                 <strong class="jxr_keyword">for</strong> (<strong class="jxr_keyword">int</strong> i = 0; i &lt; exonBoundries.size() - 1; i = i + 2) {
<a class="jxr_linenumber" name="L132" href="#L132">132</a>                     <strong class="jxr_keyword">int</strong> cdsBioStart = exonBoundries.get(i) + bioStart;
<a class="jxr_linenumber" name="L133" href="#L133">133</a>                     <strong class="jxr_keyword">int</strong> cdsBioEnd = exonBoundries.get(i + 1) + bioStart;
<a class="jxr_linenumber" name="L134" href="#L134">134</a>                     runningFrameLength = runningFrameLength + Math.abs(cdsBioEnd - cdsBioStart) + 1;
<a class="jxr_linenumber" name="L135" href="#L135">135</a>                     <em class="jxr_comment">//String cdsName = transcriptName + "-cds-" + cdsBioStart + "-" + cdsBioEnd;</em>
<a class="jxr_linenumber" name="L136" href="#L136">136</a> 
<a class="jxr_linenumber" name="L137" href="#L137">137</a>                     <em class="jxr_comment">//AccessionID cdsAccessionID = new AccessionID(cdsName);</em>
<a class="jxr_linenumber" name="L138" href="#L138">138</a>                     <em class="jxr_comment">//ExonSequence exonSequence = geneSeq.addExon(cdsAccessionID, cdsBioStart, cdsBioEnd);</em>
<a class="jxr_linenumber" name="L139" href="#L139">139</a>                     <strong class="jxr_keyword">int</strong> remainder = runningFrameLength % 3;
<a class="jxr_linenumber" name="L140" href="#L140">140</a>                     <em class="jxr_comment">//int frame = 0;</em>
<a class="jxr_linenumber" name="L141" href="#L141">141</a>                     <strong class="jxr_keyword">if</strong> (remainder == 1) {
<a class="jxr_linenumber" name="L142" href="#L142">142</a>                         <em class="jxr_comment">//frame = 2; // borrow 2 from next CDS region</em>
<a class="jxr_linenumber" name="L143" href="#L143">143</a>                     } <strong class="jxr_keyword">else</strong> <strong class="jxr_keyword">if</strong> (remainder == 2) {
<a class="jxr_linenumber" name="L144" href="#L144">144</a>                         <em class="jxr_comment">//frame = 1;</em>
<a class="jxr_linenumber" name="L145" href="#L145">145</a>                     }
<a class="jxr_linenumber" name="L146" href="#L146">146</a>                     <em class="jxr_comment">//CDSSequence cdsSequence = transcriptSequence.addCDS(cdsAccessionID, cdsBioStart, cdsBioEnd, frame);</em>
<a class="jxr_linenumber" name="L147" href="#L147">147</a>                 }
<a class="jxr_linenumber" name="L148" href="#L148">148</a> 
<a class="jxr_linenumber" name="L149" href="#L149">149</a> 
<a class="jxr_linenumber" name="L150" href="#L150">150</a>             } <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L151" href="#L151">151</a>                 <strong class="jxr_keyword">if</strong> (throwExceptionGeneNotFound) {
<a class="jxr_linenumber" name="L152" href="#L152">152</a>                     <strong class="jxr_keyword">throw</strong> <strong class="jxr_keyword">new</strong> Exception(dnaSequence.getAccession().toString() + <span class="jxr_string">" not found"</span>);
<a class="jxr_linenumber" name="L153" href="#L153">153</a>                 }
<a class="jxr_linenumber" name="L154" href="#L154">154</a>                 logger.info(<span class="jxr_string">"Gene not found {}"</span>, dnaSequence.getAccession().toString());
<a class="jxr_linenumber" name="L155" href="#L155">155</a>             }
<a class="jxr_linenumber" name="L156" href="#L156">156</a> 
<a class="jxr_linenumber" name="L157" href="#L157">157</a>         }
<a class="jxr_linenumber" name="L158" href="#L158">158</a>         <strong class="jxr_keyword">return</strong> chromosomeSequenceList;
<a class="jxr_linenumber" name="L159" href="#L159">159</a>     }
<a class="jxr_linenumber" name="L160" href="#L160">160</a> 
<a class="jxr_linenumber" name="L161" href="#L161">161</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L162" href="#L162">162</a> <em class="jxr_javadoccomment">     * Output a gff3 feature file that will give the length of each scaffold/chromosome in the fasta file.</em>
<a class="jxr_linenumber" name="L163" href="#L163">163</a> <em class="jxr_javadoccomment">     * Used for gbrowse so it knows length.</em>
<a class="jxr_linenumber" name="L164" href="#L164">164</a> <em class="jxr_javadoccomment">     * @param fastaSequenceFile</em>
<a class="jxr_linenumber" name="L165" href="#L165">165</a> <em class="jxr_javadoccomment">     * @param gffFile</em>
<a class="jxr_linenumber" name="L166" href="#L166">166</a> <em class="jxr_javadoccomment">     * @throws Exception</em>
<a class="jxr_linenumber" name="L167" href="#L167">167</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L168" href="#L168">168</a>     <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> outputFastaSequenceLengthGFF3(File fastaSequenceFile, File gffFile) <strong class="jxr_keyword">throws</strong> Exception {
<a class="jxr_linenumber" name="L169" href="#L169">169</a>         LinkedHashMap&lt;String, DNASequence&gt; dnaSequenceList = FastaReaderHelper.readFastaDNASequence(fastaSequenceFile);
<a class="jxr_linenumber" name="L170" href="#L170">170</a>         String fileName = fastaSequenceFile.getName();
<a class="jxr_linenumber" name="L171" href="#L171">171</a>         FileWriter fw = <strong class="jxr_keyword">new</strong> FileWriter(gffFile);
<a class="jxr_linenumber" name="L172" href="#L172">172</a>         String newLine = System.getProperty(<span class="jxr_string">"line.separator"</span>);
<a class="jxr_linenumber" name="L173" href="#L173">173</a>         fw.write(<span class="jxr_string">"##gff-version 3"</span> + newLine);
<a class="jxr_linenumber" name="L174" href="#L174">174</a>         <strong class="jxr_keyword">for</strong> (DNASequence dnaSequence : dnaSequenceList.values()) {
<a class="jxr_linenumber" name="L175" href="#L175">175</a>             String gff3line = dnaSequence.getAccession().getID() + <span class="jxr_string">"\t"</span> + fileName + <span class="jxr_string">"\t"</span> + <span class="jxr_string">"contig"</span> + <span class="jxr_string">"\t"</span> + <span class="jxr_string">"1"</span> + <span class="jxr_string">"\t"</span> + dnaSequence.getBioEnd() + <span class="jxr_string">"\t.\t.\t.\tName="</span> + dnaSequence.getAccession().getID() + newLine;
<a class="jxr_linenumber" name="L176" href="#L176">176</a>             fw.write(gff3line);
<a class="jxr_linenumber" name="L177" href="#L177">177</a>         }
<a class="jxr_linenumber" name="L178" href="#L178">178</a>         fw.close();
<a class="jxr_linenumber" name="L179" href="#L179">179</a>     }
<a class="jxr_linenumber" name="L180" href="#L180">180</a> 
<a class="jxr_linenumber" name="L181" href="#L181">181</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L182" href="#L182">182</a> <em class="jxr_javadoccomment">     * Loads Fasta file and GFF2 feature file generated from the geneid prediction algorithm</em>
<a class="jxr_linenumber" name="L183" href="#L183">183</a> <em class="jxr_javadoccomment">     *</em>
<a class="jxr_linenumber" name="L184" href="#L184">184</a> <em class="jxr_javadoccomment">     * @param fastaSequenceFile</em>
<a class="jxr_linenumber" name="L185" href="#L185">185</a> <em class="jxr_javadoccomment">     * @param gffFile</em>
<a class="jxr_linenumber" name="L186" href="#L186">186</a> <em class="jxr_javadoccomment">     * @return</em>
<a class="jxr_linenumber" name="L187" href="#L187">187</a> <em class="jxr_javadoccomment">     * @throws Exception</em>
<a class="jxr_linenumber" name="L188" href="#L188">188</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L189" href="#L189">189</a>     <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">public</strong> LinkedHashMap&lt;String, ChromosomeSequence&gt; loadFastaAddGeneFeaturesFromGeneIDGFF2(File fastaSequenceFile, File gffFile) <strong class="jxr_keyword">throws</strong> Exception {
<a class="jxr_linenumber" name="L190" href="#L190">190</a>         LinkedHashMap&lt;String, DNASequence&gt; dnaSequenceList = FastaReaderHelper.readFastaDNASequence(fastaSequenceFile);
<a class="jxr_linenumber" name="L191" href="#L191">191</a>         LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList = GeneFeatureHelper.getChromosomeSequenceFromDNASequence(dnaSequenceList);
<a class="jxr_linenumber" name="L192" href="#L192">192</a>         <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> listGenes = GeneIDGFF2Reader.read(gffFile.getAbsolutePath());
<a class="jxr_linenumber" name="L193" href="#L193">193</a>         addGeneIDGFF2GeneFeatures(chromosomeSequenceList, listGenes);
<a class="jxr_linenumber" name="L194" href="#L194">194</a>         <strong class="jxr_keyword">return</strong> chromosomeSequenceList;
<a class="jxr_linenumber" name="L195" href="#L195">195</a>     }
<a class="jxr_linenumber" name="L196" href="#L196">196</a> 
<a class="jxr_linenumber" name="L197" href="#L197">197</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L198" href="#L198">198</a> <em class="jxr_javadoccomment">     * Load GFF2 feature file generated from the geneid prediction algorithm and map features onto the chromosome sequences</em>
<a class="jxr_linenumber" name="L199" href="#L199">199</a> <em class="jxr_javadoccomment">     *</em>
<a class="jxr_linenumber" name="L200" href="#L200">200</a> <em class="jxr_javadoccomment">     * @param chromosomeSequenceList</em>
<a class="jxr_linenumber" name="L201" href="#L201">201</a> <em class="jxr_javadoccomment">     * @param listGenes</em>
<a class="jxr_linenumber" name="L202" href="#L202">202</a> <em class="jxr_javadoccomment">     * @throws Exception</em>
<a class="jxr_linenumber" name="L203" href="#L203">203</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L204" href="#L204">204</a>     <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> addGeneIDGFF2GeneFeatures(LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList, <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> listGenes) <strong class="jxr_keyword">throws</strong> Exception {
<a class="jxr_linenumber" name="L205" href="#L205">205</a>         Collection&lt;String&gt; geneIds = listGenes.attributeValues(<span class="jxr_string">"gene_id"</span>);
<a class="jxr_linenumber" name="L206" href="#L206">206</a>         <strong class="jxr_keyword">for</strong> (String geneid : geneIds) {
<a class="jxr_linenumber" name="L207" href="#L207">207</a>             <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> gene = listGenes.selectByAttribute(<span class="jxr_string">"gene_id"</span>, geneid);
<a class="jxr_linenumber" name="L208" href="#L208">208</a>             <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> geneFeature = gene.get(0);
<a class="jxr_linenumber" name="L209" href="#L209">209</a>             ChromosomeSequence seq = chromosomeSequenceList.get(geneFeature.seqname());
<a class="jxr_linenumber" name="L210" href="#L210">210</a>             geneid = geneid.replaceAll(<span class="jxr_string">"_"</span>, <span class="jxr_string">".G"</span>);
<a class="jxr_linenumber" name="L211" href="#L211">211</a>             AccessionID geneAccessionID = <strong class="jxr_keyword">new</strong> AccessionID(geneid);
<a class="jxr_linenumber" name="L212" href="#L212">212</a>             GeneSequence geneSequence = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L213" href="#L213">213</a>             Collection&lt;String&gt; transcriptids = gene.attributeValues(<span class="jxr_string">"gene_id"</span>);
<a class="jxr_linenumber" name="L214" href="#L214">214</a>             <strong class="jxr_keyword">for</strong> (String transcriptid : transcriptids) {
<a class="jxr_linenumber" name="L215" href="#L215">215</a>                 <em class="jxr_comment">// get all the individual features (exons, CDS regions, etc.) of this gene</em>
<a class="jxr_linenumber" name="L216" href="#L216">216</a>                 <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> transcriptFeature = listGenes.selectByAttribute(<span class="jxr_string">"gene_id"</span>, transcriptid);
<a class="jxr_linenumber" name="L217" href="#L217">217</a>                 transcriptid = transcriptid.replaceAll(<span class="jxr_string">"_"</span>, <span class="jxr_string">".G"</span>);
<a class="jxr_linenumber" name="L218" href="#L218">218</a> 
<a class="jxr_linenumber" name="L219" href="#L219">219</a> 
<a class="jxr_linenumber" name="L220" href="#L220">220</a> 
<a class="jxr_linenumber" name="L221" href="#L221">221</a> 
<a class="jxr_linenumber" name="L222" href="#L222">222</a>                 <em class="jxr_comment">//      String seqName = feature.seqname();</em>
<a class="jxr_linenumber" name="L223" href="#L223">223</a>                 <em class="jxr_comment">//FeatureI startCodon = null;</em>
<a class="jxr_linenumber" name="L224" href="#L224">224</a>                 <em class="jxr_comment">//FeatureI stopCodon = null;</em>
<a class="jxr_linenumber" name="L225" href="#L225">225</a>                 Integer startCodonBegin = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L226" href="#L226">226</a>                 Integer stopCodonEnd = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L227" href="#L227">227</a>                 <em class="jxr_comment">//String startCodonName = "";</em>
<a class="jxr_linenumber" name="L228" href="#L228">228</a>                 <em class="jxr_comment">//String stopCodonName = "";</em>
<a class="jxr_linenumber" name="L229" href="#L229">229</a> 
<a class="jxr_linenumber" name="L230" href="#L230">230</a> 
<a class="jxr_linenumber" name="L231" href="#L231">231</a>                 <em class="jxr_comment">// now select only the coding regions of this gene</em>
<a class="jxr_linenumber" name="L232" href="#L232">232</a>                 <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> firstFeatures = transcriptFeature.selectByType(<span class="jxr_string">"First"</span>);
<a class="jxr_linenumber" name="L233" href="#L233">233</a>                 <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> terminalFeatures = transcriptFeature.selectByType(<span class="jxr_string">"Terminal"</span>);
<a class="jxr_linenumber" name="L234" href="#L234">234</a>                 <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> internalFeatures = transcriptFeature.selectByType(<span class="jxr_string">"Internal"</span>);
<a class="jxr_linenumber" name="L235" href="#L235">235</a>                 <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> singleFeatures = transcriptFeature.selectByType(<span class="jxr_string">"Single"</span>);
<a class="jxr_linenumber" name="L236" href="#L236">236</a>                 <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> cdsFeatures = <strong class="jxr_keyword">new</strong> <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a>();
<a class="jxr_linenumber" name="L237" href="#L237">237</a>                 cdsFeatures.add(firstFeatures);
<a class="jxr_linenumber" name="L238" href="#L238">238</a>                 cdsFeatures.add(terminalFeatures);
<a class="jxr_linenumber" name="L239" href="#L239">239</a>                 cdsFeatures.add(internalFeatures);
<a class="jxr_linenumber" name="L240" href="#L240">240</a>                 cdsFeatures.add(singleFeatures);
<a class="jxr_linenumber" name="L241" href="#L241">241</a>                 <em class="jxr_comment">// sort them</em>
<a class="jxr_linenumber" name="L242" href="#L242">242</a>                 cdsFeatures = cdsFeatures.sortByStart();
<a class="jxr_linenumber" name="L243" href="#L243">243</a>                 Strand strand = Strand.POSITIVE;
<a class="jxr_linenumber" name="L244" href="#L244">244</a>                 <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> feature = cdsFeatures.get(0);
<a class="jxr_linenumber" name="L245" href="#L245">245</a>                 <strong class="jxr_keyword">if</strong> (feature.location().isNegative()) {
<a class="jxr_linenumber" name="L246" href="#L246">246</a>                     strand = Strand.NEGATIVE;
<a class="jxr_linenumber" name="L247" href="#L247">247</a>                 }
<a class="jxr_linenumber" name="L248" href="#L248">248</a>                 <strong class="jxr_keyword">if</strong> (startCodonBegin == <strong class="jxr_keyword">null</strong>) {
<a class="jxr_linenumber" name="L249" href="#L249">249</a>                     <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> firstFeature = cdsFeatures.get(0);
<a class="jxr_linenumber" name="L250" href="#L250">250</a>                     <strong class="jxr_keyword">if</strong> (strand == Strand.NEGATIVE) {
<a class="jxr_linenumber" name="L251" href="#L251">251</a>                         startCodonBegin = firstFeature.location().bioEnd();
<a class="jxr_linenumber" name="L252" href="#L252">252</a>                     } <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L253" href="#L253">253</a>                         startCodonBegin = firstFeature.location().bioStart();
<a class="jxr_linenumber" name="L254" href="#L254">254</a>                     }
<a class="jxr_linenumber" name="L255" href="#L255">255</a>                 }
<a class="jxr_linenumber" name="L256" href="#L256">256</a> 
<a class="jxr_linenumber" name="L257" href="#L257">257</a>                 <strong class="jxr_keyword">if</strong> (stopCodonEnd == <strong class="jxr_keyword">null</strong>) {
<a class="jxr_linenumber" name="L258" href="#L258">258</a> 
<a class="jxr_linenumber" name="L259" href="#L259">259</a>                     <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> lastFeature = cdsFeatures.get(cdsFeatures.size() - 1);
<a class="jxr_linenumber" name="L260" href="#L260">260</a>                     <strong class="jxr_keyword">if</strong> (strand == Strand.NEGATIVE) {
<a class="jxr_linenumber" name="L261" href="#L261">261</a>                         stopCodonEnd = lastFeature.location().bioStart();
<a class="jxr_linenumber" name="L262" href="#L262">262</a>                     } <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L263" href="#L263">263</a>                         stopCodonEnd = lastFeature.location().bioEnd();
<a class="jxr_linenumber" name="L264" href="#L264">264</a>                     }
<a class="jxr_linenumber" name="L265" href="#L265">265</a>                 }
<a class="jxr_linenumber" name="L266" href="#L266">266</a>                 <em class="jxr_comment">//for gtf ordering can be strand based so first is last and last is first</em>
<a class="jxr_linenumber" name="L267" href="#L267">267</a>                 <strong class="jxr_keyword">if</strong> (startCodonBegin &gt; stopCodonEnd) {
<a class="jxr_linenumber" name="L268" href="#L268">268</a>                     <strong class="jxr_keyword">int</strong> temp = startCodonBegin;
<a class="jxr_linenumber" name="L269" href="#L269">269</a>                     startCodonBegin = stopCodonEnd;
<a class="jxr_linenumber" name="L270" href="#L270">270</a>                     stopCodonEnd = temp;
<a class="jxr_linenumber" name="L271" href="#L271">271</a>                 }
<a class="jxr_linenumber" name="L272" href="#L272">272</a> 
<a class="jxr_linenumber" name="L273" href="#L273">273</a>                 AccessionID transcriptAccessionID = <strong class="jxr_keyword">new</strong> AccessionID(transcriptid);
<a class="jxr_linenumber" name="L274" href="#L274">274</a>                 <strong class="jxr_keyword">if</strong> (geneSequence == <strong class="jxr_keyword">null</strong>) {
<a class="jxr_linenumber" name="L275" href="#L275">275</a>                     geneSequence = seq.addGene(geneAccessionID, startCodonBegin, stopCodonEnd, strand);
<a class="jxr_linenumber" name="L276" href="#L276">276</a>                     geneSequence.setSource(((Feature) feature).source());
<a class="jxr_linenumber" name="L277" href="#L277">277</a>                 } <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L278" href="#L278">278</a>                     <em class="jxr_comment">//if multiple transcripts for one gene make sure the gene is defined as the min and max start/end</em>
<a class="jxr_linenumber" name="L279" href="#L279">279</a> 
<a class="jxr_linenumber" name="L280" href="#L280">280</a>                     <strong class="jxr_keyword">if</strong> (startCodonBegin &lt; geneSequence.getBioBegin()) {
<a class="jxr_linenumber" name="L281" href="#L281">281</a>                         geneSequence.setBioBegin(startCodonBegin);
<a class="jxr_linenumber" name="L282" href="#L282">282</a>                     }
<a class="jxr_linenumber" name="L283" href="#L283">283</a>                     <strong class="jxr_keyword">if</strong> (stopCodonEnd &gt; geneSequence.getBioBegin()) {
<a class="jxr_linenumber" name="L284" href="#L284">284</a>                         geneSequence.setBioEnd(stopCodonEnd);
<a class="jxr_linenumber" name="L285" href="#L285">285</a>                     }
<a class="jxr_linenumber" name="L286" href="#L286">286</a> 
<a class="jxr_linenumber" name="L287" href="#L287">287</a>                 }
<a class="jxr_linenumber" name="L288" href="#L288">288</a>                 TranscriptSequence transcriptSequence = geneSequence.addTranscript(transcriptAccessionID, startCodonBegin, stopCodonEnd);
<a class="jxr_linenumber" name="L289" href="#L289">289</a>                 <em class="jxr_comment">/*</em>
<a class="jxr_linenumber" name="L290" href="#L290">290</a> <em class="jxr_comment">                if (startCodon != null) {</em>
<a class="jxr_linenumber" name="L291" href="#L291">291</a> <em class="jxr_comment">                    if (startCodonName == null || startCodonName.length() == 0) {</em>
<a class="jxr_linenumber" name="L292" href="#L292">292</a> <em class="jxr_comment">                        startCodonName = transcriptid + "-start_codon-" + startCodon.location().bioStart() + "-" + startCodon.location().bioEnd();</em>
<a class="jxr_linenumber" name="L293" href="#L293">293</a> <em class="jxr_comment">                    }</em>
<a class="jxr_linenumber" name="L294" href="#L294">294</a> <em class="jxr_comment">                    transcriptSequence.addStartCodonSequence(new AccessionID(startCodonName), startCodon.location().bioStart(), startCodon.location().bioEnd());</em>
<a class="jxr_linenumber" name="L295" href="#L295">295</a> <em class="jxr_comment">                }</em>
<a class="jxr_linenumber" name="L296" href="#L296">296</a> <em class="jxr_comment">                if (stopCodon != null) {</em>
<a class="jxr_linenumber" name="L297" href="#L297">297</a> <em class="jxr_comment">                    if (stopCodonName == null || stopCodonName.length() == 0) {</em>
<a class="jxr_linenumber" name="L298" href="#L298">298</a> <em class="jxr_comment">                        stopCodonName = transcriptid + "-stop_codon-" + stopCodon.location().bioStart() + "-" + stopCodon.location().bioEnd();</em>
<a class="jxr_linenumber" name="L299" href="#L299">299</a> <em class="jxr_comment">                    }</em>
<a class="jxr_linenumber" name="L300" href="#L300">300</a> <em class="jxr_comment">                    transcriptSequence.addStopCodonSequence(new AccessionID(stopCodonName), stopCodon.location().bioStart(), stopCodon.location().bioEnd());</em>
<a class="jxr_linenumber" name="L301" href="#L301">301</a> <em class="jxr_comment">                }</em>
<a class="jxr_linenumber" name="L302" href="#L302">302</a> <em class="jxr_comment">                */</em>
<a class="jxr_linenumber" name="L303" href="#L303">303</a> 
<a class="jxr_linenumber" name="L304" href="#L304">304</a>                 <strong class="jxr_keyword">for</strong> (FeatureI cdsFeature : cdsFeatures) {
<a class="jxr_linenumber" name="L305" href="#L305">305</a>                     <a href="../../../../org/biojava/nbio/genome/parsers/gff/Feature.html">Feature</a> cds = (Feature) cdsFeature;
<a class="jxr_linenumber" name="L306" href="#L306">306</a>                     String cdsName = cds.getAttribute(<span class="jxr_string">"transcript_name"</span>);
<a class="jxr_linenumber" name="L307" href="#L307">307</a>                     <strong class="jxr_keyword">if</strong> (cdsName == <strong class="jxr_keyword">null</strong> || cdsName.length() == 0) {
<a class="jxr_linenumber" name="L308" href="#L308">308</a>                         cdsName = transcriptid + <span class="jxr_string">"-cds-"</span> + cds.location().bioStart() + <span class="jxr_string">"-"</span> + cds.location().bioEnd();
<a class="jxr_linenumber" name="L309" href="#L309">309</a>                     }
<a class="jxr_linenumber" name="L310" href="#L310">310</a>                     AccessionID cdsAccessionID = <strong class="jxr_keyword">new</strong> AccessionID(cdsName);
<a class="jxr_linenumber" name="L311" href="#L311">311</a>                     <em class="jxr_comment">//ExonSequence exonSequence = geneSequence.addExon(cdsAccessionID, cdsFeature.location().bioStart(), cdsFeature.location().bioEnd());</em>
<a class="jxr_linenumber" name="L312" href="#L312">312</a>                     CDSSequence cdsSequence = transcriptSequence.addCDS(cdsAccessionID, cdsFeature.location().bioStart(), cdsFeature.location().bioEnd(), cds.frame());
<a class="jxr_linenumber" name="L313" href="#L313">313</a>                     cdsSequence.setSequenceScore(cds.score());
<a class="jxr_linenumber" name="L314" href="#L314">314</a>                 }
<a class="jxr_linenumber" name="L315" href="#L315">315</a>             }
<a class="jxr_linenumber" name="L316" href="#L316">316</a>         }
<a class="jxr_linenumber" name="L317" href="#L317">317</a> 
<a class="jxr_linenumber" name="L318" href="#L318">318</a>     }
<a class="jxr_linenumber" name="L319" href="#L319">319</a> 
<a class="jxr_linenumber" name="L320" href="#L320">320</a>     <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">public</strong> LinkedHashMap&lt;String, ChromosomeSequence&gt; getChromosomeSequenceFromDNASequence(LinkedHashMap&lt;String, DNASequence&gt; dnaSequenceList) {
<a class="jxr_linenumber" name="L321" href="#L321">321</a>         LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList = <strong class="jxr_keyword">new</strong> LinkedHashMap&lt;String, ChromosomeSequence&gt;();
<a class="jxr_linenumber" name="L322" href="#L322">322</a>         <strong class="jxr_keyword">for</strong> (String key : dnaSequenceList.keySet()) {
<a class="jxr_linenumber" name="L323" href="#L323">323</a>             DNASequence dnaSequence = dnaSequenceList.get(key);
<a class="jxr_linenumber" name="L324" href="#L324">324</a>             ChromosomeSequence chromosomeSequence = <strong class="jxr_keyword">new</strong> ChromosomeSequence(dnaSequence.getProxySequenceReader()); <em class="jxr_comment">//we want the underlying sequence but don't need storage</em>
<a class="jxr_linenumber" name="L325" href="#L325">325</a>             chromosomeSequence.setAccession(dnaSequence.getAccession());
<a class="jxr_linenumber" name="L326" href="#L326">326</a>             chromosomeSequenceList.put(key, chromosomeSequence);
<a class="jxr_linenumber" name="L327" href="#L327">327</a>         }
<a class="jxr_linenumber" name="L328" href="#L328">328</a>         <strong class="jxr_keyword">return</strong> chromosomeSequenceList;
<a class="jxr_linenumber" name="L329" href="#L329">329</a>     }
<a class="jxr_linenumber" name="L330" href="#L330">330</a> 
<a class="jxr_linenumber" name="L331" href="#L331">331</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L332" href="#L332">332</a> <em class="jxr_javadoccomment">     * Lots of variations in the ontology or descriptors that can be used in GFF3 which requires writing a custom parser to handle a GFF3 generated or used</em>
<a class="jxr_linenumber" name="L333" href="#L333">333</a> <em class="jxr_javadoccomment">     * by a specific application. Probably could be abstracted out but for now easier to handle with custom code to deal with gff3 elements that are not</em>
<a class="jxr_linenumber" name="L334" href="#L334">334</a> <em class="jxr_javadoccomment">     * included but can be extracted from other data elements.</em>
<a class="jxr_linenumber" name="L335" href="#L335">335</a> <em class="jxr_javadoccomment">     * @param fastaSequenceFile</em>
<a class="jxr_linenumber" name="L336" href="#L336">336</a> <em class="jxr_javadoccomment">     * @param gffFile</em>
<a class="jxr_linenumber" name="L337" href="#L337">337</a> <em class="jxr_javadoccomment">     * @param lazyloadsequences If set to true then the fasta file will be parsed for accession id but sequences will be read from disk when needed to save memory</em>
<a class="jxr_linenumber" name="L338" href="#L338">338</a> <em class="jxr_javadoccomment">     * @return</em>
<a class="jxr_linenumber" name="L339" href="#L339">339</a> <em class="jxr_javadoccomment">     * @throws Exception</em>
<a class="jxr_linenumber" name="L340" href="#L340">340</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L341" href="#L341">341</a>     <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">public</strong> LinkedHashMap&lt;String, ChromosomeSequence&gt; loadFastaAddGeneFeaturesFromGmodGFF3(File fastaSequenceFile, File gffFile,<strong class="jxr_keyword">boolean</strong> lazyloadsequences) <strong class="jxr_keyword">throws</strong> Exception {
<a class="jxr_linenumber" name="L342" href="#L342">342</a>         LinkedHashMap&lt;String, DNASequence&gt; dnaSequenceList = FastaReaderHelper.readFastaDNASequence(fastaSequenceFile,lazyloadsequences);
<a class="jxr_linenumber" name="L343" href="#L343">343</a>         LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList = GeneFeatureHelper.getChromosomeSequenceFromDNASequence(dnaSequenceList);
<a class="jxr_linenumber" name="L344" href="#L344">344</a>         <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> listGenes = GFF3Reader.read(gffFile.getAbsolutePath());
<a class="jxr_linenumber" name="L345" href="#L345">345</a>         addGmodGFF3GeneFeatures(chromosomeSequenceList, listGenes);
<a class="jxr_linenumber" name="L346" href="#L346">346</a>         <strong class="jxr_keyword">return</strong> chromosomeSequenceList;
<a class="jxr_linenumber" name="L347" href="#L347">347</a>     }
<a class="jxr_linenumber" name="L348" href="#L348">348</a> 
<a class="jxr_linenumber" name="L349" href="#L349">349</a>     <em class="jxr_javadoccomment">/**</em>
<a class="jxr_linenumber" name="L350" href="#L350">350</a> <em class="jxr_javadoccomment">     * Load GFF3 file using mRNA as the gene feature as not all GFF3 files are complete</em>
<a class="jxr_linenumber" name="L351" href="#L351">351</a> <em class="jxr_javadoccomment">     * @param chromosomeSequenceList</em>
<a class="jxr_linenumber" name="L352" href="#L352">352</a> <em class="jxr_javadoccomment">     * @param listGenes</em>
<a class="jxr_linenumber" name="L353" href="#L353">353</a> <em class="jxr_javadoccomment">     * @throws Exception</em>
<a class="jxr_linenumber" name="L354" href="#L354">354</a> <em class="jxr_javadoccomment">     */</em>
<a class="jxr_linenumber" name="L355" href="#L355">355</a>     <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> addGmodGFF3GeneFeatures(LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList, <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> listGenes) <strong class="jxr_keyword">throws</strong> Exception {
<a class="jxr_linenumber" name="L356" href="#L356">356</a> 
<a class="jxr_linenumber" name="L357" href="#L357">357</a> 
<a class="jxr_linenumber" name="L358" href="#L358">358</a>         <em class="jxr_comment">// key off mRNA as being a known feature that may or may not have a parent gene</em>
<a class="jxr_linenumber" name="L359" href="#L359">359</a> 
<a class="jxr_linenumber" name="L360" href="#L360">360</a> 
<a class="jxr_linenumber" name="L361" href="#L361">361</a>         <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> mRNAFeatures = listGenes.selectByType(<span class="jxr_string">"mRNA"</span>);
<a class="jxr_linenumber" name="L362" href="#L362">362</a>         LinkedHashMap&lt;String,FeatureList&gt; featureIDHashMap = FeatureHelper.buildFeatureAtrributeIndex(<span class="jxr_string">"ID"</span>, listGenes);
<a class="jxr_linenumber" name="L363" href="#L363">363</a>         LinkedHashMap&lt;String,FeatureList&gt; featureParentHashMap = FeatureHelper.buildFeatureAtrributeIndex(<span class="jxr_string">"Parent"</span>, listGenes);
<a class="jxr_linenumber" name="L364" href="#L364">364</a> 
<a class="jxr_linenumber" name="L365" href="#L365">365</a>         <strong class="jxr_keyword">for</strong> (FeatureI f : mRNAFeatures) {
<a class="jxr_linenumber" name="L366" href="#L366">366</a>             String geneID;
<a class="jxr_linenumber" name="L367" href="#L367">367</a>             String geneNote = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L368" href="#L368">368</a>             String geneSource = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L369" href="#L369">369</a>             String sequenceName = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L370" href="#L370">370</a>             ChromosomeSequence seq = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L371" href="#L371">371</a>             GeneSequence geneSequence = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L372" href="#L372">372</a> 
<a class="jxr_linenumber" name="L373" href="#L373">373</a>             <a href="../../../../org/biojava/nbio/genome/parsers/gff/Feature.html">Feature</a> mRNAFeature = (Feature) f;
<a class="jxr_linenumber" name="L374" href="#L374">374</a>             String mRNAID = mRNAFeature.getAttribute(<span class="jxr_string">"ID"</span>);
<a class="jxr_linenumber" name="L375" href="#L375">375</a>             String mRNAsource = mRNAFeature.source();
<a class="jxr_linenumber" name="L376" href="#L376">376</a>             String mRNANote = mRNAFeature.getAttribute(<span class="jxr_string">"Note"</span>);
<a class="jxr_linenumber" name="L377" href="#L377">377</a>             String mRNAParent = mRNAFeature.getAttribute(<span class="jxr_string">"Parent"</span>);
<a class="jxr_linenumber" name="L378" href="#L378">378</a>             <strong class="jxr_keyword">if</strong> (mRNAParent != <strong class="jxr_keyword">null</strong> &amp;&amp; mRNAParent.length() &gt; 0) {
<a class="jxr_linenumber" name="L379" href="#L379">379</a>                <em class="jxr_comment">// FeatureList geneFeatureList = listGenes.selectByAttribute("ID", mRNAParent);</em>
<a class="jxr_linenumber" name="L380" href="#L380">380</a>                 <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> geneFeatureList = featureIDHashMap.get(mRNAParent);
<a class="jxr_linenumber" name="L381" href="#L381">381</a>                 <a href="../../../../org/biojava/nbio/genome/parsers/gff/Feature.html">Feature</a> geneFeature = (Feature) geneFeatureList.get(0);
<a class="jxr_linenumber" name="L382" href="#L382">382</a>                 geneID = geneFeature.getAttribute(<span class="jxr_string">"ID"</span>);
<a class="jxr_linenumber" name="L383" href="#L383">383</a>                 geneNote = geneFeature.getAttribute(<span class="jxr_string">"Note"</span>);
<a class="jxr_linenumber" name="L384" href="#L384">384</a>                 geneSource = ((Feature) geneFeature).source();
<a class="jxr_linenumber" name="L385" href="#L385">385</a>                 sequenceName = geneFeature.seqname();
<a class="jxr_linenumber" name="L386" href="#L386">386</a> 
<a class="jxr_linenumber" name="L387" href="#L387">387</a>                 <em class="jxr_comment">//</em>
<a class="jxr_linenumber" name="L388" href="#L388">388</a>             } <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L389" href="#L389">389</a>                 <em class="jxr_comment">//deal with cases where no parent gene is given</em>
<a class="jxr_linenumber" name="L390" href="#L390">390</a>                 geneID = mRNAID;
<a class="jxr_linenumber" name="L391" href="#L391">391</a>                 geneSource = mRNAsource;
<a class="jxr_linenumber" name="L392" href="#L392">392</a>                 sequenceName = mRNAFeature.seqname();
<a class="jxr_linenumber" name="L393" href="#L393">393</a>             }
<a class="jxr_linenumber" name="L394" href="#L394">394</a> 
<a class="jxr_linenumber" name="L395" href="#L395">395</a>             seq = chromosomeSequenceList.get(sequenceName);
<a class="jxr_linenumber" name="L396" href="#L396">396</a> 
<a class="jxr_linenumber" name="L397" href="#L397">397</a>             AccessionID geneAccessionID = <strong class="jxr_keyword">new</strong> AccessionID(geneID);
<a class="jxr_linenumber" name="L398" href="#L398">398</a> 
<a class="jxr_linenumber" name="L399" href="#L399">399</a>           <em class="jxr_comment">//  FeatureList mRNAChildren = listGenes.selectByAttribute("Parent", mRNAID);</em>
<a class="jxr_linenumber" name="L400" href="#L400">400</a>             <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> mRNAChildren = featureParentHashMap.get(mRNAID);
<a class="jxr_linenumber" name="L401" href="#L401">401</a>             <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> cdsFeatures = mRNAChildren.selectByType(<span class="jxr_string">"CDS"</span>);
<a class="jxr_linenumber" name="L402" href="#L402">402</a>             <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> feature = cdsFeatures.get(0);
<a class="jxr_linenumber" name="L403" href="#L403">403</a>             Strand strand = Strand.POSITIVE;
<a class="jxr_linenumber" name="L404" href="#L404">404</a> 
<a class="jxr_linenumber" name="L405" href="#L405">405</a>             <strong class="jxr_keyword">if</strong> (feature.location().isNegative()) {
<a class="jxr_linenumber" name="L406" href="#L406">406</a>                 strand = Strand.NEGATIVE;
<a class="jxr_linenumber" name="L407" href="#L407">407</a>             }
<a class="jxr_linenumber" name="L408" href="#L408">408</a>             cdsFeatures = cdsFeatures.sortByStart();
<a class="jxr_linenumber" name="L409" href="#L409">409</a> 
<a class="jxr_linenumber" name="L410" href="#L410">410</a> 
<a class="jxr_linenumber" name="L411" href="#L411">411</a> 
<a class="jxr_linenumber" name="L412" href="#L412">412</a> 
<a class="jxr_linenumber" name="L413" href="#L413">413</a> 
<a class="jxr_linenumber" name="L414" href="#L414">414</a> 
<a class="jxr_linenumber" name="L415" href="#L415">415</a> 
<a class="jxr_linenumber" name="L416" href="#L416">416</a>             <em class="jxr_comment">//String seqName = feature.seqname();</em>
<a class="jxr_linenumber" name="L417" href="#L417">417</a>             <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> startCodon = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L418" href="#L418">418</a>             <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> stopCodon = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L419" href="#L419">419</a>             Integer startCodonBegin = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L420" href="#L420">420</a>             Integer stopCodonEnd = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L421" href="#L421">421</a>             String startCodonName = <span class="jxr_string">""</span>;
<a class="jxr_linenumber" name="L422" href="#L422">422</a>             String stopCodonName = <span class="jxr_string">""</span>;
<a class="jxr_linenumber" name="L423" href="#L423">423</a>             <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> startCodonList = mRNAChildren.selectByType(<span class="jxr_string">"five_prime_UTR"</span>);
<a class="jxr_linenumber" name="L424" href="#L424">424</a>             <strong class="jxr_keyword">if</strong> (startCodonList != <strong class="jxr_keyword">null</strong> &amp;&amp; startCodonList.size() &gt; 0) {
<a class="jxr_linenumber" name="L425" href="#L425">425</a>                 startCodon = startCodonList.get(0);
<a class="jxr_linenumber" name="L426" href="#L426">426</a>                 <strong class="jxr_keyword">if</strong> (strand == Strand.NEGATIVE) {
<a class="jxr_linenumber" name="L427" href="#L427">427</a>                     startCodonBegin = startCodon.location().bioEnd();
<a class="jxr_linenumber" name="L428" href="#L428">428</a>                 } <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L429" href="#L429">429</a>                     startCodonBegin = startCodon.location().bioStart();
<a class="jxr_linenumber" name="L430" href="#L430">430</a>                 }
<a class="jxr_linenumber" name="L431" href="#L431">431</a>                 startCodonName = startCodon.getAttribute(<span class="jxr_string">"ID"</span>);
<a class="jxr_linenumber" name="L432" href="#L432">432</a>             }
<a class="jxr_linenumber" name="L433" href="#L433">433</a> 
<a class="jxr_linenumber" name="L434" href="#L434">434</a>             <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> stopCodonList = mRNAChildren.selectByType(<span class="jxr_string">"three_prime_UTR"</span>);
<a class="jxr_linenumber" name="L435" href="#L435">435</a> 
<a class="jxr_linenumber" name="L436" href="#L436">436</a>             <strong class="jxr_keyword">if</strong> (stopCodonList != <strong class="jxr_keyword">null</strong> &amp;&amp; stopCodonList.size() &gt; 0) {
<a class="jxr_linenumber" name="L437" href="#L437">437</a>                 stopCodon = stopCodonList.get(0);
<a class="jxr_linenumber" name="L438" href="#L438">438</a>                 <strong class="jxr_keyword">if</strong> (strand == Strand.NEGATIVE) {
<a class="jxr_linenumber" name="L439" href="#L439">439</a>                     stopCodonEnd = stopCodon.location().bioStart();
<a class="jxr_linenumber" name="L440" href="#L440">440</a>                 } <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L441" href="#L441">441</a>                     stopCodonEnd = stopCodon.location().bioEnd();
<a class="jxr_linenumber" name="L442" href="#L442">442</a>                 }
<a class="jxr_linenumber" name="L443" href="#L443">443</a>                 stopCodonName = stopCodon.getAttribute(<span class="jxr_string">"ID"</span>);
<a class="jxr_linenumber" name="L444" href="#L444">444</a> 
<a class="jxr_linenumber" name="L445" href="#L445">445</a>             }
<a class="jxr_linenumber" name="L446" href="#L446">446</a> 
<a class="jxr_linenumber" name="L447" href="#L447">447</a> 
<a class="jxr_linenumber" name="L448" href="#L448">448</a> 
<a class="jxr_linenumber" name="L449" href="#L449">449</a> 
<a class="jxr_linenumber" name="L450" href="#L450">450</a>             <strong class="jxr_keyword">if</strong> (startCodonBegin == <strong class="jxr_keyword">null</strong>) {
<a class="jxr_linenumber" name="L451" href="#L451">451</a>                 <strong class="jxr_keyword">if</strong> (strand == Strand.NEGATIVE) {
<a class="jxr_linenumber" name="L452" href="#L452">452</a>                     <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> firstFeature = cdsFeatures.get(0);
<a class="jxr_linenumber" name="L453" href="#L453">453</a> 
<a class="jxr_linenumber" name="L454" href="#L454">454</a>                     startCodonBegin = firstFeature.location().bioEnd();
<a class="jxr_linenumber" name="L455" href="#L455">455</a>                 } <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L456" href="#L456">456</a>                     <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> firstFeature = cdsFeatures.get(0);
<a class="jxr_linenumber" name="L457" href="#L457">457</a> 
<a class="jxr_linenumber" name="L458" href="#L458">458</a>                     startCodonBegin = firstFeature.location().bioStart();
<a class="jxr_linenumber" name="L459" href="#L459">459</a>                 }
<a class="jxr_linenumber" name="L460" href="#L460">460</a>             }
<a class="jxr_linenumber" name="L461" href="#L461">461</a> 
<a class="jxr_linenumber" name="L462" href="#L462">462</a>             <strong class="jxr_keyword">if</strong> (stopCodonEnd == <strong class="jxr_keyword">null</strong>) {
<a class="jxr_linenumber" name="L463" href="#L463">463</a>                 <strong class="jxr_keyword">if</strong> (strand == Strand.NEGATIVE) {
<a class="jxr_linenumber" name="L464" href="#L464">464</a>                     <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> lastFeature = cdsFeatures.get(cdsFeatures.size() - 1);
<a class="jxr_linenumber" name="L465" href="#L465">465</a>                     stopCodonEnd = lastFeature.location().bioStart();
<a class="jxr_linenumber" name="L466" href="#L466">466</a>                 } <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L467" href="#L467">467</a>                     <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> lastFeature = cdsFeatures.get(cdsFeatures.size() - 1);
<a class="jxr_linenumber" name="L468" href="#L468">468</a>                     stopCodonEnd = lastFeature.location().bioEnd();
<a class="jxr_linenumber" name="L469" href="#L469">469</a>                 }
<a class="jxr_linenumber" name="L470" href="#L470">470</a>             }
<a class="jxr_linenumber" name="L471" href="#L471">471</a>             <em class="jxr_comment">//for gtf ordering can be strand based so first is last and last is first</em>
<a class="jxr_linenumber" name="L472" href="#L472">472</a>             <strong class="jxr_keyword">if</strong> (startCodonBegin &gt; stopCodonEnd) {
<a class="jxr_linenumber" name="L473" href="#L473">473</a>                 <strong class="jxr_keyword">int</strong> temp = startCodonBegin;
<a class="jxr_linenumber" name="L474" href="#L474">474</a>                 startCodonBegin = stopCodonEnd;
<a class="jxr_linenumber" name="L475" href="#L475">475</a>                 stopCodonEnd = temp;
<a class="jxr_linenumber" name="L476" href="#L476">476</a>             }
<a class="jxr_linenumber" name="L477" href="#L477">477</a> 
<a class="jxr_linenumber" name="L478" href="#L478">478</a> 
<a class="jxr_linenumber" name="L479" href="#L479">479</a> 
<a class="jxr_linenumber" name="L480" href="#L480">480</a>             AccessionID transcriptAccessionID = <strong class="jxr_keyword">new</strong> AccessionID(mRNAID);
<a class="jxr_linenumber" name="L481" href="#L481">481</a>             geneSequence = seq.getGene(geneID);
<a class="jxr_linenumber" name="L482" href="#L482">482</a>             <strong class="jxr_keyword">if</strong> (geneSequence == <strong class="jxr_keyword">null</strong>) {
<a class="jxr_linenumber" name="L483" href="#L483">483</a>                 geneSequence = seq.addGene(geneAccessionID, startCodonBegin, stopCodonEnd, strand);
<a class="jxr_linenumber" name="L484" href="#L484">484</a>                 geneSequence.setSource(geneSource);
<a class="jxr_linenumber" name="L485" href="#L485">485</a>                 <strong class="jxr_keyword">if</strong> (geneNote != <strong class="jxr_keyword">null</strong> &amp;&amp; geneNote.length() &gt; 0) {
<a class="jxr_linenumber" name="L486" href="#L486">486</a>                     geneSequence.addNote(geneNote);
<a class="jxr_linenumber" name="L487" href="#L487">487</a>                 }
<a class="jxr_linenumber" name="L488" href="#L488">488</a>             } <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L489" href="#L489">489</a> 
<a class="jxr_linenumber" name="L490" href="#L490">490</a>                 <strong class="jxr_keyword">if</strong> (startCodonBegin &lt; geneSequence.getBioBegin()) {
<a class="jxr_linenumber" name="L491" href="#L491">491</a>                     geneSequence.setBioBegin(startCodonBegin);
<a class="jxr_linenumber" name="L492" href="#L492">492</a>                 }
<a class="jxr_linenumber" name="L493" href="#L493">493</a>                 <strong class="jxr_keyword">if</strong> (stopCodonEnd &gt; geneSequence.getBioBegin()) {
<a class="jxr_linenumber" name="L494" href="#L494">494</a>                     geneSequence.setBioEnd(stopCodonEnd);
<a class="jxr_linenumber" name="L495" href="#L495">495</a>                 }
<a class="jxr_linenumber" name="L496" href="#L496">496</a> 
<a class="jxr_linenumber" name="L497" href="#L497">497</a>             }
<a class="jxr_linenumber" name="L498" href="#L498">498</a>             TranscriptSequence transcriptSequence = geneSequence.addTranscript(transcriptAccessionID, startCodonBegin, stopCodonEnd);
<a class="jxr_linenumber" name="L499" href="#L499">499</a>             transcriptSequence.setSource(mRNAsource);
<a class="jxr_linenumber" name="L500" href="#L500">500</a>             <strong class="jxr_keyword">if</strong> (mRNANote != <strong class="jxr_keyword">null</strong> &amp;&amp; mRNANote.length() &gt; 0) {
<a class="jxr_linenumber" name="L501" href="#L501">501</a>                 transcriptSequence.addNote(mRNANote);
<a class="jxr_linenumber" name="L502" href="#L502">502</a> 
<a class="jxr_linenumber" name="L503" href="#L503">503</a>             }
<a class="jxr_linenumber" name="L504" href="#L504">504</a>             <strong class="jxr_keyword">if</strong> (startCodon != <strong class="jxr_keyword">null</strong>) {
<a class="jxr_linenumber" name="L505" href="#L505">505</a>                 <strong class="jxr_keyword">if</strong> (startCodonName == <strong class="jxr_keyword">null</strong> || startCodonName.length() == 0) {
<a class="jxr_linenumber" name="L506" href="#L506">506</a>                     startCodonName = mRNAID + <span class="jxr_string">"-start_codon-"</span> + startCodon.location().bioStart() + <span class="jxr_string">"-"</span> + startCodon.location().bioEnd();
<a class="jxr_linenumber" name="L507" href="#L507">507</a>                 }
<a class="jxr_linenumber" name="L508" href="#L508">508</a>                 transcriptSequence.addStartCodonSequence(<strong class="jxr_keyword">new</strong> AccessionID(startCodonName), startCodon.location().bioStart(), startCodon.location().bioEnd());
<a class="jxr_linenumber" name="L509" href="#L509">509</a>             }
<a class="jxr_linenumber" name="L510" href="#L510">510</a>             <strong class="jxr_keyword">if</strong> (stopCodon != <strong class="jxr_keyword">null</strong>) {
<a class="jxr_linenumber" name="L511" href="#L511">511</a>                 <strong class="jxr_keyword">if</strong> (stopCodonName == <strong class="jxr_keyword">null</strong> || stopCodonName.length() == 0) {
<a class="jxr_linenumber" name="L512" href="#L512">512</a>                     stopCodonName = mRNAID + <span class="jxr_string">"-stop_codon-"</span> + stopCodon.location().bioStart() + <span class="jxr_string">"-"</span> + stopCodon.location().bioEnd();
<a class="jxr_linenumber" name="L513" href="#L513">513</a>                 }
<a class="jxr_linenumber" name="L514" href="#L514">514</a>                 transcriptSequence.addStopCodonSequence(<strong class="jxr_keyword">new</strong> AccessionID(stopCodonName), stopCodon.location().bioStart(), stopCodon.location().bioEnd());
<a class="jxr_linenumber" name="L515" href="#L515">515</a>             }
<a class="jxr_linenumber" name="L516" href="#L516">516</a> 
<a class="jxr_linenumber" name="L517" href="#L517">517</a>             <strong class="jxr_keyword">for</strong> (FeatureI cdsFeature : cdsFeatures) {
<a class="jxr_linenumber" name="L518" href="#L518">518</a>                 <a href="../../../../org/biojava/nbio/genome/parsers/gff/Feature.html">Feature</a> cds = (Feature) cdsFeature;
<a class="jxr_linenumber" name="L519" href="#L519">519</a>                 String cdsNote = cdsFeature.getAttribute(<span class="jxr_string">"Note"</span>);
<a class="jxr_linenumber" name="L520" href="#L520">520</a>                 String cdsSource = cds.source();
<a class="jxr_linenumber" name="L521" href="#L521">521</a>                 String cdsName = cds.getAttribute(<span class="jxr_string">"ID"</span>);
<a class="jxr_linenumber" name="L522" href="#L522">522</a>                 <strong class="jxr_keyword">if</strong> (cdsName == <strong class="jxr_keyword">null</strong> || cdsName.length() == 0) {
<a class="jxr_linenumber" name="L523" href="#L523">523</a>                     cdsName = mRNAID + <span class="jxr_string">"-cds-"</span> + cds.location().bioStart() + <span class="jxr_string">"-"</span> + cds.location().bioEnd();
<a class="jxr_linenumber" name="L524" href="#L524">524</a>                 }
<a class="jxr_linenumber" name="L525" href="#L525">525</a>                 AccessionID cdsAccessionID = <strong class="jxr_keyword">new</strong> AccessionID(cdsName);
<a class="jxr_linenumber" name="L526" href="#L526">526</a>                 ExonSequence exonSequence = geneSequence.addExon(cdsAccessionID, cdsFeature.location().bioStart(), cdsFeature.location().bioEnd());
<a class="jxr_linenumber" name="L527" href="#L527">527</a>                 exonSequence.setSource(cdsSource);
<a class="jxr_linenumber" name="L528" href="#L528">528</a>                 <strong class="jxr_keyword">if</strong> (cdsNote != <strong class="jxr_keyword">null</strong> &amp;&amp; cdsNote.length() &gt; 0) {
<a class="jxr_linenumber" name="L529" href="#L529">529</a>                     exonSequence.addNote(cdsNote);
<a class="jxr_linenumber" name="L530" href="#L530">530</a>                 }
<a class="jxr_linenumber" name="L531" href="#L531">531</a>                 transcriptSequence.addCDS(cdsAccessionID, cdsFeature.location().bioStart(), cdsFeature.location().bioEnd(), cds.frame());
<a class="jxr_linenumber" name="L532" href="#L532">532</a>             }
<a class="jxr_linenumber" name="L533" href="#L533">533</a>             geneSequence.addIntronsUsingExons();
<a class="jxr_linenumber" name="L534" href="#L534">534</a> 
<a class="jxr_linenumber" name="L535" href="#L535">535</a>         }
<a class="jxr_linenumber" name="L536" href="#L536">536</a> 
<a class="jxr_linenumber" name="L537" href="#L537">537</a>     }
<a class="jxr_linenumber" name="L538" href="#L538">538</a> 
<a class="jxr_linenumber" name="L539" href="#L539">539</a>     <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">public</strong> LinkedHashMap&lt;String, ChromosomeSequence&gt; loadFastaAddGeneFeaturesFromGlimmerGFF3(File fastaSequenceFile, File gffFile) <strong class="jxr_keyword">throws</strong> Exception {
<a class="jxr_linenumber" name="L540" href="#L540">540</a>         LinkedHashMap&lt;String, DNASequence&gt; dnaSequenceList = FastaReaderHelper.readFastaDNASequence(fastaSequenceFile);
<a class="jxr_linenumber" name="L541" href="#L541">541</a>         LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList = GeneFeatureHelper.getChromosomeSequenceFromDNASequence(dnaSequenceList);
<a class="jxr_linenumber" name="L542" href="#L542">542</a>         <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> listGenes = GFF3Reader.read(gffFile.getAbsolutePath());
<a class="jxr_linenumber" name="L543" href="#L543">543</a>         addGlimmerGFF3GeneFeatures(chromosomeSequenceList, listGenes);
<a class="jxr_linenumber" name="L544" href="#L544">544</a>         <strong class="jxr_keyword">return</strong> chromosomeSequenceList;
<a class="jxr_linenumber" name="L545" href="#L545">545</a>     }
<a class="jxr_linenumber" name="L546" href="#L546">546</a> 
<a class="jxr_linenumber" name="L547" href="#L547">547</a>     <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> addGlimmerGFF3GeneFeatures(LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList, <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> listGenes) <strong class="jxr_keyword">throws</strong> Exception {
<a class="jxr_linenumber" name="L548" href="#L548">548</a>         <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> mRNAFeatures = listGenes.selectByType(<span class="jxr_string">"mRNA"</span>);
<a class="jxr_linenumber" name="L549" href="#L549">549</a>         <strong class="jxr_keyword">for</strong> (FeatureI f : mRNAFeatures) {
<a class="jxr_linenumber" name="L550" href="#L550">550</a>             <a href="../../../../org/biojava/nbio/genome/parsers/gff/Feature.html">Feature</a> mRNAFeature = (Feature) f;
<a class="jxr_linenumber" name="L551" href="#L551">551</a>             String geneid = mRNAFeature.getAttribute(<span class="jxr_string">"ID"</span>);
<a class="jxr_linenumber" name="L552" href="#L552">552</a>             String source = mRNAFeature.source();
<a class="jxr_linenumber" name="L553" href="#L553">553</a> 
<a class="jxr_linenumber" name="L554" href="#L554">554</a>             <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> gene = listGenes.selectByAttribute(<span class="jxr_string">"Parent"</span>, geneid);
<a class="jxr_linenumber" name="L555" href="#L555">555</a>             <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> geneFeature = gene.get(0);
<a class="jxr_linenumber" name="L556" href="#L556">556</a>             ChromosomeSequence seq = chromosomeSequenceList.get(geneFeature.seqname());
<a class="jxr_linenumber" name="L557" href="#L557">557</a>             AccessionID geneAccessionID = <strong class="jxr_keyword">new</strong> AccessionID(geneid);
<a class="jxr_linenumber" name="L558" href="#L558">558</a>             GeneSequence geneSequence = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L559" href="#L559">559</a> 
<a class="jxr_linenumber" name="L560" href="#L560">560</a>             <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> cdsFeatures = gene.selectByType(<span class="jxr_string">"CDS"</span>);
<a class="jxr_linenumber" name="L561" href="#L561">561</a>             <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> feature = cdsFeatures.get(0);
<a class="jxr_linenumber" name="L562" href="#L562">562</a>             Strand strand = Strand.POSITIVE;
<a class="jxr_linenumber" name="L563" href="#L563">563</a> 
<a class="jxr_linenumber" name="L564" href="#L564">564</a>             <strong class="jxr_keyword">if</strong> (feature.location().isNegative()) {
<a class="jxr_linenumber" name="L565" href="#L565">565</a>                 strand = Strand.NEGATIVE;
<a class="jxr_linenumber" name="L566" href="#L566">566</a>             }
<a class="jxr_linenumber" name="L567" href="#L567">567</a>             cdsFeatures = cdsFeatures.sortByStart();
<a class="jxr_linenumber" name="L568" href="#L568">568</a> 
<a class="jxr_linenumber" name="L569" href="#L569">569</a> 
<a class="jxr_linenumber" name="L570" href="#L570">570</a> 
<a class="jxr_linenumber" name="L571" href="#L571">571</a> 
<a class="jxr_linenumber" name="L572" href="#L572">572</a> 
<a class="jxr_linenumber" name="L573" href="#L573">573</a> 
<a class="jxr_linenumber" name="L574" href="#L574">574</a> 
<a class="jxr_linenumber" name="L575" href="#L575">575</a>             <em class="jxr_comment">//String seqName = feature.seqname();</em>
<a class="jxr_linenumber" name="L576" href="#L576">576</a>             <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> startCodon = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L577" href="#L577">577</a>             <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> stopCodon = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L578" href="#L578">578</a>             Integer startCodonBegin = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L579" href="#L579">579</a>             Integer stopCodonEnd = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L580" href="#L580">580</a>             String startCodonName = <span class="jxr_string">""</span>;
<a class="jxr_linenumber" name="L581" href="#L581">581</a>             String stopCodonName = <span class="jxr_string">""</span>;
<a class="jxr_linenumber" name="L582" href="#L582">582</a>             <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> startCodonList = gene.selectByAttribute(<span class="jxr_string">"Note"</span>, <span class="jxr_string">"initial-exon"</span>);
<a class="jxr_linenumber" name="L583" href="#L583">583</a>             <strong class="jxr_keyword">if</strong> (startCodonList != <strong class="jxr_keyword">null</strong> &amp;&amp; startCodonList.size() &gt; 0) {
<a class="jxr_linenumber" name="L584" href="#L584">584</a>                 startCodon = startCodonList.get(0);
<a class="jxr_linenumber" name="L585" href="#L585">585</a>                 <strong class="jxr_keyword">if</strong> (strand == Strand.NEGATIVE) {
<a class="jxr_linenumber" name="L586" href="#L586">586</a>                     startCodonBegin = startCodon.location().bioEnd();
<a class="jxr_linenumber" name="L587" href="#L587">587</a>                 } <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L588" href="#L588">588</a>                     startCodonBegin = startCodon.location().bioStart();
<a class="jxr_linenumber" name="L589" href="#L589">589</a>                 }
<a class="jxr_linenumber" name="L590" href="#L590">590</a>                 startCodonName = startCodon.getAttribute(<span class="jxr_string">"ID"</span>);
<a class="jxr_linenumber" name="L591" href="#L591">591</a>             }
<a class="jxr_linenumber" name="L592" href="#L592">592</a> 
<a class="jxr_linenumber" name="L593" href="#L593">593</a>             <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> stopCodonList = gene.selectByAttribute(<span class="jxr_string">"Note"</span>, <span class="jxr_string">"final-exon"</span>);
<a class="jxr_linenumber" name="L594" href="#L594">594</a> 
<a class="jxr_linenumber" name="L595" href="#L595">595</a>             <strong class="jxr_keyword">if</strong> (stopCodonList != <strong class="jxr_keyword">null</strong> &amp;&amp; stopCodonList.size() &gt; 0) {
<a class="jxr_linenumber" name="L596" href="#L596">596</a>                 stopCodon = stopCodonList.get(0);
<a class="jxr_linenumber" name="L597" href="#L597">597</a>                 <strong class="jxr_keyword">if</strong> (strand == Strand.NEGATIVE) {
<a class="jxr_linenumber" name="L598" href="#L598">598</a>                     stopCodonEnd = stopCodon.location().bioStart();
<a class="jxr_linenumber" name="L599" href="#L599">599</a>                 } <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L600" href="#L600">600</a>                     stopCodonEnd = stopCodon.location().bioEnd();
<a class="jxr_linenumber" name="L601" href="#L601">601</a>                 }
<a class="jxr_linenumber" name="L602" href="#L602">602</a>                 stopCodonName = stopCodon.getAttribute(<span class="jxr_string">"ID"</span>);
<a class="jxr_linenumber" name="L603" href="#L603">603</a> 
<a class="jxr_linenumber" name="L604" href="#L604">604</a>             }
<a class="jxr_linenumber" name="L605" href="#L605">605</a> 
<a class="jxr_linenumber" name="L606" href="#L606">606</a> 
<a class="jxr_linenumber" name="L607" href="#L607">607</a> 
<a class="jxr_linenumber" name="L608" href="#L608">608</a> 
<a class="jxr_linenumber" name="L609" href="#L609">609</a>             <strong class="jxr_keyword">if</strong> (startCodonBegin == <strong class="jxr_keyword">null</strong>) {
<a class="jxr_linenumber" name="L610" href="#L610">610</a>                 <strong class="jxr_keyword">if</strong> (strand == Strand.NEGATIVE) {
<a class="jxr_linenumber" name="L611" href="#L611">611</a>                     <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> firstFeature = cdsFeatures.get(0);
<a class="jxr_linenumber" name="L612" href="#L612">612</a> 
<a class="jxr_linenumber" name="L613" href="#L613">613</a>                     startCodonBegin = firstFeature.location().bioEnd();
<a class="jxr_linenumber" name="L614" href="#L614">614</a>                 } <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L615" href="#L615">615</a>                     <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> firstFeature = cdsFeatures.get(0);
<a class="jxr_linenumber" name="L616" href="#L616">616</a> 
<a class="jxr_linenumber" name="L617" href="#L617">617</a>                     startCodonBegin = firstFeature.location().bioStart();
<a class="jxr_linenumber" name="L618" href="#L618">618</a>                 }
<a class="jxr_linenumber" name="L619" href="#L619">619</a>             }
<a class="jxr_linenumber" name="L620" href="#L620">620</a> 
<a class="jxr_linenumber" name="L621" href="#L621">621</a>             <strong class="jxr_keyword">if</strong> (stopCodonEnd == <strong class="jxr_keyword">null</strong>) {
<a class="jxr_linenumber" name="L622" href="#L622">622</a>                 <strong class="jxr_keyword">if</strong> (strand == Strand.NEGATIVE) {
<a class="jxr_linenumber" name="L623" href="#L623">623</a>                     <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> lastFeature = cdsFeatures.get(cdsFeatures.size() - 1);
<a class="jxr_linenumber" name="L624" href="#L624">624</a>                     stopCodonEnd = lastFeature.location().bioStart();
<a class="jxr_linenumber" name="L625" href="#L625">625</a>                 } <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L626" href="#L626">626</a>                     <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> lastFeature = cdsFeatures.get(cdsFeatures.size() - 1);
<a class="jxr_linenumber" name="L627" href="#L627">627</a>                     stopCodonEnd = lastFeature.location().bioEnd();
<a class="jxr_linenumber" name="L628" href="#L628">628</a>                 }
<a class="jxr_linenumber" name="L629" href="#L629">629</a>             }
<a class="jxr_linenumber" name="L630" href="#L630">630</a>             <em class="jxr_comment">//for gtf ordering can be strand based so first is last and last is first</em>
<a class="jxr_linenumber" name="L631" href="#L631">631</a>             <strong class="jxr_keyword">if</strong> (startCodonBegin &gt; stopCodonEnd) {
<a class="jxr_linenumber" name="L632" href="#L632">632</a>                 <strong class="jxr_keyword">int</strong> temp = startCodonBegin;
<a class="jxr_linenumber" name="L633" href="#L633">633</a>                 startCodonBegin = stopCodonEnd;
<a class="jxr_linenumber" name="L634" href="#L634">634</a>                 stopCodonEnd = temp;
<a class="jxr_linenumber" name="L635" href="#L635">635</a>             }
<a class="jxr_linenumber" name="L636" href="#L636">636</a> 
<a class="jxr_linenumber" name="L637" href="#L637">637</a> 
<a class="jxr_linenumber" name="L638" href="#L638">638</a> 
<a class="jxr_linenumber" name="L639" href="#L639">639</a>             AccessionID transcriptAccessionID = <strong class="jxr_keyword">new</strong> AccessionID(geneid);
<a class="jxr_linenumber" name="L640" href="#L640">640</a>             <strong class="jxr_keyword">if</strong> (geneSequence == <strong class="jxr_keyword">null</strong>) {
<a class="jxr_linenumber" name="L641" href="#L641">641</a>                 geneSequence = seq.addGene(geneAccessionID, startCodonBegin, stopCodonEnd, strand);
<a class="jxr_linenumber" name="L642" href="#L642">642</a>                 geneSequence.setSource(source);
<a class="jxr_linenumber" name="L643" href="#L643">643</a>             } 
<a class="jxr_linenumber" name="L644" href="#L644">644</a>             <em class="jxr_comment">/*</em>
<a class="jxr_linenumber" name="L645" href="#L645">645</a> <em class="jxr_comment">            else {</em>
<a class="jxr_linenumber" name="L646" href="#L646">646</a> 
<a class="jxr_linenumber" name="L647" href="#L647">647</a> <em class="jxr_comment">                if (startCodonBegin &lt; geneSequence.getBioBegin()) {</em>
<a class="jxr_linenumber" name="L648" href="#L648">648</a> <em class="jxr_comment">                    geneSequence.setBioBegin(startCodonBegin);</em>
<a class="jxr_linenumber" name="L649" href="#L649">649</a> <em class="jxr_comment">                }</em>
<a class="jxr_linenumber" name="L650" href="#L650">650</a> <em class="jxr_comment">                if (stopCodonEnd &gt; geneSequence.getBioBegin()) {</em>
<a class="jxr_linenumber" name="L651" href="#L651">651</a> <em class="jxr_comment">                    geneSequence.setBioEnd(stopCodonEnd);</em>
<a class="jxr_linenumber" name="L652" href="#L652">652</a> <em class="jxr_comment">                }</em>
<a class="jxr_linenumber" name="L653" href="#L653">653</a> <em class="jxr_comment">            }</em>
<a class="jxr_linenumber" name="L654" href="#L654">654</a> <em class="jxr_comment">            */</em>
<a class="jxr_linenumber" name="L655" href="#L655">655</a>             TranscriptSequence transcriptSequence = geneSequence.addTranscript(transcriptAccessionID, startCodonBegin, stopCodonEnd);
<a class="jxr_linenumber" name="L656" href="#L656">656</a>             <strong class="jxr_keyword">if</strong> (startCodon != <strong class="jxr_keyword">null</strong>) {
<a class="jxr_linenumber" name="L657" href="#L657">657</a>                 <strong class="jxr_keyword">if</strong> (startCodonName == <strong class="jxr_keyword">null</strong> || startCodonName.length() == 0) {
<a class="jxr_linenumber" name="L658" href="#L658">658</a>                     startCodonName = geneid + <span class="jxr_string">"-start_codon-"</span> + startCodon.location().bioStart() + <span class="jxr_string">"-"</span> + startCodon.location().bioEnd();
<a class="jxr_linenumber" name="L659" href="#L659">659</a>                 }
<a class="jxr_linenumber" name="L660" href="#L660">660</a>                 transcriptSequence.addStartCodonSequence(<strong class="jxr_keyword">new</strong> AccessionID(startCodonName), startCodon.location().bioStart(), startCodon.location().bioEnd());
<a class="jxr_linenumber" name="L661" href="#L661">661</a>             }
<a class="jxr_linenumber" name="L662" href="#L662">662</a>             <strong class="jxr_keyword">if</strong> (stopCodon != <strong class="jxr_keyword">null</strong>) {
<a class="jxr_linenumber" name="L663" href="#L663">663</a>                 <strong class="jxr_keyword">if</strong> (stopCodonName == <strong class="jxr_keyword">null</strong> || stopCodonName.length() == 0) {
<a class="jxr_linenumber" name="L664" href="#L664">664</a>                     stopCodonName = geneid + <span class="jxr_string">"-stop_codon-"</span> + stopCodon.location().bioStart() + <span class="jxr_string">"-"</span> + stopCodon.location().bioEnd();
<a class="jxr_linenumber" name="L665" href="#L665">665</a>                 }
<a class="jxr_linenumber" name="L666" href="#L666">666</a>                 transcriptSequence.addStopCodonSequence(<strong class="jxr_keyword">new</strong> AccessionID(stopCodonName), stopCodon.location().bioStart(), stopCodon.location().bioEnd());
<a class="jxr_linenumber" name="L667" href="#L667">667</a>             }
<a class="jxr_linenumber" name="L668" href="#L668">668</a> 
<a class="jxr_linenumber" name="L669" href="#L669">669</a>             <strong class="jxr_keyword">for</strong> (FeatureI cdsFeature : cdsFeatures) {
<a class="jxr_linenumber" name="L670" href="#L670">670</a>                 <a href="../../../../org/biojava/nbio/genome/parsers/gff/Feature.html">Feature</a> cds = (Feature) cdsFeature;
<a class="jxr_linenumber" name="L671" href="#L671">671</a>                 String cdsName = cds.getAttribute(<span class="jxr_string">"ID"</span>);
<a class="jxr_linenumber" name="L672" href="#L672">672</a>                 <strong class="jxr_keyword">if</strong> (cdsName == <strong class="jxr_keyword">null</strong> || cdsName.length() == 0) {
<a class="jxr_linenumber" name="L673" href="#L673">673</a>                     cdsName = geneid + <span class="jxr_string">"-cds-"</span> + cds.location().bioStart() + <span class="jxr_string">"-"</span> + cds.location().bioEnd();
<a class="jxr_linenumber" name="L674" href="#L674">674</a>                 }
<a class="jxr_linenumber" name="L675" href="#L675">675</a>                 AccessionID cdsAccessionID = <strong class="jxr_keyword">new</strong> AccessionID(cdsName);
<a class="jxr_linenumber" name="L676" href="#L676">676</a>                 <em class="jxr_comment">//ExonSequence exonSequence = geneSequence.addExon(cdsAccessionID, cdsFeature.location().bioStart(), cdsFeature.location().bioEnd());</em>
<a class="jxr_linenumber" name="L677" href="#L677">677</a>                 transcriptSequence.addCDS(cdsAccessionID, cdsFeature.location().bioStart(), cdsFeature.location().bioEnd(), cds.frame());
<a class="jxr_linenumber" name="L678" href="#L678">678</a>             }
<a class="jxr_linenumber" name="L679" href="#L679">679</a> 
<a class="jxr_linenumber" name="L680" href="#L680">680</a>         }
<a class="jxr_linenumber" name="L681" href="#L681">681</a> 
<a class="jxr_linenumber" name="L682" href="#L682">682</a>     }
<a class="jxr_linenumber" name="L683" href="#L683">683</a> 
<a class="jxr_linenumber" name="L684" href="#L684">684</a>     <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">public</strong> LinkedHashMap&lt;String, ChromosomeSequence&gt; loadFastaAddGeneFeaturesFromGeneMarkGTF(File fastaSequenceFile, File gffFile) <strong class="jxr_keyword">throws</strong> Exception {
<a class="jxr_linenumber" name="L685" href="#L685">685</a>         LinkedHashMap&lt;String, DNASequence&gt; dnaSequenceList = FastaReaderHelper.readFastaDNASequence(fastaSequenceFile);
<a class="jxr_linenumber" name="L686" href="#L686">686</a>         LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList = GeneFeatureHelper.getChromosomeSequenceFromDNASequence(dnaSequenceList);
<a class="jxr_linenumber" name="L687" href="#L687">687</a>         <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> listGenes = GeneMarkGTFReader.read(gffFile.getAbsolutePath());
<a class="jxr_linenumber" name="L688" href="#L688">688</a>         addGeneMarkGTFGeneFeatures(chromosomeSequenceList, listGenes);
<a class="jxr_linenumber" name="L689" href="#L689">689</a>         <strong class="jxr_keyword">return</strong> chromosomeSequenceList;
<a class="jxr_linenumber" name="L690" href="#L690">690</a>     }
<a class="jxr_linenumber" name="L691" href="#L691">691</a> 
<a class="jxr_linenumber" name="L692" href="#L692">692</a>     <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">void</strong> addGeneMarkGTFGeneFeatures(LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList, <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> listGenes) <strong class="jxr_keyword">throws</strong> Exception {
<a class="jxr_linenumber" name="L693" href="#L693">693</a>         Collection&lt;String&gt; geneIds = listGenes.attributeValues(<span class="jxr_string">"gene_id"</span>);
<a class="jxr_linenumber" name="L694" href="#L694">694</a>         <strong class="jxr_keyword">for</strong> (String geneid : geneIds) {
<a class="jxr_linenumber" name="L695" href="#L695">695</a>             <em class="jxr_comment">//       if(geneid.equals("45_g")){</em>
<a class="jxr_linenumber" name="L696" href="#L696">696</a>             <em class="jxr_comment">//           int dummy =1;</em>
<a class="jxr_linenumber" name="L697" href="#L697">697</a>             <em class="jxr_comment">//       }</em>
<a class="jxr_linenumber" name="L698" href="#L698">698</a>             <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> gene = listGenes.selectByAttribute(<span class="jxr_string">"gene_id"</span>, geneid);
<a class="jxr_linenumber" name="L699" href="#L699">699</a>             <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> geneFeature = gene.get(0);
<a class="jxr_linenumber" name="L700" href="#L700">700</a>             ChromosomeSequence seq = chromosomeSequenceList.get(geneFeature.seqname());
<a class="jxr_linenumber" name="L701" href="#L701">701</a>             AccessionID geneAccessionID = <strong class="jxr_keyword">new</strong> AccessionID(geneid);
<a class="jxr_linenumber" name="L702" href="#L702">702</a>             GeneSequence geneSequence = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L703" href="#L703">703</a>             Collection&lt;String&gt; transcriptids = gene.attributeValues(<span class="jxr_string">"transcript_id"</span>);
<a class="jxr_linenumber" name="L704" href="#L704">704</a>             <strong class="jxr_keyword">for</strong> (String transcriptid : transcriptids) {
<a class="jxr_linenumber" name="L705" href="#L705">705</a>                 <em class="jxr_comment">// get all the individual features (exons, CDS regions, etc.) of this gene</em>
<a class="jxr_linenumber" name="L706" href="#L706">706</a> 
<a class="jxr_linenumber" name="L707" href="#L707">707</a> 
<a class="jxr_linenumber" name="L708" href="#L708">708</a>                 <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> transcriptFeature = listGenes.selectByAttribute(<span class="jxr_string">"transcript_id"</span>, transcriptid);
<a class="jxr_linenumber" name="L709" href="#L709">709</a>                 <em class="jxr_comment">// now select only the coding regions of this gene</em>
<a class="jxr_linenumber" name="L710" href="#L710">710</a>                 <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> cdsFeatures = transcriptFeature.selectByType(<span class="jxr_string">"CDS"</span>);
<a class="jxr_linenumber" name="L711" href="#L711">711</a>                 <em class="jxr_comment">// sort them</em>
<a class="jxr_linenumber" name="L712" href="#L712">712</a>                 cdsFeatures = cdsFeatures.sortByStart();
<a class="jxr_linenumber" name="L713" href="#L713">713</a> 
<a class="jxr_linenumber" name="L714" href="#L714">714</a>                 <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> feature = cdsFeatures.get(0);
<a class="jxr_linenumber" name="L715" href="#L715">715</a>                 Strand strand = Strand.POSITIVE;
<a class="jxr_linenumber" name="L716" href="#L716">716</a> 
<a class="jxr_linenumber" name="L717" href="#L717">717</a>                 <strong class="jxr_keyword">if</strong> (feature.location().isNegative()) {
<a class="jxr_linenumber" name="L718" href="#L718">718</a>                     strand = Strand.NEGATIVE;
<a class="jxr_linenumber" name="L719" href="#L719">719</a>                 }
<a class="jxr_linenumber" name="L720" href="#L720">720</a> 
<a class="jxr_linenumber" name="L721" href="#L721">721</a>                 <em class="jxr_comment">//String seqName = feature.seqname();</em>
<a class="jxr_linenumber" name="L722" href="#L722">722</a>                 <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> startCodon = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L723" href="#L723">723</a>                 <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> stopCodon = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L724" href="#L724">724</a>                 Integer startCodonBegin = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L725" href="#L725">725</a>                 Integer stopCodonEnd = <strong class="jxr_keyword">null</strong>;
<a class="jxr_linenumber" name="L726" href="#L726">726</a>                 String startCodonName = <span class="jxr_string">""</span>;
<a class="jxr_linenumber" name="L727" href="#L727">727</a>                 String stopCodonName = <span class="jxr_string">""</span>;
<a class="jxr_linenumber" name="L728" href="#L728">728</a>                 <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> startCodonList = transcriptFeature.selectByType(<span class="jxr_string">"start_codon"</span>);
<a class="jxr_linenumber" name="L729" href="#L729">729</a>                 <strong class="jxr_keyword">if</strong> (startCodonList != <strong class="jxr_keyword">null</strong> &amp;&amp; startCodonList.size() &gt; 0) {
<a class="jxr_linenumber" name="L730" href="#L730">730</a>                     startCodon = startCodonList.get(0);
<a class="jxr_linenumber" name="L731" href="#L731">731</a>                     <strong class="jxr_keyword">if</strong> (strand == Strand.POSITIVE) {
<a class="jxr_linenumber" name="L732" href="#L732">732</a>                         startCodonBegin = startCodon.location().bioStart();
<a class="jxr_linenumber" name="L733" href="#L733">733</a>                     } <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L734" href="#L734">734</a>                         startCodonBegin = startCodon.location().bioEnd();
<a class="jxr_linenumber" name="L735" href="#L735">735</a>                     }
<a class="jxr_linenumber" name="L736" href="#L736">736</a>                     startCodonName = startCodon.getAttribute(<span class="jxr_string">"transcript_name"</span>);
<a class="jxr_linenumber" name="L737" href="#L737">737</a>                 }
<a class="jxr_linenumber" name="L738" href="#L738">738</a> 
<a class="jxr_linenumber" name="L739" href="#L739">739</a>                 <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureList.html">FeatureList</a> stopCodonList = transcriptFeature.selectByType(<span class="jxr_string">"stop_codon"</span>);
<a class="jxr_linenumber" name="L740" href="#L740">740</a> 
<a class="jxr_linenumber" name="L741" href="#L741">741</a>                 <strong class="jxr_keyword">if</strong> (stopCodonList != <strong class="jxr_keyword">null</strong> &amp;&amp; stopCodonList.size() &gt; 0) {
<a class="jxr_linenumber" name="L742" href="#L742">742</a>                     stopCodon = stopCodonList.get(0);
<a class="jxr_linenumber" name="L743" href="#L743">743</a>                     <strong class="jxr_keyword">if</strong> (strand == Strand.POSITIVE) {
<a class="jxr_linenumber" name="L744" href="#L744">744</a>                         stopCodonEnd = stopCodon.location().bioEnd();
<a class="jxr_linenumber" name="L745" href="#L745">745</a>                     } <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L746" href="#L746">746</a>                         stopCodonEnd = stopCodon.location().bioStart();
<a class="jxr_linenumber" name="L747" href="#L747">747</a>                     }
<a class="jxr_linenumber" name="L748" href="#L748">748</a> 
<a class="jxr_linenumber" name="L749" href="#L749">749</a>                     stopCodonName = stopCodon.getAttribute(<span class="jxr_string">"transcript_name"</span>);
<a class="jxr_linenumber" name="L750" href="#L750">750</a> 
<a class="jxr_linenumber" name="L751" href="#L751">751</a>                 }
<a class="jxr_linenumber" name="L752" href="#L752">752</a> 
<a class="jxr_linenumber" name="L753" href="#L753">753</a> 
<a class="jxr_linenumber" name="L754" href="#L754">754</a> 
<a class="jxr_linenumber" name="L755" href="#L755">755</a> 
<a class="jxr_linenumber" name="L756" href="#L756">756</a>                 <strong class="jxr_keyword">if</strong> (startCodonBegin == <strong class="jxr_keyword">null</strong>) {
<a class="jxr_linenumber" name="L757" href="#L757">757</a>                     <strong class="jxr_keyword">if</strong> (strand == Strand.NEGATIVE) {
<a class="jxr_linenumber" name="L758" href="#L758">758</a>                         <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> firstFeature = cdsFeatures.get(0);
<a class="jxr_linenumber" name="L759" href="#L759">759</a> 
<a class="jxr_linenumber" name="L760" href="#L760">760</a>                         startCodonBegin = firstFeature.location().bioEnd();
<a class="jxr_linenumber" name="L761" href="#L761">761</a>                     } <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L762" href="#L762">762</a>                         <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> firstFeature = cdsFeatures.get(0);
<a class="jxr_linenumber" name="L763" href="#L763">763</a> 
<a class="jxr_linenumber" name="L764" href="#L764">764</a>                         startCodonBegin = firstFeature.location().bioStart();
<a class="jxr_linenumber" name="L765" href="#L765">765</a>                     }
<a class="jxr_linenumber" name="L766" href="#L766">766</a>                 }
<a class="jxr_linenumber" name="L767" href="#L767">767</a> 
<a class="jxr_linenumber" name="L768" href="#L768">768</a>                 <strong class="jxr_keyword">if</strong> (stopCodonEnd == <strong class="jxr_keyword">null</strong>) {
<a class="jxr_linenumber" name="L769" href="#L769">769</a>                     <strong class="jxr_keyword">if</strong> (strand == Strand.NEGATIVE) {
<a class="jxr_linenumber" name="L770" href="#L770">770</a>                         <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> lastFeature = cdsFeatures.get(cdsFeatures.size() - 1);
<a class="jxr_linenumber" name="L771" href="#L771">771</a>                         stopCodonEnd = lastFeature.location().bioStart();
<a class="jxr_linenumber" name="L772" href="#L772">772</a>                     } <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L773" href="#L773">773</a>                         <a href="../../../../org/biojava/nbio/genome/parsers/gff/FeatureI.html">FeatureI</a> lastFeature = cdsFeatures.get(cdsFeatures.size() - 1);
<a class="jxr_linenumber" name="L774" href="#L774">774</a>                         stopCodonEnd = lastFeature.location().bioEnd();
<a class="jxr_linenumber" name="L775" href="#L775">775</a>                     }
<a class="jxr_linenumber" name="L776" href="#L776">776</a>                 }
<a class="jxr_linenumber" name="L777" href="#L777">777</a>                 <em class="jxr_comment">//for gtf ordering can be strand based so first is last and last is first</em>
<a class="jxr_linenumber" name="L778" href="#L778">778</a>                 <strong class="jxr_keyword">if</strong> (startCodonBegin &gt; stopCodonEnd) {
<a class="jxr_linenumber" name="L779" href="#L779">779</a>                     <strong class="jxr_keyword">int</strong> temp = startCodonBegin;
<a class="jxr_linenumber" name="L780" href="#L780">780</a>                     startCodonBegin = stopCodonEnd;
<a class="jxr_linenumber" name="L781" href="#L781">781</a>                     stopCodonEnd = temp;
<a class="jxr_linenumber" name="L782" href="#L782">782</a>                 }
<a class="jxr_linenumber" name="L783" href="#L783">783</a> 
<a class="jxr_linenumber" name="L784" href="#L784">784</a>                 AccessionID transcriptAccessionID = <strong class="jxr_keyword">new</strong> AccessionID(transcriptid);
<a class="jxr_linenumber" name="L785" href="#L785">785</a>                 <strong class="jxr_keyword">if</strong> (geneSequence == <strong class="jxr_keyword">null</strong>) {
<a class="jxr_linenumber" name="L786" href="#L786">786</a>                     geneSequence = seq.addGene(geneAccessionID, startCodonBegin, stopCodonEnd, strand);
<a class="jxr_linenumber" name="L787" href="#L787">787</a>                     geneSequence.setSource(((Feature) feature).source());
<a class="jxr_linenumber" name="L788" href="#L788">788</a>                 } <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L789" href="#L789">789</a>                     <em class="jxr_comment">//if multiple transcripts for one gene make sure the gene is defined as the min and max start/end</em>
<a class="jxr_linenumber" name="L790" href="#L790">790</a> 
<a class="jxr_linenumber" name="L791" href="#L791">791</a>                     <strong class="jxr_keyword">if</strong> (startCodonBegin &lt; geneSequence.getBioBegin()) {
<a class="jxr_linenumber" name="L792" href="#L792">792</a>                         geneSequence.setBioBegin(startCodonBegin);
<a class="jxr_linenumber" name="L793" href="#L793">793</a>                     }
<a class="jxr_linenumber" name="L794" href="#L794">794</a>                     <strong class="jxr_keyword">if</strong> (stopCodonEnd &gt; geneSequence.getBioBegin()) {
<a class="jxr_linenumber" name="L795" href="#L795">795</a>                         geneSequence.setBioEnd(stopCodonEnd);
<a class="jxr_linenumber" name="L796" href="#L796">796</a>                     }
<a class="jxr_linenumber" name="L797" href="#L797">797</a> 
<a class="jxr_linenumber" name="L798" href="#L798">798</a>                 }
<a class="jxr_linenumber" name="L799" href="#L799">799</a>                 TranscriptSequence transcriptSequence = geneSequence.addTranscript(transcriptAccessionID, startCodonBegin, stopCodonEnd);
<a class="jxr_linenumber" name="L800" href="#L800">800</a>                 <strong class="jxr_keyword">if</strong> (startCodon != <strong class="jxr_keyword">null</strong>) {
<a class="jxr_linenumber" name="L801" href="#L801">801</a>                     <strong class="jxr_keyword">if</strong> (startCodonName == <strong class="jxr_keyword">null</strong> || startCodonName.length() == 0) {
<a class="jxr_linenumber" name="L802" href="#L802">802</a>                         startCodonName = transcriptid + <span class="jxr_string">"-start_codon-"</span> + startCodon.location().bioStart() + <span class="jxr_string">"-"</span> + startCodon.location().bioEnd();
<a class="jxr_linenumber" name="L803" href="#L803">803</a>                     }
<a class="jxr_linenumber" name="L804" href="#L804">804</a>                     transcriptSequence.addStartCodonSequence(<strong class="jxr_keyword">new</strong> AccessionID(startCodonName), startCodon.location().bioStart(), startCodon.location().bioEnd());
<a class="jxr_linenumber" name="L805" href="#L805">805</a>                 }
<a class="jxr_linenumber" name="L806" href="#L806">806</a>                 <strong class="jxr_keyword">if</strong> (stopCodon != <strong class="jxr_keyword">null</strong>) {
<a class="jxr_linenumber" name="L807" href="#L807">807</a>                     <strong class="jxr_keyword">if</strong> (stopCodonName == <strong class="jxr_keyword">null</strong> || stopCodonName.length() == 0) {
<a class="jxr_linenumber" name="L808" href="#L808">808</a>                         stopCodonName = transcriptid + <span class="jxr_string">"-stop_codon-"</span> + stopCodon.location().bioStart() + <span class="jxr_string">"-"</span> + stopCodon.location().bioEnd();
<a class="jxr_linenumber" name="L809" href="#L809">809</a>                     }
<a class="jxr_linenumber" name="L810" href="#L810">810</a>                     transcriptSequence.addStopCodonSequence(<strong class="jxr_keyword">new</strong> AccessionID(stopCodonName), stopCodon.location().bioStart(), stopCodon.location().bioEnd());
<a class="jxr_linenumber" name="L811" href="#L811">811</a>                 }
<a class="jxr_linenumber" name="L812" href="#L812">812</a> 
<a class="jxr_linenumber" name="L813" href="#L813">813</a>                 <strong class="jxr_keyword">for</strong> (FeatureI cdsFeature : cdsFeatures) {
<a class="jxr_linenumber" name="L814" href="#L814">814</a>                     <a href="../../../../org/biojava/nbio/genome/parsers/gff/Feature.html">Feature</a> cds = (Feature) cdsFeature;
<a class="jxr_linenumber" name="L815" href="#L815">815</a>                     <em class="jxr_comment">// for genemark it appears frame of 2 =1 and frame of 1 = 2</em>
<a class="jxr_linenumber" name="L816" href="#L816">816</a>                     <em class="jxr_comment">// doesn't matter when you string cds regions together as one block</em>
<a class="jxr_linenumber" name="L817" href="#L817">817</a>                     <em class="jxr_comment">// but does make a difference when you try to make a protein sequence for each CDS region where</em>
<a class="jxr_linenumber" name="L818" href="#L818">818</a>                     <em class="jxr_comment">// you give up or borrow based on the frame value</em>
<a class="jxr_linenumber" name="L819" href="#L819">819</a>                     <em class="jxr_comment">// compared with gff like files and docs for geneid and glimmer where geneid and glimmer both do it the same</em>
<a class="jxr_linenumber" name="L820" href="#L820">820</a>                     <em class="jxr_comment">// way that appears to match the gff3 docs.</em>
<a class="jxr_linenumber" name="L821" href="#L821">821</a>                     <strong class="jxr_keyword">int</strong> frame = cds.frame();
<a class="jxr_linenumber" name="L822" href="#L822">822</a>                     <strong class="jxr_keyword">if</strong> (frame == 1) {
<a class="jxr_linenumber" name="L823" href="#L823">823</a>                         frame = 2;
<a class="jxr_linenumber" name="L824" href="#L824">824</a>                     } <strong class="jxr_keyword">else</strong> <strong class="jxr_keyword">if</strong> (frame == 2) {
<a class="jxr_linenumber" name="L825" href="#L825">825</a>                         frame = 1;
<a class="jxr_linenumber" name="L826" href="#L826">826</a>                     } <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L827" href="#L827">827</a>                         frame = 0;
<a class="jxr_linenumber" name="L828" href="#L828">828</a>                     }
<a class="jxr_linenumber" name="L829" href="#L829">829</a>                     String cdsName = cds.getAttribute(<span class="jxr_string">"transcript_name"</span>);
<a class="jxr_linenumber" name="L830" href="#L830">830</a>                     <strong class="jxr_keyword">if</strong> (cdsName == <strong class="jxr_keyword">null</strong> || cdsName.length() == 0) {
<a class="jxr_linenumber" name="L831" href="#L831">831</a>                         cdsName = transcriptid + <span class="jxr_string">"-cds-"</span> + cds.location().bioStart() + <span class="jxr_string">"-"</span> + cds.location().bioEnd();
<a class="jxr_linenumber" name="L832" href="#L832">832</a>                     }
<a class="jxr_linenumber" name="L833" href="#L833">833</a>                     AccessionID cdsAccessionID = <strong class="jxr_keyword">new</strong> AccessionID(cdsName);
<a class="jxr_linenumber" name="L834" href="#L834">834</a>                     <em class="jxr_comment">//ExonSequence exonSequence = geneSequence.addExon(cdsAccessionID, cdsFeature.location().bioStart(), cdsFeature.location().bioEnd());</em>
<a class="jxr_linenumber" name="L835" href="#L835">835</a>                     transcriptSequence.addCDS(cdsAccessionID, cdsFeature.location().bioStart(), cdsFeature.location().bioEnd(), frame);
<a class="jxr_linenumber" name="L836" href="#L836">836</a>                 }
<a class="jxr_linenumber" name="L837" href="#L837">837</a>             }
<a class="jxr_linenumber" name="L838" href="#L838">838</a>         }
<a class="jxr_linenumber" name="L839" href="#L839">839</a> 
<a class="jxr_linenumber" name="L840" href="#L840">840</a>     }
<a class="jxr_linenumber" name="L841" href="#L841">841</a> 
<a class="jxr_linenumber" name="L842" href="#L842">842</a>     <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">public</strong> LinkedHashMap&lt;String, ProteinSequence&gt; getProteinSequences(Collection&lt;ChromosomeSequence&gt; chromosomeSequences) <strong class="jxr_keyword">throws</strong> Exception {
<a class="jxr_linenumber" name="L843" href="#L843">843</a>         LinkedHashMap&lt;String, ProteinSequence&gt; proteinSequenceHashMap = <strong class="jxr_keyword">new</strong> LinkedHashMap&lt;String, ProteinSequence&gt;();
<a class="jxr_linenumber" name="L844" href="#L844">844</a>         <strong class="jxr_keyword">for</strong> (ChromosomeSequence dnaSequence : chromosomeSequences) {
<a class="jxr_linenumber" name="L845" href="#L845">845</a>             <strong class="jxr_keyword">for</strong> (GeneSequence geneSequence : dnaSequence.getGeneSequences().values()) {
<a class="jxr_linenumber" name="L846" href="#L846">846</a>                 <strong class="jxr_keyword">for</strong> (TranscriptSequence transcriptSequence : geneSequence.getTranscripts().values()) {
<a class="jxr_linenumber" name="L847" href="#L847">847</a>                     <em class="jxr_comment">//TODO remove?</em>
<a class="jxr_linenumber" name="L848" href="#L848">848</a> <em class="jxr_comment">//                    DNASequence dnaCodingSequence = transcriptSequence.getDNACodingSequence();</em>
<a class="jxr_linenumber" name="L849" href="#L849">849</a> <em class="jxr_comment">//                    logger.info("CDS={}", dnaCodingSequence.getSequenceAsString());</em>
<a class="jxr_linenumber" name="L850" href="#L850">850</a> 
<a class="jxr_linenumber" name="L851" href="#L851">851</a>                     <strong class="jxr_keyword">try</strong> {
<a class="jxr_linenumber" name="L852" href="#L852">852</a>                         ProteinSequence proteinSequence = transcriptSequence.getProteinSequence();
<a class="jxr_linenumber" name="L853" href="#L853">853</a>                        
<a class="jxr_linenumber" name="L854" href="#L854">854</a> <em class="jxr_comment">//                        logger.info("{} {}", proteinSequence.getAccession().getID(), proteinSequence);</em>
<a class="jxr_linenumber" name="L855" href="#L855">855</a>                         <strong class="jxr_keyword">if</strong> (proteinSequenceHashMap.containsKey(proteinSequence.getAccession().getID())) {
<a class="jxr_linenumber" name="L856" href="#L856">856</a>                             <strong class="jxr_keyword">throw</strong> <strong class="jxr_keyword">new</strong> Exception(<span class="jxr_string">"Duplicate protein sequence id="</span> + proteinSequence.getAccession().getID() + <span class="jxr_string">" found at Gene id="</span> + geneSequence.getAccession().getID());
<a class="jxr_linenumber" name="L857" href="#L857">857</a>                         } <strong class="jxr_keyword">else</strong> {
<a class="jxr_linenumber" name="L858" href="#L858">858</a>                             proteinSequenceHashMap.put(proteinSequence.getAccession().getID(), proteinSequence);
<a class="jxr_linenumber" name="L859" href="#L859">859</a>                         }
<a class="jxr_linenumber" name="L860" href="#L860">860</a>                     } <strong class="jxr_keyword">catch</strong> (Exception e) {
<a class="jxr_linenumber" name="L861" href="#L861">861</a>                         logger.error(<span class="jxr_string">"Exception: "</span>, e);
<a class="jxr_linenumber" name="L862" href="#L862">862</a>                     }
<a class="jxr_linenumber" name="L863" href="#L863">863</a> 
<a class="jxr_linenumber" name="L864" href="#L864">864</a>                 }
<a class="jxr_linenumber" name="L865" href="#L865">865</a> 
<a class="jxr_linenumber" name="L866" href="#L866">866</a>             }
<a class="jxr_linenumber" name="L867" href="#L867">867</a>         }
<a class="jxr_linenumber" name="L868" href="#L868">868</a>         <strong class="jxr_keyword">return</strong> proteinSequenceHashMap;
<a class="jxr_linenumber" name="L869" href="#L869">869</a>     }
<a class="jxr_linenumber" name="L870" href="#L870">870</a> 
<a class="jxr_linenumber" name="L871" href="#L871">871</a>     <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">public</strong> LinkedHashMap&lt;String, GeneSequence&gt; getGeneSequences(Collection&lt;ChromosomeSequence&gt; chromosomeSequences) <strong class="jxr_keyword">throws</strong> Exception {
<a class="jxr_linenumber" name="L872" href="#L872">872</a>         LinkedHashMap&lt;String, GeneSequence&gt; geneSequenceHashMap = <strong class="jxr_keyword">new</strong> LinkedHashMap&lt;String, GeneSequence&gt;();
<a class="jxr_linenumber" name="L873" href="#L873">873</a>         <strong class="jxr_keyword">for</strong> (ChromosomeSequence chromosomeSequence : chromosomeSequences) {
<a class="jxr_linenumber" name="L874" href="#L874">874</a>             <strong class="jxr_keyword">for</strong> (GeneSequence geneSequence : chromosomeSequence.getGeneSequences().values()) {
<a class="jxr_linenumber" name="L875" href="#L875">875</a>                 geneSequenceHashMap.put(geneSequence.getAccession().getID(), geneSequence);
<a class="jxr_linenumber" name="L876" href="#L876">876</a>             }
<a class="jxr_linenumber" name="L877" href="#L877">877</a>         }
<a class="jxr_linenumber" name="L878" href="#L878">878</a> 
<a class="jxr_linenumber" name="L879" href="#L879">879</a>         <strong class="jxr_keyword">return</strong> geneSequenceHashMap;
<a class="jxr_linenumber" name="L880" href="#L880">880</a>     }
<a class="jxr_linenumber" name="L881" href="#L881">881</a> 
<a class="jxr_linenumber" name="L882" href="#L882">882</a>     <strong class="jxr_keyword">public</strong> <strong class="jxr_keyword">static</strong> <strong class="jxr_keyword">void</strong> main(String args[]) <strong class="jxr_keyword">throws</strong> Exception {
<a class="jxr_linenumber" name="L883" href="#L883">883</a> <em class="jxr_comment">/*        if (false) {</em>
<a class="jxr_linenumber" name="L884" href="#L884">884</a> <em class="jxr_comment">            LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList = GeneFeatureHelper.loadFastaAddGeneFeaturesFromGeneMarkGTF(new File("Scaffolds.fna"), new File("genemark_hmm.gtf"));</em>
<a class="jxr_linenumber" name="L885" href="#L885">885</a> <em class="jxr_comment">            LinkedHashMap&lt;String, ProteinSequence&gt; proteinSequenceList = GeneFeatureHelper.getProteinSequences(chromosomeSequenceList.values());</em>
<a class="jxr_linenumber" name="L886" href="#L886">886</a> <em class="jxr_comment">        }</em>
<a class="jxr_linenumber" name="L887" href="#L887">887</a> 
<a class="jxr_linenumber" name="L888" href="#L888">888</a> <em class="jxr_comment">        if (false) {</em>
<a class="jxr_linenumber" name="L889" href="#L889">889</a> <em class="jxr_comment">            LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList = GeneFeatureHelper.loadFastaAddGeneFeaturesFromGlimmerGFF3(new File("Scaffolds.fna"), new File("glimmerhmm.gff"));</em>
<a class="jxr_linenumber" name="L890" href="#L890">890</a> <em class="jxr_comment">            LinkedHashMap&lt;String, ProteinSequence&gt; proteinSequenceList = GeneFeatureHelper.getProteinSequences(chromosomeSequenceList.values());</em>
<a class="jxr_linenumber" name="L891" href="#L891">891</a> <em class="jxr_comment">            //  for (ProteinSequence proteinSequence : proteinSequenceList.values()) {</em>
<a class="jxr_linenumber" name="L892" href="#L892">892</a> <em class="jxr_comment">            //      logger.info(proteinSequence.getAccession().getID() + " " + proteinSequence);</em>
<a class="jxr_linenumber" name="L893" href="#L893">893</a> <em class="jxr_comment">            //  }</em>
<a class="jxr_linenumber" name="L894" href="#L894">894</a> <em class="jxr_comment">            FastaWriterHelper.writeProteinSequence(new File("predicted_glimmer.faa"), proteinSequenceList.values());</em>
<a class="jxr_linenumber" name="L895" href="#L895">895</a> 
<a class="jxr_linenumber" name="L896" href="#L896">896</a> <em class="jxr_comment">        }</em>
<a class="jxr_linenumber" name="L897" href="#L897">897</a> <em class="jxr_comment">        if (false) {</em>
<a class="jxr_linenumber" name="L898" href="#L898">898</a> <em class="jxr_comment">            GeneFeatureHelper.outputFastaSequenceLengthGFF3(new File("Scaffolds.fna"), new File("scaffolds.gff3"));</em>
<a class="jxr_linenumber" name="L899" href="#L899">899</a> <em class="jxr_comment">        }</em>
<a class="jxr_linenumber" name="L900" href="#L900">900</a> 
<a class="jxr_linenumber" name="L901" href="#L901">901</a> <em class="jxr_comment">*/</em>
<a class="jxr_linenumber" name="L902" href="#L902">902</a> 
<a class="jxr_linenumber" name="L903" href="#L903">903</a>         <strong class="jxr_keyword">try</strong> {
<a class="jxr_linenumber" name="L904" href="#L904">904</a> 
<a class="jxr_linenumber" name="L905" href="#L905">905</a>             <strong class="jxr_keyword">if</strong> (<strong class="jxr_keyword">true</strong>) {
<a class="jxr_linenumber" name="L906" href="#L906">906</a>                 <em class="jxr_comment">//File fastaSequenceFile = new File("Scaffolds.fna");</em>
<a class="jxr_linenumber" name="L907" href="#L907">907</a>                 <em class="jxr_comment">//File gff3File = new File("geneid-v6.gff3");</em>
<a class="jxr_linenumber" name="L908" href="#L908">908</a>                 <em class="jxr_comment">//LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceHashMap = GeneFeatureHelper.loadFastaAddGeneFeaturesFromGmodGFF3(fastaSequenceFile, gff3File,true);</em>
<a class="jxr_linenumber" name="L909" href="#L909">909</a>             }
<a class="jxr_linenumber" name="L910" href="#L910">910</a> 
<a class="jxr_linenumber" name="L911" href="#L911">911</a>         } <strong class="jxr_keyword">catch</strong> (Exception e) {
<a class="jxr_linenumber" name="L912" href="#L912">912</a>             logger.error(<span class="jxr_string">"Exception: "</span>, e);
<a class="jxr_linenumber" name="L913" href="#L913">913</a>         }
<a class="jxr_linenumber" name="L914" href="#L914">914</a> 
<a class="jxr_linenumber" name="L915" href="#L915">915</a>     }
<a class="jxr_linenumber" name="L916" href="#L916">916</a> }
</pre>
<hr/>
<div id="footer">Copyright &#169; 2000&#x2013;2015 <a href="http://www.biojava.org">BioJava</a>. All rights reserved.</div>
</body>
</html>
