<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_71) on Fri Jan 30 08:29:55 PST 2015 -->
<meta http-equiv="Content-Type" content="text/html" charset="UTF-8">
<title>Index (biojava-genome 4.0.0 API)</title>
<meta name="date" content="2015-01-30">
<link rel="stylesheet" type="text/css" href="./stylesheet.css" title="Style">
</head>
<body>
<script type="text/javascript"><!--
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="Index (biojava-genome 4.0.0 API)";
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar_top">
<!--   -->
</a><a href="#skip-navbar_top" title="Skip navigation links"></a><a name="navbar_top_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="./overview-summary.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li>Use</li>
<li><a href="./overview-tree.html">Tree</a></li>
<li><a href="./deprecated-list.html">Deprecated</a></li>
<li class="navBarCell1Rev">Index</li>
<li><a href="./help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="./index.html?index-all.html" target="_top">Frames</a></li>
<li><a href="index-all.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="./allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<div class="contentContainer"><a href="#_A_">A</a>&nbsp;<a href="#_B_">B</a>&nbsp;<a href="#_C_">C</a>&nbsp;<a href="#_D_">D</a>&nbsp;<a href="#_E_">E</a>&nbsp;<a href="#_F_">F</a>&nbsp;<a href="#_G_">G</a>&nbsp;<a href="#_H_">H</a>&nbsp;<a href="#_I_">I</a>&nbsp;<a href="#_L_">L</a>&nbsp;<a href="#_M_">M</a>&nbsp;<a href="#_N_">N</a>&nbsp;<a href="#_O_">O</a>&nbsp;<a href="#_P_">P</a>&nbsp;<a href="#_R_">R</a>&nbsp;<a href="#_S_">S</a>&nbsp;<a href="#_T_">T</a>&nbsp;<a href="#_U_">U</a>&nbsp;<a href="#_V_">V</a>&nbsp;<a href="#_W_">W</a>&nbsp;<a name="_A_">
<!--   -->
</a>
<h2 class="title">A</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html#add(org.biojava.nbio.genome.parsers.gff.FeatureI)">add(FeatureI)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Add specified feature to the end of the list.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html#add(java.util.Collection)">add(Collection&lt;FeatureI&gt;)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Add all features in the specified list or collection to this list.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/GeneFeatureHelper.html#addGeneIDGFF2GeneFeatures(java.util.LinkedHashMap,%20org.biojava.nbio.genome.parsers.gff.FeatureList)">addGeneIDGFF2GeneFeatures(LinkedHashMap&lt;String, ChromosomeSequence&gt;, FeatureList)</a></span> - Static method in class org.biojava.nbio.genome.<a href="./org/biojava/nbio/genome/GeneFeatureHelper.html" title="class in org.biojava.nbio.genome">GeneFeatureHelper</a></dt>
<dd>
<div class="block">Load GFF2 feature file generated from the geneid prediction algorithm and map features onto the chromosome sequences</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/GeneFeatureHelper.html#addGeneMarkGTFGeneFeatures(java.util.LinkedHashMap,%20org.biojava.nbio.genome.parsers.gff.FeatureList)">addGeneMarkGTFGeneFeatures(LinkedHashMap&lt;String, ChromosomeSequence&gt;, FeatureList)</a></span> - Static method in class org.biojava.nbio.genome.<a href="./org/biojava/nbio/genome/GeneFeatureHelper.html" title="class in org.biojava.nbio.genome">GeneFeatureHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/GeneFeatureHelper.html#addGlimmerGFF3GeneFeatures(java.util.LinkedHashMap,%20org.biojava.nbio.genome.parsers.gff.FeatureList)">addGlimmerGFF3GeneFeatures(LinkedHashMap&lt;String, ChromosomeSequence&gt;, FeatureList)</a></span> - Static method in class org.biojava.nbio.genome.<a href="./org/biojava/nbio/genome/GeneFeatureHelper.html" title="class in org.biojava.nbio.genome">GeneFeatureHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/GeneFeatureHelper.html#addGmodGFF3GeneFeatures(java.util.LinkedHashMap,%20org.biojava.nbio.genome.parsers.gff.FeatureList)">addGmodGFF3GeneFeatures(LinkedHashMap&lt;String, ChromosomeSequence&gt;, FeatureList)</a></span> - Static method in class org.biojava.nbio.genome.<a href="./org/biojava/nbio/genome/GeneFeatureHelper.html" title="class in org.biojava.nbio.genome">GeneFeatureHelper</a></dt>
<dd>
<div class="block">Load GFF3 file using mRNA as the gene feature as not all GFF3 files are complete</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html#addIndex(java.lang.String)">addIndex(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Add an attribute that will be used as index for queries</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html#addIndexes(java.util.List)">addIndexes(List&lt;String&gt;)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Add a list of attributes that will be used as indexes for queries</div>
</dd>
<dt><a href="./org/biojava/nbio/genome/App.html" title="class in org.biojava.nbio.genome"><span class="strong">App</span></a> - Class in <a href="./org/biojava/nbio/genome/package-summary.html">org.biojava.nbio.genome</a></dt>
<dd>
<div class="block">Hello world!</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/App.html#App()">App()</a></span> - Constructor for class org.biojava.nbio.genome.<a href="./org/biojava/nbio/genome/App.html" title="class in org.biojava.nbio.genome">App</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Feature.html#attributes()">attributes()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Feature.html" title="class in org.biojava.nbio.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block">Get the string of key/value attributes.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html#attributeValues(java.lang.String)">attributeValues(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Create a collection of the unique values for the specified key.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html#available()">available()</a></span> - Method in class org.biojava.nbio.genome.parsers.twobit.<a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html" title="class in org.biojava.nbio.genome.parsers.twobit">TwoBitParser</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_B_">
<!--   -->
</a>
<h2 class="title">B</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#bioEnd()">bioEnd()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Get end index, in biocoordinates.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#bioStart()">bioStart()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Get start index, in biocoordinates.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#bioStrand()">bioStrand()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Get character representation of strand.</div>
</dd>
<dt><a href="./org/biojava/nbio/genome/homology/BlastHomologyHits.html" title="class in org.biojava.nbio.genome.homology"><span class="strong">BlastHomologyHits</span></a> - Class in <a href="./org/biojava/nbio/genome/homology/package-summary.html">org.biojava.nbio.genome.homology</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/homology/BlastHomologyHits.html#BlastHomologyHits()">BlastHomologyHits()</a></span> - Constructor for class org.biojava.nbio.genome.homology.<a href="./org/biojava/nbio/genome/homology/BlastHomologyHits.html" title="class in org.biojava.nbio.genome.homology">BlastHomologyHits</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/genome/query/BlastXMLQuery.html" title="class in org.biojava.nbio.genome.query"><span class="strong">BlastXMLQuery</span></a> - Class in <a href="./org/biojava/nbio/genome/query/package-summary.html">org.biojava.nbio.genome.query</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/query/BlastXMLQuery.html#BlastXMLQuery(java.lang.String)">BlastXMLQuery(String)</a></span> - Constructor for class org.biojava.nbio.genome.query.<a href="./org/biojava/nbio/genome/query/BlastXMLQuery.html" title="class in org.biojava.nbio.genome.query">BlastXMLQuery</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html#bounds()">bounds()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">The union of all locations of all features in this list, mapped to the positive strand.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureHelper.html#buildFeatureAtrributeIndex(java.lang.String,%20org.biojava.nbio.genome.parsers.gff.FeatureList)">buildFeatureAtrributeIndex(String, FeatureList)</a></span> - Static method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureHelper.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureHelper</a></dt>
<dd>
<div class="block">Build a list of individual features to allow easy indexing and to avoid iterating through large genome gff3 files
 The index for the returned HashMap is the value of the attribute used to build the index</div>
</dd>
</dl>
<a name="_C_">
<!--   -->
</a>
<h2 class="title">C</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html#close()">close()</a></span> - Method in class org.biojava.nbio.genome.parsers.twobit.<a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html" title="class in org.biojava.nbio.genome.parsers.twobit">TwoBitParser</a></dt>
<dd>
<div class="block">Method closes current sequence and it's necessary to invoke it before setting
 new current sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html#closeParser()">closeParser()</a></span> - Method in class org.biojava.nbio.genome.parsers.twobit.<a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html" title="class in org.biojava.nbio.genome.parsers.twobit">TwoBitParser</a></dt>
<dd>
<div class="block">Method closes random access file descriptor.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html#compareTo(org.biojava.nbio.genome.parsers.cytoband.Cytoband)">compareTo(Cytoband)</a></span> - Method in class org.biojava.nbio.genome.parsers.cytoband.<a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html" title="class in org.biojava.nbio.genome.parsers.cytoband">Cytoband</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#compareTo(org.biojava.nbio.genome.parsers.genename.GeneChromosomePosition)">compareTo(GeneChromosomePosition)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#compareTo(org.biojava.nbio.genome.parsers.genename.GeneName)">compareTo(GeneName)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#contains(org.biojava.nbio.genome.parsers.gff.Location)">contains(Location)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Check if this location contains the other.</div>
</dd>
<dt><a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html" title="class in org.biojava.nbio.genome.parsers.cytoband"><span class="strong">Cytoband</span></a> - Class in <a href="./org/biojava/nbio/genome/parsers/cytoband/package-summary.html">org.biojava.nbio.genome.parsers.cytoband</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html#Cytoband()">Cytoband()</a></span> - Constructor for class org.biojava.nbio.genome.parsers.cytoband.<a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html" title="class in org.biojava.nbio.genome.parsers.cytoband">Cytoband</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/genome/parsers/cytoband/CytobandParser.html" title="class in org.biojava.nbio.genome.parsers.cytoband"><span class="strong">CytobandParser</span></a> - Class in <a href="./org/biojava/nbio/genome/parsers/cytoband/package-summary.html">org.biojava.nbio.genome.parsers.cytoband</a></dt>
<dd>
<div class="block">Parses the cytoband (karyotype) file from UCSC.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/cytoband/CytobandParser.html#CytobandParser()">CytobandParser()</a></span> - Constructor for class org.biojava.nbio.genome.parsers.cytoband.<a href="./org/biojava/nbio/genome/parsers/cytoband/CytobandParser.html" title="class in org.biojava.nbio.genome.parsers.cytoband">CytobandParser</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_D_">
<!--   -->
</a>
<h2 class="title">D</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html#DEFAULT_BUFFER_SIZE">DEFAULT_BUFFER_SIZE</a></span> - Variable in class org.biojava.nbio.genome.parsers.twobit.<a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html" title="class in org.biojava.nbio.genome.parsers.twobit">TwoBitParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneNamesParser.html#DEFAULT_GENENAMES_URL">DEFAULT_GENENAMES_URL</a></span> - Static variable in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneNamesParser.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneNamesParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/cytoband/CytobandParser.html#DEFAULT_LOCATION">DEFAULT_LOCATION</a></span> - Static variable in class org.biojava.nbio.genome.parsers.cytoband.<a href="./org/biojava/nbio/genome/parsers/cytoband/CytobandParser.html" title="class in org.biojava.nbio.genome.parsers.cytoband">CytobandParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePositionParser.html#DEFAULT_MAPPING_URL">DEFAULT_MAPPING_URL</a></span> - Static variable in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePositionParser.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePositionParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#distance(org.biojava.nbio.genome.parsers.gff.Location)">distance(Location)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Return distance between this location and the other location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#downstream(int)">downstream(int)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Return the adjacent location of specified length directly downstream of this location.</div>
</dd>
</dl>
<a name="_E_">
<!--   -->
</a>
<h2 class="title">E</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#end()">end()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Get the ending index.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#endsAfter(org.biojava.nbio.genome.parsers.gff.Location)">endsAfter(Location)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Check if this location ends after other location ends.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#endsBefore(org.biojava.nbio.genome.parsers.gff.Location)">endsBefore(Location)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Check if this location ends before other location ends.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#equals(org.biojava.nbio.genome.parsers.genename.GeneName)">equals(GeneName)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#equals(org.biojava.nbio.genome.parsers.gff.Location)">equals(Location)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Compare locations for equality.</div>
</dd>
</dl>
<a name="_F_">
<!--   -->
</a>
<h2 class="title">F</h2>
<dl>
<dt><a href="./org/biojava/nbio/genome/parsers/gff/Feature.html" title="class in org.biojava.nbio.genome.parsers.gff"><span class="strong">Feature</span></a> - Class in <a href="./org/biojava/nbio/genome/parsers/gff/package-summary.html">org.biojava.nbio.genome.parsers.gff</a></dt>
<dd>
<div class="block">A Feature corresponds to a single row in a GFF file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Feature.html#Feature(org.biojava.nbio.genome.parsers.gff.Feature)">Feature(Feature)</a></span> - Constructor for class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Feature.html" title="class in org.biojava.nbio.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block">Make a copy of the specified feature.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Feature.html#Feature(java.lang.String,%20java.lang.String,%20java.lang.String,%20org.biojava.nbio.genome.parsers.gff.Location,%20java.lang.Double,%20int,%20java.lang.String)">Feature(String, String, String, Location, Double, int, String)</a></span> - Constructor for class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Feature.html" title="class in org.biojava.nbio.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block">Construct a new Feature from raw data (usually a GFF row).</div>
</dd>
<dt><a href="./org/biojava/nbio/genome/parsers/gff/FeatureHelper.html" title="class in org.biojava.nbio.genome.parsers.gff"><span class="strong">FeatureHelper</span></a> - Class in <a href="./org/biojava/nbio/genome/parsers/gff/package-summary.html">org.biojava.nbio.genome.parsers.gff</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureHelper.html#FeatureHelper()">FeatureHelper()</a></span> - Constructor for class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureHelper.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/genome/parsers/gff/FeatureI.html" title="interface in org.biojava.nbio.genome.parsers.gff"><span class="strong">FeatureI</span></a> - Interface in <a href="./org/biojava/nbio/genome/parsers/gff/package-summary.html">org.biojava.nbio.genome.parsers.gff</a></dt>
<dd>
<div class="block">A feature on a sequence (for example, an exon or a gene), defined by a location
 and a set of attributes encoded as key/value pairs.</div>
</dd>
<dt><a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html" title="class in org.biojava.nbio.genome.parsers.gff"><span class="strong">FeatureList</span></a> - Class in <a href="./org/biojava/nbio/genome/parsers/gff/package-summary.html">org.biojava.nbio.genome.parsers.gff</a></dt>
<dd>
<div class="block">A list of FeatureI objects implemented using a Java ArrayList; corresponds to a GFF file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html#FeatureList()">FeatureList()</a></span> - Constructor for class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Construct an empty list.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html#FeatureList(java.util.Collection)">FeatureList(Collection&lt;FeatureI&gt;)</a></span> - Constructor for class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Construct a new list containing the same features
 as the specified list.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Feature.html#frame()">frame()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Feature.html" title="class in org.biojava.nbio.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block">Get frame (aka phase).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#fromBio(int,%20int,%20char)">fromBio(int, int, char)</a></span> - Static method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Create location from "biocoordinates", as in GFF file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#fromBioExt(int,%20int,%20char,%20int)">fromBioExt(int, int, char, int)</a></span> - Static method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Create a location from MAF file coordinates, which represent negative
 strand locations as the distance from the end of the sequence.</div>
</dd>
</dl>
<a name="_G_">
<!--   -->
</a>
<h2 class="title">G</h2>
<dl>
<dt><a href="./org/biojava/nbio/genome/parsers/gff/GCStats.html" title="class in org.biojava.nbio.genome.parsers.gff"><span class="strong">GCStats</span></a> - Class in <a href="./org/biojava/nbio/genome/parsers/gff/package-summary.html">org.biojava.nbio.genome.parsers.gff</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/GCStats.html#GCStats()">GCStats()</a></span> - Constructor for class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/GCStats.html" title="class in org.biojava.nbio.genome.parsers.gff">GCStats</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename"><span class="strong">GeneChromosomePosition</span></a> - Class in <a href="./org/biojava/nbio/genome/parsers/genename/package-summary.html">org.biojava.nbio.genome.parsers.genename</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#GeneChromosomePosition()">GeneChromosomePosition()</a></span> - Constructor for class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePosition</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePositionParser.html" title="class in org.biojava.nbio.genome.parsers.genename"><span class="strong">GeneChromosomePositionParser</span></a> - Class in <a href="./org/biojava/nbio/genome/parsers/genename/package-summary.html">org.biojava.nbio.genome.parsers.genename</a></dt>
<dd>
<div class="block">A parser that parses a file from the UCSC genome browser that contains mapping of gene name to chromosome positions</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePositionParser.html#GeneChromosomePositionParser()">GeneChromosomePositionParser()</a></span> - Constructor for class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePositionParser.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePositionParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/genome/GeneFeatureHelper.html" title="class in org.biojava.nbio.genome"><span class="strong">GeneFeatureHelper</span></a> - Class in <a href="./org/biojava/nbio/genome/package-summary.html">org.biojava.nbio.genome</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/GeneFeatureHelper.html#GeneFeatureHelper()">GeneFeatureHelper()</a></span> - Constructor for class org.biojava.nbio.genome.<a href="./org/biojava/nbio/genome/GeneFeatureHelper.html" title="class in org.biojava.nbio.genome">GeneFeatureHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/genome/parsers/gff/GeneIDGFF2Reader.html" title="class in org.biojava.nbio.genome.parsers.gff"><span class="strong">GeneIDGFF2Reader</span></a> - Class in <a href="./org/biojava/nbio/genome/parsers/gff/package-summary.html">org.biojava.nbio.genome.parsers.gff</a></dt>
<dd>
<div class="block">http://www.bioperl.org/wiki/GTF
 Read and write FeatureLists as GFF/GTF formatted files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/GeneIDGFF2Reader.html#GeneIDGFF2Reader()">GeneIDGFF2Reader()</a></span> - Constructor for class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/GeneIDGFF2Reader.html" title="class in org.biojava.nbio.genome.parsers.gff">GeneIDGFF2Reader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/genome/parsers/geneid/GeneIDXMLReader.html" title="class in org.biojava.nbio.genome.parsers.geneid"><span class="strong">GeneIDXMLReader</span></a> - Class in <a href="./org/biojava/nbio/genome/parsers/geneid/package-summary.html">org.biojava.nbio.genome.parsers.geneid</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/geneid/GeneIDXMLReader.html#GeneIDXMLReader(java.lang.String)">GeneIDXMLReader(String)</a></span> - Constructor for class org.biojava.nbio.genome.parsers.geneid.<a href="./org/biojava/nbio/genome/parsers/geneid/GeneIDXMLReader.html" title="class in org.biojava.nbio.genome.parsers.geneid">GeneIDXMLReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/genome/parsers/gff/GeneMarkGTFReader.html" title="class in org.biojava.nbio.genome.parsers.gff"><span class="strong">GeneMarkGTFReader</span></a> - Class in <a href="./org/biojava/nbio/genome/parsers/gff/package-summary.html">org.biojava.nbio.genome.parsers.gff</a></dt>
<dd>
<div class="block">http://www.bioperl.org/wiki/GTF
 Read and write FeatureLists as GFF/GTF formatted files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/GeneMarkGTFReader.html#GeneMarkGTFReader()">GeneMarkGTFReader()</a></span> - Constructor for class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/GeneMarkGTFReader.html" title="class in org.biojava.nbio.genome.parsers.gff">GeneMarkGTFReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename"><span class="strong">GeneName</span></a> - Class in <a href="./org/biojava/nbio/genome/parsers/genename/package-summary.html">org.biojava.nbio.genome.parsers.genename</a></dt>
<dd>
<div class="block">A simple bean that contains gene name information as available from www.genenames.org</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#GeneName()">GeneName()</a></span> - Constructor for class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/genome/parsers/genename/GeneNamesParser.html" title="class in org.biojava.nbio.genome.parsers.genename"><span class="strong">GeneNamesParser</span></a> - Class in <a href="./org/biojava/nbio/genome/parsers/genename/package-summary.html">org.biojava.nbio.genome.parsers.genename</a></dt>
<dd>
<div class="block">Parses a file from the www.genenames.org website that contains a mapping of human gene names to other databases</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneNamesParser.html#GeneNamesParser()">GeneNamesParser()</a></span> - Constructor for class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneNamesParser.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneNamesParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#getAccessionNr()">getAccessionNr()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/cytoband/CytobandParser.html#getAllCytobands(java.net.URL)">getAllCytobands(URL)</a></span> - Method in class org.biojava.nbio.genome.parsers.cytoband.<a href="./org/biojava/nbio/genome/parsers/cytoband/CytobandParser.html" title="class in org.biojava.nbio.genome.parsers.cytoband">CytobandParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/cytoband/CytobandParser.html#getAllCytobands(java.io.InputStream)">getAllCytobands(InputStream)</a></span> - Method in class org.biojava.nbio.genome.parsers.cytoband.<a href="./org/biojava/nbio/genome/parsers/cytoband/CytobandParser.html" title="class in org.biojava.nbio.genome.parsers.cytoband">CytobandParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#getApprovedName()">getApprovedName()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#getApprovedSymbol()">getApprovedSymbol()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Feature.html#getAttribute(java.lang.String)">getAttribute(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Feature.html" title="class in org.biojava.nbio.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block">Get value of specified attribute key.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureI.html#getAttribute(java.lang.String)">getAttribute(String)</a></span> - Method in interface org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureI.html" title="interface in org.biojava.nbio.genome.parsers.gff">FeatureI</a></dt>
<dd>
<div class="block">Get the attribute value for this key.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Feature.html#getAttributeOld(java.lang.String)">getAttributeOld(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Feature.html" title="class in org.biojava.nbio.genome.parsers.gff">Feature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Feature.html#getAttributes()">getAttributes()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Feature.html" title="class in org.biojava.nbio.genome.parsers.gff">Feature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureI.html#getAttributes()">getAttributes()</a></span> - Method in interface org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureI.html" title="interface in org.biojava.nbio.genome.parsers.gff">FeatureI</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#getBegin()">getBegin()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#getCdsEnd()">getCdsEnd()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#getCdsStart()">getCdsStart()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html#getChromosome()">getChromosome()</a></span> - Method in class org.biojava.nbio.genome.parsers.cytoband.<a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html" title="class in org.biojava.nbio.genome.parsers.cytoband">Cytoband</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#getChromosome()">getChromosome()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#getChromosome()">getChromosome()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePositionParser.html#getChromosomeMappings()">getChromosomeMappings()</a></span> - Static method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePositionParser.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePositionParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePositionParser.html#getChromosomeMappings(java.io.InputStream)">getChromosomeMappings(InputStream)</a></span> - Static method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePositionParser.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePositionParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/GeneFeatureHelper.html#getChromosomeSequenceFromDNASequence(java.util.LinkedHashMap)">getChromosomeSequenceFromDNASequence(LinkedHashMap&lt;String, DNASequence&gt;)</a></span> - Static method in class org.biojava.nbio.genome.<a href="./org/biojava/nbio/genome/GeneFeatureHelper.html" title="class in org.biojava.nbio.genome">GeneFeatureHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html#getCurrentSequencePosition()">getCurrentSequencePosition()</a></span> - Method in class org.biojava.nbio.genome.parsers.twobit.<a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html" title="class in org.biojava.nbio.genome.parsers.twobit">TwoBitParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/geneid/GeneIDXMLReader.html#getDNACodingSequences()">getDNACodingSequences()</a></span> - Method in class org.biojava.nbio.genome.parsers.geneid.<a href="./org/biojava/nbio/genome/parsers/geneid/GeneIDXMLReader.html" title="class in org.biojava.nbio.genome.parsers.geneid">GeneIDXMLReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html#getEnd()">getEnd()</a></span> - Method in class org.biojava.nbio.genome.parsers.cytoband.<a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html" title="class in org.biojava.nbio.genome.parsers.cytoband">Cytoband</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#getEnd()">getEnd()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#getEnsemblGeneId()">getEnsemblGeneId()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#getExonCount()">getExonCount()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#getExonEnds()">getExonEnds()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#getExonStarts()">getExonStarts()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html#getFile()">getFile()</a></span> - Method in class org.biojava.nbio.genome.parsers.twobit.<a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html" title="class in org.biojava.nbio.genome.parsers.twobit">TwoBitParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/GCStats.html#getGCStats(java.util.Collection)">getGCStats(Collection&lt;DNASequence&gt;)</a></span> - Static method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/GCStats.html" title="class in org.biojava.nbio.genome.parsers.gff">GCStats</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/GCStats.html#getGCStatsString(java.util.Collection)">getGCStatsString(Collection&lt;String&gt;)</a></span> - Static method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/GCStats.html" title="class in org.biojava.nbio.genome.parsers.gff">GCStats</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#getGenebankId()">getGenebankId()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#getGeneName()">getGeneName()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneNamesParser.html#getGeneNames()">getGeneNames()</a></span> - Static method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneNamesParser.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneNamesParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneNamesParser.html#getGeneNames(java.io.InputStream)">getGeneNames(InputStream)</a></span> - Static method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneNamesParser.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneNamesParser</a></dt>
<dd>
<div class="block">Get a list of GeneNames from an input stream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/GeneFeatureHelper.html#getGeneSequences(java.util.Collection)">getGeneSequences(Collection&lt;ChromosomeSequence&gt;)</a></span> - Static method in class org.biojava.nbio.genome.<a href="./org/biojava/nbio/genome/GeneFeatureHelper.html" title="class in org.biojava.nbio.genome">GeneFeatureHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#getHgncId()">getHgncId()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/query/BlastXMLQuery.html#getHitsQueryDef(double)">getHitsQueryDef(double)</a></span> - Method in class org.biojava.nbio.genome.query.<a href="./org/biojava/nbio/genome/query/BlastXMLQuery.html" title="class in org.biojava.nbio.genome.query">BlastXMLQuery</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html#getLocus()">getLocus()</a></span> - Method in class org.biojava.nbio.genome.parsers.cytoband.<a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html" title="class in org.biojava.nbio.genome.parsers.cytoband">Cytoband</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/homology/BlastHomologyHits.html#getMatches(java.io.File,%20double)">getMatches(File, double)</a></span> - Static method in class org.biojava.nbio.genome.homology.<a href="./org/biojava/nbio/genome/homology/BlastHomologyHits.html" title="class in org.biojava.nbio.genome.homology">BlastHomologyHits</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#getOmimId()">getOmimId()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#getOrientation()">getOrientation()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#getPreviousNames()">getPreviousNames()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#getPreviousSymbols()">getPreviousSymbols()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/GeneFeatureHelper.html#getProteinSequences(java.util.Collection)">getProteinSequences(Collection&lt;ChromosomeSequence&gt;)</a></span> - Static method in class org.biojava.nbio.genome.<a href="./org/biojava/nbio/genome/GeneFeatureHelper.html" title="class in org.biojava.nbio.genome">GeneFeatureHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/geneid/GeneIDXMLReader.html#getProteinSequences()">getProteinSequences()</a></span> - Method in class org.biojava.nbio.genome.parsers.geneid.<a href="./org/biojava/nbio/genome/parsers/geneid/GeneIDXMLReader.html" title="class in org.biojava.nbio.genome.parsers.geneid">GeneIDXMLReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#getRefseqIds()">getRefseqIds()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html#getSequenceNames()">getSequenceNames()</a></span> - Method in class org.biojava.nbio.genome.parsers.twobit.<a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html" title="class in org.biojava.nbio.genome.parsers.twobit">TwoBitParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/cytoband/StainType.html#getStainTypeFromString(java.lang.String)">getStainTypeFromString(String)</a></span> - Static method in enum org.biojava.nbio.genome.parsers.cytoband.<a href="./org/biojava/nbio/genome/parsers/cytoband/StainType.html" title="enum in org.biojava.nbio.genome.parsers.cytoband">StainType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html#getStart()">getStart()</a></span> - Method in class org.biojava.nbio.genome.parsers.cytoband.<a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html" title="class in org.biojava.nbio.genome.parsers.cytoband">Cytoband</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#getStatus()">getStatus()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#getSynonyms()">getSynonyms()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#getTranscriptionEnd()">getTranscriptionEnd()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#getTranscriptionStart()">getTranscriptionStart()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html#getType()">getType()</a></span> - Method in class org.biojava.nbio.genome.parsers.cytoband.<a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html" title="class in org.biojava.nbio.genome.parsers.cytoband">Cytoband</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#getUniprot()">getUniprot()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/genome/homology/GFF3FromUniprotBlastHits.html" title="class in org.biojava.nbio.genome.homology"><span class="strong">GFF3FromUniprotBlastHits</span></a> - Class in <a href="./org/biojava/nbio/genome/homology/package-summary.html">org.biojava.nbio.genome.homology</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/homology/GFF3FromUniprotBlastHits.html#GFF3FromUniprotBlastHits()">GFF3FromUniprotBlastHits()</a></span> - Constructor for class org.biojava.nbio.genome.homology.<a href="./org/biojava/nbio/genome/homology/GFF3FromUniprotBlastHits.html" title="class in org.biojava.nbio.genome.homology">GFF3FromUniprotBlastHits</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/genome/parsers/gff/GFF3Reader.html" title="class in org.biojava.nbio.genome.parsers.gff"><span class="strong">GFF3Reader</span></a> - Class in <a href="./org/biojava/nbio/genome/parsers/gff/package-summary.html">org.biojava.nbio.genome.parsers.gff</a></dt>
<dd>
<div class="block">http://www.bioperl.org/wiki/GTF
 Read and write FeatureLists as GFF/GTF formatted files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/GFF3Reader.html#GFF3Reader()">GFF3Reader()</a></span> - Constructor for class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/GFF3Reader.html" title="class in org.biojava.nbio.genome.parsers.gff">GFF3Reader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/genome/parsers/gff/GFF3Writer.html" title="class in org.biojava.nbio.genome.parsers.gff"><span class="strong">GFF3Writer</span></a> - Class in <a href="./org/biojava/nbio/genome/parsers/gff/package-summary.html">org.biojava.nbio.genome.parsers.gff</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/GFF3Writer.html#GFF3Writer()">GFF3Writer()</a></span> - Constructor for class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/GFF3Writer.html" title="class in org.biojava.nbio.genome.parsers.gff">GFF3Writer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Feature.html#group()">group()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Feature.html" title="class in org.biojava.nbio.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block">Get the first item (everything before first semicolon, if it has one)
 in the attribute field, which is assumed to
 be a group identifer.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureI.html#group()">group()</a></span> - Method in interface org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureI.html" title="interface in org.biojava.nbio.genome.parsers.gff">FeatureI</a></dt>
<dd>
<div class="block">Get the group id of the feature.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html#groupValues()">groupValues()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Create a collection of all unique group ids in the list, as defined
 by the group() method of the features.</div>
</dd>
</dl>
<a name="_H_">
<!--   -->
</a>
<h2 class="title">H</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Feature.html#hasAttribute(java.lang.String)">hasAttribute(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Feature.html" title="class in org.biojava.nbio.genome.parsers.gff">Feature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Feature.html#hasAttribute(java.lang.String,%20java.lang.String)">hasAttribute(String, String)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Feature.html" title="class in org.biojava.nbio.genome.parsers.gff">Feature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureI.html#hasAttribute(java.lang.String)">hasAttribute(String)</a></span> - Method in interface org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureI.html" title="interface in org.biojava.nbio.genome.parsers.gff">FeatureI</a></dt>
<dd>
<div class="block">Check if the feature has a value defined for the specified key.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureI.html#hasAttribute(java.lang.String,%20java.lang.String)">hasAttribute(String, String)</a></span> - Method in interface org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureI.html" title="interface in org.biojava.nbio.genome.parsers.gff">FeatureI</a></dt>
<dd>
<div class="block">Check if the feature attributes include the specified key/value pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html#hasAttribute(java.lang.String)">hasAttribute(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Check if any feature in list has the specified attribute key.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html#hasAttribute(java.lang.String,%20java.lang.String)">hasAttribute(String, String)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Check if any feature in list has the specified attribute key/value pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html#hasGaps(int)">hasGaps(int)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Check size of gaps between successive features in list.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/LocIterator.html#hasNext(int,%20int)">hasNext(int, int)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/LocIterator.html" title="class in org.biojava.nbio.genome.parsers.gff">LocIterator</a></dt>
<dd>
<div class="block">Check if next window of specified size is available.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/LocIterator.html#hasNext()">hasNext()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/LocIterator.html" title="class in org.biojava.nbio.genome.parsers.gff">LocIterator</a></dt>
<dd>
<div class="block">Check if next window of default size is available.</div>
</dd>
</dl>
<a name="_I_">
<!--   -->
</a>
<h2 class="title">I</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#intersection(org.biojava.nbio.genome.parsers.gff.Location)">intersection(Location)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Return the intersection, or null if no overlap.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#isAfter(org.biojava.nbio.genome.parsers.gff.Location)">isAfter(Location)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Check if this location is entirely after the other location (no overlap).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#isBefore(org.biojava.nbio.genome.parsers.gff.Location)">isBefore(Location)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Check if this location is entirely before other location (no overlap).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#isNegative()">isNegative()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Check if location is on negative strand.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#isSameStrand(org.biojava.nbio.genome.parsers.gff.Location)">isSameStrand(Location)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Check if this location is on same strand as other location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#iterator()">iterator()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Create a location iterator over this location with a window size of 1 and
 an increment of +1 (successive symbols from start to end).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#iterator(int,%20int)">iterator(int, int)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Create a location iterator over this location, 
 using specified window size and increment.</div>
</dd>
</dl>
<a name="_L_">
<!--   -->
</a>
<h2 class="title">L</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#length()">length()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Get length of range.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/GeneFeatureHelper.html#loadFastaAddGeneFeaturesFromGeneIDGFF2(java.io.File,%20java.io.File)">loadFastaAddGeneFeaturesFromGeneIDGFF2(File, File)</a></span> - Static method in class org.biojava.nbio.genome.<a href="./org/biojava/nbio/genome/GeneFeatureHelper.html" title="class in org.biojava.nbio.genome">GeneFeatureHelper</a></dt>
<dd>
<div class="block">Loads Fasta file and GFF2 feature file generated from the geneid prediction algorithm</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/GeneFeatureHelper.html#loadFastaAddGeneFeaturesFromGeneMarkGTF(java.io.File,%20java.io.File)">loadFastaAddGeneFeaturesFromGeneMarkGTF(File, File)</a></span> - Static method in class org.biojava.nbio.genome.<a href="./org/biojava/nbio/genome/GeneFeatureHelper.html" title="class in org.biojava.nbio.genome">GeneFeatureHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/GeneFeatureHelper.html#loadFastaAddGeneFeaturesFromGlimmerGFF3(java.io.File,%20java.io.File)">loadFastaAddGeneFeaturesFromGlimmerGFF3(File, File)</a></span> - Static method in class org.biojava.nbio.genome.<a href="./org/biojava/nbio/genome/GeneFeatureHelper.html" title="class in org.biojava.nbio.genome">GeneFeatureHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/GeneFeatureHelper.html#loadFastaAddGeneFeaturesFromGmodGFF3(java.io.File,%20java.io.File,%20boolean)">loadFastaAddGeneFeaturesFromGmodGFF3(File, File, boolean)</a></span> - Static method in class org.biojava.nbio.genome.<a href="./org/biojava/nbio/genome/GeneFeatureHelper.html" title="class in org.biojava.nbio.genome">GeneFeatureHelper</a></dt>
<dd>
<div class="block">Lots of variations in the ontology or descriptors that can be used in GFF3 which requires writing a custom parser to handle a GFF3 generated or used
 by a specific application.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/GeneFeatureHelper.html#loadFastaAddGeneFeaturesFromUpperCaseExonFastaFile(java.io.File,%20java.io.File,%20boolean)">loadFastaAddGeneFeaturesFromUpperCaseExonFastaFile(File, File, boolean)</a></span> - Static method in class org.biojava.nbio.genome.<a href="./org/biojava/nbio/genome/GeneFeatureHelper.html" title="class in org.biojava.nbio.genome">GeneFeatureHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html#loadFragment(long,%20int)">loadFragment(long, int)</a></span> - Method in class org.biojava.nbio.genome.parsers.twobit.<a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html" title="class in org.biojava.nbio.genome.parsers.twobit">TwoBitParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Feature.html#location()">location()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Feature.html" title="class in org.biojava.nbio.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block">Get location of feature.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureI.html#location()">location()</a></span> - Method in interface org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureI.html" title="interface in org.biojava.nbio.genome.parsers.gff">FeatureI</a></dt>
<dd>
<div class="block">Get the location of the feature.</div>
</dd>
<dt><a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff"><span class="strong">Location</span></a> - Class in <a href="./org/biojava/nbio/genome/parsers/gff/package-summary.html">org.biojava.nbio.genome.parsers.gff</a></dt>
<dd>
<div class="block">A location on a sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#Location(int,%20int)">Location(int, int)</a></span> - Constructor for class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Construct new location from coordinates.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#Location(org.biojava.nbio.genome.parsers.gff.Location)">Location(Location)</a></span> - Constructor for class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Clone other location.</div>
</dd>
<dt><a href="./org/biojava/nbio/genome/parsers/gff/LocIterator.html" title="class in org.biojava.nbio.genome.parsers.gff"><span class="strong">LocIterator</span></a> - Class in <a href="./org/biojava/nbio/genome/parsers/gff/package-summary.html">org.biojava.nbio.genome.parsers.gff</a></dt>
<dd>
<div class="block">Move a sliding window over a Location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/LocIterator.html#LocIterator(org.biojava.nbio.genome.parsers.gff.Location,%20int,%20int)">LocIterator(Location, int, int)</a></span> - Constructor for class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/LocIterator.html" title="class in org.biojava.nbio.genome.parsers.gff">LocIterator</a></dt>
<dd>
<div class="block">Construct an iterator that slides a window over a Location.</div>
</dd>
</dl>
<a name="_M_">
<!--   -->
</a>
<h2 class="title">M</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/App.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.genome.<a href="./org/biojava/nbio/genome/App.html" title="class in org.biojava.nbio.genome">App</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/GeneFeatureHelper.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.genome.<a href="./org/biojava/nbio/genome/GeneFeatureHelper.html" title="class in org.biojava.nbio.genome">GeneFeatureHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/homology/GFF3FromUniprotBlastHits.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.genome.homology.<a href="./org/biojava/nbio/genome/homology/GFF3FromUniprotBlastHits.html" title="class in org.biojava.nbio.genome.homology">GFF3FromUniprotBlastHits</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/cytoband/CytobandParser.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.genome.parsers.cytoband.<a href="./org/biojava/nbio/genome/parsers/cytoband/CytobandParser.html" title="class in org.biojava.nbio.genome.parsers.cytoband">CytobandParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/geneid/GeneIDXMLReader.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.genome.parsers.geneid.<a href="./org/biojava/nbio/genome/parsers/geneid/GeneIDXMLReader.html" title="class in org.biojava.nbio.genome.parsers.geneid">GeneIDXMLReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePositionParser.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePositionParser.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePositionParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneNamesParser.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneNamesParser.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneNamesParser</a></dt>
<dd>
<div class="block">parses a file from the genenames website</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Feature.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Feature.html" title="class in org.biojava.nbio.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>&nbsp;</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>&nbsp;</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/GeneIDGFF2Reader.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/GeneIDGFF2Reader.html" title="class in org.biojava.nbio.genome.parsers.gff">GeneIDGFF2Reader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/GeneMarkGTFReader.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/GeneMarkGTFReader.html" title="class in org.biojava.nbio.genome.parsers.gff">GeneMarkGTFReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/GFF3Reader.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/GFF3Reader.html" title="class in org.biojava.nbio.genome.parsers.gff">GFF3Reader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/GFF3Writer.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/GFF3Writer.html" title="class in org.biojava.nbio.genome.parsers.gff">GFF3Writer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>&nbsp;</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/LocIterator.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/LocIterator.html" title="class in org.biojava.nbio.genome.parsers.gff">LocIterator</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>&nbsp;</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.genome.parsers.twobit.<a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html" title="class in org.biojava.nbio.genome.parsers.twobit">TwoBitParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/query/BlastXMLQuery.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.genome.query.<a href="./org/biojava/nbio/genome/query/BlastXMLQuery.html" title="class in org.biojava.nbio.genome.query">BlastXMLQuery</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/query/OutputHitsGFF.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.genome.query.<a href="./org/biojava/nbio/genome/query/OutputHitsGFF.html" title="class in org.biojava.nbio.genome.query">OutputHitsGFF</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/uniprot/UniprotToFasta.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.genome.uniprot.<a href="./org/biojava/nbio/genome/uniprot/UniprotToFasta.html" title="class in org.biojava.nbio.genome.uniprot">UniprotToFasta</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/util/SplitFasta.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.nbio.genome.util.<a href="./org/biojava/nbio/genome/util/SplitFasta.html" title="class in org.biojava.nbio.genome.util">SplitFasta</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#minus()">minus()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Return location that is in same position on negative strand.</div>
</dd>
</dl>
<a name="_N_">
<!--   -->
</a>
<h2 class="title">N</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/LocIterator.html#next()">next()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/LocIterator.html" title="class in org.biojava.nbio.genome.parsers.gff">LocIterator</a></dt>
<dd>
<div class="block">Get next window of default size, then increment position by default amount.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/LocIterator.html#next(int,%20int)">next(int, int)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/LocIterator.html" title="class in org.biojava.nbio.genome.parsers.gff">LocIterator</a></dt>
<dd>
<div class="block">Get next window of specified size, then increment position by specified amount.</div>
</dd>
</dl>
<a name="_O_">
<!--   -->
</a>
<h2 class="title">O</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html#omitOverlapping(java.lang.String,%20org.biojava.nbio.genome.parsers.gff.Location,%20boolean)">omitOverlapping(String, Location, boolean)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Create a list of all features that do not overlap the specified location on the specified sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#opposite()">opposite()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Return location that is in same position on opposite strand.</div>
</dd>
<dt><a href="./org/biojava/nbio/genome/package-summary.html">org.biojava.nbio.genome</a> - package org.biojava.nbio.genome</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/genome/homology/package-summary.html">org.biojava.nbio.genome.homology</a> - package org.biojava.nbio.genome.homology</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/genome/parsers/cytoband/package-summary.html">org.biojava.nbio.genome.parsers.cytoband</a> - package org.biojava.nbio.genome.parsers.cytoband</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/genome/parsers/geneid/package-summary.html">org.biojava.nbio.genome.parsers.geneid</a> - package org.biojava.nbio.genome.parsers.geneid</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/genome/parsers/genename/package-summary.html">org.biojava.nbio.genome.parsers.genename</a> - package org.biojava.nbio.genome.parsers.genename</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/genome/parsers/gff/package-summary.html">org.biojava.nbio.genome.parsers.gff</a> - package org.biojava.nbio.genome.parsers.gff</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/genome/parsers/twobit/package-summary.html">org.biojava.nbio.genome.parsers.twobit</a> - package org.biojava.nbio.genome.parsers.twobit</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/genome/query/package-summary.html">org.biojava.nbio.genome.query</a> - package org.biojava.nbio.genome.query</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/genome/uniprot/package-summary.html">org.biojava.nbio.genome.uniprot</a> - package org.biojava.nbio.genome.uniprot</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/genome/util/package-summary.html">org.biojava.nbio.genome.util</a> - package org.biojava.nbio.genome.util</dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/GeneFeatureHelper.html#outputFastaSequenceLengthGFF3(java.io.File,%20java.io.File)">outputFastaSequenceLengthGFF3(File, File)</a></span> - Static method in class org.biojava.nbio.genome.<a href="./org/biojava/nbio/genome/GeneFeatureHelper.html" title="class in org.biojava.nbio.genome">GeneFeatureHelper</a></dt>
<dd>
<div class="block">Output a gff3 feature file that will give the length of each scaffold/chromosome in the fasta file.</div>
</dd>
<dt><a href="./org/biojava/nbio/genome/query/OutputHitsGFF.html" title="class in org.biojava.nbio.genome.query"><span class="strong">OutputHitsGFF</span></a> - Class in <a href="./org/biojava/nbio/genome/query/package-summary.html">org.biojava.nbio.genome.query</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/query/OutputHitsGFF.html#OutputHitsGFF()">OutputHitsGFF()</a></span> - Constructor for class org.biojava.nbio.genome.query.<a href="./org/biojava/nbio/genome/query/OutputHitsGFF.html" title="class in org.biojava.nbio.genome.query">OutputHitsGFF</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#overlaps(org.biojava.nbio.genome.parsers.gff.Location)">overlaps(Location)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Check if this location and other location overlap.</div>
</dd>
</dl>
<a name="_P_">
<!--   -->
</a>
<h2 class="title">P</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#percentOverlap(org.biojava.nbio.genome.parsers.gff.Location)">percentOverlap(Location)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Return percent overlap of two locations.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#plus()">plus()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Return location that is in same position on plus strand.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#prefix(int)">prefix(int)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">The part of this location before the specified position.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#prefix(org.biojava.nbio.genome.parsers.gff.Location)">prefix(Location)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">The part of this location before the other location (not inclusive).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html#printFastaSequence()">printFastaSequence()</a></span> - Method in class org.biojava.nbio.genome.parsers.twobit.<a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html" title="class in org.biojava.nbio.genome.parsers.twobit">TwoBitParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html#printFastaSequence(long)">printFastaSequence(long)</a></span> - Method in class org.biojava.nbio.genome.parsers.twobit.<a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html" title="class in org.biojava.nbio.genome.parsers.twobit">TwoBitParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/homology/GFF3FromUniprotBlastHits.html#process(java.io.File,%20double,%20java.util.LinkedHashMap,%20java.io.OutputStream)">process(File, double, LinkedHashMap&lt;String, GeneSequence&gt;, OutputStream)</a></span> - Method in class org.biojava.nbio.genome.homology.<a href="./org/biojava/nbio/genome/homology/GFF3FromUniprotBlastHits.html" title="class in org.biojava.nbio.genome.homology">GFF3FromUniprotBlastHits</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/homology/GFF3FromUniprotBlastHits.html#process(java.util.LinkedHashMap,%20java.util.LinkedHashMap,%20java.io.OutputStream)">process(LinkedHashMap&lt;String, ArrayList&lt;String&gt;&gt;, LinkedHashMap&lt;String, GeneSequence&gt;, OutputStream)</a></span> - Method in class org.biojava.nbio.genome.homology.<a href="./org/biojava/nbio/genome/homology/GFF3FromUniprotBlastHits.html" title="class in org.biojava.nbio.genome.homology">GFF3FromUniprotBlastHits</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/query/OutputHitsGFF.html#process(java.io.File,%20java.io.File,%20java.io.File,%20double,%20double,%20boolean,%20boolean)">process(File, File, File, double, double, boolean, boolean)</a></span> - Method in class org.biojava.nbio.genome.query.<a href="./org/biojava/nbio/genome/query/OutputHitsGFF.html" title="class in org.biojava.nbio.genome.query">OutputHitsGFF</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/uniprot/UniprotToFasta.html#process(java.lang.String,%20java.lang.String)">process(String, String)</a></span> - Method in class org.biojava.nbio.genome.uniprot.<a href="./org/biojava/nbio/genome/uniprot/UniprotToFasta.html" title="class in org.biojava.nbio.genome.uniprot">UniprotToFasta</a></dt>
<dd>
<div class="block">Convert a Uniprot sequence file to a fasta file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/util/SplitFasta.html#processNucleotides(java.io.File,%20java.lang.String,%20java.io.File)">processNucleotides(File, String, File)</a></span> - Method in class org.biojava.nbio.genome.util.<a href="./org/biojava/nbio/genome/util/SplitFasta.html" title="class in org.biojava.nbio.genome.util">SplitFasta</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_R_">
<!--   -->
</a>
<h2 class="title">R</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/GeneIDGFF2Reader.html#read(java.lang.String)">read(String)</a></span> - Static method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/GeneIDGFF2Reader.html" title="class in org.biojava.nbio.genome.parsers.gff">GeneIDGFF2Reader</a></dt>
<dd>
<div class="block">Read a file into a FeatureList.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/GeneMarkGTFReader.html#read(java.lang.String)">read(String)</a></span> - Static method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/GeneMarkGTFReader.html" title="class in org.biojava.nbio.genome.parsers.gff">GeneMarkGTFReader</a></dt>
<dd>
<div class="block">Read a file into a FeatureList.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/GFF3Reader.html#read(java.lang.String,%20java.util.List)">read(String, List&lt;String&gt;)</a></span> - Static method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/GFF3Reader.html" title="class in org.biojava.nbio.genome.parsers.gff">GFF3Reader</a></dt>
<dd>
<div class="block">Read a file into a FeatureList.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/GFF3Reader.html#read(java.lang.String)">read(String)</a></span> - Static method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/GFF3Reader.html" title="class in org.biojava.nbio.genome.parsers.gff">GFF3Reader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html#read()">read()</a></span> - Method in class org.biojava.nbio.genome.parsers.twobit.<a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html" title="class in org.biojava.nbio.genome.parsers.twobit">TwoBitParser</a></dt>
<dd>
<div class="block">Method reads 1 nucleotide from sequence stream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/LocIterator.html#remainder()">remainder()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/LocIterator.html" title="class in org.biojava.nbio.genome.parsers.gff">LocIterator</a></dt>
<dd>
<div class="block">Get portion of bounding location that has not yet been retrieved by next() method.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/LocIterator.html#remove()">remove()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/LocIterator.html" title="class in org.biojava.nbio.genome.parsers.gff">LocIterator</a></dt>
<dd>
<div class="block">Unsupported.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html#reset()">reset()</a></span> - Method in class org.biojava.nbio.genome.parsers.twobit.<a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html" title="class in org.biojava.nbio.genome.parsers.twobit">TwoBitParser</a></dt>
<dd>
<div class="block">Method resets current position to the begining of sequence stream.</div>
</dd>
</dl>
<a name="_S_">
<!--   -->
</a>
<h2 class="title">S</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Feature.html#score()">score()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Feature.html" title="class in org.biojava.nbio.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block">Get score.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html#selectByAttribute(java.lang.String,%20java.lang.String)">selectByAttribute(String, String)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Create a list of all features that include the specified attribute key/value pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html#selectByAttribute(java.lang.String)">selectByAttribute(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Create a list of all features that include the specified attribute key.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html#selectByGroup(java.lang.String)">selectByGroup(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Create a list of all features that have the specified group id, as defined by
 the group() method of the features.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html#selectByType(java.lang.String)">selectByType(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Create a list of all features that are of the specified type, as defined by
 the type() method of the features.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html#selectByUserData(java.lang.String,%20java.lang.Object)">selectByUserData(String, Object)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Create a list of all features that include the specified key/value pair in their userMap().</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html#selectByUserData(java.lang.String)">selectByUserData(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Create a list of all features that include the specified key in their userMap().</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html#selectOverlapping(java.lang.String,%20org.biojava.nbio.genome.parsers.gff.Location,%20boolean)">selectOverlapping(String, Location, boolean)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Create a list of all features that overlap the specified location on the specified
 sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Feature.html#seqname()">seqname()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Feature.html" title="class in org.biojava.nbio.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block">Get the sequence name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureI.html#seqname()">seqname()</a></span> - Method in interface org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureI.html" title="interface in org.biojava.nbio.genome.parsers.gff">FeatureI</a></dt>
<dd>
<div class="block">Get the sequence name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#setAccessionNr(java.lang.String)">setAccessionNr(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#setApprovedName(java.lang.String)">setApprovedName(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#setApprovedSymbol(java.lang.String)">setApprovedSymbol(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#setCdsEnd(java.lang.Integer)">setCdsEnd(Integer)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#setCdsStart(java.lang.Integer)">setCdsStart(Integer)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html#setChromosome(java.lang.String)">setChromosome(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.cytoband.<a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html" title="class in org.biojava.nbio.genome.parsers.cytoband">Cytoband</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#setChromosome(java.lang.String)">setChromosome(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#setChromosome(java.lang.String)">setChromosome(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html#setCurrentSequence(java.lang.String)">setCurrentSequence(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.twobit.<a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html" title="class in org.biojava.nbio.genome.parsers.twobit">TwoBitParser</a></dt>
<dd>
<div class="block">Method open nucleotide stream for sequence with given name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html#setCurrentSequencePosition(long)">setCurrentSequencePosition(long)</a></span> - Method in class org.biojava.nbio.genome.parsers.twobit.<a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html" title="class in org.biojava.nbio.genome.parsers.twobit">TwoBitParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html#setEnd(java.lang.Integer)">setEnd(Integer)</a></span> - Method in class org.biojava.nbio.genome.parsers.cytoband.<a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html" title="class in org.biojava.nbio.genome.parsers.cytoband">Cytoband</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#setEnsemblGeneId(java.lang.String)">setEnsemblGeneId(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#setExonCount(int)">setExonCount(int)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#setExonEnds(java.util.List)">setExonEnds(List&lt;Integer&gt;)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#setExonStarts(java.util.List)">setExonStarts(List&lt;Integer&gt;)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#setGenebankId(java.lang.String)">setGenebankId(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#setGeneName(java.lang.String)">setGeneName(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#setHgncId(java.lang.String)">setHgncId(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html#setLocus(java.lang.String)">setLocus(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.cytoband.<a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html" title="class in org.biojava.nbio.genome.parsers.cytoband">Cytoband</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#setOmimId(java.lang.String)">setOmimId(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#setOrientation(java.lang.Character)">setOrientation(Character)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#setPreviousNames(java.lang.String)">setPreviousNames(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#setPreviousSymbols(java.lang.String)">setPreviousSymbols(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#setRefseqIds(java.lang.String)">setRefseqIds(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html#setStart(java.lang.Integer)">setStart(Integer)</a></span> - Method in class org.biojava.nbio.genome.parsers.cytoband.<a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html" title="class in org.biojava.nbio.genome.parsers.cytoband">Cytoband</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#setStatus(java.lang.String)">setStatus(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#setSynonyms(java.lang.String)">setSynonyms(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#setTranscriptionEnd(java.lang.Integer)">setTranscriptionEnd(Integer)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#setTranscriptionStart(java.lang.Integer)">setTranscriptionStart(Integer)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html#setType(org.biojava.nbio.genome.parsers.cytoband.StainType)">setType(StainType)</a></span> - Method in class org.biojava.nbio.genome.parsers.cytoband.<a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html" title="class in org.biojava.nbio.genome.parsers.cytoband">Cytoband</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#setUniprot(java.lang.String)">setUniprot(String)</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html#skip(long)">skip(long)</a></span> - Method in class org.biojava.nbio.genome.parsers.twobit.<a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html" title="class in org.biojava.nbio.genome.parsers.twobit">TwoBitParser</a></dt>
<dd>
<div class="block">Method skips n nucleotides in sequence stream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html#sortByStart()">sortByStart()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Create a new list that is ordered by the starting index of the features' locations.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Feature.html#source()">source()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Feature.html" title="class in org.biojava.nbio.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block">Get source (aka method).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html#splice(org.biojava.nbio.core.sequence.DNASequence)">splice(DNASequence)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Concatenate successive portions of the specified sequence
 using the feature locations in the list.</div>
</dd>
<dt><a href="./org/biojava/nbio/genome/util/SplitFasta.html" title="class in org.biojava.nbio.genome.util"><span class="strong">SplitFasta</span></a> - Class in <a href="./org/biojava/nbio/genome/util/package-summary.html">org.biojava.nbio.genome.util</a></dt>
<dd>
<div class="block">Utility to write each Fasta entry to a unique file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/util/SplitFasta.html#SplitFasta()">SplitFasta()</a></span> - Constructor for class org.biojava.nbio.genome.util.<a href="./org/biojava/nbio/genome/util/SplitFasta.html" title="class in org.biojava.nbio.genome.util">SplitFasta</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/nbio/genome/parsers/cytoband/StainType.html" title="enum in org.biojava.nbio.genome.parsers.cytoband"><span class="strong">StainType</span></a> - Enum in <a href="./org/biojava/nbio/genome/parsers/cytoband/package-summary.html">org.biojava.nbio.genome.parsers.cytoband</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#start()">start()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Get starting index (origin 0).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#startsAfter(org.biojava.nbio.genome.parsers.gff.Location)">startsAfter(Location)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Check if this location starts after the other location starts.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#startsBefore(org.biojava.nbio.genome.parsers.gff.Location)">startsBefore(Location)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Check if this location starts before other location starts.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#suffix(int)">suffix(int)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">The part of this location after the specified position.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#suffix(org.biojava.nbio.genome.parsers.gff.Location)">suffix(Location)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">The part of this location after the other location (not inclusive).</div>
</dd>
</dl>
<a name="_T_">
<!--   -->
</a>
<h2 class="title">T</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.genome.parsers.cytoband.<a href="./org/biojava/nbio/genome/parsers/cytoband/Cytoband.html" title="class in org.biojava.nbio.genome.parsers.cytoband">Cytoband</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneChromosomePosition.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneChromosomePosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.genome.parsers.genename.<a href="./org/biojava/nbio/genome/parsers/genename/GeneName.html" title="class in org.biojava.nbio.genome.parsers.genename">GeneName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Feature.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Feature.html" title="class in org.biojava.nbio.genome.parsers.gff">Feature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureI.html#toString()">toString()</a></span> - Method in interface org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureI.html" title="interface in org.biojava.nbio.genome.parsers.gff">FeatureI</a></dt>
<dd>
<div class="block">A string representation of the feature.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureList.html" title="class in org.biojava.nbio.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Return a string representation of all features in this list.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Return a string representation of location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/LocIterator.html#toString()">toString()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/LocIterator.html" title="class in org.biojava.nbio.genome.parsers.gff">LocIterator</a></dt>
<dd>
<div class="block">Get string representation of iterator.</div>
</dd>
<dt><a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html" title="class in org.biojava.nbio.genome.parsers.twobit"><span class="strong">TwoBitParser</span></a> - Class in <a href="./org/biojava/nbio/genome/parsers/twobit/package-summary.html">org.biojava.nbio.genome.parsers.twobit</a></dt>
<dd>
<div class="block">downloaded from http://storage.bioinf.fbb.msu.ru/~roman/TwoBitParser.java

 Class is a parser of UCSC Genome Browser file format .2bit used to store
 nucleotide sequence information.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html#TwoBitParser(java.io.File)">TwoBitParser(File)</a></span> - Constructor for class org.biojava.nbio.genome.parsers.twobit.<a href="./org/biojava/nbio/genome/parsers/twobit/TwoBitParser.html" title="class in org.biojava.nbio.genome.parsers.twobit">TwoBitParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/cytoband/StainType.html#type">type</a></span> - Variable in enum org.biojava.nbio.genome.parsers.cytoband.<a href="./org/biojava/nbio/genome/parsers/cytoband/StainType.html" title="enum in org.biojava.nbio.genome.parsers.cytoband">StainType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Feature.html#type()">type()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Feature.html" title="class in org.biojava.nbio.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block">Get feature type, such as "exon" or "CDS".</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureI.html#type()">type()</a></span> - Method in interface org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureI.html" title="interface in org.biojava.nbio.genome.parsers.gff">FeatureI</a></dt>
<dd>
<div class="block">Get the feature type, for example, "exon", "CDS", etc.</div>
</dd>
</dl>
<a name="_U_">
<!--   -->
</a>
<h2 class="title">U</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#union(org.biojava.nbio.genome.parsers.gff.Location)">union(Location)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Return the union.</div>
</dd>
<dt><a href="./org/biojava/nbio/genome/uniprot/UniprotToFasta.html" title="class in org.biojava.nbio.genome.uniprot"><span class="strong">UniprotToFasta</span></a> - Class in <a href="./org/biojava/nbio/genome/uniprot/package-summary.html">org.biojava.nbio.genome.uniprot</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/uniprot/UniprotToFasta.html#UniprotToFasta()">UniprotToFasta()</a></span> - Constructor for class org.biojava.nbio.genome.uniprot.<a href="./org/biojava/nbio/genome/uniprot/UniprotToFasta.html" title="class in org.biojava.nbio.genome.uniprot">UniprotToFasta</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#upstream(int)">upstream(int)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Return the adjacent location of specified length directly upstream of this location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Feature.html#userData()">userData()</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Feature.html" title="class in org.biojava.nbio.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block">Get HashMap of user data.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/FeatureI.html#userData()">userData()</a></span> - Method in interface org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/FeatureI.html" title="interface in org.biojava.nbio.genome.parsers.gff">FeatureI</a></dt>
<dd>
<div class="block">Get HashMap of user data.</div>
</dd>
</dl>
<a name="_V_">
<!--   -->
</a>
<h2 class="title">V</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/cytoband/StainType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.nbio.genome.parsers.cytoband.<a href="./org/biojava/nbio/genome/parsers/cytoband/StainType.html" title="enum in org.biojava.nbio.genome.parsers.cytoband">StainType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/cytoband/StainType.html#values()">values()</a></span> - Static method in enum org.biojava.nbio.genome.parsers.cytoband.<a href="./org/biojava/nbio/genome/parsers/cytoband/StainType.html" title="enum in org.biojava.nbio.genome.parsers.cytoband">StainType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
</dl>
<a name="_W_">
<!--   -->
</a>
<h2 class="title">W</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/Location.html#window(int,%20int)">window(int, int)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/Location.html" title="class in org.biojava.nbio.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Enable a "sliding window" iteration over a location 
 to use with Java's "for" loop construct.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/GeneIDGFF2Reader.html#write(org.biojava.nbio.genome.parsers.gff.FeatureList,%20java.lang.String)">write(FeatureList, String)</a></span> - Static method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/GeneIDGFF2Reader.html" title="class in org.biojava.nbio.genome.parsers.gff">GeneIDGFF2Reader</a></dt>
<dd>
<div class="block">Write features in FeatureList to file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/nbio/genome/parsers/gff/GFF3Writer.html#write(java.io.OutputStream,%20java.util.LinkedHashMap)">write(OutputStream, LinkedHashMap&lt;String, ChromosomeSequence&gt;)</a></span> - Method in class org.biojava.nbio.genome.parsers.gff.<a href="./org/biojava/nbio/genome/parsers/gff/GFF3Writer.html" title="class in org.biojava.nbio.genome.parsers.gff">GFF3Writer</a></dt>
<dd>
<div class="block">Output gff3 format for a DNA Sequence</div>
</dd>
</dl>
<a href="#_A_">A</a>&nbsp;<a href="#_B_">B</a>&nbsp;<a href="#_C_">C</a>&nbsp;<a href="#_D_">D</a>&nbsp;<a href="#_E_">E</a>&nbsp;<a href="#_F_">F</a>&nbsp;<a href="#_G_">G</a>&nbsp;<a href="#_H_">H</a>&nbsp;<a href="#_I_">I</a>&nbsp;<a href="#_L_">L</a>&nbsp;<a href="#_M_">M</a>&nbsp;<a href="#_N_">N</a>&nbsp;<a href="#_O_">O</a>&nbsp;<a href="#_P_">P</a>&nbsp;<a href="#_R_">R</a>&nbsp;<a href="#_S_">S</a>&nbsp;<a href="#_T_">T</a>&nbsp;<a href="#_U_">U</a>&nbsp;<a href="#_V_">V</a>&nbsp;<a href="#_W_">W</a>&nbsp;</div>
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar_bottom">
<!--   -->
</a><a href="#skip-navbar_bottom" title="Skip navigation links"></a><a name="navbar_bottom_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="./overview-summary.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li>Use</li>
<li><a href="./overview-tree.html">Tree</a></li>
<li><a href="./deprecated-list.html">Deprecated</a></li>
<li class="navBarCell1Rev">Index</li>
<li><a href="./help-doc.html">Help</a></li>
</ul>
<div class="aboutLanguage"><em><script src="http://www.google-analytics.com/urchin.js" 							type="text/javascript"></script><script 							type="text/javascript">_uacct = 							"UA-1326640-1";urchinTracker();</script></em></div>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="./index.html?index-all.html" target="_top">Frames</a></li>
<li><a href="index-all.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="./allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
<p class="legalCopy"><small>Copyright &#169; 2000-2015 <a href="http://www.biojava.org">BioJava</a>. All Rights Reserved.</small></p>
</body>
</html>
